



<!DOCTYPE html>
<html lang="en">
  
  <head>
    
      <meta charset="utf-8">
      <title>Bokeh Plot</title>
      
      
        
          
        
        
          
        <script type="text/javascript" src="https://cdn.bokeh.org/bokeh/release/bokeh-2.0.1.min.js" integrity="sha384-JpP8FXbgAZLkfur7LiK3j9AGBhHNIvF742meBJrjO2ShJDhCG2I1uVvW+0DUtrmc" crossorigin="anonymous"></script>
        <script type="text/javascript">
            Bokeh.set_log_level("info");
        </script>
        
      
      
    
  </head>
  
  
  <body>
    
      
        
          
          
            
              <div class="bk-root" id="72c29b4b-5dd3-4d6c-8154-5e52b5f613f3" data-root-id="1087"></div>
            
          
        
      
      
        <script type="application/json" id="1278">
          {"72ff5b54-16ef-4148-b5df-f79ee92fabad":{"roots":{"references":[{"attributes":{},"id":"1127","type":"BasicTickFormatter"},{"attributes":{},"id":"1107","type":"WheelZoomTool"},{"attributes":{},"id":"1110","type":"SaveTool"},{"attributes":{},"id":"1094","type":"LinearScale"},{"attributes":{"overlay":{"id":"1112"}},"id":"1108","type":"BoxZoomTool"},{"attributes":{"background_fill_alpha":0.6,"items":[{"id":"1133"}]},"id":"1132","type":"Legend"},{"attributes":{},"id":"1109","type":"ResetTool"},{"attributes":{},"id":"1129","type":"BasicTickFormatter"},{"attributes":{"fill_color":{"field":"desc","transform":{"id":"1086"}},"size":{"units":"screen","value":7},"x":{"field":"x"},"y":{"field":"y"}},"id":"1122","type":"Scatter"},{"attributes":{"label":{"field":"labels"},"renderers":[{"id":"1124"}]},"id":"1133","type":"LegendItem"},{"attributes":{"callback":null},"id":"1111","type":"TapTool"},{"attributes":{},"id":"1096","type":"LinearScale"},{"attributes":{},"id":"1106","type":"PanTool"},{"attributes":{"active_drag":"auto","active_inspect":"auto","active_multi":null,"active_scroll":"auto","active_tap":"auto","tools":[{"id":"1085"},{"id":"1106"},{"id":"1107"},{"id":"1108"},{"id":"1109"},{"id":"1110"},{"id":"1111"}]},"id":"1113","type":"Toolbar"},{"attributes":{},"id":"1103","type":"BasicTicker"},{"attributes":{},"id":"1099","type":"BasicTicker"},{"attributes":{"high":11,"low":0,"palette":["#1f77b4","#aec7e8","#ff7f0e","#ffbb78","#2ca02c","#98df8a","#d62728","#ff9896","#9467bd","#c5b0d5","#8c564b","#c49c94"]},"id":"1086","type":"LinearColorMapper"},{"attributes":{"axis":{"id":"1098"},"ticker":null},"id":"1101","type":"Grid"},{"attributes":{},"id":"1092","type":"DataRange1d"},{"attributes":{"fill_alpha":{"value":0.1},"fill_color":{"field":"desc","transform":{"id":"1086"}},"line_alpha":{"value":0.1},"size":{"units":"screen","value":7},"x":{"field":"x"},"y":{"field":"y"}},"id":"1123","type":"Scatter"},{"attributes":{"axis":{"id":"1102"},"dimension":1,"ticker":null},"id":"1105","type":"Grid"},{"attributes":{"below":[{"id":"1098"}],"center":[{"id":"1101"},{"id":"1105"},{"id":"1132"}],"left":[{"id":"1102"}],"plot_height":500,"plot_width":1300,"renderers":[{"id":"1124"}],"title":{"id":"1088"},"toolbar":{"id":"1113"},"toolbar_location":"above","x_range":{"id":"1090"},"x_scale":{"id":"1094"},"y_range":{"id":"1092"},"y_scale":{"id":"1096"}},"id":"1087","subtype":"Figure","type":"Plot"},{"attributes":{},"id":"1090","type":"DataRange1d"},{"attributes":{"formatter":{"id":"1129"},"ticker":{"id":"1103"}},"id":"1102","type":"LinearAxis"},{"attributes":{"text":"Clustering of the COVID-19 Literature with t-SNE and K-Means"},"id":"1088","type":"Title"},{"attributes":{"formatter":{"id":"1127"},"ticker":{"id":"1099"}},"id":"1098","type":"LinearAxis"},{"attributes":{"source":{"id":"1084"}},"id":"1125","type":"CDSView"},{"attributes":{},"id":"1140","type":"UnionRenderers"},{"attributes":{"data":{"abstract":["AbstractViruses possessing class I fusion proteins require proteolytic activation by host cell proteases to mediate fusion with the host cell membrane. The mammalian SPINT2 gene encodes a protease inhibitor that targets trypsin-like serine proteases. Here we show the protease inhibitor, SPINT2, restricts cleavage-activation efficiently for a range of influenza viruses and for human metapneumovirus (HMPV). SPINT2 treatment resulted in the cleavage and fusion inhibition of full-length influenza A/CA/04/09 (H1N1) HA, A/Aichi/68 (H3N2) HA, A/Shanghai/2/2013 (H7N9) HA and HMPV F when activated by trypsin, recombinant matriptase or KLK5. We also demonstrate that SPINT2 was able to reduce viral growth of influenza A/CA/04/09 H1N1 and A/X31 H3N2 in cell culture by inhibiting matriptase or TMPRSS2. Moreover, inhibition efficacy did not differ whether SPINT2 was added at the time of infection or 24 hours post-infection. Our data suggest that the SPINT2 inhibitor has a strong potential to serve as a novel broad-spectrum antiviral.","AbstractThe new coronavirus COVID-19, also known as SARS-CoV-2, has infected more than 300,000 patients and become a global health emergency due to the very high risk of spread and impact of COVID-19. There are no specific drugs or vaccines against COVID-19, thus effective antiviral agents are still urgently needed to combat this virus. Herein, the FEP (free energy perturbation)-based screening strategy is newly derived as a rapid protocol to accurately reposition potential agents against COVID-19 by targeting viral proteinase Mpro. Restrain energy distribution (RED) function was derived to optimize the alchemical pathway of FEP, which greatly accelerated the calculations and first made FEP possible in the virtual screening of the FDA-approved drugs database. As a result, fifteen out of twenty-five drugs validated in vitro exhibited considerable inhibitory potencies towards Mpro. Among them, the most potent Mpro inhibitor dipyridamole potentially inhibited NF-\u03baB signaling pathway and inflammatory responses, and has just finished the first round clinical trials. Our result demonstrated that the FEP-based screening showed remarkable advantages in prompting drug repositioning against COVID-19.","Arrays of regularly spaced nucleosomes dominate chromatin and are often phased by alignment to reference sites like active promoters. How the distances between nucleosomes (spacing), and between phasing sites and nucleosomes are determined remains unclear, and specifically, how ATP dependent chromatin remodelers impact these features. Here, we used genome-wide reconstitution to probe how Saccharomyces cerevisiae ATP dependent remodelers generate phased arrays of regularly spaced nucleosomes. We find that remodelers bear a functional element named the \u2018ruler\u2019 that determines spacing and phasing in a remodeler-specific way. We use structure-based mutagenesis to identify and tune the ruler element residing in the Nhp10 and Arp8 modules of the INO80 remodeler complex. Generally, we propose that a remodeler ruler regulates nucleosome sliding direction bias in response to (epi)genetic information. This finally conceptualizes how remodeler-mediated nucleosome dynamics determine stable steady-state nucleosome positioning relative to other nucleosomes, DNA bound factors, DNA ends and DNA sequence elements.","AbstractIn the current COVID-19 pandemic context, proposing and validating effective treatments represents a major challenge. However, the lack of biologically relevant pre-clinical experimental models of SARS-CoV-2 infection as a complement of classic cell lines represents a major barrier for scientific and medical progress. Here, we advantageously used human reconstituted airway epithelial models of nasal or bronchial origin to characterize viral infection kinetics, tissue-level remodeling of the cellular ultrastructure and transcriptional immune signatures induced by SARS-CoV-2. Our results underline the relevance of this model for the preclinical evaluation of antiviral candidates. Foremost, we provide evidence on the antiviral efficacy of remdesivir and the therapeutic potential of the remdesivir-diltiazem combination as a rapidly available option to respond to the current unmet medical need imposed by COVID-19.One Sentence SummaryNew insights on SARS-CoV-2 biology and drug combination therapies against COVID-19.","SARS-CoV-2 is a newly emerged coronavirus responsible for the current COVID-19 pandemic that has resulted in more than one million infections and 73,000 deaths1,2. Vaccine and therapeutic discovery efforts are paramount to curb the pandemic spread of this zoonotic virus. The SARS-CoV-2 spike (S) glycoprotein promotes entry into host cells and is the main target of neutralizing antibodies. Here we describe multiple monoclonal antibodies targeting SARS-CoV-2 S identified from memory B cells of a SARS survivor infected in 2003. One antibody, named S309, potently neutralizes SARS-CoV-2 and SARS-CoV pseudoviruses as well as authentic SARS-CoV-2 by engaging the S receptor-binding domain. Using cryo-electron microscopy and binding assays, we show that S309 recognizes a glycan-containing epitope that is conserved within the sarbecovirus subgenus, without competing with receptor attachment. Antibody cocktails including S309 along with other antibodies identified here further enhanced SARS-CoV-2 neutralization and may limit the emergence of neutralization-escape mutants. These results pave the way for using S309 and S309-containing antibody cocktails for prophylaxis in individuals at high risk of exposure or as a post-exposure therapy to limit or treat severe disease.","ABSTRACTMotivationRecombinant protein production is a widely used technique in the biotechnology and biomedical industries, yet only a quarter of target proteins are soluble and can therefore be purified.ResultsWe have discovered that global structural flexibility, which can be modeled by normalised B-factors, accurately predicts the solubility of 12,216 recombinant proteins expressed in Escherichia coli. We have optimised B-factors, and derived a new set of values for solubility scoring that further improves prediction accuracy. We call this new predictor the \u2018Solubility-Weighted Index\u2019 (SWI). Importantly, SWI outperforms many existing protein solubility prediction tools. Furthermore, we have developed \u2018SoDoPE\u2019 (Soluble Domain for Protein Expression), a web interface that allows users to choose a protein region of interest for predicting and maximising both protein expression and solubility.AvailabilityThe SoDoPE web server and source code are freely available at https://tisigner.com/sodope and https://github.com/Gardner-BinfLab/TISIGNER-ReactJS, respectively. The code and data for reproducing our analysis can be found at https://github.com/Gardner-BinfLab/SoDoPE_paper2020.","AbstractUntargeted accurate strain-level classification of a priori unidentified organisms using tandem mass spectrometry is a challenging task. Reference databases often lack taxonomic depth, limiting peptide assignments to the species level. However, the extension with detailed strain information increases runtime and decreases statistical power. In addition, larger databases contain a higher number of similar proteomes.We present TaxIt, an iterative workflow to address the increasing search space required for MS/MS-based strain-level classification of samples with unknown taxonomic origin. TaxIt first applies reference sequence data for initial identification of species candidates, followed by automated acquisition of relevant strain sequences for low level classification. Furthermore, proteome similarities resulting in ambiguous taxonomic assignments are addressed with an abundance weighting strategy to improve candidate confidence.We apply our iterative workflow on several samples of bacterial and viral origin. In comparison to non-iterative approaches using unique peptides or advanced abundance correction, TaxIt identifies microbial strains correctly in all examples presented (with one tie), thereby demonstrating the potential for untargeted and deeper taxonomic classification. TaxIt makes extensive use of public, unrestricted and continuously growing sequence resources such as the NCBI databases and is available under open-source license at https://gitlab.com/rki_bioinformatics.","A recent outbreak of novel coronavirus (SARS-CoV-2), the causative agent of COVID-19, has spread rapidly all over the world. Human immunodeficiency virus (HIV) is another deadly virus and causes acquired immunodeficiency syndrome (AIDS). Rapid and early detection of these viruses will facilitate early intervention and reduce disease transmission risk. Here, we present an All-In-One Dual CRISPR-Cas12a (termed \"AIOD-CRISPR\") assay method for simple, rapid, ultrasensitive, one-pot, and visual detection of coronavirus SARS-CoV-2 and HIV virus. In our AIOD CRISPR assay, a pair of crRNAs was introduced to initiate dual CRISPR-Cas12a detection and improve detection sensitivity. The AIOD-CRISPR assay system was successfully utilized to detect nucleic acids (DNA and RNA) of SARS-CoV-2 and HIV with a sensitivity of few copies. Also, it was evaluated by detecting HIV-1 RNA extracted from human plasma samples, achieving a comparable sensitivity with real-time RT-PCR method. Thus, our method has a great potential for developing next-generation point-of-care molecular diagnostics.","AbstractMotivationIn viruses, post-translational modifications (PTMs) are essential for their life cycle. Recognizing viral PTMs is very important for better understanding the mechanism of viral infections and finding potential drug targets. However, a database of comprehensive PTM information on viruses is not available.ResultsTo fill this gap, we developed a viral post-translational modification database (VPTMdb) for collecting systematic information of viral PTM data. The VPTMdb contains 912 PTM sites that integrate 414 experimental-confirmed PTM sites with 98 proteins in 45 human viruses manually extracted from 162 publications and 498 PTMs extracted from UniProtKB/Swiss-Prot. Moreover, we created a novel sequence-based classifier named VPTMpre to help users predict viral protein phosphorylated sites. Finally, an online web server was implemented for users to download viral PTM data and predict phosphorylated sites of interest.Availability and ImplementationVPTMdb can be accessed at http://vptmdb.com:8787/VPTMdb/Contactzhaoll@ioz.ac.cn or zouquan@nclab.net","AbstractBats are unique among mammals due to the ability of powered flight and exceptional longevity. They are also asymptomatic hosts for numerous viruses, including recently emerged zoonotic Henipaviruses Nipah and Hendra, which are highly pathogenic for humans and other mammals. Better understanding of how bats control viral infection requires development of relevant permissive cellular experimental models. By applying a somatic reprogramming protocol to Pteropus bat primary cells, using a novel combination of ESRRB, CDX2, and c-MYC transcription factors, we generated bat reprogrammed cells exhibiting stem cell-like characteristics and a neural stem cell-like molecular signature. These cells present a unique interferon-stimulated transcriptomic signature and both produce and respond to interferon type-I, highlighting differences between stem cells from bats and other mammals. In contrast to primary bat cells, these reprogrammed cells are highly susceptible to infection by Henipavirus, thereby enabling isolation of new bat viruses, study of virus-bat interactions, and better understanding of bat biology.Summary sentenceSomatic reprogramming provides new bat stem cells with unique immune properties and original viral permissivness","AbstractRNA viruses are capable of rapid host shifting, typically due to a point mutation that confers expanded host range. As additional point mutations are necessary for further expansions, epistasis among host range mutations can potentially affect the mutational neighborhood and frequency of niche expansion. We mapped the mutational neighborhood of host range expansion using three genotypes of the dsRNA bacteriophage phi6 (wildtype and two isogenic host range mutants) on the novel host Pseudomonas syringae pv. atrofaciens (PA). Sanger sequencing of fifty PA mutant clones for each genotype and population Illumina sequencing both revealed the same high frequency mutations allowing infection of PA. Wildtype phi6 had at least nine different ways of mutating to enter the novel host, eight of which are in p3 (host attachment protein gene), and 13/50 clones had unchanged p3 genes. However, the two isogenic mutants had dramatically restricted neighborhoods: only one or two mutations, all in p3. Deep sequencing revealed that wildtype clones without mutations in p3 likely had changes in p12 (morphogenic protein), a region that was not polymorphic for the two isogenic host range mutants. Sanger sequencing confirmed that 10/13 of the wildtype phi6 clones had nonsynonymous mutations in p12 and two others had point mutations in p9 and p5 \u2013 none of these genes had previously been associated with host range expansion in phi6. We demonstrate, for the first time, epistatic constraint in an RNA virus due to host range mutations themselves, which has implications for models of serial host range expansion.ImportanceRNA viruses mutate rapidly and frequently expand their host ranges to infect novel hosts, leading to serial host shifts. Using an RNA bacteriophage model system (Pseudomonas phage phi6), we studied the impact of pre-existing host range mutations on another host range expansion. Results from both clonal Sanger and Illumina sequencing show extant host range mutations dramatically narrow the neighborhood of potential host range mutations compared to wildtype phi6. This research suggests that serial host shifting viruses may follow a small number of molecular paths to enter additional novel hosts. We also identified new genes involved in phi6 host range expansion, expanding our knowledge of this important model system in experimental evolution.","The COVID-19 pandemic has resulted in increased need for diagnostic testing using reverse transcriptase real-time PCR (RT-PCR). An exponential increase in demand has resulted in a shortage of numerous reagents in particular those associated with the lysis buffer required to extract the viral RNA. Herein, we describe a rapid collective effort by hospital laboratory scientists, academic researchers and the biopharma industry to generate a validated lysis buffer. We have formulated a 4M Guanidinium thiocyanate (GITC)/ Triton X-100 Lysis buffer which provides comparable results with the recommended reagents. This buffer will ease the burden on hospital labs in their heroic efforts diagnose a large population of patients.","AbstractOver the course of the last several million years of evolution, humans likely have been plagued by hundreds or perhaps thousands of epidemics. Little is known about such ancient epidemics and a deep evolutionary perspective on current pathogenic threats is lacking. The study of past epidemics has typically been limited in temporal scope to recorded history, and in physical scope to pathogens that left sufficient DNA behind, such as Yersinia pestis during the Great Plague.Host genomes however offer an indirect way to detect ancient epidemics beyond the current temporal and physical limits. Arms races with pathogens have shaped the genomes of the hosts by driving a large number of adaptations at many genes, and these signals can be used to detect and further characterize ancient epidemics.Here, we detect the genomic footprints left by ancient viral epidemics that took place in the past ~50,000 years in the 26 human populations represented in the 1,000 Genomes Project. By using the enrichment in signals of adaptation at ~4,500 host loci that interact with specific types of viruses, we provide evidence that RNA viruses have driven a particularly large number of adaptive events across diverse human populations. These results suggest that different types of viruses may have exerted different selective pressures during human evolution. Knowledge of these past selective pressures will provide a deeper evolutionary perspective on current pathogenic threats.","NaN","AbstractIntroductionBacterial infection is the leading cause of death in children globally. Clinical algorithms to identify children who are likely to benefit from antimicrobial treatment remain suboptimal. Biomarkers that accurately identify serious bacterial infection (SBI) could improve diagnosis and clinical management. Lipocalin 2 (LCN2) and neutrophil collagenase (MMP-8) are neutrophil-derived biomarkers associated with bacterial infection.MethodsWe evaluated LCN2 and MMP-8 as candidate biomarkers in 40 healthy controls and 151 febrile children categorised confirmed SBI, probable SBI, or viral infection. The diagnostic performance of LCN2 and MMP-8 to predict SBI was estimated by the area under the receiver operating characteristic curve (AUROC) and compared to the performance of C-reactive protein (CRP).ResultsPlasma LCN2 and MMP-8 concentration were predictive of SBI. The AUROC (95% CI) for LCN2, MMP8 and CRP to predict SBI was 0.88 (0.82-0.94); 0.80 (0.72-0.87) and 0.89 (0.84-0.94), respectively. The diagnostic performance of LCN2 in combination with CRP was significantly superior to either marker alone: AUROC 0.92 (95% CI: 0.88-0.96).ConclusionLCN2 is a sensitive and specific predictor of SBI in children which could be used to improve clinical management and antimicrobial stewardship. LCN2 should be further evaluated in prospective clinical studies.","The burgeoning epidemic caused by novel coronavirus 2019 (2019-nCoV) is currently a global concern. Angiotensin-converting enzyme-2 (ACE2) is a receptor of 2019-nCoV spike 1 protein (S1) and mediates viral entry into host cells. Despite the abundance of ACE2 in small intestine, few digestive symptoms are observed in patients infected by 2019-nCoV. Herein, we investigated the interactions between ACE2 and human defensins (HDs) specifically secreted by intestinal Paneth cells. The lectin-like HD5, rather than HD6, bound ACE2 with a high affinity of 39.3 nM and weakened the subsequent recruitment of 2019-nCoV S1. The cloak of HD5 on the ligand-binding domain of ACE2 was confirmed by molecular dynamic simulation. A remarkable dose-dependent preventive effect of HD5 on 2019-nCoV S1 binding to intestinal epithelial cells was further evidenced by in vitro experiments. Our findings unmasked the innate defense function of lectin-like intestinal defensin against 2019-nCoV, which may provide new insights into the prevention and treatment of 2019-nCoV infection.","The airway epithelium is a key protective barrier whose integrity is preserved by the self-renewal and differentiation of basal progenitor cells. Epithelial cells are central to the pathogenesis of multiple lung diseases. In chronic diseases, increasing age is a principle risk factor. In acute diseases, such as COVID-19, children suffer less severe symptoms than adults and have a lower rate of mortality. Few studies have explored differences between airway epithelial cells in children and adults to explain this age dependent variation in diseases. Here, we perform bulk RNA sequencing studies in laser-capture microdissected whole epithelium, FACS-sorted basal cells and cultured basal cells, as well as in vitro cell proliferation experiments, to address the intrinsic molecular differences between paediatric and adult airway basal cells. We find that, while the cellular composition of the paediatric and adult tracheobronchial epithelium is broadly similar, in cell culture, paediatric airway epithelial cells displayed higher colony forming ability, better in vitro growth and outcompeted adult cells in competitive proliferation assays. In RNA sequencing experiments, we observed potentially important differences in airway epithelial gene expression between samples from children and adults. However, genes known to be associated with SARS-CoV-2 infection were not differentially expressed between children and adults. Our results chart cell-intrinsic differences in transcriptional profile and regenerative capacity between proximal airway epithelial cells of children and adults.","While RNA viruses thrive with massive structural and functional diversity, their genomes size variation is particularly low, ranging only from ~2-to-33 kb. Here, I present the characterization of RNA sequences corresponding to the first virus associated with Aplysia californica . Genome structure and domain architecture suggest that the identified virus is a novel member of Nidovirales . The proposed aplysia californica nido-like virus (AcNV), with a genome sequence of ca.35,906 nt, represents the longest ever recorded RNA virus yet. Phylogenetic insights indicate that AcNV clusters in a major phylloclade of unclassified invertebrate nidoviruses, Roniviridae , and Mesoniviridae . Basal branching in this emerging cluster could indicate that AcNV is a member of a novel divergent clade within Nidovirales . Further, virus RNA detection in multiple independent studies suggests that AcNV is neurotropic with a broad cell/tissue/organ tropism, supported by AcNV occurrence in diverse organs, including the first detection of a Nidovirales in single specific neurons.","ABSTRACTSchizophrenia is an idiopathic disorder that affects approximately 1% of the human population, and presents with persistent delusions, hallucinations, and disorganized behaviors. Antipsychotics are the standard pharmacological treatment for schizophrenia, but are frequently discontinued by patients due to inefficacy and/or side effects. Chronic treatment with the typical antipsychotic haloperidol causes tardive dyskinesia (TD), which manifests as involuntary and often irreversible orofacial movements in around 30% of patients. Mice treated with haloperidol develop many of the features of TD, including jaw tremors, tongue protrusions, and vacuous chewing movements (VCMs). In this study, we used genetically diverse Collaborative Cross (CC) recombinant inbred inter-cross (RIX) mice to elucidate the genetic basis of antipsychotic-induced adverse drug reactions (ADRs). We performed a battery of behavioral tests in 840 mice from 73 RIX lines (derived from 62 CC strains) treated with haloperidol or placebo in order to monitor the development of ADRs. We used linear mixed models to test for strain and treatment effects. We observed highly significant strain effects for almost all behavioral measurements investigated (p&lt;0.001). Further, we observed strong strain-by-treatment interactions for most phenotypes, particularly for changes in distance traveled, vertical activity, and extrapyramidal symptoms (EPS). Estimates of overall heritability ranged from 0.21 (change in body weight) to 0.4 (VCMs and change in distance traveled) while the portion attributable to the interactions of treatment and strain ranged from 0.01 (for change in body weight) to 0.15 (for change in EPS). Interestingly, close to 30% of RIX mice exhibited VCMs, a sensitivity to haloperidol exposure, approximately similar to the rate of TD in humans chronically exposed to haloperidol. Understanding the genetic basis for the susceptibility to antipsychotic ADRs may be possible in mouse, and extrapolation to humans could lead to safer therapeutic approaches for schizophrenia.","AbstractArtificial nanoparticles accumulate a protein corona layer in biological fluids, which significantly influences their bioactivity. As nanosized obligate intracellular parasites, viruses share many biophysical properties with artificial nanoparticles in extracellular environments and here we show that respiratory syncytial virus (RSV) and herpes simplex virus 1 (HSV-1) accumulate a rich and distinctive protein corona in different biological fluids. Moreover, we show that corona pre-coating differentially affects viral infectivity and immune cell activation. Additionally, we demonstrate that viruses bind amyloidogenic peptides in their corona and catalyze amyloid formation via surface-assisted heterogeneous nucleation. Importantly, we show that HSV-1 catalyzes the aggregation of the amyloid beta peptide (A\u03b242), a major constituent of amyloid plaques in Alzheimer\u2019s disease, in-vitro and in animal models. Our results highlight the viral protein corona as an acquired structural layer that is critical for viral-host interactions and illustrate a mechanistic convergence between viral and amyloid pathologies.","ABSTRACTVaccine efficacy against susceptibility to infection (VES), regardless of symptoms, is an important endpoint of vaccine trials for pathogens with a high proportion of asymptomatic infection, as such infections may contribute to onward transmission and outcomes such as Congenital Zika Syndrome. However, estimating VESis resource-intensive. We aim to identify methods to accurately estimate VEswhen limited information is available and resources are constrained. We model an individually randomized vaccine trial by generating a network of individuals and simulating an epidemic. The disease natural history follows a Susceptible, Exposed, Infectious and Symptomatic or Infectious and Asymptomatic, Recovered model. We then use seven approaches to estimate VES, and we also estimate vaccine efficacy against progression to symptoms (VEP). A corrected relative risk and an interval censored Cox model accurately estimate VESand only require serologic testing of participants once, while a Cox model using only symptomatic infections returns biased estimates. Only acquiring serological endpoints in a 10% sample and imputing the remaining infection statuses yields unbiased VESestimates across values of R0and accurate estimates of VEPfor higher values. Identifying resource-preserving methods for accurately estimating VESis important in designing trials for diseases with a high proportion of asymptomatic infection.","AbstractMost fungal viruses have been identified in plant pathogens, whereas the presence of viral particles in human pathogenic fungi is less well studied. In the present study, we observed extrachromosomal double-stranded RNA (dsRNA) segments in various clinical isolates of Malassezia species. Malassezia is the most dominant fungal genus on the human skin surface, and species in this group are considered to be etiological factors in various skin diseases including dandruff, seborrheic dermatitis, and atopic dermatitis. We identified novel dsRNA segments and our sequencing results revealed that the virus, named MrV40, belongs to the Totiviridae family and contains an additional satellite dsRNA segment encoding a novel protein. The transcriptome of virus-infected M. restricta cells was compared to that of virus-free cells, and the results showed that transcripts involved in ribosomal biosynthesis were down regulated and those involved in energy production and programmed cell death were increased in abundance. Moreover, transmission electron microscopy revealed significantly larger vacuoles for virus-infected M. restricta cells, indicating that MrV40 infection dramatically altered M. restricta physiology. Our analysis also revealed that a viral nucleic acid from MrV40 induces a TLR3-mediated inflammatory immune response in bone marrow-derived dendritic cells (BMDCs) and this result suggests that a viral element contributes to the pathogenesis of Malassezia.ImportanceMalassezia is the most dominant fungal genus on the human skin surface and is associated with various skin diseases including dandruff and seborrheic dermatitis. Among Malassezia species, M. restricta is the most widely observed species on the human skin. In the current study, we identified a novel dsRNA virus, named MrV40, in M. restricta and characterized the sequences and structure of the viral genome along with an independent satellite dsRNA viral segment. Moreover, we found altered expression of genes involved in ribosomal synthesis and programmed cell death, indicating that virus infection altered the physiology of the fungal host cells. Our data also showed that the viral nucleic acid from MrV40 induces a TLR3-mediated inflammatory immune response in bone marrow-derived dendritic cells (BMDCs), indicating that a viral element likely contributes to the pathogenesis of Malassezia. This is the first study to identify and characterize a novel mycovirus in Malassezia.","ABSTRACTMicroglia are the resident myeloid cells in the central nervous system (CNS). The majority of microglial population relies on Csf1r signaling for survival and maintenance. However, a small subset of microglia in the murine brain can survive without Csf1r signaling, and reestablishes homeostasis after Csf1r signaling returns. Using single-cell RNA-seq, we characterized the heterogeneous microglial populations under Csf1r inhibition, including microglia lacking homeostatic markers and populations with elevated markers of monocytes, granulocytes and dendritic cells. Importantly, Mac2 is distinctively expressed in a subset of Csf1r-independent microglia cells, which were highly proliferative and shared striking similarities with those of microglial progenitors in yolk sac and early embryos. Lineage-tracing revealed that the Mac2+ population is of microglial origin and does not come from periphery monocytes. In non-treated mouse brains, Mac2+ microglia exhibited progenitor transcriptomic signature indistinguishable from those survived csf1r inhibition, supporting Mac2+ progenitor-like cells are present among homeostatic microglia.","ABSTRACTHuman coronavirus HKU1 (HCoV-HKU1) is associated with respiratory disease and is prevalent worldwide, but in vitro model for virus replication is lacking. Interaction between the coronaviral spike (S) protein and its receptor is the major determinant of virus tissue and host specificity, but virus entry is a complex process requiring a concerted action of multiple cellular elements. Here, we show that KLK13 is required for the infection of the human respiratory epithelium and is sufficient to mediate the entry of HCoV-HKU1 to non-permissive RD cells. We also demonstrated HCoV-HKU1 S protein cleavage by KLK13 in the S1/S2 region, proving that KLK13 is the priming enzyme for this virus. Summarizing, we show for the first time that protease distribution and specificity predetermines the tissue and cell specificity of the virus and may also regulate interspecies transmission. It is also of importance that presented data may be relevant for the emerging coronaviruses, including SARS-CoV-2 and may help to understand the differences in their zoonotic potential.","AbstractNeuromodulation plays a critical role in brain function in both health and disease. New optical tools, and their validation in biological tissues, are needed that can image neuromodulation with high spatial and temporal resolution, which will add an important new dimension of information to neuroscience research. Here, we demonstrate the use of a catecholamine nanosensor with fluorescent emission in the 1000-1300 nm near-infrared window to measure dopamine transmission in ex vivo brain slices. These near-infrared catecholamine nanosensors (nIRCats) represent a broader class of nanosensors that can be synthesized from non-covalent conjugation of single wall carbon nanotubes (SWNT) with single strand oligonucleotides. We show that nIRCats can be used to detect catecholamine efflux in brain tissue driven by both electrical stimulation or optogenetic stimulation. Spatial analysis of electrically-evoked signals revealed dynamic regions of interest approximately 2 microns in size in which transients scaled with simulation intensity. Optogenetic stimulation of dopaminergic terminals produced similar transients, whereas optogenetic stimulation of glutamatergic terminals showed no effect on nIRCat signal. Bath application of nomifensine prolonged nIRCat fluorescence signal, consistent with reuptake blockade of dopamine. We further show that the chemically synthetic molecular recognition elements of nIRCats permit measurement of dopamine dynamics in the presence of dopamine receptor agonists and antagonists. These nIRCat nanosensors may be advantageous for future use because i) they do not require virus delivery, gene delivery, or protein expression, ii) their near-infrared fluorescence facilitates imaging in optically scattering brain tissue and is compatible for use in conjunction with other optical neuroscience tool sets, iii) the broad availability of unique near-infrared colors have the potential for simultaneous detection of multiple neurochemical signals, and iv) they are compatible with pharmacology. Together, these data suggest nIRCats and other nanosensors of this class can serve as versatile new optical tools to report dynamics of extracellular neuromodulation in the brain.","The recent Coronavirus Disease 2019 (COVID-19) causes an immense health crisis to global public health. The World Health Organization (WHO) declared the COVID-19 as a pandemic. The COVID-19 is the etiologic agent of a recently arose disease caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). Presently, there is no vaccine available against this emerged viral disease. Therefore, it is indeed a need of the hour to develop an effectual and safe vaccine against this decidedly pandemic disease. In the current study, we collected SARS-CoV-2 genome from Indian geographical origin against human host, further more using reverse vaccinology and immunoinformatics tools here we claim effective vaccine candidates that can be mile stone in battle against COVID19. This novel study divulged two promising antigenic peptide GVYFASTEK and NFRVQPTESIV from surface glycoproteins (protein accession no. - QIA98583.1 and QHS34546.1) of SARS-CoV-2, which were predicated to be interacted with class I and class II MHC alleles and showed up to 90% conservancy and high value of antigenicity. Subsequently, the molecular docking studies were verified molecular interaction of these prime antigenic peptides with the residues of HLA-A*11-01 allele for MHC Class I and HLA DRB1*04-01 allele for MHC class II. After vigorous analysis, these peptides were predicted to be suitable epitopes which are capable to elicit the strong cell-mediated immune response against the SARS-CoV-2. Consequences from the present study could facilitate selecting SARS-CoV-2 epitopes for vaccine production pipelines in the immediate future. This novel research will certainly pave the way for a fast, reliable and virtuous platform to provide timely countermeasure of this dangerous pandemic disease, COVID-19. Keywords: COVID-19, SARS-CoV-2, Immunoinformatics, Reverse vaccinology, Molecular docking, Epitope, Vaccine candidates.","AbstractThe role of individual case characteristics, such as symptoms or demographics, in norovirus transmissibility is poorly understood. Six nursing home norovirus outbreaks occurring in South Carolina, U.S. from 2014 to 2016 were examined. We aimed to quantify the contribution of symptoms and other case characteristics in norovirus transmission using the reproduction number (REi) as an estimate of individual case infectivity and to examine how transmission changes over the course of an outbreak. Individual estimates of REi were calculated using a maximum likelihood procedure to infer the average number of secondary cases generated by each case. The associations between case characteristics and REi were estimated using a multivariate mixed linear model. Outbreaks began with one to three index case(s) with large estimated REi\u2019s (range: 1.48 to 8.70) relative to subsequent cases. Of the 209 cases, 155 (75%) vomited, 164 (79%) had diarrhea, and 158 (76%) were nursing home residents (vs. staff). Cases who vomited infected 2.74 (95% CI: 1.90, 3.94) more individuals than non-vomiters, cases with diarrhea infected 1.62 (95% CI: 1.09, 2.41) more individuals than cases without diarrhea, and resident-cases infected 1.69 (95% CI: 1.18, 2.42) more individuals than staff-cases. Index cases tended to be residents (vs. staff) who vomited and infected considerably more secondary cases compared to non-index cases. Results suggest that individuals, particularly residents, who vomit are more infectious and tend to drive norovirus transmission in U.S. nursing home norovirus outbreaks. While diarrhea also plays a role in norovirus transmission, it is to a lesser degree than vomiting in these settings. Results lend support for prevention and control measures that focus on cases who vomit, particularly if those cases are residents.Author summaryThe majority of all norovirus outbreaks reported to the CDC occur in long-term care facilities (LTCFs), including nursing homes, where older residents are at risk for more severe or prolonged infection. Because there is currently no publicly available norovirus vaccine, sound control measures are key to controlling norovirus outbreaks, but there is little evidence that standard control measures are effective in reducing the size and/or duration of LTCF norovirus outbreaks. Hence, studies leading to a better understanding of disease spread and prevention of additional cases, and thus more effective control measures, are needed. To this end, we aimed to quantify factors associated with norovirus transmission and to examine how transmission changes over the course of an outbreak. We show that vomiting and, to a lesser extent, diarrhea are critical in initiating and sustaining norovirus transmission in U.S. nursing home norovirus outbreaks. We also show that nursing home residents, rather than staff, are the primary drivers of transmission. Results suggest that control measures focusing on cases who vomit, particularly if those cases are residents, would be most effective at curtailing norovirus transmission in these settings.","ABSTRACTMotivationLong-read sequencing technologies are invaluable for determining complex RNA transcript architectures but are error-prone. Numerous \u201chybrid correction\u201d algorithms have been developed for genomic data that correct long reads by exploiting the accuracy and depth of short reads sequenced from the same sample. These algorithms are not suited for correcting more complex transcriptome sequencing data.ResultsWe have created a novel algorithm called TALC (Transcription Aware Long Read Correction) which models changes in RNA expression and isoform representation in a weighted De-Bruijn graph to correct long reads from transcriptome studies. We show that transcription aware correction by TALC improves the accuracy of the whole spectrum of downstream RNA-seq applications and is thus necessary for transcriptome analyses that use long read technology.Availability and ImplementationTALC is implemented in C++ and available at https://gitlab.igh.cnrs.fr/lbroseus/TALC.Contactwilliam.ritchie@igh.cnrs.fr","Recent reports suggest an association between COVID-19 and altered olfactory function. Here we analyze bulk and single cell RNA-Seq datasets to identify cell types in the olfactory epithelium that express molecules that mediate infection by SARS-CoV-2 (CoV-2), the causal agent in COVID-19. We find in both mouse and human datasets that olfactory sensory neurons do not express two key genes involved in CoV-2 entry, ACE2 and TMPRSS2. In contrast, olfactory epithelial support cells and stem cells express both of these genes, as do cells in the nasal respiratory epithelium. Taken together, these findings suggest possible mechanisms through which CoV-2 infection could lead to anosmia or other forms of olfactory dysfunction.","AbstractGenetic subtyping of viruses and bacteria is a critical tool for visualizing and modeling their geographic distribution and temporal dynamics. Quantifying viral dynamics is of particular importance for the novel coronavirus responsible for COVID-19, SARS-CoV-2. Effective containment strategies and potential future therapeutic and vaccine strategies will likely require a precise and quantitative understanding of viral transmission and evolution. In this paper, we employ an entropy-based analysis to identify mutational signatures of SARS-CoV-2 strains in the GISAID database available as of April 5, 2020. Our analysis method identifies nucleotide sites within the viral genome which are highly informative of variation between the viral genomes sequenced in different individuals. These sites are used to characterize individual virus sequence with a characteristic Informative Subtype Marker (ISM). The ISMs provide signatures that can be efficiently and rapidly utilized to quantitatively trace viral dynamics through geography and time. We show that by analyzing the ISM of currently available SARS-CoV-2 sequences, we are able to profile international and interregional differences in viral subtype, and visualize the emergence of viral subtypes in different countries over time. To validate and demonstrate the utility of ISM-based subtyping: (1) We show the distinct genetic subtypes of European infections, in which early on infections are related to the viral subtypes that has become dominant in Italy followed by the development of local subtypes, (2) We distinguish subtypes associated with outbreaks in distinct parts of the United States, identify the development of a local subtype potentially due to community to transmission and distinguish it from the predominant subtype in New York, suggesting that the outbreak in New York is linked to imported cases from Europe. (3) We present results that quantitatively show the temporal behavior of the emergence of SARS-CoV-2 from localization in China to a pattern of distinct regional subtypes as the virus spreads throughout the world over time. Accordingly, we show that genetic subtyping using entropy-based ISMs can play an important complementary role to phylogenetic tree-based analysis, such as the Nextstrain [9] project, in efficiently quantifying SARS-CoV-2 dynamics to enable modeling, data-mining, and machine learning tools. Following from this initial study, we have developed a pipeline to dynamically generate ISMs for newly added SARS-CoV-2 sequences and generate updated visualization of geographical and temporal dynamics, and made it available on Github at https://github.com/EESI/ISM.","Germline variation at immunoglobulin gene (IG) loci is critical for pathogen-mediated immunity, but establishing complete reference sequences in these regions is problematic because of segmental duplications and somatically rearranged source DNA. We sequenced BAC clones from the essentially haploid hydatidiform mole, CHM1, across the light chain IG loci, kappa (IGK) and lambda (IGL), creating single haplotype representations of these regions. The IGL haplotype is 1.25Mb of contiguous sequence with four novel V gene and one novel C gene alleles and an 11.9kbp insertion. The IGK haplotype consists of two 644kbp proximal and 466kbp distal contigs separated by a gap also present in the reference genome sequence. Our effort added an additional 49kbp of unique sequence extending into this gap. The IGK haplotype contains six novel V gene and one novel J gene alleles and a 16.7kbp region with increased sequence identity between the two IGK contigs, exhibiting signatures of interlocus gene conversion. Our data facilitated the first comparison of nucleotide diversity between the light and IG heavy (IGH) chain haplotypes within a single genome, revealing a three to six fold enrichment in the IGH locus, supporting the theory that the heavy chain may be more important in determining antigenic specificity.","AbstractCOVID-19 pandemic has resulted so far 14,395,16 confirmed cases with 85,711 deaths from the 212 countries, or territories. Due to multifacet issues and challenges in implementation of the safety &amp; preventive measures, inconsistent coordination between societies-governments and most importanly lack of specific vaccine to SARS-CoV-2, the spread of Wuhan originated virus is still uprising after taking a heavy toll on human life. In the present study, we mapped several immunogenic epitopes (B-cell, T-cell, and IFN-gamma) over the entire structural proteins of SARS-CoV-2 and by applying various computational and immunoinformatics approaches, we designed a multi-epitope peptide based vaccine that predicted high immunogenic response in the largest proportion of world\u2019s human population. To ensure high expression of the recombinant vaccine in E. coli, codon optimization and in-silico cloning were also carried out. The designed vaccine with high molecular affinity to TLR3 and TLR4, was found capable to initiate effective innate and adaptive immune response. The immune simulation also suggested uprising high levels of both B-cell and T-cell mediated immunity which on subsequent exposure cleared antigen from the system. The proposed vaccine found promising by yielding desired results and hence, should be tested by practical experimentations for its functioning and efficacy to neutralize SARS-CoV-2.","Viruses need to hijack the translational machinery of the host cell for a productive infection to happen. However, given the dynamic landscape of tRNA pools among tissues, it is unclear whether different viruses infecting different tissues have adapted their codon usage toward their tropism. Here, we collect the coding sequences of over 500 human-infecting viruses and determine that tropism explains changes in codon usage. Using an in silico model of translational efficiency, we validate the correspondence of the viral codon usage with the translational machinery of their tropism. In particular, we propose that the improved translational adaptation to the upper respiratory airways of the pandemic agent SARS-CoV-2 coronavirus could enhance its transmissibility. Furthermore, this correspondence is specifically defined in early viral proteins, as upon infection cells undergo reprogramming of tRNA pools that favors the translation of late counterparts.","AbstractThe ongoing outbreak of a new coronavirus (2019-nCoV) causes an epidemic of acute respiratory syndrome in humans. 2019-nCoV rapidly spread to national regions and multiple other countries, thus, pose a serious threat to public health. Recent studies show that spike (S) proteins of 2019-nCoV and SARS-CoV may use the same host cell receptor called angiotensin-converting enzyme 2 (ACE2) for entering into host cells. The affinity between ACE2 and 2019-nCoV S is much higher than ACE2 binding to SARS-CoV S protein, explaining that why 2019-nCoV seems to be more readily transmitted from the human to human. Here, we reported that ACE2 can be significantly upregulated after infection of various viruses including SARS-CoV and MERS-CoV. Basing on findings here, we propose that coronavirus infection can positively induce its cellular entry receptor to accelerate their replication and spread, thus drugs targeting ACE2 expression may be prepared for the future emerging infectious diseases caused by this cluster of viruses.","AbstractA novel coronavirus (severe acute respiratory syndrome coronavirus 2, SARS-CoV-2) emerged in late 2019, causing an outbreak of pneumonia [coronavirus disease 2019 (COVID-19)] in Wuhan, China, which then rapidly spread globally. Although the use of ready-made reaction mixes can enable more rapid PCR-based diagnosis of COVID-19, the need to transport and store these mixes at low temperatures presents challenges to already overburdened logistics networks. Here, we present an optimized freeze-drying procedure that allows SARS-CoV-2 PCR mixes to be transported and stored at ambient temperatures, without loss of activity. Additive-supplemented PCR mixes were freeze-dried. The residual moisture of the freeze-dried PCR mixes was measured by Karl-Fischer titration. We found that freeze-dried PCR mixes with \u223c1.2% residual moisture are optimal for storage, transport, and reconstitution. The sensitivity, specificity, and repeatability of the freeze-dried reagents were similar to those of freshly prepared, wet reagents. The freeze-dried mixes retained activity at room temperature (18\u223c25\u00b0C) for 28 days, and for 14 and 10 days when stored at 37\u00b0C and 56\u00b0C, respectively. The uptake of this approach will ease logistical challenges faced by transport networks and make more cold storage space available at diagnosis and hospital laboratories. This method can also be applied to the generation of freeze-dried PCR mixes for the detection of other pathogens.","Understanding the molecular features that made SARS-CoV-2 a highly infectious virus is crucial for the development of targeted therapies and an effective vaccine. Following the recent report of a furin-like cleavage site1, unique to SARS-CoV-2, we characterized several additional residues positioned within the crown of the S glycoprotein under positive/diversifying selection along the ancestral lineage of current pandemic strains. Residue V483 in near proximity to the ACE2 binding domain, which may indeed affect the enhanced ability of currently circulating strains to infect host cells. Molecular dynamic simulations revealed long-range covariant movements with furin cleavage, correlated with pre-fusion conformation of SARS-CoV-2 S glycoprotein monomer. Residue T333 is hinge of this movement that gives further advantage to bind cell receptors facilitating fusion and entry. Evolutionary analysis revealed that current lineage is likely to have emerged from the reservoir after a recombination event between bat and pangolin ancestors. Recombination impacted residues located in the binding receptor region of S glycoprotein that are identical to pangolin ancestor. Interestingly, several recombination hotspots were detected throughout the genome with the exception of the S glycoprotein, which exhibits a coldspot in the N terminal region, likely the result of negative selection against further changes that may reduce its optimal fitness.","A group of biologists, ARTIC Network, has proposed a multiplexed PCR primer set for whole-genome analysis of the novel coronavirus, SARS-CoV-2, soon after the start of COVID-19 epidemics was realized. The primer set was adapted by many researchers worldwide and has already contributed to the high-quality and prompt genome epidemiology of this rapidly spreading viral disease. We have also seen the great performance of their primer set and protocol; the primer set amplifies all desired 98 PCR amplicons with fairly small amplification bias from clinical samples with relatively high viral load. However, we also observed an acute drop of reads derived from some amplicons especially amplicon 18 and 76 in \"pool 2\" as a sample's viral load decreases. We suspected this low coverage issue was due to dimer formation between primers targeting those two amplicons. Indeed, replacement of just one of those primers, nCoV-2019_76_RIGHT, to a newly designed primer resulted in a drastic improvement of coverages at both regions targeted by the amplicons 18 and 76. Given this result, we further replaced four primers in \"pool 1\" with each respective alternative. These modifications also improved coverage in eight amplicons particularly in samples with low viral load. The results of our experiments clearly indicate that primer dimer formation is one critical cause of coverage bias in ARTIC protocol. Importantly, some of the problematic primers are detectable by observing primer dimers in raw NGS sequence reads and replacing them with alternatives as shown in this study. We expect a continuous improvement of the ARTIC primer set will extend the limit for completion of SARS-CoV-2 genomes to samples with lower viral load, that supports better genomic epidemiology and mitigation of spread of this pathogen.","In this paper, we study the dynamics of a vector-borne disease model with two transmission paths: direct transmission through contact and indirect transmission through vector. The direct transmission is considered to be a non-monotone incidence function to describe the psychological effect of some severe diseases among the population when the number of infected hosts is large and/or the disease possesses high case fatality rate. The system has a disease-free equilibrium which is locally asymptomatically stable when the basic reproduction number (R_0) is less than unity and may have up to four endemic equilibria. Analytical expression representing the epidemic growth rate is obtained for the system. Sensitivity of the two transmission pathways were compared with respect to the epidemic growth rate. We numerically find that the direct transmission coefficient is more sensitive than the indirect transmission coefficient with respect to R_0 and the epidemic growth rate. Local stability of endemic equilibria is studied. Further, the global asymptotic stability of the endemic equilibrium is proved using Li and Muldowney geometric approach. The explicit condition for which the system undergoes backward bifurcation is obtained. The basic model also exhibits the hysteresis phenomenon which implies diseases will persist even when R_0&lt;1 although the system undergoes a forward bifurcation and this phenomenon is rarely observed in disease models. Consequently, our analysis suggests that the diseases with multiple transmission routes exhibit bi-stable dynamics. However, efficient application of temporary control in bi-stable regions will curb the disease to lower endemicity. In addition, increase in transmission heterogeneity will increase the chance of disease eradication.","SummaryBackgroundCOVID-19 caused by SARA-CoV-2 is a disaster sweeping over 200 countries, and more than 2,150,000 people are suffering from the disease and 140,000 people died. ACE2 is a receptor protein of SARS- CoV-2, and TMPRSS2 promotes virus proliferation and transmission. Some patients developed multiple organ dysfunction syndromes other than lungs. Therefore, studying the viral susceptibility of other organs is important for a deeper understanding of viral pathogenesis.MethodsThe advantage of scRNA-seq data is the identification of cell types by clustering the gene expression of cells. ACE2 and TMPRSS2 are highly expressed in AT2 of lungs, we compared the ACE2 and TMPRSS2 expression levels of cell types from 31 organs, with AT2 of lungs to evaluate the risk of the viral infection using scRNA-seq data.FindingsFor the first time, we found the brain, gall bladder, and fallopian tube are vulnerable to COVID-19 infection. Besides, the nose, heart, small intestine, large intestine, esophagus, testis and kidney are also identified to be high-risk organs with high expression levels of ACE2 and TMPRSS2. Moreover, the susceptible organs are grouped into three risk levels based on the TMPRSS2 expression. As a result, the respiratory system, digestive system and reproductive system are at the top-risk level to COVID-19 infection.InterpretationThis study provides evidence for COVID-19 infection in the human nervous system, digestive system, reproductive system, respiratory system, circulatory system and urinary system using scRNA-seq data, which helps for the clinical diagnosis and treatment of patients.","When humans indicate on which hand a tactile stimulus occurred, they often err when their hands are crossed. This finding seemingly supports the view that the automatically determined touch location in external space affects limb assignment: the crossed right hand is localized in left space, and this conflict presumably provokes hand assignment errors. Here, participants judged on which hand the first of two stimuli, presented during a bimanual movement, had occurred, and then indicated its external location by a reach-to-point movement. When participants incorrectly chose the hand stimulated second, they pointed to where that hand had been at the correct, first time point, though no stimulus had occurred at that location. This behavior suggests that stimulus localization depended on hand assignment, not vice versa. It is, thus, incompatible with the notion of automatic computation of external stimulus location upon occurrence. Instead, humans construct external touch location post-hoc and on demand.","Virus entry is a multistep process. It initiates when the virus attaches to the host cell and ends when the viral contents reach the cytosol. Genetically unrelated viruses can subvert analogous subcellular mechanisms and use similar trafficking pathways for successful entry. Antiviral strategies targeting early steps of infection are therefore appealing, particularly when the probability for successful interference through a common step is highest. We describe here potent inhibitory effects on content release and infection by chimeric VSV containing the envelope proteins of EBOV (VSV-EBOV) or SARS-CoV-2 (VSV-SARS-CoV-2) elicited by Apilimod and Vacuolin-1, small molecule inhibitors of the main endosomal Phosphatidylinositol-3-Phosphate/Phosphatidylinositol 5-Kinase, PIKfyve. We also describe potent inhibition of SARS-CoV-2 strain 2019-nCoV/USA-WA1/2020 by Apilimod. These results define new tools for studying the intracellular trafficking of pathogens elicited by inhibition of PIKfyve kinase and suggest the potential for targeting this kinase in developing a small-molecule antiviral against SARS-CoV-2.","AbstractDomain growth plays an important role in many biological systems, and so the inclusion of domain growth in models of these biological systems is important to understanding how these biological systems function. In this work we present methods to include the effects of domain growth on the evolution of spatial correlations in a continuum approximation of a lattice-based model of cell motility and proliferation. We show that, depending on the way in which domain growth is implemented, different steady-state densities are predicted for an agent population. Furthermore, we demonstrate that the way in which domain growth is implemented can result in the evolution of the agent density depending on the size of the domain. Continuum approximations that ignore spatial correlations cannot capture these behaviours, while those that account for spatial correlations do. These results will be of interest to researchers in developmental biology, as they suggest that the nature of domain growth can determine the characteristics of cell populations.","ABSTRACTZika Virus (ZIKV) is a causative agent for poor pregnancy outcome and fetal developmental abnormalities, including microcephaly and eye defects. As a result, ZIKV is now a confirmed teratogen. Understanding host-pathogen interactions, specifically cellular perturbations caused by ZIKV, can provide novel therapeutic targets. In order to complete viral replication, viral pathogens control the host cellular machineries and regulate various factors, including long noncoding RNA (lncRNA) genes, at transcriptional levels. The role of lncRNA genes in the pathogenesis of ZIKV-mediated microcephaly and eye defects is currently unknown. To gain additional insights, we focused on profiling the differentially expressed lncRNA genes during ZIKV infection in mammalian cells. For this study, we employed a contemporary clinical Zika viral isolate, PRVABC59, of Asian genotype. We utilized an unbiased RNA sequencing approach to profile the lncRNA transcriptome in ZIKV infected Vero cells. We identified a total of 121 lncRNA genes that are differentially regulated at 48 hours post-infection. The majority of these genes are independently validated by reverse-transcription qPCR. A notable observation was that the lncRNAs, MALAT1 (Metastasis Associated Lung Adenocarcinoma Transcript 1) and NEAT1 (Nuclear Paraspeckle Assembly Transcript 1), are down-regulated upon Zika viral infection. MALAT1 and NEAT1 are known as nuclear localized RNAs that regulate gene expression and cell proliferation. Protein-lncRNA interaction maps revealed that MALAT1 and NEAT1 share common interacting partners and form a larger network comprising of 71 cellular factors. ZIKV-mediated dysregulation of these two regulatory lncRNAs can alter the expression of respective target genes and associated biological functions, an important one being cell division. In conclusion, this investigation is the first to provide insight into the biological connection of lncRNAs and ZIKV which can be further explored for developing antiviral therapy and understanding fetal developmental processes.","Here we present a web interface that implements a comprehensive mechanistic model of the SARS-CoV-2 disease map in which the detailed activity of the human signaling circuits related to the viral infection and the different antiviral responses, including immune and inflammatory activities, can be inferred from gene expression experiments. Moreover, given to the mechanistic properties of the model, the effect of potential interventions, such as knock-downs, over-expression or drug effects (currently the system models the effect of more than 8000 DrugBank drugs) can be studied in specific conditions. This tool, with the holistic, systems biology approach to the understanding of the complexities of the viral infection process, will become an important asset in the search for efficient antiviral treatments. The tool is freely available at: http://hipathia.babelomics.org/covid19/.","SummarySARS-CoV-2, a member of the coronavirus family, has caused a global public health emergency.1 Based on our analysis of hepatitis C virus and coronavirus replication, and the molecular structures and activities of viral inhibitors, we previously reasoned that the FDA-approved heptatitis C drug EPCLUSA (Sofosbuvir/Velpatasvir) should inhibit coronaviruses, including SARS-CoV-2.2 Here, using model polymerase extension experiments, we demonstrate that the activated triphosphate form of Sofosbuvir is incorporated by low-fidelity polymerases and SARS-CoV RNA-dependent RNA polymerase (RdRp), and blocks further incorporation by these polymerases; the activated triphosphate form of Sofosbuvir is not incorporated by a host-like high-fidelity DNA polymerase. Using the same molecular insight, we selected two other anti-viral agents, Alovudine and AZT (an FDA approved HIV/AIDS drug) for evaluation as inhibitors of SARS-CoV RdRp. We demonstrate the ability of two HIV reverse transcriptase inhibitors, 3\u2019-fluoro-3\u2019-deoxythymidine triphosphate and 3\u2019-azido-3\u2019-deoxythymidine triphosphate (the active triphosphate forms of Alovudine and AZT), to be incorporated by SARS-CoV RdRp where they also terminate further polymerase extension. Given the 98% amino acid similarity of the SARS-CoV and SARS-CoV-2 RdRps, we expect these nucleotide analogues would also inhibit the SARS-CoV-2 polymerase. These results offer guidance to further modify these nucleotide analogues to generate more potent broad-spectrum anti-coronavirus agents.","AbstractThe emergence of a novel coronavirus, SARS-CoV-2, at the end of 2019 has resulted in widespread human infections across the globe. While genetically distinct from SARS-CoV, the etiological agent that caused an outbreak of severe acute respiratory syndrome (SARS) in 2003, both coronaviruses exhibit receptor binding domain (RBD) conservation and utilize the same host cell receptor, angiotensin-converting enzyme 2 (ACE2), for virus entry. Therefore, it will be important to test the cross-reactivity of antibodies that have been previously generated against the surface spike (S) glycoprotein of SARS-CoV in order to aid research on the newly emerged SARS-CoV-2. Here, we show that an immunogenic domain in the S2 subunit of SARS-CoV S is highly conserved in multiple strains of SARS-CoV-2. Consistently, four murine monoclonal antibodies (mAbs) raised against this immunogenic SARS-CoV fragment were able to recognise the S protein of SARS-CoV-2 expressed in a mammalian cell line. Importantly, one of them (mAb 1A9) was demonstrated to detect S in SARS-CoV-2-infected cells. To our knowledge, this is the first study showing that mAbs targeting the S2 domain of SARS-CoV can cross-react with SARS-CoV-2 and this observation is consistent with the high sequence conservation in the S2 subunit. These cross-reactive mAbs may serve as tools useful for SARS-CoV-2 research as well as for the development of diagnostic assays for its associated coronavirus disease COVID-19.","AbstractElucidating antiviral CD8 T lymphocyte responses to SARS-CoV-2 may shed light on the heterogeneity of clinical outcomes and inform vaccine or therapeutic approaches. To facilitate the evaluation of antiviral CD8 T cell responses to SARS-CoV-2, we generated a publicly accessible database of epitopes predicted to bind any class I HLA protein across the entire SARS-CoV-2 proteome. While a subset of epitopes from earlier betacoronaviruses, such as SARS-CoV (SARS), have been validated experimentally, validation systems are often biased toward specific HLA haplotypes (notably HLA-A*02:01) that only account for a fraction of the haplotypes of individuals affected by the SARS-CoV-2 pandemic. To enable evaluation of epitopes across individuals with a variety of HLA haplotypes, we computed the predicted binding affinities between 9-mer peptides derived from the annotated SARS-CoV-2 peptidome across 9,360 MHC class I HLA-A, -B, and -C alleles. There were 6,748 unique combinations of peptides and HLA alleles (pMHCs) with a predicted binding affinity of less than 500nM, including 1,103 unique peptides and 1,022 HLA alleles, spanning 11 annotated superfamilies. These peptides were derived from all 11 proteins spanning the SARS-CoV-2 peptidome, including peptides that have previously been validated experimentally. We also show evidence that these previously validated epitopes may be relevant in other HLA contexts. This complete dataset is available publicly: gs://pici-covid19-data-resources/mhci/peptide_predictions.","AbstractA novel coronavirus pneumonia initially identified in Wuhan, China and provisionally named 2019-nCoV has surged in the public. In anticipation of substantial burdens on healthcare system following this human-to-human spread, we aim to scrutinise the currently available information and evaluate the burden of healthcare systems during this outbreak in Wuhan. We applied a modified SIR model to project the actual number of infected cases and the specific burdens on isolation wards and intensive care units, given the scenarios of different diagnosis rates as well as different public health intervention efficacy. Our estimates suggest the actual number of infected cases could be much higher than the reported, with estimated 26,701 cases (as of 28th January 2020) assuming 50% diagnosis rate if no public health interventions were implemented. The estimated burdens on healthcare system could be largely reduced if at least 70% efficacy of public health intervention is achieved.","SummaryCoronaviruses such as the newly discovered virus from Wuhan, China, 2019-nCoV, and the viruses that cause SARS and MERS, have resulted in regional and global public health emergencies. Based on our molecular insight that the hepatitis C virus and the coronavirus use a similar viral genome replication mechanism, we reasoned that the FDA-approved drug EPCLUSA (Sofosbuvir/Velpatasvir) for the treatment of hepatitis C will also inhibit the above coronaviruses, including 2019-nCoV. To develop broad spectrum anti-viral agents, we further describe a novel strategy to design and synthesize viral polymerase inhibitors, by combining the ProTide Prodrug approach used in the development of Sofosbuvir with the use of 3\u2019-blocking groups that we have previously built into nucleotide analogues that function as polymerase terminators.","ABSTRACTZoonotic coronaviruses (CoVs) are significant threats to global health, as exemplified by the recent emergence of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2)1. Host immune responses to CoV are complex and regulated in part through antiviral interferons. However, the interferon-stimulated gene products that inhibit CoV are not well characterized2. Here, we show that interferon-inducible lymphocyte antigen 6 complex, locus E (LY6E) potently restricts cellular infection by multiple CoVs, including SARS-CoV, SARS-CoV-2, and Middle East respiratory syndrome coronavirus (MERS-CoV). Mechanistic studies revealed that LY6E inhibits CoV entry into cells by interfering with spike protein-mediated membrane fusion. Importantly, mice lacking Ly6e in hematopoietic cells were highly susceptible to murine CoV infection. Exacerbated viral pathogenesis in Ly6e knockout mice was accompanied by loss of hepatic and splenic immune cells and reduction in global antiviral gene pathways. Accordingly, we found that Ly6e directly protects primary B cells and dendritic cells from murine CoV infection. Our results demonstrate that LY6E is a critical antiviral immune effector that controls CoV infection and pathogenesis. These findings advance our understanding of immune-mediated control of CoV in vitro and in vivo, knowledge that could help inform strategies to combat infection by emerging CoV.","ABSTRACTTo combat the pandemic of the coronavirus disease (COVID-19), numerous governments have established phone hotlines to prescreen potential cases. These hotlines have struggled with the volume of callers, leading to wait times of hours or, even, an inability to contact health authorities. Symptoma is a symptom-to-disease digital health assistant that can differentiate more than 20,000 diseases with an accuracy of more than 90%. We tested the accuracy of Symptoma to identify COVID-19 using a set of diverse clinical cases combined with case reports of COVID-19. We showed that Symptoma can accurately distinguish COVID-19 in 96.32% of clinical cases. When considering only COVID-19 symptoms and risk factors, Symptoma identified 100% of those infected when presented with only three signs. Lastly, we showed that Symptoma\u2019s accuracy far exceeds that of simple \u201cyes-no\u201d questionnaires widely available online. In summary, Symptoma provides unparalleled accuracy in systematically identifying cases of COVID-19 while also considering over 20,000 other diseases. Furthermore, Symptoma allows free text input, furthered with disease-specific follow up questions, in 36 languages. Combined, these results and accessibility give Symptoma the potential to be a key tool in the global fight against COVID-19. The Symptoma predictor is freely available online at https://www.symptoma.com.","ABSTRACTViruses commonly antagonize innate immune pathways that are primarily driven by Nuclear Factor-\u03baB (NF-\u03baB), Interferon Regulatory Factor (IRF) and Signal Transducer and Activator of Transcription (STAT) family of transcription factors. Such a strategy allows viruses to evade immune surveillance and maximize their replication. Using an unbiased RNA-seq based approach to measure gene expression induced by transfected viral genomic RNA (vgRNA) and reovirus infection, we discovered that mammalian reovirus inhibits host cell innate immune signaling. We found that while vgRNA and reovirus infection both induce a similar IRF dependent gene expression program, gene expression driven by the NF-\u03baB family of transcription factors is lower in infected cells. Potent agonists of NF-\u03baB, such as Tumor Necrosis Factor alpha (TNF\u03b1) and vgRNA, failed to induce NF-\u03baB dependent gene expression in infected cells. We demonstrate that NF-\u03baB signaling is blocked due to loss of critical members of the Inhibitor of KappaB Kinase (IKK) complex, NF-\u03baB Essential MOdifier (NEMO) and IKK\u03b2. The loss of the IKK complex components prevents nuclear translocation and phosphorylation of NF-\u03baB, thereby preventing gene expression. Our studies demonstrate that reovirus infection selectively blocks NF-\u03baB, likely to counteract its antiviral effects and promote efficient viral replication.IMPORTANCEHost cells mount a response to curb virus replication in infected cells and prevent infection of neighboring, as yet uninfected cells. The NF-\u03baB family of proteins is important for the cell to mediate this response. In this study, we show that in cells infected with mammalian reovirus, NF-\u03baB is inactive. Further, we demonstrate that NF-\u03baB is rendered inactive because virus infection results in reduced levels of upstream intermediaries (called IKKs) that are needed for NF-\u03baB function. Based on previous evidence that active NF-\u03baB limits reovirus infection, we conclude that inactivating NF-\u03baB is a viral strategy to produce a cellular environment that is favorable for virus replication.","AbstractFaced with the current large-scale public health emergency, collecting, sorting, and analyzing biomedical information related to the \u201ccoronavirus\u201d should be done as quickly as possible to gain a global perspective, which is a basic requirement for strengthening epidemic control capacity. However, for human researchers studying the viruses and the hosts, the vast amount of information available cannot be processed effectively and in a timely manner, particularly when the scientific understanding may be limited, which can further lower the information processing efficiency. We present TWIRLS, a method that can automatically acquire, organize, and classify information. Additionally, independent functional data sources can be added to build an inference system using a machine-based approach, which can provide relevant knowledge to help human researchers quickly establish subject cognition and to make more effective decisions. TWIRLS can automatically analyze more than three million words in more than 14,000 literature articles in only 4 hours. Combining with generalized gene interaction databases creates a data interface that can help researchers to further analyze the information. Using the TWIRLS system, we found that an important regulatory factor angiotensin-converting enzyme 2 (ACE2) may be involved in the host pathological changes on binding to the coronavirus after infection. After triggering functional changes in ACE2/AT2R, an imbalance in the steady-state cytokine regulatory axis involving the Renin-Angiotensin System and IP-10 leads to a cytokine storm.","While the SARS-CoV-2 pandemic is hardly hitting the world, it is of extreme importance that significant in vitro observations guide the quick set up of clinical trials. In this study, we evidence that the anti-SARS-CoV2 activity of a clinically achievable hydroxychloroquine concentration is maximized only when administered before and after the infection of Vero E6 cells. This strongly suggests that only a combined prophylactic and therapeutic use of hydroxychloroquine may be effective in limiting viral replication in patients.","AbstractAcute lung inflammation has severe morbidity, as seen in COVID-19 patients. Lung inflammation is accompanied or led by massive accumulation of neutrophils in pulmonary capillaries (\u201cmargination\u201d). We sought to identify nanostructural properties that predispose nanoparticles to accumulate in pulmonary marginated neutrophils, and therefore to target severely inflamed lungs. We designed a library of nanoparticles and conducted an in vivo screen of biodistributions in naive mice and mice treated with lipopolysaccharides. We found that supramolecular organization of protein in nanoparticles predicts uptake in inflamed lungs. Specifically, nanoparticles with agglutinated protein (NAPs) efficiently home to pulmonary neutrophils, while protein nanoparticles with symmetric structure (e.g. viral capsids) are ignored by pulmonary neutrophils. We validated this finding by engineering protein-conjugated liposomes that recapitulate NAP targeting to neutrophils in inflamed lungs. We show that NAPs can diagnose acute lung injury in SPECT imaging and that NAP-like liposomes can mitigate neutrophil extravasation and pulmonary edema arising in lung inflammation. Finally, we demonstrate that ischemic ex vivo human lungs selectively take up NAPs, illustrating translational potential. This work demonstrates that structure-dependent interactions with neutrophils can dramatically alter the biodistribution of nanoparticles, and NAPs have significant potential in detecting and treating respiratory conditions arising from injury or infections.","Viruses evolve extremely quickly, so reliable methods for viral host prediction are necessary to safeguard biosecurity and biosafety alike. Novel human-infecting viruses are difficult to detect with standard bioinformatics workflows. Here, we predict whether a virus can infect humans directly from next-generation sequencing reads. We show that deep neural architectures significantly outperform both shallow machine learning and standard, homology-based algorithms, cutting the error rates in half and generalizing to taxonomic units distant from those presented during training. We propose a new approach for convolutional filter visualization to disentangle the information content of each nucleotide from its contribution to the final classification decision. Nucleotide-resolution maps of the learned associations between pathogen genomes and the infectious phenotype can be used to detect virulence-related genes in novel agents, as we show here for the SARS-CoV-2 coronavirus, unknown before it caused a COVID-19 pandemic in 2020.","The global pandemic of severe acute pneumonia syndrome (COVID-19) caused by SARS-CoV-2 urgently calls for prevention and intervention strategies. The densely glycosylated spike (S) protein highly exposed on the viral surface is a determinant for virus binding and invasion into host cells as well as elicitation of a protective host immune response. Herein, we characterized the site-specific N-glycosylation of SARS-CoV-2 S protein using stepped collision energy (SCE) mass spectrometry (MS). Following digestion with two complementary proteases to cover all potential N-glycosylation sequons and integrated N-glycoproteomics analysis, we revealed the N-glycosylation profile of SARS-CoV-2 S proteins at the levels of intact N-glycopeptides and glycosites, along with the glycan composition and site-specific number of glycans. All 22 potential canonical N-glycosites were identified in S protein protomer. Of those, 18 N-glycosites were conserved between SARS-CoV and SARS-CoV-2 S proteins. Nearly all glycosites were preserved among the 753 SARS-CoV-2 genome sequences available in the public influenza database Global Initiative on Sharing All Influenza Data. By comparison, insect cell-expressed SARS-CoV-2 S protein contained 38 N-glycans, which were primarily assigned to the high-mannose type N-glycans, whereas the human cell-produced protein possessed up to 140 N-glycans largely belonging to the complex type. In particular, two N-glycosites located in the structurally exposed receptor-binding domain of S protein exhibited a relatively conserved N-glycan composition in human cells. This N-glycosylation profiling and determination of differences between distinct expression systems could shed light on the infection mechanism and promote development of vaccines and targeted drugs.","Next-generation sequencing is increasingly being used to study samples composed of mixtures of organisms, such as in clinical applications where the presence of a pathogen at very low abundance may be highly important. We present an analytical method (SIANN: Strain Identification by Alignment to Near Neighbors) specifically designed to rapidly detect a set of target organisms in mixed samples that achieves a high degree of species- and strain-specificity by aligning short sequence reads to the genomes of near neighbor organisms, as well as that of the target. Empirical benchmarking alongside the current state-of-the-art methods shows an extremely high Positive Predictive Value, even at very low abundances of the target organism in a mixed sample. SIANN is available as an Illumina BaseSpace app, as well as through Signature Science, LLC. SIANN results are presented in a streamlined report designed to be comprehensible to the non-specialist user, providing a powerful tool for rapid species detection in a mixed sample. By focusing on a set of (customizable) target organisms and their near neighbors, SIANN can operate quickly and with low computational requirements while delivering highly accurate results.","Abstract2019-Novel Coronavirus (2019-nCoV) is the pathogen of Corona Virus Disease 2019. Nucleic acid detection of 2019-nCoV is one of the key indicators for clinical diagnosis. However, the positive rate is only 30-50%. Currently, fluorescent quantitative RT-PCR technology is mainly used to detect 2019-nCoV. According to \u201cThe Laboratory Technical Guidelines for Detection 2019-nCoV (Fourth Edition)\u201d issued by National Health and Commission of China and \u201cThe Experts\u2019 Consensus on Nucleic Acid Detection of 2019-nCoV\u201d released by Chinese Society of Laboratory Medicine, the human samples must be placed under 56\u00b0C or higher to inactivate the viruses in order to keep the inspectors from virus infection before the nucleic acids were isolated as the template of qRT-PCR. In this study, we demonstrated that the virus inactivation treatment disrupts its genome integrity seriously when using porcine epidemic diarrhea virus (vaccine), a kind of coronavirus, as a model. Our results showed that only 50.11% of the detectable viral templates left after the inactivation of 56 \u00b0C for 30 minutes and only 3.36% left after the inactivation of 92 \u00b0C for 5 minutes when the samples were preserved by Hank\u2019s solution, one of an isotonic salt solutions currently suggested. However, the detectable templates of viral nucleic acids can be unchanged after the samples were incubated at 56 \u00b0C or higher if the samples were preserved with an optimized solution to protect the RNA from being disrupted. We therefore highly recommend to carry out systematic investigation on the impact of high temperature inactivation on the integrity of 2019-nCoV genome and develop a sample preservation solution to protect the detectable templates of 2019-nCoV nucleic acids from high temperature inactivation damage.","AbstractMotivationUnderstanding the relationship between the sequence, structure, binding energy, binding kinetics and binding thermodynamics of protein-protein interactions is crucial to understanding cellular signaling, the assembly and regulation of molecular complexes, the mechanisms through which mutations lead to disease, and protein engineering.ResultsWe present SKEMPI 2.0, a major update to our database of binding free energy changes upon mutation for structurally resolved protein-protein interactions. This version now contains manually curated binding data for 7085 mutations, an increase of 133%, including changes in kinetics for 1844 mutations, enthalpy and entropy changes for 443 mutations, and 440 mutations which abolish detectable binding.AvailabilityThe database is available at https://life.bsc.es/pid/skempi2/","ABSTRACTThe respiratory tract surface is protected from inhaled pathogens by a secreted layer of mucus that is rich in mucin glycoproteins. Disrupted mucus production is a cardinal feature of chronic respiratory diseases but how this alteration affect interactions between mucins and pathogens is complex and poorly understood. Here, we identify a central and unexpected role for the major airway mucin MUC5AC in pathogenesis of virus-induced exacerbations of chronic obstructive pulmonary disease (COPD). Virus induction of MUC5AC is augmented in COPD compared to healthy subjects, is enhanced in frequent exacerbators and correlates with inflammation, symptom severity and secondary bacterial infection during exacerbation. MUC5AC is functionally related to inflammation as MUC5AC-deficient (Muc5ac-/-) mice had attenuated rhinovirus-induced airway inflammation whilst exogenous MUC5AC glycoprotein administration augmented virus-induced inflammatory responses and bacterial load. Mechanistically, MUC5AC-augmentation of rhinovirus-induced inflammation occurred through release of extracellular adenosine triphosphate (ATP). Therapeutic suppression of virus-induced MUC5AC release using an epidermal growth factor receptor (EGFR) inhibitor ameliorated exaggerated pro-inflammatory responses in a mouse COPD exacerbation model. Collectively, these studies demonstrate previously unrecognised pro-inflammatory effects of MUC5AC during infection and thus highlight a key unforeseen role in driving COPD exacerbation severity.","AbstractIntroductionCOVID-19, is caused by a new strain of coronavirus called SARS-coronavirus-2 (SARS-CoV-2), which is a positive sense single strand RNA virus. In humans, it binds to angiotensin converting enzyme 2 (ACE2) with the help a structure on its surface called the S-spike. COVID-19 poses intriguing issues with imperative relevance to clinicians. The pathogenesis of GI symptoms, diabetes-associated mortality, and disease recurrence in COVID-19 are of particular relevance because they cannot be sufficiently explained from the existing knowledge of the viral diseases. Tissue specific variation of ACE2 expression in healthy individuals can help in understanding the pathophysiological basis the aforementioned collection of symptoms.Materials and MethodsThe data were downloaded from the Human Protein Atlas available at (https://www.proteinatlas.org/search/ACE2) and the tissue specific expression (both mRNA and protein) of ACE2 as yielded from the studies with RNA sequencing and immunohistochemistry (IHC) was analyzed as a function of the various components of the digestive tract. A digestive system specific functional enrichment map of ACE2 gene was created using g:profiler (https://biit.cs.ut.ee/gprofiler/gost) utility and the data were visualized using Cytoscape software, version 3.7.2 (https://cytoscape.org/).ResultsThe correlated expression (genomic and proteomic) of ACE2 (to which SARS-CoV-2 binds through the S-spike) was found to be enriched in the lower gastrointestinal tract (GIT) (highest in small intestine, followed by colon and rectum), and was undetectable in the upper GIT components: mouth cavity (tongue, oral mucosa, and salivary glands), esophagus, and stomach. High expression of ACE2 was noted in the glandular cells as well as in the enterocytes in the lining epithelium (including brush border epithelium). Among other digestive system organs, gallbladder (GB) showed high expression of ACE2 in glandular cells, while any protein expression was undetectable in liver and pancreas.ConclusionsBased on the findings of this study and supportive evidence from the literature we propose that a SARS-CoV-2 binding with ACE2 mediates dysregulation of the sodium dependent nutrient transporters and hence may be a plausible basis for the digestive symptoms in COVID-19 patients. ACE2 mediated dysregulation of sodium dependent glucose transporter (SGLT1 or SLC5A1) in the intestinal epithelium also links it to the pathogenesis of diabetes mellitus which can be a possible reason for the associated mortality in COVID-19 patients with diabetes. High expression of ACE2 in mucosal cells of the intestine and GB make these organs potential sites for the virus entry and replication. Continued replication of the virus at these ACE2 enriched sites may be a basis for the disease recurrence reported in some, thought to be cured, patients.HighlightsSARS-CoV-2 binding receptor ACE2 expression is enriched in human intestineACE2 regulates neutral amino acid (B0AT1) and glucose transporter (SGLT1) in intestinal epitheliumMucus secreted from GI cells may protect SARS-CoV-2 from harsh pH levels of the digestive juicesContinued replication of SARS-CoV-2 in GI cells may cause disease recurrenceGraphical Abstract","AbstractViruses must hijack cellular translation machinery to efficiently express viral genes. In many cases, this is impeded by cellular stress responses. These stress responses swiftly relocate and repurpose translation machinery, resulting in global inhibition of translation and the aggregation of stalled 48S mRNPs into cytoplasmic foci called stress granules. This results in translational silencing of all mRNAs excluding those beneficial for the cell to resolve the specific stress. For example, expression of antiviral factors is maintained during viral infection. Here we investigated stress granule regulation by Gammacoronavirus infectious bronchitis virus (IBV), which causes the economically important poultry disease, infectious bronchitis. Interestingly, we found that IBV is able to inhibit multiple cellular stress granule signaling pathways whilst at the same time IBV replication also results in induction of seemingly canonical stress granules in a proportion of infected cells. Moreover, IBV infection uncouples translational repression and stress granule formation and both processes are independent of eIF2\u03b1 phosphorylation. These results provide novel insights into how IBV modulates cellular translation and antiviral stress signaling.","AbstractGenome-wide association studies (GWAS) of systemic lupus erythematosus (SLE) explain only \u223c15% of genetic risk, indicating genes of modest effect remain to be discovered. Association clustering methods such as OASIS are more apt at identifying modest genetic effects. 410 genes were mapped to previously identified OASIS GWAS SLE loci and investigated for expression in SLE GEO datasets. GSE50395 dataset from Cordoba was used for validation. Blood eQTL for significant SNPs in SLE loci and STRING for functional pathways of differentially expressed genes was used. Confirmatory qPCR on monocytes of 12 SLE patients and controls was performed. We identified 55 genes that were differentially expressed in at least 2 SLE GEO datasets with all probes directionally aligned. DDX11 was downregulated in both GEO (P=3.60E-02) and Cordoba (P=8.02E-03) datasets and confirmed by qPCR (P=0.001). The most significant SNP, rs3741869 (P=3.2E-05) in OASIS locus 12p11.21, containing DDX11, was a cis-eQTL regulating DDX11 expression (P=8.62E-05). DDX11 interacted with multiple genes including STAT1/STAT4. Genomic convergence with OASIS and multiple expression datasets identifies novel genes. DDX11, RNA helicase involved in genome stability, is repressed in SLE.Summary StatementMore than 100 genes for SLE have been identified but they explain only \u223c15% of heritability. GWAS are challenged by risk genes of modest effect. Using locus-based GWAS mapping and multiple gene expression replications, DDX11 was identified as a novel SLE gene. DDX11, repressed in SLE, may be used as a clinical diagnostic tool.","Molecular phylogenetics plays a key role in comparative genomics and has an increasingly-significant impacts on science, industry, government, public health, and society. We posit that the current phylogenetic protocol is missing two critical steps, and that their absence allows model misspecification and confirmation bias to unduly influence our phylogenetic estimates. Based on the potential offered by well-established but under-used procedures (e.g., assessment of phylogenetic assumptions and tests of goodness-of-fit), we introduce a new phylogenetic protocol that will reduce confirmation bias and increase the accuracy of phylogenetic estimates.","The recent outbreak of SARS-CoV-2 (2019-nCoV) virus has highlighted the need for fast and efficacious vaccine development. Stimulation of a proper immune response that leads to protection is highly dependent on presentation of epitopes to circulating T-cells via the HLA complex. SARS-CoV-2 is a large RNA virus and testing of all overlapping peptides in vitro to deconvolute an immune response is not feasible. Therefore HLA-binding prediction tools are often used to narrow down the number of peptides to test. We tested 15 epitope-HLA-binding prediction tools, and using an in vitro peptide MHC stability assay, we assessed 777 peptides that were predicted to be good binders across 11 MHC allotypes. In this investigation of potential SARS-CoV-2 epitopes we found that current prediction tools vary in performance when assessing binding stability, and they are highly dependent on the MHC allotype in question. Designing a COVID-19 vaccine where only a few epitope targets are included is therefore a very challenging task. Here, we present 174 SARS-CoV-2 epitopes with high prediction binding scores, validated to bind stably to 11 HLA allotypes. Our findings may contribute to the design of an efficacious vaccine against COVID-19.","ABSTRACTAlthough bats are increasingly becoming the focus of scientific studies due to their unique properties, these exceptional animals are still among the least studied mammals. Assembly quality and completeness of bat genomes vary a lot and especially non-coding RNA (ncRNA) annotations are incomplete or simply missing. Accordingly, standard bioinformatics pipelines for gene expression analysis often ignore ncRNAs such as microRNAs or long antisense RNAs. The main cause of this problem is the use of incomplete genome annotations. We present a complete screening for ncRNAs within 16 bat genomes. NcRNAs affect a remarkable variety of vital biological functions, including gene expression regulation, RNA processing, RNA interference and, as recently described, regulatory processes in viral infections. Within all investigated bat assemblies we annotated 667 ncRNA families including 162 snoRNAs and 193 miRNAs as well as rRNAs, tRNAs, several snRNAs and IncRNAs, and other structural ncRNA elements. We validated our ncRNA candidates by six RNA-Seq data sets and show significant expression patterns that have never been described before in a bat species on such a large scale. Our annotations will be usable as a resource (Electronic Supplement) for deeper studying of bat evolution, ncRNAs repertoire, gene expression and regulation, ecology, and important host-virus interactions.Supplementary informationis available at rna.uni-jena.de/supplements/bats, the Open Science Framework (doi.org/10.17605/OSF.IO/4CMDN), and GitHub (github.com/rnajena/bats_ncrna).","AbstractBackgroundsAn ongoing outbreak of a novel coronavirus (2019-nCoV) pneumonia hit a major city of China, Wuhan, December 2019 and subsequently reached other provinces/regions of China and countries. We present estimates of the basic reproduction number, R0, of 2019-nCoV in the early phase of the outbreak.MethodsAccounting for the impact of the variations in disease reporting rate, we modelled the epidemic curve of 2019-nCoV cases time series, in mainland China from January 10 to January 24, 2020, through the exponential growth. With the estimated intrinsic growth rate (\u03b3), we estimated R0 by using the serial intervals (SI) of two other well-known coronavirus diseases, MERS and SARS, as approximations for the true unknown SI.FindingsThe early outbreak data largely follows the exponential growth. We estimated that the mean R0 ranges from 2.24 (95%CI: 1.96-2.55) to 3.58 (95%CI: 2.89-4.39) associated with 8-fold to 2-fold increase in the reporting rate. We demonstrated that changes in reporting rate substantially affect estimates of R0.ConclusionThe mean estimate of R0 for the 2019-nCoV ranges from 2.24 to 3.58, and significantly larger than 1. Our findings indicate the potential of 2019-nCoV to cause outbreaks.","ABSTRACTCOVID-19, SARS, and MERS are featured by fibrinolytic dysfunction. To test the role of the fibrinolytic niche in the regeneration of alveolar epithelium, we compared the self-renewing capacity of alveolar epithelial type 2 (AT2) cells and its differentiation to AT1 cells between wild type (wt) and fibrinolytic niche deficient mice (Plau\u2212/\u2212 and Serpine1Tg). A significant reduction in both proliferation and differentiation of deficient AT2 cells was observed in vivo and in 3D organoid cultures. This decrease was mainly restored by uPA derived A6 peptide, a binding fragment to CD44 receptors. The proliferative and differential rate of CD44+ AT2 cells was greater than that of CD44\u2212 controls. There was a reduction in transepithelial ion transport in deficient monolayers compared to wt cells. Moreover, we found a marked suppression in total AT2 cells and CD44+ subpopulation in lungs from brain dead patients with acute respiratory distress syndrome (ARDS) and a mouse model infected by influenza viruses. Thus, we demonstrate that the fibrinolytic niche can regulate AT2-mediated homeostasis and regeneration via a novel uPA-A6-CD44+-ENaC cascade.","COVID-19 has become a global pandemic caused by a novel coronavirus SARS-CoV-2. Understanding the origins of SARS-CoV-2 is critical for deterring future zoonosis and for drug discovery and vaccine development. We show evidence of strong purifying selection around the receptor binding motif (RBM) in the spike gene and in other genes among bat, pangolin and human coronaviruses, indicating similar strong evolutionary constraints in different host species. We also demonstrate that SARS-CoV-2's entire RBM was introduced through recombination with coronaviruses from pangolins, possibly a critical step in the evolution of SARS-CoV-2's ability to infect humans. Similar purifying selection in different host species and frequent recombination among coronaviruses suggest a common evolutionary mechanism that could lead to new emerging human coronaviruses.","SUMMARYSARS-CoV-2 is a zoonotic virus that has caused a pandemic of severe respiratory disease\u2014COVID-19\u2014 within several months of its initial identification. Comparable to the first SARS-CoV, this novel coronavirus\u2019s surface Spike (S) glycoprotein mediates cell entry via the human ACE-2 receptor, and, thus, is the principal target for the development of vaccines and immunotherapeutics. Molecular information on the SARS-CoV-2 S glycoprotein remains limited. Here we report the crystal structure of the SARS-CoV-2 S receptor-binding-domain (RBD) at a the highest resolution to date, of 1.95 \u00c5. We identified a set of SARS-reactive monoclonal antibodies with cross-reactivity to SARS-CoV-2 RBD and other betacoronavirus S glycoproteins. One of these antibodies, CR3022, was previously shown to synergize with antibodies that target the ACE-2 binding site on the SARS-CoV RBD and reduce viral escape capacity. We determined the structure of CR3022, in complex with the SARS-CoV-2 RBD, and defined a broadly reactive epitope that is highly conserved across betacoronaviruses. This epitope is inaccessible in the \u201cclosed\u201d prefusion S structure, but is accessible in \u201copen\u201d conformations. This first-ever resolution of a human antibody in complex with SARS-CoV-2 and the broad reactivity of this set of antibodies to a conserved betacoronavirus epitope will allow antigenic assessment of vaccine candidates, and provide a framework for accelerated vaccine, immunotherapeutic and diagnostic strategies against SARS-CoV-2 and related betacoronaviruses.HIGHLIGHTSHigh resolution structure of the SARS-CoV-2 Receptor-Binding-Domain (RBD).Recognition of the SARS-CoV-2 RBD by SARS-CoV antibodies.Structure of the SARS-COV-2 RBD in complex with antibody CR3022.Identification of a cryptic site of vulnerability on the SARS-CoV-2 Spike.","ABSTRACTThe viral cytotoxic T lymphocyte (CTL) epitope peptides presented by classical MHC-I molecules require the assembly of a peptide-MHC-I-\u03b22m (aka pMHC-I) trimolecular complex for TCR recognition, which is the critical activation link for triggering antiviral T cell immunity. Ursidae includes 5 genera and 8 species; however, research on T cell immunology in this family, especially structural immunology, is lacking. In this study, the structure of the key trimolecular complex pMHC-1 (aka pAime-128), which binds a peptide from canine distemper virus, was solved for the first time using giant panda as a representative species of Ursidae. The structural characteristics of the giant panda pMHC-I complex, including the unique pockets in the peptide-binding groove (PBG), were analyzed in detail. Comparing the panda pMHC-I to others in the bear family and extending the comparison to other mammals revealed distinct features. The interaction between MHC-I and \u03b22m, the features of pAime-128 involved in TCR docking and CD8 binding, the anchor sites in the PBG, and the CTL epitopes of potential viruses that infect pandas were concretely clarified. Unique features of pMHC-I viral antigen presentation in the panda were revealed by solving the three-dimensional structure of pAime-128. The distinct characteristics of pAime-128 indicate an unusual event that emerged during the evolution of the MHC system in the bear family. These results provide a new platform for research on panda CTL immunity and the design of vaccines for application in the bear family.IMPORTANCEUrsidae includes 5 genera and 8 species; however, the study of its immunology, especially structural immunology, is extremely rare to date. In this paper, we first crystallized the key complex pMHC-I, taking the giant panda as its representative species. Structural characteristics of the giant panda pMHC-I complexes, contains the unique pockets of PBG were analyzed in detail. Comparison of the panda pMHC-I in the bear family and other mammals, almost definite features was displayed. Meanwhile, the interaction between HC and LV, the unique features of pMHC-I in the CD8 binding and TCR docking, validation of anchor site in the PBG, and epitopes of potential viruses infected with the pandas, were concretely clarified. These unique characteristics of pMHC-I clearly indicate an unusual situation during the evolution of MHC molecules in the endangered pandas. These results also provide a novel platform for further study of panda T cell immunology and vaccines.","AbstractE-probe Diagnostic for Nucleic acid Analysis (EDNA) is a bioinformatic tool originally developed to detect plant pathogens in metagenomic databases. However, enhancements made to EDNA increased its capacity to conduct hypothesis directed detection of specific gene targets present in transcriptomic databases. To target specific pathogenicity factors used by the pathogen to infect its host or other targets of interest, e-probes need to be developed for transcripts related to that function. In this study, EDNA transcriptomics (EDNAtran) was developed to detect the expression of genes related to aflatoxin production at the transcriptomic level. E-probes were designed from genes up-regulated during A. flavus aflatoxin production. EDNAtran detected gene transcripts related to aflatoxin production in a transcriptomic database from corn, where aflatoxin was produced. The results were significantly different from e-probes being used in the transcriptomic database where aflatoxin was not produced (atoxigenic AF36 strain and toxigenic AF70 in Potato Dextrose Broth).","We present a timely evaluation of the Chinese 2019-nCov epidemic in its initial phase, where 2019-nCov demonstrates comparable transmissibility but lower fatality rates than SARS and MERS. A quick diagnosis that leads to case isolation and integrated interventions will have a major impact on its future trend. Nevertheless, as China is facing its Spring Festival travel rush and the epidemic has spread beyond its borders, further investigation on its potential spatiotemporal transmission pattern and novel intervention strategies are warranted.","AbstractThe spread of the novel coronavirus (SARS-CoV-2) has triggered a global emergency, that demands urgent solutions for detection and therapy to prevent escalating health, social and economic impacts. The spike protein (S) of this virus enables binding to the human receptor ACE2, and hence presents a prime target for vaccines preventing viral entry into host cells1. The S proteins from SARS-CoV-1 and SARS-CoV-2 are similar2, but structural differences in the receptor binding domain (RBD) preclude the use of SARS-CoV-1\u2013specific neutralizing antibodies to inhibit SARS-CoV-23. Here we used comparative pangenomic analysis of all sequenced Betacoronaviruses to reveal that, among all core gene clusters present in these viruses, the envelope protein E shows a variant shared by SARS and SARS-Cov2 with two completely-conserved key functional features, an ion-channel and a PDZ-binding Motif (PBM). These features trigger a cytokine storm that activates the inflammasome, leading to increased edema in lungs causing the acute respiratory distress syndrome (ARDS)4-6, the leading cause of death in SARS-CoV-1 and SARS-CoV-2 infection7,8. However, three drugs approved for human use may inhibit SARS-CoV-1 and SARS-CoV-2 Protein E, either acting upon the ion channel (Amantadine and Hexamethylene amiloride9,10) or the PBM (SB2035805), thereby potentially increasing the survival of the host, as already demonstrated for SARS-CoV-1in animal models. Hence, blocking the SARS protein E inhibits development of ARDS in vivo. Given that our results demonstrate that the protein E subcluster for the SARS clade is quasi-identical for the key functional regions of SARS-CoV-1 and SARS-CoV-2, we conclude that use of approved drugs shown to act as SARS E protein inhibitors can help prevent further casualties from COVID-2019 while vaccines and other preventive measures are being developed.","The ongoing pandemic spread of a novel human coronavirus, SARS-COV-2, associated with severe pneumonia disease (COVID-19), has resulted in the generation of thousands of virus genome sequences. The rate of genome generation is unprecedented, yet there is currently no coherent nor accepted scheme for naming the expanding phylogenetic diversity of SARS-CoV-2. We present a rational and dynamic virus nomenclature that uses a phylogenetic framework to identify those lineages that contribute most to active spread. Our system is made tractable by constraining the number and depth of hierarchical lineage labels and by flagging and declassifying virus lineages that become unobserved and hence are likely inactive. By focusing on active virus lineages and those spreading to new locations this nomenclature will assist in tracking and understanding the patterns and determinants of the global spread of SARS-CoV-2.","AbstractBats provide key ecosystem services such as crop pest regulation, pollination, seed dispersal, and soil fertilization. Bats are also major hosts for biological agents responsible for zoonoses, such as coronaviruses (CoVs). The islands of the Western Indian Ocean are identified as a major biodiversity hotspot, with more than 50 bat species. In this study, we tested 1,013 bats belonging to 36 species from Mozambique, Madagascar, Mauritius, Mayotte, Reunion Island and Seychelles, based on molecular screening and partial sequencing of the RNA-dependent RNA polymerase gene. In total, 88 bats (8.7%) tested positive for coronaviruses, with higher prevalence in Mozambican bats (20.5% \u00b1 4.9%) as compared to those sampled on islands (4.5% \u00b1 1.5%). Phylogenetic analyses revealed a large diversity of \u03b1- and \u03b2-CoVs and a strong signal of co-evolution between CoVs and their bat host species, with limited evidence for host-switching, except for bat species sharing day roost sites.ImportanceThis is the first study to report the presence of coronaviruses (CoVs) in bats in Mayotte, Mozambique and Reunion Island, and in insectivorous bats in Madagascar. Eight percent of the tested bats were positive for CoVs, with higher prevalence in continental Africa than on islands. A high genetic diversity of \u03b1- and \u03b2-CoVs was found, with strong association between bat host and virus phylogenies, supporting a long history of co-evolution between bats and their associated CoVs in the Western Indian Ocean. These results highlight that strong variation between islands does exist and is associated with the composition of the bat species community on each island. Future studies should investigate whether CoVs detected in these bats have a potential for spillover in other hosts.","AbstractUnderstanding interspecific viral transmission is key to understanding viral ecology and evolution, disease spillover into humans, and the consequences of global change. Prior work has demonstrated that macroecological factors drive viral sharing in some mammalian groups, but analyses have never attempted to predict viral sharing in a pan-mammalian context. Here we show that host phylogenetic similarity and geographic range overlap are strong, nonlinear predictors of viral sharing among species across the entire mammal class. Using these traits, we predict global viral sharing patterns across 4196 mammal species and show that our simulated network successfully predicts viral sharing and reservoir host status using internal validation and an external dataset. We predict high rates of mammalian viral sharing in the tropics, particularly among rodents and bats, and that within- and between-order sharing differs geographically and taxonomically. Our results emphasize the importance of macroecological factors in shaping mammalian viral communities, and provide a robust, general model to predict viral host range and guide pathogen surveillance and conservation efforts.","AbstractIn December 2019, a novel coronavirus named SARS-CoV-2 first reported in Wuhan, China, emerged and rapidly spread to numerous other countries globally, causing the current pandemic. SARS-CoV-2 causes acute infection of the respiratory tract (COVID-19) that can result in severe disease and lethality. Currently, there is no approved antiviral drug for treating COVID-19 patients and there is an urgent need for specific antiviral therapies and vaccines.In order for SARS-CoV-2 to enter cells, its surface glycoprotein spike (S) must be cleaved at two different sites by host cell proteases, which therefore represent potential drug targets. In the present study we investigated which host cell proteases activate the SARS-CoV-2 S protein in Calu-3 human airway epithelial cells. We show that S can be cleaved by both the proprotein convertase furin at the S1/S2 site and the transmembrane serine protease 2 (TMPRSS2) at the S2\u2019 site. We demonstrate that TMPRSS2 is essential for activation of SARS-CoV-2 S in Calu-3 cells through antisense-mediated knockdown of TMPRSS2 expression. Further, we show that SARS-CoV-2 replication can be efficiently inhibited by two synthetic inhibitors of TMPRSS2 and also by the broad range serine protease inhibitor aprotinin. Additionally, SARS-CoV-2 replication was also strongly inhibited by the synthetic furin inhibitor MI-1851. Combining various TMPRSS2 inhibitors with MI-1851 produced more potent antiviral activity against SARS-CoV-2 than an equimolar amount of any single serine protease inhibitor. In contrast, inhibition of endosomal cathepsins by E64d did not affect virus replication.Our data demonstrate that both TMPRSS2 and furin are essential for SARS-CoV-2 activation in human airway cells and are promising drug targets for the treatment of COVID-19 either by targeting one of these proteases alone or by a combination of furin and TMPRSS2 inhibitors. Therefore, this approach has a high therapeutic potential for treatment of COVID-19.","Abstract2019-nCoV, which is a novel coronavirus emerged in Wuhan, China, at the end of 2019, has caused at least infected 11,844 as of Feb 1, 2020. However, there is no specific antiviral treatment or vaccine currently. Very recently report had suggested that novel CoV would use the same cell entry receptor, ACE2, as the SARS-CoV. In this report, we generated a novel recombinant protein by connecting the extracellular domain of human ACE2 to the Fc region of the human immunoglobulin IgG1. An ACE2 mutant with low catalytic activity was also used in the study. The fusion proteins were then characterized. Both fusion proteins has high affinity binding to the receptor-binding domain (RBD) of SARS-CoV and 2019-nCoV and exerted desired pharmacological properties. Moreover, fusion proteins potently neutralized SARS-CoV and 2019-nCoV in vitro. As these fusion proteins exhibit cross-reactivity against coronaviruses, they could have potential applications for diagnosis, prophylaxis, and treatment of 2019-nCoV.","AbstractMany mammalian carnivore species have been reduced to small, isolated populations by habitat destruction, fragmentation, poaching, and human conflict. Their limited genetic variability and increased exposure to domestic animals such as dogs place them at risk of further losses from infectious diseases. In India, domestic and feral dogs are associated with villages in and around protected areas, and may serve as reservoirs of pathogens to the carnivores within. India\u2019s Kanha Tiger Reserve (KTR) is home to a number of threatened and endangered mammalian carnivores including tiger (Panthera tigris), leopard (Panthera pardus), wolf (Canis lupus), and dhole (Cuon alpinus). It also has more than 150 villages with associated dog populations. We found that dog populations ranged from 14 to 45/village (3.7 to 23.7/km2), and did not vary with village area, human population size, or distance from the KTR\u2019s core area, though they all increased between summer 2014 and winter 2015, primarily through reproduction. No dog tested positive for rabies but seroprevalence levels to three other generalist viral pathogens were high in summer (N=67) and decreased somewhat by winter (N=168): canine parvovirus (83.6% to 68.4%), canine distemper virus (50.7% to 30.4%) and canine adenovirus (41.8% to 30.9%). The declines in seroprevalence were primarily due to new recruitments by birth and these were not yet exposed to the viruses. Wild carnivores frequently entered the villages, as shown by tracks, scats, kills and other indicators, and the dogs are known to leave the villages so that encounters between dogs and wild carnivores may be common. We conclude that there is a large population of unvaccinated dogs in and around Kanha Tiger Reserve, with high levels of seroprevalence to pathogens with broad host ranges and these dogs which interact with wild carnivores, therefore posing a high risk of disease spillover to the wild carnivores.","The spread of SARS-CoV-2 since December 2019 has become a pandemic and impacted many aspects of human society. Here, we analyzed genetic variation of SARS-CoV-2 and its related coronavirus and found the evidence of intergenomic recombination. After correction for mutational bias, analysis of 137 SARS-CoV-2 genomes as of 2/23/2020 revealed the excess of low frequency mutations on both synonymous and nonsynonymous sites which is consistent with recent origin of the virus. In contrast to adaptive evolution previously reported for SARS-CoV in its brief epidemic in 2003, our analysis of SARS-CoV-2 genomes shows signs of relaxation of selection. The sequence similarity of the spike receptor binding domain between SARS-CoV-2 and a sequence from pangolin is probably due to an ancient intergenomic introgression. Therefore, SARS-CoV-2 might have cryptically circulated within humans for years before being recently noticed. Data from the early outbreak and hospital archives are needed to trace its evolutionary path and reveal critical steps required for effective spreading. Two mutations, 84S in orf8 protein and 251V in orf3 protein, occurred coincidentally with human intervention. The 84S first appeared on 1/5/2020 and reached a plateau around 1/23/2020, the lockdown of Wuhan. 251V emerged on 1/21/2020 and rapidly increased its frequency. Thus, the roles of these mutations on infectivity need to be elucidated. Genetic diversity of SARS-CoV-2 collected from China was two time higher than those derived from the rest of the world. In addition, in network analysis, haplotypes collected from Wuhan city were at interior and have more mutational connections, both of which are consistent with the observation that the outbreak of cov-19 was originated from China.","A novel coronavirus SARS-CoV-2, also called novel coronavirus 2019 (nCoV-19), started to circulate among humans around December 2019, and it is now widespread as a global pandemic. The disease caused by SARS-CoV-2 virus is called COVID-19, which is highly contagious and has an overall mortality rate of 6.4% as of April 15, 2020. There is no vaccine or antiviral available for SARS-CoV-2. In this study, we report our discovery of inhibitors targeting the SARS-CoV-2 main protease (Mpro). Using the FRET-based enzymatic assay, several inhibitors including boceprevir, GC-376, calpain inhibitors II, XII, and MG-132 were identified to have potent activity with single-digit to submicromolar IC50 values in the enzymatic assay. The mechanism of action of the hits was further characterized using enzyme kinetic studies and thermal shift binding assays. Significantly, four compounds (boceprevir, GC-376, calpain inhibitors II and XII) inhibit SARS-CoV-2 viral replication in cell culture with EC50 values ranging from 0.49 to 3.37 micromolar. Notably, boceprevir, calpain inhibitors II and XII represent novel chemotypes that are distinct from known Mpro inhibitors. Overall, the compounds identified herein provide promising starting points for the further development of SARS-CoV-2 therapeutics.","AbstractThree anti-HIV drugs, ritonavir, lopinavir and darunavir, might have therapeutic effect on coronavirus disease 2019 (COVID-19). In this study, the structure models of two severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) proteases, coronavirus endopeptidase C30 (CEP_C30) and papain like viral protease (PLVP), were built by homology modeling. Ritonavir, lopinavir and darunavir were then docked to the models, respectively, followed by energy minimization of the protease-drug complexes. In the simulations, ritonavir can bind to CEP_C30 most suitably, and induce significant conformation changes of CEP_C30; lopinavir can also bind to CEP_C30 suitably, and induce significant conformation changes of CEP_C30; darunavir can bind to PLVP suitably with slight conformation changes of PLVP. It is suggested that the therapeutic effect of ritonavir and lopinavir on COVID-19 may be mainly due to their inhibitory effect on CEP_C30, while ritonavir may have stronger efficacy; the inhibitory effect of darunavir on SARS-CoV-2 and its potential therapeutic effect may be mainly due to its inhibitory effect on PLVP.","AbstractThe bacterial bipartite transferrin receptor is an iron acquisition system that is required for survival by several key human and animal pathogens. It consists of the TonB-dependent transporter Transferrin binding protein A (TbpA) and the surface lipoprotein Transferrin binding protein B (TbpB). Curiously, the Tbps are only found in host specific pathogens, and are themselves host specific, meaning that they will bind to the transferrin of their host species, but not to those of other animal species. While this phenomenon has long been established, neither the steps in the evolutionary process that led to this exquisite adaptation for the host, nor the steps that could alter it, are known. We sought to gain insight into these processes by studying Tbp specificity in Histophilus somni, a major pathogen of cattle. A past study showed that whole cells of H. somni specifically bind bovine transferrin, but not transferrin from sheep and goats, two bovids whose transferrins share 93% amino acid sequence identity with bovine transferrin. To our surprise, we found that H. somni can use sheep and goat transferrins as iron sources for growth, and that HsTbpB, but not HsTbpA, has detectable affinity for sheep and goat transferrins. Furthermore, a third transferrin binding protein, HsTbpA2, also showed affinity for sheep and goat transferrins. Our results show that H. somni TbpB and TbpA2 act to broaden the host transferrin recognition range of H. somni.ImportanceHost restricted pathogens infect a single host species or a narrow range of host species. Histophilus somni, a pathogen that incurs severe economic losses for the cattle industry, infects cattle, sheep, and goats, but not other mammals. The transferrin binding proteins, TbpA and TbpB, are thought to be a key iron acquisition system in H. somni, however, surprisingly, they were also shown to be cattle transferrin-specific. In our study we find that H. somni TbpB, and another little-studied Tbp, TbpA2, bind sheep and goat transferrins as well as bovine transferrin. Our results suggest that TbpA2 may have allowed for host range expansion, and provide a mechanism for how host specificity in Tbp containing pathogens can be altered.","Summary: Genome sequences constitute the primary evidence on the origin and spread of the 2019-2020 Covid-19 pandemic. Rapid comparative analysis of coronavirus SARS-CoV-2 genomes is critical for disease control, outbreak forecasting, and developing clinical interventions. CoV Genome Tracker is a web portal dedicated to trace Covid-19 out-breaks in real time using a haplotype network, an accurate and scalable representation of genomic changes in a rapidly evolving population. We resolve the direction of mutations by using a bat-associated genome as outgroup. At a broader evolutionary time scale, a compan-ion browser provides gene-by-gene and codon-by-codon evolutionary rates to facilitate the search for molecular targets of clinical interventions. Availability and Implementation: CoV Genome Tracker is publicly available at http://cov.genometracker.org and updated weekly with the data downloaded from GISAID (http://gisaid.org). The website is implemented with a custom JavaScript script based on jQuery (https://jquery.com) and D3-force (https://github.com/d3/d3-force). Contact: weigang@genectr.hunter.cuny.edu, City University of New York, Hunter College. Supplementary Information: All supporting scripts developed in JavaScript, Python, BASH, and PERL programming languages are available as Open Source at the GitHub re-pository https://github.com/weigangq/cov-browser.","AbstractKnowledge of the host factors required for norovirus replication has been hindered by the challenges associated with culturing human noroviruses. We have combined proteomic analysis of the viral translation and replication complexes with a CRISPR screen, to identify host factors required for norovirus infection. The core stress granule component G3BP1 was identified as a host factor essential for efficient human and murine norovirus infection, demonstrating a conserved function across theNorovirusgenus. Furthermore, we show that G3BP1 functions in the novel paradigm of viral VPg-dependent translation initiation, contributing to the assembly of translation complexes on the VPg-linked viral positive sense RNA genome by facilitating 40S recruitment. Our data suggest that G3BP1 functions by providing viral RNA a competitive advantage over capped cellular RNAs, uncovering a novel function for G3BP1 in the life cycle of positive sense RNA viruses and identifying the first host factor with pan-norovirus pro-viral activity.","Many viral infections can be prevented by immunizing with live, attenuated vaccines. Early methods of attenuation were hit-and-miss, and these are now much improved by genetic engineering. But even current attenuation methods operate on the principle of genetic harm, reducing the virus's ability to grow, which in turn limits the host immune response below that of infection by wild-type. We use mathematical models of the dynamics of virus and its control by innate and adaptive immunity to explore the tradeoff between attenuation of virus growth and the generation of immunity. Our analysis suggests that directed attenuation that disables key viral defenses against the host immune responses may attenuate viral growth without compromising, and potentially even enhancing the generation of immunity. We explore which immune evasion pathways should be attenuated and how attenuating multiple pathways could lead to robust attenuation of pathology with enhancement of immunity.","AbstractWe recently described CUT&amp;Tag, a general strategy for epigenomic profiing in which antibody-tethered Tn5 transposase integrates DNA sequencing adapters at sites of specific chromatin protein binding or histone modification in intact cells or nuclei. Here we introduce a simplified CUT&amp;Tag method that can be performed at home to help ameliorate the interruption of bench research caused by COVID-19 physical distancing requirements. All steps beginning with frozen nuclei are performed in single PCR tubes through to barcoded library amplication and clean-up, ready for pooling and DNA sequencing. Our CUT&amp;Tag@home protocol has minimal equipment, reagent and supply needs and does not require handling of toxic or biologically active materials. We show that data quality and reproducibility for samples down to ~100 nuclei compare favorably to datasets produced using lab-based CUT&amp;Tag and other chromatin profiling methods. We use CUT&amp;Tag@home with antibodies to trimethylated histone H3 lysine-4, \u221236, \u221227 and-9 to comprehensively profile the epigenome of human K562 cells, consisting respectively of active gene regulatory elements, transcribed gene bodies, developmentally silenced domains and constitutively silenced parasitic elements.","AbstractAn epidemic can be characterized by its speed (i.e., the exponential growth rate r) and strength (i.e., the reproductive number \u211b). Disease modelers have historically placed much more emphasis on strength, in part because the effectiveness of an intervention strategy is typically evaluated on this scale. Here, we develop a mathematical framework for this classic, strength-based paradigm and show that there is a corresponding speed-based paradigm which can provide complementary insights. In particular, we note that r = 0 is a threshold for disease spread, just like \u211b = 1, and show that we can measure the speed and strength of an intervention on the same scale as the speed and strength of an epidemic, respectively. We argue that, just as the strength-based paradigm provides the clearest insight into certain questions, the speed-based paradigm provides the clearest view in other cases. As an example, we show that evaluating the prospects of \u201ctest-and-treat\u201d interventions against the human immunodeficiency virus (HIV) can be done more clearly on the speed than strength scale, given uncertainty in the proportion of HIV spread that happens early in the course of infection. We suggest that disease modelers should avoid over-emphasizing the reproductive number at the expense of the exponential growth rate, but instead look at these as complementary measures.","AbstractThe 2013-2016 West Africa EBOV epidemic was the biggest EBOV outbreak to date. An analysis of virus-specific CD8+ T-cell immunity in 30 survivors showed that 26 of those individuals had a CD8+ response to at least one EBOV protein. The dominant response (25/26 subjects) was specific to the EBOV nucleocapsid protein (NP). It has been suggested that epitopes on the EBOV NP could form an important part of an effective T-cell vaccine for Ebola Zaire. We show that a 9-amino-acid peptide NP44-52 (YQVNNLEEI) located in a conserved region of EBOV NP provides protection against morbidity and mortality after mouse adapted EBOV challenge. A single vaccination in a C57BL/6 mouse using an adjuvanted microsphere peptide vaccine formulation containing NP44-52 is enough to confer immunity in mice. Our work suggests that a peptide vaccine based on CD8+ T-cell immunity in EBOV survivors is conceptually sound and feasible. Nucleocapsid proteins within COVID-19 contain multiple class I epitopes with predicted HLA restrictions consistent with broad population coverage. A similar approach to a CTL vaccine design may be possible for that virus.","As the COVID-19 outbreak spreads, there is a growing need for a compilation of conserved RNA genome regions in the SARS-CoV-2 virus along with their structural propensities to guide development of antivirals and diagnostics. Using sequence alignments spanning a range of betacoronaviruses, we rank genomic regions by RNA sequence conservation, identifying 79 regions of length at least 15 nucleotides as exactly conserved over SARS-related complete genome sequences available near the beginning of the COVID-19 outbreak. We then confirm the conservation of the majority of these genome regions across 739 SARS-CoV-2 sequences reported to date from the current COVID-19 outbreak, and we present a curated list of 30 \u2032SARS-related-conserved\u2032 regions. We find that known RNA structured elements curated as Rfam families and in prior literature are enriched in these conserved genome regions, and we predict additional conserved, stable secondary structures across the viral genome. We provide 106 \u2032SARS-CoV-2-conserved-structured\u2032 regions as potential targets for antivirals that bind to structured RNA. We further provide detailed secondary structure models for the 5\u2032 UTR, frame-shifting element, and 3\u2032 UTR. Last, we predict regions of the SARS-CoV-2 viral genome have low propensity for RNA secondary structure and are conserved within SARS-CoV-2 strains. These 59 \u2032SARS-CoV-2-conserved-unstructured\u2032 genomic regions may be most easily targeted in primer-based diagnostic and oligonucleotide-based therapeutic strategies.","AbstractLung injury and fibrosis represent the most significant outcomes of severe and acute lung disorders, including COVID-19. However, there are still no effective drugs to treat lung injury and fibrosis. In this study, we report the generation of clinical-grade human embryonic stem cells (hESCs)-derived immunity- and matrix-regulatory cells (IMRCs) manufactured under good manufacturing practice (GMP) requirements, that can treat lung injury and fibrosis in vivo. We generate IMRCs through sequentially differentiating hESCs with xeno-free reagents. IMRCs possess a unique gene expression profile distinct from umbilical cord mesenchymal stem cells (UCMSCs), such as higher levels of proliferative, immunomodulatory and anti-fibrotic genes. Moreover, intravenous delivery of IMRCs inhibits both pulmonary inflammation and fibrosis in mouse models of lung injury, and significantly improves the survival rate of the recipient mice in a dose-dependent manner, likely through their paracrine functions. IMRCs are superior to both primary UCMSCs and FDA-approved pirfenidone, with an excellent efficacy and safety profile in mice, monkeys and two severely ill COVID-19 patients in our pilot study. In light of recent public health crises involving pneumonia, acute lung injury (ALI) and acute respiratory distress syndrome (ARDS), our findings indicate that IMRCs are ready for clinical trials on lung disorders.","AbstractBACKGROUNDIn the acute respiratory distress syndrome (ARDS), elevated alveolar surface tension, T, may increase ventilation-induced lung injury. Exogenous surfactant therapy has not reduced ARDS mortality. Sulforhodamine B (SRB), which acts with albumin to improve native lung surfactant efficacy, could be an alternative T-lowering therapeutic. We test whether substances suspected of elevating T in ARDS raise T in the lungs \u2013 where, unlike in most in vitro tests, the surfactant monolayer is intact \u2013 and test the abilities of exogenous surfactant and SRB to reduce T.METHODSIn isolated rat lungs, we micropuncture a surface alveolus and instill a solution containing a substance purported to raise T in ARDS: control saline, cell debris, secretory phospholipase A2 (sPLA2), acid or mucins. We test each substance alone; with albumin, to model proteinaceous edema liquid; with albumin and subsequent exogenous surfactant; or with albumin and SRB. We determine T by combining servo-nulling pressure measurement with confocal microscopy, and applying the Laplace relation.RESULTSIn the control group, saline, albumin and Infasurf do not alter T; SRB reduces T below normal. With albumin, the experimental substances raise T. With cell debris, surfactant does not alter T; SRB normalizes T. With sPLA2, surfactant normalizes T; SRB reduces T. With acid or mucins, neither surfactant nor SRB alters T.CONCLUSIONSThe inability of surfactant to counter cell debris may contribute to the failure of surfactant therapy for ARDS. For non-aspiration ARDS, SRB, which can be delivered intravascularly to target injured lung regions, holds promise as a treatment.SummaryIn the acute respiratory distress syndrome (ARDS), surface tension, T, is believed to be elevated and surfactant therapy has failed to reduce mortality. Here, we test whether various substances suggested to contribute to elevated T in ARDS in fact raise T in the lungs. And we test the ability of exogenous surfactant and of a potential alternative therapeutic, sulforhodamine B (SRB), to reduce T. We find exogenous surfactant unable to lower T in the presence of cell debris but SRB, which can be administered intravascularly, a candidate for lowering T in non-aspiration ARDS.","COVID-19, caused by SARS-CoV-2 infection, has recently been announced as a pandemic all over the world. Plenty of diagnostic, preventive and therapeutic knowledges have been enriched from clinical studies since December 2019. However, animal models, particularly non-human primate models, are urgently needed for critical questions that could not be answered in clinical patients, evaluations of anti-viral drugs and vaccines. In this study, two families of non-human primates, old world monkeys (12 Macaca mulatta, 6 Macaca fascicularis) and new world monkeys (6 Callithrix jacchus), were experimentally inoculated with SARS-CoV-2. Clinical signs were recorded. Samples were collected for analysis of viral shedding, viremia and histopathological examination. Increased body temperature was observed in 100% (12/12) M. mulatta, 33.3% (2/6) M. fascicularis and none (0/6) of C. jacchus post inoculation of SARS-CoV-2. All of M. mulatta and M. fascicularis showed chest radiographic abnormality. Viral genomes were detected in nasal swabs, throat swabs, anal swabs and blood from all 3 species of monkeys. Viral shedding from upper respiratory reached the peak between day 6 and day 8 post inoculation. From necropsied M. mulatta and M. fascicularis, tissues showing virus positive were mainly lung, weasand, bronchus and spleen. No viral genome was seen in any of tissues from 2 necropsied C. jacchus. Severe gross lesions and histopathological changes were observed in lung, heart and stomach of SARS-CoV-2 infected animals. In summary, we have established a NHP model for COVID-19, which could be used to evaluate drugs and vaccines, and investigate viral pathogenesis. M. mulatta is the most susceptible to SARS-CoV2 infection, followed by M. fascicularis and C. jacchus.","Motivation: Coronaviridae are a family of positive-sense RNA viruses capable of infecting humans and animals. These viruses usually cause a mild to moderate upper respiratory tract infection, however, they can also cause more severe symptoms, gastrointestinal and central nervous system diseases. These viruses are capable of flexibly adapting to new environments, hence health threats from coronavirus are constant and long-term. Immunogenic spike proteins are glyco-proteins found on the surface of Coronaviridae particles that mediate entry to host cells. The aim of this study was to train deep learning neural networks to produce simulated spike protein sequences, which may be able to aid in knowledge and/or vaccine design by creating alternative possible spike sequences that could arise from zoonotic sources in future. Results: Here we have trained deep learning recurrent neural networks (RNN) to provide computer simulated coronavirus spike protein sequences in the style of previously known sequences and examine their characteristics. Training used a dataset of alpha, beta, gamma and delta coronavirus spike sequences. In a test set of 100 simulated sequences, all 100 had most significant BLAST matches to Spike proteins in searches against NCBI non-redundant dataset (NR) and also possessed concomitant Pfam domain matches. Conclusions: Simulated sequences from the neural network may be able to guide us in future with prospective targets for vaccine discovery in advance of a potential novel zoonosis. We may effectively be able to fast-forward through evolution using neural networks to investigate sequences that could arise.","AbstractInfluenza is a major global public health threat as a result of its highly pathogenic variants, large zoonotic reservoir, and pandemic potential. Metagenomic viral sequencing offers the potential of a diagnostic test for influenza which also provides insights on transmission, evolution and drug resistance, and simultaneously detects other viruses. We therefore set out to apply Oxford Nanopore Technology to metagenomic sequencing of respiratory samples. We generated influenza reads down to a limit of detection of 102-103 genome copies/ml in pooled samples, observing a strong relationship between the viral titre and the proportion of influenza reads (p = 4.7\u00d710-5). Applying our methods to clinical throat swabs, we generated influenza reads for 27/27 samples with high-to-mid viral titres (Cycle threshold (Ct) values &lt;30) and 6/13 samples with low viral titres (Ct values 30-40). No false positive reads were generated from 10 influenza-negative samples. Thus Nanopore sequencing operated with 83% sensitivity (95% CI 67-93%) and 100% specificity (95% CI 69-100%) compared to the current diagnostic standard. Coverage of full length virus was dependent on sample composition, being negatively influenced by increased host and bacterial reads. However, at high influenza titres, we were able to reconstruct &gt;99% complete sequence for all eight gene segments. We also detected Human Coronavirus and generated a near complete Human Metapneumovirus genome from clinical samples. While further optimisation is required to improve sensitivity, this approach shows promise for the Nanopore platform to be used in the diagnosis and genetic analysis of influenza and other respiratory viruses.","SARS-CoV-2 emerged in December 2019 in Wuhan, China and has since infected over 1.5 million people, of which over 100,000 have died. As SARS-CoV-2 spreads across the planet, speculations remain about the evolution of the virus and the range of human cells that can be infected by SARS-CoV-2. In this study, we report the isolation of SARS-CoV-2 from two COVID-19 patients in Toronto, Canada. We determined the genomic sequences of the two isolates and identified single nucleotide changes in representative populations of our virus stocks. More importantly, we have tested a wide range of human immune cells for infectivity with SARS-CoV-2. We confirm from our studies that human primary peripheral blood mononuclear cells (PBMCs) are not permissive to SARS-CoV-2. As SARS-CoV-2 continues to spread globally, it is essential to monitor any small nucleotide polymorphisms in the virus and to continue to isolate circulating strains of the virus to determine cell susceptibility and pathogenicity using in vitro and in vivo infection models.","ABSTRACTThe advent of the genomic era has made elucidating gene function at large scale a pressing challenge. ORFeome collections, whereby almost all ORFs of a given species are cloned and can be subsequently leveraged in multiple functional genomic approaches, represent valuable resources towards this endeavor. Here we provide novel, genome-scale tools for the study of Candida albicans, a commensal yeast that is also responsible for frequent superficial and disseminated infections in humans. We have generated an ORFeome collection composed of 5,102 ORFs cloned in a Gateway\u2122 donor vector, representing 83% of the currently annotated coding sequences of C. albicans. Sequencing data of the cloned ORFs are available in the CandidaOrfDB database at http://candidaorfeome.eu. We also engineered 49 expression vectors with a choice of promoters, tags, and selection markers and demonstrated their applicability to the study of target ORFs transferred from the C. albicans ORFeome. In addition, the use of the ORFeome in the detection of protein-protein interaction was demonstrated. Mating-compatible strains as well as Gateway\u2122-compatible two-hybrid vectors were engineered, validated and used in a proof of concept experiment. These unique and valuable resources should greatly facilitate future functional studies in C. albicans and the elucidation of mechanisms that underlie its pathogenicity.","AbstractA novel coronavirus (the SARS-CoV-2) has been identified in January 2020 as the causal pathogen for COVID-19 pneumonia, an outbreak started near the end of 2019 in Wuhan, China. The SARS-CoV-2 was found to be closely related to the SARS-CoV, based on the genomic analysis. The Angiotensin converting enzyme 2 protein (ACE2) utilized by the SARS-CoV as a receptor was found to facilitate the infection of SARS-CoV-2, initiated by the binding of the spike protein to the human ACE2. Using homology modeling and molecular dynamics (MD) simulation methods, we report here the detailed structure of the ACE2 in complex with the receptor binding domain (RBD) of the SARS-CoV-2 spike protein. The predicted model is highly consistent with the experimentally determined complex structures. Besides the binding interface reported in the crystal structures, new plausible binding poses were revealed from all-atom MD simulations. The simulation data helped identify critical residues at the complex interface and provided more details about the interactions between the SARS-CoV-2 RBD and human ACE2. Two mutants mimicking rat ACE2 were modeled to study the mutation effects on RBD binding to ACE2. The simulations showed that the N-terminal helix and the K353 are very important for the tight binding of the complex, the mutants were found to alter the binding modes of the CoV2-RBD to ACE2.","AbstractOdor memories are exceptionally robust and essential for animal survival. The olfactory (piriform) cortex has long been hypothesized to encode odor memories, yet the cellular substrates for olfactory learning and memory remain unknown. Here, using intersectional, cFos-based genetic manipulations (\u201cFos-tagging\u201d), we show that olfactory fear conditioning activates sparse and distributed ensembles of neurons in mouse piriform cortex. We demonstrate that chemogenetic silencing of these Fos-tagged piriform ensembles selectively interferes with odor fear memory retrieval, but does not compromise basic odor detection and discrimination. Furthermore, chemogenetic reactivation of piriform neurons that were Fos-tagged during olfactory fear conditioning causes a decrease in exploratory behavior, mimicking odor-evoked fear memory recall. Together, our experiments identify odor-specific ensembles of piriform neurons as necessary and sufficient for odor fear memory recall.","ABSTRACTEnveloped viruses gain entry into host cells by fusing with cellular membranes, a step required for virus replication. Coronaviruses, including the severe acute respiratory syndrome coronavirus (SARS-CoV), Middle East respiratory syndrome coronavirus (MERS-CoV), and infectious bronchitis virus (IBV), fuse at the plasma membrane or use receptor-mediated endocytosis and fuse with endosomes depending on the cell or tissue type. The virus Spike (S) protein mediates fusion with the host cell membrane. We have shown previously that an Abl kinase inhibitor, imatinib, significantly reduces SARS-CoV and MERS-CoV viral titers and prevents endosomal entry by HIV SARS S and MERS S pseudotyped virions. SARS-CoV and MERS-CoV are classified as BSL-3 viruses, which can make experimentation into the cellular mechanisms involved in infection more challenging. Here, we use IBV, a BSL-2 virus, as a model for studying the role of Abl kinase activity during coronavirus infection. We found that imatinib and two specific Abl kinase inhibitors, GNF2 and GNF5, reduce IBV titers by blocking the first round of virus infection. Additionally, all three drugs prevented IBV S-induced syncytia formation prior to the hemifusion step. Our results indicate that membrane fusion (both virus-cell and cell-cell) is blocked in the presence of Abl kinase inhibitors. Studying the effects of Abl kinase inhibitors on IBV will be useful in identifying host cell pathways required for coronavirus infection. This will provide insight into possible therapeutic targets to treat infections by current as well as newly emerging coronaviruses.","AbstractVirophages are satellite-like double stranded DNA viruses whose replication requires the presence of two biological entities, a giant virus and a protist. In this report, we present the first evidence of host range expansion in a virophage. We demonstrated that the Guarani virophage was able to spontaneously expand its viral host range to replicate with two novel giant viruses that were previously nonpermissive to this virophage. We were able to characterize a potential genetic determinant of this cross-species infection. We then highlighted the relevant impact of this host adaptation on giant viruses and protists by demonstrating that coinfection with the mutant virophage abolishes giant virus production and rescues the host cell population from lysis. The results of our study help to elucidate the parasitic lifestyle of virophages and their interactions with giant viruses and protists.","SUMMARYDevelopment of antiviral therapeutics emphasizes minimization of the effective dose and maximization of the toxic dose, first in cell culture and later in animal models. Long-term success of an antiviral therapeutic is determined not only by its efficacy but also by the duration of time required for drug-resistance to evolve. We have developed a microfluidic device comprised of ~6000 wells, with each well containing a microstructure to capture single cells. We have used this device to characterize enterovirus inhibitors with distinct mechanisms of action. In contrast to population methods, single-cell analysis reveals that each class of inhibitor interferes with the viral infection cycle in a manner that can be distinguished by principal component analysis. Single-cell analysis of antiviral candidates reveals not only efficacy but also properties of the members of the viral population most sensitive to the drug, the stage of the lifecycle most affected by the drug, and perhaps even if the drug targets an interaction of the virus with its host.","AbstractSevere acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has spread across more than 200 countries and regions, leading to an unprecedented medical burden and live lost. SARS-CoV-2 specific antivirals or prophylactic vaccines are not available. Neutralizing antibodies provide efficient blockade for viral infection and are a promising category of biological therapies. Using SARS-CoV-2 spike RBD as a bait, we have discovered a panel of humanized single domain antibodies (sdAbs). These sdAbs revealed binding kinetics with the equilibrium dissociation constant (KD) of 0.7~33 nM. The monomeric sdAbs showed half maximal inhibitory concentration (IC50) of 0.003~0.3 \u03bcg/mL in pseudotyped particle neutralization assay, and 0.23~0.50 \u03bcg/mL in authentic SARS-CoV-2 neutralization assay. Competitive ligand-binding data suggested that the sdAbs either completely blocked or significantly inhibited the association between SARS-CoV-2 RBD and viral entry receptor ACE2. Finally, we showed that fusion of the human IgG1 Fc to sdAbs improved their neutralization activity by tens of times. These results reveal the novel SARS-CoV-2 RBD targeting sdAbs and pave a road for antibody drug development.","ABSTRACTCilia are organelles specialized for movement and signaling. To infer when during animal evolution signaling pathways became associated with cilia, we characterized the proteomes of cilia from three organisms: sea urchins, sea anemones and choanoflagellates. From these ciliomes, we identified 437 high confidence ciliary candidate proteins conserved in mammals, including known regulators of Hh, GPCR and TRP channel signaling. The phylogenetic profiles of their ciliary association indicate that the Hh and GPCR pathways were linked to cilia before the origin of bilateria and TRP channels before the origin of animals. We demonstrated that some of the candidates not previously implicated in ciliary biology localized to cilia and further investigated ENKUR, a TRP channel-interacting protein that we identified in the cilia of all three organisms. In animals, ENKUR is expressed by cells with motile cilia, ENKUR localizes to cilia in diverse organisms and, in both Xenopus laevis and mice, ENKUR is required for patterning the left/right axis. Moreover, mutation of ENKUR causes situs inversus in humans. Thus, proteomic profiling of cilia from diverse eukaryotes defines a conserved ciliary proteome, reveals ancient connections to Hh, GPCR and TRP channel signaling, and uncovers a novel ciliary protein that controls vertebrate development and human disease.","Quantitative reverse transcription polymerase chain reaction (RT-qPCR) assay is the gold standard recommended to test for acute SARS-CoV-2 infection. It has been used by the Centers for Disease Control and Prevention (CDC) and several other companies in their Emergency Use Authorization (EUA) assays. With many PCR-based molecular assays, an extraction step is routinely used as part of the protocol. This step can take up a significant amount of time and labor, especially if the extraction is performed manually. Long assay time, partly caused by slow sample preparation steps, has created a large backlog when testing patient samples suspected of COVID-19. Using flu and RSV clinical specimens, we have collected evidence that the RT-qPCR assay can be performed directly on patient sample material from a nasal swab immersed in virus transport medium (VTM) without an RNA extraction step. We have also used this approach to test for the direct detection of SARS-CoV-2 reference materials spiked in VTM. Our data, while preliminary, suggest that using a few microliters of these untreated samples still can lead to sensitive test results. If RNA extraction steps can be omitted without significantly affecting clinical sensitivity, the turn-around time of COVID-19 tests and the backlog we currently experience can be reduced drastically. Next, we will confirm our findings using patient samples.","AbstractFollowing the Severe Acute Respiratory Syndrome coronavirus (SARS-CoV) in 2002 and Middle East Respiratory Syndrome coronavirus (MERS-CoV) in 2012, the novel coronavirus SARS-CoV-2 emerged at the end of 2019 as a highly pathogenic infectious agent that rapidly spread around the world. SARS-CoV-2 shares high sequence homology with SARS-CoV and causes acute, highly lethal pneumonia coronavirus disease 2019 (COVID-19) with clinical symptoms similar to those reported for SARS-CoV. Like other betacoronaviruses, SARS-CoV-2 encode four major structural proteins: Spike (S), Membrane (M), Nucleocapsid (N) and Envelope (E). SARS-CoV E protein is abundant in infected cells and plays a crucial role in viral particle assembly. Moreover, SARS coronaviruses lacking E are attenuated in vivo, suggesting that CoV E may act as a critical virulence factor not only in SARS-CoV but also in the case of the new coronavirus SARS-CoV-2. Ectopic expression of SARS-CoV E was previously shown to trigger apoptosis (cell suicide) of T lymphocytes, lymphopenia being a common feature observed in fatal cases following viral infections. Importantly, T-cell apoptosis was shown to involve interaction between the C-terminal region of SARS-CoV E and the Bcl-2 family member Bcl-xL, which acts as a potent anti-apoptotic protein. Here we provide the first observation that the SARS-CoV E and SARS-CoV-2 E proteins share a conserved Bcl-2 Homology 3 (BH3)-like motif in their C-terminal region, a well-studied motif shown to be necessary for SARS-CoV E binding to Bcl-xL. We used available sequence data for SARS-CoV-2 and related coronaviruses, in combination with structural information, to study the structure to biological activity relationships of SARS-CoV-2 E in relation with its BH3-like motif. Our analysis of the SARS-CoV E interactome further revealed that the predicted SARS-CoV-2 network is extensively wired to the Bcl-2 apoptotic switch. Research is therefore needed to establish if SARS-CoV-2 E targets prosurvival Bcl-2 homologs to modulate cell viability, as part of a coronavirus strategy to interfere with apoptosis. The identification of small molecules (or the repurposing of existing drugs) able to disrupt SARS-CoV-2 E BH3-mediated interactions might provide a targeted therapeutic approach for COVID-19 treatment. Recombinant SARS-CoV-2 expressing an E protein with a deleted or mutated BH3-like motif might also be of interest for the design of a live, attenuated vaccine.","AbstractThe new coronavirus (2019-nCoV) outbreak from December 2019 in Wuhan, Hubei, China, has been declared a global public health emergency. Angiotensin I converting enzyme 2 (ACE2), is the host receptor by 2019-nCov to infect human cells. Although ACE2 is reported to be expressed in lung, liver, stomach, ileum, kidney and colon, its expressing levels are rather low, especially in the lung. 2019-nCoV may use co-receptors/auxiliary proteins as ACE2 partner to facilitate the virus entry. To identify the potential candidates, we explored the single cell gene expression atlas including 119 cell types of 13 human tissues and analyzed the single cell co-expression spectrum of 51 reported RNA virus receptors and 400 other membrane proteins. Consistent with other recent reports, we confirmed that ACE2 was mainly expressed in lung AT2, liver cholangiocyte, colon colonocytes, esophagus keratinocytes, ileum ECs, rectum ECs, stomach epithelial cells, and kidney proximal tubules. Intriguingly, we found that the candidate co-receptors, manifesting the most similar expression patterns with ACE2 across 13 human tissues, are all peptidases, including ANPEP, DPP4 and ENPEP. Among them, ANPEP and DPP4 are the known receptors for human CoVs, suggesting ENPEP as another potential receptor for human CoVs. We also conducted \u201cCellPhoneDB\u201d analysis to understand the cell crosstalk between CoV-targets and their surrounding cells across different tissues. We found that macrophages frequently communicate with the CoVs targets through chemokine and phagocytosis signaling, highlighting the importance of tissue macrophages in immune defense and immune pathogenesis.","AbstractCoronaviruses (CoVs) are important human pathogens with significant zoonotic potential. Progress has been made toward identifying potential vaccine candidates for highly pathogenic human CoVs, including use of attenuated viruses that lack the CoV envelope (E) protein or express E mutants. However, no approved vaccines or anti-viral therapeutics exist. CoVs assemble by budding into the lumen of the early Golgi prior to exocytosis. The small CoV E protein plays roles in assembly, virion release, and pathogenesis. CoV E has a single hydrophobic domain (HD), is targeted to Golgi membranes, and has cation channel activity in vitro. The E protein from the avian infectious bronchitis virus (IBV) has dramatic effects on the secretory system, which requires residues in the HD. Mutation of the HD of IBV E during infection results in impaired growth kinetics, impaired release of infectious virions, accumulation of IBV S protein on the plasma membrane when compared IBV WT infected cells, and aberrant cleavage of IBV S on the surface of virions. We previously reported the formation of two distinct oligomeric pools of IBV E in transfected and infected cells. Disruption of the secretory pathway by IBV E correlates with a form that is likely monomeric, suggesting that the effects on the secretory pathway are independent of E ion channel activity. Here, we present evidence suggesting that the monomeric form of IBV E correlates with a rise in the pH of the Golgi lumen. We demonstrate that infection with IBV induces neutralization of Golgi luminal pH, promoting a model in which IBV E alters the secretory pathway through interaction with host cells factors, protecting IBV spike protein (S) from premature cleavage and leading to the efficient release of infectious virus from the cells.","AbstractThe MinION was used to evaluate upper respiratory disease infections using both whole genome amplification (WGA), targeted sequencing, and was found to have tremendous potential for field use. The MinION nanopore sequencer was been released to community testers for evaluation using a variety of sequencing applications. The MinION was used to evaluate upper respiratory disease infections using both whole genome amplification and targeted sequencing, and was found to have tremendous potential for field use. In this study, we tested the ability of the MinION nanopore sequencer to accurately identify and differentiate clinical bacterial and viral samples via targeted sequencing and whole genome sequencing. The current nanopore technology has limitations with respect to error rate but has steadily improved with development of new flow cells and kits. Upper respiratory disease organisms were successfully identified and differentiated down to the strain level with 87-98% alignment to our reference genome database. The ability to differentiate strains by amplicon and whole genome sequencing on the MinION was accomplished despite the observed average per 100-base error rate averaged 1.2E-01. This study offers evidence of the utility of sequencing to identify and differentiate both viral and bacterial species present within clinical samples.","AbstractCoronavirus disease 2019 (COVID-19) is a global health concern as it continues to spread within China and beyond. The causative agent of this disease, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), belongs to the genus Betacoronavirus which also includes severe acute respiratory syndrome related coronavirus (SARSr-CoV) and Middle East respiratory syndrome related coronavirus (MERSr-CoV). Codon usage of viral genes are believed to be subjected to different selection pressures in different host environments. Previous studies on codon usage of influenza A viruses can help identify viral host origins and evolution trends, however, similar studies on coronaviruses are lacking. In this study, global correspondence analysis (CA), within-group correspondence analysis (WCA) and between-group correspondence analysis (BCA) were performed among different genes in coronavirus viral sequences. The amino acid usage pattern of SARS-CoV-2 was generally found similar to bat and human SARSr-CoVs. However, we found greater synonymous codon usage differences between SARS-CoV-2 and its phylogenetic relatives on spike and membrane genes, suggesting these two genes of SARS-CoV-2 are subjected to different evolutionary pressures.","As a first step toward the development of diagnostic and therapeutic tools to fight the Coronavirus disease (COVID-19), it is important to characterize CD8+ T cell epitopes in the SARS-CoV-2 peptidome that can trigger adaptive immune responses. Here, we use RosettaMHC, a comparative modeling approach which leverages existing high-resolution X-ray structures from peptide/MHC complexes available in the Protein Data Bank, to derive physically realistic 3D models for high-affinity SARS-CoV-2 epitopes. We outline an application of our method to model 439 9mer and 279 10mer predicted epitopes displayed by the common allele HLA-A*02:01, and we make our models publicly available through an online database (https://rosettamhc.chemistry.ucsc.edu). As more detailed studies on antigen-specific T cell recognition become available, RosettaMHC models of antigens from different strains and HLA alleles can be used as a basis to understand the link between peptide/HLA complex structure and surface chemistry with immunogenicity, in the context of SARS-CoV-2 infection.","ABSTRACTBackgroundPatients with frequent exacerbations represent a chronic obstructive pulmonary disease (COPD) sub-group requiring better treatment options. The aim of this study was to determine the innate immune mechanisms that underlie susceptibility to frequent exacerbations in COPD.MethodsWe measured sputum expression of immune mediators and bacterial loads in samples from patients with COPD at stable state and during virus-associated exacerbations. Ex vivo immune responses to rhinovirus infection in differentiated bronchial epithelial cells (BECs) sampled from patients with COPD were additionally evaluated. Patients were stratified as frequent exacerbators (\u22652 exacerbations in the preceding year) or infrequent exacerbators (&lt;2 exacerbations in the preceding year) with comparisons made between these groups.ResultsFrequent exacerbators had reduced sputum cell mRNA expression of the anti-viral immune mediators type I and III interferons and reduced interferon-stimulated gene (ISG) expression when clinically stable and during virus-associated exacerbation. RV-induction of interferon and ISGs ex vivo was also impaired in differentiated BECs from frequent exacerbators. Frequent exacerbators also had reduced sputum levels of the anti-microbial peptide mannose-binding lectin (MBL)-2 with an associated increase in sputum bacterial loads at 2 weeks following virus-associated exacerbation onset. MBL-2 levels correlated negatively with bacterial loads during exacerbation.ConclusionThese data implicate deficient airway innate immunity in the increased propensity to exacerbations observed in some patients with COPD. Therapeutic approaches to boost innate antimicrobial immunity in the lung could be a viable strategy for prevention/treatment of frequent exacerbations.","As recently classified as a pandemic by WHO, novel Corononavirus 2019 has affected almost every corner of the globe causing human deaths in a range of hundred thousands. The virus having its roots in Wuhan (China) has been spread over the world by its own property to change itself accordingly. These changes correspond to its transmission and pathogenicity due to which the concept of social distancing appeared into the picture. In this paper, a few findings from the whole genome sequence analysis of viral genome sequences submitted from India are presented. The data used for analysis comprises 440 collective genome sequences of virus submitted in GenBank, GISAID, and SRA projects, from around the world as well as 28 viral sequences from India. Multiple sequence alignment of all genome sequences was performed and analysed. A novel non-synonymous mutation 4809C&gt;T (S1515F) in NSP3 gene of SARS-CoV2 Indian strains is reported along with other frequent and important changes from around the world: 3037C&gt;T, 14408C&gt;T, and 23403A&gt;G. The novel change was observed in samples collected in the month of March, whereas was found to be absent in samples collected in January with the respective persons travel history to China. Phylogenetic analysis clustered the sequences with this change as one separate clade. Mutation was predicted as stabilising change by insilco tool DynaMut. A second patient in the world to our knowledge with multiple (Wuhan and USA) strain contraction was observed in this study. The infected person is among the two early infected patients with travel history to China. Strains sequenced in Iran stood out to have different variants, as most of the reported frequent variants were not observed. The objective of this paper is to highlight the similarities and changes observed in the submitted Indian viral strains. This helps to keep track on the activity, that how virus is changing into a new subtype. Major strains observed were European with the novel change in India and other being emergent clade of Iran. Its important to observe the changes in NSP3 gene, as this gene has been reported with extensive positive selection as well as potential drug target. Extensive Positive Selection Drives the Evolution of Nonstructural Proteins. With the limited number of sequences this was the only frequent novel non-synonymous change observed from Indian strains, thereby making this change vulnerable for investigation in future. This paper has a special focus on tracking of Indian viral sequences submitted in public domain.","AbstractCells of all mammals recognize double-stranded RNA (dsRNA) as a foreign material. In response, they release interferons (IFNs) and activate a ubiquitously expressed pseudokinase/endoribonuclease RNase L. RNase L executes regulated RNA decay and halts global translation. Here we developed a biosensor for 2\u2019,5\u2019-oligoadenylate (2-5A), the natural activator of RNase L. We found that 2-5A was acutely synthesized by cells in response to dsRNA sensing, which immediately triggered cellular RNA cleavage by RNase L and arrested host protein synthesis. However, translation-arrested cells still transcribed IFN-stimulated genes (ISGs) and secreted IFNs of types I and III (IFN-\u03b2 and IFN-\u03bb). Our data suggests that IFNs escape from the action of RNase L on translation. We propose that 2-5A/RNase L pathway serves to rapidly and accurately suppress basal protein synthesis, preserving privileged production of defense proteins of the innate immune system.SignificanceRNase L is a mammalian enzyme that can stop global protein synthesis during interferon response. Cells must balance the need to make interferons (which are proteins) with the risk to lose cell-wide translation due to RNase L. This balance can most simply be achieved if RNase L was activated late in the interferon response. However, we show by engineering a biosensor for the RNase L pathway, that on the contrary, RNase L activation precedes interferon synthesis. Further, translation of interferons evades the action of RNase L. Our data suggest that RNase L facilitates a switch of protein synthesis from homeostasis to specific needs of innate immune signaling.","Early characterization is essential to control the spread of emerging viruses, such as the Zika Virus outbreak in 2014. A major challenge is the identification of potential hosts for novel viruses. We introduce an algorithm to identify the host range of a virus from its raw genome sequence that will be a useful tool to understand host-virus relationships.","AbstractThe novel coronavirus (CoV) disease termed COVID-19 (Coronavirus Disease-19) caused by SARS-CoV-2 (Severe Acute Respiratory Syndrome Coronavirus-2) is causing a massive pandemic worldwide, threatening public health systems across the globe. During this ongoing COVID-19 outbreak, nucleic acid detection has played an important role in early diagnosis. Here we report a SARS-CoV-2 detection protocol using a CRISPR-based CRISPR diagnostic platform - CDetection (Cas12b-mediated DNA detection). By combining sample treatment protocols and nucleic acid amplification methods with CDetection, we have established an integrated viral nucleic acid detection platform - CASdetec (CRISPR-assisted detection). The detection limit of CASdetec for SARS-CoV-2 pseudovirus is 1 \u00d7 104 copies/mL, with no cross reactivity observed. Our assay design and optimization process can provide guidance for future CRISPR-based nucleic acid detection assay development and optimization.","AbstractAs the initial step in carbohydrate catabolism in cells, the substrate-specific transporters via active transport and facilitated diffusion play a decisive role in passage of sugars through the plasma membrane into the cytoplasm. The SecY complex (SecYEG) in bacteria forms a membrane channel responsible for protein translocation. This work demonstrates that weakening the sealability of the SecY channel allowed free diffusion of sugars, including glucose, fructose, mannose, xylose, arabinose, and lactose, into the engineered cells, facilitating its rapid growth on a wide spectrum of monosaccharides and bypassing/reducing stereospecificity, transport saturation, competitive inhibition, and carbon catabolite repression (CCR), which are usually encountered with the specific sugar transporters. The SecY channel is structurally conserved in prokaryotes, thus it may be engineered to serve as a unique and universal transporter for bacteria to passage sugars as demonstrated in Escherichia coli and Clostridium acetobutylicum.","SummaryThe unprecedented epidemic of pneumonia caused by a novel coronavirus, HCoV-19, in China and beyond has caused public health concern at a global scale. Although bats are regarded as the most likely natural hosts for HCoV-191,2, the origins of the virus remain unclear. Here, we report a novel bat-derived coronavirus, denoted RmYN02, identified from a metagenomics analysis of samples from 227 bats collected from Yunnan Province in China between May and October, 2019. RmYN02 shared 93.3% nucleotide identity with HCoV-19 at the scale of the complete virus genome and 97.2% identity in the 1ab gene in which it was the closest relative of HCoV-19. In contrast, RmYN02 showed low sequence identity (61.3%) to HCoV-19 in the receptor binding domain (RBD) and might not bind to angiotensin-converting enzyme 2 (ACE2). Critically, however, and in a similar manner to HCoV-19, RmYN02 was characterized by the insertion of multiple amino acids at the junction site of the S1 and S2 subunits of the Spike (S) protein. This provides strong evidence that such insertion events can occur in nature. Together, these data suggest that HCoV-19 originated from multiple naturally occurring recombination events among those viruses present in bats and other wildlife species.","The greatly accelerated economic growth during the Anthropocene has resulted in astonishing improvements in many aspects of human well-being, but has also caused the acceleration of risks, such as the interlinked biodiversity and climate crisis. Here, we report on another risk: the accelerated infectious disease risk associated with the number and geographic spread of human infectious disease outbreaks. Using the most complete, reliable, and up-to-date database on human infectious disease outbreaks (GIDEON), we show that the number of disease outbreaks, the number of diseases involved in these outbreaks, and the number of countries affected have increased during the entire Anthropocene. Furthermore, the spatial distribution of these outbreaks is becoming more globalized in the sense that the overall modularity of the disease networks across the globe has decreased, meaning disease outbreaks have become increasingly pandemic in their nature. This decrease in modularity is correlated with the increase in air traffic. We finally show that those countries and regions which are most central within these disease networks tend to be countries with higher GDPs. Therefore, one cost of increased global mobility and greater economic growth is the increased risk of disease outbreaks and their faster and wider spread. We briefly discuss three different scenarios which decision-makers might follow in light of our results.","AbstractHearing and balance deficits have been reported during and following treatment with the antimalarial drug chloroquine. However, experimental work examining the direct actions of chloroquine on mechanoreceptive hair cells in common experimental models is lacking. This study examines the effects of chloroquine on hair cells using two common experimental models: the zebrafish lateral line and neonatal mouse cochlear cultures. Zebrafish larvae were exposed to varying concentrations of chloroquine phosphate or hydroxychloroquine for 1 hr or 24 hr, and hair cells assessed by antibody staining. A significant, dose-dependent reduction in the number of surviving hair cells was seen across conditions for both exposure periods. Hydroxycholroquine showed similar toxicity. In mouse cochlear cultures, chloroquine damage was specific to outer hair cells in tissue from the cochlear basal turn, consistent with susceptibility to other ototoxic agents. These findings suggest a need for future studies employing hearing and balance monitoring during exposure to chloroquine and related compounds, particularly with interest in these compounds as therapeutics against viral infections including coronavirus.","ABSTRACTInfectious bronchitis virus (IBV) is a member of the genus Gammacoronavirus and the causative agent of avian infectious bronchitis. IBV has a single-stranded, positive-sense RNA genome ~27 kb in length and, like all coronaviruses, produces a set of sub-genomic messenger RNAs (sgmRNAs) synthesised via the viral polymerase. Here, we used RNA sequencing (RNASeq) and ribosome profiling (RiboSeq) to delineate gene expression in the IBV M41-CK and Beau-CK strains at sub-codon resolution. Quantification of reads flanking the programmed ribosomal frameshifting (PRF) signal at the genomic RNA ORF1a/ORF1b junction revealed that PRF in IBV is highly efficient (33\u201340%), consistent with in vitro measurements. Triplet phasing of the profiling data allowed precise determination of reading frames and revealed the translation of two intergenic genes (4b and 4c on sgmRNA4), which are widely conserved across IBV isolates. RNASeq revealed two novel transcription junction sites in the attenuated Beau-CK strain, one of which would generate a sgmRNA encoding a ribosomally occupied ORF in the viral 3\u2019 untranslated region (dORF). Within IBV transcripts, the nucleocapsid (N) protein was unexpectedly found to be inefficiently translated, despite being an abundant structural component of mature IBV virions. Finally, we demonstrate that the host cell response to IBV occurs primarily at the level of transcription, with a global up-regulation of immune-related mRNA transcripts following infection, and comparatively modest changes in the translation efficiencies of host genes.IMPORTANCEIBV is a major avian pathogen and presents a substantial economic burden to the poultry industry. Improved vaccination strategies are urgently needed to curb the global spread of this pathogen, and the development of suitable vaccine candidates will be aided by an improved understanding of IBV molecular biology. Our high-resolution data have enabled a precise study of transcription and translation in both pathogenic and attenuated forms of IBV, and expand our understanding of gammacoronaviral gene expression. We demonstrate that gene expression shows considerable intra-species variation, with single nucleotide polymorphisms associated with altered production of sgmRNA transcripts, and our RiboSeq data sets enabled us to uncover novel ribosomally occupied ORFs in both strains. We also identify numerous cellular genes and gene networks that are differentially expressed during virus infection, giving insights into the host cell reponse to IBV infection.","AbstractBackgroundIn recent years, metagenomic Next-Generation Sequencing (mNGS) has increasingly been used for an accurate assumption-free virological diagnosis. However, the systematic workflow evaluation on clinical respiratory samples and implementation of quality controls (QCs) is still lacking.MethodsA total of 3 QCs were implemented and processed through the whole mNGS workflow: a notemplate-control to evaluate contamination issues during the process; an internal and an external QC to check the integrity of the reagents, equipment, the presence of inhibitors, and to allow the validation of results for each sample. The workflow was then evaluated on 37 clinical respiratory samples from patients with acute respiratory infections previously tested for a broad panel of viruses using semi-quantitative real-time PCR assays (28 positive samples including 6 multiple viral infections; 9 negative samples). Selected specimens included nasopharyngeal swabs (n = 20), aspirates (n = 10), or sputums (n = 7).ResultsThe optimal spiking level of the internal QC was first determined in order to be sufficiently detected without overconsumption of sequencing reads. According to QC validation criteria, mNGS results were validated for 34/37 selected samples. For valid samples, viral genotypes were accurately determined for 36/36 viruses detected with PCR (viral genome coverage ranged from 0.6% to 100%, median = 67.7%). This mNGS workflow allowed the detection of DNA and RNA viruses up to a semi-quantitative PCR Ct value of 36. The six multiple viral infections involving 2 to 4 viruses were also fully characterized. A strong correlation between results of mNGS and real-time PCR was obtained for each type of viral genome (R2 ranged from 0.72 for linear single-stranded (ss) RNA viruses to 0.98 for linear ssDNA viruses).ConclusionsAlthough the potential of mNGS technology is very promising, further evaluation studies are urgently needed for its routine clinical use within a reasonable timeframe. The approach described herein is crucial to bring standardization and to ensure the quality of the generated sequences in clinical setting. We provide an easy-to-use single protocol successfully evaluated for the characterization of a broad and representative panel of DNA and RNA respiratory viruses in various types of clinical samples.","AbstractThe emergence and rapid worldwide spread of the novel coronavirus disease, COVID-19, has prompted concerted efforts to find successful treatments. The causative virus, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), uses its spike (S) protein to gain entry into host cells. Therefore, the S protein presents a viable target to develop a directed therapy. Here, we deployed an integrated artificial intelligence with molecular dynamics simulation approach to provide new details of the S protein structure. Based on a comprehensive structural analysis of S proteins from SARS-CoV-2 and previous human coronaviruses, we found that the protomer state of S proteins is structurally flexible. Without the presence of a stabilizing beta sheet from another protomer chain, two regions in the S2 domain and the hinge connecting the S1 and S2 subunits lose their secondary structures. Interestingly, the region in the S2 domain was previously identified as an immunodominant site in the SARS-CoV-1 S protein. We anticipate that the molecular details elucidated here will assist in effective therapeutic development for COVID-19.","AbstractIn eukaryotes, the degradation of cellular mRNAs is accomplished by Xrn1p and the cytoplasmic exosome. Because viral RNAs often lack canonical caps or poly-A tails, they can also be vulnerable to degradation by these host exonucleases. Yeast lack sophisticated mechanisms of innate and adaptive immunity, but do use RNA degradation as an antiviral defense mechanism. One model is that the RNA of yeast viruses is subject to degradation simply as a side effect of the intrinsic exonuclease activity of proteins involved in RNA metabolism. Contrary to this model, we find a highly refined, species-specific relationship between Xrn1p and the \u201cL-A\u201d totiviruses of different Saccharomyces yeast species. We show that the gene XRN1 has evolved rapidly under positive natural selection in Saccharomyces yeast, resulting in high levels of Xrn1p protein sequence divergence from one yeast species to the next. We also show that these sequence differences translate to differential interactions with the L-A virus, where Xrn1p from S. cerevisiae is most efficient at controlling the L-A virus that chronically infects S. cerevisiae, and Xrn1p from S. kudriavzevii is most efficient at controlling the L-A-like virus that we have discovered within S. kudriavzevii. All Xrn1p orthologs are equivalent in their interaction with another virus-like parasite, the Ty1 retrotransposon. Thus, the activity of Xrn1p against totiviruses is not simply an incidental consequence of the enzymatic activity of Xrn1p, but rather Xrn1p co-evolves with totiviruses to maintain its potent antiviral activity and limit viral propagation in Saccharomyces yeasts. Consistent with this, we demonstrated that Xrn1p physically interacts with the Gag protein encoded by the L-A virus, suggesting a host-virus interaction that is more complicated than just Xrn1p-mediated nucleolytic digestion of viral RNAs.","AbstractThe 2019-nCoV outbreak has become a global health risk. Editing by host deaminases is an innate restriction process to counter viruses, and it is not yet known whether it operates against coronaviruses. Here we analyze RNA sequences from bronchoalveolar lavage fluids derived from two Wuhan patients. We identify nucleotide changes that may be signatures of RNA editing: Adenosine-to-Inosine changes from ADAR deaminases and Cytosine-to-Uracil changes from APOBEC ones. A mutational analysis of genomes from different strains of human-hosted Coronaviridae reveals patterns similar to the RNA editing pattern observed in the 2019-nCoV transcriptomes. Our results suggest that both APOBECs and ADARs are involved in Coronavirus genome editing, a process that may shape the fate of both virus and patient.","AbstractWe live in a world of emerging new diseases and old diseases resurging in more aggressive forms. Drug development by pharmaceutical companies is a market-driven and costly endeavor, and thus it is often a challenge when drugs are needed for diseases endemic only to certain regions or which affect only a few patients. However, biomedical open data is accessible and reusable for reanalysis and generation of a new hypotheses and discovery. In this study, we leverage biomedical data and tools to analyze available data on Nipah Virus (NiV) infection. NiV infection is an emerging zoonosis that is transmissible to humans and is associated with high mortality rates. In this study, explored the application of computational drug repositioning and chemogenomic enrichment analyses using host transcriptome data to match drugs that could reverse the virus-induced gene signature. We performed analyses using two gene signatures: i) A previously published gene signature (n=34), and ii) a gene signature generated using the characteristic direction method (n= 5,533). Our predictive framework suggests that several drugs including FDA approved therapies like beclometasone, trihexyphenidyl, S-propranolol etc. could modulate the NiV infection induced gene signatures in endothelial cells. A target specific analysis of CXCL10 also suggests the potential application of Eldelumab, an investigative therapy for Crohn\u2019s disease and ulcerative colitis, as a putative candidate for drug repositioning. To conclude, we also discuss challenges and opportunities in clinical trials (n-of-1 and adaptive trials) for repositioned drugs. Further follow-up studies including biochemical assays and clinical trials are required to identify effective therapies for clinical use. Our proof-of-concept study highlights that translational bioinformatics methods including gene expression analyses and computational drug repositioning could augment epidemiological investigations in the context of an emerging disease with no effective treatment.","AbstractThe COVID-19 disease has plagued over 110 countries and has resulted in over 4,000 deaths within 10 weeks. We compare the interaction between the human ACE2 receptor and the SARS-CoV-2 spike protein with that of other pathogenic coronaviruses using molecular dynamics simulations. SARS-CoV, SARS-CoV-2, and HCoV-NL63 recognize ACE2 as the natural receptor but present a distinct binding interface to ACE2 and a different network of residue-residue contacts. SARS-CoV and SARS-CoV-2 have comparable binding affinities achieved by balancing energetics and dynamics. The SARS-CoV-2\u2013ACE2 complex contains a higher number of contacts, a larger interface area, and decreased interface residue fluctuations relative to SARS-CoV. These findings expose an exceptional evolutionary exploration exerted by coronaviruses toward host recognition. We postulate that the versatility of cell receptor binding strategies has immediate implications on therapeutic strategies.One Sentence SummaryMolecular dynamics simulations reveal a temporal dimension of coronaviruses interactions with the host receptor.","Influenza viruses, corona viruses and related pneumotropic viruses cause sickness and death partly by inducing a hyper-proinflammatory response by immune cells and cytokines in the host airway. Here we show that the cardiac glycoside digitoxin suppresses this response induced by influenza virus strain A/Wuhan/H3N2/359/95 in the cotton rat lung. The cytokines TNFa, GRO/KC, MIP2, MCP1, TGFb, and IFNg. are significantly reduced. Since the hyper-proinflammatory overproduction of cytokines is a host response, we suggest that digitoxin may have therapeutic potential for not only influenza and but also for coronovirus infections.","ABSTRACTReproductive success in the eight founder strains of the Collaborative Cross (CC) was measured using a diallel-mating scheme. Over a 48-month period we generated 4,448 litters, and 24,782 weaned pups were used across 16 different published experiments. We identified factors that affect the average litter size in a cross by estimating the overall contribution of parent-of-origin, heterosis, inbred, and epistatic effects using a Bayesian zero-truncated overdispersed Poisson mixed model. The phenotypic variance of litter size has a substantial contribution (79%) from unexplained and environmental sources, but no detectable effect of seasonality. Most of the explained variance was due to additive effects (9.2%) and parental sex (maternal vs paternal strain; 5.8%), with epistasis accounting for 3.4%. Within the parental effects, the effect of the dam\u2019s strain explained more than the sire\u2019s strain (13.2% vs. 1.8%), and the dam\u2019s strain effects account for 74.2% of total variation explained. Dams from strains C57BL/6J and NOD/ShiLtJ increased the expected litter size by a mean of 1.66 and 1.79 pups, whereas dams from strains WSB/EiJ, PWK/PhJ, and CAST/EiJ reduced expected litter size by a mean of 1.51, 0.81, and 0.90 pups. Finally, there was no strong evidence for strain-specific effects on sex ratio distortion. Overall, these results demonstrate that strains vary substantially in their reproductive ability depending on their genetic background and that litter size is largely determined by dam.strain rather than sire.strain effects, as expected. This analysis adds to our understanding of factors that influence litter size in mammals, and also helps to explain breeding successes and failures in the extinct lines and surviving CC strains.","AbstractIn this paper, ACE2 gene of pigs was cloned and the purified protein was obtained via the prokaryotic expression system. Polyclonal antibody of high titer and sensitivity was obtained using Wastar rats immunization method and is then used to determine of the expression of ACE2 using immunohistochemistry. The sequence of ACE2 in pigs covered 2418 nucleotides and coded 805 amino acid (aa) residues. Sequence homology analysis showed that the ACE2 sequence in pigs is highly conserved among species at the nucleotide and amino acid levels. Genetic evolution analysis revealed that ACE2 gene in pigs has the shortest genetic distance with that in goats while residing in a totally different branch from that in zebra fishes. Analysis of protein structure predicted that ACE2 protein is a transmembrane secreted protein with high hydrophilicity, containing a signal peptide sequences locating between 1aa to 17aa. The ACE2 fusion protein expressed (under the induction with 1.0 mmol/L IPTG for 10 h) was of approximately 100 kDa and mainly existed in inclusion body. Wastar rats immunization showed that the titer of the anti-ACE2 antiserum in rats was 1: 3200. Western blot showed that the antibody binds specifically. Immunohistochemistry showed that the ACE2 protein was expressed in all major tissues of pigs. It is the first time that polyclonal antibody of ACE2 in pigs was obtained and the expression of ACE2 was confirmed. These results will provide a basis for investigating on ACE2\u2019s biological activity in pigs.","AbstractIntrinsically disordered regions (IDRs) challenge the well-established sequence-structure-function paradigm for describing protein function and evolution. Here, we direct a combination of biophysical and cellular studies to further our understanding of how the intrinsically disordered C-terminal tail of FtsZ contributes to cell division in rod-shaped bacteria. FtsZ is a modular protein that encompasses a conserved GTPase domain and a highly variable intrinsically disordered C-terminal tail (CTT). The CTT is essential for forming the cytokinetic Z-ring. Despite poor sequence conservation of the CTT, the patterning of oppositely charged residues, which refers to the extent of linear mixing / segregation of oppositely charged residues within CTT sequences is bounded within a narrow range. To assess the impact of evolutionary bounds on charge patterning within CTT sequences we performed experiments, aided by sequence design, to quantify the impact of changing the patterning of oppositely charged residues within the CTT on the functions of FtsZ from B. subtilis. Z-ring formation is robust if and only if the extent of linear mixing / segregation of oppositely charged residues within the CTT sequences is within evolutionarily observed bounds. Otherwise, aberrant, CTT-mediated, FtsZ assemblies impair Z-ring formation. The complexities of CTT sequences also have to be above a threshold value because FtsZ variants with low complexity CTTs are not tolerated in cells. Taken together, our results suggest that CTT sequences have evolved to be \u201cjust right\u201d and that this is achieved through an optimal extent of charge patterning while maintaining the sequence complexity above a threshold value.","AbstractTranslational control of memory processes is a tightly regulated process where the coordinated interaction and modulation of translation factors provides a permissive environment for protein synthesis during memory formation. Existing methods used to block translation lack the spatiotemporal precision to investigate cell-specific contributions to consolidation of long-term memories. Here, we have developed a novel chemogenetic mouse resource for cell type-specific and drug-inducible protein synthesis inhibition (ciPSI) that utilizes an engineered version of the catalytic kinase domain of dsRNA-activated protein (PKR). ciPSI allows rapid and reversible phosphorylation of eIF2\u03b1 causing a block on general translation by 50% in vivo. Using this resource, we discovered that temporally structured pan-neuronal protein synthesis is required for consolidation of long-term auditory threat memory. Targeted protein synthesis inhibition in CamK2\u03b1 expressing glutamatergic neurons in lateral amygdala (LA) impaired long-term memory, which was recovered with artificial chemogenetic reactivation at the cost of stimulus generalization. Conversely, genetically reducing phosphorylation of eIF2\u03b1 in CamK2\u03b1 positive neurons in LA enhanced memory strength, but was accompanied with reduced memory fidelity and behavior inflexibility. Our findings provide evidence for a finely tuned translation program during consolidation of long-term threat memories.","AbstractNatural selection acting on synonymous mutations in protein-coding genes influences genome composition and evolution. In viruses, introducing synonymous mutations in genes encoding structural proteins can drastically reduce viral growth, providing a means to generate potent, live attenuated vaccine candidates. However, an improved understanding of what compositional features are under selection and how combinations of synonymous mutations affect viral growth is needed to predictably attenuate viruses and make them resistant to reversion. We systematically recoded all non-overlapping genes of the bacteriophage \u03a6X174 with codons rarely used in its E. coli host. The fitness of recombinant viruses decreases as additional deoptimizing mutations are made to the genome, although not always linearly, and not consistently across genes. Combining deoptimizing mutations may reduce viral fitness more or less than expected from the effect size of the constituent mutations and we point out difficulties in untangling correlated compositional features. We test our model by optimizing the same genes and find that the relationship between codon usage and fitness does not hold for optimization, suggesting that wild-type \u03a6X174 is at a fitness optimum. This work highlights the need to better understand how selection acts on patterns of synonymous codon usage across the genome and provides a convenient system to investigate the genetic determinants of virulence.","Pandemics, even more than other scientific questions, require swift integration of knowledge and identifiers. In a setting where there is a large number of loosely connected projects and initiatives, we need a common ground, also known as a \"commons\". Wikidata, a public knowledge graph aligned with Wikipedia, is such a commons, but Wikidata may not always have the right schema for the urgent questions. In this paper, we address this problem by showing how a data schema required for the integration can be modelled with entity schemas represented by shape expressions. As a telling example, we describe the process of aligning resources on the genomics of the SARS-CoV-2 virus and related viruses as well as how shape expressions can be defined for Wikidata helping others studying the SARS-CoV-2 pandemic. How this model can be used to make data between various resources interoperable, is demonstrated by integrating data from NCBI Taxonomy, NCBI Genes, UniProt, and WikiPathways. Based on that model, a set of automated applications or bots were written for regular updates of these sources in Wikidata and added to a platform for automatically running these updates. Although this workflow is developed and applied in the context of the SARS-CoV-2 pandemic, it was also applied to other human coronaviruses (MERS, SARS, SARS-CoV-2, Human Coronavirus NL63, Human coronavirus 229E, Human coronavirus HKU1, Human coronavirus OC4) to demonstrate its broader applicability.","ABSTRACTSevere Acute Respiratory Syndrome Coronavirus 2 is rapidly spreading around the world. There is no existing vaccine or proven drug to prevent infections and stop virus proliferation. Although this virus is similar to human and animal SARS- and MERS-CoVs the detailed information about SARS-CoV-2 proteins structures and functions is urgently needed to rapidly develop effective vaccines, antibodies and antivirals. We applied high-throughput protein production and structure determination pipeline at the Center for Structural Genomics of Infectious Diseases to produce SARS-CoV-2 proteins and structures. Here we report the high-resolution crystal structure of endoribonuclease Nsp15/NendoU from SARS-CoV-2 \u2013 a virus causing current world-wide epidemics. We compare this structure with previously reported models of Nsp15 from SARS and MERS coronaviruses.","AbstractSevere acute respiratory syndrome coronavirus 2 (SARS-CoV-2) poses an acute threat to public health and the world economy, especially because no approved specific drugs or vaccines are available. Pharmacological modulation of metabolism-dependent cellular pathways such as autophagy reduced propagation of highly pathogenic Middle East respiratory syndrome (MERS)-CoV.Here we show that SARS-CoV-2 infection limits autophagy by interfering with multiple metabolic pathways and that compound-driven interventions aimed at autophagy induction reduce SARS-CoV-2 propagation in vitro. In-depth analyses of autophagy signaling and metabolomics indicate that SARS-CoV-2 reduces glycolysis and protein translation by limiting activation of AMP-protein activated kinase (AMPK) and mammalian target of rapamycin complex 1 (mTORC1). Infection also downregulates autophagy-inducing spermidine, and facilitates AKT1/SKP2-dependent degradation of autophagy-initiating Beclin-1 (BECN1). Targeting of these pathways by exogenous administration of spermidine, AKT inhibitor MK-2206, and the Beclin-1 stabilizing, antihelminthic drug niclosamide inhibited SARS-CoV-2 propagation by 85, 88, and &gt;99%, respectively. In sum, SARS-CoV-2 infection causally diminishes autophagy. A clinically approved and well-tolerated autophagy-inducing compound shows potential for evaluation as a treatment against SARS-CoV-2.","Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is the causative agent of the Coronavirus disease 2019 (COVID-19) which is currently negatively affecting the population and disrupting the global economy. SARS-CoV-2 belongs to the +RNA virus family that utilize single-stranded positive-sense RNA molecules as genomes. SARS-CoV-2, like other coronaviruses, has an unusually large genome for a +RNA virus that encodes four structural proteins - the matrix (M), small envelope (E), spike (S) and nucleocapsid phosphoprotein (N) - and sixteen nonstructural proteins (nsp1-16) that together ensure replication of the virus in the host cell. The nucleocapsid phosphoprotein N is essential for linking the viral genome to the viral membrane. Its N-terminal RNA binding domain (N-NTD) captures the RNA genome while the C-terminal domain anchors the ribonucleoprotein complex to the viral membrane via its interaction with the M protein. Here, we characterized the structure of the N-NTD and its interaction with RNA using NMR spectroscopy. We observed a positively charged canyon on the surface of the N-NTD lined with arginine residues suggesting a putative RNA binding site. Next, we performed an NMR titration experiment using an RNA duplex. The observed changes in positions of signals in the N-NTD NMR spectra allowed us to construct a model of the N-NTD in complex with RNA.","Emerging and re-emerging infectious diseases, such as SARS, MERS, Zika and highly pathogenic influenza present a major threat to public health1\u20133. Despite intense research effort, how, when and where novel diseases appear are still the source of considerable uncertainly. A severe respiratory disease was recently reported in the city of Wuhan, Hubei province, China. At the time of writing, at least 62 suspected cases have been reported since the first patient was hospitalized on December 12nd 2019. Epidemiological investigation by the local Center for Disease Control and Prevention (CDC) suggested that the outbreak was associated with a sea food market in Wuhan. We studied seven patients who were workers at the market, and collected bronchoalveolar lavage fluid (BALF) from one patient who exhibited a severe respiratory syndrome including fever, dizziness and cough, and who was admitted to Wuhan Central Hospital on December 26th 2019. Next generation metagenomic RNA sequencing4 identified a novel RNA virus from the family Coronaviridae designed WH-Human-1 coronavirus (WHCV).Phylogenetic analysis of the complete viral genome (29,903 nucleotides) revealed that WHCV was most closely related (89.1% nucleotide similarity similarity) to a group of Severe Acute Respiratory Syndrome (SARS)-like coronaviruses (genus Betacoronavirus, subgenus Sarbecovirus) previously sampled from bats in China and that have a history of genomic recombination. This outbreak highlights the ongoing capacity of viral spill-over from animals to cause severe disease in humans.","AbstractTemperate phage genomes are highly variable mosaic collections of genes that infect a bacterial host, integrate into the host\u2019s genome or replicate as low copy number plasmids, and are regulated to switch from the lysogenic to lytic cycles to generate new virions and escape their host. Genomes from most Bacterial phyla contain at least one or more prophages. We updated our PhiSpy algorithm to improve detection of prophages and to provide a web-based framework for PhiSpy. We have used this algorithm to identify 36,488 prophage regions from 11,941 bacterial genomes, including almost 600 prophages with no known homology to any proteins. Transfer RNA genes were abundant in the prophages, many of which alleviate the limits of translation efficiency due to host codon bias and presumably enable phages to surpass the normal capacity of the hosts\u2019 translation machinery. We identified integrase genes in 15,765 prophages (43% of the prophages). The integrase was routinely located at either end of the integrated phage genome, and was used to orient and align prophage genomes to reveal their underlying organization. The conserved genome alignments of phages recapitulate early, middle, and late gene order in transcriptional control of phage genes, and demonstrate that gene order, presumably selected by transcription timing and/or coordination among functional modules has been stably conserved throughout phage evolution.","AbstractHerein we describe the Archaea diversity in a shallow pond in the Cuatro Ci\u00e9negas Basin (CCB), Northeast Mexico, with fluctuating hypersaline conditions containing elastic microbial mats that can form small domes where their anoxic inside reminds us of the characteristics of the Archaean Eon, rich in methane and sulfur gases; thus, we named this site the Archaean Domes (AD). These domes only form after heavy rains that are rare in the Chihuahuan desert. CCB is a unique oasis with hundreds of ponds, containing endemic species of animals, plants and highly diverse and unique microbial communities, despite its very biased stoichiometry, due mostly to extreme low phosphorus content (soils, water columns and sediments). This extreme oligotrophy has favored survival of ancestral microorganisms. Whole metagenome sequencing approach was performed for this unusual site in three different seasons to assess the extent of the Archaea biodiversity, with a focus on extremophiles, since members of the Archaea had been underrepresented in different study sites within the oasis. We found a highly diverse Archaea community compassing \u223c5% of the metagenomes. The archaeal portion in all three metagenomes maintained its abundance and most of the strains showed to form a resilient core during three seasonal samplings (2016-2017), despite environmental fluctuations. However, relative abundances of all 230 archaeal OTUs (defined using a 97% cutoff) were low enough (&lt;0.1%) to be considered part of the rare biosphere. AD finding and their description within CCB confirms that this particular pond is the most diverse for Archaea that we are aware of and opens new paths for understanding the forces that once drove and keep shaping microbial community assemblage.","COVID-19 outbreak is still threatening the public health. Therefore, in the middle of the pandemic, all kind of knowledge on SARS-CoV-2 may help us to find the solution. Determining the 3D structures of the proteins involved in host-pathogen interactions are of great importance in the fight against infection. Besides, post-translational modifications of the protein on 3D structure should be revealed in order to understand the protein function since these modifications are responsible for the host-pathogen interaction. Based on these, we predicted O-glycosylation and phosphorylation positions using full amino acid sequence of S1 protein. Candidate positions were further analyzed with enzyme binding activity, solvent accessibility, surface area parameters and the positions determined with high accuracy rate were used to design full 3D glycoprotein structure of the S1 protein using carbohydrate force field. In addition, the interaction between the C-type lectin CD209L and \u03b1-mannose residues was examined and carbohydrate recognition positions were predicted. We suggest these positions as a potential target for the inhibition of the initial binding of SARS-CoV-2 S1 protein to the host cell.","Macroscopic descriptions of populations commonly assume that encounters between individuals are well mixed; i.e., each individual has an equal chance of coming into contact with any other individual. Relaxing this assumption can be challenging though, due to the difficulty of acquiring detailed knowledge about the non-random nature of encounters. Here, we fitted a mathematical model of dengue virus transmission to spatial time series data from Pakistan and compared maximum-likelihood estimates of \u201cmixing parameters\u201d when disaggregating data across an urban-rural gradient. We show that dynamics across this gradient are subject not only to differing transmission intensities but also to differing strengths of nonlinearity due to differences in mixing. We furthermore show that neglecting spatial variation in mixing can lead to substantial underestimates of the level of effort needed to control a pathogen with vaccines or other control efforts. We complement this analysis with relevant contemporary environmental drivers of dengue.","COVID-19 is the most rapidly expanding coronavirus outbreak in the past two decades. To provide a swift response to a novel outbreak, prior knowledge from similar outbreaks is essential. Here, we study the volume of research conducted on previous coronavirus outbreaks, specifically SARS and MERS, relative to other infectious diseases by analyzing over 45 million papers from the last 20 years. Our results demonstrate that previous coronavirus outbreaks have been understudied compared to other viruses. We also show that the research volume of emerging infectious diseases is very high after an outbreak and drops drastically upon the containment of the disease. This can yield inadequate research and limited investment in gaining a full understanding of novel coronavirus management and prevention. Independent of the outcome of the current COVID-19 outbreak, we believe that measures should be taken to encourage sustained research in the field.","The pandemic of Corona Virus Disease 2019 (COVID-19) caused by SARS-CoV-2 has become a global crisis. The replication of SARS-CoV-2 requires the viral RNA-dependent RNA polymerase (RdRp), a direct target of the antiviral drug, Remdesivir. Here we report the structure of the SARS-CoV-2 RdRp either in the apo form or in complex with a 50-base template-primer RNA and Remdesivir at a resolution range of 2.5-2.8 \u00c5. The complex structure reveals that the partial double-stranded RNA template is inserted into the central channel of the RdRp where Remdesivir is incorporated into the first replicated base pair and terminates the chain elongation. Our structures provide critical insights into the working mechanism of viral RNA replication and a rational template for drug design to combat the viral infection.","ABSTRACTCancer gene therapy is a great promising tool for cancer therapeutics due to the specific targeting based on the cancerous gene expression background. Binary systems based on site-specific recombination are one of the most effective potential approaches for cancer gene therapy. In these systems, a cancer specific promoter expresses a site-specific recombinase/integrase that in turn controls the expression of a toxin gene. In the current study, we have developed a new HK022 bacteriophage Integrase (Int) based binary system activating a Diphtheria toxin (DTA) gene expression specifically in cancer cells. We have demonstrated the efficiency, and the high specificity of the system in vitro in cell cultures and in vivo in a lung cancer mouse model. Strikingly, different apoptotic and anti-apoptotic factors demonstrated a remarkable efficacy killing capability of the Int-based binary system compared to the conventional hTERT-DTA mono system in the LLC-Kat lung cancer mice model; we observed that the active hTERT promoter down regulation by the transcription factors Mad-1 is the cornerstone of this phenomenon. The new Int-based binary system offers advantages over already known counterparts and may therefore be developed into a safer and efficient cancer treatment technology.","Viral genetic variants are widely known to influence disease progression among infected humans. Given the recent and rapid emergence of pandemic SARS-CoV-2 infection, the cause of COVID-19 disease, viral protein variants have attracted research interest. However, little has yet been written about genetic risk factors among human hosts. Human genetic variation has proven to affect disease progression and outcome for important diseases such as HIV infection and malaria infestation. The fact that the human ACE2 protein is encoded on the X chromosome means that males who carry rare ACE2 coding variants will express those variants in all ACE2-expressing cells, whereas females will typically express those variants in a mosaic distribution determined by early X-inactivation events. This sex-based difference in ACE2 expression has unique implications for epidemiological studies designed to assess host genetic factors influencing progression from asymptomatic SARS-coV-2 infection to COVID-19. Here we present theoretical modelling of rare ACE2 coding variants documented to occur naturally in several human superpopulations and subpopulations, and show that rare variants predicted to affect the binding of ACE2 to the SARS-CoV-2 spike protein exist in people. Though the rs4646116 (p.Lys26Arg) allele is found in 1 in 70 Ashkenazi Jewish males, and in 1 in 172 non-Finnish European males, this allele is found at higher frequencies in females. Furthermore, the class of missense ACE2 alleles predicted to affect SARS-CoV-2 binding are found in aggregate among 1.43% and 2.16% of Ashkenazi males and females, respectively, as well as in 0.58% and 1.24% of European males and females outside of Finland. These alleles are rarer in other population groups, and almost absent from East Asians genotyped to date. Though we are aware that full genome-wide and exome-wide sequencing studies may ultimately be required to assess human genetic susceptibility to SARS-CoV-2 fully, we argue on the basis of strong prior probabilities that genotyping of this class of alleles is justified in cases of atypical SARS-CoV-2 diseases, such as asymptomatic super-spreaders (if any are identified), and in neonatal/paediatric-onset COVID-19 disease. Even relatively rare susceptibility factors (1% or fewer carriers) may become quantitatively important in the context of hundreds of thousands of infections. A small number of asymptomatic carriers, or a small number of super-spreaders, or a small segment of the population that is disproportionately likely to require intensive care, can magnify the medical, social and economic impacts of a pandemic of this size. The speed of the pandemic and the large number of affected cases worldwide justify efforts to identify all possible risk factors for adverse outcomes, including efforts to identify genetic susceptibility factors in human hosts.","AbstractThe spread of the COVID-19 caused by the SARS-CoV-2 outbreak has been growing since its first identification in December 2019. The publishing of the first SARS-CoV-2 genome made a valuable source of data to study the details about its phylogeny, evolution, and interaction with the host. Protein-protein binding assays have confirmed that Angiotensin-converting enzyme 2 (ACE2) is more likely to be the cell receptor via which the virus invades the host cell. In the present work, we provide an insight into the interaction of the viral spike Receptor Binding Domain (RBD) from different coronavirus isolates with host ACE2 protein. We used homology-based protein-protein docking, binding energy estimation, and decomposition, aiming to predict both qualitative and quantitative aspects of the interaction. Using in silico structural modelling, we brought additional evidence that the interface segment of the spike protein RBD might be acquired by SARS-CoV-2 via a complex evolutionary process rather than mutation accumulation. We also highlighted the relevance of Q493 and P499 amino acid residues of SARS-CoV-2 RBD for binding to hACE2 and maintaining the stability of the interface. Finally, we studied the impact of eight different variants located at the interaction surface of ACE2, on the complex formation with SARS-CoV-2 RBD. We found that none of them is likely to disrupt the interaction with the viral RBD of SARS-CoV-2.","1AbstractThe ongoing outbreak of the novel coronavirus disease 2019 (COVID-19) originating from Wuhan, China, draws worldwide concerns due to its long incubation period and strong infectivity. Although RT-PCR-based molecular diagnosis techniques are being widely applied for clinical diagnosis currently, timely and accurate diagnosis are still limited due to labour intensive and time-consuming operations of these techniques. To address the issue, herein we report the synthesis of poly (amino ester) with carboxyl groups (PC)-coated magnetic nanoparticles (pcMNPs), and the development of pcMNPs-based viral RNA extraction method for the sensitive detection of COVID-19 causing virus, the SARS-CoV-2. This method combines the lysis and binding steps into one step, and the pcMNPs-RNA complexes can be directly introduced into subsequent RT-PCR reactions. The simplified process can purify viral RNA from multiple samples within 20 min using a simple manual method or an automated high-throughput approach. By identifying two different regions (ORFlab and N gene) of viral RNA, a 10-copy sensitivity and a strong linear correlation between 10 and 105 copies of SARS-CoV-2 pseudovirus particles are achieved. Benefitting from the simplicity and excellent performances, this new extraction method can dramatically reduce the turn-around time and operational requirements in current molecular diagnosis of COVID-19, in particular for the early clinical diagnosis.","The ongoing pandemic of the coronavirus disease 2019 (COVID-19) is an infectious disease caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV2). We have performed an integrated sequence-based analysis of SARS-CoV2 genomes from different geographical locations in order to identify its unique features absent in SARS-CoV and other related coronavirus family genomes, conferring unique infection, facilitation of transmission, virulence and immunogenic features to the virus. The phylogeny of the genomes yields some interesting results. Systematic gene level mutational analysis of the genomes has enabled us to identify several unique features of the SARS-CoV2 genome, which includes a unique mutation in the spike surface glycoprotein (A930V (24351C&gt;T)) in the Indian SARS-CoV2, absent in other strains studied here. We have also predicted the impact of the mutations in the spike glycoprotein function and stability, using computational approach. To gain further insights into host responses to viral infection, we predict that antiviral host-miRNAs may be controlling the viral pathogenesis. Our analysis reveals nine host miRNAs which can potentially target SARS-CoV2 genes. Interestingly, the nine miRNAs do not have targets in SARS and MERS genomes. Also, hsa-miR-27b is the only unique miRNA which has a target gene in the Indian SARS-CoV2 genome. We also predicted immune epitopes in the genomes.","AbstractBackgroundThe current outbreak caused by novel coronavirus (2019-nCoV) in China has become a worldwide concern. As of 28 January 2020, there were 4631 confirmed cases and 106 deaths, and 11 countries or regions were affected.MethodsWe downloaded the genomes of 2019-nCoVs and similar isolates from the Global Initiative on Sharing Avian Influenza Database (GISAID and nucleotide database of the National Center for Biotechnology Information (NCBI). Lasergene 7.0 and MEGA 6.0 softwares were used to calculate genetic distances of the sequences, to construct phylogenetic trees, and to align amino acid sequences. Bayesian coalescent phylogenetic analysis, implemented in the BEAST software package, was used to calculate the molecular clock related characteristics such as the nucleotide substitution rate and the most recent common ancestor (tMRCA) of 2019-nCoVs.ResultsAn isolate numbered EPI_ISL_403928 showed different phylogenetic trees and genetic distances of the whole length genome, the coding sequences (CDS) of ployprotein (P), spike protein (S), and nucleoprotein (N) from other 2019-nCoVs. There are 22, 4, 2 variations in P, S, and N at the level of amino acid residues. The nucleotide substitution rates from high to low are 1\u00b705 \u00d7 10\u22122 (nucleotide substitutions/site/year, with 95% HPD interval being 6.27 \u00d7 10\u22124 to 2.72 \u00d7 10\u22122) for N, 5.34 \u00d7 10\u22123 (5.10 \u00d7 10\u22124, 1.28 \u00d7 10\u22122) for S, 1.69 \u00d7 10\u22123 (3.94 \u00d7 10\u22124, 3.60 \u00d7 10\u22123) for P, 1.65 \u00d7 10\u22123 (4.47 \u00d7 10\u22124, 3.24 \u00d7 10\u22123) for the whole genome, respectively. At this nucleotide substitution rate, the most recent common ancestor (tMRCA) of 2019-nCoVs appeared about 0.253-0.594 year before the epidemic.ConclusionOur analysis suggests that at least two different viral strains of 2019-nCoV are involved in this outbreak that might occur a few months earlier before it was officially reported.","AbstractUrodynamic studies, used to understand bladder function, diagnose bladder disease, and develop treatments for dysfunctions, are ideally performed with awake subjects. However, in animal models, especially cats (a common model of spinal cord injury and associated bladder pathology), anesthesia is often required for these procedures and can be a research confounder. This study compared the effects of select agents (dexmedetomidine, alfaxalone, propofol, isoflurane, and \u03b1-chloralose) on urodynamic (\u0394pressure, bladder capacity, bladder compliance, non-voiding contractions, bladder pressure slopes) and anesthetic (change in heart rate [\u0394HR], average heart rate [HR], reflexes, induction/recovery times) parameters in repeated cystometrograms across five adult male cats. \u0394pressure was greatest with propofol, bladder capacity was highest with \u03b1-chloralose, non-voiding contractions were greatest with \u03b1-chloralose. Propofol and dexmedetomidine had the highest bladder pressure slopes during the initial and final portions of the cystometrograms respectively. Cats progressed to a deeper plane of anesthesia (lower HR, smaller \u0394HR, decreased reflexes) under dexmedetomidine, compared to propofol and alfaxalone. Time to induction was shortest with propofol, and time to recovery was shortest with dexmedetomidine. These agent-specific differences in urodynamic and anesthetic parameters in cats will facilitate appropriate study-specific anesthetic choices.","AbstractThe current practice for diagnosis of SARS-CoV-2 infection relies on PCR testing of nasopharyngeal or respiratory specimens in a symptomatic patient at high epidemiologic risk. This testing strategy likely underestimates the true prevalence of infection, creating the need for serologic methods to detect infections missed by the limited testing to date. Here, we describe the development of a coronavirus antigen microarray containing immunologically significant antigens from SARS-CoV-2, in addition to SARS-CoV, MERS-CoV, common human coronavirus strains, and other common respiratory viruses. A preliminary study of human sera collected prior to the SARS-CoV-2 pandemic demonstrates overall high IgG reactivity to common human coronaviruses and low IgG reactivity to epidemic coronaviruses including SARS-CoV-2, with some cross-reactivity of conserved antigenic domains including S2 domain of spike protein and nucleocapsid protein. This array can be used to answer outstanding questions regarding SARS-CoV-2 infection, including whether baseline serology for other coronaviruses impacts disease course, how the antibody response to infection develops over time, and what antigens would be optimal for vaccine development.","AbstractCoronaviruses encode a mix of highly conserved and novel genes as well as genetic elements necessary for infection and pathogenesis, raising the possibility for common targets for attenuation and therapeutic design. In this study, we focus on the highly conserved nonstructural protein (NSP) 16, a viral 2\u2019O methyl-transferase (MTase) that encodes critical functions in immune modulation and infection. Using reverse genetics, we disrupted a key motif in the conserved KDKE motif of MERS NSP16 (D130A) and evaluated the effect on viral infection and pathogenesis. While the absence of 2\u2019O MTase activity had only marginal impact on propagation and replication in Vero cells, the MERS dNSP16 mutant demonstrated significant attenuation relative to control both in primary human airway cultures and in vivo. Further examination indicated the MERS dNSP16 mutant had a type I IFN based attenuation and was partially restored in the absence of IFIT molecules. Importantly, the robust attenuation permitted use of MERS dNSP16 as a live attenuated vaccine platform protecting from challenge with a mouse adapted MERS-CoV strain. These studies demonstrate the importance of the conserved 2\u2019O MTase activity for CoV pathogenesis and highlight NSP16 as a conserved universal target for rapid live attenuated vaccine design in an expanding CoV outbreak setting.SignificanceCoronavirus emergence in both human and livestock represents a significant threat to global public health, as evidenced by the sudden emergence of SARS-CoV, MERS-CoV, PEDV and swine delta coronavirus in the 21st century. These studies describe an approach that effectively targets the highly conserved 2\u2019O methyl-transferase activity of coronaviruses for attenuation. With clear understanding of the IFN/IFIT based mechanism, NSP16 mutants provide a suitable target for a live attenuated vaccine platform as well as therapeutic development for both current and future emergent CoV strains. Importantly, other approaches targeting other conserved pan-coronavirus functions have not yet proven effective against MERS-CoV, illustrating the broad applicability of targeting viral 2\u2019O MTase function across coronaviruses.","AbstractThe current practice for diagnosis of COVID-19, based on SARS-CoV-2 PCR testing of pharyngeal or respiratory specimens in a symptomatic patient at high epidemiologic risk, likely underestimates the true prevalence of infection. Serologic methods can more accurately estimate the disease burden by detecting infections missed by the limited testing performed to date. Here, we describe the validation of a coronavirus antigen microarray containing immunologically significant antigens from SARS-CoV-2, in addition to SARS-CoV, MERS-CoV, common human coronavirus strains, and other common respiratory viruses. A comparison of antibody profiles detected on the array from control sera collected prior to the SARS-CoV-2 pandemic versus convalescent blood specimens from virologically confirmed COVID-19 cases demonstrates complete discrimination of these two groups. This array can be used as a diagnostic tool, as an epidemiologic tool to more accurately estimate the disease burden of COVID-19, and as a research tool to correlate antibody responses with clinical outcomes.","ABSTRACTThe outbreak of COVID-19, which is caused by SARS-CoV-2 virus, continues to spread globally, but there is currently very little understanding of the epitopes on the virus. In this study, we have determined the crystal structure of the receptor-binding domain (RBD) of the SARS-CoV-2 spike (S) protein in complex with CR3022, a neutralizing antibody previously isolated from a convalescent SARS patient. CR3022 targets a highly conserved epitope that enables cross-reactive binding between SARS-CoV-2 and SARS-CoV. Structural modeling further demonstrates that the binding site can only be accessed when at least two RBDs on the trimeric S protein are in the \u201cup\u201d conformation. Overall, this study provides structural and molecular insight into the antigenicity of SARS-CoV-2.ONE SENTENCE SUMMARYStructural study of a cross-reactive SARS antibody reveals a conserved epitope on the SARS-CoV-2 receptor-binding domain.","AbstractOne of the key characteristics of the transmission dynamics of infectious diseases is the generation time which refers to the time interval between the infection of a secondary case and the infection of its infector. The generation time distribution together with the reproduction number determines the rate at which an infection spreads in a population. When defining the generation time distribution at a calendar time t two definitions are plausible according whether we regard t as the infection time of the infector or the infection time of the infectee. The resulting measurements are respectively called forward generation time and backward generation time. It has been observed that the mean forward generation time contracts around the peak of an epidemic. This contraction effect has previously been attributed to either competition among potential infectors or depletion of susceptibles in the population. The first explanation requires many infectives for contraction to occur whereas the latter explanation suggests that contraction occurs even when there are few infectives. With a simulation study we show that both competition and depletion cause the mean forward generation time to contract. Our results also reveal that the distribution of the infectious period and the reproduction number have a strong effect on the size and timing of the contraction, as well as on the mean value of the generation time in both forward and backward scheme.Author summaryInfectious diseases remain one of the greatest threats to human health and commerce, and the analysis of epidemic data is one of the most important applications of statistics in public health. Thus, having reliable estimates of fundamental infectious diseases parameters is critical for public health decision-makers in order to take appropriate actions for the global prevention and management of outbreaks and other health emergencies. A key example is given by the prediction models of the reproduction numbers: these rely on the generation time distribution that is usually estimated from contact tracing data collected at a precise calendar time. The forward scheme is used in such a prediction model and the knowledge of its evolution over time is crucial to correctly estimate the parameters of interest. It is therefore important to characterize the causes that lead to the contraction of the mean forward generation time during the course of an outbreak.In this paper, we firstly identify the impact of the epidemiological quantities as reproduction number, infectious period and population size on the mean forward and backward generation time. Moreover, we analyze the phenomena of competition among infectives and depletion of susceptible individuals highlighting their effects on the contraction of the mean forward generation time. The upshot of this investigation is that the variance of the infectious period distribution and the reproduction number have a strong impact on the generation times affecting both the mean value and the evolution over time. Furthermore, competition and depletion can both cause contraction even for small values of the reproduction number suggesting that, in epidemic models where the generation time is considered time-inhomogeneous, estimators accounting for both depletion and competing risks are to be preferred in the inference of the generation interval distributions.","Cryo-EM structures of transcription pre-initiation complex (PIC) and initiation complex (IC) of yeast mitochondrial RNA polymerase show fully resolved transcription bubbles and explain promoter melting, template alignment, DNA scrunching, transition into elongation, and abortive synthesis. Promoter melting initiates in PIC with MTF1 trapping the -4 to -2 non-template (NT) bases in its NT-groove. Transition to IC is marked by a large-scale movement that aligns the template with RNA at the active site. RNA synthesis scrunches the NT strand into an NT-loop, which interacts with centrally positioned MTF1 C-tail. Steric clashes of the C-tail with RNA:DNA and NT-loop, and dynamic scrunching-unscrunching of DNA explain abortive synthesis and transition into elongation. Capturing the catalytically active IC-state with UTP\u03b1S poised for incorporation enables modeling toxicity of antiviral nucleosides/nucleotides.","AbstractBackgroundViral encephalitis is a dangerous compromise between the need to robustly clear pathogen from the brain and the need to protect neurons from bystander injury. Theiler's murine encephalomyelitis virus (TMEV) infection of C57Bl/6 mice is a model of viral encephalitis in which the compromise results in hippocampal damage and permanent neurological sequelae. We previously identified brain infiltrating inflammatory monocytes as the primary driver of this hippocampal pathology, but the mechanisms involved in recruiting these cells to the brain were unclear.MethodsChemokine expression levels in the hippocampus were assessed by microarray, ELISA, RT-PCR, and immunofluorescence. Monocyte infiltration during acute TMEV infection was measured by flow cytometry. CCL2 levels were manipulated by immunodepletion and by specific removal from neurons in mice generated by crossing a line expressing the Cre recombinase behind the synapsin promoter to animals with floxed CCL2.ResultsInoculation of the brain with TMEV induced hippocampal production of the proinflammatory chemokine CCL2 that peaked at 6 hours postinfection, whereas inoculation with UV-inactivated TMEV did not elicit this response. Immunofluorescence revealed that hippocampal neurons expressed high levels of CCL2 at this timepoint. Genetic deletion of CCR2 and systemic immunodepletion of CCL2 abrogated or blunted the infiltration of inflammatory monocytes into the brain during acute infection. Specific genetic deletion of CCL2 from neurons reduced serum and hippocampal CCL2 levels and inhibited inflammatory monocyte infiltration into the brain.ConclusionsWe conclude that intracranial inoculation with infectious TMEV rapidly induces the expression of CCL2 in neurons, and this cellular source is necessary for CCR2-dependent infiltration of inflammatory monocytes into the brain during the most acute stage of encephalitis. These findings highlight a unique role for neuronal production of chemokines in the initiation of leukocytic infiltration into the infected central nervous system.","AbstractStrongyloidiasis is a prevailing helminth infection ubiquitous in tropical and subtropical areas. However, prevalence data are scarce in migrant populations.This study aims at evaluating the prevalence of S. stercoralis at hospital level in migrant populations or long term travellers being attended in out-patient and in-patient units as part of a systematic screening implemented in 6 Spanish hospitals. A cross-sectional study was conducted and systematic screening for S. stercoralis infection using serological tests was offered to all eligible participants. The overall seroprevalence of S. stercoralis was 9.04% (95% confidence interval [95%CI] 7.76 \u221210.31). The seroprevalence of people with a risk of infection acquired in Africa and Latin America was 9.35% (95%CI 7.01-11.69), 9.22% (7.5-10.93), respectively. The number of individuals coming from Asian countries was significantly smaller and the overall prevalence in these countries was 2.9% (95%CI \u22120.3; \u22126.2). There was only one case (1/14 (7.14%) from an individual from East European countries. The seroprevalence in units attending potentially immunosuppressed patients was significantly lower (5.64%) compared with the seroprevalence in other units of the hospital (10.20%) or Tropical diseases units (13.33%) (p&lt;0.001). Conclusions: We report a hospital-based systematic screening of strongyloidiasis with a seroprevalence of almost 10% in a mobile population coming from endemic areas suggesting the need of implementing strongyloidiasis screening in hospitalized patients coming from endemic areas, particularly if they are at risk of immunosuppression.Author summaryStrongyloidiasis is an infection caused by the helminth Strongyloides stercoralis which is ubiquitous in tropical and subtropical areas. In the rest of the countries, it is also frequent in migrants coming from tropical and subtropical areas. The disease is more severe when an infected subject has an impaired immune system. Within this study we have evaluated the prevalence of this infection in people being attended in six Spanish hospitals. The prevalence was around 9%, being higher in Africa and Latin America compared with other regions. In addition, the prevalence in patients with an impaired immune system (immunosuppression) was lower compared with people non suffering immunosuppression. These results suggest that the prevalence of strongyloidiasis is quite high among migrants living in Spain and that a screening programme should be designed, particularly in immunosuppressed patients that are at more risk of suffering severe complications of the infection.","SUMMARYAll viruses balance interactions between cellular machinery co-opted to support replication and host factors deployed to halt the infection. We used gene correlation analysis to perform an unbiased screen for host factors involved in influenza A virus (FLUAV) infection. Our screen identified the cellular factor epidermal growth factor receptor pathway substrate 8 (EPS8) as the highest confidence pro-viral candidate. Knockout and overexpression of EPS8 confirmed its importance in enhancing FLUAV infection and titers. Loss of EPS8 did not affect virion attachment, uptake, or fusion. Rather, our data show that EPS8 specifically functions during virion uncoating. EPS8 physically associated with incoming virion components, and subsequent nuclear import of released ribonucleoprotein complexes was significantly delayed in the absence of EPS8. Our study identified EPS8 as a host factor important for uncoating, a crucial step of FLUAV infection during which the interface between the virus and host is still being discovered.","AbstractDrug susceptibility, defined by the minimal inhibitory concentration (MIC), often does not predict whether fungal infections will respond to therapy in the clinic. Tolerance at supra-MIC antifungal drug concentrations is rarely quantified and current clinical recommendations suggest it be ignored. Here, we measured and characterized drug-response variables that could influence the outcomes of fungal infections and be generalizable across major clades of Candida albicans, one of the most frequently isolated human fungal pathogens. We quantified antifungal tolerance as the fraction of growth (FoG) above the MIC and found that it is clearly distinct from susceptibility/resistance measured as MIC. Instead, tolerance is due to the slow growth of subpopulations of cells that overcome drug stress more efficiently than the rest of the population, and correlates inversely with the accumulation of intracellular drug. Importantly, many adjuvant drugs used together with fluconazole, a fungistatic drug, reduce tolerance without affecting resistance. These include inhibitors of major stress response hubs such as Hsp90, calcineurin, PKC1 and TOR. Accordingly, in an invertebrate infection model, adjuvant combination therapy was significantly more effective than fluconazole alone in treating highly tolerant isolates and did not improve the treatment of isolates with low tolerance levels. Furthermore, isolates recovered from immunocompetent patients with persistent candidemia displayed significantly higher tolerance than isolates that were readily cleared by fluconazole. Thus, tolerance correlates with the response to fluconazole therapy in patients and may help predict whether infections will respond to fluconazole alone. Similarly, measuring tolerance may provide a useful clinical parameter for choosing appropriate therapeutic strategies to overcome persistent clinical candidemia.","AbstractThe ability to comprehensively characterize exposures and immune responses to viral infections will be critical to better understanding human health and disease. We previously described the VirScan system, a phage-display based technology for profiling antibody binding to a comprehensive library of peptides designed to represent the human virome. The previous VirScan analytical approach did not fully account for disproportionate representation of viruses in the library or for antibody cross-reactivity among sequences shared by related viruses. Here we present the \u2018AntiViral Antibody Response Deconvolution Algorithm\u2019 (\u2018AVARDA\u2019), a multi-module software package for analyzing VirScan datasets. AVARDA provides a probabilistic assessment of infection at species-level resolution by considering alignment of all library peptides to each other and to all human viruses. We employed AVARDA to analyze VirScan data from a cohort of encephalitis patients with either known viral infections or undiagnosed etiologies. By comparing acute and convalescent sera, AVARDA successfully confirmed or detected antibody responses to human herpesviruses 1, 3, 4, 5, and 6, thereby improving the rate of diagnosing viral encephalitis in this cohort by 62.5%. We further assessed AVARDA\u2019s utility in the setting of an epidemiological study, demonstrating its ability to determine infections acquired in a child followed prospectively from infancy. We consider ways in which AVARDA\u2019s conceptual framework may be further developed in the future and describe how its analyses may be extended beyond investigations of viral infection. AVARDA, in combination with VirScan and other pan-pathogen serological techniques, is likely to find broad utility in the epidemiology and diagnosis of infectious diseases.","AbstractThe rapid spread of COVID-19 motivates development of antivirals targeting conserved molecular machinery of the SARS-CoV-2 virus. The SARS-CoV-2 genome includes conserved RNA elements that offer potential targets for RNA-targeting small-molecule drugs, but 3D structures of most of these elements have not been experimentally characterized. Here, we provide a dataset called \u2018FARFAR2-SARS-CoV-2\u2019, a collection of 3D coordinates modeled using Rosetta\u2019s FARFAR2 algorithm, including de novo models for thirteen RNA elements in SARS-CoV-2 and homology models for a fourteenth. These elements comprise SL1, SL2, SL3, SL4, SL5, putative SL6 and SL7 in the extended 5\u2032 UTR, as well as the entire extended 5\u2032 UTR; the frameshifting element (FSE) from the SARS-CoV-2 ORF1a/b gene and a putative dimer of FSE; and the extended pseudoknot, hypervariable region, and the s2m of the 3\u2032 UTR, as well as the entire 3\u2032 UTR. For five of these elements (SL1, SL2, SL3, FSE, s2m), convergence of lowest predicted energy structures supports their accuracy in capturing low energy states that might be targeted for small molecule binding. To aid efforts to discover small molecule RNA binders guided by computational models, we provide a second benchmarking dataset called \u2018FARFAR2-Apo-Riboswitch\u2019, which consists of similarly prepared Rosetta-FARFAR2 models for RNA riboswitch aptamer regions that bind small molecules. Both datasets include up to 400 3D models for each RNA element, which may facilitate drug discovery approaches targeting dynamic ensembles of low-energy excited states of RNA molecules.","AbstractThe genus Torovirus (subfamily Torovirinae, family Coronaviridae, order Nidovirales) encompasses a range of species that infect domestic ungulates including cattle, sheep, goats, pigs and horses, causing an acute self-limiting gastroenteritis. Using the prototype species equine torovirus (EToV) we performed parallel RNA sequencing (RNA-seq) and ribosome profiling (Ribo-seq) to analyse the relative expression levels of the known torovirus proteins and transcripts, chimaeric sequences produced via discontinuous RNA synthesis (a characteristic of the nidovirus replication cycle) and changes in host transcription and translation as a result of EToV infection. RNA sequencing confirmed that EToV utilises a unique combination of discontinuous and non-discontinuous RNA synthesis to produce its subgenomic RNAs; indeed, we identified transcripts arising from both mechanisms that would result in sgRNAs encoding the nucleocapsid. Our ribosome profiling analysis revealed that ribosomes efficiently translate two novel CUG-initiated ORFs, located within the so-called 5\u2019 UTR. We have termed the resulting proteins U1 and U2. Comparative genomic analysis confirmed that these ORFs are conserved across all available torovirus sequences and the inferred amino acid sequences are subject to purifying selection, indicating that U1 and U2 are functionally relevant. This study provides the first high-resolution analysis of transcription and translation in this neglected group of livestock pathogens.ImportanceToroviruses infect cattle, goats, pigs and horses worldwide and can cause gastrointestinal disease. There is no treatment or vaccine and their ability to spill over into humans has not been assessed. These viruses are related to important human pathogens including severe acute respiratory syndrome (SARS) coronavirus and they share some common features, however the mechanism that they use to produce subgenomic RNA molecules differs. Here we performed deep sequencing to determine how equine torovirus produces subgenomic RNAs. In doing so, we also identified two previously unknown open reading frames \u201chidden\u201d within the genome. Together these results highlight the similarities and differences between this domestic animal virus and related pathogens of humans and livestock.","SummaryHeat shock triggers an instant reprogramming of gene and enhancer transcription, but whether cells encode a memory to stress, at the level of nascent transcription, has remained unknown. Here, we measured transcriptional response to acute heat stress in unconditioned cells and in daughters of cells that had been exposed to a single or multiple heat shocks. Tracking RNA Polymerase II (Pol II) genome-wide at nucleotide-resolution revealed that cells precisely remember their transcriptional identity throughout stress, restoring Pol II distribution at gene bodies and enhancers upon recovery. However, single heat shock primed faster gene-induction in the daughter cells by increasing promoter-proximal Pol II pausing, and accelerating the pause-release. In repeatedly stressed cells, both basal and inducible transcription was refined, and pre-mRNA processing decelerated, which retained transcripts on chromatin and reduced recycling of the transcription machinery. These results mechanistically uncovered how the steps of pause-release and termination maintain transcriptional memory over mitosis.Highlights-Cell type-specific transcription precisely recovers after heat-induced reprogramming-Single heat shock primes genes for accelerated induction over mitotic divisionsviaincreased promoter-proximal Pol II pausing and faster pause-release-Multiple heat shocks refine basal and inducible transcription over mitotic divisions to support survival of the daughter cells-Decelerated termination at active genes reduces recycling of Pol II to heat-activated promoters and enhancers-HSF1 increases the rate of promoter-proximal pause-releaseviadistal and proximal regulatory elements","AbstractLung disease causes significant morbidity and mortality, and is exacerbated by environmental injury, e.g. through lipopolysaccharide (LPS) or ozone (O3). Toll-like receptors (TLRs) orchestrate immune responses to injury by recognizing pathogen- or danger-associated molecular patterns. TLR4, the prototypic receptor for LPS, also mediates inflammation after O3, triggered by endogenous hyaluronan. Regulation of TLR4 signaling is incompletely understood. TLR5, the flagellin receptor, is expressed in alveolar macrophages, and regulates immune responses to environmental injury. Using in vivo animal models of TLR4-mediated inflammations (LPS, O3, hyaluronan), we show that TLR5 impacts the in vivo response to LPS, hyaluronan and O3. We demonstrate that immune cells of human carriers of a dominant negative TLR5 allele have decreased inflammatory response to O3 exposure ex vivo and LPS exposure in vitro. Using primary murine macrophages, we find that TLR5 physically associates with TLR4 and biases TLR4 signaling towards the MyD88 pathway. Our results suggest an updated paradigm for TLR4/TLR5 signaling.","AbstractRhinoviruses (RVs) occur more frequently than other viruses and more often in people displaying symptoms than in those without. RVs exacerbate chronic airway disease and confound the clinical diagnosis of influenza-like illness. We sought to estimate the spectrum of RV diversity, RV species seasonality and to breakdown RV involvement in respiratory virus co-detections by comprehensive molecular testing of a convenience collection of airway sample extracts from patients with suspected respiratory infections, collected during 2001.RVs were the most common virus detected. We were able to genotype \u223c90% of RV detections, identifying 70 distinct RVs, spanning all three species. RV-Bs were under-represented. We found RV species co-circulated at times, although one species usually dominated. Each species displayed a bimodal distribution.Notably, RVs and influenza A viruses (IFAV) seldom co-occurred, supporting their roles as primary pathogens of the airway among acutely ill infants. Whether RV circulation has a moderating or controlling effect on the IFAV season or is controlled by it cannot be determined from these data.Despite the frequent perception that RVs commonly co-occur with another virus, our findings indicated this was not the case. Nearly 80% of RV detections occurred alone. Understanding more about population-level interference between viruses may allow us to harness aspects of it to generate a non-specific antiviral intervention that mimics a putative protective effect.For routine respiratory virus screening to best serve the patient, RV testing should be a principal component of any acute respiratory illness testing algorithm throughout the year.","AbstractPhylogenomic analysis of SARS-Cov-2 as available from publicly available repositories suggests the presence of 4 prevalent groups of viral episomes (super-clades), which are mostly associated with outbreaks in distinct geographic locations (China, USA and Europe). While levels of genomic variability between SARS-Cov-2 isolates are limited, to our knowledge, it is not clear whether the observed patterns of variability in viral super-clades reflect ongoing adaptation of SARS-Cov-2, or merely genetic drift and founder effects. Here, we analyze more than 1100 complete, high quality SARS-Cov-2 genome sequences, and provide evidence for the absence of distinct evolutionary patterns/signatures in the genomes of the currently known major clades of SARS-Cov-2. Our analyses suggest that the presence of distinct viral episomes at different geographic locations are consistent with founder effects, coupled with the rapid spread of this novel virus. We observe that while cross species adaptation of the virus is associated with hypervariability of specific genomic regions (including the RDB domain of the spike protein), the more variable genomic regions between extant SARS-Cov-2 episomes correspond with the 3\u2019 and 5\u2019 UTRs, suggesting that at present viral protein coding genes should not be subjected to different adaptive evolutionary pressures in different viral strains. All in all, while - also in the light of the limited variability of SARS-Cov-2 - this study can not be conclusive, we believe that the evidence presented here is strongly consistent with the notion that the biased geographic distribution of SARS-Cov-2 isolates should not be associated with adaptive evolution of this novel pathogen.","SummaryAsthma exacerbations are triggered by rhinovirus infections. We employed a systems biology approach to delineate upper airway gene network patterns underlying asthma exacerbation phenotypes in children. Cluster analysis unveiled distinct IRF7hi versus IRF7lo molecular phenotypes, the former exhibiting robust upregulation of Th1/type I interferon responses and the latter an alternative signature marked by upregulation of cytokine and growth factor signalling and downregulation of interferon gamma. The two phenotypes also produced distinct clinical phenotypes. For IRF7lo versus IRF7hi: symptom duration prior to hospital presentation was more than twice as long from initial symptoms (p=0.011) and nearly three times as long for cough (p&lt;0.001); the odds ratio of admission to hospital was increased more than four-fold (p=0.018); and time to recurrence was shorter (p=0.015). In summary, our findings demonstrate that asthma exacerbations in children can be divided into IRF7hi versus IRF7lo phenotypes with associated differences in clinical phenotypes.AbbreviationsAHR, airway hyperresponsiveness; ARG1, Arginase 1, CSF3, Colony Stimulating Factor 3; CD38, Cluster of Differentiation 38; CD163, Cluster of Differentiation 163; cDCs, conventional (or myeloid) dendritic cells; DDX60, DExD/H-Box Helicase 60; ED, Emergency Department; EGF, Epidermal Growth Factor; ERK, Extracellular signal-Regulated Kinase; FCER1G, Fc Fragment Of IgE Receptor Ig; HMBS, Hydroxymethylbilane Synthase; IFNg, Interferon Gamma; IFNL1, Interferon Lambda 1; IL-1R2, Interleukin 1 Receptor Type 2; IRF7, Interferon Regulatory Factor 7; ISG15, Interferon-stimulated gene 15; MDA5, Melanoma Differentiation-Associated protein 5; MX1, Myxovirus Resistance Protein 1; NAD, nicotinamide adenine dinucleotide; NCR1, Natural cytotoxicity triggering receptor 1; OSM, Oncostatin M; PD-L1, Programmed Death-Ligand 1; PPIA, Peptidylprolyl Isomerase A; PPIB Peptidylprolyl Isomerase B; RSAD2, Radical S-adenosyl methionine domain-containing protein 2; RSV, respiratory syncytial virus; RT-qPCR, quantitative reverse transcription PCR; RV, rhinovirus; sPLA2, secretory Phospholipase A2; TGFb, Transforming Growth Factor beta; THBS1, Thrombospondin 1; TNF, Tumor Necrosis Factor; TLR2, Toll-like Receptor 2.","AbstractCommercial multiplex assays, built on different chemistries and platforms are widely available for simultaneous detection of pathogens that cause respiratory infections. However, these tests are often difficult to implement in a resource limited setting because of high cost. In this study, we developed and validated a method for simultaneous testing of common respiratory pathogens (Respanel) by real-time PCR in a convenient, strip-tube array format. Primers and probes for sixteen PCR assays were selected from the literature or newly designed. Following optimization of individual PCR assays, strip-tube arrays were prepared by dispensing primer-probe mixes (PPM) into two sets of 8-tube strips. Nucleic acid extracts from specimens were mixed with PCR master mix, and dispensed column-wise into 2X8-wells of a 96-well plate. PPMs from strip-tubes were then added to the wells using a multichannel pipette for real-time PCR. Individual PCR assays were optimized using previously known specimens (n=397) with 91%-100% concordance with culture, DFA or PCR results. Respanel was then tested in a routine manner at two different sites using specimens (n=147) previously tested by Qiagen Resplex I&amp;II or Fast-Track Diagnostics Respiratory Pathogens 21 assays. The sensitivity, specificity and accuracy of Respanel were 94%, 95% and 95%, respectively, against Resplex and 88%, 100% and 99%, respectively, against FTDRP21. Respanel detected 48% more pathogens (p&lt;0.05) than Resplex but the rate of pathogen detection was not significantly different from FTDRP21. Respanel is a convenient and inexpensive assay that is more sensitive than Resplex and comparable to FTDRP21 for the detection of common respiratory pathogens.","AbstractBackgroundThe adenosine A2A receptor (A2AR) heteromerizes with the dopamine D2 receptor (D2R). In order to explore their functional interaction, we engineered previously stable single-polypeptide-chain (sc) A2AR/D2LR: whether the molecular entity of the striatal A2AR/D2R antagonism, i.e., scA2AR/D2Rs are just A2AR/D2R with the antagonism, remains unresolved.New MethodTo further clarify the heteromerization through the scA2AR/D2LR, we here designed supramolecularly \u2018exclusive\u2019 monomers and dimers, using the C\u03b52 domain of IgE-Fc or apoproteins of the bacterial light-harvesting antenna complex.ResultsA concept of the recptor protein assembly regulation, i.e., the selective monomer/non-obligate dimer formation was obtained. Although none of these new fusions were constructed or tested, we could aim at obtaining heterodimer-specific agents, using the scA2AR/D2R. Whether the resulting designs were explained feasibly and rationally was addressed. The structure and function of the non-obligate dimer were here discussed through scA2AR/D2R, focusing on the procedure of the membrane protein design and methods for transient protein-protein interactions.Summary and OutlookGiven that upon being expressed and allosteric regulation occurs regardless of specific signal to non-specific noise (S/N) ratio, the supramolecular designs, allowing us to express selectively monomer/non-obligate dimer of class A GPCR, are experimentally testable and will be used to confirm in vivo that such low S/N ratio interaction between A2AR and D2LR functions in the dopamine neurotransmission in the striatum.Abbreviations:A2ARadenosine A2A receptor3HA-A2ARA2AR tagged with a triple HA epitopeBchlbacteriochlorophyllBRETbioluminescence resonance energy transferCcarboxy-terminalCDcluster of differentiationD2LR and D2SRthe long and short form of dopamine D2 receptor, respectivelyFabantigen binding fragmentFcFc fragmentFc\u03b5RIhigh affinity receptor for IgEFRETfluorescence resonance energy transferG4San amino acid sequence consisting of a four-glycine-repeat followed by a serine residueGABA\u03b3-aminobutyric acidGABABGABA type B receptorGPCRG protein-coupled receptorGttransducinHAhemagglutininHIVhuman immunodeficiency virusICintracellular loopsIgimmunoglobulinLHlight-harvesting antenna complexmAbmonoclonal antibodyMrmolecular weightNamino-terminalPDParkinson\u2019s diseasePSphotosystemRCreaction centerRlucRenilla luciferasescsingle-chainTMtransmembrane3Dthree-dimensional","AbstractViruses with RNA genomes dominate the eukaryotic virome, reaching enormous diversity in animals and plants. The recent advances of metaviromics prompted us to perform a detailed phylogenomic reconstruction of the evolution of the dramatically expanded global RNA virome. The only universal gene among RNA viruses is the RNA-dependent RNA polymerase (RdRp). We developed an iterative computational procedure that alternates the RdRp phylogenetic tree construction with refinement of the underlying multiple sequence alignments. The resulting tree encompasses 4,617 RNA virus RdRps and consists of 5 major branches, 2 of which include positive-sense RNA viruses, 1 is a mix of positive-sense (+) RNA and double-stranded (ds) RNA viruses, and 2 consist of dsRNA and negative-sense (\u2212) RNA viruses, respectively. This tree topology implies that dsRNA viruses evolved from +RNA viruses on at least two independent occasions, whereas -RNA viruses evolved from dsRNA viruses. Reconstruction of RNA virus evolution using the RdRp tree as the scaffold suggests that the last common ancestors of the major branches of +RNA viruses encoded only the RdRp and a single jelly-roll capsid protein. Subsequent evolution involved independent capture of additional genes, particularly, those encoding distinct RNA helicases, enabling replication of larger RNA genomes and facilitating virus genome expression and virus-host interactions. Phylogenomic analysis reveals extensive gene module exchange among diverse viruses and horizontal virus transfer between distantly related hosts. Although the network of evolutionary relationships within the RNA virome is bound to further expand, the present results call for a thorough reevaluation of the RNA virus taxonomy.IMPORTANCEThe majority of the diverse viruses infecting eukaryotes have RNA genomes, including numerous human, animal, and plant pathogens. Recent advances of metagenomics have led to the discovery of many new groups of RNA viruses in a wide range of hosts. These findings enable a far more complete reconstruction of the evolution of RNA viruses than what was attainable previously. This reconstruction reveals the relationships between different Baltimore Classes of viruses and indicates extensive transfer of viruses between distantly related hosts, such as plants and animals. These results call for a major revision of the existing taxonomy of RNA viruses.","AbstractSynthetic biology based diagnostic technologies have improved upon gold standard diagnostic methodologies by decreasing cost, increasing accuracy, and enhancing portability. However there has been little effort in adapting these technologies towards applications related to point-of-use monitoring of plant and crop health. Here, we take a step towards this vision by developing an approach that couples isothermal amplification of specific plant pathogen genomic sequences with customizable synthetic RNA regulators that are designed to trigger the production of a colorimetric output in cell-free gene expression reactions. We demonstrate our system can sense viral derived sequences with high-sensitivity and specificity, and can be utilized to directly detect viruses from infected plant material. Furthermore, we demonstrate that the entire system can operate using only body heat and naked-eye visual analysis of outputs. We anticipate these strategies to be important components of user-friendly and deployable diagnostic systems that can be configured to detect a range of important plant pathogens.","SUMMARYThe recent global outbreak of viral pneumonia designated as Coronavirus Disease 2019 (COVID-19) by coronavirus (SARS-CoV-2) has threatened global public health and urged to investigate its source. Whole genome analysis of SARS-CoV-2 revealed ~96% genomic similarity with bat CoV (RaTG13) and clustered together in phylogenetic tree. Furthermore, RaTGl3 also showed 97.43% spike protein similarity with SARS-CoV-2 suggesting that RaTGl3 is the closest strain. However, RBD and key amino acid residues supposed to be crucial for human-to-human and cross-species transmission are homologues between SARS-CoV-2 and pangolin CoVs. These results from our analysis suggest that SARS-CoV-2 is a recombinant virus of bat and pangolin CoVs. Moreover, this study also reports mutations in coding regions of 125 SARS-CoV-2 genomes signifying its aptitude for evolution. In short, our findings propose that homologous recombination has been occurred between bat and pangolin CoVs that triggered cross-species transmission and emergence of SARS-CoV-2, and, during the ongoing outbreak, SARS-CoV-2 is still evolving for its adaptability.","AbstractCOVID-19 caused by the emerging human coronavirus, SARS-CoV-2, has become a global pandemic, leading a serious threat to human health. So far, there is none vaccines or specific antiviral drugs approved for that. Therapeutic antibodies for SARS-CoV-2, was obtained from hyper immune equine plasma in this study. Herein, SARS-CoV-2 RBD with gram level were obtained through Chinese hamster ovary cells high-density fermentation. The binding of RBD to SARS-CoV-2 receptor, human ACE2, was verified and the efficacy of RBD in vivo was tested on mice and then on horses. As a result, RBD triggered high-titer neutralizing antibodies in vivo, and immunoglobulin fragment F(ab\u2019)2 was prepared from horse antisera through removing Fc. Neutralization test demonstrated that RBD-specific F(ab\u2019)2 inhibited SARS-CoV-2 with EC50 at 0.07 \u03bcg/ml, showing a potent inhibitory effect on SARS-CoV-2. These results highlights as RBD-specific F(ab\u2019)2 as therapeutic candidate for SARS-CoV-2.","ABSTRACTRATIONALECurrent microbiologic diagnostics often fail to identify the etiology of lower respiratory tract infections (LRTI) in hematopoietic cellular transplant recipients (HCT), which precludes the implementation of targeted therapies.OBJECTIVESTo address the need for improved LRTI diagnostics, we evaluated the utility of metagenomic next generation sequencing (mNGS) of bronchoalveolar lavage (BAL) to detect microbial pathogens in HCT patients with acute respiratory illnesses.METHODSWe enrolled 22 post-HCT adults ages 19-69 years with acute respiratory illnesses who underwent BAL at the University of Michigan between January 2012 and May 2013. mNGS was performed on BAL fluid to detect microbes and simultaneously assess the host transcriptional response. Results were compared against conventional microbiologic assays.MEASUREMENTS &amp; MAIN RESULTSmNGS demonstrated 100% sensitivity for detecting respiratory microbes (human metapneumovirus, respiratory syncytial virus, Stenotrophomonas maltophilia, human herpesvirus 6 and cytomegalovirus) when compared to standard testing. Previously unrecognized LRTI pathogens were identified in six patients for whom standard testing was negative (human coronavirus 229E, human rhinovirus A, Corynebacterium propinquum and Streptococcus mitis); findings were confirmed by independent PCR and 16S rRNA sequencing. Relative to patients without infection, patients with infection had increased expression of immunity related genes (p=0.022) and significantly lower diversity of their respiratory microbiome (p=0.017).CONCLUSIONSCompared to conventional diagnostics, mNGS enhanced detection of pathogens in BAL fluid from HCT patients. Furthermore, our results suggest that combining unbiased microbial pathogen detection with assessment of host gene biomarkers of immune response may hold promise for enhancing the diagnosis of post-HCT respiratory infections.","SummaryThe recent pandemic of coronavirus disease 2019 (COVID-19) is caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). COVID-19 was first reported in China (December 2019) and now prevalent in \u223c170 countries across the globe. Entry of SARS-CoV-2 into mammalian cells require the binding of viral Spike (S) proteins to the ACE2 (angiotensin converting enzyme 2) receptor. Once entered the S protein is primed by a specialised serine protease, TMPRSS2 (Transmembrane Serine Protease 2) in the host cell. Importantly, beside respiratory symptoms, consistent with other common respiratory virus infection when patients become viraemic, a significant number of COVID-19 patients also develop liver comorbidities. We explored if specific target cell-type in the mammalian liver, could be implicated in disease pathophysiology other than the general deleterious response to cytokine storms. Here we employed single-cell RNA-seq (scRNA-seq) to survey the human liver and identified potentially implicated liver cell-type for viral ingress. We report the co-expression of ACE2 and TMPRSS2 in a TROP2+ liver progenitor population. Importantly, we fail to detect the expression of ACE2 in hepatocyte or any other liver (immune and stromal) cell types. These results indicated that in COVID-19 associated liver dysfunction and cell death, viral infection of TROP2+ progenitors in liver may significantly impaired liver regeneration and could lead to pathology.Highlights- EPCAM+ Liver progenitors co-express ACE2 and TMPRSS2- ACE2 and TMPRSS2 expression is highest in TROP2high progenitors- ACE2 and TMPRSS2 cells express cholangiocyte biased fate markers- ACE2 and TMPRSS2 positive cells are absent in human fetal liver","AbstractSpreading dynamics analysis is an important and interesting topic since it has many applications such as rumor or disease controlling, viral marketing and information recommending. Many state-of-the-art researches focus on predicting infection scale or threshold. Few researchers pay attention to the predicting of infection nodes from a snapshot. With developing of precision marketing, recommending and, controlling, how to predict infection nodes precisely from snapshot becomes a key issue in spreading dynamics analysis. In this paper, a probability based prediction model is presented so as to estimate the infection nodes from a snapshot of spreading. Experimental results on synthetic and real networks demonstrate that the model proposed could predict the infection nodes precisely in the sense of probability.","ABSTRACTThe outbreak of the coronavirus disease 2019 (COVID-19), caused by the Severe Acute Respiratory Syndrome coronavirus 2 (SARS-CoV-2), has infected more than 100,000 people worldwide with over 3,000 deaths since December 2019. There is no cure for COVID-19 and the vaccine development is estimated to require 12-18 months. Here we demonstrate a CRISPR-Cas13-based strategy, PAC-MAN (Prophylactic Antiviral CRISPR in huMAN cells), for viral inhibition that can effectively degrade SARS-CoV-2 sequences and live influenza A virus (IAV) genome in human lung epithelial cells. We designed and screened a group of CRISPR RNAs (crRNAs) targeting conserved viral regions and identified functional crRNAs for cleaving SARS-CoV-2. The approach is effective in reducing respiratory cell viral replication for H1N1 IAV. Our bioinformatic analysis showed a group of only six crRNAs can target more than 90% of all coronaviruses. The PAC-MAN approach is potentially a rapidly implementable pan-coronavirus strategy to deal with emerging pandemic strains.","AbstractThe emergence of novel SARS coronavirus 2 (SARS-CoV-2) in 2019 has triggered an ongoing global pandemic of severe pneumonia-like disease designated as coronavirus disease 2019 (COVID-19). To date, more than 2.1 million confirmed cases and 139,500 deaths have been reported worldwide, and there are currently no medical countermeasures available to prevent or treat the disease. As the development of a vaccine could require at least 12-18 months, and the typical timeline from hit finding to drug registration of an antiviral is &gt;10 years, repositioning of known drugs can significantly accelerate the development and deployment of therapies for COVID-19. To identify therapeutics that can be repurposed as SARS-CoV-2 antivirals, we profiled a library of known drugs encompassing approximately 12,000 clinical-stage or FDA-approved small molecules. Here, we report the identification of 30 known drugs that inhibit viral replication. Of these, six were characterized for cellular dose-activity relationships, and showed effective concentrations likely to be commensurate with therapeutic doses in patients. These include the PIKfyve kinase inhibitor Apilimod, cysteine protease inhibitors MDL-28170, Z LVG CHN2, VBY-825, and ONO 5334, and the CCR1 antagonist MLN-3897. Since many of these molecules have advanced into the clinic, the known pharmacological and human safety profiles of these compounds will accelerate their preclinical and clinical evaluation for COVID-19 treatment.","AbstractHealth-care and public health professionals rely on accurate, real-time monitoring of infectious diseases for outbreak preparedness and response. Early detection of outbreaks is improved by systems that are pathogen-specific. We describe a system, FilmArray\u00ae Trend, for rapid disease reporting that is syndrome-based but pathogen-specific. Results from a multiplex molecular diagnostic test are sent directly to a cloud database. www.syndromictrends.com presents these data in near real-time. Trend preserves patient privacy by removing or obfuscating patient identifiers. We summarize the respiratory pathogen results, for 20 organisms from 344,000 patient samples acquired as standard of care testing over the last four years from 20 clinical laboratories in the United States. The majority of pathogens show influenza-like seasonality, rhinovirus has fall and spring peaks and adenovirus and bacterial pathogens show constant detection over the year. Interestingly, the rate of pathogen co-detections, on average 7.7%, matches predictions based on the relative abundance of organisms present.","Abstract2019 Novel Coronavirus (2019-nCoV) is a virus identified as the cause of the outbreak of pneumonia first detected in Wuhan, China. Investigations on the transmissibility, severity, and other features associated with this virus are ongoing. Currently, there is no vaccine or therapeutic antibody to prevent the infection, and more time is required to develop an effective immune strategy against the pathogen. In contrast, specific inhibitors targeting the key protease involved in replication and proliferation of the virus are the most effective means to alleviate the epidemic. The main protease of SARS-CoV is essential for the life cycle of the virus, which showed 96.1% of similarity with the main proteaseof 2019-nCoV, is considered to be an attractive target for drug development. In this study, we have identified 4 small molecular drugs with high binding capacity with SARS-CoV main protease by high-throughput screening based on the 8,000 clinical drug libraries, all these drugs have been widely used in clinical applications with guaranteed safety, which may serve as promising candidates to treat the infection of 2019-nCoV.","AbstractEmerging infectious diseases arising from pathogen spillover from mammals to humans comprise a substantial health threat. Tracing virus origin and predicting the most likely host species for future spillover events are major objectives in One Health disciplines. However, the species that share pathogens most widely with other mammals, and the role of different wildlife groups in sharing viruses with humans remain poorly identified. To address this challenge, we applied network analysis and Bayesian hierarchical models to a global database of mammal-virus associations. We show that domesticated mammals and some primates hold the most central positions in networks of known mammal-virus associations. We revealed strong evidence that DNA viruses were phylogenetically more host specific than RNA viruses, while the frequencies of sharing viruses among hosts and the proportion of zoonotic viruses in hosts were larger for RNA than DNA viruses. Among entire host-virus networks, Carnivora and Chiroptera hold central positions for mainly sharing RNA viruses with other host species, while network centrality of Primates scored relatively high for sharing DNA viruses. Ungulates hold central positions for sharing both RNA and DNA viruses. Acknowledging the role of domestic species in addition to host and virus traits in patterns of virus sharing is necessary to improve our understanding of virus spread and spillover in times of global change.","ABSTRACTWe utilized a cell-level approach to examine glycolytic pathways in the DLPFC of subjects with schizophrenia (n=16) and control (n=16) subjects and found decreased mRNA expression of glycolytic enzymes in pyramidal neurons, but not astrocytes. To replicate these novel bioenergetic findings, we probed independent datasets for bioenergetic targets and found similar abnormalities. Next, we used a novel strategy to build a schizophrenia bioenergetic profile by a tailored application of the Library of Integrated Network-Based Cellular Signatures data portal (iLINCS) and investigated connected cellular pathways, kinases, and transcription factors using Enrichr. Finally, with the goal of identifying drugs capable of \u201creversing\u201d the bioenergetic schizophrenia signature, we performed a connectivity analysis with iLINCS and identified peroxisome proliferator-activated receptor (PPAR) agonists as promising therapeutic targets. We administered a PPAR agonist to the GluN1 knockdown model of schizophrenia and found it improved long-term memory. Taken together, our findings suggest that tailored bioinformatics approaches, coupled with the LINCS library of transcriptional signatures of chemical and genetic perturbagens may be employed to identify novel treatment strategies for schizophrenia and related diseases.","AbstractsFSHD is a genetically dominant myopathy caused by mutations that cause expression of the normally silent DUX4 gene. This transcription factor has been shown to interfere with myogenesis when misexpressed at very low levels in myoblasts, and to cause cell death when overexpressed at high levels. A previous report using adeno-associated virus to deliver high levels of DUX4 to mouse skeletal muscle demonstrated severe pathology that was suppressed on a p53 knockout background, implying that DUX4 acted through the p53 pathway. Here, we investigate the p53-dependence of DUX4 using both in vitro cellular and the transgenic iDUX4[2.7] mouse models. We find that inhibiting p53 has no effect on the cytoxicity of DUX4 in vitro. When crossed onto the p53 null background, we find no suppression of the male-specific lethality or skin phenotypes of the DUX4 transgene, and find that primary myoblasts from this mouse are still killed by DUX4 expression. These data challenge the notion that the p53 pathway is central to the pathogenicity of DUX4.Summary StatementDUX4 is thought to mediate cytopathology through p53. Here, DUX4 is shown to kill primary myoblasts and promote pathological phenotypes in the iDUX4[2.7] mouse model on the p53-null background, calling into question this notion.","ABSTRACTCoronaviruses (CoV) are positive-sense RNA viruses that infect numerous mammalian and avian species and are capable of causing severe and lethal disease in humans. CoVs encode several innate immune antagonists that interact with the host innate immune response to facilitate efficient viral replication. CoV non-structural protein 14 (nsp14) encodes 3'-to-5' exoribonuclease activity (ExoN), which performs a proofreading function and is required for high-fidelity replication. Outside of the order Nidovirales, arenaviruses are the only RNA viruses that encode an ExoN, which functions to degrade dsRNA replication intermediates. In this study, we tested the hypothesis that CoV ExoN may also function to antagonize the innate immune response. We demonstrate that viruses lacking ExoN activity [ExoN(-)] are sensitive to cellular pretreatment with interferon beta (IFN-\u03b2) in a dose-dependent manner. In addition, ExoN(-) virus replication was attenuated in wild-type bone marrow-derived macrophages (BMMs) and partially restored in interferon alpha/beta receptor deficient (IFNAR-/-) BMMs. ExoN(-) virus replication did not result in IFN-\u03b2 gene expression, and in the presence of an IFN-\u03b2-mediated antiviral state, ExoN(-) viral RNA levels were not substantially reduced relative to untreated. However, ExoN(-) virus generated from IFN-\u03b2 pretreated cells had reduced specific infectivity and decreased relative fitness, suggesting that ExoN(-) virus generated during an antiviral state is less viable to establish a subsequent infection. Overall, our data suggest MHV ExoN activity is required for resistance to the innate immune response and antiviral mechanisms affecting the viral RNA sequence and/or an RNA modification act on viruses lacking ExoN activity.","AbstractBats host virulent zoonotic viruses without experiencing disease. A mechanistic understanding of the impact of bats\u2019 virus hosting capacities, including uniquely constitutive immune pathways, on cellular-scale viral dynamics is needed to elucidate zoonotic emergence. We carried out virus infectivity assays on bat cell lines expressing induced and constitutive immune phenotypes, then developed a theoretical model of our in vitro system, which we fit to empirical data. Best fit models recapitulated expected immune phenotypes for representative cell lines, supporting robust antiviral defenses in bat cells that correlated with higher estimates for within-host viral propagation rates. In general, heightened immune responses limit pathogen-induced cellular morbidity, which can facilitate the establishment of rapidly-propagating persistent infections within-host. Rapidly-replicating viruses that have evolved with bat immune systems will likely cause enhanced virulence following emergence into secondary hosts with immune systems that diverge from those unique to bats.","AbstractAngiotensin-converting enzyme 2 (ACE2) has been suggested to be the cellular receptor for the new coronavirus (2019-nCoV) that is causing the coronavirus disease 2019 (COVID-19). Like other coronaviruses such as the SARS-CoV, the 2019-nCoV uses the receptor binding domain (RBD) of the surface spike glycoprotein (S protein) to engage ACE2. We most recently determined the structure of the full-length human ACE2 in complex with a neutral amino acid transporter B0AT1. Here we report the cryo-EM structure of the full-length human ACE2 bound to the RBD of the 2019-nCoV at an overall resolution of 2.9 \u00c5 in the presence of B0AT1. The local resolution at the ACE2-RBD interface is 3.5 \u00c5, allowing analysis of the detailed interactions between the RBD and the receptor. Similar to that for the SARS-CoV, the RBD of the 2019-nCoV is recognized by the extracellular peptidase domain (PD) of ACE2 mainly through polar residues. Pairwise comparison reveals a number of variations that may determine the different affinities between ACE2 and the RBDs from these two related viruses.","AbstractFinding vaccine or specific antiviral treatment for global pandemic of virus diseases (such as the ongoing COVID-19) requires rapid analysis, annotation and evaluation of metagenomic libraries to enable a quick and efficient screening of nucleotide sequences. Traditional sequence alignment methods are not suitable and there is a need for fast alignment-free techniques for sequence analysis. Information theory and data compression algorithms provide a rich set of mathematical and computational tools to capture essential patterns in biological sequences. In 2013, our research group (Nagaraj et al., Eur. Phys. J. Special Topics 222(3-4), 2013) has proposed a novel measure known as Effort-To-Compress (ETC) based on the notion of compression-complexity to capture the information content of sequences. In this study, we propose a compression-complexity based distance measure for automatic identification of SARS coronavirus strains from a set of viruses using only short fragments of nucleotide sequences. We also demonstrate that our proposed method can correctly distinguish SARS-CoV-2 from SARS-CoV-1 viruses by analyzing very short segments of nucleotide sequences. This work could be extended further to enable medical practitioners in automatically identifying and characterizing SARS coronavirus strain in a fast and efficient fashion using short and/or incomplete segments of nucleotide sequences. Potentially, the need for sequence assembly can be circumvented.NoteThe main ideas and results of this research were first presented at the International Conference on Nonlinear Systems and Dynamics (CNSD-2013) held at Indian Institute of Technology, Indore, December 12, 2013. In this manuscript, we have extended our preliminary analysis to include SARS-CoV-2 virus as well.","AbstractThe process of molecular adaptation following a cross-species virus transmission event is currently poorly understood. Here, we identified 137 protein sites that experienced deceleration in their rate of evolution along the HIV-1/SIV phylogeny, likely indicating gain-of-function and consequent adaptation. The majority of such events occurred in parallel to cross-species transmission events and varied between HIV-1 groups, indicating independent adaptation strategies. The evolutionary rate decelerations we found were particularly prominent in accessory proteins that counteract host antiviral restriction factors, suggesting that these factors are a major barrier to viral adaptation to a new host. Surprisingly, we observed that the non-pandemic HIV-1 group O, derived from gorillas, exhibited more rate deceleration events than the pandemic group M, derived from chimpanzees. We suggest that the species barrier is higher when the genetic distance of the hosts increases. Our approach paves the way for subsequent studies on cross-species transfers in other major pathogens.","SummaryMonitoring the mutation dynamics of SARS-CoV-2 is critical for the development of effective approaches to contain the pathogen. By analyzing 106 SARS-CoV-2 and 39 SARS genome sequences, we provided direct genetic evidence that SARS-CoV-2 has a much lower mutation rate than SARS. Minimum Evolution phylogeny analysis revealed the putative original status of SARS-CoV-2 and the early-stage spread history. The discrepant phylogenies for the spike protein and its receptor binding domain proved a previously reported structural rearrangement prior to the emergence of SARS-CoV-2. Despite that we found the spike glycoprotein of SARS-CoV-2 is particularly more conserved, we identified a mutation that leads to weaker receptor binding capability, which concerns a SARS-CoV-2 sample collected on 27th January 2020 from India. This represents the first report of a significant SARS-CoV-2 mutant, and raises the alarm that the ongoing vaccine development may become futile in future epidemic if more mutations were identified.HighlightsBased on the currently available genome sequence data, we proved that SARS-COV-2 genome has a much lower mutation rate and genetic diversity than SARS during the 2002-2003 outbreak.The spike (S) protein encoding gene of SARS-COV-2 is found relatively more conserved than other protein-encoding genes, which is a good indication for the ongoing antiviral drug and vaccine development.Minimum Evolution phylogeny analysis revealed the putative original status of SARS-CoV-2 and the early-stage spread history.We confirmed a previously reported rearrangement in the S protein arrangement of SARS-COV-2, and propose that this rearrangement should have occurred between human SARS-CoV and a bat SARS-CoV, at a time point much earlier before SARS-COV-2 transmission to human.We provided first evidence that a mutated SARS-COV-2 with reduced human ACE2 receptor binding affinity have emerged in India based on a sample collected on 27th January 2020.","AbstractAn outbreak of the Corona Virus Disease 2019 (COVID-19), caused by the severe acute respiratory syndrome CoV-2 (SARS-CoV-2), began in Wuhan and spread globally. Recently, it has been reported that discharged patients in China and elsewhere were testing positive after recovering. However, it remains unclear whether the convalescing patients have a risk of \u201crelapse\u201d or \u201creinfection\u201d. The longitudinal tracking of re-exposure after the disappeared symptoms of the SARS-CoV-2-infected monkeys was performed in this study. We found that weight loss in some monkeys, viral replication mainly in nose, pharynx, lung and gut, as well as moderate interstitial pneumonia at 7 days post-infection (dpi) were clearly observed in rhesus monkeys after the primary infection. After the symptoms were alleviated and the specific antibody tested positively, the half of infected monkeys were rechallenged with the same dose of SARS-CoV-2 strain. Notably, neither viral loads in nasopharyngeal and anal swabs along timeline nor viral replication in all primary tissue compartments at 5 days post-reinfection (dpr) was found in re-exposed monkeys. Combined with the follow-up virologic, radiological and pathological findings, the monkeys with re-exposure showed no recurrence of COVID-19, similarly to the infected monkey without rechallenge. Taken together, our results indicated that the primary SARS-CoV-2 infection could protect from subsequent exposures, which have the reference of prognosis of the disease and vital implications for vaccine design.","AbstractBluetongue virus (BTV) is an arbovirus transmitted by blood-feeding midges to a wide range of wild and domestic ruminants. In this report, we showed that BTV, through its virulence non-structural protein NS3 (BTV-NS3), is able to activate the MAPK/ERK pathway. In response to growth factors, the MAPK/ERK pathway activates cell survival, differentiation, proliferation and protein translation but can also lead to the production of several inflammatory cytokines. By combining immunoprecipitation of BTV-NS3 and mass spectrometry analysis from both BTV-infected and NS3-transfected cells, we identified the serine/threonine-protein kinase B-Raf (BRAF), a crucial player of the MAPK/ERK pathway, as a new cellular interactor of BTV-NS3. BRAF silencing led to a significant decrease of the MAPK/ERK activation by BTV supporting a model where BTV-NS3 interacts with BRAF to activate this signaling cascade. Furthermore, the intrinsic ability of BTV-NS3 to bind BRAF and activate the MAPK/ERK pathway is conserved throughout multiple serotypes/strains but appears to be specific to BTV compared to other members of Orbivirus genus. Inhibition of MAPK/ERK pathway with U0126 reduced viral titers, suggesting that BTV manipulates this pathway for its own replication. Therefore, the activation of the MAPK/ERK pathway by BTV-NS3 could benefit to BTV replication by promoting its own viral protein synthesis but could also explain the deleterious inflammation associated with tissue damages as already observed in severe cases of BT disease. Altogether, our data provide molecular mechanisms to explain the role of BTV-NS3 as a virulence factor and determinant of pathogenesis.ImportanceBluetongue Virus (BTV) is responsible of the non-contagious arthropod-borne disease Bluetongue (BT) transmitted to ruminants by blood-feeding midges. Despite the fact that BTV has been extensively studied, we still have little understanding of the molecular determinants of BTV virulence. In this report, we found that the virulence protein NS3 interacts with BRAF, a key component of the MAPK/ERK pathway. In response to growth factors, this pathway promotes cell survival, increases protein translation but also contributes to the production of inflammatory cytokines. We showed that BTV-NS3 enhances the MAPK/ERK pathway and this activation is BRAF-dependent. Our results demonstrate, at the molecular level, how a single virulence factor has evolved to target a cellular function to ensure its viral replication. On the other hand, our findings could also explain the deleterious inflammation associated with tissue damages as already observed in severe cases of BT disease.","AbstractGeneration intervals, defined as the time between when an individual is infected and when that individual infects another person, link two key quantities that describe an epidemic: the reproductive number,, and the rate of exponential growth,r. Generation intervals are often measured through contact tracing by identifying who infected whom. We study how observed intervals differ from \u201cintrinsic\u201d intervals that could be estimated by tracing individual-level infectiousness, and identify both spatial and temporal effects, including censoring (due to observation time), and the effects of susceptible depletion at various spatial scales. Early in an epidemic, we expect the variation in the observed generation intervals to be mainly driven by the censoring and the population structure near the source of disease spread; therefore, we predict that correcting observed intervals for the effect of temporal censoring butnotfor spatial effects will provide a spatially informed \u201ceffective\u201d generation-interval distribution, which will correctly linkrand. We develop and test statistical methods for temporal corrections of generation intervals, and confirm our prediction using individual-based simulations on an empirical network.","The COVID-19 pandemic caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is raging across the world, leading to a mortality rate of 3.4%. As a potential vaccine and therapeutic target, the nucleocapsid protein of SARS-CoV-2 (nCoVN) functions in packaging the viral genome and viral self-assembly. To investigate the biological effect of nCoVN to human induced pluripotent stem cells (iPSC), genetically engineered iPSC overexpressing nCoVN (iPSC-nCoVN) were generated by lentiviral expression systems. Unexpectedly, the morphology and proliferation rate of iPSC were changed after nCoVN expressing for two weeks. The pluripotency markers SSEA4 and TRA-1-81 were not detectable in iPSC-nCoVN. Meanwhile, iPSC-nCoVN lost the ability for differentiation into cardiomyocytes when using a routine differentiation protocol. Our data suggested that nCoVN disrupted the pluripotent properties of iPSC and turned them into fibroblasts, which provided a new insight to the pathogenic mechanism of SARS-CoV-2.","AbstractLinear plasmids with almost identical compact genetic organization have been found in the cytoplasm of yeast species from nine genera. We employed pGKL1,2 plasmids from Kluyveromyces lactis as a model to investigate the previously unstudied transcriptome of yeast cytoplasmic linear plasmids. We performed 5\u2019 and 3\u2019 RACE analysis of all the pGKL1,2 mRNAs and found them not 3\u2019 polyadenylated and containing mostly uncapped 5\u2019 poly(A) leaders that are not complementary to the plasmid DNA. The degree of 5\u2019 capping and/or 5\u2019 polyadenylation is specific to each gene and is controlled by the corresponding promoter regions. We refined the description of the pGKL1,2 promoters and found new alternative promoters of several genes. We also provide evidence that K2ORF3 encodes an mRNA cap guanine-N7-methyltransferase and that 5\u2019 capped pGKL1,2 transcripts contain N7-methylated caps. Translation of pGKL1,2 transcripts is enhanced in Ism1\u0394 and pab1\u0394 strains and is independent of eIF4E and Pab1 translation factors. We suggested a model of a primitive regulation of pGKL1,2 plasmids gene expression where degree of 5\u2019 mRNA capping, degree of 5\u2019 non-template polyadenylation and presence of negative regulators as PAB1 and Lsm1 play an important role. Our data also suggest a close relationship between linear plasmids and poxviruses.","The COVID-19 pandemic caused by SARS-CoV-2 is a public-health emergency of international concern and thus calling for the development of safe and effective therapeutics and prophylactics particularly a vaccine to protect against the infection. SARS-CoV-2 spike glycoprotein is an attractive candidate for vaccine, antibodies and inhibitor development because of many roles it plays in attachment, fusion and entry into the host cell. In this study, we characterized the SARS-CoV-2 spike glycoprotein by immune-informatics techniques to put forward potential B and T cell epitopes, followed by the use of epitopes in construction of a multi-epitope peptide vaccine construct (MEPVC). The MEPVC revealed robust host immune system simulation with high production of immunoglobulins, cytokines and interleukins. Stable conformation of the MEPVC with a representative innate immune TLR3 receptor was observed involving strong hydrophobic and hydrophilic chemical interactions, along with enhanced contribution from salt-bridges towards inter-molecular stability. Molecular dynamics simulation in solution aided further in interpreting strong affinity of the MEPVC for TLR3. This stability is the attribute of several vital residues from both TLR3 and MEPVC as shown by radial distribution function (RDF) and a novel analytical tool axial frequency distribution (AFD). Comprehensive binding free energies estimation was provided at the end that concluded major domination by electrostatic and minor from van der Waals. Summing all, the designed MEPVC has tremendous potential of providing protective immunity against COVID-19 and thus has the potential to be considered in experimental studies.","AbstractBackgroundThe ongoing COVID-19 pandemic has created an urgency to identify novel vaccine targets for protective immunity against SARS-CoV-2. Consistent with observations for SARS-CoV, a closely related coronavirus responsible for the 2003 SARS outbreak, early reports identify a protective role for both humoral and cell-mediated immunity for SARS CoV-2.MethodsIn this study, we leveraged HLA-I and HLA-II T cell epitope prediction tools from RECON\u00ae (Real-time Epitope Computation for ONcology), our bioinformatic pipeline that was developed using proteomic profiling of individual HLA-I and HLA-II alleles to predict rules for peptide binding to a diverse set of such alleles. We applied these binding predictors to viral genomes from the Coronaviridae family, and specifically to identify SARS-CoV-2 T cell epitopes.ResultsTo test the suitability of these tools to identify viral T cell epitopes, we first validated HLA-I and HLA-II predictions on Coronaviridae family epitopes deposited in the Virus Pathogen Database and Analysis Resource (ViPR) database. We then use our HLA-I and HLA-II predictors to identify 11,776 HLA-I and 7,991 HLA-II candidate binding peptides across all 12 open reading frames (ORFs) of SARS-CoV-2. This extensive list of identified candidate peptides is driven by the length of the ORFs and the significant number of HLA-I and HLA-II alleles that we are able to predict (74 and 83, respectively), providing over 99% coverage for the US, European and Asian populations, for both HLA-I and HLA-II. From our SARS-CoV-2 predicted peptide-HLA-I allele pairs, 368 pairs identically matched previously reported pairs in the ViPR database, originating from other forms of coronaviruses. 320 of these pairs (89.1%) had a positive MHC-binding assay result. This analysis reinforces the validity our predictions.ConclusionsUsing this bioinformatic platform, we identify multiple putative epitopes for CD4+ and CD8+ T cells whose HLA binding properties cover nearly the entire population and thus may be effective when included in prophylactic vaccines against SARS-CoV-2 to induce broad cellular immunity.","AbstractModel-based epidemiological assessment is useful to support decision-making at the beginning of an emerging Aedes-transmitted outbreak. However, early forecasts are generally unreliable as little information is available in the first few incidence data points. Here, we show how past Aedes-transmitted epidemics help improve these predictions. The approach was applied to the 2015-2017 Zika virus epidemics in three islands of the French West Indies, with historical data including other Aedes-transmitted diseases (Chikungunya and Zika) in the same and other locations. Hierarchical models were used to build informative a priori distributions on the reproduction ratio and the reporting rates. The accuracy and sharpness of forecasts improved substantially when these a priori distributions were used in models for prediction. For example, early forecasts of final epidemic size obtained without historical information were 3.3 times too high on average (range: 0.2 to 5.8) with respect to the eventual size, but were far closer (1.1 times the real value on average, range: 0.4 to 1.5) using information on past CHIKV epidemics in the same places. Likewise, the 97.5% upper bound for maximal incidence was 15.3 times (range: 2.0 to 63.1) the actual peak incidence, and became much sharper at 2.4 times (range: 1.3 to 3.9) the actual peak incidence with informative a priori distributions. Improvements were more limited for the date of peak incidence and the total duration of the epidemic. The framework can adapt to all forecasting models at the early stages of emerging Aedes-transmitted outbreaks.","We dissect the mechanism of SARS-CoV-2 in human lung hosts from the initial phase of receptor binding to viral replication machinery using a systems biological approach. Two independent lung protein interactome were contracted to revel the signaling process on receptor activation and host protein hijacking machinery in the pathogenesis virus. Most of the functional hubs interactome proteins were associated with SARS-CoV-2 infection. Also, 52% of the proteins related to cardiovascular disease, and they involved in viral infection, which causes vulnerability and severity in the risk individuals. We also showed five interconnecting cascades between hubs of both networks that explore the preparation of the optimal environment in the host for viral replication initiated at the phase of receptor attachment. Interestingly, we propose that seven potential miRNAs targeting the intermediate phase that connects receptor and viral replication process a better choice as a drug for SARS-CoV-2.","AbstractAs the infection of 2019-nCoV coronavirus is quickly developing into a global pneumonia epidemic, careful analysis of its transmission and cellular mechanisms is sorely needed. In this report, we re-analyzed the computational approaches and findings presented in two recent manuscripts by Ji et al. (https://doi.org/10.1002/jmv.25682) and by Pradhan et al. (https://doi.org/10.1101/2020.01.30.927871), which concluded that snakes are the intermediate hosts of 2019-nCoV and that the 2019-nCoV spike protein insertions shared a unique similarity to HIV-1. Results from our re-implementation of the analyses, built on larger-scale datasets using state-of-the-art bioinformatics methods and databases, do not support the conclusions proposed by these manuscripts. Based on our analyses and existing data of coronaviruses, we concluded that the intermediate hosts of 2019-nCoV are more likely to be mammals and birds than snakes, and that the \u201cnovel insertions\u201d observed in the spike protein are naturally evolved from bat coronaviruses.","AbstractThe recent pandemic created due to Novel Coronavirus (nCOV-2019) from Wuhan, China demanding a large scale of a general health emergency. This demands novel research on the vaccine to fight against this pandemic situation, re-purposing of the existing drugs, phylogenetic analysis to identify the origin and determine the similarity with other known viruses, etc. The very preliminary task from the research community is to analyze the wide verities of existing related research articles, which is very much time-consuming in such situations where each minute counts for saving hundreds of human lives. The entire manual processing is even lower down the efficiency in mining the information. We have developed a complete automatic literature mining system that delivers efficient and fast mining from existing biomedical literature databases. With the help of modern-day deep learning algorithms, our system also delivers a summarization of important research articles that provides ease and fast comprehension of critical research articles. The system is currently scanning nearly 1,46,115,136 English words from 29,315 research articles in not greater than 1.5 seconds with multiple search keywords. Our research article presents the criticality of literature mining, especially in pandemic situations with the implementation and online deployment of the system.","AbstractProtein kinase R (PKR) plays a major role in activating host immunity during infection by sensing dsRNA produced by viruses. Once activated by dsRNA, PKR phosphorylates the translation factor eIF2\u03b1, halting cellular translation. Many viruses have methods of inhibiting PKR activation or its downstream effects, circumventing protein synthesis shutdown. These include sequestering dsRNA or producing proteins that bind to and inhibit PKR activation. Here we describe our finding that in multiple cell types, PKR was depleted during mouse adenovirus type 1 (MAV-1) infection. MAV-1 did not appear to be targeting PKR at a transcriptional or translational level because total PKR mRNA levels and levels of PKR mRNA bound to polysomes were unchanged or increased during MAV-1 infection. However, inhibiting the proteasome reduced the PKR depletion seen in MAV-1-infected cells, whereas inhibiting the lysosome had no effect. This suggests that proteasomal degradation alone is responsible for PKR degradation during MAV-1 infection. Time course experiments indicate that the degradation occurs early after infection. Infecting cells with UV-inactivated virus prevented PKR degradation, whereas inhibiting viral DNA replication did not. Together these results suggest that an early viral gene is responsible. Degradation of PKR is a rare mechanism to oppose PKR activity, and it has only been described in six RNA viruses. To our knowledge, this is the first example of a DNA virus counteracting PKR by degrading it.ImportanceThe first line of defense in cells during viral infection is the innate immune system, which is activated by different viral products. PKR is a part of this innate immune system and is induced by interferon and activated by dsRNA produced by DNA and RNA viruses. PKR is such an important part of the antiviral response that many viral families have gene products to counteract its activation or the resulting effects of its activity. Although a few RNA viruses degrade PKR, this method of counteracting PKR has not been reported for any DNA viruses. MAV-1 does not encode virus-associated RNAs, a human adenoviral defense against PKR activation. Instead, MAV-1 degrades PKR, and it is the first DNA virus reported to do so. The innate immune evasion by PKR degradation is a previously unidentified way for a DNA virus to circumvent the host antiviral response.","AbstractATP-dependent chromatin remodelling proteins represent a diverse family of proteins that share ATPase domains that are adapted to regulate protein-DNA interactions. Here we present structures of the yeast Chd1 protein engaged with nucleosomes in the presence of the transition state mimic ADP-beryllium fluoride. The path of DNA strands through the ATPase domains indicates the presence of contacts conserved with single strand translocases and additional contacts with both strands that are unique to Snf2 related proteins. The structure provides connectivity between rearrangement of ATPase lobes to a closed, nucleotide bound state and the sensing of linker DNA. Two turns of linker DNA are prised off the surface of the histone octamer as a result of Chd1 binding, and both the histone H3 tail and ubiquitin conjugated to lysine 120 are re-orientated towards the unravelled DNA. This indicates how changes to nucleosome structure can alter the way in which histone epitopes are presented.","The ongoing SARS-CoV-2 pandemic demands rapid identification of immunogenic targets for the design of efficient vaccines and serological detection tools. In this report, using pools of overlapping linear peptides and functional assays, we present two immunodominant regions on the spike glycoprotein that were highly recognized by neutralizing antibodies in the sera of COVID-19 convalescent patients. One is highly specific to SARS-CoV-2, and the other is a potential pan-coronavirus target.","AbstractAlpha herpesviruses naturally infect the peripheral nervous system, and can spread to the central nervous system causing severe deadly or debilitating disease. Because alpha herpesviruses spread along synaptic circuits, and infected neurons exhibit altered electrophysiology and increased spontaneous firing, we hypothesized that alpha herpesviruses use activity-dependent synaptic vesicle-like regulated secretory mechanisms for egress and spread from neurons. To address this hypothesis, we used a compartmentalized primary neuron culture system to measure egress and spread of pseudorabies virus (PRV), pharmacological and optogenetics approaches to modulate neuronal firing activity, and a live-cell fluorescence microscopy assay to directly visualize the exocytosis of individual virus particles from infected neurons. Using tetrodotoxin to silence neuronal activity, we observed no inhibition of virus spread, and using potassium chloride or optogenetics to elevate neuronal activity, we also show no increase in virus spread. Using a live-cell fluorescence microscopy method to directly measure virus egress from infected neurons, we observed no association between virus particle exocytosis and intracellular Ca2+ signaling. Finally, we observed virus particle exocytosis occurs in association with constitutive secretory Rab GTPases, Rab6a and Rab8a, not Rab proteins that are associated with the Ca2+-regulated secretory pathway in neurons, Rab3a and Rab11a. Therefore, we conclude that alpha herpesvirus egress and spread is independent of neuronal activity and Ca2+ signaling because virus particle exocytosis uses constitutive secretory mechanisms in neurons.Author SummaryAlpha herpesviruses, including important human pathogens Herpes Simplex Virus 1 and 2, and Varicella-Zoster Virus, are among the very few viruses that naturally infect the nervous system. These viruses cause recurrent herpetic and zosteriform lesions, peripheral neuropathies, and deadly or debilitating central nervous system diseases. Many of the molecular and cellular mechanisms of viral egress and spread remain unknown, particularly in the context of specialized neuronal cell biology. Our results indicate that elevated firing activity of infected neurons is not functionally or mechanistically linked to virus egress and spread; therefore, therapies targeting peripheral neuropathic symptoms, elevated neuronal activity, and synaptic vesicle secretory mechanisms are unlikely to affect virus spread in the nervous system.","AbstractTransmission between hosts is a critical part of the viral lifecycle. Recent studies of viral transmission have used genome sequence data to evaluate the number of particles transmitted between hosts, and the role of selection as it operates during the transmission process. However, the interpretation of sequence data describing transmission events is a challenging task. We here present a novel and comprehensive framework for using short-read sequence data to understand viral transmission events. Our model describes transmission as an event involving whole viruses, rather than independent alleles. We demonstrate how selection and noisy sequence data may each affect inferences of the population bottleneck, and identify circumstances in which selection for increased viral transmission may or may not be identified. Applying our model to data from a previous experimental transmission study, we show that our approach grants a more quantitative insight into viral transmission, inferring that between 2 to 6 viruses initiated infection, and allowing for a more informed interpretation of transmission events. While our model is here applied to influenza transmission, the framework we present is highly generalisable to other systems. Our work provides new opportunities for studying viral transmission.","Angiotensin-converting enzyme 2 (ACE2) has been identified as a critical receptor for severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). This has led to extensive speculation on the role of ACE2 in disease severity, and in particular, whether variation in its expression can explain higher mortality in older individuals. We examine this question in mouse lung and show that 24-month old mice have significantly reduced ACE2 mRNA expression relative to 3-month old mice. The differences appear to be localized to ciliated cells.","SummaryProgrammed ribosomal frameshifting is the controlled slippage of the translating ribosome to an alternative frame. This tightly regulated process is widely employed by human viruses such as HIV and SARS coronavirus and is critical for their life cycle and virulence. It is also utilized from yeast to human to implement a feedback control mechanism to regulate polyamine levels. Here, we developed a high-throughput, fluorescence-based approach to assess the frameshifting potential of a sequence. We designed and tested &gt;12,000 sequences based on 15 viral and human frameshifting events, allowing us to elucidate the rules governing ribosomal frameshifting in a systematic way and to discover novel regulatory inputs based on amino acid properties and tRNA availability. We assessed the natural variation in HIV gag-pol frameshifting rates by testing &gt;500 clinical isolates and identified subtype-specific differences as well as associations between viral load in patients and the optimality of gag-pol frameshifting rates. We further devised computational models that accurately predict frameshifting potential (up to auROC=0.93) and frameshifting rates (up to Pearson r=0.81) of novel variants, including subtle differences between HIV clinical isolates (r=0.60). Taken together, this systematic approach can contribute to the development of antiviral agents acting on programmed ribosomal frameshifting.","AbstractWith an ongoing threat posed by circulating zoonotic strains, new strategies are required to prepare for the next emergent coronavirus (CoV). Previously, groups had targeted conserved coronavirus proteins as a strategy to generate live-attenuated vaccine strains against current and future CoVs. With this in mind, we explored whether manipulation of CoV NSP16, a conserved 2\u2019O methyltransferase (MTase), could provide a broad attenuation platform against future emergent strains. Using the SARS-CoV mouse model, a NSP16 mutant vaccine was evaluated for protection from heterologous challenge, efficacy in the aging host, and potential for reversion to pathogenesis. Despite some success, concerns for virulence in the aged and potential for reversion makes targeting NSP16 alone an untenable approach. However, combining a 2\u2019O MTase mutation with a previously described CoV fidelity mutant produced a vaccine strain capable of protection from heterologous virus challenge, efficacy in aged mice, and no evidence for reversion. Together, the results indicate that targeting the CoV 2\u2019O MTase in parallel with other conserved attenuating mutations may provide a platform strategy for rapidly generating live-attenuated coronavirus vaccines.SignificanceEmergent coronaviruses remain a significant threat to global public health and rapid response vaccine platforms are needed to stem future outbreaks. However, failure of many previous CoV vaccine formulations has clearly highlighted the need to test efficacy under different conditions and especially in vulnerable populations like the aged and immune-compromised. This study illustrates that despite success in young models, the NSP16 mutant carries too much risk for pathogenesis and reversion in vulnerable models to be used as a stand-alone vaccine strategy. Importantly, the NSP16 mutation can be paired with other attenuating approaches to provide robust protection from heterologous challenge and in vulnerable populations. Coupled with increased safety and reduced pathogenesis, the study highlights the potential for NSP16 attenuation as a major component of future live-attenuated coronavirus vaccines.","Protein structures are crucial for understanding their biological activities. Since the outbreak of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), there is an urgent need to understand the biological behavior of the virus and provide a basis for developing effective therapies. Since the proteome of the virus was determined, some of the protein structures could be determined experimentally, and others were predicted via template-based modeling approaches. However, tertiary structures for several proteins are still not available from experiment nor they could be accurately predicted by template-based modeling because of lack of close homolog structures. Previous efforts to predict structures for these proteins include efforts by DeepMind and the Zhang group via machine learning-based structure prediction methods, i.e. AlphaFold and C-I-TASSER. However, the predicted models vary greatly and have not yet been subjected to refinement. Here, we are reporting new predictions from our in-house structure prediction pipeline. The pipeline takes advantage of inter-residue contact predictions from trRosetta, a machine learning-based method. The predicted models were further improved by applying molecular dynamics simulation-based refinement. We also took the AlphaFold models and refined them by applying the same refinement method. Models based on our structure prediction pipeline and the refined AlphaFold models were analyzed and compared with the C-I-TASSER models. All of our models are available at https://github.com/feiglab/sars-cov-2-proteins .","AbstractThe outbreaks of 2002/2003 SARS, 2012/2015 MERS and 2019/2020 Wuhan respiratory syndrome clearly indicate that genome evolution of an animal coronavirus (CoV) may enable it to acquire human transmission ability, and thereby to cause serious threats to global public health. It is widely accepted that CoV human transmission is driven by the interactions of its spike protein (S-protein) with human receptor on host cell surface; so, quantitative evaluation of these interactions may be used to assess the human transmission capability of CoVs. However, quantitative methods directly using viral genome data are still lacking. Here, we perform large-scale protein-protein docking to quantify the interactions of 2019-nCoV S-protein receptor-binding domain (S-RBD) with human receptor ACE2, based on experimental SARS-CoV S-RBD-ACE2 complex structure. By sampling a large number of thermodynamically probable binding conformations with Monte Carlo algorithm, this approach successfully identified the experimental complex structure as the lowest-energy receptor-binding conformations, and hence established an experiment-based strength reference for evaluating the receptor-binding affinity of 2019-nCoV via comparison with SARS-CoV. Our results show that this binding affinity is about 73% of that of SARS-CoV, supporting that 2019-nCoV may cause human transmission similar to that of SARS-CoV. Thus, this study presents a method for rapidly assessing the human transmission capability of a newly emerged CoV and its mutant strains, and demonstrates that post-genome analysis of protein-protein interactions may provide early scientific guidance for viral prevention and control.","AbstractBased on genome-scale loss-of-function screens we discovered that Topoisomerase III-\u00df (TOP3B), a human topoisomerase that acts on DNA and RNA, is required for yellow fever virus and dengue virus-2 replication. Remarkably, we found that TOP3B is required for efficient replication of all positive-sense-single stranded RNA viruses tested, including SARS-CoV-2. While there are no drugs that specifically inhibit this topoisomerase, we posit that TOP3B is an attractive anti-viral target.","AbstractSevere acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is the etiological agent of the ongoing pandemic of 2019 CoV disease (COVID-19), which is already responsible for far more deaths than were reported during the previous public health emergencies of international concern provoked by two related pathogenic coronaviruses (CoVs) from 2002 and 2012. The identification of any clinically approved drug that could be repurposed to combat COVID-19 would allow the rapid implementation of potentially life-saving procedures to complement social distancing and isolation protocols. The major protease (Mpro) of SARS-CoV-2 is considered a promising target for drug interventions, based on results from related CoVs with lopinavir (LPV) an HIV protease inhibitor, that that can inhibit the Mpro of 2002 SARS-CoV. However, limited evidence exists for other clinically approved anti-retroviral protease inhibitors that may bind more efficiently to Mpro from SARS-CoV-2 and block its replication. Of high interest is atazanavir (ATV) due to its documented bioavailability within the respiratory tract, which motivated our evaluation on its ability to impair SARS-CoV-2 replication through a series of in vitro experiments. A molecular dynamic analysis showed that ATV could dock in the active site of SARS-CoV-2 Mpro with greater strength than LPV and occupied the substrate cleft on the active side of the protease throughout the entire molecular dynamic analysis. In a cell-free protease assay, ATV was determined to block Mpro activity at a concentration of 10 \u03bcM. Next, a series of assays with in vitro models of virus infection/replications were performed using three cell types, Vero cells, a human pulmonary epithelial cell line and primary human monocytes, which confirmed that ATV could inhibit SARS-CoV-2 replication, alone or in combination with ritonavir (RTV). In addition, the virus-induced levels of IL-6 and TNF-\u03b1 were reduced in the presence of these drugs, which performed better than chloroquine, a compound recognized for its anti-viral and anti-inflammatory activities. Together, our data strongly suggest that ATV and ATV/RTV should be considered among the candidate repurposed drugs undergoing clinical trials in the fight against COVID-19.","The COVID-19 pandemic has caused an unprecedented global public health and economy crisis. The origin and emergence of its causal agent, SARS-CoV-2, in the human population remains mysterious, although bat and pangolin were proposed to be the natural reservoirs. Strikingly, comparing to the SARS-CoV-2-like CoVs identified in bats and pangolins, SARS-CoV-2 harbors a polybasic furin cleavage site in its spike (S) glycoprotein. SARS-CoV-2 uses human ACE2 as its receptor to infect cells. Receptor recognition by the S protein is the major determinant of host range, tissue tropism, and pathogenesis of coronaviruses. In an effort to search for the potential intermediate or amplifying animal hosts of SARS-CoV-2, we examined receptor activity of ACE2 from 14 mammal species and found that ACE2 from multiple species can support the infectious entry of lentiviral particles pseudotyped with the wild-type or furin cleavage site deficient S protein of SARS-CoV-2. ACE2 of human/rhesus monkey and rat/mouse exhibited the highest and lowest receptor activity, respectively. Among the remaining species, ACE2 from rabbit and pangolin strongly bound to the S1 subunit of SARS-CoV-2 S protein and efficiently supported the pseudotyped virus infection. These findings have important implications for understanding potential natural reservoirs, zoonotic transmission, human-to-animal transmission, and use of animal models.","AbstractThe role of the RNAi/Dicer/Ago system to degrade RNA viruses has been elusive, which prompt authors to think that interferon (IFN) synthesis is essential, relegating the dsRNAs as accessory function. We investigate SARS-CoV-2 genome responsible of the new deadly COVID-19 pandemic for the theoretical possibilities to engage intra pairing within the viral RNA and also hybrid pairing with human transcriptome. Segmental pieces of RNAs that originate from SARS-CoV-2 were computationally searched as a potential source of one strand, the complementary strand being from the host transcriptome. We therefore considered perfect complementarity of host RNA with any piece of SARS-CoV-2 RNA as a collection of theoretical siRNAs potentially Dicer substrates. Few human genes seems targeted by SARS-CoV-2 RNA, among them mitochondrial deubiquitinase USP30 and a subunit of ubiquitin protein ligase complex FBXO21 could explain premature death of infected cell by the collapse of mitochondria.","AbstractA new porcine coronavirus SADS-CoV was recently identified from suckling piglets with severe diarrhea in southern China and its genome sequence is most identical (~95% identity) to that of bat \u03b1-coronavirus HKU2. It again indicates bats are the natural reservoir of many coronaviruses that have great potential for cross-species transmission to animals and humans by recombination and/or mutation. Here we report the cryo-EM structures of HKU2 and SADS-CoV spike glycoprotein trimers at 2.38 \u00c5 and 2.83 \u00c5 resolution, respectively. HKU2 and SADS-CoV spikes exhibit very high structural similarity, with subtle differences mainly distributed in the NTD and CTD of the S1 subunit responsible for cell attachment and receptor binding. We systematically analyzed and compared the NTD, CTD, SD1 and SD2 domains of the S1 subunit and the S2 subunit of HKU2 spike with those of \u03b1-, \u03b2-, \u03b3-, and \u03b4-coronavirus spikes. The results show that the NTD and CTD of HKU2/SADS-CoV are probably the most ancestral in the evolution of spike. Although the S2 subunit mediating membrane fusion is highly conserved, the connecting region after fusion peptide in HKU2/SADS-CoV S2 subunit also adopts a conformation distinct from other coronaviruses. These results structurally demonstrate a close evolutionary relationship between HKU2 /SADS-CoV and \u03b2-coronavirus spikes and provide new insights into the evolution and cross-species transmission of coronaviruses.","Porcine epidemic diarrhoea virus (PEDV) devastates the health of piglets but may not infect piglets whose CMP-N-glycolylneuraminic acid hydroxylase (CMAH) gene mutated (knockouts, KO) by using CRISPR/Cas9 gene editing techniques. This hypothesis was tested by using KO piglets challenged with PEDV. Two single-guide RNAs targeting the CMAH gene and Cas9 mRNA were microinjected into the cytoplasm of newly fertilized eggs; 4 live founders generated and proven to be biallelic KO, lacking detectable N-glycolylneuraminic acid (NGNA). The founders were bred, and homozygous offspring were obtained. Two-day-old (in exps. I and III) and 3-day-old (in exp. II) KO and wild-type (WT) piglets were inoculated with TCID50 1x103 PEDV and then fed 20 mL of infant formula (in exps. I and II) or sow\u2019s colostrum (in exp. III) every 4 hours. In exp. III, the colostrum was offered 6 times and was then replaced with Ringer/5% glucose solution. At 72 hours post-PEDV inoculation (hpi), the animals were euthanized for necropsy, and their intestines were sampled. In all 3 experiments, the piglets showed apparent outward clinical manifestations suggesting that infection occurred despite the CMAH KO. In exp. I, all 6 WT piglets and only 1 of 6 KO piglets died at 72 hpi. Histopathology and immunofluorescence staining showed that the villus epithelial cells of WT piglets were severely exfoliated, but only moderate exfoliation and enterocyte vacuolization was observed in KO piglets. In exp. II, delayed clinical symptoms appeared, yet the immunofluorescence staining/histopathologic inspection (I/H) scores of the two groups differed little. In exp. III, the animals exhibited clinical and pathological signs after inoculation similar to those in exp. II. These results suggest that porcine CMAH KO with nullified NGNA expression are not immune to PEDV but that this KO may lessen the severity of the infection and delay its occurrence.","AbstractPredictive models for the spatial spread of infectious diseases has received much attention in recent years as tools for the management of infectious diseas outbreaks. Prominently, various versions of the so-called gravity model, borrowed from transportation theory, have been used. However, the original literature suggests that the model has some potential misspecifications inasmuch as it fails to capture higher-order interactions among population centers. The fields of economics, geography and network sciences holds alternative formulations for the spatial coupling within and among conurbations. These includes Stouffer\u2019s rank model, Fotheringham\u2019s competing destinations model and the radiation model of Simini et al. Since the spread of infectious disease reflects mobility through the filter of age-specific susceptibility and infectivity and since, moreover, disease may alter spatial behavior, it is essential to confront with epidemiological data on spread. To study their relative merit we, accordingly, fit variants of these models to the uniquely detailed dataset of prevaccination measles in the 954 cities and towns of England and Wales over the years 1944-65 and compare them using a consistent likelihood framework. We find that while the gravity model is a reasonable first approximation, both Stouffer\u2019s rank model, an extended version of the radiation model and the Fotheringham competing destinations model provide significantly better fits, Stouffer\u2019s model being the best. Through a new method of spatially disaggregated likelihoods we identify areas of relatively poorer fit, and show that it is indeed in densely-populated conurbations that higher order spatial interactions are most important. Our main conclusion is that it is premature to narrow in on a single class of models for predicting spatial spread of infectious disease. The supplemental materials contain all code for reproducing the results and applying the methods to other data sets.Author summaryThe ability to predict how infectious disease will spread is of great importance in the face of the numerous emergent and re-emergent pathogens that currently threatening human well-being. We identified a variety of alternative models that predict human mobility as as a function of population distribution across a landscape. These consider some models that account for pair-wise interactions between population centers, as well as some that allow for higher-order interactions. We trained the models using a uniquely rich spatiotemporal data set on pre-vaccination measles in England and wales (1944-65), which comprises more than a million records from 954 cities and towns. Likelihood rankings of the different models reveal strong evidence for higher-order interactions in the form of competition among cities as destinations for travelers and, thus, dilution of spatial transmission. The currently most commonly used so-called \u2018gravity\u2019 models were far from the best in capturing spatial disease dynamics.","AbstractA novel coronavirus has been identified as the causative agent of a massive outbreak of atypical pneumonia originating at Wuhan, Hubei province, China. Involved in the formation of the coronavirus replication complex, the viral main protease (Mpro, also called 3CLpro) represents an attractive target for therapy. We determined the crystal structure of the unliganded Mpro at 1.75 \u00c5 resolution and used this structure to guide optimization of a series of alpha-ketoamide inhibitors. The main goal of the optimization efforts was improvement of the pharmacokinetic properties of the compounds. We further describe 1.95- and 2.20-\u00c5 crystal structures of the complex between the enzyme and the most potent alpha-ketoamide optimized this way. These structures will form the basis for further development of these compounds to antiviral drugs.","The ongoing COVID-19 pandemic has reached more than 200 countries and territories worldwide. Given the large requirement of SARS-CoV-2 diagnosis and considering that RNA extraction kits are in short supply, we investigated whether two commercial RT-qPCR kits were compatible with direct SARS-CoV-2 detection from nasopharyngeal swab samples. We show that one of the tested kits is fully compatible with direct SARS-CoV-2 detection suggesting that omission of RNA extraction step should be considered in SARS-CoV-2 diagnosis.","ABSTRACTRespiratory tract infections (RTI) are responsible for over 4 million deaths per year worldwide with pathobiont carriage a required precursor to infection. Through a cross-sectional community-based nasal self-swabbing study we sought to determine carriage epidemiology for respiratory pathogens amongst bacteria (Streptococcus pneumoniae, Haemophilus influenza, Moraxella catarrhalis, Staphylococcus aureus, Pseudomonas aeruginosa and Neisseria meningitidis) and viruses (RSV, Influenza viruses A and B, Rhinovirus/Enterovirus, Coronavirus, Parainfluenza viruses 1-3 and Adenovirus (ADV)). Carriage of bacterial and viral species was shown to vary with participant age, recent RTI and the presence of other species. The spatial structure of microbial respiratory communities was less nested (more disordered) in the young (0-4 years) and those with recent RTI. Species frequency distributions were flatter than random expectation in young individuals (X2 = 20.42, p = 0.002), indicating spatial clumping of species consistent with facilitative relationships amongst them. Deviations from a neutral model of ecological niches were observed for samples collected in the summer and from older individuals (those aged 5-17, 18-64 and \u226565 years) but not in samples collected from winter, younger individuals (those aged 0-4 years), individuals with recent RTI and individuals without recent RTI, demonstrating the importance of both neutral and niche processes in respiratory community assembly. The application of epidemiological methods and ecological theory to sets of respiratory tract samples has yielded novel insights into the factors that drive microbial community composition, such as seasonality and age, as well as species patterns and interactions within the nose.","AbstractThere is currently an urgent unmet need to increase coronavirus disease 2019 (COVID-19) testing capability to effectively respond to the COVID-19 pandemic. However, the current shortage in RNA extraction reagents as well as limitations in qPCR protocols have resulted in bottlenecks in testing capacity. Herein, we describe a novel molecular diagnostic for COVID-19 based on Sanger sequencing. This assay uses the addition of a frame-shifted spike-in, a modified PCR master mix, and custom Sanger sequencing data analysis to detect and quantify SARS-CoV-2 RNA at a limit of detection comparable to existing qPCR-based assays, at 10-20 genome copy equivalents. Crucially, our assay was able to detect SARS-CoV-2 RNA from viral particles suspended in transport media that was directly added to the PCR master mix, suggesting that RNA extraction can be skipped entirely without any degradation of test performance. Since Sanger sequencing instruments are widespread in clinical laboratories and commonly have built-in liquid handling automation to support up to 3840 samples per instrument per day, the widespread adoption of qSanger COVID-19 diagnostics can unlock more than 1,000,000 tests per day in the US.","AbstractViruses have high mutation rates and generally exist as a mixture of variants in biological samples. Next-generation sequencing (NGS) approach has surpassed Sanger for generating long viral sequences, yet how variants affect NGS de novo assembly remains largely unexplored. Our results from &gt;15,000 simulated experiments showed that presence of variants can turn an assembly of one genome into tens to thousands of contigs. This \u201cvariant interference\u201d (VI) is highly consistent and reproducible by ten most used de novo assemblers, and occurs independent of genome length, read length, and GC content. The main driver of VI is pairwise identities between viral variants. These findings were further supported by in silico simulations, where selective removal of minor variant reads from clinical datasets allow the \u201crescue\u201d of full viral genomes from fragmented contigs. These results call for careful interpretation of contigs and contig numbers from de novo assembly in viral deep sequencing.","AbstractEpidemiological models are dominated by SEIR (Susceptible, Exposed, Infected and Removed) dynamical systems formulations and their elaborations. These formulations can be continuous or discrete, deterministic or stochastic, or spatially homogeneous or heterogeneous, the latter often embracing a network formulation. Here we review the continuous and discrete deterministic and discrete stochastic formulations of the SEIR dynamical systems models, and we outline how they can be easily and rapidly constructed using the Numerus Model Builder, a graphically-driven coding platform. We also demonstrate how to extend these models to a metapopulation setting using both the Numerus Model Builder network and geographical mapping tools.","ABSTRACTOne unexplored aspect of HIV-1 genetic architecture is how codon choice influences population diversity and evolvability. Here we compared the development of HIV-1 resistance to protease inhibitors (PIs) between wild-type (WT) virus and a synthetic virus (MAX) carrying a codon-pair re-engineered protease sequence including 38 (13%) synonymous mutations. WT and MAX viruses showed indistinguishable replication in MT-4 cells or PBMCs. Both viruses were subjected to serial passages in MT-4 cells with selective pressure from the PIs atazanavir (ATV) and darunavir (DRV). After 32 successive passages, both the WT and MAX viruses developed phenotypic resistance to PIs (IC50 14.6 \u00b1 5.3 and 21.2 \u00b1 9 nM for ATV, and 5. 9 \u00b1 1.0 and 9.3 \u00b1 1.9 for DRV, respectively). Ultra-deep sequence clonal analysis revealed that both viruses harbored previously described resistance mutations to ATV and DRV. However, the WT and MAX virus proteases showed different resistance variant repertoires, with the G16E and V77I substitutions observed only in WT, and the L33F, S37P, G48L, Q58E/K, and L89I substitutions detected only in MAX. Remarkably, G48L and L89I are rarely found in vivo in PI-treated patients. The MAX virus showed significantly higher nucleotide and amino acid diversity of the propagated viruses with and without PIs (P &lt; 0.0001), suggesting higher selective pressure for change in this recoded virus. Our results indicate that HIV-1 protease position in sequence space delineates the evolution of its mutant spectra. Nevertheless, the investigated synonymously recoded variant showed mutational robustness and evolvability similar to the WT virus.IMPORTANCELarge-scale synonymous recoding of virus genomes is a new tool for exploring various aspects of virus biology. Synonymous virus genome recoding can be used to investigate how a virus\u2019s position in sequence space defines its mutant spectrum, evolutionary trajectory, and pathogenesis. In this study, we evaluated how synonymous recoding of the human immunodeficiency virus type 1 (HIV-1) protease impacts the development of protease inhibitor (PI) resistance. HIV-1 protease is a main target of current antiretroviral therapies. Our present results demonstrate that the wild-type (WT) virus and the virus with the recoded protease exhibited different patterns of resistance mutations after PI treatment. Nevertheless, the developed PI resistance phenotype was indistinguishable between the recoded virus and the WT virus, suggesting that the synonymously recoded protease HIV-1 and the WT protease virus were equally robust and evolvable.","AbstractGenome-wide analysis of the mode of action of GSK983, a potent antiviral agent, led to the identification of dihydroorotate dehydrogenase (DHODH) as its target, along with the discovery that knockdown of genes in pyrimidine salvage pathways sensitized cells to GSK983. Because GSK983 is an ineffective antiviral in the presence of physiological uridine concentrations, we explored combining GSK983 with pyrimidine salvage inhibitors. We synthesized and evaluated analogs of cyclopentenyl uracil (CPU), an inhibitor of uridine salvage. We found that CPU was efficiently converted into its triphosphates in cells. When combined with GSK983, it led to large drops in cellular UTP and CTP pools. Consequently, CPU-GSK983 suppressed dengue virus replication in the presence of physiological concentrations of uridine. In addition, the CPU-GSK983 combination markedly enhanced the effect of RNA-dependent RNA polymerase (RdRp) inhibition on viral genome infection. Our findings highlight a new host-targeting strategy for potentiating the antiviral activities of RdRp inhibitors.","AbstractUsing the RNA-dependent RNA polymerase (RdRp) from poliovirus (PV) as our model system, we have shown that Lys-359 in motif-D functions as a general acid in the mechanism of nucleotidyl transfer. A K359H (KH) RdRp derivative is slow and faithful relative to wild-type enzyme. In the context of the virus, RdRp-coding sequence evolves, selecting for the following substitutions: I331F (IF, motif-C) and P356S (PS, motif-D). We have evaluated IF-KH, PS-KH, and IF-PS-KH viruses and enzymes. The speed and fidelity of each double mutant are equivalent. Each exhibits a unique recombination phenotype, with IF-KH being competent for copy-choice recombination and PS-KH being competent for forced-copy-choice recombination. Although the IF-PS-KH RdRp exhibits biochemical properties within twofold of wild type, the virus is impaired substantially for recombination in cells. We conclude that there are biochemical properties of the RdRp in addition to speed and fidelity that determine the mechanism and efficiency of recombination. The interwoven nature of speed, fidelity, the undefined property suggested here, and recombination makes it impossible to attribute a single property of the RdRp to fitness. However, the derivatives described here may permit elucidation of the importance of recombination on the fitness of the viral population in a background of constant polymerase speed and fidelity.SignificanceThe availability of a \u201cuniversal\u201d method to create attenuated viruses for use as vaccine strains would permit a rapid response to outbreaks of newly emerging viruses. Targeting RdRp fidelity has emerged as such a universal approach. However, because polymerase fidelity and speed are inextricably linked, the effort to attribute the attenuated phenotype to a single biochemical property of the RdRp may be futile. Here, we show that this circumstance is even more complex. We provide evidence for the existence of a biochemical parameter that combines with fidelity and speed to govern the mechanism and/or efficiency of recombination. We conclude that the field will be served best by continued emphasis on discovery of manipulatable functions of the RdRp instead of debating the importance of individual properties.","AbstractPersistent bacterial bronchitis is a leading cause of chronic wet cough in young children. This study aimed to characterise the respiratory bacterial microbiota of healthy children and to assess the impact of the changes associated with the development of persistent bacterial bronchitis.Blind, protected brushings were obtained from 20 healthy controls and 24 children with persistent bacterial bronchitis, with an additional directed sample obtained from persistent bacterial bronchitis patients. DNA was extracted, quantified using a 16S rRNA gene quantitative PCR assay prior to microbial community analysis by 16S rRNA gene sequencing.No significant difference in bacterial diversity or community composition (R2 = 0.01, P = 0.36) was observed between paired blind and non-blind brushes, showing that blind brushings are a valid means of accessing the airway microbiota. This has important implications for collecting lower respiratory samples from healthy children. A significant decrease in bacterial diversity (P &lt; 0.001) and change in community composition (R2 = 0.08, P = 0.004) was observed between controls and patients. Bacterial communities within patients with PBB were dominated by Proteobacteria, and indicator species analysis showed that Haemophilus and Neisseria were significantly associated with the patient group. In 15 (52.9%) cases the dominant organism by sequencing was not identified by standard routine clinical culture.The bacteria present in the lungs of patients with persistent bacterial bronchitis were less diverse in terms of richness and evenness. The results validate the clinical diagnosis, and suggest that more attention to bacterial communities in children with chronic cough may lead to more rapid recognition of this condition with earlier treatment and reduction in disease burden.","Background: COVID-19 has become a worldwide pandemic. It is caused by a novel coronavirus named SARS-CoV-2 with elusive origin. SARS-CoV-2 infects mammalian cells by binding to ACE2, a transmembrane protein. Therefore, the conservation of ACE2 and its expression pattern across mammalian species, which are yet to be comprehensively investigated, may provide valuable insights into tracing potential hosts of SARS-CoV-2. Methods: We analyzed gene conservation of ACE2 across mammals and collected more than 140 transcriptome datasets from human and common mammalian species, including presumed hosts of SARS-CoV-2 and other animals in close contact with humans. In order to enable comparisons across species and tissues, we used a unified pipeline to quantify and normalize ACE2 expression levels. Results: We first found high conservation of ACE2 genes among common mammals at both DNA and peptide levels, suggesting that a broad range of mammalian species can potentially be the hosts of SARS-CoV-2. Next, we showed that high level of ACE2 expression in certain human tissues is consistent with clinical symptoms of COVID-19 patients. Furthermore, we observed that ACE2 expressed in a species-specific manner in the mammals examined. Notably, high expression in skin and eyes in cat and dog suggested that these animals may play roles in transmitting SARS-CoV-2 to humans. Conclusions: Through building the first atlas of ACE2 expression in pets and livestock, we identified species and tissues susceptible to SARS-CoV-2 infection, yielding novel insights into the viral transmission.","AbstractAlternative splicing is a widespread phenomenon in metazoans by which single genes are able to produce multiple isoforms of the gene product. However, this has been poorly characterised in apicomplexans, a major phylum of some of the most important global parasites. Efforts have been hampered by atypical transcriptomic features, such as the high AT content of Plasmodium RNA, but also the limitations of short read sequencing in deciphering complex splicing events. In this study, we utilised the long read direct RNA sequencing platform developed by Oxford Nanopore Technologies (ONT) to survey the alternative splicing landscape of Toxoplasma gondii and Plasmodium falciparum. We find that while native RNA sequencing has a reduced throughput, it allows us to obtain full-length or near full-length transcripts with comparable quantification to Illumina sequencing. By comparing this data with available gene models, we find widespread alternative splicing, particular intron retention, in these parasites. Most of these transcripts contain premature stop codons, suggesting that in these parasites, alternative splicing represents a pathway to transcriptomic diversity, rather than expanding proteomic diversity. Moreover, alternative splicing rates are comparable between parasites, suggesting a shared splicing machinery, despite notable transcriptomic differences between the parasites. This work highlights a strategy in using long read sequencing to understand splicing events at the whole transcript level, and has implications in future interpretation of RNA-seq studies.","The newly emerged pandemic coronavirus, SARS-CoV-2, has posed a significant public health threat worldwide. However, the mode of virus transmission and tissue tropism is not well established yet. Recent findings of substantial liver damage in patients and ACE2+ cholangiocytes in healthy liver tissues prompted us to hypothesize that human liver ductal organoids could serve as a model to determine the susceptibility and mechanisms underlining the liver damage upon SARS-CoV-2 infection. By single-cell RNA sequencing, we found that long-term liver ductal organoid culture preserved the human specific ACE2+ population of cholangiocytes. Moreover, human liver ductal organoids were permissive to SARS-CoV-2 infection and support robust replication. Notably, virus infection impaired the barrier and bile acid transporting functions of cholangiocytes through dysregulation of genes involved in tight junction formation and bile acid transportation, which could explain the bile acid accumulation and consequent liver damage in patients. These results indicate that control of liver damage caused directly by viral infection should be valued in treating COVID-19 patients. Our findings also provide an application of human organoids in investigating the tropism and pathogenesis of SARS-CoV-2, which would facilitate novel drug discovery.","AbstractRepurposing existing drugs is a timely option to cope with COVID-19. We predicted therapeutic candidates that could reverse the gene expression of coronavirus-infected host cells. Thirteen expression signatures computed from various experimental conditions and preclinical models could be reversed by those compounds known to be effective against SARS- or MERS-CoV, as well as the drug candidates recently shown to be effective against SARS-CoV-2. We selected ten novel candidates to further evaluate their in vitro efficacy against SARS-CoV-2 infection. Four compounds bortezomib, dactolisib, alvocidib and methotrexate inhibited the formation of virus infection-induced cytopathic effect in Vero E6 cells at 1 \u00b5M, yet such a concentration seems toxic to the cells as well. While the evaluation in other permissive cells and the prediction of toxicity are needed to optimize and minimize their antiviral activity and cytotoxicity, respectively, this computational approach has the potential to rapidly and rationally identify drug candidates against COVID-19.","AbstractCytokine storms are drivers of pathology and mortality in myriad viral infections affecting the human population. In SARS-CoV-2-infected patients, the strength of the cytokine storm has been associated with increased risk of acute respiratory distress syndrome, myocardial damage, and death. However, the therapeutic value of attenuating the cytokine storm in COVID-19 remains to be defined. Here, we report results obtained using a novel mouse model of lethal sterile anti-viral immune responses. Using a mouse model of Down syndrome (DS) with a segmental duplication of a genomic region encoding four of the six interferon receptor genes (Ifnrs), we demonstrate that these animals overexpress Ifnrs and are hypersensitive to IFN stimulation. When challenged with viral mimetics that activate Toll-like receptor signaling and IFN anti-viral responses, these animals overproduce key cytokines, show exacerbated liver pathology, rapidly lose weight, and die. Importantly, the lethal immune hypersensitivity, accompanying cytokine storm, and liver hyperinflammation are blocked by treatment with a JAK1-specific inhibitor. Therefore, these results point to JAK1 inhibition as a potential strategy for attenuating the cytokine storm and consequent organ failure during overdrive immune responses. Additionally, these results indicate that people with DS, who carry an extra copy of the IFNR gene cluster encoded on chromosome 21, should be considered at high risk during the COVID-19 pandemic.One Sentence SummaryInhibition of the JAK1 kinase prevents pathology and mortality caused by a rampant innate immune response in mice.","There is interest in the use of chloroquine/hydroxychloroquine (CQ/HCQ) and azithromycin (AZT) in COVID-19 therapy. Employing cystic fibrosis respiratory epithelial cells, here we show that drugs AZT and ciprofloxacin (CPX) act as acidotropic lipophilic weak bases and confer in vitro effects on intracellular organelles similar to the effects of CQ. These seemingly disparate FDA-approved antimicrobials display a common property of modulating pH of endosomes and trans-Golgi network. We believe this may in part help understand the potentially beneficial effects of CQ/HCQ and AZT in COVID-19, and that the present considerations of HCQ and AZT for clinical trials should be extended to CPX.","AbstractDuring Drosophila oogenesis, the localization and translational regulation of maternal transcripts relies on RNA-binding proteins (RBPs). Many of these RBPs localize several mRNAs and may have additional direct interaction partners to regulate their functions. Using immunoprecipitation from whole Drosophila ovaries coupled to mass spectrometry, we examined protein-protein associations of 6 GFP-tagged RBPs expressed at physiological levels. Analysis of the interaction network and further validation in human cells allowed us to identify 26 previously unknown associations, besides recovering several well characterized interactions. We identified interactions between RBPs and several splicing factors, providing links between nuclear and cytoplasmic events of mRNA regulation. Additionally, components of the translational and RNA decay machineries were selectively co-purified with some baits, suggesting a mechanism for how RBPs may regulate maternal transcripts. Given the evolutionary conservation of the studied RBPs, the interaction network presented here provides the foundation for future functional and structural studies of mRNA localization across metazoans.","For successful infection, viruses must recognize their respective host cells. A common mechanism of host recognition by viruses is to utilize a portion of the host cell as a receptor. Bacteriophage Sf6, which infects Shigella flexneri, uses lipopolysaccharide as a primary receptor and then requires interaction with a secondary receptor, a role that can be fulfilled by either outer membrane proteins (Omp) A or C. Our previous work showed that specific residues in the loops of OmpA mediate Sf6 infection. To better understand Sf6 interactions with OmpA loop variants, we determined the kinetics of these interactions through the use of biolayer interferometry, an optical biosensing technique that yields data similar to surface plasmon resonance. Here, we successfully tethered whole Sf6 virions, determined the binding constant of Sf6 to OmpA to be 36 nM. Additionally, we showed that Sf6 bound to five variant OmpAs and the resulting kinetic parameters varied only slightly. Based on these data, we propose a model in which Sf6: Omp receptor recognition is not solely based on kinetics, but likely also on the ability of an Omp to induce a conformational change that results in productive infection.","ABSTRACTMicroRNA precursors (pre-miRNAs) are short hairpin RNAs that are rapidly processed into mature microRNAs (miRNAs) in the cytoplasm. Due to their low abundance in cells, sequencing-based studies of pre-miRNAs have been limited. We successfully enriched for and deep sequenced pre-miRNAs in human cells by capturing these RNAs during their interaction with Argonaute (AGO) proteins. Using this approach, we detected &gt; 350 pre-miRNAs in human cells and &gt; 250 pre-miRNAs in a reanalysis of a similar study in mouse cells. We uncovered widespread trimming and non-templated additions to the 3\u2019 ends of pre- and mature miRNAs. Additionally, we created an index for microRNA precursor processing efficiency. This analysis revealed a subset of pre-miRNAs that produce low levels of mature miRNAs despite abundant precursors, including an annotated miRNA in the 5\u2019 UTR of the DiGeorge syndrome critical region 8 mRNA transcript. This led us to search for AGO-associated stem-loops originating from other mRNA species, which identified hundreds of putative pre-miRNAs derived from human and mouse mRNAs. In summary, we provide a wealth of information on mammalian pre-miRNAs, and identify novel microRNA and microRNA-like elements localized in mRNAs.","The emergence of SARS-CoV-2 has resulted in more than 200,000 infections and nearly 9,000 deaths globally so far. This novel virus is thought to have originated from an animal reservoir, and acquired the ability to infect human cells using the SARS-CoV cell receptor hACE2. In the wake of a global pandemic it is essential to improve our understanding of the evolutionary dynamics surrounding the origin and spread of a novel infectious disease. One way theory predicts selection pressures should shape viral evolution is to enhance binding with host cells. We first assessed evolutionary dynamics in select betacoronavirus spike protein genes to predict where these genomic regions are under directional or purifying selection between divergent viral lineages at various scales of relatedness. With this analysis, we determine a region inside the receptor-binding domain with putative sites under positive selection interspersed among highly conserved sites, which are implicated in structural stability of the viral spike protein and its union with human receptor hACE2. Next, to gain further insights into factors associated with coronaviruses recognition of the human host receptor, we performed modeling studies of five different coronaviruses and their potential binding to hACE2. Modeling results indicate that interfering with the salt bridges at hot spot 353 could be an effective strategy for inhibiting binding, and hence for the prevention of coronavirus infections. We also propose that a glycine residue at the receptor binding domain of the spike glycoprotein can have a critical role in permitting bat variants of the coronaviruses to infect human cells.","AbstractWhen people are forced to be isolated from one another, do they crave social interactions in the same way a hungry person craves food? To address this question, we used functional magnetic resonance imaging (fMRI) to measure neural responses in participants (n=40) evoked by food and social cues after ten hours of mandated fasting or total social isolation. After isolation, people felt lonely and craved social interaction. Midbrain regions showed increased activation to food cues after fasting and to social cues after isolation; these responses were correlated with self-reported craving. Neural patterns in response to food cues when participants were hungry generalized to social cues after isolation. Our results support the intuitive idea that acute isolation causes social craving, similar to hunger.","Influenza A virus (IAV) enters cells by binding to sialic acid on the cell surface. To accomplish this while avoiding immobilization by sialic acid in host mucus, viruses rely on a balance between the receptor-binding protein hemagglutinin (HA) and the receptor-cleaving protein neuraminidase (NA). Although genetic aspects of this balance are well-characterized, little is known about how the spatial organization of these proteins in the viral envelope may contribute. Using site-specific fluorescent labeling and super-resolution microscopy, we show that HA and NA are asymmetrically distributed on the surface of filamentous viruses, creating an organization of binding and cleaving activities that causes viruses to step consistently away from their NA-rich pole. This Brownian ratchet-like diffusion produces persistent directional mobility that resolves the conflicting needs of the virus to both penetrate mucus and stably attach to the underlying cells, and could contribute to the prevalence of the filamentous phenotype in clinical isolates of IAV.","ABSTRACTRationaleSeveral studies have estimated basic production number of novel coronavirus pneumonia (NCP). However, the time-varying transmission dynamics of NCP during the outbreak remain unclear.ObjectivesWe aimed to estimate the basic and time-varying transmission dynamics of NCP across China, and compared them with SARS.MethodsData on NCP cases by February 7, 2020 were collected from epidemiological investigations or official websites. Data on severe acute respiratory syndrome (SARS) cases in Guangdong Province, Beijing and Hong Kong during 2002-2003 were also obtained. We estimated the doubling time, basic reproduction number (R0) and time-varying reproduction number (Rt) of NCP and SARS.Measurements and main resultsAs of February 7, 2020, 34,598 NCP cases were identified in China, and daily confirmed cases decreased after February 4. The doubling time of NCP nationwide was 2.4 days which was shorter than that of SARS in Guangdong (14.3 days), Hong Kong (5.7 days) and Beijing (12.4 days). The R0 of NCP cases nationwide and in Wuhan were 4.5 and 4.4 respectively, which were higher than R0 of SARS in Guangdong (R0=2.3), Hongkong (R0=2.3), and Beijing (R0=2.6). The Rt for NCP continuously decreased especially after January 16 nationwide and in Wuhan. The R0 for secondary NCP cases in Guangdong was 0.6, and the Rt values were less than 1 during the epidemic.ConclusionsNCP may have a higher transmissibility than SARS, and the efforts of containing the outbreak are effective. However, the efforts are needed to persist in for reducing time-varying reproduction number below one.At a Glance CommentaryScientific Knowledge on the SubjectSince December 29, 2019, pneumonia infection with 2019-nCoV, now named as Novel Coronavirus Pneumonia (NCP), occurred in Wuhan, Hubei Province, China. The disease has rapidly spread from Wuhan to other areas. As a novel virus, the time-varying transmission dynamics of NCP remain unclear, and it is also important to compare it with SARS.What This Study Adds to the FieldWe compared the transmission dynamics of NCP with SARS, and found that NCP has a higher transmissibility than SARS. Time-varying production number indicates that rigorous control measures taken by governments are effective across China, and persistent efforts are needed to be taken for reducing instantaneous reproduction number below one.","As the SARS-CoV-2 pandemic is rapidly progressing, the need for the development of an effective vaccine is critical. A promising approach for vaccine development is to generate, through codon pair deoptimization, an attenuated virus. This approach carries the advantage that it only requires limited knowledge specific to the virus in question, other than its genome sequence. Therefore, it is well suited for emerging viruses for which we may not have extensive data. We performed comprehensive in silico analyses of several features of SARS-CoV-2 genomic sequence (e.g., codon usage, codon pair usage, dinucleotide/junction dinucleotide usage, RNA structure around the frameshift region) in comparison with other members of the coronaviridae family of viruses, the overall human genome, and the transcriptome of specific human tissues such as lung, which are primarily targeted by the virus. Our analysis identified the spike (S) and nucleocapsid (N) proteins as promising targets for deoptimization and suggests a roadmap for SARS-CoV-2 vaccine development, which can be generalizable to other viruses.","ABSTRACTThe ongoing coronavirus disease (COVID-19) pandemic caused by Severe Acute Respiratory Syndrome Coronavirus-2 (SARS-CoV-2) is a global public health concern due to relatively easy person-to-person transmission and the current lack of effective antiviral therapy. However, the exact molecular mechanisms of SARS-CoV-2 pathogenesis remain largely unknown. We exploited an integrated proteomics approach to systematically investigate intra-viral and virus-host interactomes for the identification of unrealized SARS-CoV-2 host targets and participation of cellular proteins in the response to viral infection using peripheral blood mononuclear cells (PBMCs) isolated from COVID-19 patients. Using this approach, we elucidated 251 host proteins targeted by SARS-CoV-2 and more than 200 host proteins that are significantly perturbed in COVID-19 derived PBMCs. From the interactome, we further identified that non-structural protein nsp9 and nsp10 interact with NKRF, a NF-\u041aB repressor, and may precipitate the strong IL-8/IL-6 mediated chemotaxis of neutrophils and overexuberant host inflammatory response observed in COVID-19 patients. Our integrative study not only presents a systematic examination of SARS-CoV-2-induced perturbation of host targets and cellular networks to reflect disease etiology, but also reveals insights into the mechanisms by which SARS-CoV-2 triggers cytokine storms and represents a powerful resource in the quest for therapeutic intervention.","ABSTRACTPositive-sense single-stranded RNA viruses form the largest and most diverse group of eukaryote-infecting viruses. Their genomes comprise one or more segments of coding-sense RNA that function directly as messenger RNAs upon release into the cytoplasm of infected cells. Positive-sense RNA viruses are generally accepted to encode proteins solely on the positive strand. However, we previously identified a surprisingly long (~1000 codons) open reading frame (ORF) on the negative strand of some members of the family Narnaviridae which, together with RNA bacteriophages of the family Leviviridae, form a sister group to all other positive-sense RNA viruses. Here, we completed the genomes of three mosquito-associated narnaviruses, all of which have the long reverse-frame ORF. We systematically identified narnaviral sequences in public data sets from a wide range of sources, including arthropod, fungi and plant transcriptomic datasets. Long reverse-frame ORFs are widespread in one clade of narnaviruses, where they frequently occupy &gt;95% of the genome. The reverse-frame ORFs correspond to a specific avoidance of CUA, UUA and UCA codons (i.e. stop codon reverse complements) in the forward-frame RNA-dependent RNA polymerase ORF. However, absence of these codons cannot be explained by other factors such as inability to decode these codons or GC3 bias. Together with other analyses, we provide the strongest evidence yet of coding capacity on the negative strand of a positive-sense RNA virus. As these ORFs comprise some of the longest known overlapping genes, their study may be of broad relevance to understanding overlapping gene evolution and de novo origin of genes.","AbstractMotivationZoonosis, the natural transmission of infections from animals to humans, is a far-reaching global problem. The recent outbreaks of Zika virus, Ebola virus and Corona virus are examples of viral zoonosis, which occur more frequently due to globalization. In the case of a virus outbreak, it is helpful to know which host organism was the original carrier of the virus. Once the reservoir or intermediate host is known, it can be isolated to prevent further spreading of the viral infection. Recent approaches aim to predict a viral host based on the viral genome, often in combination with the potential host genome and arbitrarily selected features. These methods have a clear limitation in either the number of different hosts they can predict or the accuracy of their prediction.ResultsHere, we present a fast and accurate deep learning approach for viral host prediction, which is based on the viral genome sequence only. To ensure a high prediction accuracy, we developed an effective selection approach for the training data to avoid biases due to a highly unbalanced number of known sequences per virus-host combinations. We tested our deep neural network on three different virus species (influenza A, rabies lyssavirus, rotavirus A). We reached for each virus species an AUG between 0.93 and 0.98, outperforming previous approaches and allowing highly accurate predictions while only using fractions (100-400 bp) of the viral genome sequences. We show that deep neural networks are suitable to predict the host of a virus, even with a limited amount of sequences and highly unbalanced available data. The deep neural networks trained for this approach build the core of the virus-host predicting tool VIDHOP (Virus Deep learning HOst Prediction).AvailabilityThe trained models for the prediction of the host for the viruses influenza A, rabies lyssavirus, rotavirus A are implemented in the tool VIDHOP. This tool is freely available under https://github.com/flomock/vidhop.Supplementary informationSupplementary data are available at DOI 10.17605/OSF.IO/UXT7N","Advanced biomaterials provide versatile ways to spatially and temporally control immune cell activity, potentially enhancing their therapeutic potency and safety. Precise cell modulation demands multi-modal display of functional proteins with controlled densities on biomaterials. Here, we develop an artificial immune cell engager (AICE) platform \u2013 biodegradable particles onto which multiple proteins are densely loaded with ratiometric control via short nucleic acid tethers. We demonstrate the impact of AICE with varying ratios of anti-CD3 and anti-CD28 antibodies onex vivoexpansion of human primary T cells. We also show that AICE can be used to control the activity of engineered T cellsin vivo. AICE injected intratumorally can provide a local priming signal for systemically administered AND-gate chimeric antigen receptor T cells, driving local tumor clearance while sparing uninjected tumors that model potentially cross-reactive healthy tissues. This modularly functionalized biomaterial thus provides a flexible platform to achieve sophisticated control over cell-based immunotherapies.","AbstractZika virus is a single-stranded, positive-sense RNA virus of the family Flaviviridae, which has recently undergone a rapid expansion among humans in the Western Hemisphere. Here, we report a high-resolution map of ribosomal occupancy of the Zika virus genome during infection of mammalian and insect cells, obtained by ribosome profiling. In contrast to some other flaviviruses such as West Nile, we find no evidence for substantial frameshift-induced ribosomal drop-off during translation of the viral polyprotein, indicating that Zika virus must use alternative mechanisms to downregulate levels of catalytically active viral polymerase. We also show that high levels of ribosome-protected fragments map in-frame to two previously overlooked upstream open reading frames (uORFs) initiating at CUG and UUG codons, with likely consequences for the efficiency of polyprotein expression. Curiously, in African isolates of Zika virus, the two uORFs are fused in-frame into a single uORF. A parallel RNA-Seq analysis reveals the 5\u2032 end position of the subgenomic flavivirus RNA in mammalian and insect cells. Together, these provide the first analysis of flavivirus gene expression by ribosome profiling.Author SummaryRecent Zika virus outbreaks have been associated with congenital diseases and neurological complications. An enhanced understanding of the molecular biology of this pathogen may contribute towards the development of improved treatment and control methods. We present a single-codon resolution analysis of Zika virus translation in mammalian and mosquito cells using ribosome profiling. The analysis revealed two hitherto uncharacterized uORFs in the 5\u2032 leader of Zika virus Brazilian isolate PE243, both of which are occupied by ribosomes during infection. In contrast, these two uORFs are fused into a single uORF in African isolates. This observation provides a new avenue for further investigations into potential factors involved in the emergence of Zika virus from a rarely detected pathogen into a major epidemic.","AbstractInfluenza viruses and rhinoviruses are responsible for a large number of acute respiratory viral infections in human populations and are detected as co-pathogens within hosts. Clinical and epidemiological studies suggest that co-infection by rhinovirus and influenza virus may reduce disease severity and that they may also interfere with each other\u2019s spread within a host population. To determine how co-infection by these two unrelated respiratory viruses affects pathogenesis, we established a mouse model using a minor serogroup rhinovirus (RV1B) and mouse-adapted influenza A virus (PR8). Infection of mice with RV1B two days before PR8 reduced pathogenesis of mild to moderate, but not severe PR8 infections. Disease attenuation was associated with an early inflammatory response in the lungs and enhanced clearance of PR8. However, co-infection by RV1B did not reduce PR8 viral loads early in infection or inhibit replication of PR8 within respiratory epithelia or in vitro. Inflammation in co-infected mice remained focal, in comparison to diffuse inflammation and damage in the lungs of mice infected by PR8. These findings suggest that RV1B stimulates an early immune response that clears PR8 while limiting excessive pulmonary inflammation. The timing of RV1B co-infection was a critical determinant of protection, suggesting that sufficient time is needed to induce this response. Finally, disease attenuation was not unique to RV1B: co-infection by a murine coronavirus two days before PR8 also reduced disease severity. This model will be critical for understanding the mechanisms responsible for attenuation of influenza disease during co-infection by unrelated respiratory viruses.","AbstractIn recent years, many studies have drawn attention to the important role of collective awareness and human behaviour during epidemic outbreaks. A number of modelling efforts have investigated the interaction between the disease transmission dynamics and human behaviour change mediated by news coverage and by information spreading in the population. Yet, given the scarcity of data on public awareness during an epidemic, few studies have relied on empirical data. Here, we use fine-grained, geo-referenced data from three online sources \u2013 Wikipedia, the GDELT Project and the Internet Archive \u2013 to quantify population-scale information seeking about the 2016 Zika virus epidemic in the U.S., explicitly linking such behavioural signal to epidemiological data. Geolocalized Wikipedia pageview data reveal that visiting patterns of Zika-related pages in Wikipedia were highly synchronized across the United States and largely explained by exposure to national television broadcast. Contrary to the assumption of some theoretical models, news volume and Wikipedia visiting patterns were not significantly correlated with the magnitude or the extent of the epidemic. Attention to Zika, in terms of Zika-related Wikipedia pageviews, was high at the beginning of the outbreak, when public health agencies raised an international alert and triggered media coverage, but subsequently exhibited an activity profile that suggests nonlinear dependencies and memory effects in the relation between information seeking, media pressure, and disease dynamics. This calls for a new and more general modelling framework to describe the interaction between media exposure, public awareness and disease dynamics during epidemic outbreaks.","RNA degradation by RNase L during 2-5A-mediated decay (2-5AMD) is a conserved mammalian stress response to viral and endogenous double-stranded RNA (dsRNA). 2-5AMD onsets rapidly and facilitates a switch of protein synthesis from homeostasis to production of interferons (IFNs). To understand the mechanism of this protein synthesis reprogramming, we examined 2-5AMD in human cells. 2-5AMD triggers polysome collapse characteristic of a translation initiation defect, but translation initiation complexes and ribosomes purified from the translation-arrested cells remain functional. Using spike-in RNA-seq we found that basal messenger RNAs (mRNAs) rapidly decay, while mRNAs encoding IFNs and IFN-stimulated genes evade 2-5AMD and accumulate. The IFN evasion results from the combined effect of better mRNA stability and positive feedback amplification in the IFN response. Therefore, 2-5AMD and transcription act in concert to revamp the cellular mRNA composition. The resulting preferential accumulation of innate immune mRNAs establishes 'prioritized' synthesis of defense proteins.","AbstractThe G-quadruplex structure forming motifs have recently emerged as a novel therapeutic drug target in various human pathogens. Herein, we report three highly conserved G-quadruplex motifs (SE-PGQ-1, 2, and3) in genome of all the 412 strains of Salmonella enterica. Bioinformatics analysis inferred the presence of SE-PGQ-1 in the regulatory region of mgtA, presence of SE-PGQ-2 in the open reading frame of entA and presence of SE-PGQ-3 in the promoter region of malE and malK genes. The products of mgtA and entA are involved in transport and homeostasis of Mg2+ and Fe3+ ion and thereby required for bacterial survival in the presence of reactive nitrogen/oxygen species produced by the host macrophages, whereas, malK and malE genes are involved in transport of maltose sugar, that is one of the major carbon source in the gastrointestinal tract of human. The formation of stable intramolecular G-quadruplex structures by SE-PGQs was confirmed by employing CD, EMSA and NMR spectroscopy. Cellular studies revealed the inhibitory effect of 9-amino acridine on Salmonella enterica growth. Next, CD melting analysis demonstrated the stabilizing effect of 9-amino acridine on SE-PGQs. Further, polymerase inhibition and RT-qPCR assays emphasize the biological relevance of predicted G-quadruplex in the expression of PGQ possessing genes and demonstrate the G-quadruplexes as a potential drug target for the devolping novel therapeutics for combating Salmonella enterica infection.Author SummarySince last several decades\u2019 scientific community has witnessed a rapid increase in number of such human pathogenic bacterial species that acquired resistant to multiple antibacterial agents. Currently, emergence of multidrug-resistant strains remain a major public health concern for clinical investigators that rings a global alarm to search for novel and highly conserved drug targets. Recently, G-quadruplex structure forming nucleic acid sequences were endorsed as highly conserved Drug target for preventing infection of several human pathogens including viral and protozoan species. Therefore, here we explored the presence G-quadruplex forming motif in genome of Salmonella enterica bacteria that causes food poisoning, and enteric fever in human. The formation of intra molecular G-quadruplex structure in four genes (mgtA, entA, malE and malK) was confirmed by NMR, CD and EMSA. The 9-amino acridine, a known G-quadruplex binder has been shown to stabilize the predicted G-quadruplex motif and decreases the expressioin of G-quadruplex hourbouring genes using RT-PCR and cellular toxicity assay. This study concludes the presence of G-quadruplex motifs in essential genes of Salmonella enterica genome as a novel and conserved drug target and 9-amino acridine as candidate small molecule for preventing the infection of Salmonella enterica using a G4 mediated inhibition mechanism.","AbstractMyxovirus resistance protein B (MxB) is an interferon-inducible restriction factor of HIV-1 that blocks nuclear import of the viral genome. Evidence suggests that MxB recognizes higher-order interfaces of the HIV capsid lattice, but the mechanistic details of this interaction are not known. Previous studies have mapped the restriction activity of MxB to its N-terminus encompassing a triple arginine motif 11RRR13. Here we demonstrate a direct and specific interaction between the MxB N-terminus and helical assemblies of HIV-1 capsid protein (CA) using highly purified recombinant proteins. We performed thorough mutagenesis to establish the detailed molecular requirements for the CA interaction with MxB. The results map MxB binding to the interface of three CA hexamers, specifically interactions between positively charged MxB N-terminal residues and negatively charged CA residues. Our crystal structures show that the CA mutations affecting MxB interaction and restriction do not alter the conformation of capsid assembly. In addition, 30 microsecond long all-atom molecular dynamics (MD) simulations of the complex between the MxB N-terminus and the HIV CA tri-hexamer interface show persistent MxB binding and identify a MxB-binding pocket surrounded by three CA hexamers. These results establish the molecular details of the binding of a lattice-sensing host factor onto HIV capsid, and provide insight into how MxB recognizes HIV capsid for the restriction of HIV-1 infection.Author summaryThe human antiviral protein MxB is a restriction factor that fights HIV infection. Previous experiments have demonstrated that MxB targets the HIV capsid, a protein shell that protects the viral genome. To make the conical shaped capsid, HIV CA proteins are organized into a lattice composed of hexamer and pentamer building blocks, providing many interfaces for host proteins to recognize. Through extensive biochemical and biophysical studies and molecular dynamics simulations, we show that MxB is targeting the HIV capsid by recognizing the region created at the intersection of three CA hexamers. We are further able to map this interaction to a few CA residues, located in a negatively-charged well at the interface between the three CA hexamers. This work provides detailed residue-level mapping of the targeted capsid interface and how MxB interacts. This information could inspire the development of capsid-targeting therapies for HIV.","ABSTRACTSince its first recorded appearance in December 2019, a novel coronavirus (SARS-CoV-2) causing the disease COVID-19 has resulted in more than 2,000,000 infections and 128,000 deaths. Currently there is no proven treatment for COVID-19 and there is an urgent need for the development of vaccines and therapeutics. Coronavirus spike glycoproteins play a critical role in coronavirus entry into the host cells, as they provide host cell recognition and membrane fusion between virus and host cell. Thus, they emerged as popular and promising drug targets. Crystal structures of spike protein in its closed and open states were resolved very recently in March 2020. These structures comprise 77% of the sequence and provide almost the complete protein structure. Based on down and up positions of receptor binding domain (RBD), spike protein can be in a receptor inaccessible closed or receptor accessible open state, respectively. Starting from closed and open state crystal structures, and also 16 intermediate conformations, an extensive set of all-atom molecular dynamics (MD) simulations in the presence of explicit water and ions were performed. Simulations show that in its down position, RBD has significantly lower mobility compared to its up position; probably caused by the 6 interdomain salt bridges of RBD in down position compared to 3 in up position. Free energy landscapes based on MD simulations revealed a semi-open state located between closed and open states. Minimum energy pathway between down and up positions comprised a gradual salt bridge switching mechanism. Furthermore, although significantly lower than open state, ACE2 binding surface of RBD contained a partial solvent accessibility in its closed state.","ABSTRARCTThe fast accumulation of viral metagenomic data has contributed significantly to new RNA virus discovery. However, the short read size, complex composition, and large data size can all make taxonomic analysis difficult. In particular, commonly used alignment-based methods are not ideal choices for detecting new viral species. In this work, we present a novel hierarchical classification model named CHEER, which can conduct read-level taxonomic classification from order to genus for new species. By combining k-mer embedding-based encoding, hierarchically organized CNNs, and carefully trained rejection layer, CHEER is able to assign correct taxonomic labels for reads from new species. We tested CHEER on both simulated and real sequencing data. The results show that CHEER can achieve higher accuracy than popular alignment-based and alignment-free taxonomic assignment tools. The source code, scripts, and pre-trained parameters for CHEER are available via GitHub: https://github.com/KennthShang/CHEER.","AbstractEstimating temporal changes in a target population from phylogenetic or count data is an important problem in ecology and epidemiology. Reliable estimates can provide key insights into the climatic and biological drivers influencing the diversity or structure of that population and evidence hypotheses concerning its future growth or decline. In infectious disease applications, the individuals infected across an epidemic form the target population. The renewal model estimates the effective reproduction number, R, of the epidemic from counts of its observed cases. The skyline model infers the effective population size, N, underlying a phylogeny of sequences sampled from that epidemic. Practically, R measures ongoing epidemic growth while N informs on historical caseload. While both models solve distinct problems, the reliability of their estimates depends on p-dimensional piecewise-constant functions. If p is misspecified, the model might underfit significant changes or overfit noise and promote a spurious understanding of the epidemic, which might misguide intervention policies or misinform forecasts. Surprisingly, no transparent yet principled approach for optimising p exists. Usually, p is heuristically set, or obscurely controlled via complex algorithms. We present a computable and interpretable p-selection method based on the minimum description length (MDL) formalism of information theory. Unlike many standard model selection techniques, MDL accounts for the additional statistical complexity induced by how parameters interact. As a result, our method optimises p so that R and N estimates properly adapt to the available data. It also outperforms comparable Akaike and Bayesian information criteria on several classification problems. Our approach requires some knowledge of the parameter space and exposes the similarities between renewal and skyline models.","AbstractMiddle East respiratory syndrome coronavirus (MERS-CoV) is a zoonotic virus from camels causing significant mortality and morbidity in humans in the Arabian Peninsula. The epidemiology of the virus remains poorly understood, and while case-based and seroepidemiological studies have been employed extensively throughout the epidemic, viral sequence data have not been utilised to their full potential. Here we use existing MERS-CoV sequence data to explore its phylodynamics in two of its known major hosts, humans and camels. We employ structured coalescent models to show that long-term MERS-CoV evolution occurs exclusively in camels, whereas humans act as a transient, and ultimately terminal host. By analysing the distribution of human outbreak cluster sizes and zoonotic introduction times we show that human outbreaks in the Arabian peninsula are driven by seasonally varying zoonotic transfer of viruses from camels. Without heretofore unseen evolution of host tropism, MERS-CoV is unlikely to become endemic in humans.","ABSTRACTRotavirus (RV) replicates in round-shaped cytoplasmic viral factories although how they assemble remains unknown.During RV infection, NSP5 undergoes hyperphosphorylation, which is primed by the phosphorylation of a single serine residue. The role of this post-translational modification in the formation of viroplasms and its impact on the virus replication remains obscure. Here we investigated the role of NSP5 during RV infection by taking advantage of a modified fully tractable reverse genetics system. An NSP5 trans-complementing cell line was used to generate and characterise several recombinant rotaviruses (rRVs) with mutations in NSP5. We demonstrate that a rRV lacking NSP5, was completely unable to assemble viroplasms and to replicate, confirming its pivotal role in rotavirus replication.A number of mutants with impaired NSP5 phosphorylation were generated to further interrogate the function of this post-translational modification in the assembly of replication-competent viroplasms. We showed that the rRV mutant strains exhibit impaired viral replication and the ability to assemble round-shaped viroplasms in MA104 cells. Furthermore, we have investigated the mechanism of NSP5 hyper-phosphorylation during RV infection using NSP5 phosphorylation-negative rRV strains, as well as MA104-derived stable transfectant cell lines expressing either wt NSP5 or selected NSP5 deletion mutants. Our results indicate that NSP5 hyper-phosphorylation is a crucial step for the assembly of round-shaped viroplasms, highlighting the key role of the C-terminal tail of NSP5 in the formation of replication-competent viral factories. Such a complex NSP5 phosphorylation cascade may serve as a paradigm for the assembly of functional viral factories in other RNA viruses.IMPORTANCERotavirus (RV) double-stranded RNA genome is replicated and packaged into virus progeny in cytoplasmic structures termed viroplasms. The non-structural protein NSP5, which undergoes a complex hyperphosphorylation process during RV infection, is required for the formation of these virus-induced organelles. However, its roles in viroplasm formation and RV replication have never been directly assessed due to the lack of a fully tractable reverse genetics (RG) system for rotaviruses. Here we show a novel application of a recently developed RG system by establishing a stable trans-complementing NSP5-producing cell line required to rescue rotaviruses with mutations in NSP5. This approach allowed us to provide the first direct evidence of the pivotal role of this protein during RV replication. Furthermore, using recombinant RV mutants we shed light on the molecular mechanism of NSP5 hyperphosphorylation during infection and its involvement in the assembly and maturation of replication-competent viroplasms.","AbstractRNA knockdown in vivo carries significant potential for disease modelings and therapies. Despite the emerging approaches of CRISPR/Cas9-mediated permanent knock out of targeted genes, strategies targeting RNA for disruption are advantageous in the treatment of acquired metabolic disorders when permanent modification of the genome DNA is not appropriate, and RNA virus infection diseases when pathogenic DNA is not available (such as SARS-Cov-2 and MERS infections). Recently, Cas13d, a family of RNA-targeting CRISPR effectors, has been shown to accomplish robust down-regulation of cellular RNAs in mammalian cells in vitro. Among the various Cas13d subtypes, CasRx (RfxCas13d) showed the most potent RNA knockdown efficiency in HEK293T cells. However, the RNA-targeting activity of Cas13d still needs to be verified in vivo. In this study, the CasRx system was demonstrated to efficiently and functionally knock down genes related to metabolism functions, including Pten, Pcsk9 and lncLstr, in mouse hepatocytes. CasRx-mediated simultaneous knockdown of multiple genes was also achieved by sgRNA arrays, providing a useful strategy to modulate complex metabolism networks. Moreover, the AAV (adeno-associated virus)-mediated delivery of CasRx and Pcsk9 sgRNAs into mouse liver successfully decreased serum PCSK9, resulting in significant reduction of serum cholesterol levels. Importantly, CasRx-mediated knockdown of Pcsk9 is reversible and Pcsk9 could be repeatedly down-regulated, providing an effective strategy to reversibly modulate metabolic genes. The present work supplies a successful proof-of-concept trial that suggests efficient and regulatory knockdown of target metabolic genes for a designed metabolism modulation in the liver.","ABSTRACTDue to its economic importance to in poultry industry, the biology and pathogenesis of infectious bronchitis virus (IBV) have been investigated extensively. However, the molecular mechanisms involved in IBV entry are not well characterized. In this study, systematic approaches were used to dissect IBV entry process in various susceptible cells. First, we observed that lipid rafts were involved in IBV attachment. Second, low pH in intracyplasmic vesicles was required for virus entry. By using the specific clathrin mediated endocytosis (CME) inhibitor or knock down of clathrin heavy chain (CHC), we demonstrated that IBV mainly utilized the CME for its entry. Furthermore, GTPase dynamin1 was involved in virus containing vesicle scission and internalization. Surprisingly, CME adaptor Eps15 had no effect on IBV internalization. Third, the penetration of IBV into cells led to active cytoskeleton rearrangement. After internalization, virus particles moved along with the classical endosome/lysosome track, as evidenced by co-localization of R18 labeled IBV with vehicle markers Rab5/Rab7/LAMP1 along with the infection time course. Functional inactivation of Rab5 and Rab7 significantly inhibited IBV infection. VCP, a protein helps early endosome maturation, was involved virus trafficking. Finally, by using the dual R18/DiOC labeled IBV, we observed that membrane fusion with late endosome/lysosome membranes was induced between 2-3 h.p.i.. Taken together, our findings demonstrate that IBV virions attach to lipid rafts and are internalized into cells via CME, move along with early/late endosomes-lysosomes, finally fuse with late endosome-lysosome membranes, release virus genome into cytoplasm. This study provides comprehensive images of IBV attachment-internalization-trafficking-fusion steps.IMPORTANCEIBV, the avian coronavirus isolated in 1937, infects chicken and causes economic loss in poultry industry. It has been reported that the entry of IBV requires low pH. However, the molecular mechanisms underlying IBV internalization and trafficking remain to be clarified. Therefore, we employed multiple chemical and molecular approaches to dissect the entry mechanisms of IBV in susceptible cells. Our results showed IBV entry was significantly inhibited when clathrin-mediated endocytosis (CME) was blocked by chemical inhibitor or depletion of clathrin protein. Moreover, by using R18-labeled IBV, we found that IBV particles attached to lipid rafts, led to actin rearrangement, and moved along with the entire endosomal system. R18/DiOC labeling method showed that IBV fused with late endosomes or lysosomes. This is the first report to describe the entire entry process of IBV, allowing for a better understanding of the infection process of group III avian coronavirus.","The COVID-19 pandemic is caused by the single-stranded RNA virus severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), a virus of zoonotic origin that was first detected in Wuhan, China in December 2019. There is evidence that homologous recombination contributed to this cross-species transmission. Since that time the virus has demonstrated a high propensity for human-to-human transmission. Here we report two newly identified adjacent amino acid polymorphisms in the nucleocapsid at positions 203 and 204 (R203K/G204R) due to three adjacent nucleotide changes across the two codons (i.e. AGG GGA to AAA CGA). This new strain within the LGG clade may have arisen by a form of homologous recombination from the core sequence (CS-B) of the transcription-regulating sequences of SAS-CoV-2 itself and has rapidly increased to approximately one third of reported sequences from Europe during the month of March 2020. We note that these polymorphisms are predicted to reduce the binding of an overlying putative HLA-C*07-restricted epitope and that HLA-C*07 is prevalent in Caucasians being carried by &gt;40% of the population. The findings suggest that homologous recombination may have occurred since its introduction into humans and be a mechanism for increased viral fitness and adaptation of SARS-CoV-2 to human populations.","ABSTRACTThe international spread of the novel, pathogenic SARS-coronavirus 2 (SARS-CoV-2) poses a global challenge on both healthcare and society. A multitude of research efforts worldwide aim at characterizing the cellular factors involved in viral transmission in order to reveal therapeutic targets. For a full understanding of the susceptibility for SARS-CoV-2 infection, the cell type-specific expression of the host cell surface receptor is necessary. The key protein suggested to be involved in host cell entry is Angiotensin I converting enzyme 2 (ACE2), and its expression has been reported in various human organs, in some cases with inconsistent or contradictory results. Here, we aim to verify a reliable expression profile of ACE2 in all major human tissues and cell types. Based on stringently validated immunohistochemical analysis and high-throughput mRNA sequencing from several datasets, we found that ACE2 expression is mainly localized to microvilli of the intestinal tract and renal proximal tubules, gallbladder epithelium, testicular Sertoli cells and Leydig cells, glandular cells of seminal vesicle and cardiomyocytes. The expression in several other previously reported locations, including alveolar type II (AT2) cells, could not be confirmed. Furthermore, ACE2 expression was absent in the AT2 lung carcinoma cell line A549, often used as a model for viral replication studies. Our analysis suggests that the expression of ACE2 in the human respiratory system appears to be limited, and the expression of the receptor in lung or respiratory epithelia on the protein level is yet to be confirmed. This raises questions regarding the role of ACE2 for infection of human lungs and highlights the need to further explore the route of transmission during SARS-CoV-2 infection.","The COVID-19 pandemic, caused by the novel coronavirus SARS-CoV-2, creates an urgent need for identifying molecular mechanisms that mediate viral entry, propagation, and tissue pathology. Cell membrane bound angiotensin-converting enzyme 2 (ACE2) and associated proteases, transmembrane protease serine 2 (TMPRSS2) and Cathepsin L (CTSL), were previously identified as mediators of SARS-CoV2 cellular entry. Here, we assess the cell type-specific RNA expression of ACE2, TMPRSS2, and CTSL through an integrated analysis of 107 single-cell and single-nucleus RNA-Seq studies, including 22 lung and airways datasets (16 unpublished), and 85 datasets from other diverse organs. Joint expression of ACE2 and the accessory proteases identifies specific subsets of respiratory epithelial cells as putative targets of viral infection in the nasal passages, airways, and alveoli. Cells that co-express ACE2 and proteases are also identified in cells from other organs, some of which have been associated with COVID-19 transmission or pathology, including gut enterocytes, corneal epithelial cells, cardiomyocytes, heart pericytes, olfactory sustentacular cells, and renal epithelial cells. Performing the first meta-analyses of scRNA-seq studies, we analyzed 1,176,683 cells from 282 nasal, airway, and lung parenchyma samples from 164 donors spanning fetal, childhood, adult, and elderly age groups, associate increased levels of ACE2, TMPRSS2, and CTSL in specific cell types with increasing age, male gender, and smoking, all of which are epidemiologically linked to COVID-19 susceptibility and outcomes. Notably, there was a particularly low expression of ACE2 in the few young pediatric samples in the analysis. Further analysis reveals a gene expression program shared by ACE2+TMPRSS2+ cells in nasal, lung and gut tissues, including genes that may mediate viral entry, subtend key immune functions, and mediate epithelial-macrophage cross-talk. Amongst these are IL6, its receptor and co-receptor, IL1R, TNF response pathways, and complement genes. Cell type specificity in the lung and airways and smoking effects were conserved in mice. Our analyses suggest that differences in the cell type-specific expression of mediators of SARS-CoV-2 viral entry may be responsible for aspects of COVID-19 epidemiology and clinical course, and point to putative molecular pathways involved in disease susceptibility and pathogenesis.","One of the challenges in the current COVID-19 crisis is the time and cost of performing tests especially for large-scale population surveillance. Since, the probability of testing positive in large population studies is expected to be small (&lt;15%), therefore, most of the test outcomes will be negative. Here, we propose the use of agglomerative sampling which can prune out multiple negative cases in a single test by intelligently combining samples from different individuals. The proposed scheme builds on the assumption that samples from the population may not be independent of each other. Our simulation results show that the proposed sampling strategy can significantly increase testing capacity under resource constraints: on average, a saving of ~40% tests can be expected assuming a positive test probability of 10% across the given samples. The proposed scheme can also be used in conjunction with heuristic or Machine Learning guided clustering for improving the efficiency of large-scale testing further. The code for generating the simulation results for this work is available here: https://github.com/foxtrotmike/AS.","SARS-CoV-2 is the etiological agent responsible for the COVID-19 outbreak in Wuhan. Specific antiviral drug are urgently needed to treat COVID-19 infections. The main protease (Mpro) of SARS-CoV-2 is a key CoV enzyme that plays a pivotal role in mediating viral replication and transcription, which makes it an attractive drug target. In an effort to rapidly discover lead compounds targeting Mpro, two compounds (11a and 11b) were designed and synthesized, both of which exhibited excellent inhibitory activity with an IC50 value of 0.05 \u03bcM and 0.04 \u03bcM respectively. Significantly, both compounds exhibited potent anti-SARS-CoV-2 infection activity in a cell-based assay with an EC50 value of 0.42 \u03bcM and 0.33 \u03bcM, respectively. The X-ray crystal structures of SARS-CoV-2 Mpro in complex with 11a and 11b were determined at 1.5 \u00c5 resolution, respectively. The crystal structures showed that 11a and 11b are irreversible, the aldehyde groups of which are bound covalently to Cys145 of Mpro. Both compounds are promising drug leads with clinical potential that merits further studies.","Mycetoma is a neglected tropical disease, endemic in many tropical and subtropical regions, characterised by massive deformity and disability and can be fatal if untreated early and appropriately. Interleukins (IL) -35 and IL-37 are newly discovered cytokines that play an important role in suppressing the immune system. However, the expression of these interleukins in patients with Madurella mycetomatis (M. mycetomatis) induced eumycetoma has not yet been explored. This study aims to determine the levels of the IL-1 family (IL-1\u03b2, IL-37) and IL-12 family (IL-12, IL-35) in a group of these patients and the association between these cytokines levels and the patients' demographic characteristics. The present, a case-control study was conducted at the Mycetoma Research Centre, Soba University Hospital, University of Khartoum, Sudan and it included 140 individuals. They were divided into two groups; group I: healthy controls [n = 70; median age 25 years (range 12 to 70 years)]. Group II: mycetoma patients [n = 70 patients; median age 25 (range 13 to 70 years)]. Cytokines levels were measured in sera using enzyme-linked immunosorbent assay (ELISA). There was no significant correlation between the IL-1\u03b2 and IL-12 levels and the lesions size and disease duration, whereas levels of IL-37 and IL-35 were significantly correlated with that. The analysis of the risk factors of higher circulatory levels of IL-37 in patients of mycetoma showed a significant negative association with IL-1\u03b2 cytokine, where a unit increment in IL-1\u03b2 will decrease the levels of IL-37 by 35.28 pg/ml. The levels of IL-37 among the patients with a duration of mycetoma infection \u2264 one year had significantly decreased by an average of 18.45 compared to patients with a mycetoma infection's duration of \u2265 5years (reference group). Furthermore, the risk factors of higher levels of IL-35 in mycetoma patients revealed a significant negative association with IL-12, as a unit increment in IL-12 decreases the levels of IL-35by 8.99 pg/ml (p &lt; 0.001). Levels of IL-35 among the patients with duration of mycetoma infection \u2264 one year had significantly decreased (p-value = 0.002) on average by 41.82 compared to patients with a duration of mycetoma infection \u2265 five years (reference group). In conclusion, this study indicates that both IL-35 and IL-37 are negatively associated with the levels of IL-1\u03b2 and IL-12 in eumycetoma mycetoma infection; and high levels of IL-37 and IL-35 may have a negative impact on disease progression.","Abstract2019-nCov has caused more than 80 deaths as of 27 January 2020 in China, and infection cases have been reported in more than 10 countries. However, there is no approved drug to treat the disease. 2019-nCov Mpro is a potential drug target to combat the virus. We built homology models based on SARS Mpro structures, and docked 1903 small molecule drugs to the models. Based on the docking score and the 3D similarity of the binding mode to the known Mpro ligands, 4 drugs were selected for binding free energy calculations. Both MM/GBSA and SIE methods voted for nelfinavir, with the binding free energy of \u221224.69\u00b10.52 kcal/mol and \u22129.42\u00b10.04 kcal/mol, respectively. Therefore, we suggested that nelfinavir might be a potential inhibitor against 2019-nCov Mpro.","AbstractWhile the ecology of infectious disease is a rich field with decades worth of empirical evidence and theory, there are aspects that remain relatively under-examined. One example is the importance of the free-living survival stage of certain pathogens, and especially is cases where they are transmitted indirectly between hosts through an environmental reservoir intermediate. In this study, we develop an integrated, broadly applicable mathematical method to examine diseases fitting this description\u2014the waterborne, abiotic and other indirectly transmitted (W.A.I.T.) infection framework. To demonstrate its utility, we construct realistic models of two very different epidemic scenarios: cholera in a densely populated setting with limited access to clean drinking water and hepatitis C virus in an urban setting of injection-drug users. Using these two exemplars, we find that the W.A.I.T. model fortifies the centrality of reservoir dynamics in the \u201csit and wait\u201d infection strategy, and provides a way to simulate a diverse set of intervention strategies.","Motivation: Over the past years, NGS has become a crucial workhorse for open-view pathogen diagnostics. Yet, long turnaround times result from using massively parallel high-throughput technologies as the analysis can only be performed after sequencing has finished. The interpretation of results can further be challenged by contaminations, clinically irrelevant sequences, and the sheer amount and complexity of the data. Results: We implemented PathoLive, a real-time diagnostics pipeline for the detection of pathogens from clinical samples hours before sequencing has finished. Based on real-time alignment with HiLive2, mappings are scored with respect to common contaminations, low-entropy areas, and sequences of widespread, non-pathogenic organisms. The results are visualized using an interactive taxonomic tree that provides an easily interpretable overview of the relevance of hits. For a human plasma sample that was spiked in vitro with six pathogenic viruses, all agents were clearly detected after only 40 of 200 sequencing cycles. For a real-world sample from Sudan the results correctly indicated the presence of Crimean-Congo hemorrhagic Fever Virus. In a second real-world dataset from the 2019 SARS-CoV-2 outbreak in Wuhan, we found the presence of a SARS Coronavirus as the most relevant hit without the novel virus reference genome being included in the database. For all samples, clinically irrelevant hits were correctly deemphasized. Our approach is valuable to obtain fast and accurate NGS-based pathogen identifications and correctly prioritize and visualize them based on their clinical significance. Availability: PathoLive is open source and available on GitLab (https://gitlab.com/rki_bioinformatics/PathoLive) and BioConda (conda install -c bioconda patholive).","ABSTRACTIt has been recently estimated that one-third of the world population will be infected by Zika virus, but unfortunately so far there is no vaccine or medicine available. In particular, the special concern on the vaccine treatment to Zika and Dengue arising from antibody-dependent enhancement strongly emphasizes the irreplaceable role of its NS2B-NS3 protease (NS2B-NS3pro) as a target for anti-Zika drug discovery/design due to its absolutely-essential role in viral replication. Very recently we identified two small molecules inhibit Zika NS2B-NS3pro in non-competitive mode, with Ki values of 0.57 and 2.02 \u00b5M respective for p-Nitrophenyl-p-guanidino benzoate and qucertin. Here, by molecular docking, we show that although one is designed compound while another is a natural product, both molecules bind to the same pocket on the back of the substrate-binding pocket of Zika NS2B-NS3pro. As the two inhibitors fundamentally differ from cn-716, the only known peptidomimetic boronic acid inhibitor in both structure scaffolds and inhibitory modes, our discovery might open up a new avenue for the future development of allosteric inhibitors, which is highly demanded to achieve therapeutic inhibition of flaviviral NS2B-NS3pro complexes. Furthermore, as qucertin is abundant in many vegetables and fruits such caper, lovage, tea and red onion, our results should benefit the public to immediately fight Zika virus.","AbstractPandemic coronavirus disease 2019 (COVID-19) is caused by the emerging severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), for which there are no efficacious vaccines or therapeutics that are urgently needed. We expressed three versions of spike (S) proteins\u2014receptor binding domain (RBD), S1 subunit and S ectodomain\u2014in insect cells. RBD appears monomer in solutions, whereas S1 and S associate into homotrimer with substantial glycosylation. The three proteins confer excellent antigenicity with six convalescent COVID-19 patient sera. Cryo-electron microscopy (cryo-EM) analyses indicate that the SARS-CoV-2 S trimer dominate in a unique conformation distinguished from the classic prefusion conformation of coronaviruses by the upper S1 region at lower position ~15 \u00c5 proximal to viral membrane. Such conformation is proposed as an early prefusion state for the SARS-CoV-2 spike that may broaden the knowledge of coronavirus and facilitate vaccine development.","ABSTRACTThe coronavirus (CoV) RNA genome is the largest among single-stranded positive sense RNA viruses. CoVs encode a proofreading 3\u2032\u21925\u2032exoribonuclease within nonstructural protein 14 (nsp14-ExoN) that is responsible for CoV high-fidelity replication. Alanine substitution of ExoN catalytic residues [ExoN(-)] in SARS-CoV and murine hepatitis virus (MHV) disrupts ExoN activity, yielding viable mutant viruses with defective replication, up to 20-fold decreased fidelity, and increased susceptibility to nucleoside analogs. To test the stability of the ExoN(-) genotype and phenotype, we passaged MHV-ExoN(-) 250 times in cultured cells (P250), in parallel with WT-MHV. Compared to MHV-ExoN(-) P3, MHV-ExoN(-) P250 demonstrated enhanced replication, reduced susceptibility to nucleoside analogs, and increased competitive fitness. However, passage did not select for complete or partial reversion at the ExoN-inactivating mutations. We identified novel amino acid changes within the RNA-dependent RNA polymerase (nsp12-RdRp) and nsp14 of MHV-ExoN(-) P250 that partially account for the observed changes in replication, susceptibility to nucleoside analogs, and competitive fitness observed in the passaged virus population, indicating that additional determinants can compensate for the activities of nsp14-ExoN. Our results suggest that while selection favors restoration of replication fidelity in ExoN(-) CoVs, there may be a significant barrier to ExoN(-) reversion. These results also support the hypothesis that high-fidelity replication is linked to CoV fitness and identify additional candidate proteins that may regulate CoV replication fidelity.IMPORTANCEUnique among RNA viruses, CoVs encode a proofreading exoribonuclease (ExoN) in nsp14 that mediates high-fidelity RNA genome replication. Proofreading-deficient CoVs with disrupted ExoN activity [ExoN(-)] are either non-viable or have significant defects in replication, RNA synthesis, fidelity, fitness, and virulence. In this study, we show that ExoN(-) murine hepatitis virus can adapt over long-term passage for increased replication and fitness without reverting the ExoN-inactivating mutations. Passage-adapted ExoN(-) mutants also demonstrate increasing resistance to nucleoside analogs that is only partially explained by secondary mutations in nsp12 and nsp14. These data suggest that enhanced resistance to nucleoside analogs is mediated by the interplay of multiple replicase proteins and support the proposed link between CoV fidelity and fitness.","SummaryThe Mad1-Mad2 heterodimer is the catalytic hub of the spindle assembly checkpoint (SAC), which controls mitosis through assembly of a multi-subunit anaphase inhibitor, the mitotic checkpoint complex (MCC) [1, 2]. Mad1-Mad2 first catalyzes MCC assembly at interphase nuclear pores [3], then migrates to kinetochores at nuclear envelope breakdown (NEBD) and resumes MCC assembly until bipolar spindle attachment is complete [1, 2]. There is significant debate about the factor(s) involved in targeting Mad1-Mad2 to kinetochores in higher eukaryotes [4-9]. Through gene editing and live-cell imaging, we found that the human Rod-Zw10-Zwilch (RZZ) complex is dispensable for cell viability and initial recruitment of Mad1-Mad2 to kinetochores at NEBD, but then becomes necessary to tether Mad1-Mad2 at kinetochores and sustain SAC arrest in cells challenged with spindle poisons. We also show that RZZ forms the mesh-like fibrous corona, a structural expansion of the outer kinetochore important for timely chromosome congression [10-13] once Mps1 phosphorylates the N-terminus of Rod. Artificially tethering Mad1-Mad2 to kinetochores enabled long-term mitotic arrest in the absence of RZZ. Conversely, blocking early RZZ-independent recruitment of Mad1-Mad2 eliminated the transient SAC response in RZZ-null cells. We conclude that RZZ drives structural changes in the outer kinetochore that facilitate chromosome bi-orientation and chronic SAC transduction, a key determinant of cytotoxicity during anti-mitotic drug therapy [14-16].","AbstractUsing incipient lines of the Collaborative Cross (CC), a murine genetic reference population, we previously identified a quantitative trait loci (QTL) associated with low SARS-CoV titer. In this study, we integrated sequence information and RNA expression of genes within the QTL to identify mucin 4 (Muc4) as a high priority candidate for controlling SARS-CoV titer in the lung. To test this hypothesis, we infected Muc4-/- mice and found that female, but not male, Muc4-/- mice developed more weight loss and disease following infection with SARS-CoV. Female Muc4-/- mice also had more difficulty breathing despite reduced lung pathology; however, no change in viral titers was observed. Comparing across viral families, studies with chikungunya virus, a mosquito-borne arthralgic virus, suggests that Muc4\u2019s impact on viral pathogenesis may be widespread. Although not confirming the original titer QTL, our data identifies a role for Muc4 in the SARS-CoV disease and viral pathogenesis.ImportanceGiven the recent emergence of SARS-CoV-2, this work suggest that Muc4 expression plays a protective role in female mice not conserved in male mice following SARS-CoV infection. With the SARS-CoV-2 outbreak continuing, treatments that modulate or enhance Muc4 activity may provide an avenue for treatment and improved outcomes. In addition, the work highlights the importance of studying host factors including host genetics and biological sex as key parameters influencing infection and disease outcomes.","1AbstractThe confident high-throughput identification of small molecules remains one of the most challenging tasks in mass spectrometry-based metabolomics. SIRIUS has become a powerful tool for the interpretation of tandem mass spectra, and shows outstanding performance for identifying the molecular formula of a query compound, being the first step of structure identification. Nevertheless, the identification of both molecular formulas for large compounds above 500 Daltons and novel molecular formulas remains highly challenging. Here, we present ZODIAC, a network-based algorithm for the de novo estimation of molecular formulas. ZODIAC reranks SIRIUS\u2019 molecular formula candidates, combining fragmentation tree computation with Bayesian statistics using Gibbs sampling. Through careful algorithm engineering, ZODIAC\u2019s Gibbs sampling is very swift in practice. ZODIAC decreases incorrect annotations 16.2-fold on a challenging plant extract dataset with most compounds above 700 Dalton; we then show improvements on four additional, diverse datasets. Our analysis led to the discovery of compounds with novel molecular formulas such as C24H47BrNO8P which, as of today, is not present in any publicly available molecular structure databases.","AbstractNon-viral gene delivery is constrained by the dwell time that most synthetic nucleic acid nanocarriers spend inside endosomal compartments. In order to overcome this endosomal-release bottleneck, methods are required that measure nanocarrier uptake kinetics and transfection efficiency simultaneously. Here, we employ live-cell imaging on single-cell arrays (LISCA) to study the delivery-time distribution of lipid-based mRNA complexes under varied serum conditions. By fitting a translation-maturation model to hundreds of individual eGFP reporter fluorescence time courses, the protein expression onset times and the expression rates after transfection are determined. Using this approach, we find that delivery timing and protein expression rates are not intrinsically correlated at the single-cell level, even though population-averaged values of both parameters conjointly change as a function of increasing external serum protein fraction. Lipofectamine mediated delivery showed decreased transfection efficiency and longer delivery times with increasing serum protein concentration. This is in contrast to ionizable lipid nanoparticles (LNPs) mediated transfer, which showed increased efficiency and faster uptake in the presence of serum. In conclusion, the interdependences of single-cell expression rates and onset timing provide additional clues on uptake and release mechanisms, which are useful for improving nucleic acid delivery.","AbstractBackgroundAs of February 25, 2019, 875 cases of Ebola virus disease (EVD) were reported in North Kivu and Ituri Provinces, Democratic Republic of Congo. Since the beginning of October, the outbreak has largely shifted into regions in which active armed conflict has occurred, and in which EVD cases and their contacts have been difficult for health workers to reach. We used available data on the current outbreak with case-count time series from prior outbreaks to project the short-term and long-term course of the outbreak.MethodsFor short- and long-term projections, we modeled Ebola virus transmission using a stochastic branching process that assumes gradually quenching transmission rates estimated from past EVD outbreaks, with outbreak trajectories conditioned on agreement with the course of the current outbreak, and with multiple levels of vaccination coverage. We used two regression models to estimate similar projection periods. Short- and long-term projections were estimated using negative binomial autoregression and Theil-Sen regression, respectively. We also used Gott\u2019s rule to estimate a baseline minimum-information projection. We then constructed an ensemble of forecasts to be compared and recorded for future evaluation against final outcomes. From August 20, 2018 to February 25, 2019, short-term model projections were validated against known case counts.ResultsDuring validation of short-term projections, from one week to four weeks, we found models consistently scored higher on shorter-term forecasts. Based on case counts as of February 25, the stochastic model projected a median case count of 933 cases by February 18 (95% prediction interval: 872\u20131054) and 955 cases by March 4 (95% prediction interval: 874\u20131105), while the auto-regression model projects median case counts of 889 (95% prediction interval: 876\u2013933) and 898 (95% prediction interval: 877\u2013983) cases for those dates, respectively. Projected median final counts range from 953 to 1,749. Although the outbreak is already larger than all past Ebola outbreaks other than the 2013\u20132016 outbreak of over 26,000 cases, our models do not project that it is likely to grow to that scale. The stochastic model estimates that vaccination coverage in this outbreak is lower than reported in its trial setting in Sierra Leone.ConclusionsOur projections are concentrated in a range up to about 300 cases beyond those already reported. While a catastrophic outbreak is not projected, it is not ruled out, and prevention and vigilance are warranted. Prospective validation of our models in real time allowed us to generate more accurate short-term forecasts, and this process may prove useful for future real-time short-term forecasting. We estimate that transmission rates are higher than would be seen under target levels of 62% coverage due to contact tracing and vaccination, and this model estimate may offer a surrogate indicator for the outbreak response challenges.Author summaryAs of February 25, 2019, 875 cases of Ebola virus disease (EVD) were reported in North Kivu and Ituri Provinces, Democratic Republic of Congo. Since the beginning of October 2018, the outbreak has largely shifted into regions in which active armed conflict has been reported, and in which EVD cases and their contacts have been difficult for health workers to reach. We used an ensemble of models to estimate EVD transmission rates and to forecast the short- and long-term course of the outbreak. Our models project that a final size of roughly up to 300 additional cases is most likely, and estimate that transmission rates are higher than would be seen under optimal levels of contact tracing and vaccination. While a catastrophic outbreak is not projected, is it not ruled out, and prevention and vigilance are warranted.","AbstractWhile numerous studies have explored single-word naming, few have evaluated the behavioral and neural correlates of more naturalistic language, like connected speech, which we produce every day. Here, in a retrospective analysis of 120 participants at least six months following left hemisphere stroke, we evaluated the distribution of word errors (paraphasias) and associated brain damage during connected speech (picture description) and object naming. While paraphasias in connected speech and naming shared underlying neural substrates, analysis of the distribution of paraphasias suggested that lexical-semantic load is likely reduced during connected speech. Using voxelwise lesion-symptom mapping (VLSM), we demonstrated that verbal (real word: semantically related and unrelated) and sound (phonemic and neologistic) paraphasias during both connected speech and naming loaded onto the left hemisphere ventral and dorsal streams of language, respectively. Furthermore, for the first time using both connected speech and naming data, we localized semantically related paraphasias to more anterior left hemisphere temporal cortex and unrelated paraphasias to more posterior left temporal and temporoparietal cortex. The connected speech results, in particular, highlight a gradient of specificity as one translates visual recognition from left temporo-occipital cortex to posterior and subsequently anterior temporal cortex. The robustness of VLSM results for sound paraphasias derived during connected speech was notable, in that analyses performed on sound paraphasias from the connected speech task, and not the naming task, demonstrated significant results following removal of lesion volume variance and related apraxia of speech variance. Therefore, connected speech may be a particularly sensitive task on which to evaluate further lexical-phonological processing in the brain. The results presented here demonstrate the related, though different, distribution of paraphasias during connected speech, confirm that paraphasias arising in connected speech and single-word naming likely share neural origins, and endorse the need for continued evaluation of the neural substrates of connected speech processes.","AbstractAngiotensin-converting enzyme 2 (ACE2) is the surface receptor for SARS coronavirus (SARS-CoV), directly interacting with the spike glycoprotein (S protein). ACE2 is also suggested to be the receptor for the new coronavirus (2019-nCoV), which is causing a serious epidemic in China manifested with severe respiratory syndrome. B0AT1 (SLC6A19) is a neutral amino acid transporter whose surface expression in intestinal cells requires ACE2. Here we present the 2.9 \u00c5 resolution cryo-EM structure of full-length human ACE2 in complex with B0AT1. The complex, assembled as a dimer of ACE2-B0AT1 heterodimers, exhibits open and closed conformations due to the shifts of the peptidase domains (PDs) of ACE2. A newly resolved Collectrin-like domain (CLD) on ACE2 mediates homo-dimerization. Structural modelling suggests that the ACE2-B0AT1 complex can bind two S proteins simultaneously, providing important clues to the molecular basis for coronavirus recognition and infection.","AbstractPositive-sense RNA viruses hijack intracellular membranes that provide niches for viral RNA synthesis and a platform for interactions with host proteins. However, little is known about host factors at the interface between replicase complexes and the host cytoplasm. We engineered a biotin ligase into a coronaviral replication/transcription complex (RTC) and identified &gt;500 host proteins constituting the RTC microenvironment. siRNA-silencing of each RTC-proximal host factor demonstrated importance of vesicular trafficking pathways, ubiquitin-dependent and autophagy-related processes, and translation initiation factors. Notably, detection of translation initiation factors at the RTC was instrumental to visualize and demonstrate active translation proximal to replication complexes of several coronaviruses.Collectively, we establish a spatial link between viral RNA synthesis and diverse host factors of unprecedented breadth. Our data may serve as a paradigm for other positive-strand RNA viruses and provide a starting point for a comprehensive analysis of critical virus-host interactions that represent targets for therapeutic intervention.","AbstractThe recent outbreak of coronavirus disease (COVID-19) caused by SARS-CoV-2 infection in Wuhan, China has posed a serious threat to global public health. To develop specific anti-coronavirus therapeutics and prophylactics, the molecular mechanism that underlies viral infection must first be confirmed. Therefore, we herein used a SARS-CoV-2 spike (S) protein-mediated cell-cell fusion assay and found that SARS-CoV-2 showed plasma membrane fusion capacity superior to that of SARS-CoV. We solved the X-ray crystal structure of six-helical bundle (6-HB) core of the HR1 and HR2 domains in SARS-CoV-2 S protein S2 subunit, revealing that several mutated amino acid residues in the HR1 domain may be associated with enhanced interactions with HR2 domain. We previously developed a pan-coronavirus fusion inhibitor, EK1, which targeted HR1 domain and could inhibit infection by divergent human coronaviruses tested, including SARS-CoV and MERS-CoV. We then generated a series of lipopeptides and found that the EK1C4 was the most potent fusion inhibitor against SARS-CoV-2 S protein-mediated membrane fusion and pseudovirus infection with IC50s of 1.3 and 15.8 nM, about 241- and 149-fold more potent than that of EK1 peptide, respectively. EK1C4 was also highly effective against membrane fusion and infection of other human coronavirus pseudoviruses tested, including SARS-CoV and MERS-CoV, as well as SARSr-CoVs, potently inhibiting replication of 4 live human coronaviruses, including SARS-CoV-2. Intranasal application of EK1C4 before or after challenge with HCoV-OC43 protected mice from infection, suggesting that EK1C4 could be used for prevention and treatment of infection by currently circulating SARS-CoV-2 and emerging SARSr-CoVs.","AbstractThe development of new drugs is costly, time consuming, and often accompanied with safety issues. Drug repurposing can avoid the expensive and lengthy process of drug development by finding new uses for already approved drugs. In order to repurpose drugs effectively, it is useful to know which proteins are targeted by which drugs. Computational models that estimate the interaction strength of new drug\u2013target pairs have the potential to expedite drug repurposing. Several models have been proposed for this task. However, these models represent the drugs as strings, which is not a natural way to represent molecules. We propose a new model called GraphDTA that represents drugs as graphs and uses graph neural networks to predict drug\u2013target affinity. We show that graph neural networks not only predict drug\u2013target affinity better than non-deep learning models, but also outperform competing deep learning methods. Our results confirm that deep learning models are appropriate for drug\u2013target binding affinity prediction, and that representing drugs as graphs can lead to further improvements.Availability of data and materialsThe proposed models are implemented in Python. Related data, pre-trained models, and source code are publicly available at https://github.com/thinng/GraphDTA.All scripts and data needed to reproduce the post-hoc statistical analysis are available from https://doi.org/10.5281/zenodo.3603523.","AbstractIn times of public crises, including the current Covid-19 pandemic, rapid dissemination of relevant scientific knowledge is of paramount importance. The duration of scholarly journals\u2019 publication process is one of the main factors hindering quick delivery of new information. While proper editorial assessment and peer review obviously require some time, turnaround times for medical journals can be up to several months, which is undesirable in the era of a crisis. Following initiatives of medical journals and scholarly publishers to accelerate their publication process, this study assesses whether medical journals have indeed managed to speed up their publication process for Covid-19 related articles. It studies the duration of 14 medical journals\u2019 publication process both during and prior to the current pandemic. Assessing a total of 669 articles, the study concludes that medical journals have indeed drastically accelerated the publication process for Covid-19 related articles since the outbreak of the pandemic. Compared to articles published in the same journals before the pandemic, turnaround times have decreased on average by 49%. The largest decrease in number of days between submission and publication of articles was due to a decrease in the number of days required for peer review. For articles not related to Covid-19, no acceleration of the publication process is found. While the acceleration of journals\u2019 publication process is laudable from the perspective of quick information dissemination, it also raises concerns relating to the quality of the peer review process and the quality of the resulting publications.","AbstractSodium-bile acid cotransporter, also denominated sodium-taurocholate cotransporting polypeptide (NTCP) is an integral membrane protein with multiple hydrophobic transmembrane domains. The third extracellular domain of NTCP presents a stretch of nine aminoacids (KGIVISLVL) that is characterized by pronounced hydrophobicity and serves as receptor for a protein, preS1, showing the hydrophobic epta-peptide sequence NPLGFFP. Vitamin D-binding protein macrophage activating factor (DBP-MAF) is a multifunctional protein that is characterized by two hydrophobic regions able to bind fatty acids and vitamin D, respectively. Here we demonstrate that NTCP and DBP-MAF show significant sequence similarities as far as hydrophobic stretches of aminoacids are concerned. Alignment of the sequence of seven aminoacids preceding the 157-KGIVISLVL-165 stretch of NTCP shows four aminoacids that are identical to those of the corresponding sequence of DBP-MAF, and two that are conserved substitutions. In addition, in the sequence of DBP-MAF that is aligned with the sequence YKGIVISLVL of NTCP, there are two contiguous negatively charged aminoacids (ED) and, in the preceding epta-peptide sequence, there are three negatively charged aminoacids (D-ED), whereas in the corresponding sequence of NTCP there are only two (D--D) that are not contiguous. This concentration of negatively charged aminoacids may be involved in binding of protein inserts characterized by high density of positively charges residues. The alternating hydrophobic and electrostatic interactions described in this paper may help elucidating the biological roles of these proteins as far as protein-protein interactions are concerned.","AbstractHost shifts - where a pathogen jumps between different host species - are an important source of emerging infectious disease. With ongoing climate change there is an increasing need to understand the effect changes in temperature may have on emerging infectious disease. We investigated whether species\u2019 susceptibilities change with temperature and ask if susceptibility is greatest at different temperatures in different species. We infected 45 species of Drosophilidae with an RNA virus and measured how viral load changes with temperature. We found the host phylogeny explained a large proportion of the variation in viral load at each temperature, with strong phylogenetic correlations between viral loads across temperature. The variance in viral load increased with temperature, whilst the mean viral load did not, such that as temperature increased the most susceptible species become more susceptible, and the least susceptible less so. We found no significant relationship between a species\u2019 susceptibility across temperatures and proxies for thermal optima; critical thermal maximum and minimum or basal metabolic rate. These results suggest that whilst the rank order of species susceptibilities can remain the same with changes in temperature, the likelihood of host shifts into a given species may increase or decrease.Author SummaryEmerging infectious diseases are often the result of a host shift, where a pathogen jumps from one host species into another. Understanding the factors underlying host shifts is a major goal for infectious disease researchers. This effort has been further complicated by the fact that host-parasite interactions are now taking place in a period of unprecedented global climatic warming. Here, we ask how host shifts are affected by temperature by carrying out experimental infections using an RNA virus across a wide range of related species, at three different temperatures. We find that as temperature increases the most susceptible species become more susceptible, and the least susceptible less so. This has important consequences for our understanding of host shift events in a changing climate, and suggests that temperature changes may affect the likelihood of a host shift into certain species.","The emergence of the betacoronavirus, SARS-CoV-2 that causes COVID-19, represents a significant threat to global human health. Vaccine development is focused on the principal target of the humoral immune response, the spike (S) glycoprotein, that mediates cell entry and membrane fusion. SARS-CoV-2 S gene encodes 22 N-linked glycan sequons per protomer, which likely play a role in immune evasion and occluding immunogenic protein epitopes. Here, using a site-specific mass spectrometric approach, we reveal the glycan structures on a recombinant SARS-CoV-2 S immunogen. This analysis enables mapping of the glycan-processing states across the viral spike. We show how SARS-CoV-2 S glycans differ from typical host glycan processing, which may have implications in viral pathobiology and vaccine design.","The COVID-19 pandemic demands assimilation of all available biomedical knowledge to decode its mechanisms of pathogenicity and transmission. Despite the recent renaissance in unsupervised neural networks for decoding unstructured natural languages, a platform for the real-time synthesis of the exponentially growing biomedical literature and its comprehensive triangulation with deep omic insights is not available. Here, we present the nferX platform for dynamic inference from over 45 quadrillion possible conceptual associations extracted from unstructured biomedical text, and their triangulation with Single Cell RNA-sequencing based insights from over 25 tissues (https://academia.nferx.com/). Using this platform, we identify intersections between the pathologic manifestations of COVID-19 and the comprehensive expression profile of the SARS-CoV-2 receptor ACE2. We find that tongue keratinocytes, airway club cells, and ciliated cells are likely under-appreciated targets of SARS-CoV-2 infection, in addition to type II pneumocytes and olfactory epithelial cells. We further identify mature small intestinal enterocytes as a possible hotspot of COVID-19 fecal-oral transmission, where an intriguing maturation-correlated transcriptional signature is shared between ACE2 and the other coronavirus receptors DPP4 (MERS-CoV) and ANPEP (\u03b1-coronavirus). This study demonstrates how a holistic data science platform can leverage unprecedented quantities of structured and unstructured publicly available data to accelerate the generation of impactful biological insights and hypotheses.","AbstractTo compare the secondary structures of RNA molecules we developed the CROSSalign method. CROSSalign is based on the combination of the Computational Recognition Of Secondary Structure (CROSS) algorithm to predict the RNA secondary structure at single-nucleotide resolution using sequence information, and the Dynamic Time Warping (DTW) method to align profiles of different lengths. We applied CROSSalign to investigate the structural conservation of long non-coding RNAs such as XIST and HOTAIR as well as ssRNA viruses including HIV. In a pool of sequences with the same secondary structure CROSSalign accurately recognizes repeat A of XIST and domain D2 of HOTAIR and outperforms other methods based on covariance modelling. CROSSalign can be applied to perform pair-wise comparisons and is able to find homologues between thousands of matches identifying the exact regions of similarity between profiles of different lengths. The algorithm is freely available at the webpage http://service.tartaglialab.com//new_submission/CROSSalign.","AbstractAstroviruses (AstVs) are RNA viruses responsible for infection of a large diversity of avian and mammalian species, including bats, livestock, and humans. We investigated AstV infection in a free-tailed bat species, Mormopterus francoismoutoui, endemic to Reunion Island. A total of 190 guano samples were collected in a maternity colony during 19 different sampling sessions, between June 2016 and June 2017. Biological material was tested for the presence of the AstV RNA-dependent RNA-polymerase (RdRp) gene with a pan-AstV semi-nested polymerase chain reaction assay. In total, 15 guano samples (7.9%) tested positive, with high genetic diversity of the partial RdRp gene sequences among positive samples. A phylogenetic analysis further revealed that the detected viruses were genetically related to AstVs reported in reptiles, dogs, and pigs, but did not cluster with AstVs commonly found in bats. Although more investigation need to be conducted to assess the level of infected bats in the studied population, our findings suggest that Reunion free-tailed bats are exposed to AstV, and that cross-species transmission may occur with other hosts sharing the same habitat.","AbstractNeuropsychiatric systemic lupus erythematosus (NPSLE) is a severe complication associated with the neurotoxic effects of circulating autoantibodies in the central nervous system (CNS) manifested frequently as a learning and memory deficit. Pristane-induced lupus in BALB/c female mice is an experimental model that resembles some clinical and immunological SLE pathogenesis associated with environmental factors. Nevertheless, there is no experimental evidence that relate pristane-induced lupus with cognitive dysfunction associated with autoantibodies production.ObjectiveTo evaluate cognitive impairment related to memory deficits in a pristane-induced lupus BALB/c female mice related to mRNA expression levels of NR2A/2B hippocampal subunits in short and long-term memory task at 7 and 12 weeks after LPS exposition (7wLPS and 12wLPS) in a behavioral test with the employment of Barnes maze.MethodsFifty-four female BALB/c mice of 8-12 weeks old were included in 2 experimental groups: 7 and 12 weeks after lypopolissacharide (LPS) exposure and classified in subgroups (control, pristane and pristane+LPS). To determine cognitive dysfunction, mice were tested in a Barnes maze. Serum anti-Sm antibodies and relative expression of hippocampal NR2A/NR2B subunits were quantified.ResultsPristane and pristane+LPS mice showed a prolonged escape latency at 7wLPS than at 12wLPS in short-term memory. Downregulation of hippocampal NR2A subunit was more evident than NR2B in pristane and pristane+LPS at 7wLPS and 12wLPS. The anti-Sm autoantibodies levels correlate with the relative expression of NR2A.ConclusionDownregulation of hippocampal NR2A/2B subunits in the pristane-model of lupus in BALB/c mice may be related to anti-Sm autoantibodies production with the consequence of cognitive impairment in early stages of autoimmune disease.","ABSTRACTHost plasma membrane protein SERINC5 is incorporated into budding retrovirus particles where it blocks subsequent entry into susceptible target cells. Three accessory proteins encoded by diverse retroviruses, HIV-1 Nef, EIAV S2, and MLV Glycogag, each independently disrupt SERINC5 antiviral activity, by redirecting SERINC5 from the site of virion assembly on the plasma membrane to an internal RAB7+ endosomal compartment. Pseudotyping retroviruses with particular glycoproteins, e.g., the vesicular stomatitis glycoprotein (VSV G), renders the infectivity of particles resistant to inhibition by virion-associated SERINC5. To better understand viral determinants for SERINC5-sensitivity, the effect of SERINC5 was assessed using HIV-1, MLV, and M-PMV virion cores, pseudotyped with glycoproteins from Arenavirus, Coronavirus, Filovirus, Rhabdovirus, Paramyxovirus, and Orthomyxovirus genera. Infectivity of particles, pseudotyped with HIV-1, amphotropic-MLV, or influenza virus glycoproteins, was decreased by SERINC5, whether the core was provided by HIV-1, MLV, or M-PMV. Particles generated by all three cores, and pseudotyped with glycoproteins from either avian leukosis virus-A, human endogenous retrovirus K (HERV-K), ecotropic-MLV, HTLV-1, Measles morbillivirus, lymphocytic choriomeningitis mammarenavirus (LCMV), Marburg virus, Ebola virus, severe acute respiratory syndrome-related coronavirus (SARS-CoV), or VSV, were insensitive to SERINC5. In contrast, particles pseudotyped with M-PMV, RD114, or rabies virus (RABV) glycoproteins were sensitive to SERINC5, but only with particular retroviral cores. Resistance to SERINC5 by particular glycoproteins did not correlate with reduced SERINC5 incorporation into particles or with the route of viral entry. These findings indicate that some non-retroviruses may be sensitive to SERINC5 and that, in addition to the viral glycoprotein, the retroviral core influences sensitivity to SERINC5.IMPORTANCEThe importance of SERINC5 for inhibition of retroviruses is underscored by convergent evolution among three non-monophyletic retroviruses, each of which encodes a structurally unrelated SERINC5 inhibitor. One of these retroviruses causes tumors in mice, a second anemia in horses, and a third causes AIDS. SERINC5 is incorporated into retrovirus particles where it blocks entry into target cells, via a mechanism that is dependent on the viral glycoprotein. Here we demonstrate that retroviruses pseudotyped with glycoproteins from several non-retroviruses are also inhibited by SERINC5, suggesting that enveloped viruses other than retroviruses may also be inhibited by SERINC5. Additionally, we found that sensitivity to SERINC5 is determined by the retrovirus core, as well as by the glycoprotein. By better understanding how SERINC5 inhibits viruses we hope to extend fundamental understanding of virus replication and of the native role of SERINC5 in cells, and perhaps to advance the development of new antiviral strategies.","AbstractClavibacter is an agriculturally important genus comprising a single species, Clavibacter michiganensis, and multiple subspecies, including, C. michiganensis subsp. nebraskensis which causes Goss\u2019s wilt/blight of corn and accounts for high yield losses - listed among the five most significant diseases of corn in the United States of America. Our research objective was to develop a robust and rapid multiplex TaqMan real-time PCR (qPCR) to detect C. michiganensis in general and C. michiganensis subsp. nebraskensis with enhanced reliability and accuracy by adding non-complementary AT sequences to the 5\u2019 end of the forward and reverse primers. Comparative genomic analyses were performed to identify unique and conserved gene regions for primer and probe design. The unique genomic regions, ABC transporter ATP-binding protein CDS/ABC-transporter permease and MFS transporter were determined for specific detection of C. michiganensis and C. m. subsp. nebraskensis, respectively. The AT-rich sequences at the 5\u2019 position of the primers enhanced the reaction efficiency and sensitivity of rapid qPCR cycling; the reliability, accuracy and high efficiency of the developed assay was confirmed after testing with 59 strains from inclusivity and exclusivity panels \u2013 no false positives or false negatives were detected. The assays were also validated through naturally and artificially infected corn plant samples; all samples were detected for C. michiganensis and C. m. subsp. nebraskensis with 100% accuracy. The assay with 5\u2019 AT-rich sequences detected up to 10- and 100-fg of C. michiganensis and C. michiganensis subsp. nebraskensis genome targets, respectively. No adverse effect was observed when sensitivity assays were spiked with host genomic DNA. Addition of 5\u2019 AT rich sequences enhanced the qPCR reaction efficiency from 0.82 (M = -3.83) and 0.91 (M = -3.54) to 1.04 (with optimum slope value; M = -3.23) for both C. michiganensis and C. michiganensis subsp. nebraskensis, respectively; a increase of 10-fold sensitivity was also obtained with C. michiganensis primer set. The methodology proposed here can be used to optimize the reaction efficiency and to harmonize the diagnostic protocols which have prodigious applications in routine diagnostics, biosecurity and microbial forensics.","AbstractNovel coronavirus has wrecked medical and health care facilities claiming ~5% death tolls globally. All efforts to contain the pathogenesis either using inhibitory drugs or vaccines largely remained futile due to a lack of better understanding of the genomic feature of this virus. In the present study, we compared the 2019-nCoV with other coronaviruses, which indicated that bat-SARS like coronavirus could be a probable ancestor of the novel coronavirus. The protein sequence similarity of pangolin-hCoV and bat-hCoV with human coronavirus was higher as compared to their nucleotide similarity denoting the occurrence of more synonymous mutations in the genome. Phylogenetic and alignment analysis of 591 novel coronaviruses of different clades from Group I to Group V revealed several mutations and concomitant amino acid changes. Detailed investigation on nucleotide substitution unfolded 100 substitutions in the coding region of which 43 were synonymous and 57 were of non-synonymous type. The non-synonymous substitutions resulting into 57 amino acid changes were found to be distributed over different hCoV proteins with maximum on spike protein. An important di-amino acid change RG to KR was observed in ORF9 protein. Additionally, several interesting features of the novel coronavirus genome have been highlighted in respect to various other human infecting viruses which may explain extreme pathogenicity, infectivity and simultaneously the reason behind failure of the antiviral therapies.SignificanceThis study presents a comprehensive phylogenetic analysis of SARS-CoV2 isolates to understand discrete mutations that are occurring between patient samples. This analysis will provide an explanation for varying treatment efficacies of different inhibitory drugs and a future direction towards a combinatorial treatment therapies based on the kind of mutation in the viral genome.","Human severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is most closely related, by average genetic distance, to two coronaviruses isolated from bats, RaTG13 and RmYN02. However, there is a segment of high amino acid similarity between human SARS-CoV-2 and a pangolin isolated strain, GD410721, in the receptor binding domain (RBD) of the spike protein, a pattern that can be caused by either recombination or by convergent amino acid evolution driven by natural selection. We perform a detailed analysis of the synonymous divergence, which is less likely to be affected by selection than amino acid divergence, between human SARS-CoV-2 and related strains. We show that the synonymous divergence between the bat derived viruses and SARS-CoV-2 is larger than between GD410721 and SARS-CoV-2 in the RBD, providing strong additional support for the recombination hypothesis. However, the synonymous divergence between pangolin strain and SARS-CoV-2 is also relatively high, which is not consistent with a recent recombination between them, instead it suggests a recombination into RaTG13. We also find a 14-fold increase in the dN/dS ratio from the lineage leading to SARS-CoV-2 to the strains of the current pandemic, suggesting that the vast majority of non-synonymous mutations currently segregating within the human strains have a negative impact on viral fitness. Finally, we estimate that the time to the most recent common ancestor of SARS-CoV-2 and RaTG13 or RmYN02 based on synonymous divergence, is 51.71 years (95% C.I., 28.11-75.31) and 37.02 years (95% C.I., 18.19-55.85), respectively.","AbstractSteroid compounds, which are expected to have dual functions in blocking host inflammation and MERS-CoV replication, were screened from a chemical library. Within this library, ciclesonide, an inhaled corticosteroid, suppressed human coronavirus replication in cultured cells, but did not suppress replication of respiratory syncytial virus or influenza virus. The effective concentration of ciclesonide to block SARS-CoV-2 (the cause of COVID-19) replication (EC90) was 6.3 \u03bcM. After the eleventh consecutive MERS-CoV passage in the presence of ciclesonide, a resistant mutation was generated, which resulted in an amino acid substitution (A25V) in nonstructural protein (NSP) 15, as identified using reverse genetics. A recombinant virus with the mutation was also resistant to ciclesonide suppression of viral replication. These observations suggest that the effect of ciclesonide was specific to coronavirus, suggesting this is a candidate drug for treatment of patients suffering MERS or COVID-19.","AbstractSARS-CoV-2 and SARS-CoV share a common human receptor ACE2. Protein-protein interaction structure modeling indicates that spike-RBD of the two viruses also has similar overall binding conformation and binding free energy to ACE2. In vitro assays using recombinant ACE2 proteins and ACE2 expressing cells confirmed the two coronaviruses\u2019 similar binding affinities to ACE2. The above studies provide experimental supporting evidences and possible explanation for the high transmissibility observed in the SARS-CoV-2 outbreak. Potent ACE2-blocking SARS-CoV neutralizing antibodies showed limited cross-binding and neutralizing activities to SARS-CoV-2. ACE2-non-blocking SARS-CoV RBD antibodies, though with weaker neutralizing activities against SARS-CoV, showed positive cross-neutralizing activities to SARS-CoV-2 with an unknown mechanism. These findings suggest a trade-off between the efficacy and spectrum for therapeutic antibodies to different coronaviruses, and hence highlight the possibilities and challenges in developing broadly protecting antibodies and vaccines against SARS-CoV-2 and its future mutants.","SARS CoV2 is a novel highly virulent pathogen which gains entry to human cells by binding with the cell surface receptor angiotensin converting enzyme (ACE2). We contrasted the binding interactions between human ACE2 and coronavirus spike protein receptor binding domains (RBDs) from (i) SARS-CoV2, (ii) the related but less virulent OC43 (Singapore COVID-19 strain), and (iii) the 2002 epidemic-causing SARS CoV1. We find that the RBDs of the spike protein of SARS-CoV2 are highly optimized to achieve the strongest possible binding interaction with ACE2 which is consistent with its enhanced pathogenicity. SARS CoV2 RBDs form the most stable complex with ACE2 (64.91%, and 28.07% higher binding scores than SARS CoV1 and OC43, respectively) while occupying the greatest number of residues in the ATR1 binding site. In fact, the spike protein RBDs from SARS-CoV2 out-compete the angiotensin 2 receptor type I (ATR1) which is the native binding partner of ACE2 by 64% in terms of binding affinity (quantified using the Rosetta scoring function). They accomplish this through strong electrostatic attachments with every fourth residue on the N-terminus alpha-helix (starting from Ser19 to Asn53) as the turn of the helix makes these residues solvent accessible. These results offer a computational explanation for the increased pathogenicity of SARS CoV2 and allude to therapeutic modalities by identifying and rank-ordering the ACE2 residues involved in binding with the virus.","We calculated the structural properties of &gt;2500 coronaviruses and computed &gt;100000 human protein interactions with severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). Using the CROSS method, we found that the SARS-CoV-2 region encompassing nucleotides 23000 and 24000 is highly conserved at the structural level, while the region 1000 nucleotides up-stream varies significantly. The two sequences code for a domain of the spike S protein that binds to the host receptor angiotensin-converting enzyme 2 (ACE2) that mediates human infection and in the homologue from Middle East respiratory syndrome coronavirus (MERS-CoV) interacts with sialic acids. We predicted structured regions at the 5 prime and 3 prime ends where our calculations indicate strong propensity to bind proteins. Using the catRAPID method, we computed 3500 protein interactions with the 5 prime end and identified Cyclin T1 CCNT1, ATP-dependent RNA helicase DDX1, Zinc Finger Protein ZNF175 and other 20 high-confidence candidate partners. We propose these proteins, also implicated in HIV replication, to be further investigated for a better understanding of host-virus interaction mechanisms.","AbstractThe beginning of 2020 has seen the emergence of COVID-19 outbreak caused by a novel coronavirus, Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2). There is an imminent need to better understand this new virus and to develop ways to control its spread. In this study, we sought to gain insights for vaccine design against SARS-CoV-2 by considering the high genetic similarity between SARS-CoV-2 and SARS-CoV, which caused the outbreak in 2003, and leveraging existing immunological studies of SARS-CoV. By screening the experimentally-determined SARS-CoV-derived B cell and T cell epitopes in the immunogenic structural proteins of SARS-CoV, we identified a set of B cell and T cell epitopes derived from the spike (S) and nucleocapsid (N) proteins that map identically to SARS-CoV-2 proteins. As no mutation has been observed in these identified epitopes among the available SARS-CoV-2 sequences (as of 9 February 2020), immune targeting of these epitopes may potentially offer protection against this novel virus. For the T cell epitopes, we performed a population coverage analysis of the associated MHC alleles and proposed a set of epitopes that is estimated to provide broad coverage globally, as well as in China. Our findings provide a screened set of epitopes that can help guide experimental efforts towards the development of vaccines against SARS-CoV-2.","In 2012, the genome of a novel rhabdovirus, Bas-Congo virus, was discovered in the acute-phase serum of a Congolese patient with presumed viral hemorrhagic fever. In the absence of a replicating virus isolate, fulfilling Koch's postulates to determine whether Bas-Congo virus is indeed a human virus and/or pathogen has been impossible. However, experiments with vesiculoviral particles pseudotyped with Bas-Congo glycoprotein suggested that Bas-Congo virus particles can enter cells from multiple animals, including humans. In 2015, genomes of two related viruses, Ekpoma virus 1 and Ekpoma virus 2, were detected in human sera in Nigeria. Isolates could not be obtained. Phylogenetic analyses led to the classification of Bas-Congo virus, Ekpoma virus 1, and Ekpoma virus 2 in the same genus, Tibrovirus, together with five biting midge-borne rhabdoviruses (i.e., Beatrice Hill virus, Bivens Arm virus, Coastal Plains virus, Sweetwater Branch virus, and Tibrogargan virus) not known to infect humans. Using individual recombinant vesiculoviruses expressing the glycoproteins of all eight known tibroviruses and more than 75 cell lines representing different animal species, we demonstrate that the glycoproteins of all tibroviruses can mediate vesiculovirus particle entry into human, bat, nonhuman primate, cotton rat, boa constrictor, and Asian tiger mosquito cells. Using four of five isolated authentic tibroviruses (i.e., Bivens Arm virus, Coastal Plains virus, Sweetwater Branch virus, and Tibrogargan virus), our experiments indicate that many cell types may be partially resistant to tibrovirus replication after virion cell entry. Consequently, experimental data solely obtained from experiments using tibrovirus surrogate systems (e.g., vesiculoviral pseudotypes, recombinant vesiculoviruses) cannot be used to predict whether Bas-Congo virus, or any other tibrovirus, infects humans.","SummaryThe flavivirus Zika virus (ZIKV) activates ribonuclease L (RNase L) catalytic antiviral function during infection, yet deletion of RNase L decreases ZIKV production, suggesting a proviral role of RNase L. In this study, we reveal that latent RNase L supports ZIKV replication factory (RF) assembly. Deletion of RNase L induced broader cellular distribution of ZIKV dsRNA and NS3 compared with densely concentrated RFs detected in WT cells. An inactive form of RNase L was sufficient to contain ZIKV genome and dsRNA within a smaller area, which increased levels of viral RNA within RFs as well as infectious ZIKV released from the cell. We used a microtubule stabilization drug to demonstrate that RNase L deletion impaired the cytoskeleton rearrangements that are required for proper generation of RFs. During infection with dengue or West Nile Kunjin viruses, RNase L decreased virus production, suggesting that RNase L proviral function is specific to ZIKV.","AbstractGuided by the principle of primum non nocere (first do no harm), we report a cautionary note on the potential fatal toxicity of chloroquine (CQ) or hydroxychloroquine (HCQ) in combination with anti-diabetic drug metformin. We observed that the combination of CQ or HCQ and metformin, which were used in our studies as potential anti-cancer drugs, killed 30-40% of mice. While our observations in mice may not translate to toxicity in humans, the reports that CQ or HCQ has anti-COVID-19 activity, the use of CQ resulting in toxicity and at least one death, and the recent Emergency Use Authorization (EUA) for CQ and HCQ by the US Food and Drug Administration (FDA) prompted our report. Here we report the lethality of CQ or HCQ in combination with metformin as a warning of its potential serious clinical toxicity. We hope that our report will be helpful to stimulate pharmacovigilance and monitoring of adverse drug reactions with the use of CQ or HCQ, particularly in combination with metformin.","AbstractZika virus (ZIKV) is a member of the family Flaviviridae. In 2015, ZIKV triggered a large epidemic in Brazil and spread across Latin America. In November of that year, the Brazilian Ministry of Health reported a 20-fold increase in cases of neonatal microcephaly, which corresponds geographically and temporally to the ZIKV outbreak. ZIKV was isolated from the brain tissue of a fetus diagnosed with microcephaly, and recent studies in mice models revealed that ZIKV infection may cause brain defects by influencing brain cell developments. Unfortunately, the mechanisms by which ZIKV alters neurophysiological development remain unknown. MicroRNAs (miRNAs) are small noncoding RNAs that regulate post-transcriptional gene expression by translational repression. In order to gain insight into the possible role of ZIKV-mediated miRNA signaling dysfunction in brain-tissue development, we computationally predicted new miRNAs encoded by the ZIKV genome and their effective hybridization with transcripts from human genes previously shown to be involved in microcephalia. The results of these studies suggest a possible role of these miRNAs on the expression of human genes associated with this disease. Besides, a new ZIKV miRNA was predicted in the 3\u2019stem loop (3\u2019 SL) of the 3\u2019untranslated region (3\u2019UTR) of the ZIKV genome, suggesting the role of the 3\u2019UTR of flaviviruses as a source of miRNAs.","AbstractThe same nucleotide sequence can encode two protein products in different reading frames. Overlapping gene regions encode higher levels of intrinsic structural disorder (ISD) than non-overlapping genes (39% vs. 25% in our viral dataset). This might be because of the intrinsic properties of the genetic code, because one member per pair was recently born de novo in a process that favors high ISD, or because high ISD relieves increased evolutionary constraint imposed by dual-coding. Here we quantify the relative contributions of these three alternative hypotheses. We estimate that the recency of de novo gene birth explains 32% or more of the elevation in ISD in overlapping regions of viral genes. While the two reading frames within a same-strand overlapping gene pair have markedly different ISD tendencies that must be controlled for, their effects cancel out to make no net contribution to ISD. The remaining elevation of ISD in the older members of overlapping gene pairs, presumed due to the need to alleviate evolutionary constraint, was already present prior to the origin of the overlap. Same-strand overlapping gene birth events can occur in two different frames, favoring high ISD either in the ancestral gene or in the novel gene; surprisingly, most de novo gene birth events contained completely within the body of an ancestral gene favor high ISD in the ancestral gene (23 phylogenetically independent events vs. 1). This can be explained by mutation bias favoring the frame with more start codons and fewer stop codons.","AbstractThe virus has caused much mortality and morbidity to humans, and still posed a serious threat to the global public health. The virome with the human-infection potential is far from complete. Novel viruses have been discovered at an unprecedented pace as the rapid development of viral metagenomics. However, there is still a lack of a method for rapidly identifying the virus with the human-infection potential. This study built several machine learning models for discriminating the human-infecting viruses from other viruses based on the frequency of k-mers in the viral genomic sequences. The k-nearest neighbor (KNN) model could predict the human-infecting virus with an accuracy of over 90%. Even for the KNN models built on the contigs as short as 1kb, they performed comparably to those built on the viral genomes, suggesting that the models could be used to identify the human-infecting virus from the viral metagenomic sequences. This work could help for discovery of novel human-infecting virus in metagenomics studies.","AbstractPrecise estimation of genetic substitution patterns is critical for accurate reconstruction of pathogen phylogenies. Few studies of viral evolution account for variations of mutation rate across a single gene. This is especially true when considering evolution of segmented viruses where individual segments are short, encoding for few proteins. However, the structural and functional partitions of these proteins could provide valuable information for more accurate inference of viral evolution, due to the disparate immune selection pressure on different functional domains. Accurately reconstructed evolutionary features on specific functional domains can in turn provide biological information on viral protein and immune targets for vaccine design. In this study we developed and evaluated a structurally informed partitioning scheme that accounts for rate variation among immunogenic head and stalk domains of the surface protein hemagglutinin (HA) of influenza viruses. We evaluated the model fit and performance of four different models - HKY, SRD06 codon, HKY with a structurally informed partitioning scheme, SRD06 with a structurally informed partitioning scheme on pandemic A/H1N1pdm09, seasonal A/H1N1postpdm, A/H3N2, B-Yamagata-like and Victoria-like lineages, and two highly pathogenic avian influenza A viruses H5Nx and H7N9. Results showed that structurally informed partitioning with SRD06 performed better for all datasets with decisively statistical support. Significantly faster nucleotide substitution rates for head domain, compared to stalk domain was observed and may provide insight for stalk derived broadly-reactive vaccine design. Taken together, integrating a functionally informed partitioning scheme based on protein structures of immune targets allows for significant improvement of phylogenetic analysis and providing important biological insights.","AbstractCoronavirus Disease 2019 (COVID-19) and severe acute respiratory syndrome (SARS)-related coronaviruses (e.g., 2019-nCoV and SARS-CoV) are phylogenetically distantly related, but both are capable of infecting human hosts via the same receptor, angiotensin-converting enzyme 2, and cause similar clinical and pathological features, suggesting their phenotypic convergence. Yet, the molecular basis that underlies their phenotypic convergence remains unknown. Here, we used a recently developed molecular phyloecological approach to examine the molecular basis leading to their phenotypic convergence. Our genome-level analyses show that the spike protein, which is responsible for receptor binding, has undergone significant Darwinian selection along the branches related to 2019-nCoV and SARS-CoV. Further examination shows an unusually high proportion of evolutionary convergent amino acid sites in the receptor binding domain (RBD) of the spike protein between COVID-19 and SARS-related CoV clades, leading to the phylogenetic uniting of their RBD protein sequences. In addition to the spike protein, we also find the evolutionary convergence of its partner protein, ORF3a, suggesting their possible co-evolutionary convergence. Our results demonstrate a strong adaptive evolutionary convergence between COVID-19 and SARS-related CoV, possibly facilitating their adaptation to similar or identical receptors. Finally, it should be noted that many observed bat SARS-like CoVs that have an evolutionary convergent RBD sequence with 2019-nCoV and SARS-CoV may be pre-adapted to human host receptor ACE2, and hence would be potential new coronavirus sources to infect humans in the future.","AbstractTraditionally, the emergence of coronaviruses (CoVs) has been attributed to a gain in receptor binding in a new host. Our previous work with SARS-like viruses argued that bats already harbor CoVs with the ability to infect humans without adaptation. These results suggested that additional barriers limit the emergence of zoonotic CoV. In this work, we describe overcoming host restriction of two MERS-like bat CoVs using exogenous protease treatment. We found that the spike protein of PDF2180-CoV, a MERS-like virus found in a Ugandan bat, could mediate infection of Vero and human cells in the presence of exogenous trypsin. We subsequently show that the bat virus spike can mediate infection of human gut cells, but is unable to infect human lung cells. Using receptor-blocking antibodies, we show that infection with the PDF2180 spike does not require MERS-CoV receptor DPP4 and antibodies developed against the MERS spike receptor-binding domain and S2 portion are ineffective in neutralizing the PDF2180 chimera. Finally, we found that addition of exogenous trypsin also rescues replication of HKU5-CoV, a second MERS-like group 2c CoV. Together, these results indicate that proteolytic cleavage of the spike, not receptor binding, is the primary infection barrier for these two group 2c CoVs. Coupled with receptor binding, proteolytic activation offers a new parameter to evaluate emergence potential of CoVs and offer a means to recover previously unrecoverable zoonotic CoV strains.ImportanceOverall, our studies demonstrate that proteolytic cleavage is the primary barrier to infection for a subset of zoonotic coronaviruses. Moving forward, the results argue that both receptor binding and proteolytic cleavage of the spike are critical factors that must be considered for evaluating the emergence potential and risk posed by zoonotic coronaviruses. In addition, the findings also offer a novel means to recover previously uncultivable zoonotic coronavirus strains and argue that other tissues, including the digestive tract, could be a site for future coronavirus emergence events in humans.","AbstractPolyomaviruses (PyVs) are small, double-stranded DNA tumor viruses carried by diverse vertebrates. PyVs have previously been considered highly host restricted in mammalian hosts, with host-switching events thought rare or nonexistent. Prior investigations have revealed short-range host-switching events of PyVs in two different African bat species within the horseshoe bat genus Rhinolophus. Herein, we have conducted a systematic investigation of PyVs in 1,083 archived bat samples collected from five provinces across China, and identified 192 PyVs from 186 bats from 15 host species within 6 families (Rhinolophidae, Vespertilionidae, Hipposideridae, Emballonuridae, Miniopteridae and Pteropodidae) representing 28 newly-described PyVs, indicative of extensive genetic diversity of bat PyVs. Surprisingly, two PyVs were identified in multiple bat species from different families, and another PyV clustered phylogenetically with PyVs carried by bats from a different host family, indicative of three inter-family PyV host-switching events. The time to most recent common ancestor (tMRCA) of the three events was estimated at 0.02-11.6 million years ago (MYA), which is inconsistent with the estimated tMRCA of their respective bat hosts (36.3-66.7 MYA), and is most parsimoniously explained by host-switching events. PyVs identified from geographically separated Chinese horseshoe bat species in the present study showed close genetic identities, and clustered with each other and with PyVs from African horseshoe bats, allowing assessment of the effects of positive selection in VP1 within the horseshoe bat family Rhinolophidae. Correlation analysis indicated that co-evolution with their hosts contributed much more to evolutionary divergence of PyV than geographic distance. In conclusion, our findings provide the first evidence of inter-family host-switching events of PyV in mammals and challenge the prevailing evolutionary paradigm for strict host restriction of mammalian PyVs.Author summarySince the discovery of murine polyomavirus in the 1950s, polyomaviruses (PyVs) have been considered both genetically stable and highly host-restricted in their mammalian hosts. In this study, we have identified multiple cases of host-switching events of PyVs by large scale surveillance in diverse bat species collected in China. These host-switching events occurred between bat families living in the same colony, indicating that a large population with frequent contacts between different bat species may represent an ecological niche facilitating PyV host-switching. The cases studied involved members of bats from several families, including horseshoe bats, which were previously found to harbor a number of highly virulent viruses to both humans and domestic animals. Our findings have provided evidence that even highly host-specific DNA viruses can transmit between bats of different species and indicate an increased propensity for spillover events involving horseshoe bats. We propose an evolutionary scheme for bat-borne PyVs in which intra-host divergence and host-switching has generated the diverse PyVs in present day bats. This scheme provides a useful model to study the evolution of PyVs in other hosts and, potentially, the modeling of bat zoonoses and the transmission of other DNA viruses in other mammals, including humans.","AbstractThe ongoing Zika epidemic in the Americas has challenged public health surveillance, response, and control systems. Even as the epidemic appears to be near its end in the Americas, it is unclear whether substantial Zika virus transmission may still be ongoing. This issue is exacerbated by large discrepancies in local case reporting and significant delays in detecting outbreaks due to surveillance gaps. To uncover locations with lingering outbreaks in the Americas, we investigated travel-associated Zika cases diagnosed in the United States and Europe to identify signatures of transmission dynamics that were not captured by local reporting. We found that a large and unreported Zika outbreak occurred in Cuba during 2017, a year after peak transmission in neighboring countries, with cases still appearing in 2018. By sequencing Zika virus from infected travelers, we show that the 2017 outbreak in Cuba was sparked by long-lived lineages of Zika virus introduced from multiple places in the Americas a year prior. Our data suggest that while aggressive mosquito control in Cuba may initially have been effective at mitigating Zika virus transmission, in the absence of vaccines, herd immunity, or strong international coordination, such control measures may need to be maintained to be effective. Our study highlights how Zika virus may still be \u2018silently\u2019 spreading in the Americas and provides a framework for more accurately understanding outbreak dynamics.","AbstractUnderstanding and quantifying the transmission of zoonotic pathogens is essential for directing public health responses, especially for pathogens capable of transmission between humans. However, determining a pathogen\u2019s transmission dynamics is complicated by challenges often encountered in zoonotic disease surveillance, including unobserved sources of transmission (both human and zoonotic), limited spatial information, and unknown scope of surveillance. In this work, we present a model-based inference method that addresses these challenges for subcritical zoonotic pathogens using a spatial model with two levels of mixing. After demonstrating the robustness of the method using simulation studies, we apply the new method to a dataset of human monkeypox cases detected during an active surveillance program from 1982-1986 in the Democratic Republic of the Congo (DRC). Our results provide estimates of the reproductive number and spillover rate of monkeypox during this surveillance period and suggest that most human-to-human transmission events occur over distances of 30km or less. Taking advantage of contact-tracing data available for a subset of monkeypox cases, we find that around 80% of contact-traced links could be correctly recovered from transmission trees inferred using only date and location. Our results highlight the importance of identifying the appropriate spatial scale of transmission, and show how even imperfect spatiotemporal data can be incorporated into models to obtain reliable estimates of human-to-human transmission patterns.Author SummarySurveillance datasets are often the only sources of information about the ecology and epidemiology of zoonotic infectious diseases. Methods that can extract as much information as possible from these datasets therefore provide a key advantage for informing our understanding of the disease dynamics and improving our ability to choose the optimal intervention strategy. We developed and tested a likelihood-based inference method based on a mechanistic model of the spillover and human-to-human transmission processes. We first used simulated datasets to explore which information about the disease dynamics of a subcritical zoonotic pathogen could be successfully extracted from a line-list surveillance dataset with non-localized spatial information and unknown geographic coverage. We then applied the method to a dataset of human monkeypox cases detected during an active surveillance program in the Democratic Republic of the Congo between 1982 and 1986 to obtain estimates of the reproductive number, spillover rate, and spatial dispersal of monkeypox in humans.","AbstractNoncanonical nucleic acid structures play important roles in the regulation of molecular processes. Considering the importance of the ongoing coronavirus crisis, we decided to evaluate genomes of all coronaviruses sequenced to date (stated more broadly, the order Nidovirales) to determine if they contain noncanonical nucleic acid structures. We discovered much evidence of putative G-quadruplex sites and even much more of inverted repeats (IRs) loci, which in fact are ubiquitous along the whole genomic sequence and indicate a possible mechanism for genomic RNA packaging. The most notable enrichment of IRs was found inside 5\u2032UTR for IRs of size 12+ nucleotides, and the most notable enrichment of putative quadruplex sites (PQSs) was located before 3\u2032UTR, inside 5\u2032UTR, and before mRNA. This indicates crucial regulatory roles for both IRs and PQSs. Moreover, we found multiple G-quadruplex binding motifs in human proteins having potential for binding of SARS-CoV-2 RNA. Noncanonical nucleic acids structures in Nidovirales and in novel SARS-CoV-2 are therefore promising druggable structures that can be targeted and utilized in the future.","Although single cell RNA sequencing studies have begun providing compendia of cell expression profiles, it has proven more difficult to systematically identify and localize all molecular cell types in individual organs to create a full molecular cell atlas. Here we describe droplet- and plate-based single cell RNA sequencing applied to ~75,000 human lung and blood cells, combined with a multi-pronged cell annotation approach, which have allowed us to define the gene expression profiles and anatomical locations of 58 cell populations in the human lung, including 41 of 45 previously known cell types or subtypes and 14 new ones. This comprehensive molecular atlas elucidates the biochemical functions of lung cell types and the cell-selective transcription factors and optimal markers for making and monitoring them; defines the cell targets of circulating hormones and predicts local signaling interactions including sources and targets of chemokines in immune cell trafficking and expression changes on lung homing; and identifies the cell types directly affected by lung disease genes and respiratory viruses. Comparison to mouse identified 17 molecular types that appear to have been gained or lost during lung evolution and others whose expression profiles have been substantially altered, revealing extensive plasticity of cell types and cell-type-specific gene expression during organ evolution including expression switches between cell types. This atlas provides the molecular foundation for investigating how lung cell identities, functions, and interactions are achieved in development and tissue engineering and altered in disease and evolution.","ABSTRACTPleiotropy, the concept that a single gene controls multiple distinct traits, is prevalent in most organisms and has broad implications for medicine and agriculture. Identifying the molecular mechanisms underlying pleiotropy has the power to unveil previously unknown biological connections between seemingly unrelated traits. Additionally, the discovery of pleiotropic genes increases our understanding of both genetic and phenotypic complexity by characterizing novel gene functions. Quantitative trait locus (QTL) mapping has been used to identify several pleiotropic regions in many organisms. However, gene knockout studies are needed to eliminate the possibility of tightly linked, non-pleiotropic loci. Here, we use a panel of 296 recombinant inbred advanced intercross lines of Caenorhabditis elegans and a high-throughput fitness assay to identify a single large-effect QTL on the center of chromosome V associated with variation in responses to eight chemotherapeutics. We validate this QTL with near-isogenic lines and pair genome-wide gene expression data with drug response traits to perform mediation analysis, leading to the identification of a pleiotropic candidate gene, scb-1. Using deletion strains created by genome editing, we show that scb-1, which was previously implicated in response to bleomycin, also underlies responses to other double-strand DNA break-inducing chemotherapeutics. This finding provides new evidence for the role of scb-1 in the nematode drug response and highlights the power of mediation analysis to identify causal genes.","AbstractIntroductionA novel severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) was recently identified as the pathogen responsible for the COVID-19 outbreak. SARS-CoV-2 triggers severe pneumonia, which leads to acute respiratory distress syndrome and death in severe cases. As reported, SARS-CoV-2 is 80% genetically identical to the 2003 SARS-CoV virus. Angiotensin-converting enzyme 2 (ACE2) has been identified as the main receptor for entry of both SARS-CoV and SARS-CoV-2 into human cells. ACE2 is normally expressed in cardiovascular and lung type II alveolar epithelial cells, where it positively modulates the RAS system that regulates blood flow, pressure, and fluid homeostasis. Thus, virus-induced reduction of ACE2 gene expression is considered to make a significant contribution to severe acute respiratory failure. Chromatin remodelling plays a significant role in the regulation of ACE2 gene expression and the activity of regulatory elements within the genome.MethodsHere, we integrated data on physical chromatin interactions within the genome organization (captured by Hi-C) with tissue-specific gene expression data to identify spatial expression quantitative trait loci (eQTLs) and thus regulatory elements located within the ACE2 gene.ResultsWe identified regulatory elements within ACE2 that control the expression of PIR, CA5B, and VSP13C in the lung. The gene products of these genes are involved in inflammatory responses, de novo pyrimidine and polyamine synthesis, and the endoplasmic reticulum, respectively.ConclusionOur study, although limited by the fact that the identification of the regulatory interactions is putative until proven by targeted experiments, supports the hypothesis that viral silencing of ACE2 alters the activity of gene regulatory regions and promotes an intra-cellular environment suitable for viral replication.","The ongoing COVID-19 pandemic has already caused devastating losses. Early evidence shows that the exponential spread of COVID-19 can be slowed by restrictive isolation measures, but these place a tremendous burden on society. Moreover, once these restrictions are lifted, the exponential spread is likely to re-emerge. It has been suggested that population-scale testing can help break the cycle of isolation and spread, but current detection methods are not capable of such large-scale processing. Here we propose LAMP-Seq, a barcoded Reverse-Transcription Loop-mediated Isothermal Amplification (RT-LAMP) protocol that could dramatically reduce the cost and complexity of population-scale testing. In this approach, individual samples are processed in a single heat step, producing barcoded amplicons that can be shipped to a sequencing center, pooled, and analyzed en masse. Using unique barcode combinations per sample from a compressed barcode space enables extensive pooling, significantly reducing cost and organizational efforts. Given the low cost and scalability of next-generation sequencing, we believe that this method can be affordably scaled to analyze millions of samples per day using existing sequencing infrastructure.","AbstractDrugs and drug delivery systems have to traverse multiple biological barriers to achieve therapeutic efficacy. In diseases of mucosal-associated tissues such as cystic fibrosis (CF), successful delivery of gene and drug therapies remains a significant challenge due to an abnormally concentrated viscoelastic mucus, which prevents ~99% of all drugs and particles from penetrating the mucus barrier and the underlying epithelia for effective therapy, resulting in decreased survival. We used combinatorial peptide-presenting phage libraries and next-generation sequencing to identify hydrophilic, close to net-neutral charged peptides that penetrate the mucus barrier ex vivo in sputum from CF patients with ~600-fold better penetration than a positively charged control. After mucus penetration, nanoparticles conjugated with our selected peptides successfully translocated into lung epithelial cells derived from CF patients and demonstrated up to three-fold improved cell uptake compared to non-modified carboxylated- and gold standard PEGylated-nanoparticles. The selected peptides act as surface chemistries with synergistic functions to significantly improve the ability of drug delivery systems to overcome the human mucosal barriers and provide efficient cellular internalization. Our screening strategy provides a biologically-based discovery assay that directly addresses transport through mucus and cell barriers and has the potential to advance drug and gene delivery to multiple mucosal barriers.","AbstractThe outbreak of COVID-19 has caused serious epidemic events in China and other countries. With the rapid spread of COVID-19, it is urgent to explore the pathogenesis of this novel coronavirus. However, the foundational research of COVID-19 is very weak. Although angiotensin converting enzyme 2 (ACE2) is the reported receptor of SARS-CoV-2, information about SARS-CoV-2 invading airway epithelial cells is very limited. Based on the analysis of the Human Protein Atlas database, we compared the virus-related receptors of epithelial-derived cells from different organs and found potential key molecules in the local microenvironment for SARS-CoV-2 entering airway epithelial cells. In addition, we found that these proteins were associated with virus reactive proteins in host airway epithelial cells, which may promote the activation of the immune system and the release of inflammatory factors. Our findings provide a new research direction for understanding the potential microenvironment required by SARS-CoV-2 infection in airway epithelial, which may assist in the discovery of potential drug targets against SARS-CoV-2 infection.","AbstractThere is concern about a new coronavirus, the 2019-nCoV, as a global public health threat. In this article, we provide a preliminary evolutionary and molecular epidemiological analysis of this new virus. A phylogenetic tree has been built using the 15 available whole genome sequence of 2019-nCoV and 12 whole genome sequences highly similar sequences available in gene bank (5 from SARS, 2 from MERS and 5 from Bat SARS-like Coronavirus). FUBAR analysis shows that the Nucleocapsid and the Spike Glycoprotein has some sites under positive pressure while homology modelling helped to explain some molecular and structural differences between the viruses. The phylogenetic tree showed that 2019.nCoV significantly clustered with Bat SARS-like Coronavirus sequence isolated in 2015, whereas structural analysis revealed mutation in S and nucleocapsid proteins. From these results, 2019nCoV could be considered a coronavirus distinct from SARS virus, probably transmitted from bats or another host where mutations conferred upon it the ability to infect humans.","Objectives: SARS-CoV-2 whole-genome analysis has identified three large clades spreading worldwide, designated G, V and S. This study aims to analyze the diffusion of SARS-CoV-2 in Spain/Europe. Methods: Maximum likelihood phylogenetic and Bayesian phylodynamic analyses have been performed to estimate the most probable temporal and geographic origin of different phylogenetic clusters and the diffusion pathways of SARS-CoV-2. Results: Phylogenetic analyses of the first 28 SARS-CoV-2 whole genome sequences obtained from patients in Spain revealed that most of them are distributed in G and S clades (13 sequences in each) with the remaining two sequences branching in the V clade. Eleven of the Spanish viruses of the S clade and six of the G clade grouped in two different monophyletic clusters (S-Spain and G-Spain, respectively), with the S-Spain cluster also comprising 8 sequences from 6 other countries from Europe and the Americas. The most recent common ancestor (MRCA) of the SARS-CoV-2 pandemic was estimated in the city of Wuhan, China, around November 24, 2019, with a 95% highest posterior density (HPD) interval from October 30-December 17, 2019. The origin of S-Spain and G-Spain clusters were estimated in Spain around February 14 and 18, 2020, respectively, with a possible ancestry of S-Spain in Shanghai. Conclusions: Multiple SARS-CoV-2 introductions have been detected in Spain and at least two resulted in the emergence of locally transmitted clusters, with further dissemination of one of them to at least 6 other countries. These results highlight the extraordinary potential of SARS-CoV-2 for rapid and widespread geographic dissemination.","AbstractCOVID-19 is prevalent in the elderly. Old individuals are more likely to develop pneumonia and respiratory failure due to alveolar damage, suggesting that lung senescence may increase the susceptibility to SARS-CoV-2 infection and replication. Considering that human coronavirus (HCoVs; SARS-CoV-2 and SARS-CoV) require host cellular factors for infection and replication, we analyzed Genotype-Tissue Expression (GTEx) data to test whether lung aging is associated with transcriptional changes in human protein-coding genes that potentially interact with these viruses. We found decreased expression of the gene tribbles homolog 3 (TRIB3) during aging in male individuals, and its protein was predicted to interact with HCoVs nucleocapsid protein and RNA-dependent RNA polymerase. Using publicly available lung single-cell data, we found TRIB3 expressed mainly in alveolar epithelial cells that express SARS-CoV-2 receptor ACE2. Functional enrichment analysis of age-related genes, in common with SARS-CoV-induced perturbations, revealed genes associated with the mitotic cell cycle and surfactant metabolism. Given that TRIB3 was previously reported to decrease virus infection and replication, the decreased expression of TRIB3 in aged lungs may help explain why older male patients are related to more severe cases of the COVID-19. Thus, drugs that stimulate TRIB3 expression should be evaluated as a potential therapy for the disease.","Evolution couples differences in ambient pH to biological function through protonatable groups, in particular those that switch from buried to exposed and alter protonation state in doing so. We present a tool focusing on structure-based discovery and display of these groups. Since prediction of buried group pKas is computationally intensive, solvent accessibility of ionisable groups is displayed, from which the user can iteratively select pKa calculation centers. Results are color-coded, with emphasis on buried groups. Utility is demonstrated with coronaviruses, which exhibit variable dependence on the acidic pH of the endocytotic pathway. After benchmarking with variants of murine hepatitis virus, a pair of conserved histidine residues are identified that are predicted to be electrostatically frustrated at acidic pH in a common structural core of pre- and post-fusion coronavirus spike proteins. We suggest that an intermediate expanded conformation at endosomal pH could relax the frustration, allowing histidine protonation, and facilitating conformational conversion. This tool is available at http://www.protein-sol.manchester.ac.uk/pka/.","AbstractIn numerous instances, tracking the biological significance of a nucleic acid sequence can be augmented through the identification of environmental niches in which the sequence of interest is present. Many metagenomic datasets are now available, with deep sequencing of samples from diverse biological niches. While any individual metagenomic dataset can be readily queried using web-based tools, meta-searches through all such datasets are less accessible. In this brief communication, we demonstrate such a meta-meta-genomic approach, examining close matches to the Wuhan coronavirus 2019-nCoV in all high-throughput sequencing datasets in the NCBI Sequence Read Archive accessible with the keyword \u201cvirome\u201d. In addition to the homology to bat coronaviruses observed in descriptions of the 2019-nCoV sequence (F. Wu et al. 2020, Nature, doi.org/10.1038/s41586-020-2008-3; P. Zhou et al. 2020, Nature, doi.org/10.1038/s41586-020-2012-7), we note a strong homology to numerous sequence reads in a metavirome dataset generated from the lungs of deceased Pangolins reported by Liu et al. (Viruses 11:11, 2019, http://doi.org/10.3390/v11110979). Our observations are relevant to discussions of the derivation of 2019-nCoV and illustrate the utility and limitations of meta-metagenomic search tools in effective and rapid characterization of potentially significant nucleic acid sequences.ImportanceMeta-metagenomic searches allow for high-speed, low-cost identification of potentially significant biological niches for sequences of interest.","AbstractHeterogeneity in disease transmission is widespread and, when not accounted for, can produce unpredictable outbreaks of infectious disease. Despite this, precisely how different sources of variation in host traits drive heterogeneity in disease transmission is poorly understood. Here we dissected the sources of variation in pathogen transmission using Drosophila melanogaster and Drosophila C Virus as a host-pathogen model system. We found that infected lifespan, viral growth, virus shedding, and viral load at death were all significantly influenced by fly genetic background, sex and female mating status. To understand how variation in each of these traits may generate heterogeneity in disease transmission, we estimated individual transmission potential by integrating data on virus shedding and lifespan alongside previously collected data on social aggregation. We found that \u223c15% of between-individual heterogeneity in disease transmission was explained by a significant interaction between genetic and sex-specific variation. We also characterised the amount of variation in viral load, virus shedding, and lifespan following infection that could be explained by genetic background and sex. Amongst the determinants of individual variation in disease transmission these sources of host variation play roles of varying importance, with genetic background generally playing the largest role. Our results highlight the importance of characterising sources of variation in multiple host traits when studying disease transmission at the individual-level.","In December 2019, an outbreak of atypical pneumonia (Coronavirus disease 2019 -COVID-19) associated with a novel coronavirus (SARS-CoV-2) was reported in Wuhan city, Hubei province, China. The outbreak was traced to a seafood wholesale market and human to human transmission was confirmed. The rapid spread and the death toll of the new epidemic warrants immediate intervention. The intra-host genomic variability of SARS-CoV-2 plays a pivotal role in the development of effective antiviral agents and vaccines, but also in the design of accurate diagnostics. We analyzed NGS data derived from clinical samples of three Chinese patients infected with SARS-CoV-2, in order to identify small- and large-scale intra-host variations in the viral genome. We identified tens of low- or higher-frequency single nucleotide variations (SNVs) with variable density across the viral genome, affecting 7 out of 10 protein-coding viral genes. The majority of these SNVs corresponded to missense changes. The annotation of the identified SNVs but also of all currently circulating strain variations revealed colocalization of intra-host but also strain specific SNVs with primers and probes currently used in molecular diagnostics assays. Moreover, we de-novo assembled the viral genome, in order to isolate and validate intra-host structural variations and recombination breakpoints. The bioinformatics analysis disclosed genomic rearrangements over poly-A / poly-U regions located in ORF1ab and spike (S) gene, including a potential recombination hot-spot within S gene. Our results highlight the intra-host genomic diversity and plasticity of SARS-CoV-2, pointing out genomic regions that are prone to alterations. The isolated SNVs and genomic rearrangements, reflect the intra-patient capacity of the polymorphic quasispecies, which may arise rapidly during the outbreak, allowing immunological escape of the virus, offering resistance to anti-viral drugs and affecting the sensitivity of the molecular diagnostics assays.","AbstractViral detection is critical for controlling disease spread and progression. Recent emerging threats including the Zika and Ebola virus outbreaks highlight the cost and difficulty in responding rapidly. In low-resource areas, a key obstacle is quick and accurate detection of viruses near the point of care. To address these challenges, we develop a platform for low-cost and rapid detection of viral RNA with DNA nanoswitches designed to mechanically reconfigure in response to specific viruses. Using Zika virus as a model system, we show non-enzymatic detection of viral RNA to the attomole level, with selective and multiplexed detection between related viruses and viral strains. For clinical-level sensitivity in biological fluids, we paired the assay with a sample preparation step using either RNA extraction or isothermal pre-amplification. Our assay can be performed with minimal or no lab infrastructure, and is readily adaptable (with \u223c24-hour development time) to detect other viruses. Given this versatility, we expect that further development and field implementation will improve our ability to detect emergent viral threats and ultimately limit their impact.","In December 2019, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) emerged in Wuhan, Hubei Province, China. No specific treatment has been established against coronavirus disease-2019 (COVID-19) so far. Therefore, it is urgently needed to identify effective antiviral agents for the treatment of this disease, and several approved drugs such as lopinavir have been evaluated. Here, we report that nelfinavir, an HIV-1 protease inhibitor, potently inhibited replication of SARS-CoV-2. The effective concentrations for 50% and 90% inhibition (EC50 and EC90) of nelfinavir were 1.13 micro M and 1.76 micro M respectively, the lowest of the nine HIV-1 protease inhibitors including lopinavir. The trough and peak serum concentrations of nelfinavir were three to six times higher than EC50 of this drug. These results suggest that nelfinavir is a potential candidate drug for the treatment of COVID-19 and should be assessed in patients with COVID-19.","AbstractHuman infection with Middle East Respiratory Syndrome Coronavirus (MERS-CoV) is driven by recurring dromedary-to-human spill-over events, leading decision-makers to consider dromedary vaccination. Dromedary vaccine candidates in the development pipeline are showing hopeful results, but gaps in our understanding of the epidemiology of MERS-CoV in dromedaries must be addressed to design and evaluate potential vaccination strategies. We systematically reviewed the published literature reporting seroprevalence and/or prevalence of active MERS-CoV infection in dromedary populations from both cross-sectional and longitudinal studies, including 60 studies in our qualitative syntheses. MERS-CoV seroprevalence increased with age up to 80-100% in adult dromedaries supporting geographically wide spread endemicity of MERS-CoV in dromedaries in both the Arabian Peninsula and countries exporting dromedaries from Africa. The high prevalence of active infection measured in juveniles and at sites where dromedary populations mix should guide further investigation \u2013 particularly of dromedary movement \u2013 and inform vaccination strategy design.","AbstractTo have epidemic potential, a pathogen must be able to spread in human populations, but of human-infective RNA viruses only a minority can do so. We investigated the evolution of human transmissibility through parallel analyses of 1755 virus genome sequences from 39 RNA virus genera. We identified 57 lineages containing human-transmissible species and estimated that at least 74% of these lineages have evolved directly from non-human viruses in other mammals or birds, a public health threat recently designated \u201cDisease X\u201d. Human-transmissible viruses rarely evolve from virus lineages that can infect but not transmit between humans. This result cautions against focussing surveillance and mitigation efforts narrowly on currently known human-infective virus lineages and supports calls for a better understanding of RNA virus diversity in non-human hosts.","AbstractPreviously, the cyclophilin inhibitors cyclosporin A (CsA) and Alisporivir (ALV) were shown to inhibit the replication of diverse RNA viruses, including arteriviruses and coronaviruses, which both belong to the orderNidovirales. Here we aimed to identify arterivirus proteins involved in the mode-of-action of cyclophilin inhibitors and to investigate how these compounds inhibit arterivirus RNA synthesis in the infected cell. Repeated passaging of the arterivirus prototype equine arteritis virus (EAV) in the presence of CsA revealed that reduced drug sensitivity is associated with the emergence of adaptive mutations in nonstructural protein 5 (nsp5), one of the transmembrane subunits of the arterivirus replicase polyprotein. Introduction of singular nsp5 mutations (nsp5 Q21R, Y113H, or A134V) led to a \u223c2-fold decrease in sensitivity to CsA treatment, whereas combinations of mutations further increased EAV\u2019s CsA resistance. The detailed experimental characterization of engineered EAV mutants harboring CsA-resistance mutations implicated nsp5 in arterivirus RNA synthesis. Particularly, in anin vitroassay, EAV RNA synthesis was far less sensitive to CsA treatment when nsp5 contained the adaptive mutations mentioned above. Interestingly, for increased sensitivity to the closely-related drug ALV CsA-resistant nsp5 mutants required the incorporation of an additional adaptive mutation, which resided in nsp2 (H114R), another transmembrane subunit of the arterivirus replicase. Our study provides the first evidence for the involvement of nsp2 and nsp5 in the mechanism underlying the inhibition of arterivirus replication by cyclophilin inhibitors.ImportanceCurrently, no approved treatments are available to combat infections with nidoviruses, a group of plus-stranded RNA viruses including important zoonotic and veterinary pathogens. Previously, the cyclophilin inhibitors cyclosporin A (CsA) and Alisporivir (ALV) were shown to inhibit the replication of diverse nidoviruses (both arteriviruses and coronaviruses), and may thus represent a class of pan-nidovirus inhibitors. Here, using the arterivirus prototype equine arteritis virus, we have established that resistance to CsA and ALV treatment is associated with adaptive mutations in two trans-membrane subunits of the viral replication complex, nonstructural proteins 2 and 5. This is the first evidence for the involvement of specific replicase subunits of nidoviruses in the mechanism underlying the inhibition of their replication by cyclophilin inhibitors. Understanding this mechanism of action is of major importance to guide future drug design, both for nidoviruses and other RNA viruses inhibited by these compounds.","AbstractAgeing results in a decline in immune function. We showed previously that healthy older humans (&gt;65 years old) have reduced antigen-specific cutaneous immunity to varicella zoster virus (VZV) antigen challenge. This was associated with p38 MAP kinase driven inflammation that was induced by mild tissue injury caused by the injection of the antigen itself. Here we show that non-specific injury induced by injection of air or saline into the skin of older adults recruits CCR2+CD14+ monocytes by CCL2 produced by senescent fibroblasts. These monocytes reduced TRM proliferation via secretion of prostaglandin E2 (PGE2). Pre-treatment with a p38-MAPK inhibitor (Losmapimod) in older adults in vivo significantly decreased CCL2 expression, recruitment of monocyte into the skin, COX2 expression and PGE2 production. This enhanced the VZV response in the skin. Therefore, local inflammation arising from interaction between senescent cells and monocytes leads to immune decline in the skin during ageing, a process that can be reversed.SummaryInflammation resulting from tissue injury blocks antigen-specific cutaneous immunity during ageing. Monocytes recruited to the skin inhibit TRM function through COX2-derived prostaglandin E2 production. Blocking inflammation and resulting prostaglandin E2 production with a p38-MAP kinase inhibitor significantly enhances cutaneous antigen-specific responses.","N4-hydroxycytidine (NHC) has been recently reported to have promising antiviral activity against SARS-CoV-2. To join worldwide efforts in identifying potential drug targets against this pandemic, the target landscape of NHC was defined by extracting all known targets of its chemical neighborhood, including drugs, analogues, and metabolites, and by performing target predictions from two independent platforms, following the recent Public Health Assessment via Structural Evaluation (PHASE) protocol. The analysis provides a list of over 30 protein targets that could be useful in future design activities of new COVID-19 antivirals. The relevance for existing drugs within the same chemical space, such as remdesivir, is also discussed.","Predicting infectious disease dynamics is a central challenge in disease ecology. Models that can assess which individuals are most at risk of being exposed to a pathogen not only provide valuable insights into disease transmission and dynamics but can also guide management interventions. Constructing such models for wild animal populations, however, is particularly challenging; often only serological data is available on a subset of individuals and non-linear relationships between variables are common. Here we take advantage of the latest advances in statistical machine learning to construct pathogen-risk models that automatically incorporate complex non-linear relationships with minimal statistical assumptions from ecological data with missing values. Our approach compares multiple machine learning algorithms in a unified environment to find the model with the best predictive performance and uses game theory to better interpret results. We apply this framework on two major pathogens that infect African lions: canine distemper virus (CDV) and feline parvovirus. Our modelling approach provided enhanced predictive performance compared to more traditional approaches, as well as new insights into disease risks in a wild population. We were able to efficiently capture and visualise strong non-linear patterns, as well as model complex interactions between variables in shaping exposure risk from CDV and feline parvovirus. For example, we found that lions were more likely to be exposed to CDV at a young age but only in low rainfall years. When combined with our data calibration approach, our framework helped us to answer questions about risk of pathogen exposure which are difficult to address with previous methods. Our framework not only has the potential to aid in predicting disease risk in animal populations, but also can be used to build robust predictive models suitable for other ecological applications such as modelling species distribution or diversity patterns.","AbstractNutritional stress, and especially a dearth of pollen, is considered an important factor associated with honey bee colony losses. We used pollen-restricted colonies as a model to study the nutritional stress conditions experienced in colonies within intensively cultivated agricultural areas. This model was complemented by the establishment of an experimental design, which allowed us to uncouple the effect of nutrition, behavior and age in colonies of similar size and demography. We used this system to determine the effect of pollen restriction on workers\u2019 behavioral development. Then, we analyzed the effect of nutritional stress, behavior and age on the expression of key physiological genes involved in the regulation of division of labor. Finally, we analyzed the effects of these variables on the expression of immune genes and the titers of honey bee viruses. Our results show that pollen restriction led to an increased number of precocious foragers and this behavioral transition was associated with important changes in the expression of nutritionally regulated physiological genes, immunity and viral titers. Vitellogenin (vg) and major royal jelly protein 1 (mrjp1) were the most predictive markers of nutrition and behavior. The expression of immune genes was primarily affected by behavior, with higher levels in foragers. Deformed wing virus (DWV) titers were significantly affected by behavior and nutritional status, with higher titer in foragers and increased levels associated with pollen ingestion. Correlation analyses support the predominant effect of behavior on immunity and susceptibility to viral infection, revealing that both immune genes and DWV exhibited strong negative correlations with genes associated with nursing, but positive correlations with genes associated with foraging. Our results provide valuable insights into the physiological mechanisms by which nutritional stress induce precocious foraging and increased susceptibility to viral infections.","Viruses interact with hundreds to thousands of proteins in mammals, yet adaptation against viruses has only been studied in a few proteins specialized in antiviral defense. Whether adaptation to viruses typically involves only specialized antiviral proteins or affects a broad array of proteins is unknown. Here, we analyze adaptation in ~1,300 virus-interacting proteins manually curated from a set of 9,900 proteins conserved across mammals. We show that viruses (i) use the more evolutionarily constrained proteins from the cellular functions they hijack and that (ii) despite this high constraint, virus-interacting proteins account for a high proportion of all protein adaptation in humans and other mammals. Adaptation is elevated in virus-interacting proteins across all functional categories, including both immune and non-immune functions. Our results demonstrate that viruses are one of the most dominant drivers of evolutionary change across mammalian and human proteomes.","AbstractPARP2 belongs to a family of proteins involved in cell differentiation, DNA damage repair, cellular energy expenditure, chromatin modeling and cell differentiation. In addition to these overlapping functions with PARP1, PARP2 participates in spermatogenesis, T-cell maturation, extraembryonic endoderm formation and adipogenesis. The function(s) of PARP2 is far from complete, and the mechanism(s) by which the gene and protein are regulated are unknown. In this study, we found that two different mechanisms are used in vitro to regulate PARP2 levels. In the presence of serum, PARP2 is degraded through the ubiquitin-proteasome pathway, however, when serum is removed, PARP2 is rapidly sequestered into an SDS- and urea-insoluble fraction. This sequestration is relieved by serum in a dose-dependent manner, and again PARP2 is detected by immunoblotting. Furthermore, and despite the presence of a putative serum response element in the PARP2 gene, transcription is not affected by serum deprivation. These observations that PARP2 is tightly regulated by distinct pathways highlights the critical roles PARP2 plays under different physiological conditions.","In recent years nidoviruses have emerged as an important respiratory pathogen of reptiles, affecting especially captive python populations. In pythons, nidovirus infection induces an inflammation of the upper respiratory and alimentary tract which can develop into a severe and often fatal proliferative pneumonia. We observed pyogranulomatous and fibrinonecrotic lesions in organ systems other than the respiratory tract during full post mortem examinations on 30 nidovirus RT-PCR positive pythons of varying species originating from Switzerland and Spain. The observations prompted us to study whether the atypical tissue tropism associates with previously unknown nidoviruses or changes in the nidovirus genome. RT-PCR and inoculation of Morelia viridis cell cultures served to recruit the cases and to obtain virus isolates. Immunohistochemistry and immunofluorescence staining against nidovirus nucleoprotein demonstrated that the virus not only infects a broad spectrum of epithelial (respiratory and alimentary epithelium, hepatocytes, renal tubules, pancreatic ducts etc.), but also intravascular monocytes, intralesional macrophages and endothelial cells. By next-generation sequencing we obtained full length genome for a novel nidovirus species circulating in Switzerland. Analysis of viral genomes recovered from pythons showing nidovirus infection-associated respiratory or systemic disease did not explain the observed phenotypes. The results indicate that python nidoviruses have a broad cell and tissue tropism, further suggesting that the course of infection could vary and involve lesions in a broad spectrum of tissues and organ systems as a consequence of monocyte-mediated systemic spread of the virus.","AbstractNovel infectious diseases continue to emerge within human populations. Predictive studies have begun to identify pathogen traits associated with emergence. However, emerging pathogens vary widely in virulence, a key determinant of their ultimate risk to public health. Here, we use structured literature searches to review the virulence of each of the 214 known human-infective RNA virus species. We then use a machine learning framework to determine whether viral virulence can be predicted by ecological traits including human-to-human transmissibility, transmission routes, tissue tropisms and host range. Using severity of clinical disease as a measurement of virulence, we identified potential risk factors using predictive classification tree and random forest ensemble models. The random forest model predicted literature-assigned disease severity of test data with 90.3% accuracy, compared to a null accuracy of 74.2%. In addition to viral taxonomy, the ability to cause systemic infection, having renal and/or neural tropism, direct contact or respiratory transmission, and limited (0 &lt; R0 \u2264 1) human-to-human transmissibility were the strongest predictors of severe disease. We present a novel, comparative perspective on the virulence of all currently known human RNA virus species. The risk factors identified may provide novel perspectives in understanding the evolution of virulence and elucidating molecular virulence mechanisms. These risk factors could also improve planning and preparedness in public health strategies as part of a predictive framework for novel human infections.Author SummaryNewly emerging infectious diseases present potentially serious threats to global health. Although studies have begun to identify pathogen traits associated with the emergence of new human diseases, these do not address why emerging infections vary in the severity of disease they cause, often termed \u2018virulence\u2019. We test whether ecological traits of human viruses can act as predictors of virulence, as suggested by theoretical studies. We conduct the first systematic review of virulence across all currently known human RNA virus species. We adopt a machine learning approach by constructing a random forest, a model that aims to optimally predict an outcome using a specific structure of predictors. Predictions matched literature-assigned ratings for 28 of 31 test set viruses. Our predictive model suggests that higher virulence is associated with infection of multiple organ systems, nervous systems or the renal systems. Higher virulence was also associated with contact-based or airborne transmission, and limited capability to transmit between humans. These risk factors may provide novel starting points for questioning why virulence should evolve and identifying causative mechanisms of virulence. In addition, our work could suggest priority targets for infectious disease surveillance and future public health risk strategies.BlurbComparative analysis using machine learning shows specificity of tissue tropism and transmission biology can act as predictive risk factors for virulence of human RNA viruses.","Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), the cause of human coronavirus disease 2019 (COVID-19), emerged in Wuhan, China in December 2019. The virus rapidly spread globally, resulting in a public-health crisis including more than one million cases and tens of thousands of deaths. Here, we describe the identification and evaluation of commercially available reagents and assays for the molecular detection of SARS-CoV-2 in infected formalin fixed paraffin embedded (FFPE) cell pellets. We identified a suitable rabbit polyclonal anti-SARS-CoV spike protein antibody and a mouse monoclonal anti-SARS-CoV nucleocapsid protein (NP) antibody for cross detection of the respective SARS-CoV-2 proteins by immunohistochemistry (IHC) and immunofluorescence assay (IFA). Next, we established RNAscope in situ hybridization (ISH) to detect SARS-CoV-2 RNA. Furthermore, we established a multiplex fluorescence ISH (mFISH) to detect positive-sense SARS-CoV-2 RNA and negative-sense SARS-CoV-2 RNA (a replicative intermediate indicating viral replication). Finally, we developed a dual staining assay using IHC and ISH to detect SARS-CoV-2 antigen and RNA in the same FFPE section. These reagents and assays will accelerate COVID-19 pathogenesis studies in humans and in COVID-19 animal models.","AbstractSARS-CoV-2, a novel coronavirus (CoV), has recently emerged causing an ongoing outbreak of viral pneumonia around the world. While genetically distinct from the original SARS-CoV, both group 2B CoVs share similar genome organization and origins to coronaviruses harbored in bats. Importantly, initial guidance has used insights from SARS-CoV infection to inform treatment and public health strategies. In this report, we evaluate type-I Interferon (IFN-I) sensitivity of SARS-CoV-2 relative to the original SARS-CoV. Our results indicate that while SARS-CoV-2 maintains similar viral replication kinetics to SARS-CoV in Vero cell, the novel CoV is much more sensitive to IFN-I pretreatment. Examining transcriptional factor activation and interferon stimulated gene (ISG) induction, SARS-CoV-2 in the context of type I IFN induces phosphorylation of STAT1 and increased ISG proteins. In contrast, the original SARS-CoV has no evidence for STAT1 phosphorylation or ISG protein increases even in the presence of type I IFN pretreatment. Finally, we examined homology between SARS-CoV and SARS-CoV-2 in viral proteins shown to be interferon antagonist. The absence of open reading frame (ORF) 3b and significant changes to ORF6 suggest the two key IFN antagonists may not maintain equivalent function in SARS-CoV-2. Together, the results identify key differences in susceptibility to the IFN-I response between SARS-CoV and SARS-CoV-2. that could help inform disease progression, treatment options, and animal model development.ImportanceWith the ongoing outbreak of COVID-19 disease, differences between the SARS-CoV-2 and the original SARS-CoV could be leveraged to inform disease progression and eventual treatment options. In addition, these findings could have key implications for animal model development as well as further research into how SARS-CoV-2 modulates the type I IFN response early during infection.Article SummarySARS-CoV-2 has similar replication kinetics to SARS-CoV, but demonstrates significant sensitivity to type I interferon treatment.","Abstract The current classification of animal viruses primarily relates to the virus molecular world, the genomic architecture and the corresponding host-cell infection cycle. This virus centered perspective does not make allowance for the precept that virus fitness hinges on the virus transmission success. Virus transmission reflects the infection-shedding-transmission dynamics and, with it, the organ system involvement and other, macroscopic dimensions of the host environment. This study examines the transmission ecology of the world main livestock viruses, 36 in total, belonging to eleven different families, and a mix of RNA, DNA and retroviruses. Viruses are virtually ranked in an outer- to inner-body fashion, based on the shifting organ system involvement and associated infection-shedding-transmission dynamics. As a next step, this ranking is disentangled with the aim to contrast two main host ecologies, poultry plus pig production and ruminant plus equine husbandry, as well as to create a distinction among the RNA, DNA and retroviruses, also ranked in an outer- to inner-body fashion. Spearman correlation reveals the matches among these various virus traits, as pertaining to the two host-ecologies, four infection-shedding-transmission related variables, and the three virus genomes. The collective results reveal the outer- to inner-body shifts in the interplay of host environment, virus-host interactions, and nature of the virus. Two opposing virus evolution pathways emerge, respectively for generalist type, outer-body and for specialist type, inner-body viruses. The ecological virus classification here presented is broadly consistent with the current virus classification system and offers the advantage of bringing substance and cohesion to the interrelationships among viruses and virus families.","AbstractThe novel coronavirus disease first identified in 2019 in Wuhan, China (COVID-19) has become a serious global public health concern. One current issue is the ability to adequately screen for the virus causing COVID-2 (SARS-CoV-2). Here we demonstrate the feasibility of shotgun proteomics as a SARS-CoV-2 screening method, through the detection of viral peptides in proteolytically digested body fluids. Using in silico methods, we generated trypsin-based shotgun proteomics methods optimized for LCMS systems from 5 commercial instrument vendors (Thermo, SCIEX, Waters, Shimadzu, and Agilent). First, we generated protein FASTA files and their protein digest maps. Second, the FASTA files were used to generate spectral libraries based on experimental data. Third, transition lists were derived from the spectral libraries using the vendor neutral and open Skyline software environment. Finally, we identified 17 post-translational modifications using linear motif modeling.Abstract Figure","AbstractProtein therapeutics represent a significant and growing component of the modern pharmacopeia, but their potential to treat human disease is limited because most proteins fail to traffic across biological membranes. Recently, we discovered that cell-permeant miniature proteins (CPMPs) containing a precisely defined, penta-arginine motif traffic readily to the cytosol and nucleus with efficiencies that rival those of hydrocarbon-stapled peptides active in animals and man. Like many cell-penetrating peptides (CPPs), CPMPs enter the endocytic pathway; the difference is that CPMPs are released efficiently from endosomes while other CPPs are not. Here, we seek to understand how CPMPs traffic from endosomes into the cytosol and what factors contribute to the efficiency of endosomal release. First, using two complementary cell-based assays, we exclude endosomal rupture as the primary means of endosomal escape. Next, using a broad spectrum of techniques, including an RNA interference (RNAi) screen, fluorescence correlation spectroscopy (FCS), and confocal imaging, we identify VPS39\u2014a gene encoding a subunit of the homotypic fusion and protein sorting (HOPS) complex\u2014as a critical determinant in the trafficking of CPMPs and hydrocarbon-stapled peptides to the cytosol. Although CPMPs neither inhibit nor activate HOPS function, HOPS activity is essential to efficiently deliver CPMPs to the cytosol. Subsequent multi-color confocal imaging studies identify CPMPs within the endosomal lumen, particularly within the intraluminal vesicles (ILVs) of Rab7+ and Lamp1+ endosomes that are the products of HOPS-mediated fusion. These results suggest that CPMPs require HOPS to reach ILVs\u2014an environment that serves as a prerequisite for efficient endosomal escape.","Zoonotic coronavirus (CoV) infections, like those responsible for the current SARS-CoV-2 epidemic, cause grave international public health concern. In infected cells, the CoV RNA-synthesizing machinery associates with modified endoplasmic reticulum membranes that are transformed into the viral replication organelle (RO). While double-membrane vesicles (DMVs) appear to be a pan -coronavirus RO element, studies to date describe an assortment of additional coronavirus-induced membrane structures. Despite much speculation, it remains unclear which RO element(s) accommodate viral RNA synthesis. Here we provide detailed 2D and 3D analyses of CoV ROs and show that diverse CoVs essentially induce the same membrane modifications, including the small open double-membrane spherules (DMSs) previously thought to be restricted to gamma- and delta-CoV infections and proposed as sites of replication. Metabolic labelling of newly-synthesized viral RNA followed by quantitative EM autoradiography revealed abundant viral RNA synthesis associated with DMVs in cells infected with the beta-CoVs MERS-CoV and SARS-CoV, and the gamma-CoV infectious bronchitis virus. RNA synthesis could not be linked to DMSs or any other cellular or virus-induced structure. Our results provide a unifying model of the CoV RO and clearly establish DMVs as the central hub for viral RNA synthesis and a potential drug target in coronavirus infection.","AbstractUsing a Systems Biology approach, we integrated genomic, transcriptomic, proteomic, and molecular structure information to provide a holistic understanding of the COVID-19 pandemic. The expression data analysis of the Renin Angiotensin System indicates mild nasal, oral or throat infections are likely and that the gastrointestinal tissues are a common primary target of SARS-CoV-2. Extreme symptoms in the lower respiratory system likely result from a secondary-infection possibly by a comorbidity-driven upregulation of ACE2 in the lung. The remarkable differences in expression of other RAS elements, the elimination of macrophages and the activation of cytokines in COVID-19 bronchoalveolar samples suggest that a functional immune deficiency is a critical outcome of COVID-19. We posit that using a non-respiratory system as a major pathway of infection is likely determining the unprecedented global spread of this coronavirus.One Sentence SummaryA Systems Approach Indicates Non-respiratory Pathways of Infection as Key for the COVID-19 Pandemic","AbstractBackgroundThe Middle East respiratory syndrome coronavirus (MERS-CoV) developed infections that caused serious epidemics. Special priority of awareness and prompt initiative involvement of health workers (HW) during such intensified health situation necessitated an assessment of their preparedness, appropriate attitudes and protective strategies for better efficiency modes and acceleration during emergency.MethodsHW of Prince Sultan Military Medical city in Riyadh, Saudi Arabia were reviewed through specifically designed questionnaires to acquire the demanded data. It included clinical and demographic information about the viral diseases, associated signs and symptoms, transmission and protection, and attitudes about the MERS-CoV disease.ResultsThe study was accomplished between August and November, 2017 and 477 participants of the medical city, successfully completed the study questionnaire (Appendix I). Females represented a majority and there was an indirectly proportional decrease with the increasing age. Gradual educational increase levels reflected dominance of the university degree holders. Jobs were dominated by nurses and non-Saudis were a majority whilst, the highly experienced, (&gt;10 years) were a minority. A majority recognized the viral transmission methods, popular information sources of MERS-CoV and associated medical terms. Highest scores were observed in dealing with protective aids and recognizing symptoms of disease. High adherence to hand hygiene protocols and correct washing steps were recorded. Correct and high levels were observed in taking preventive measures and avoiding infection. Participants responded correctly to negative and wrong actions that patients should refrain from. High scores were observed in taking appreciable attitudes towards oneself and towards others.ConclusionsExpatriates were majority and nurses were dominant which, necessitates Saudization of this sector. Ministry of health pamphlets and seminars were of less impact in invigilating HW, hence, more attention and efforts are demanded. HW were quite aware of the basic and emergent health policies during epidemic episodes of MERS-CoV.","SARS-CoV-2, a member of the coronavirus family, is responsible for the current COVID-19 pandemic. Based on our analysis of hepatitis C virus and coronavirus replication, and the molecular structures and activities of viral inhibitors, we previously demonstrated that three nucleotide analogues inhibit the SARS-CoV RNA-dependent RNA polymerase (RdRp). Here, using polymerase extension experiments, we have demonstrated that the active triphosphate form of Sofosbuvir (a key component of the FDA approved hepatitis C drug EPCLUSA), is incorporated by SARS-CoV-2 RdRp, and blocks further incorporation. Using the same molecular insight, we selected the active triphosphate forms of three other anti-viral agents, Alovudine, AZT (an FDA approved HIV/AIDS drug) and Tenofovir alafenamide (TAF, an FDA approved drug for HIV and hepatitis B) for evaluation as inhibitors of SARS-CoV-2 RdRp. We demonstrated the ability of these three viral polymerase inhibitors, 3\u2032-fluoro-3\u2032-deoxythymidine triphosphate, 3\u2032-azido-3\u2032-deoxythymidine triphosphate and Tenofovir diphosphate (the active triphosphate forms of Alovudine, AZT and TAF, respectively) to be incorporated by SARS-CoV-2 RdRp, where they also terminate further polymerase extension. These results offer a strong molecular basis for these nucleotide analogues to be evaluated as potential therapeutics for COVID-19.","ABSTRACTCurrent prophylactic and therapeutic strategies targeting human influenza viruses include vaccines and antivirals. Given variable rates of vaccine efficacy and antiviral resistance, alternative strategies are urgently required to improve disease outcomes. Here we describe the use of HiSeq deep sequencing to analyze host gene expression in primary human alveolar epithelial type II cells infected with highly pathogenic avian influenza H5N1 virus. At 24 hours post-infection, 623 host genes were significantly up-regulated, including the cell adhesion moleculeCEACAM1. The up-regulation ofCEACAM1was blocked in the presence of the reactive oxygen species inhibitor, apocynin. H5N1 virus infection stimulated significantly higher CEACAM1 protein expression when compared to low pathogenic PR8 H1N1 virus, suggesting a key role for CEACAM1 in influenza virus pathogenicity. Furthermore, silencing of endogenousCEACAM1resulted in reduced levels of proinflammatory cytokine/chemokine production, as well as reduced levels of virus replication following H5N1 infection. Our study provides evidence for the involvement of CEACAM1 in a clinically relevant model of H5N1 infection and may assist in the development of host-oriented antiviral strategies.","ABSTRACTSingle-cell RNA profiling of ACE2, the SARS-CoV-2 receptor, had proposed multiple tissue cells as the potential targets of SARS-CoV-2, the novel coronavirus causing the COVID-19 pandemic. However, most were not echoed by the patients\u2019 clinical manifestations, largely due to the lack of protein expression information of ACE2 and co-factors. Here, we incorporated the protein information to analyse the expression of ACE2, together with TMPRSS2 and Furin, two proteases assisting SARS-CoV-2 infection, at single cell level in situ, which we called protein-proofed single-cell RNA (pscRNA) profiling. Systemic analysis across 36 tissues revealed a rank list of candidate cells potentially vulnerable to SARS-CoV-2. The top targets are lung AT2 cells and macrophages, then cardiomyocytes and adrenal gland stromal cells, followed by stromal cells in testis, ovary and thyroid. Whereas, the polarized kidney proximal tubule cells, liver cholangiocytes and intestinal enterocytes are less likely to be the primary SARS-CoV-2 targets as ACE2 localizes at the apical region of cells, where the viruses may not readily reach. These findings are in concert with the clinical characteristics of prominent lung symptoms, frequent heart injury, and uncommon intestinal symptoms and acute kidney injury. Together, we provide a comprehensive view on the potential SARS-CoV-2 targets by pscRNA profiling, and propose that, in addition to acute respiratory distress syndrome, attentions should also be paid to the potential injuries in cardiovascular, endocrine and reproductive systems during the treatment of COVID-19 patients.","To ultimately combat the emerging COVID-19 pandemic, it is desired to develop an effective and safe vaccine against this highly contagious disease caused by the SARS-CoV-2 coronavirus. Our literature and clinical trial survey showed that the whole virus, as well as the spike (S) protein, nucleocapsid (N) protein, and membrane (M) protein, have been tested for vaccine development against SARS and MERS. However, these vaccine candidates might lack the induction of complete protection and have safety concerns. We then applied the Vaxign reverse vaccinology tool and the newly developed Vaxign-ML machine learning tool to predict COVID-19 vaccine candidates. By investigating the entire proteome of SARS-CoV-2, six proteins, including the S protein and five non-structural proteins (nsp3, 3CL-pro, and nsp8-10), were predicted to be adhesins, which are crucial to the viral adhering and host invasion. The S, nsp3, and nsp8 proteins were also predicted by Vaxign-ML to induce high protective antigenicity. Besides the commonly used S protein, the nsp3 protein has not been tested in any coronavirus vaccine studies and was selected for further investigation. The nsp3 was found to be more conserved among SARS-CoV-2, SARS-CoV, and MERS-CoV than among 15 coronaviruses infecting human and other animals. The protein was also predicted to contain promiscuous MHC-I and MHC-II T-cell epitopes, and linear B-cell epitopes localized in specific locations and functional domains of the protein. By applying reverse vaccinology and machine learning, we predicted potential vaccine targets for effective and safe COVID-19 vaccine development. We then propose that an \"Sp/Nsp cocktail vaccine\" containing a structural protein(s) (Sp) and a non-structural protein(s) (Nsp) would stimulate effective complementary immune responses.","Recombination is a process that unlinks neighbouring loci allowing for independent evolutionary trajectories within genomes of many organisms. If not properly accounted for, recombination can compromise many evolutionary analyses. In addition, when dealing with organisms that are not obligately sexually reproducing, recombination gives insight into the rate at which distinct genetic lineages come into contact. Since June, 2012, Middle East respiratory syndrome coronavirus (MERS-CoV) has caused 1106 laboratory-confirmed infections, with 421 MERS-CoV associated deaths as of April 16, 2015. Although bats are considered as the likely ultimate source of zoonotic betacoronaviruses, dromedary camels have been consistently implicated as the source of current human infections in the Middle East. In this paper we use phylogenetic methods and simulations to show that MERS-CoV genome has likely undergone numerous recombinations recently. Recombination in MERS-CoV implies frequent co-infection with distinct lineages of MERS-CoV, probably in camels given the current understanding of MERS-CoV epidemiology.","AbstractThe COVID-19 pandemic is expanding at an unprecedented rate. As a result, diagnostic services are stretched to their limit, and there is a clear need for the provision of additional diagnostic capacity. Academic laboratories, many of which are closed due to governmental lockdowns, may be in a position to support local screening capacity by adapting their current laboratory practices. Here, we describe the process of developing a SARS-Cov2 diagnostic workflow in a conventional academic Containment Level 2 (CL2) laboratory. Our outline includes simple SARS-Cov2 deactivation upon contact, the methods for a quantitative real-time reverse transcriptase PCR (qRT-PCR) detecting SARS-Cov2, a description of process establishment and validation, and some considerations for establishing a similar workflow elsewhere. This was achieved under challenging circumstances through the collaborative efforts of scientists, clinical staff, and diagnostic staff to mitigate to the ongoing crisis. Within 14 days, we created a validated COVID-19 diagnostics service for healthcare workers in our local hospital. The described methods are not exhaustive, but we hope may offer support to other academic groups aiming to set up something comparable in a short time frame.","AbstractThe appearance of the novel betacoronavirus SARS-CoV-2 represents a major threat to human health, and its diffusion around the world is causing dramatic consequences. The knowledge of the 3D structures of SARS-CoV-2 proteins can facilitate the development of therapeutic and diagnostic molecules. Specifically, comparative analyses of the structures of SARS-CoV-2 proteins and homologous proteins from previously characterized viruses, such as SARS-CoV, can reveal the common and/or distinctive traits that underlie the mechanisms of recognition of cell receptors and of molecules of the immune system.Herein, we apply our recently developed energy-based methods for the prediction of antibody-binding epitopes and protein-protein interaction regions to the Receptor Binding Domain (RBD) of the Spike proteins from SARS-CoV-2 and SARS-CoV. Our analysis focusses only on the study of the structure of RBDs in isolation, without making use of any previous knowledge of binding properties. Importantly, our results highlight structural and sequence differences among the regions that are predicted to be immunoreactive and bind/elicit antibodies. These results provide a rational basis to the observation that several SARS-CoV RDB-specific monoclonal antibodies fail to appreciably bind the SARS-CoV-2 counterpart. Furthermore, we correctly identify the region of SARS-CoV-2 RBD that is engaged by the cell receptor ACE2 during viral entry into host cells.The data, sequences and structures we present here can be useful for the development of novel therapeutic and diagnostic interventions.","Bats are notorious reservoirs of diverse, potentially zoonotic viruses, exemplified by the evolutionarily distinct, influenza A-like viruses H17N10 and H18N11 (BatIVs). The surface glycoproteins [haemagglutinin (H) and neuraminidase (N)] of BatIVs neither bind nor cleave sialic acid receptors, which suggests that these viruses employ cell attachment and entry mechanisms that differ from those of classical influenza A viruses (IAVs). Identifying the cellular factors that mediate entry and determine susceptibility to infection will help assess the host range of BatIVs. Here, we investigated a range of cell lines from different species for their susceptibility to infection by pseudotyped viruses bearing bat H17 and/or N10 envelope proteins. We show that a number of human haematopoietic cancer cell lines and the canine kidney MDCK II (but not MDCK I) cells are susceptible to H17-pseudotypes (H17-PV). We observed with microarrays and qRT-PCR that the dog leukocyte antigen DLA-DRA mRNA is overexpressed in late passaged parental MDCK and commercial MDCK II cells, compared to early passaged parental MDCK and MDCK I cells, respectively. The human orthologue HLA-DRA encodes the alpha subunit of the MHC class II HLA-DR antigen-binding heterodimer. Small interfering RNA- or neutralizing antibody-targeting HLA-DRA, drastically reduced the susceptibility of Raji B cells to H17-PV. Conversely, overexpression of HLA-DRA and its paralogue HLA-DRB1 on the surface of the unsusceptible HEK293T/17 cells conferred susceptibility to H17-PV. The identification of HLA-DR as an H17N10 entry mediator will contribute to a better understanding of the tropism of the virus and will elucidate its zoonotic transmission.","AbstractModels of host-microbe dynamics typically assume a single-host population infected by a single pathogen. In reality, many hosts form multi-species aggregations and may be infected with an assemblage of pathogens. We used a meta-transcriptomic approach to characterize the viromes of nine avian species in the Anseriformes (ducks) and Charadriiformes (shorebirds). This revealed the presence of 27 viral species, of which 24 were novel, including double-stranded RNA viruses (Picobirnaviridae and Reoviridae), single-stranded RNA viruses (Astroviridae, Caliciviridae, Picornaviridae), a retro-transcribing DNA virus (Hepadnaviridae), and a single-stranded DNA virus (Parvoviridae). These viruses comprise multi-host generalist viruses and those that are host-specific, indicative of both virome connectivity and heterogeneity. Virome connectivity was apparent in two well described multi-host virus species (avian coronavirus and influenza A virus) and a novel Rotavirus species that were shared among some Anseriform species, while heterogeneity was reflected in the absence of viruses shared between Anseriformes and Charadriiformes. Notably, within avian host families there was no significant relationship between either host taxonomy or foraging ecology and virome composition, although Anseriform species positive for influenza A virus harboured more additional viruses than those negative for influenza virus. Overall, we demonstrate complex virome structures across host species that co-exist in multi-species aggregations.","AbstractThe human microbiome, described as an accessory organ because of the crucial functions it provides, is composed of species that are uniquely found in humans1,2. Yet, surprisingly little is known about the impact of routine interpersonal contacts in shaping microbiome composition. In a relatively \u2018closed\u2019 cohort of 287 people from the Fiji Islands, where common barriers to bacterial transmission are absent, we examine putative bacterial transmission in individuals\u2019 gut and oral microbiomes using strain-level data from both core SNPs and flexible genomic regions. We find a weak signal of transmission, defined by the inferred sharing of genotypes, across many organisms that, in aggregate, reveals strong transmission patterns, most notably within households and between spouses. We find that women harbor strains more closely related to those harbored by their familial and social contacts than men; and that transmission patterns of oral- and gut-associated microbiota need not be the same. Using strain-level data alone, we are able to confidently predict a subset of spouses, highlighting the role of shared susceptibilities, behaviors or social interactions that distinguish specific links in the social network.","ABSTRACTSialic acids (Sia) are widely displayed on the surfaces of cells and tissues. Sia come in a variety of chemically modified forms, including those with acetyl modifications at the C7, C8, and C9 positions. Here, we analyzed the distribution and amounts of these acetyl modifications in different human and canine cells. As Sia or their variant forms are receptors for influenza A and influenza C viruses, we examined the effects of these modifications on virus infections. We confirmed that 9-O-acetyl and 7,9-O-acetyl modified Sia are widely but variably expressed across cell lines from both humans and canines. While they were expressed on the cell surface of canine MDCK cell lines, they were located primarily within the Golgi compartment of human HEK-293 and A549 cells. The O-acetyl modified Sia were expressed at low levels of 1-2% of total Sia in these cell lines. We knocked out and over-expressed the sialate O-acetyltransferase gene (CasD1), and knocked out the sialate O-acetylesterase gene (SIAE) using CRISPR/Cas9 editing. Knocking out CasD1 removed 7,9-O- and 9-O-acetyl Sia expression, confirming previous reports. However, over-expression of CasD1 and knockout of SIAE gave only modest increases in 9-O-acetyl levels in cells and no change in 7,9-O-acetyl levels, indicating that there are complex regulations of these modifications. These modifications were essential for influenza C infection, but had no obvious effect on influenza A infection.IMPORTANCESialic acids are key glycans that are involved in many different normal cellular functions, as well as being receptors for many pathogens. However, Sia come in diverse chemically modified forms. Here we examined and manipulated the expression of 7,9-O- and 9-O-acetyl modified Sia on cells commonly used in influenza virus and other research by engineering the enzymes that produce or remove the acetyl groups.","AbstractDuring outbreaks of deadly emerging pathogens (e.g., Ebola, MERS-CoV) and bioterror threats (e.g., smallpox), actively monitoring potentially infected individuals aims to limit disease transmission and morbidity. Guidance issued by CDC on active monitoring was a cornerstone of its response to the West Africa Ebola outbreak. There are limited data on how to balance the costs and performance of this important public health activity. We present a framework that estimates the risks and costs of specific durations of active monitoring for pathogens of significant public health concern. We analyze data from New York City\u2019s Ebola active monitoring program over a 16-month period in 2014-2016. For monitored individuals, we identified unique durations of active monitoring that minimize expected costs for those at \u201clow (but not zero) risk\u201d and \u201csome or high risk\u201d: 21 and 31 days, respectively. Extending our analysis to smallpox and MERS-CoV, we found that the optimal length of active monitoring relative to the median incubation period was reduced compared to Ebola due to less variable incubation periods. Active monitoring can save lives but is expensive. Resources can be most effectively allocated by using exposure-risk categories to modify the duration or intensity of active monitoring.","SUMMARYA novel coronavirus (nCoV-2019) was the cause of an outbreak of respiratory illness detected in Wuhan, Hubei Province, China in December of 2019. Genomic analyses of nCoV-2019 determined a 96% resemblance with a coronavirus isolated from a bat in 2013 (RaTG13); however, the receptor binding motif (RBM) of these two genomes share low sequence similarity. This divergence suggests a possible alternative source for the RBM coding sequence in nCoV-2019. We identified high sequence similarity in the RBM between nCoV-2019 and a coronavirus genome reconstructed from a viral metagenomic dataset from pangolins possibly indicating a more complex origin for nCoV-2019.","The coronavirus disease COVID-19, caused by emerging SARS-CoV-2, has posed serious threats to global public health, economic and social stabilities, calling for the prompt development of therapeutics and prophylactics. In this study, we firstly verified that SARS-CoV-2 uses human ACE2 as a cell receptor and its spike (S) protein mediates high membrane fusion activity. Comparing to that of SARS-CoV, the heptad repeat 1 (HR1) sequence in the S2 fusion protein of SARS-CoV-2 possesses markedly increased \u03b1-helicity and thermostability, as well as a higher binding affinity with its corresponding heptad repeat 2 (HR1) site. Then, we designed a HR2 sequence-based lipopeptide fusion inhibitor, termed IPB02, which showed highly poent activities in inibibiting the SARS-CoV-2 S protein-mediated cell-cell fusion and pseudovirus infection. IPB02 also inhibited the SARS-CoV pseudovirus efficiently. Moreover, the strcuture and activity relationship (SAR) of IPB02 were characterzized with a panel of truncated lipopeptides, revealing the amino acid motifs critical for its binding and antiviral capacities. Therefore, the presented results have provided important information for understanding the entry pathway of SARS-CoV-2 and the design of antivirals that target the membrane fusion step.","SARS-CoV-2, a coronavirus that newly emerged in China in late 2019 1,2 and spread rapidly worldwide, caused the first witnessed pandemic sparked by a coronavirus. As the pandemic progresses, information about the modes of transmission of SARS-CoV-2 among humans is critical to apply appropriate infection control measures and to slow its spread. Here we show that SARS-CoV-2 is transmitted efficiently via direct contact and via the air (via respiratory droplets and/or aerosols) between ferrets. Intranasal inoculation of donor ferrets resulted in a productive upper respiratory tract infection and long-term shedding, up to 11 to 19 days post-inoculation. SARS-CoV-2 transmitted to four out of four direct contact ferrets between 1 and 3 days after exposure and via the air to three out of four independent indirect recipient ferrets between 3 and 7 days after exposure. The pattern of virus shedding in the direct contact and indirect recipient ferrets was similar to that of the inoculated ferrets and infectious virus was isolated from all positive animals, showing that ferrets were productively infected via either route. This study provides experimental evidence of robust transmission of SARS-CoV-2 via the air, supporting the implementation of community-level social distancing measures currently applied in many countries in the world and informing decisions on infection control measures in healthcare settings 3.","AbstractXylella fastidiosa is an insect-borne bacterium confined to the xylem vessels of plants. This plant pathogen has a broad host range estimated to 560 plant species. Five subspecies of the pathogen with different but overlapping host ranges have been described, but only three subspecies are widely accepted, namely subspecies fastidiosa, multiplex and pauca. Initially limited to the Americas, Xf has been detected in Europe since 2013. As management of X. fastidiosa outbreaks in Europe depends on the identification of the subspecies, accurate determination of the subspecies in infected plants as early as possible is of major interest. Thus, we developed various tetraplex and triplex qPCR assays for Xylella fastidiosa detection and subspecies identification in planta in a single reaction. We designed primers and probes using SkIf, a bioinformatics tool based on k-mers, to detect specific signatures of the species and subspecies from a dataset of 58 genome sequences representative of X. fastidiosa diversity. We tested the qPCR assays on 39 target and 30 non-target strains, as well as on 13 different plant species spiked with strains of the different subspecies of X. fastidiosa, and on samples from various environmental and inoculated host plants. Sensitivity of simplex assays was equal or slightly better than the reference protocol on purified DNA. Tetraplex qPCR assays had the same sensitivity than the reference protocol and allowed X. fastidiosa detection in all spiked matrices up to 103 cells.mL\u22121. Moreover, mix infections of two to three subspecies could be detected in the same sample with tetraplex assays. In environmental plant samples, the tetraplex qPCR assays allowed subspecies identification when the current method based on multilocus sequence typing failed. The qPCR assays described here are robust and modular tools that are efficient for differentiating X. fastidiosa subspecies directly in plant samples.","The sudden emergence of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) at the end of 2019 from the Chinese province of Hubei and its subsequent pandemic spread highlight the importance of understanding the full molecular details of coronavirus infection and pathogenesis. Here, we compared a variety of replication features of SARS-CoV-2 and SARS-CoV and analysed the cytopathology caused by the two closely related viruses in the commonly used Vero E6 cell line. Compared to SARS-CoV, SARS-CoV-2 generated higher levels of intracellular viral RNA, but strikingly about 50-fold less infectious viral progeny was recovered from the culture medium. Immunofluorescence microscopy of SARS-CoV-2-infected cells established extensive cross-reactivity of antisera previously raised against a variety of nonstructural proteins, membrane and nucleocapsid protein of SARS-CoV. Electron microscopy revealed that the ultrastructural changes induced by the two SARS viruses are very similar and occur within comparable time frames after infection. Furthermore, we determined that the sensitivity of the two viruses to three established inhibitors of coronavirus replication (Remdesivir, Alisporivir and chloroquine) is very similar, but that SARS-CoV-2 infection was substantially more sensitive to pre-treatment of cells with pegylated interferon alpha. An important difference between the two viruses is the fact that - upon passaging in Vero E6 cells - SARS-CoV-2 apparently is under strong selection pressure to acquire adaptive mutations in its spike protein gene. These mutations change or delete a putative furin-like cleavage site in the region connecting the S1 and S2 domains and result in a very prominent phenotypic change in plaque assays.","The World Health Organization declared the COVID-19 epidemic a public health emergency of international concern on March 11th, 2020, and the pandemic is rapidly spreading worldwide. COVID-19 is caused by a novel coronavirus SARS-CoV-2, which enters human target cells via angiotensin converting enzyme 2 (ACE2). We used a number of bioinformatics tools to computationally characterize ACE2 by determining its cell-specific expression, putative functions, and transcriptional regulation. The small intestine expressed higher levels of ACE2 than any other organ. The large intestine, kidney and testis showed moderate signals, whereas the signal was weak in the lung specimens. Single cell RNA-Seq data indicated positive signals along the respiratory tract in the key protective cell types including the goblet and ciliary epithelial cells, as well as in the endothelial cells and type I pneumocytes. Gene ontology analysis suggested that, besides its classical role in renin-angiotensin system, ACE2 may be functionally associated with angiogenesis/blood vessel morphogenesis. A novel tool for the prediction of transcription factor binding sites identified several putative sites for determined transcription factors within the ACE2 gene promoter. Our results also confirmed that age and gender play no significant role in the regulation of ACE2 mRNA expression in the lung.","Emergence of new viral diseases is linked to mutation or recombination events. The likelihood of cross-species transmission is related to phenotypic plasticity of a virus and its capacity to produce genetically variable progeny. Herein a model is described connecting the production of genetic variability with increasing genome size. Comparing all known zoonotic viral genome sizes to known non-zoonotic viral genome sizes demonstrates that zoonotic viruses have significantly larger genomes. These results support the notion that large viral genomes are important in producing new zoonotic disease, and suggest that genome size may be a useful surrogate in screening for potential zoonotic viruses.","The beginning of 2020 brought us information about the novel coronavirus emerging in China. Rapid research resulted in the characterization of the pathogen, which appeared to be a member of the SARS-like cluster, commonly seen in bats. Despite the global and local efforts, the virus escaped the healthcare measures and rapidly spread in China and later globally, officially causing a pandemic and global crisis in March 2020. At present, different scenarios are being written to contain the virus, but the development of novel anticoronavirals for all highly pathogenic coronaviruses remains the major challenge. Here, we describe the antiviral activity of previously developed by us HTCC compound (N-(2-hydroxypropyl)-3-trimethylammonium chitosan chloride), which may be used as potential inhibitor of currently circulating highly pathogenic coronaviruses - SARS-CoV-2 and MERS-CoV.","AbstractThe positive stranded RNA genomes of picornaviruses comprise a single large open reading frame flanked by 5\u2032 and 3\u2032 untranslated regions (UTRs). Foot-and-mouth disease virus (FMDV) has an unusually large 5\u2032 UTR (1.3 kb) containing five structural domains. These include the internal ribosome entry site (IRES), which facilitates initiation of translation, and the cis-acting replication element (cre). Less well characterised structures are a 5\u2032 terminal 360 nucleotide stem-loop, a variable length poly-C-tract of approximately 100-200 nucleotides and a series of two to four tandemly repeated pseudoknots (PKs). We investigated the structures of the PKs by selective 2\u2032 hydroxyl acetylation analysed by primer extension (SHAPE) analysis and determined their contribution to genome replication by mutation and deletion experiments. SHAPE and mutation experiments confirmed the importance of the previously predicted PK structures for their function. Deletion experiments showed that although PKs are not essential for replication, they provide genomes with a competitive advantage. However, although replicons and full-length genomes lacking all PKs were replication competent, no infectious virus was rescued from genomes containing less than one PK copy. This is consistent with our earlier report describing the presence of putative packaging signals in the PK region.","AbstractSevere acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is the causative agent of Coronavirus disease 2019 (COVID-19) which is an infectious disease that spread throughout the world and was declared as a pandemic by the World Health Organization (WHO). In the present study, we analyzed genome-wide codon usage patterns in 13 SARS-CoV-2 isolates from different geo-locations (countries) by utilizing different CUB measurements. Nucleotide and di-nucleotide compositions displayed bias toward A/U content in all codon positions and CpU-ended codons preference, respectively. Relative Synonymous Codon Usage (RSCU) analysis revealed 8 common putative preferred codons among all the 13 isolates. Interestingly, all of the latter codons are A/U-ended (U-ended: 7, A-ended: 1). Cluster analysis (based on RSCU values) was performed and showed comparable results to the phylogenetic analysis (based on their whole genome sequences) indicating that the CUB pattern may reflect the evolutionary relationship between the tested isolates. To investigate the force (mutation and/or selection) influencing the pattern of CUB in SARS-CoV-2 coding sequences, we employed the following; (i). Effective number of codons (ENc), (ii). ENc-GC3 plot, (iii). Neutrality plot, and (iv) Codon adaptation index (CAI). According to their results, natural selection and/or other factors (not investigated in this study) may be the dominant force driving SARS-CoV-2 CUB. It is also worth mentioning that, by using the most expressed genes in human lung tissues as a reference set, some viral genes such as Nucleocapsid phosphoprotein, ORF7a protein, and surface glycoprotein had high CAI values which may indicate for selection force acting on their codon usage, as they play important roles in viral assembly and may help viruses avoid the host immune system. The outcome of our study may help in understanding the underlying factors involved in the evolution of SARS-CoV-2 viruses, and the interactions with their host. Also, it may aid in vaccine design strategies.","AbstractCoronavirus disease 19 (COVID-19) is an emerging global health crisis. With over 200,000 confirmed cases to date, this pandemic continues to expand, spurring research to discover vaccines and therapies. SARS-CoV-2 is the novel coronavirus responsible for this disease. It initiates entry into human cells by binding to angiotensin-converting enzyme 2 (ACE2) via the receptor binding domain (RBD) of its spike protein (S). Disrupting the SARS-CoV-2-RBD binding to ACE2 with designer drugs has the potential to inhibit the virus from entering human cells, presenting a new modality for therapeutic intervention. Peptide-based binders are an attractive solution to inhibit the RBD-ACE2 interaction by adequately covering the extended protein contact interface. Using molecular dynamics simulations based on the recently solved ACE2 and SARS-CoV-2-RBD co-crystal structure, we observed that the ACE2 peptidase domain (PD) \u03b11 helix is important for binding SARS-CoV-2-RBD. Using automated fast-flow peptide synthesis, we chemically synthesized a 23-mer peptide fragment of the ACE2 PD \u03b11 helix composed entirely of proteinogenic amino acids. Chemical synthesis of this human derived sequence was complete in 1.5 hours and after work up and isolation &gt;20 milligrams of pure material was obtained. Bio-layer interferometry revealed that this peptide specifically associates with the SARS-CoV-2-RBD with low nanomolar affinity. This peptide binder to SARS-CoV-2-RBD provides new avenues for COVID-19 treatment and diagnostic modalities by blocking the SARS-CoV-2 spike protein interaction with ACE2 and thus precluding virus entry into human cells.","AbstractWuhan Novel Coronavirus disease (COVID-19) outbreak has become a global outbreak which has raised the concern of scientific community to design and discover a definitive cure against this deadly virus which has caused deaths of numerous infected people upon infection and spreading. To date, no antiviral therapy or vaccine is available which can effectively combat the infection caused by this virus. This study was conducted to design possible epitope-based subunit vaccines against the SARS-CoV-2 virus using the approaches of reverse vaccinology and immunoinformatics. Upon continual computational experimentation three possible vaccine constructs were designed and one vaccine construct was selected as the best vaccine based on molecular docking study which is supposed to effectively act against SARS-CoV-2. Later, molecular dynamics simulation and in silico codon adaptation experiments were carried out in order to check biological stability and find effective mass production strategy of the selected vaccine. Hopefully, this study will contribute to uphold the present efforts of the researches to secure a definitive treatment against this lethal virus.","ABSTRACTThe recent severe acute respiratory syndrome, known as Corona Virus Disease 2019 (COVID-19) has spread so much rapidly and severely to induce World Health Organization (WHO) to declare state of emergency over the new coronavirus SARS-CoV-2 pandemic. While several countries have chosen the almost complete lock-down for slowing down SARS-CoV-2 spread, scientific community is called to respond to the devastating outbreak by identifying new tools for diagnosis and treatment of the dangerous COVID-19. With this aim we performed an in silico comparative modeling analysis, which allows to gain new insights about the main conformational changes occurring in the SARS-CoV-2 spike protein, at the level of the receptor binding domain (RBD), along interactions with human cells angiotensin converting enzyme 2 (ACE2) receptor, that favour human cell invasion. Furthermore, our analysis provides i) an ideal pipeline to identify already characterized antibodies that might target SARS-CoV-2 spike RBD, for preventing interactions with the human ACE2, and ii) instructions for building new possible neutralizing antibodies, according to chemical/physical space restraints and complementary determining regions (CDR) mutagenesis of the identified existing antibodies. The proposed antibodies show in silico a high affinity for SARS-CoV-2 spike RBD and can be used as reference antibodies also for building new high affinity antibodies against present and future coronavirus able to invade human cells through interactions of their spike proteins with the human ACE2. More in general, our analysis provides indications for the set-up of the right biological molecular context for investigating spike RBD-ACE2 interactions for the development of new vaccines, diagnosis kits and other treatments based on the usage or the targeting of SARS-CoV-2 spike protein.","AbstractPost-transcriptional gene silencing (PTGS) is a powerful tool to understand and control plant metabolic pathways, which is central to plant biotechnology. PTGS is commonly accomplished through delivery of small interfering RNA (siRNA) into cells. While siRNA delivery has been optimized for mammalian systems, it remains a significant challenge for plants due to the plant cell wall. Standard plant siRNA delivery methods (Agrobacterium and viruses) involve coding siRNA into DNA vectors, and are only tractable for certain plant species. Herein, we develop a nanotube-based platform for direct delivery of siRNA, and show high silencing efficiency in intact plant cells. We demonstrate that nanotubes successfully deliver siRNA and silence endogenous genes owing to effective intracellular delivery and nanotube-induced protection of siRNA from nuclease degradation. This study establishes that nanotubes, which are below the size exclusion limit of the plant cell wall, could enable a myriad of plant biotechnology applications that rely on RNA delivery.","Sickness behaviours, like lethargy, can slow the spread of pathogens across a social network. We conducted a field experiment to investigate how sickness behaviour reduces individual connectedness in a high-resolution dynamic social network. We captured 35 adult female vampire bats (Desmodus rotundus) from a wild roost. To create \"sick\" bats, we injected a random half of the bats with the immune-challenging substance, lipopolysaccharide, and injected the control group with saline. Over the next three days, we used proximity sensors to continuously track their associations under natural conditions. The \"sick\" bats showed a clear decrease in social connectedness (degree, strength, and eigenvector centrality). Bats in the control group encountered fewer \"sick\" bats and also spent less time near them. These effects varied by time of day and declined over 48 hours. High-resolution proximity data allow researchers to define network connections based on how a pathogen spreads (e.g. the minimum contact time or distance for transmission). We therefore show how the estimate of the sickness effect changes as network ties are defined using varying distances and durations of association. Tracking the effects of sickness behaviour on high-resolution dynamic social networks can help create more sophisticated simulations of pathogen transmission through structured populations.","AbstractTilorone is a 50-year-old synthetic small-molecule compound with antiviral activity that is proposed to induce interferon after oral administration. This drug is used as a broad-spectrum antiviral in several countries of the Russian Federation. We have recently described activity in vitro and in vivo against the Ebola Virus. After a broad screening of additional viruses, we now describe in vitro activity against Chikungunya virus (CHIK) and Middle Eastern Respiratory Syndrome Coronavirus (MERS-CoV).","Abstract17 years after the SARS-CoV epidemic, the world is facing the COVID-19 pandemic. COVID-19 is caused by a coronavirus named SARS-CoV-2. Given the most optimistic projections estimating that it will take more than a year to develop a vaccine, our best short term strategy may lie in identifying virus-specific targets for small molecule interventions. All coronaviruses utilize a molecular mechanism called -1 PRF to control the relative expression of their proteins. Prior analyses of SARS-CoV revealed that it utilizes a structurally unique three-stemmed mRNA pseudoknot to stimulate high rates of -1 PRF, that it also harbors a -1 PRF attenuation element. Altering -1 PRF activity negatively impacts virus replication, suggesting that this molecular mechanism may be therapeutically targeted. Here we present a comparative analysis of the original SARS-CoV and SARS-CoV-2 frameshift signals. Structural analyses reveal that the core -1 PRF signal, composed of the U UUA AAC slippery site and three-stemmed mRNA pseudoknot is highly conserved. In contrast, the upstream attenuator hairpin is less well conserved. Functional assays revealed that both elements promote similar rates of -1 PRF and that silent coding mutations in the slippery site strongly ablate -1 PRF activity. We suggest that molecules that were previously identified as inhibiting SARS-CoV mediated -1 PRF may serve as lead compounds to counter the current pandemic.","AbstractIn recent decades, many new flavi-like viruses have been discovered predominantly in different invertebrates and, as was recently shown, some of them may cause disease in humans. The Jingmen tick virus (JMTV) group holds a special place among flavi-like viruses because, in contrast to the \u201cclassic\u201d flaviviruses and other flavi-like viruses, they have a segmented ssRNA(+) genome. Two segments of the JMTV genome have homology with regions of the flavivirus genome encoding polymerase and helicase-protease proteins. JMTVs have several open reading frames (ORF) in segments encoding glycoprotein(s) and capsid protein and these ORF are specific only to them. JMTVs greatly differ in virion size.We isolated three strains of Alongshan virus (ALSV), which is a representative of the JMTV group, from adult Ixodes persulcatus ticks collected in two geographically-separated Russian regions in the tick cell line IRE/CTVM19. One of the strains persisted in the IRE/CTVM19 cells without cytopathic effect for three years. Most virions purified from tick cells were spherical with a diameter of approximately 40.5 nm. In addition, we found smaller particles of approximately 13.1 nm in diameter. We obtained full genome sequences of all four segments of two of the isolated ALSV strains, and partial sequences of one segment from the third strain. Phylogenetic analysis on genome segment 2 of the JMTV group clustered our novel strains with other ALSV strains. We found evidence for the existence of a novel upstream ORF in the glycoprotein-coding segment of ALSV and other members of the JMTV group.Significance StatementWe isolated three strains of Alongshan virus (ALSV) from adult Ixodes persulcatus ticks from two geographically separate areas of Russia in the Ixodes ricinus tick cell line IRE/CTVM19. One of the strains persisted in the IRE/CTVM19 cells without cytopathic effect for three years. Our study confirmed the value of tick cell lines in virus isolation and maintenance of persistent infection. The majority of virions of the ALSV strain Miass527 were enveloped spherical particles with a diameter of 40.5\u00b13.7 nm. We found evidence for the existence of a novel upstream ORF in the glycoprotein-coding segment of ALSV and other members of the Jingmen tick virus group.","AbstractThe emergence and outbreak of SARS-CoV-2, the causative agent of COVID-19, has rapidly become a global concern and has highlighted the need for fast, sensitive, and specific tools to surveil circulating viruses. Here we provide assay designs and experimental resources, for use with CRISPR-based nucleic acid detection, that could be valuable for ongoing surveillance. We provide assay designs for detection of 67 viral species and subspecies, including: SARS-CoV-2, phylogenetically-related viruses, and viruses with similar clinical presentation. The designs are outputs of algorithms that we are developing for rapidly designing nucleic acid detection assays that are comprehensive across genomic diversity and predicted to be highly sensitive and specific. Of our design set, we experimentally screened 4 SARS-CoV-2 designs with a CRISPR-Cas13 detection system and then extensively tested the highest-performing SARS-CoV-2 assay. We demonstrate the sensitivity and speed of this assay using synthetic targets with fluorescent and lateral flow detection. Moreover, our provided protocol can be extended for testing the other 66 provided designs. Assay designs are available at https://adapt.sabetilab.org/.","AbstractBackgroundIn silico bacterial, viral, and human truth datasets were generated to evaluate available metagenomics algorithms. Sequenced datasets include background organisms, creating ambiguity in the true source organism for each read. Bacterial and viral datasets were created with even and staggered coverage to evaluate organism identification, read mapping, and gene identification capabilities of available algorithms. These truth datasets are provided as a resource for the development and refinement of metagenomic algorithms. Algorithm performance on these truth datasets can inform decision makers on strengths and weaknesses of available algorithms and how the results may be best leveraged for bacterial and viral organism identification and characterization.Source organisms were selected to mirror communities described in the Human Microbiome Project as well as the emerging pathogens listed by the National Institute of Allergy and Infectious Diseases. The six in silico datasets were used to evaluate the performance of six leading metagenomics algorithms: MetaScope, Kraken, LMAT, MetaPhlAn, MetaCV, and MetaPhyler.ResultsAlgorithms were evaluated on runtime, true positive organisms identified to the genus and species levels, false positive organisms identified to genus and species level, read mapping, relative abundance estimation, and gene calling. No algorithm out performed the others in all categories, and the algorithm or algorithms of choice strongly depends on analysis goals. MetaPhlAn excels for bacteria and LMAT for viruses. The algorithms were ranked by overall performance using a normalized weighted sum of the above metrics, and MetaScope emerged as the overall winner, followed by Kraken and LMAT.ConclusionsSimulated FASTQ datasets with well-characterized truth data about microbial community composition reveal numerous insights about the relative strengths and weaknesses of the metagenomics algorithms evaluated. The simulated datasets are available to download from the Sequence Read Archive (SRP062063).","ABSTRACTType 1 regulatory CD4+ T (Tr1) cells express high levels of the immunosuppressive cytokine IL-10 but not the master transcription factor Foxp3, and can suppress inflammation and promote immune tolerance. In order to identify and obtain viable Tr1 cells for research and clinical applications, co-expression of CD49b and LAG3 has been proposed as a unique surface signature for both human and mouse Tr1 cells. However, recent studies have revealed that this pattern of co-expression is dependent on the stimulating conditions and the differentiation stage of the CD4+ T cells. Here, using an IL-10GFP/Foxp3RFP dual reporter transgenic murine model, we demonstrate that co-expression of CD49b and LAG3 is not restricted to the Foxp3\u2212 Tr1 cells, but is also observed in Foxp3+ T regulatory (Treg) cells and CD8+ T cells that produce IL-10. Our data indicate that IL-10-producing Tr1 cells, Treg cells and CD8+ T cells are all capable of co-expressing LAG3 and CD49b in vitro following differentiation under IL-10-inducing conditions, and in vivo following pathogenic insult or infection in the pulmonary mucosa. Our findings urge caution in the use of LAG3/CD49b co-expression to identify Tr1 cells, since it may mark IL-10-producing T cell lineages more broadly, including the Foxp3\u2212 Tr1 cells, Foxp3+ Treg cells and CD8+ T cells.","AbstractBackgroundCOVID-19 is a disease with global public health emergency that have shook the world since its\u2019 first detection in China in December, 2019. Severe acute respiratory syndrome Coronavirus 2 (SARS-CoV-2) is the pathogen responsible behind this pandemic. The lethality of different viral strains is found to vary in different geographical locations but the molecular mechanism is yet to be known.MethodsAvailable data of whole genome sequencing of different viral strains published by different countries were retrieved and then analysed using Multiple Sequence Alignment and Pair-wise Sequence Alignment leading to Phylogenetic tree construction. Each location and the corresponding genetic variations were screened in depth. Then the variations are analysed at protein level giving special emphasis on Non Synonymous amino acid substitutions. The fatality rates in different countries were matched against the mutation number, rarity of the nucleotide alterations and functional impact of the Non Synonymous changes at protein level, separately and in combination.FindingsAll the viral strains have been found to evolve from the viral strain of Taiwan (MT192759) which is 100% identical with the ancestor SARS-CoV-2 sequences of Wuhan (NC 045512.2; submitted on 5th Jan, 2020). Transition from C to T (C&gt;T) is the most frequent mutation in this viral genome and mutations A&gt;T, G&gt;A, T&gt;A are the rarest ones, found in countries with maximum fatality rate i.e Italy, Spain and Sweden. 20 Non Synonymous mutations are located in viral genome spanning Orf1ab polyprotein, Surface glycoprotein, Nucleocapsid protein etc. The functional effect on the structure and function of the protein can favourably or unfavourably interact with the host body.InterpretationThe fatality outcome depends on three important factors (a) number of mutation (b) rarity of the allelic variation and (c) functional consequence of the mutation at protein level. The molecular divergence, evolved from the ancestral strain (S) lead to extremely lethal (E), lethal(L) and non lethal (N) strains with the involvement of an Intermediate strain(I).","AbstractIn 2015, the Middle East Respiratory Syndrome Coronavirus (MERS-CoV) reached the Republic of Korea through nosocomial transmission and was the largest epidemic outside of the Arabian Peninsula. To date, despite various strategies to identify CoV interventions, only limited therapeutic options are available. To address these unmet medical needs, we used a South Korean MERS-CoV clinical isolate and screened 5,406 compounds, including United States Food and Drug Administration (FDA)-approved drugs and bioactive molecules, for their activity against the isolate. The primary assay confirmed 221 hits by dose-response curve analysis and identified 54 hits with a therapeutic index (TI) greater than 6. Time-of-addition studies with 12 FDA-approved drugs demonstrated that 8 and 4 therapeutics act on the early and late stages of the viral life cycle, respectively. Among the drugs were e.g., three cardiotonic agents (ouabain, digitoxin, digoxin) with a TI greater than 100, an anti-malaria drug (atovaquone; TI &gt;34), an inhalable corticosteroid (ciclesonide; TI &gt;6), etc. Together, our results identify potential therapeutic options for treating MERS-CoV infections and could provide a basis for agents against a wider range of coronavirus-related illnesses, including the currently emerging Coronavirus Disease 2019 (COVID-19) outbreak.","ABSTRACTWe aim to test four one-step RT real-time mastermix options for use in SARS-CoV2 real-time PCR, with three primer/probe assays targeting the N gene. The lower limit of detection is determined using a SARS CoV2 N gene RNA transcript dilution series (to 1 copy/\u00b5l) and verified using 74 nose and throat swabs.The N2 assay demonstrates the most sensitive detection of SARS-Cov-2 RNA. Three of the four mastermixes performed well, with the Takara One Step PrimeScript\u2122 III RT-PCR Kit mastermix demonstrating improved performance at the lower limit of detection.","ABSTRACTThe 3\u2032-to-5\u2032 exoribonuclease in coronavirus (CoV) nonstructural protein 14 (nsp14-ExoN) mediates RNA proofreading during genome replication. ExoN catalytic residues are arranged in three motifs: I (DE), II (E), III (D). Alanine substitution of the motif I residues (AA-E-D, four nucleotide substitutions) in murine hepatitis virus (MHV) and SARS-CoV yields viable mutants with impaired replication and fitness, increased mutation rates, and attenuated virulence in vivo. Despite these impairments, MHV- and SARS-CoV ExoN motif I AA mutants (ExoN-AA) have not reverted at motif I in diverse in vitro and in vivo environments, suggesting that profound fitness barriers prevent motif I reversion. To test this hypothesis, we engineered MHV-ExoN-AA with 1, 2 or 3 nucleotide mutations along genetic pathways to AA-to-DE reversion. We show that engineered intermediate revertants were viable but had no increased replication or competitive fitness compared to MHV-ExoN-AA. In contrast, a low passage (P10) MHV-ExoN-AA showed increased replication and competitive fitness without reversion of ExoN-AA. Finally, engineered reversion of ExoN-AA to ExoN-DE in the presence of ExoN-AA passage-adaptive mutations resulted in significant fitness loss. These results demonstrate that while reversion is possible, at least one alternative adaptive pathway is more rapidly advantageous than intermediate revertants and may alter the genetic background to render reversion detrimental to fitness. Our results provide an evolutionary rationale for lack of ExoN-AA reversion, illuminate potential multi-protein replicase interactions and coevolution, and support future studies aimed at stabilizing attenuated CoV ExoN-AA mutants.IMPORTANCECoronaviruses encode an exoribonuclease (ExoN) that is important for viral replication, fitness, and virulence, yet coronaviruses with a defective ExoN (ExoN-AA) have not reverted under diverse experimental conditions. In this study, we identify multiple impediments to MHV-ExoN-AA reversion. We show that ExoN-AA reversion is possible but evolutionarily unfavorable. Instead, compensatory mutations outside of ExoN-AA motif I are more accessible and beneficial than partial reversion. We also show that coevolution between replicase proteins over long-term passage partially compensates for ExoN-AA motif I but renders the virus inhospitable to a reverted ExoN. Our results reveal the evolutionary basis for the genetic stability of ExoN-inactivating mutations, illuminate complex functional and evolutionary relationships between coronavirus replicase proteins, and identify potential mechanisms for stabilization of ExoN-AA coronavirus mutants.","AbstractMost emerging human infectious diseases have an animal origin. Yet, while zoonotic diseases originate from a primary reservoir, most theoretical studies have principally focused on single-host processes, either exclusively humans or exclusively animals, without considering the importance of animal to human transmission for understanding the dynamics of emerging infectious diseases. Here we aim to investigate the importance of spillover transmission for explaining the number and the size of outbreaks. We propose a simple stochastic Susceptible-Infected-Recovered model with a recurrent infection of an incidental host from a reservoir (e.g. humans by a zoonotic species), considering two modes of transmission, (1) animal-to-human and (2) human-to-human. The model assumes that (i) epidemiological processes are faster than other processes such as demographics or pathogen evolution and (ii) that an epidemic occurs until there are no susceptible individuals left. The results show that during an epidemic, even when the pathogens are barely contagious, multiple outbreaks are observed due to spillover transmission. Overall, the findings demonstrate that the only consideration of direct transmission between individuals is not sufficient to explain the dynamics of zoonotic pathogens in an incidental host.","AbstractCOVID-19 virus is the cause of a debilitating and life-threatening infectious pulmonary disease that is now responsible for a global pandemic. Currently, there are no specific drugs or vaccines to contain this virus. The main protease (Mpro) of COVID-19 virus is a key enzyme, which plays an essential role in viral replication and transcription, making it an ideal drug target. An FDA-approved antineoplastic drug, carmofur, has been identified as an inhibitor that targets COVID-19 virus Mpro. However, its inhibitory mechanism is unknown. Here, we report the 1.6-\u00c5 crystal structure of COVID-19 virus Mpro in complex with carmofur. The crystal structure shows that carmofur contains an electrophilic carbonyl reactive group, which covalently binds to C145, a member of the catalytic dyad. As a result, its fatty acid tail occupies the hydrophobic S2 subsite of Mpro whilst its 5-fluorouracil head is cleaved as product of the new covalent bond that has formed. Carmofur is active in a cell based antiviral assay with an EC50 of 24.87 \u03bcM. It is therefore a promising lead compound for the development of new antivirals to target COVID-19.","SUMMARYThe severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is a currently uncontrolled pandemic and the etiological agent of coronavirus disease 2019 (COVID-19). It is important to study the host range of SARS-CoV-2 because some domestic species might harbor the virus and transmit it back to humans. In addition, insight into the ability of SARS-CoV-2 and SARS-like viruses to utilize animal orthologs of the SARS-CoV-2 receptor ACE2 might provide structural insight into improving ACE2-based viral entry inhibitors. Here we show that ACE2 orthologs of a wide range of domestic and wild animals support entry of SARS-CoV-2, as well as that of SARS-CoV-1, bat coronavirus RaTG13, and a coronavirus isolated from pangolins. Some of these species, including camels, cattle, horses, goats, sheep, pigs, cats, and rabbits may serve as potential intermediate hosts for new human transmission, and rabbits in particular may serve as a useful experimental model of COVID-19. We show that SARS-CoV-2 and SARS-CoV-1 entry could be potently blocked by recombinant IgG Fc-fusion proteins of viral spike protein receptor-binding domains (RBD-Fc) and soluble ACE2 (ACE2-Fc). Moreover, an ACE2-Fc variant, which carries a D30E mutation and has ACE2 truncated at its residue 740 but not 615, outperforms all the other ACE2-Fc variants on blocking entry of both viruses. Our data suggest that RBD-Fc and ACE2-Fc could be used to treat and prevent infection of SARS-CoV-2 and any new viral variants that emerge over the course of the pandemic.","Background: Effective therapeutics to treat COVID-19 are urgently needed. Remdesivir is a nucleotide prodrug with in vitro and in vivo efficacy against coronaviruses. Here, we tested the efficacy of remdesivir treatment in a rhesus macaque model of SARS-CoV-2 infection. Methods: To evaluate the effect of remdesivir treatment on SARS-CoV-2 disease outcome, we used the recently established rhesus macaque model of SARS-CoV-2 infection that results in transient lower respiratory tract disease. Two groups of six rhesus macaques were infected with SARS-CoV-2 and treated with intravenous remdesivir or an equal volume of vehicle solution once daily. Clinical, virological and histological parameters were assessed regularly during the study and at necropsy to determine treatment efficacy. Results: In contrast to vehicle-treated animals, animals treated with remdesivir did not show signs of respiratory disease and had reduced pulmonary infiltrates on radiographs. Virus titers in bronchoalveolar lavages were significantly reduced as early as 12hrs after the first treatment was administered. At necropsy on day 7 after inoculation, lung viral loads of remdesivir-treated animals were significantly lower and there was a clear reduction in damage to the lung tissue. Conclusions: Therapeutic remdesivir treatment initiated early during infection has a clear clinical benefit in SARS-CoV-2-infected rhesus macaques. These data support early remdesivir treatment initiation in COVID-19 patients to prevent progression to severe pneumonia.","AbstractHuman behavior can change the spread of infectious disease. There is limited understanding of how the time in the future over which individuals make a behavioral decision, their planning horizon, affects epidemic dynamics. We developed an agent-based model (along with an ODE analog) to explore the decision-making of self-interested individuals on adopting prophylactic behavior. The decision-making process incorporates prophylaxis efficacy and disease prevalence with individuals' payoffs and planning horizon. Our results show that for short and long planning horizons individuals do not consider engaging in prophylactic behavior. In contrast, individuals adopt prophylactic behavior when considering intermediate planning horizons. Such adoption, however, is not always monotonically associated with the prevalence of the disease, depending on the perceived protection efficacy and the disease parameters. Adoption of prophylactic behavior reduces the peak size while prolonging the epidemic and potentially generates secondary waves of infection. These effects can be made stronger by increasing the behavioral decision frequency or distorting an individual\u2019s perceived risk of infection.","There is an urgent necessity of effective medication against SARS CoV-2, which is producing the COVID-19 pandemic across the world. Its main protease (Mpro) represents an attractive pharmacological target due to its involvement in essential viral functions. The crystal structure of free Mpro shows a large structural resemblance with the main protease of CoV-1. Here we report that CoV-2 Mpro is however 300% more sensitive than CoV-1 Mpro in transmitting tiny structural changes across the whole protein through long-range interactions. The largest sensitivity of Mpro to structural perturbations is located exactly around the catalytic site Cyst-145, and coincides with the binding site of several of its inhibitors. These findings, based on a simplified representation of the protein as a residue network, may help in designing potent inhibitors of CoV-2 Mpro.","AbstractViral escape from CD8+ cytotoxic T lymphocyte responses correlates with disease progression and represents a significant challenge for vaccination. Here, we demonstrate that CD8+ T cell recognition of the naturally occurring MHC-I-restricted LCMV-associated immune escape variant Y4F is restored following vaccination with a proline-altered peptide ligand (APL). The APL increases MHC/peptide (pMHC) complex stability, rigidifies the peptide and facilitates T cell receptor (TCR) recognition through reduced entropy costs. Structural analyses of pMHC complexes before and after TCR binding, combined with biophysical analyses, revealed that although the TCR binds similarly to all complexes, the p3P modification alters the conformations of a very limited amount of specific MHC and peptide residues, facilitating efficient TCR recognition. This approach can be easily introduced in peptides restricted to other MHC alleles, and can be combined with currently available and future vaccination protocols in order to prevent viral immune escape.Author SummaryViral escape mutagenesis correlates often with disease progression and represents a major hurdle for vaccination-based therapies. Here, we have designed and developed a novel generation of altered epitopes that re-establish and enhance significantly CD8+ T cell recognition of a naturally occurring viral immune escape variant. Biophysical and structural analyses provide a clear understanding of the molecular mechanisms underlying this reestablished recognition. We believe that this approach can be implemented to currently available or novel vaccination approaches to efficiently restore T cell recognition of virus escape variants to control disease progression.","AbstractFibrolamellar carcinoma (FLC) is a rare, therapeutically intractable liver cancer that disproportionately affects youth. Although FLC tumors exhibit a distinct gene expression profile, the causative transcriptional mechanisms remain unclear. Here we used chromatin run-on sequencing to discover approximately 7,000 enhancers and 141 enhancer hotspots activated in FLC relative to non-malignant liver. Detailed bioinformatic analyses revealed aberrant ERK/MEK signaling and candidate master transcriptional regulators. We also defined the genes most strongly associated with aberrant FLC enhancer activity, including CA12 and SLC16A14. Treatment of FLC cell models with inhibitors of CA12 or SLC16A14 independently reduced cell viability and/or significantly enhanced the effect of MEK inhibitor cobimetinib. These findings highlight new molecular targets for drug development as well as novel drug combination approaches.","AbstractZika virus (ZIKV) is a global public health issue due to its association with severe developmental disorders in infants and neurological disorders in adults. Because ZIKV uses glycosylation of its envelope (E) protein to interact with host cell receptors to facilitate entry, these interactions could also be important for designing therapeutics and vaccines. Due to a lack of information about Asn-linked (N-glycans) on ZIKV E, we analyzed ZIKV E of various strains derived from different cells. ZIKV E proteins are extensively modified with oligomannose-, hybrid- and complex-N-glycans of a highly heterogeneous nature. Host cell-surface glycans correlated strongly with the glycomic features of ZIKV E. Mechanistically, we discovered that ZIKV N-glycans are important in viral pathogenesis, as mannose-specific C-type lectins DC-SIGN and L-SIGN mediate cell entry of ZIKV. Our findings represent the first detailed mapping of N-glycans on ZIKV E of various strains and their functional significance.","AbstractSince the emergence of SARS-CoV-2 in December 2019, Coronavirus Disease-2019 (COVID-19) has rapidly spread across the globe. Epidemiologic studies have demonstrated that age is one of the strongest risk factors influencing the morbidity and mortality of COVID-19. Here, we interrogate the transcriptional features and cellular landscapes of the aging human lung through integrative analysis of bulk and single-cell transcriptomics. By intersecting these age-associated changes with prior experimental data on host interactions between SARS-CoV-2 or its relative SARS-CoV, we identify several age-associated factors that may contribute to the heightened severity of COVID-19 in older populations. These analyses illuminate potential avenues for further studies on the relationship between the aging lung and COVID-19 pathogenesis, which may inform strategies to more effectively treat this disease.","The SARS-coronavirus 2 (SARS-CoV-2) spike (S) protein mediates entry of SARS-CoV-2 into cells expressing the angiotensin-converting enzyme 2 (ACE2). The S protein engages ACE2 through its receptor-binding domain (RBD), an independently folded 197-amino acid fragment of the 1273-amino acid S-protein protomer. Antibodies to the RBD domain of SARS-CoV (SARS-CoV-1), a closely related coronavirus which emerged in 2002-2003, have been shown to potently neutralize SARS-CoV-1 S-protein-mediated entry, and the presence of anti-RBD antibodies correlates with neutralization in SARS-CoV-2 convalescent sera. Here we show that immunization with the SARS-CoV-2 RBD elicits a robust neutralizing antibody response in rodents, comparable to 100 \u03bcg/ml of ACE2-Ig, a potent SARS-CoV-2 entry inhibitor. Importantly, anti-sera from immunized animals did not mediate antibody-dependent enhancement (ADE) of S-protein-mediated entry under conditions in which Zika virus ADE was readily observed. These data suggest that an RBD-based vaccine for SARS-CoV-2 could be safe and effective.","SummaryCylindrical protein scaffolds are thought to stabilize membrane tubules, preventing membrane fission. In contrast, Snead et al. find that when scaffold proteins assemble, bulky disordered domains within them become acutely concentrated, generating steric pressure that destabilizes tubules, driving fission.AbstractCellular membranes are continuously remodeled. The crescent-shaped bin-amphiphysinrvs (BAR) domains remodel membranes in multiple cellular pathways. Based on studies of BAR domains in isolation, the current paradigm is that they polymerize into cylindrical scaffolds that stabilize lipid tubules, preventing membrane fission. But in nature BAR domains are often part of multi-domain proteins that contain large intrinsically-disordered regions. Using in vitro and live cell assays, here we show that full-length BAR domain-containing proteins, rather than stabilizing membrane tubules, are instead surprisingly potent drivers of membrane fission. Specifically, when BAR scaffolds assemble at membrane surfaces, their bulky disordered domains become crowded, generating steric pressure that destabilizes lipid tubules. More broadly, we observe this behavior with BAR domains that have a range of curvatures. These data challenge the idea that cellular membranes adopt the curvature of BAR scaffolds, suggesting instead that the ability to concentrate disordered domains is the key requirement for membrane remodeling and fission by BAR domain-containing proteins.","AbstractSchool closure is often considered as an option to mitigate influenza epidemics because of its potential to reduce transmission in children and then in the community. The policy is still however highly debated because of controversial evidence. Moreover, the specific mechanisms leading to mitigation are not clearly identified.We introduced a stochastic spatial age-specific metapopulation model to assess the role of holiday-associated behavioral changes and how they affect seasonal influenza dynamics. The model is applied to Belgium, parameterized with country-specific data on social mixing and travel, and calibrated to the 2008/2009 influenza season. It includes behavioral changes occurring during weekend vs. weekday, and holiday vs. school-term. Several experimental scenarios are explored to identify the relevant social and behavioral mechanisms.Stochastic numerical simulations show that holidays considerably delay the peak of the season and mitigate its impact. Changes in mixing patterns are responsible for the observed effects, whereas changes in travel behavior do not alter the epidemic. Weekends are important in slowing down the season by periodically dampening transmission. Christmas holidays have the largest impact on the epidemic, however later school breaks may help in reducing the epidemic size, stressing the importance of considering the full calendar. An extension of the Christmas holiday of 1 week may further mitigate the epidemic.Changes in the way individuals establish contacts during holidays are the key ingredient explaining the mitigating effect of regular school closure. Our findings highlight the need to quantify these changes in different demographic and epidemic contexts in order to provide accurate and reliable evaluations of closure effectiveness. They also suggest strategic policies in the distribution of holiday periods to minimize the epidemic impact.","Novel SARS coronavirus (SARS-CoV-2) has caused a pandemic condition world-wide and has been declared as public health emergency of International concern by WHO in a very short span of time. The community transmission of this highly infectious virus has severely affected various parts of China, Italy, Spain and USA among others. The prophylactic solution against SARS-CoV-2 infection is challenging due to the high mutation rate of its RNA genome. Herein, we exploited a next generation vaccinology approach to construct a multi-epitope vaccine candidate against SARS-CoV-2 with high antigenicity, safety and efficacy to combat this deadly infectious agent. The whole proteome was scrutinized for the screening of highly conserved, antigenic, non-allergen and non-toxic epitopes having high population coverage that can elicit both humoral and cellular mediated immune response against COVID-19 infection. These epitopes along with four different adjuvants were utilized to construct a multi-epitope vaccine candidate that can generate strong immunological memory response having high efficacy in humans. Various physiochemical analyses revealed the formation of a stable vaccine product having a high propensity to form a protective solution against the detrimental SARS-CoV-2 strain with high efficacy. The vaccine candidate interacted with immunological receptor TLR3 with high affinity depicting the generation of innate immunity. Further, the codon optimization and in silico expression show the plausibility of the high expression and easy purification of the vaccine product. Thus, this present study provides an initial platform of the rapid generation of an efficacious protective vaccine for combating COVID-19.","We propose a novel numerical method able to determine efficiently and effectively the relationship of complementarity between portions of proteins surfaces. This innovative and general procedure, based on the representation of the molecular iso-electron density surface in terms of 2D Zernike polynomials, allows the rapid and quantitative assessment of the geometrical shape complementarity between interacting proteins, that was unfeasible with previous methods. We first tested the method with a large dataset of known protein complexes obtaining an overall area under the ROC curve of 0.69 in the blind recognition of binding sites and then applied it to investigate the features of the interaction between the Spike protein of SARS-Cov-2 and human cellular receptors. Our results indicate that SARS-CoV-2 uses a dual strategy: its spike protein could also interact with sialic acid receptors of the cells in the upper airways, in addition to the known interaction with Angiotensin-converting enzyme 2.","The recent outbreak of a new coronavirus (SARS-CoV-2) in Wuhan, China, underscores the need for understanding the evolutionary processes that drive the emergence and adaptation of zoonotic viruses in humans. Here, we show that recombination in betacoronaviruses, including human-infecting viruses like SARS-CoV and MERS-CoV, frequently encompasses the Receptor Binding Domain (RBD) in the Spike gene. We find that this common process likely led to a recombination event at least 11 years ago in an ancestor of the SARS-CoV-2 involving the RBD. As a result of this recombination event, SARS-CoV and SARS-CoV-2 share a similar genotype in RBD, including two insertions (positions 432-436 and 460-472), and alleles 427N and 436Y. Both 427N and 436Y belong to a helix that interacts with the human ACE2 receptor. Ancestral state analyses revealed that SARS-CoV-2 differentiated from its most recent common ancestor with RaTG13 by accumulating a significant number of amino acid changes in the RBD. In sum, we propose a two-hit scenario in the emergence of the SARS-CoV-2 virus whereby the SARS-CoV-2 ancestors in bats first acquired genetic characteristics of SARS-CoV by incorporation of a SARS-like RBD through recombination before 2009, and subsequently, the lineage that led to SARS-CoV-2 accumulated further, unique changes specifically in the RBD.","AbstractAdenoviruses are double-strained DNA viruses found in a great number of vertebrates, including humans. In order to understand their transmission dynamics, it is crucial, even from a human health perspective, to investigate how host traits influence their prevalence. Bats are important reservoirs for Adenoviruses, and here we use the results of recent screenings in Western Europe to evaluate the association between characteristic traits of bat species and their probability of hosting Adenoviruses, taking into account their phylogenetic relationships. Across species, we found an important phylogenetic component in the presence of Adenoviruses and mating strategy as the most determinant factor conditioning the prevalence of Adenoviruses across bat species. Contrary to other more stable mating strategies (e.g. harems), swarming could hinder transmission of Adenoviruses since this strategy implies that contacts between individuals are too short. Alternatively, bat species with more promiscuous behavior may develop a stronger immune system. Outstandingly high prevalence of Adenoviruses was reported for the Iberian species Pipistrellus pygmaeus, P. kuhlii and Nyctalus lasiopterus and we found that in the latter, males were more likely to be infected by Adenoviruses than females, due to the immunosuppressing consequence of testosterone during the mating season. As a general trend across species, we found that the number of Adenoviruses positive individuals was different across localities and that the difference in prevalence between populations was correlated with their geographic distances (P. pygmaeus). These results increase our knowledge about the transmission mechanisms of Adenoviruses.Author SummaryAdenoviruses are DNA viruses with a wide range of vertebrate hosts, including humans, causing ocular, respiratory and gastrointestinal diseases. Here, we focus on the prevalence of Adenoviruses in bats, which are known to be natural reservoir of many viruses, using the results of recent screenings for prevalence of these viruses in 33 European bat species. Our aim is to find association between Adenoviruses prevalence and biological and behavioral host traits, considering the heterogeneity both between and within species in order to have a deeper understanding of mechanisms of viral transmission.Our results highlight the importance of mating strategy: bats species using swarming as mating strategy are less likely to be infected by Adenoviruses. Moreover, we found that locality of capture can explain a higher prevalence of Adenovirus within species. However, no general pattern has been found in the analysis at individual level, suggesting a strong species specificity and complex viral transmission dynamics.","ABSTRACTDuring lytic Kaposi\u2019s sarcoma-associated herpesvirus (KSHV) infection, the viral endonu-clease SOX promotes widespread degradation of cytoplasmic messenger RNA (mRNA). However, select mRNAs, including the transcript encoding interleukin-6 (IL-6), escape SOX-induced cleavage. IL-6 escape is mediated through a 3\u2019 UTR RNA regulatory element that overrides the SOX targeting mechanism. Here, we reveal that this protective RNA element functions to broadly restrict cleavage by a range of homologous and non-homologous viral endonucleases. However, it does not impede cleavage by cellular endonucleases. The IL-6 protective sequence may be representative of a larger class of nuclease escape elements, as we identified a similar protective element in the GADD45B mRNA. The IL-6 and GADD45B-derived elements display similarities in their sequence, putative structure, and several associated RNA binding proteins. However, the overall composition of their ribonucleoprotein complexes appears distinct, leading to differences in the breadth of nucleases restricted. These findings highlight how RNA elements can selectively control transcript abundance in the background of widespread virus-induced mRNA degradation.AUTHOR SUMMARYThe ability of viruses to control the host gene expression environment is crucial to promote viral infection. Many viruses express factors that reduce host gene expression through widespread mRNA decay. However, some mRNAs escape this fate, like the transcript encoding the immunoregulatory cytokine IL-6 during KSHV infection. IL-6 escape relies on an RNA regulatory element located in its 3\u2019UTR and involves the recruitment of a protective protein complex. Here, we show that this escape extends beyond KSHV to a variety of related and unrelated viral endonucleases. However, the IL-6 element does not protect against cellular endonucleases, revealing for the first time a virus-specific nuclease escape element. We identified a related escape element in the GADD45B mRNA, which displays several similarities with the IL-6 element. However, these elements assemble a largely distinct complex of proteins, leading to differences in the breadth of their protective capacity. Collectively, these findings reveal how a putative new class of RNA elements function to control RNA fate in the background of widespread mRNA degradation by viral endonucleases.","AbstractPathogens are embedded in a complex network of microparasites that can collectively or individually alter disease dynamics and outcomes. Chronic pathogens, for example, can either facilitate or compete with subsequent pathogens thereby exacerbating morbidity and mortality. Pathogen interactions are ubiquitous in nature, but poorly understood, particularly in wild populations. We report here on ten years of serological and molecular data in African lions, leveraging comprehensive demographic and behavioral data to utilize pathogen networks to test if chronic infections shape infection by acute pathogens. We combine network and community ecology approaches to assess broad network structure and characterize associations between pathogens across spatial and temporal scales. We found significant non-random structure in the lion-pathogen co-occurrence network and identified potential facilitative and competitive interactions between acute and chronic pathogens. Our results provide a novel insight for untangling the complex associations underlying pathogen co-occurrence networks.","AbstractThe SARS-CoV-2 pandemic has been growing exponentially, affecting nearly 900 thousand people and causing enormous distress to economies and societies worldwide. A plethora of analyses based on viral sequences has already been published, in scientific journals as well as through non-peer reviewed channels, to investigate SARS-CoV-2 genetic heterogeneity and spatiotemporal dissemination. We examined full genome sequences currently available to assess the presence of sufficient information for reliable phylogenetic and phylogeographic studies in countries with the highest toll of confirmed cases. Although number of-available full-genomes is growing daily, and the full dataset contains sufficient phylogenetic information that would allow reliable inference of phylogenetic relationships, country-specific SARS-CoV-2 datasets still present severe limitations. Studies assessing within country spread or transmission clusters should be considered preliminary at best, or hypothesis generating. Hence the need for continuing concerted efforts to increase number and quality of the sequences required for robust tracing of the epidemic.Significance StatementAlthough genome sequences of SARS-CoV-2 are growing daily and contain sufficient phylogenetic information, country-specific data still present severe limitations and should be interpreted with caution.","AbstractThe outbreak of coronavirus disease (COVID-19) in China caused by SARS-CoV-2 virus continually lead to worldwide human infections and deaths. It is currently no specific viral protein targeted therapeutics yet. Viral nucleocapsid protein is a potential antiviral drug target, serving multiple critical functions during the viral life cycle. However, the structural information of SARS-CoV-2 nucleocapsid protein is yet to be clear. Herein, we have determined the 2.7 \u00c5 crystal structure of the N-terminal RNA binding domain of SARS-CoV-2 nucleocapsid protein. Although overall structure is similar with other reported coronavirus nucleocapsid protein N-terminal domain, the surface electrostatic potential characteristics between them are distinct. Further comparison with mild virus type HCoV-OC43 equivalent domain demonstrates a unique potential RNA binding pocket alongside the \u03b2-sheet core. Complemented by in vitro binding studies, our data provide several atomic resolution features of SARS-CoV-2 nucleocapsid protein N-terminal domain, guiding the design of novel antiviral agents specific targeting to SARS-CoV-2.","ABSTRACTRibosomal frameshifting during the translation of RNA is implicated in both human disease and viral infection. While previous work has uncovered many mechanistic details about single RNA frameshifting kinetics in vitro, very little is known about how single RNA frameshift in living systems. To confront this problem, we have developed technology to quantify live-cell single RNA translation dynamics in frameshifted open reading frames. Applying this technology to RNA encoding the HIV-1 frameshift sequence reveals a small subset (~8%) of the translating pool robustly frameshift in living cells. Frameshifting RNA are preferentially in multi-RNA \u201ctranslation factories,\u201d are translated at about the same rate as non-frameshifting RNA (~2 aa/sec), and can continuously frameshift for more than four rounds of translation. Fits to a bursty model of frameshifting constrain frameshifting kinetic rates and demonstrate how ribosomal traffic jams contribute to the persistence of the frameshifting state. These data provide novel insight into retroviral frameshifting and could lead to new strategies to perturb the process in living cells.","AbstractAn unexplained outbreak of feline diarrhea and vomiting, negative for common enteric viral and bacterial pathogens, was subjected to viral metagenomics and PCR. We characterized from fecal samples the genome of a novel chapparvovirus we named fechavirus that was shed by 8/17 affected cats and different feline bocaviruses shed by 9/17 cats. Also detected were nucleic acids from attenuated vaccine viruses, members of the normal feline virome, viruses found in only one or two cases, and viruses likely derived from ingested food products. Epidemiological investigation of disease signs, time of onset, and transfers of affected cats between three facilities support a possible role for this new chapparvovirus in a highly contagious feline diarrhea and vomiting disease.","AbstractThe protozoan parasite Leishmania donovani (L. donovani) causes visceral leishmaniasis, a chronic infection which is fatal when untreated. While previous studies showed that L. donovani reprograms transcription to subvert host cell functions, it remains unclear whether the parasite also alters host mRNA translation to establish a successful infection. To assess this, we compared transcriptome-wide translation in primary mouse macrophages infected with L. donovani promastigotes or amastigotes using polysome-profiling. This identified ample selective changes in translation (3,127 transcripts) which were predicted to target central cellular functions by inducing synthesis of proteins related to chromatin remodeling and RNA metabolism while inhibiting those related to intracellular trafficking and antigen presentation. Parallel quantification of protein and mRNA levels for a set of transcripts whose translation was activated upon L. donovani infection (Papbpc1, Eif2ak2, and Tgfb) confirmed, as indicated by polysome-profiling, increased protein levels despite largely unaltered mRNA levels. Mechanistic in silico analyses suggested activated translation depending on the kinase mTOR (e.g. Pabpc1) and the RNA helicase eIF4A (e.g. Tgfb) during infection. Accordingly, treatment with mTOR inhibitors torin-1 or rapamycin reversed L. donovani-induced PABPC1 without affecting corresponding transcript levels. Similarly, the production of TGF-\u03b2 decreased in presence of the eIF4A inhibitor silvestrol despite unaltered Tgfb mRNA levels. Consistent with parasite modulation of host eIF4A-sensitive translation to promote infection, silvestrol suppressed L. donovani replication within macrophages. In contrast, parasite survival was favored under mTOR inhibition. In summary, infection-associated changes in translation of mTOR- and eIF4A-sensitive mRNAs contribute to modulate mRNA metabolism and immune responses in L. donovani-infected macrophages. Although the net outcome of such translation programs favours parasite propagation, individual translation programs appear to have opposing roles during L. donovani infection, thereby suggesting their selective targeting as key for therapeutic effects.Author SummaryFine-tuning the efficiency of mRNA translation into proteins allows cells to tailor their responses to stress without the need for synthesizing new mRNA molecules. It is well established that the protozoan parasite Leishmania donovani alters transcription of specific genes to subvert host cell functions. However, discrepancies between transcriptomic and proteomic data suggest that post-transcriptional regulatory mechanisms also contribute to modulate host gene expression programs during L. donovani infection. Herein, we report that one third of protein-coding mRNAs expressed in macrophages are differentially translated upon infection with L. donovani. Our computational analyses reveal that subsets of mRNAs encoding functionally related proteins share the same directionality of translational regulation, which is likely to impact metabolic and microbicidal activity of infected cells. We also show that upregulated translation of transcripts that encode central regulators of mRNA metabolism and inflammation is sensitive to the activation of mTOR or eIF4A during infection. Finally, we observe that inhibition of eIF4A activity reduces parasite survival within macrophages while selective blockade of mTOR has the opposite effect. Thus, our study points to a dual role for translational control of host gene expression during L. donovani infection and suggests that novel regulatory nodes could be targeted for therapeutic intervention.","AbstractThe basal ganglia and thalamus play an important role in cognition, procedural learning, eye movements, control of voluntary motor movements, emotional control, habit development, and are structures that are severely impacted by neurological disorders such as Alzheimer\u2019s disease, Parkinson\u2019s disease, or Tourette syndrome. To understand the structural connectivity of cortical and subcortical circuits in the healthy human brain could thus be of pivotal importance for detecting changes in this circuitry and to start early intervention, to assess the progress of movement rehabilitation, or the effectiveness of therapeutic approaches in neuropsychiatry. While conventional magnetic resonance imaging (MRI), positron emission tomography, or magnetoencephalography are able to provide detailed information about connectivity at the macro level, the sensitivity and specificity these imaging techniques put limits on the amount of detail one can obtain when measuring in vivo connectivity of human basal ganglia and thalamus. In contrast, the multiband diffusion echo planar imaging MRI sequence, which acquires multiple slices of the brain simultaneously, enables high resolution imaging of these brain structures with only short acquisition times at 3-Tesla field strength. Here, we introduce a novel protocol that allows us to generate comprehensive in vivo participant-specific probabilistic patterns and visualizations of the structural connections that exist within basal ganglia and thalamic nuclei. Moreover, we are able to map specific parcellations of these nuclei into sub-territories based on their connectivity with primary motor-, and somatosensory cortex. The detailed subcortical structural connectivity mapping introduced in this work could benefit early intervention and therapy methods for human movement rehabilitation and for treating neuropsychiatric disorders.","AbstractBackgroundEmerging and reemerging infectious diseases such as the novel Coronavirus disease, COVID-19 and Ebola pose a significant threat to global society and test the public health community\u2019s preparedness to rapidly respond to an outbreak with effective diagnostics and therapeutics. Recent advances in next generation sequencing technologies enable rapid generation of pathogen genome sequence data, within 24 hours of obtaining a sample in some instances. With these data, one can quickly evaluate the effectiveness of existing diagnostics and therapeutics using in silico approaches. The propensity of some viruses to rapidly accumulate mutations can lead to the failure of molecular detection assays creating the need for redesigned or newly designed assays.ResultsHere we describe a bioinformatics system named BioLaboro to identify signature regions in a given pathogen genome, design PCR assays targeting those regions, and then test the PCR assays in silico to determine their sensitivity and specificity. We demonstrate BioLaboro with two use cases: Bombali Ebolavirus (BOMV) and the novel Coronavirus 2019 (SARS-CoV-2). For the BOMV, we analyzed 30 currently available real-time reverse transcription-PCR assays against the three available complete genome sequences of BOMV. Only two met our in silico criteria for successful detection and neither had perfect matches to the primer/probe sequences. We designed five new primer sets against BOMV signatures and all had true positive hits to the three BOMV genomes and no false positive hits to any other sequence. Four assays are closely clustered in the nucleoprotein gene and one is located in the glycoprotein gene. Similarly, for the SARS-CoV-2, we designed five highly specific primer sets that hit all 145 whole genomes (available as of February 28, 2020) and none of the near neighbors.ConclusionsHere we applied BioLaboro in two real-world use cases to demonstrate its capability; 1) to identify signature regions, 2) to assess the efficacy of existing PCR assays to detect pathogens as they evolve over time, and 3) to design new assays with perfect in silico detection accuracy, all within hours, for further development and deployment. BioLaboro is designed with a user-friendly graphical user interface for biologists with limited bioinformatics experience.","Since 2007, repeated outbreaks of Zika virus (ZIKV) has affected millions of people worldwide and created global health concern with major complications like microcephaly and Guillain Barre syndrome. Generally, ZIKV transmits through mosquitoes (Aedes aegypti) like other flaviviruses, but reports show blood transfusion and sexual mode of ZIKV transmission which further makes the situation alarming. Till date, there is not a single Zika specific licensed drug or vaccine present in the market. However, in recent months, several antiviral molecules have been screened against viral and host proteins. Among those, -Epigallocatechin-3-gallate (EGCG), a green tea polyphenol has shown great virucidal potential against flaviviruses including ZIKV. However, the mechanistic understanding of EGCG targeting viral proteins is not yet entirely deciphered except little is known about its interaction with viral envelope protein and viral protease. Since literature has shown significant inhibitory interactions of EGCG against various kinases and bacterial DNA gyrases; we designed our study to find inhibitory actions of EGCG against ZIKV NS3 helicase. NS3 helicase is playing a significant role in viral replication by unwinding RNA after hydrolyzing NTP. We employed molecular docking and simulation approach and found significant interactions at ATPase site and also at RNA binding site. Further, the enzymatic assay has shown significant inhibition of NTPase activity with an IC50 value of 295.7 nM and Ki of 0.387 (error 0.034) micromolar. Our study suggests the possibility that EGCG could be considered as prime backbone molecule for further broad-spectrum and multitargeted inhibitor development against ZIKV and other flaviviruses.","AbstractAs of February 20, 2020, the 2019 novel coronavirus (renamed to COVID-19) spread to 30 countries with 2130 deaths and more than 75500 confirmed cases. COVID-19 is being compared to the infamous SARS coronavirus, which resulted, between November 2002 and July 2003, in 8098 confirmed cases worldwide with a 9.6% death rate and 774 deaths. Though COVID-19 has a death rate of 2.8% as of 20 February, the 75752 confirmed cases in a few weeks (December 8, 2019 to February 20, 2020) are alarming, with cases likely being under-reported given the comparatively longer incubation period. Such outbreaks demand elucidation of taxonomic classification and origin of the virus genomic sequence, for strategic planning, containment, and treatment. This paper identifies an intrinsic COVID-19 genomic signature and uses it together with a machine learning-based alignment-free approach for an ultra-fast, scalable, and highly accurate classification of whole COVID-19 genomes. The proposed method combines supervised machine learning with digital signal processing for genome analyses, augmented by a decision tree approach to the machine learning component, and a Spearman\u2019s rank correlation coefficient analysis for result validation. These tools are used to analyze a large dataset of over 5000 unique viral genomic sequences, totalling 61.8 million bp. Our results support a hypothesis of a bat origin and classify COVID-19 as Sarbecovirus, within Betacoronavirus. Our method achieves high levels of classification accuracy and discovers the most relevant relationships among over 5,000 viral genomes within a few minutes, ab initio, using raw DNA sequence data alone, and without any specialized biological knowledge, training, gene or genome annotations. This suggests that, for novel viral and pathogen genome sequences, this alignment-free whole-genome machine-learning approach can provide a reliable real-time option for taxonomic classification.","ABSTRACTThe COVID-19 pandemic, caused by the novel coronavirus SARS-CoV-2, has resulted in a global health and economic crisis of unprecedented scale. The high transmissibility of SARS-CoV-2, combined with a lack of population immunity and prevalence of severe clinical outcomes, urges the rapid development of effective therapeutic countermeasures. Here, we report the generation of synthetic nanobodies, known as sybodies, against the receptor-binding domain (RBD) of SARS-CoV-2. In an expeditious process taking only twelve working days, sybodies were selected entirely in vitro from three large combinatorial libraries, using ribosome and phage display. We obtained six strongly enriched sybody pools against the isolated RBD and identified 63 unique anti-RBD sybodies which also interact in the context of the full-length SARS-CoV-2 spike protein. It is anticipated that compact binders such as these sybodies could feasibly be developed into an inhalable drug that can be used as a convenient prophylaxis against COVID-19. Moreover, generation of polyvalent antivirals, via fusion of anti-RBD sybodies to additional small binders recognizing secondary epitopes, could enhance the therapeutic potential and guard against escape mutants. We present full sequence information and detailed protocols for the identified sybodies, as a freely accessible resource. This report will be updated as we further characterize the identified sybodies, in terms of affinities, scaled-up purification yields, and their potential to neutralize SARS-CoV-2 infections.","One of the reasons for the fast spread of SARS-CoV-2 is the lack of accuracy in detection tools in the clinical field. Molecular techniques, such as quantitative real-time RT-PCR and nucleic acid sequencing methods, are widely used to identify pathogens. For this particular virus, however, they have an overall unsatisfying detection rate, due to its relatively recent emergence and still not completely understood features. In addition, SARS-CoV-2 is remarkably similar to other Coronaviruses, and it can present with other respiratory infections, making identification even harder. To tackle this issue, we propose an assisted detection test, combining molecular testing with deep learning. The proposed approach employs a state-of-the-art deep convolutional neural network, able to automatically create features starting from the genome sequence of the virus. Experiments on data from the Novel Coronavirus Resource (2019nCoVR) show that the proposed approach is able to correctly classify SARS-CoV-2, distinguishing it from other coronavirus strains, such as MERS-CoV, HCoV-NL63, HCoV-OC43, HCoV-229E, HCoV-HKU1, and SARS-CoV regardless of missing information and errors in sequencing (noise). From a dataset of 553 complete genome non-repeated sequences that vary from 1,260 to 31,029 bps in length, the proposed approach classifies the different coronaviruses with an average accuracy of 98.75% in a 10-fold cross-validation, identifying SARS-CoV-2 with an AUC of 98%, specificity of 0.9939 and sensitivity of 1.00 in a binary classification. Then, using the same basis, we classify SARS-CoV-2 from 384 complete viral genome sequences with human host, that contain the gene ORF1ab from the NCBI with a 10-fold accuracy of 98.17% , a specificity of 0.9797 and sensitivity of 1.00. These preliminary results seem encouraging enough to identify deep learning as a promising research venue to develop assisted detection tests for SARS-CoV-2. At this end the interaction between viromics and deep learning, will hopefully help to solve global infection problems. In addition, we offer our code and processed data to be used for diagnostic purposes by medical doctors, virologists and scientists involved in solving the SARS-CoV-2 pandemic. As more data become available we will update our system.","AbstractThe objective of this study was to evaluate how the inclusion of a blend of essential oils in milk replacer (MR) affects different outcomes of dairy heifers. The outcomes evaluated: feed intake, performance, body development, blood cells and metabolites, insulin-like growth factor-1 (IGF-1), rumen fermentation, fecal scores and respiratory scores. All outcomes were evaluated during pre-weaning (4 \u2013 60 d of age), and carryover effects on post-weaning (61 \u2013 90 d of age) periods. The experimental units utilized were 29 newborn Holstein \u00d7 Gyr crossbred dairy heifers, with genetic composition of 5/8 or more Holstein and 3/8 or less Gyr and body weight (BW) at birth of 32.2 \u00b1 5.2 kg. Experimental units were randomly assigned to either a control group (CON, n = 15) or a treatment group, consisting of supplementation of a blend of essential oils (BEO, n = 14) with1 g/d/calf (Apex Calf, Adisseo, China). During the pre-weaning phase, all heifers were fed 5 L of MR/d reconstituted at 15% (dry matter basis), divided into two equal meals. Water and starter were provided ad libitum. During the post-weaning, animals received a maximum 3 kg of starter/d, and ad libitum corn silage, divided into two meals. The outcomes feed intake, fecal and respiratory scores were evaluated daily. BW was measured every three days, while body development was recorded weekly. Blood samples were collected on 0, 30 and 60 d of age for total blood cell count, weekly to determinate \u00df-hydroxybutyrate, urea and glucose, and biweekly for IGF-1. Ruminal parameters (pH, volatile fatty acids, ammonia-N and acetate:proprionate proportion - C2:C3) were measured each 14 days. A randomized complete block design with an interaction between treatment and week was the experimental method of choice to test the hypothesis of effect of BEO on all outcomes. An ANOVA procedure was used for continuous outcomes and a non-parametric test was used for the ordered categorical outcomes both adopting a C.I. = 95%. Results indicated that there was not enough evidence to accept the alternative hypothesis of effect of BEO in MR on: feed intake, performance, body development and blood metabolites during both pre-weaning and post-weaning periods. However, results indicated that the inclusion of BEO in MR significantly affects the proportion of C2:C3 during pre and post-weaning (P \u2264 0.05). Similarly, the effect is significant for basophil (P \u2264 0.001) and platelet (P \u2264 0.05) counts in pre-weaning. The interaction between week and treatment was also significant for lymphocytes (P \u2264 0.001), revealing a long-term treatment immunological effect. Lastly, the effect on fecal scores was also significant (P \u2264 0.05) during pre-weaning, with lower values for BEO. BEO contributed for ruminal manipulation on pre-weaning and carry over effect on post-weaning; immunity improvement and a decrease morbidity of neonatal diarrhea in pre-weaning phase.","1AbstractInfectious disease interventions like contact precautions and vaccination have proven effective in disease control and elimination. The priority given to interventions can depend strongly on how virulent the pathogen is, and interventions may also depend partly for their success on social processes that respond adaptively to disease dynamics. However, mathematical models of competition between pathogen strains with differing natural history profiles typically assume that human behaviour is fixed. Here, our objective is to model the influence of social behaviour on the competition between pathogen strains with differing virulence. We couple a compartmental Susceptible-Infectious-Recovered model for a resident pathogen strain and a mutant strain with higher virulence, with a differential equation of a population where individuals learn to adopt protective behaviour from others according to the prevalence of infection of the two strains and the perceived severity of the respective strains in the population. We perform invasion analysis, time series analysis and phase plane analysis to show that perceived severities of pathogen strains and the efficacy of infection control against them can greatly impact the invasion of more virulent strain. We demonstrate that adaptive social behaviour enables invasion of the mutant strain under plausible epidemiological scenarios, even when the mutant strain has a lower basic reproductive number than the resident strain. Surprisingly, in some situations, increasing the perceived severity of the resident strain can facilitate invasion of the more virulent mutant strain. Our results demonstrate that for certain applications, it may be necessary to include adaptive social behaviour in models of the emergence of virulent pathogens, so that the models can better assist public health efforts to control infectious diseases.","ABSTRACTOne of the characteristics of RNA interference (RNAi) is systemic spread of the silencing signal among cells and tissues throughout the organism. Systemic RNAi, initiated by double-stranded RNA (dsRNA) ingestion, has been reported in diverse invertebrates, including honey bees, demonstrating environmental RNA uptake that undermines homologous gene expression. However, the question why any organism would take up RNA from the environment has remained largely unanswered. Here, we report on horizontal RNA flow among honey bees mediated by secretion and ingestion of worker and royal jelly diets. We show that ingested dsRNA spreads through the bee\u2019s hemolymph associated with a protein complex. The systemic dsRNA is secreted with the jelly and delivered to larvae via ingestion. Furthermore, we demonstrate that transmission of jelly-secreted dsRNA to larvae is biologically active and triggers gene knockdown that lasts into adulthood. Finally, RNA extracted from worker and royal jellies harbor differential naturally occurring RNA populations. Some of these RNAs corresponded to honey bee protein coding genes, transposable elements, non-coding RNA as well as bacteria, fungi and viruses. These results reveal an inherent property of honey bees to share RNA among individuals and generations. Thus, our findings suggest a transmissible RNA pathway, playing a role in social immunity and epigenetic signaling between honey bees and potentially among other closely interacting organisms.SIGNIFICANCEHoney bees are eusocial insects, living in a colony that is often described as a superorganism. RNA mobility among cells of an organism has been documented in plants and animals. Here we show that RNA spreads further in honey bees, and is horizontally transferred between individuals and across generations. We found that honey bees share biologically active RNA through secretion and ingestion of worker and royal jellies. Such RNA initiates RNA interference, which is a known defense mechanism against viral infection. Furthermore, we characterized diverse RNA profiles of worker and royal jelly, including fragmented viral RNA. Our findings demonstrate a transmissible RNA pathway with potential roles in social immunity and epigenetic signaling among members of the hive.","AbstractWe propose and discuss a model for flagellar mechanics in Euglena gracilis. We show that the peculiar non-planar shapes of its beating flagellum, dubbed \u201cspinning lasso\u201d, arise from the mechanical interactions between two of its inner components, namely, the axoneme and the paraflagellar rod. The spontaneous shape of the axoneme and the resting shape of the paraflagellar rod are incompatible. The complex non-planar configurations of the coupled system emerge as the energetically optimal compromise between the two antagonistic components. The model is able to reproduce the experimentally observed flagellar beats and their characteristic spinning lasso geometric signature, namely, travelling waves of torsion with alternating sing along the length of the flagellum.","AbstractBunyaviruses are significant human pathogens, causing diseases ranging from hemorrhagic fevers to encephalitis. Among these viruses, La Crosse virus (LACV), a member of the California serogroup, circulates in the eastern and midwestern United States. While LACV infection is often asymptomatic, dozens of cases of encephalitis are reported yearly. Unfortunately, no antivirals have been approved to treat LACV infection. Here, we developed a method to rapidly test potential antivirals against LACV infection. From this screen, we identified several potential antiviral molecules, including known antivirals. Additionally, we identified many novel antivirals that exhibited antiviral activity without affecting cellular viability. Valinomycin, a potassium ionophore, was among our top targets. We found that valinomycin exhibited potent anti-LACV activity in multiple cell types in a dose-dependent manner. Valinomycin did not affect particle stability or infectivity, suggesting that it may preclude virus replication by altering cellular potassium ions, a known determinant of LACV entry. We extended these results to other ionophores and found that the antiviral activity of valinomycin extended to other viral families including bunyaviruses (Rift Valley fever virus, Keystone virus), enteroviruses (Coxsackievirus, rhinovirus), flavirivuses (Zika), and coronaviruses (229E and MERS-CoV). In all viral infections, we observed significant reductions in virus titer in valinomycin-treated cells. In sum, we demonstrate the importance of potassium ions to virus infection, suggesting a potential therapeutic target to disrupt virus replication.ImportanceNo antivirals are approved for the treatment of bunyavirus infection. The ability to rapidly screen compounds and identify novel antivirals is one means to accelerate drug discovery for viruses with no approved treatments. We used this approach to screen hundreds of compounds against La Crosse virus, an emerging bunyavirus that causes significant disease, including encephalitis. We identified several known and previously unidentified antivirals. We focused on a potassium ionophore, valinomycin, due to its promising in vitro antiviral activity. We demonstrate that valinomycin, as well as a selection of other ionophores, exhibits activity against La Crosse virus as well as several other distantly related bunyaviruses. We finally observe that valinomycin has activity against a wide array of human viral pathogens, suggesting that disrupting potassium ion homeostasis with valinomycin may be a potent host pathway to target to quell virus infection.","AbstractEarly warning signals (EWS) identify systems approaching a critical transition, where the system undergoes a sudden change in state. For example, monitoring changes in variance or autocorrelation offers a computationally inexpensive method which can be used in real-time to assess when an infectious disease transitions to elimination. EWS have a promising potential to not only be used to monitor infectious diseases, but also to inform control policies to aid disease elimination. Previously, potential EWS have been identified for prevalence data, however the prevalence of a disease is often not known directly. In this work we identify EWS for incidence data, the standard data type collected by the Centers for Disease Control and Prevention (CDC) or World Health Organization (WHO). We show, through several examples, that EWS calculated on simulated incidence time series data exhibit vastly different behaviours to those previously studied on prevalence data. In particular, the variance displays a decreasing trend on the approach to disease elimination, contrary to that expected from critical slowing down theory; this could lead to unreliable indicators of elimination when calculated on real-world data. We derive analytical predictions which can be generalised for many epidemiological systems, and we support our theory with simulated studies of disease incidence. Additionally, we explore EWS calculated on the rate of incidence over time, a property which can be extracted directly from incidence data. We find that although incidence might not exhibit typical critical slowing down properties before a critical transition, the rate of incidence does, presenting a promising new data type for the application of statistical indicators.Author summaryThe threat posed by infectious diseases has a huge impact on our global society. It is therefore critical to monitor infectious diseases as new data become available during control campaigns. One obstacle in observing disease emergence or elimination is understanding what influences noise in the data and how this fluctuates when near to zero cases. The standard data type collected is the number of new cases per day/month/year but mathematical modellers often focus on data such as the total number of infectious people, due to its analytical properties. We have developed a methodology to monitor the standard type of data to inform whether a disease is approaching emergence or disease elimination. We have shown computationally how fluctuations change as disease data get closer towards a tipping point and our insights highlight how these observed changes can be strikingly different when calculated on different types of data.","AbstractEmerging infectious diseases increasingly threaten wildlife populations. Most studies focus on managing short-term epidemic properties, such as controlling early outbreaks. Predicting long-term endemic characteristics with limited retrospective data is more challenging. We used individual-based modelling informed by individual variation in pathogen load and transmissibility to predict long-term impacts of a lethal, transmissible cancer on Tasmanian devil (Sarcophilus harrisii) populations. For this, we employed Approximate Bayesian Computation to identify model scenarios that best matched known epidemiological and demographic system properties derived from ten years of data after disease emergence, enabling us to forecast future system dynamics. We show that the dramatic devil population declines observed thus far are likely attributable to transient dynamics. Only 21% of matching scenarios led to devil extinction within 100 years following devil facial tumour disease (DFTD) introduction, whereas DFTD faded out in 57% of simulations. In the remaining 22% of simulations, disease and host coexisted for at least 100 years, usually with long-period oscillations. Our findings show that pathogen extirpation or host-pathogen coexistence are much more likely than the DFTD-induced devil extinction, with crucial management ramifications. Accounting for individual-level disease progression and the long-term outcome of devil-DFTD interactions at the population-level, our findings suggest that immediate management interventions are unlikely to be necessary to ensure the persistence of Tasmanian devil populations. This is because strong population declines of devils after disease emergence do not necessarily translate into long-term population declines at equilibria. Our modelling approach is widely applicable to other host-pathogen systems to predict disease impact beyond transient dynamics.","AbstractInspired by bioactive biaryl-containing natural products found in plants and the marine environment, a series of synthetic compounds belonging to the azaBINOL chiral ligand family was evaluated for antiviral activity against HIV-1. Testing of 39 unique azaBINOLs in a singleround infectivity assay resulted in the identification of three promising antiviral compounds, including 7-isopropoxy-8-(naphth-1-yl)quinoline (azaBINOL B#24), which exhibited low-micromolar activity. The active compounds and several close structural analogues were further tested against three different HIV-1 envelope pseudotyped viruses as well as in a full-virus replication system (EASY-HIT). Mode-of-action studies using a time-of-addition assay indicated that azaBINOL B#24 acts after viral entry but before viral assembly and budding. HIV-1 reverse transcriptase (RT) assays that individually test for polymerase and RNase H activity were used to demonstrate that B#24 inhibits RNase H activity, most likely allosterically. Further binding analysis using bio-layer interferometry (BLI) showed that B#24 interacts with HIV-1 RT in a highly specific manner. These results indicate that azaBINOL B#24 is a potentially viable, novel lead for the development of new HIV-1 RNase H inhibitors. Furthermore, this study demonstrates that the survey of libraries of synthetic compounds, designed purely with the goal of facilitating chemical synthesis in mind, may yield unexpected and selective drug leads for the development of new antiviral agents.","AbstractA subset of intracellular mono-ADP-ribosyltransferases diphtheria toxin-like (ARTDs, aka mono-PARPs) is induced by type I interferons. Some of these mono-ARTDs feature antiviral activity while certain RNA viruses, including Chikungunya virus (CHIKV), encode mono-ADP-ribosylhydrolases, suggesting a role for mono-ADP-ribosylation (MARylation) in host-virus conflicts. CHIKV expresses four non-structural proteins (nsP1-nsP4), with nsP3 containing a macrodomain that hydrolyzes and thereby reverses protein MARylation in vitro and in cells. This de-MARylation activity is essential as hydrolase inactivating mutations result in replication defective virus. However, the substrates of MARylation during CHIKV infection are unknown and thus it is unclear how the macrodomain contributes to virus replication and how mono-ARTD-dependent MARylation confers antiviral immunity. We identified ARTD10 and ARTD12 as restriction factors for CHIKV replication in a catalytic activity-dependent manner. CHIKV replication requires processing of the non-structural polyprotein nsP1-4 by the nsP2-encoded protease and the assembly of the four individual nsPs into a functional replication complex. Expression of ARTD10 and ARTD12 resulted in a reduction of processed nsPs. Similarly, MAR hydrolase inactive CHIKV replicon mutants revealed a decrease in processed nsPs, comparable to an nsP2 protease defective mutant. This suggested that the macrodomain contributes to nsP2 protease activity. In support, a hydrolase-deficient virus was complemented by a protease-deficient virus. We hypothesized that MARylation regulates the proteolytic function of nsP2. Indeed, we found that nsP2 is MARylated by ARTD10. This inhibited nsP2 protease activity, thereby preventing polyprotein processing and consequently virus replication. This inhibition was antagonized by the MAR hydrolase activity of nsP3. Together, our findings provide a mechanistic explanation for the need of the viral MAR hydrolase for efficient replication of CHIKV.Author SummaryInfectious diseases still pose major health threats. Especially fast evolving viruses find ever new strategies to manipulate the immune response. With climate warming and increased human mobility vector-borne pathogens like Chikungunya virus (CHIKV) spread and cause world-wide epidemics. Beyond the acute phase, CHIKV patients regularly suffer from chronic rheumatism. This entails a decline in life quality and an economic burden. To date no drugs are approved and the mode of pathogenesis remains elusive. Here we describe a mechanistic function of the CHIKV nsP3 macrodomain. We found that the viral nsP2 is mono-ADP-ribosylated interfering with its auto-proteolytic function. The nsP3 macrodomain removes this modification and restores the protease activity that is essential for replication. Because macrodomains are highly conserved they might represent broad antiviral targets.","ABSTRACTG-quadruplexes are non-canonical nucleic acid structures that control transcription, replication, and recombination in organisms. G-quadruplexes are present in eukaryotes, prokaryotes, and viruses. In the latter, mounting evidence indicates their key biological activity. Since data on viruses are scattered, we here present a comprehensive analysis of putative G-quadruplexes in the genome of all known viruses that can infect humans. We show that the presence, distribution, and location of G-quadruplexes are features characteristic of each virus class and family. Our statistical analysis proves that their presence within the viral genome is orderly arranged, as indicated by the possibility to correctly assign up to two-thirds of viruses to their exact class based on the G-quadruplex classification. For each virus we provide: i) the list of all G-quadruplexes formed by GG-, GGG- and GGGG-islands present in the genome (positive and negative strands), ii) their position in the viral genome along with the known function of that region, iii) the degree of conservation among strains of each G-quadruplex in its genome context, iv) the statistical significance of G-quadruplex formation. This information is accessible from a database (http://www.medcomp.medicina.unipd.it/main_site/doku.php?id=g4virus) to allow the easy and interactive navigation of the results. The availability of these data will greatly expedite research on G-quadruplex in viruses, with the possibility to accelerate finding therapeutic opportunities to numerous and some fearsome human diseases.","AbstractThe outbreak of the 2019 novel coronavirus (SARS-CoV-2) has infected thousands of people with a large number of deaths across 26 countries. The sudden appearance of the virus leads to the limited existing therapies for SARS-CoV-2. Therefore, vaccines and antiviral medicines are in desperate need. This study took immune-informatics approaches to identify B- and T-cell epitopes for surface glycoprotein (S) of SARS-CoV-2, followed by estimating their antigenicity and interactions with the human leukocyte antigen (HLA) alleles. We identified four B cell epitopes, two MHC class-I and nine MHC class-II binding T-cell epitopes, which showed highly antigenic features. Allergenicity, toxicity and physiochemical properties analysis confirmed the specificity and selectivity of epitopes. The stability and safety of epitopes were confirmed by digestion analysis. No mutations were observed in all the selected B- and T-cell epitopes across all isolates from different locations worldwide. Epitopes were thus identified and some of them can be potential candidates for vaccine development.","AbstractThe virus receptor is key for viral infection of host cells. Identification of the virus receptor is still challenging at present. Our previous study has shown that human virus receptor proteins have some unique features including high level of N-glycosylation, high number of interaction partners and high expressions. Here, we built a random-forest model to identify human virus receptorome from human cell membrane proteins with accepted accuracy based on combination of unique features of human virus receptors and protein sequences. A total of 729 human cell membrane proteins were predicted to constitute the receptorome for the human-infecting virome. Combination of the random-forest model with protein-protein interactions between human and viruses predicted in previous studies further predicted receptors for 693 human-infecting viruses, such as the Enterovirus, Norovirus and West Nile virus. Finally, we predicted the candidate alternative receptors for the 2019-nCoV. The study would greatly facilitate identification of receptors for human-infecting virome.","Middle East respiratory syndrome coronavirus (MERS-CoV) continues to infect humans via the dromedary camel reservoir and can transmit between humans, most commonly via nosocomial transmission. Currently, no licensed vaccine is available. Previously we showed that vaccination of transgenic mice with ChAdOx1 MERS, encoding the MERS S protein, prevented disease upon lethal challenge. In the current study we show that rhesus macaques seroconverted rapidly after a single intramuscular vaccination with ChAdOx1 MERS. Upon MERS-CoV challenge vaccinated animals were protected against respiratory injury and pneumonia and had a reduction in viral load in lung tissue of several logs. Furthermore, we did not detect MERS-CoV replication in type I and II pneumocytes of ChAdOx1 MERS vaccinated animals. A prime-boost regimen of ChAdOx1 MERS boosted antibody titers, and viral replication was completely absent from the respiratory tract tissue of these rhesus macaques. Finally, we investigated the ability of ChAdOx1 MERS to protect against six different MERS-CoV strains, isolated between 2012 to 2018, from dromedary camels and humans in the Middle East and Africa. Antibodies elicited by ChAdOx1 MERS in rhesus macaques were able to neutralize all MERS-CoV strains. Vaccination of transgenic hDPP4 mice with ChAdOx1 MERS completely protected the animals against disease and lethality for all different MERS-CoV strains. The data support further clinical development of ChAdOx1 MERS supported by CEPI.","AbstractThe use of metabolomics profiling to understand metabolism under different physiological states has increased in recent years, which created the need for robust analytical platforms. Here, we present a validated method for targeted and semi-quantitative analysis of 102 polar metabolites that covers major metabolic pathways from 24 classes in a single 17.5-min assay. The method has been optimized for a wide range of biological matrices from various organisms, and involves automated sample preparation, and data processing using in-house developed R package. To ensure reliability, the method was validated for accuracy, precision, selectivity, specificity, linearity, recovery, and stability according to European Medicines Agency guidelines. We demonstrated excellent repeatability of the retention times (CV&lt;4%), calibration curves (R2\u22650.980) in their respective wide dynamic concentration ranges (CV&lt;3%), and concentrations (CV&lt;25%) of quality control samples interspersed within 25 batches analyzed over a period of one-year. The robustness was demonstrated through high correlation between metabolite concentrations measured using our method and NIST reference values (R2=0.967), including cross-platform comparability against the BIOCRATES AbsoluteIDQp180 kit (R2=0.975) and NMR analyses (R2=0.884). We have shown that our method can be successfully applied in many biomedical research fields and clinical trials, including epidemiological studies for biomarker discovery. In summary, a thorough validation demonstrated that our method is reproducible, robust, reliable, and suitable for metabolomics studies.","ABSTRACTA new betacoronavirus named SARS-CoV-2 has emerged as a new threat to global health and economy. A promising target for both diagnosis and therapeutics treatments of the new disease named COVID-19 is the coronavirus (CoV) spike (S) glycoprotein. By constant-pH Monte Carlo simulations and the PROCEEDpKa method, we have mapped the electrostatic epitopes for four monoclonal antibodies and the angiotensin-converting enzyme 2 (ACE2) on both SARS-CoV-1 and the new SARS-CoV-2 S receptor binding domain (RBD) proteins. We also calculated free energy of interactions and shown that the S RBD proteins from both SARS viruses binds to ACE2 with similar affinities. However, the affinity between the S RBD protein from the new SARS-CoV-2 and ACE2 is higher than for any studied antibody previously found complexed with SARS-CoV-1. Based on physical chemical analysis and free energies estimates, we can shed some light on the involved molecular recognition processes, their clinical aspects, the implications for drug developments, and suggest structural modifications on the CR3022 antibody that would improve its binding affinities for SARS-CoV-2 and contribute to address the ongoing international health crisis.","SUMMARYCurrently, COVID-19 caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has been widely spread around the world; nevertheless, so far there exist no specific antiviral drugs for treatment of the disease, which poses great challenge to control and contain the virus. Here, we reported a research finding that SARS-CoV-2 invaded host cells via a novel route of CD147-spike protein (SP). SP bound to CD147, a receptor on the host cells, thereby mediating the viral invasion. Our further research confirmed this finding. First, in vitro antiviral tests indicated Meplazumab, an anti-CD147 humanized antibody, significantly inhibited the viruses from invading host cells, with an EC50 of 24.86 \u03bcg/mL and IC50 of 15.16 \u03bcg/mL. Second, we validated the interaction between CD147 and SP, with an affinity constant of 1.85\u00d710\u22127M. Co-Immunoprecipitation and ELISA also confirmed the binding of the two proteins. Finally, the localization of CD147 and SP was observed in SARS-CoV-2 infected Vero E6 cells by immuno-electron microscope. Therefore, the discovery of the new route CD147-SP for SARS-CoV-2 invading host cells provides a critical target for development of specific antiviral drugs.","A pattern in which nucleotide transitions are favored several-fold over transversions is common in molecular evolution. When this pattern occurs among amino acid replacements, explanations often invoke an effect of selection, on the grounds that transitions are more conservative in their effects on proteins. However, the underlying hypothesis of conservative transitions has never been tested directly. Here we assess support for this hypothesis using direct evidence: the fitness effects of mutations in actual proteins, measured via individual or paired growth experiments. We assembled data from 8 published studies, ranging in size from 24 to 757 single-nucleotide mutations that change an amino acid. Every study has the statistical power to reveal significant effects of amino acid exchangeability, and most studies have the power to discern a binary conservative-vs-radical distinction. However, only one study suggests that transitions are significantly more conservative than transversions. In the combined set of 1239 replacements, the chance that a transition is more conservative than a transversion is 53 % (95 % confidence interval, 50 % to 56 %), compared to the null expectation of 50 %. We show that this effect is not large compared to that of most biochemical factors, and is not large enough to explain the several-fold bias observed in evolution. In short, available data have the power to verify the \"conservative transitions\" hypothesis if true, but suggest instead that selection on proteins plays at best a minor role in the observed bias.","AbstractIntroductionCryptococcosis is a ubiquitous opportunistic fungal disease caused by Cryptococcus neoformans var. grubii. It has high global morbidity and mortality among HIV patients and none-HIV carriers with 99% and 95% respectively. Furthermore, the increasing prevalence of undesired toxicity profile of antifungal, multi-drug resistant organism, and the scarcity of FDA authorized vaccines, where the hallmark in the present days. This study was undertaken to design a reliable multi-epitope peptide vaccine against highly conserved immunodominant heat shock 70KDa protein of Cryptococcus neoformans var. grubii that covers a considerable digit of the world population through implementing computational vaccinology approach.Materials and MethodsA total of 38 Sequences of Cryptococcus neoformans var. grubii\u2019s heat shock 70KDa protein were retrieved from NCBI protein database. Different prediction tools were used to analyze the aforementioned protein at Immune Epitope Database (IEDB) to discriminate the most promising T-cell and B-cell epitopes. Then the proposed epitopes were subjected to Population coverage analysis tool to compute global population\u2019s coverage. Finally, the projected epitopes were ranked based on their scores and binding modes through using Moe 2007 program.Outstanding Results and ConclusionOur prime vaccine candidate was a putative ten promising epitopes (ANYVQASEK, NYVQASEK, KSVEKPAS, TPQQPPAQ, YVYDTRGKL, FYRQGAFEL, FTQLVAAYL, FFGGKVLNF, FDYALVQHF, and FINAQLVDV). Together, these epitopes are forecasted to trigger T lymphocytes, B lymphocytes, and immunological memory with overall population coverage above 90%. Accordingly, our in silico vaccine is expected to be the future multi-epitope peptide vaccine against Cryptococcus neoformans var. grubii\u2019s heat shock 70KDa protein that covers a significant figure of the entire world citizens. Therefore, there is a definite need for experimental validation for the carefully chosen vaccine candidates in vitro and in vivo to fortify their antigenic and immunogenic potentials. Additionally, further computational studies are needed to be conducted in pathogens-derived Heat shock 70KDa protein family, as it believed to find universal epitopes that might be overlapped with other pathogens-derived Hsp70.","ABSTRACTBacterial biofilms represent an essential part of Earth\u2019s ecosystem that can cause multiple ecological, technological and health problems. The environmental resilience and sophisticated organization of biofilms are enabled by the extracellular matrix that creates a protective network of biomolecules around the bacterial community. Current anti-biofilm agents can interfere with extracellular matrix production but, being based on small molecules, are degraded by bacteria and rapidly diffuse away from biofilms. Both factors severely reduce their efficacy, while their toxicity to higher organisms create additional barriers to their practicality. In this paper we report on the ability of graphene quantum dots to effectively disperse mature Staphylococcus aureus biofilms, interfering with the self-assembly of amyloid fibers - a key structural component of the extracellular matrix. Mimicking peptide-binding biomolecules, graphene quantum dots form supramolecular complexes with phenol soluble modulins, the peptide monomers of amyloid fibers. Experimental and computational results show that graphene quantum dots efficiently dock near the N-terminus of the peptide and change the secondary structure of phenol soluble modulins, which disrupts their fibrillation and represents a novel strategy for mitigation of bacterial communities. GQD mediated staphylococcal biofilm dispersal. GQDs interact with PSM peptides and frustrate the fibrillation process. The reduction in amyloid fibers prevents robust stabilization of the biofilm. In addition, there is an increase in free monomeric and oligomeric PSM peptides which trigger dispersal events.","AbstractIdentifying viruses and phages in a metagenomics sample has important implication in improving human health, preventing viral outbreaks, and developing personalized medicine. With the rapid increase in data files generated by next generation sequencing, existing tools for identifying and annotating viruses and phages in metagenomics samples suffer from expensive running time. In this paper, we developed a stand-alone pipeline, FastViromeExplorer, for rapid identification and abundance quantification of viruses and phages in big metagenomic data. Both real and simulated data validated FastViromeExplorer as a reliable tool to accurately identify viruses and their abundances in large data, as well as in a time efficient manner.","Coronavirus disease 2019 (COVID-19) can be screened and diagnosed through the detection of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) by real-time reverse transcription polymerase chain reaction. SARS-CoV-2 nucleic acid amplification tests (NAATs) have been rapidly developed and quickly applied to clinical testing during the pandemic. However, studies evaluating the performance of these NAAT assays are limited. We evaluated the performance of four NAATs, which were marked by the Conformit\u00e9 Europ\u00e9enne and widely used in China during the pandemic. Results showed that the analytical sensitivity of the four assays was significantly lower than that claimed by the NAAT manufacturers. The limit of detection (LOD) of Daan, Sansure, and Hybribio NAATs was 3000 copies/mL, whereas the LOD of Bioperfectus NAATs was 4000 copies/mL. The results of the consistency test using 46 samples showed that Daan, Sansure, and Hybribio NAATs could detect the samples with a specificity of 100% (30/30) and a sensitivity of 100% (16 /16), whereas Bioperfectus NAAT detected the samples with a specificity of 100% (30/30) and a sensitivity 81.25% (13/16). The sensitivity of Bioperfectus NAAT was lower than that of the three other NAATs; this finding was consistent with the result that Bioperfectus NAAT had a higher LOD than the three other kinds of NAATs. The four above mentioned reagents presented high specificity; however, for the detection of the samples with low virus concentration, Bioperfectus reagent had the risk of missing detection. Therefore, the LOD should be considered in the selection of SARS-CoV-2 NAATs.","AbstractSARS-CoV-2 is known to infect the neurological, respiratory, enteric, and hepatic systems of human and has already become an unprecedented threat to global healthcare system. COVID-19, the most serious public condition caused by SARS-CoV-2 leads the world to an uncertainty alongside thousands of regular death scenes. Unavailability of specific therapeutics or approved vaccine has made the recovery of COVI-19 more troublesome and challenging. The present in silico study aimed to predict a novel chimeric vaccines by simultaneously targeting four major structural proteins via the establishment of ancestral relationship among different strains of coronaviruses. Conserved regions from the homologous protein sets of spike glycoprotein (S), membrane protein (M), envelope protein and nucleocapsid protein (N) were identified through multiple sequence alignment. The phylogeny analyses of whole genome stated that four proteins (S, E, M and N) reflected the close ancestral relation of SARS-CoV-2 to SARS-COV-1 and bat coronavirus. Numerous immunogenic epitopes (both T cell and B cell) were generated from the common fragments which were further ranked on the basis of antigenicity, transmembrane topology, conservancy level, toxicity and allergenicity pattern and population coverage analysis. Top putative epitopes were combined with appropriate adjuvants and linkers to construct a novel multiepitope subunit vaccine against COVID-19. The designed constructs were characterized based on physicochemical properties, allergenicity, antigenicity and solubility which revealed the superiority of construct V3 in terms safety and efficacy. Essential molecular dynamics and Normal Mode analysis confirmed minimal deformability of the refined model at molecular level. In addition, disulfide engineering was investigated to accelerate the stability of the protein. Molecular docking study ensured high binding affinity between construct V3 and HLA cells, as well as with different host receptors. Microbial expression and translational efficacy of the constructs were checked using pET28a(+) vector of E. coli strain K12. The development of preventive measures to combat COVID-19 infections might be aided the present study. However, the in vivo and in vitro validation might be ensured with wet lab trials using model animals for the implementation of the presented data.","AbstractCoronaviruses express a multifunctional papain-like protease, termed PLP2. PLP2 acts as a protease that cleaves the viral replicase polyprotein, and a deubiquitinating (DUB) enzyme which removes ubiquitin moieties from ubiquitin-conjugated proteins. Previous in vitro studies implicated PLP2 DUB activity as a negative regulator of the host interferon (IFN) response, but the role of DUB activity during virus infection was unknown. Here, we used X-ray structure-guided mutagenesis and functional studies to identify amino acid substitutions within the ubiquitin-binding surface of PLP2 that reduced DUB activity without affecting polyprotein processing activity. We engineered a DUB mutation (Asp1772 to Ala) into a murine coronavirus and evaluated the replication and pathogenesis of the DUB mutant virus (DUBmut) in cultured macrophages and in mice. We found that the DUBmut virus replicates similarly as the wild-type virus in cultured cells, but the DUBmut virus activates an IFN response at earlier times compared to the wild-type virus infection in macrophages, consistent with DUB activity negatively regulating the IFN response. We compared the pathogenesis of the DUBmut virus to the wild-type virus and found that the DUBmut-infected mice had a statistically significant reduction (p&lt;0.05) in viral titer in livers and spleens at day 5 post-infection, albeit both wild-type and DUBmut virus infections resulted in similar liver pathology. Overall, this study demonstrates that structure-guided mutagenesis aids the identification of critical determinants of PLP2-ubiquitin complex, and that PLP2 DUB activity plays a role as an interferon antagonist in coronavirus pathogenesis.ImportanceCoronaviruses employ a genetic economy by encoding multifunctional proteins that function in viral replication and also modify the host environment to disarm the innate immune response. The coronavirus papain-like protease 2 (PLP2) domain possesses protease activity, which cleaves the viral replicase polyprotein, and also DUB activity (de-conjugating ubiquitin/ubiquitin-like molecules from modified substrates) using identical catalytic residues. To separate the DUB activity from the protease activity, we employed a structure-guided mutagenesis approach and identified residues that are important for ubiquitin-binding. We found that mutating the ubiquitin-binding residues results in a PLP2 that has reduced DUB activity but retains protease activity. We engineered a recombinant murine coronavirus to express the DUB mutant and showed that the DUB mutant virus activated an earlier type I interferon response in macrophages and exhibited reduced pathogenesis in mice. The results of this study demonstrate that PLP2/DUB is an interferon antagonist and a virulence trait of coronaviruses.","ABSTRACTBleomycin is a powerful chemotherapeutic drug used to treat a variety of cancers. However, individual patients vary in their responses to bleomycin. The identification of genetic differences that underlie this response variation could improve treatment outcomes by tailoring bleomycin dosages to each patient. We used the model organism Caenorhabditis elegans to identify genetic determinants of bleomycin-response differences by performing linkage mapping on recombinants derived from a cross between the laboratory strain (N2) and a wild strain (CB4856). This approach identified a small genomic region on chromosome V that underlies bleomycin-response variation. Using near-isogenic lines and strains with CRISPR-Cas9 mediated deletions and allele replacements, we discovered that a novel nematode-specific gene (scb-1) is required for bleomycin resistance. Although the mechanism by which this gene causes variation in bleomycin responses is unknown, we suggest that a rare variant present in the CB4856 strain might cause differences in the potential stress-response function of scb-1 between the N2 and CB4856 strains, thereby leading to differences in bleomycin resistance.","AbstractSince December 2019, a novel coronavirus named 2019 coronavirus (2019-nCoV) has emerged in Wuhan of China and spread to several countries worldwide within just one month. Apart from fever and respiratory complications, acute kidney injury has been observed in some patients with 2019-nCoV. In a short period of time, angiotensin converting enzyme II (ACE2), have been proposed to serve as the receptor for the entry of 2019-nCoV, which is the same for severe acute respiratory syndrome coronavirus (SARS). To investigate the possible cause of kidney damage in 2019-nCoV patients, we used both published kidney and bladder cell atlas data and an independent unpublished kidney single cell RNA-Seq data generated in-house to evaluate ACE2 gene expressions in all cell types in healthy kidneys and bladders.Our results showed the enriched expression of all subtypes of proximal tubule cells of kidney and low but detectable levels of expression in bladder epithelial cells. These results indicated the urinary system is a potential route for 2019-nCoV infection, along with the respiratory system and digestion system. Our findings suggested the kidney abnormalities of SARS and 2019-nCoV patients may be due to proximal tubule cells damage and subsequent systematic inflammatory response induced kidney injury. Beyond that, laboratory tests of viruses and related indicators in urine may be needed in some special patients of 2019-nCoV.","AbstractPorcine epidemic diarrhea virus is an alphacoronavirus responsible for significant morbidity and mortality in pigs. A key determinant of viral tropism and entry, the PEDV spike protein is a key target for the host antibody response and a good candidate for a protein-based vaccine immunogen. We used electron microscopy to evaluate the PEDV spike structure, as well as pig polyclonal antibody responses to viral infection. The structure of the PEDV spike reveals a configuration similar to that of HuCoV-NL63. Several PEDV protein-protein interfaces are mediated by non-protein components including a glycan at Asn264 and two bound palmitoleic acid molecules. The polyclonal antibody response to PEDV infection shows a dominance of epitopes in the S1 region. This structural and immune characterization provides new insights into coronavirus spike stability determinants and explores the immune landscape of viral spike proteins.","AbstractThe prokaryotic transcriptome is shaped by transcriptional and posttranscriptional events that define the characteristics of an RNA, including transcript boundaries, the base modification status, and processing pathways to yield mature RNAs. Currently, a combination of several specialised short-read sequencing approaches and additional biochemical experiments are required to describe all transcriptomic features. In this study, we present native RNA sequencing of bacterial (E. coli) and archaeal (H. volcanii, P. furiosus) transcriptomes employing the Oxford Nanopore sequencing technology. Based on this approach, we could address multiple transcriptomic characteristics simultaneously with single-molecule resolution. Taking advantage of long RNA reads provided by the Nanopore platform, we could accurately (re-)annotate large transcriptional units and boundaries. Our analysis of transcription termination sites revealed that diverse termination mechanisms are in place in archaea. Moreover, we shed light on the poorly understood rRNA processing pathway in archaea and detected new processing intermediates. One of the key features of native RNA sequencing is that RNA modifications are retained. We could confirm this ability by analysing the well-known KsgA-dependent rRNA methylation sites. Notably, our analysis suggests that rRNA modifications are more abundant in a hyperthermophilic organism.","The COVID-19 pandemic has created massive demand for widespread, distributed tools for detecting SARS-CoV-2 genetic material. The hurdles to scalable testing include reagent and instrument accessibility, availability of highly-trained personnel, and large upfront investment. Here we showcase an orthogonal pipeline we call CREST (Cas13-based, Rugged, Equitable, Scalable Testing) that addresses some of these hurdles. Specifically, CREST pairs commonplace and reliable biochemical methods (PCR) with low-cost instrumentation, without sacrificing detection sensitivity. By taking advantage of simple fluorescence visualizers, CREST allows for a binary interpretation of results. CREST may provide a point-of-care solution to increase the distribution of COVID-19 surveillance.","AbstractAdenovirus is a common human pathogen that relies on host cell processes for production and processing of viral RNA. Although adenoviral promoters, splice junctions, and cleavage and polyadenylation sites have been characterized using low-throughput biochemical techniques or short read cDNA-based sequencing, these technologies do not fully capture the complexity of the adenoviral transcriptome. By combining Illumina short-read and nanopore long-read direct RNA sequencing approaches, we mapped transcription start sites and cleavage and polyadenylation sites across the adenovirus genome. The canonical viral early and late RNA cassettes were confirmed, but analysis of splice junctions within long RNA reads revealed an additional 20 novel viral transcripts. These RNAs include seven new splice junctions which lead to expression of canonical open reading frames (ORF), as well as 13 transcripts encoding for messages that potentially alter protein functions through truncations or the fusion of canonical ORFs. In addition, we also detect RNAs that bypass canonical cleavage sites and generate potential chimeric proteins by linking separate gene transcription units. Our work highlights how long-read sequencing technologies can reveal further complexity within viral transcriptomes.","Understanding the disease pathogenesis of the novel coronavirus, denoted SARS-CoV-2, is critical to the development of anti-SARS-CoV-2 therapeutics. The global propagation of the viral disease, denoted COVID-19 (\"coronavirus disease 2019\"), has unified the scientific community in searching for possible inhibitory small molecules or polypeptides. Given the known interaction between the human ACE2 (\"Angiotensin-converting enzyme 2\") protein and the SARS-CoV virus (responsible for the coronavirus outbreak circa. 2003), considerable focus has been directed towards the putative interaction between the SARS-CoV-2 Spike protein and ACE2. However, a more holistic understanding of the SARS-CoV-2 vs. human inter-species interactome promises additional putative protein-protein interactions (PPI) that may be considered targets for the development of inhibitory therapeutics. To that end, we leverage two state-of-the-art, sequence-based PPI predictors (PIPE4 &amp; SPRINT) capable of generating the comprehensive SARS-CoV-2 vs. human interactome, comprising approximately 285,000 pairwise predictions. Of these, we identify the high-scoring subset of human proteins predicted to interact with each of the 14 SARS-CoV-2 proteins by both methods, comprising 279 high-confidence putative interactions involving 225 human proteins. Notably, the Spike-ACE2 interaction was the highest ranked for both the PIPE4 and SPRINT predictors, corroborating existing evidence for this PPI. Furthermore, the PIPE-Sites algorithm was used to predict the putative subsequence that might mediate each interaction and thereby inform the design of inhibitory polypeptides intended to disrupt the corresponding host-pathogen interactions. We hereby publicly release the comprehensive set of PPI predictions and their corresponding PIPE-Sites landscapes in the following DataVerse repository: https://www.doi.org/10.5683/SP2/JZ77XA. All data and metadata are released under a CC-BY 4.0 licence. The information provided represents theoretical modeling only and caution should be exercised in its use. It is intended as a resource for the scientific community at large in furthering our understanding of SARS-CoV-2.","SUMMARYThe authors find that human cells re-entering the cell cycle from quiescence have both an impaired p53-dependent DNA replication origin licensing checkpoint and slow origin licensing. This combination makes every first S phase underlicensed and hypersensitive to replication stress.ABSTRACTTo maintain tissue homeostasis, cells transition between cell cycle quiescence and proliferation. An essential G1 process is Minichromosome Maintenance complex (MCM) loading at DNA replication origins to prepare for S phase, known as origin licensing. A p53-dependent origin licensing checkpoint normally ensures sufficient MCM loading prior to S phase entry. We used quantitative flow cytometry and live cell imaging to compare MCM loading during the long first G1 upon cell cycle entry and the shorter G1 phases in the second and subsequent cycles. We discovered that despite the longer G1 phase, the first G1 after cell cycle re-entry is significantly underlicensed. As a result, the first S phase cells are hypersensitive to replication stress. This underlicensing is from a combination of slow MCM loading with a severely compromised origin licensing checkpoint. The hypersensitivity to replication stress increases over repeated rounds of quiescence. Thus, underlicensing after cell cycle re-entry from quiescence distinguishes a higher risk cell cycle that promotes genome instability.","AbstractINTRODUCTIONEpidemic forecasting and prediction tools have the potential to provide actionable information in the midst of emerging epidemics. While numerous predictive studies were published during the 2016-2017 Zika Virus (ZIKV) pandemic, it remains unknown how timely, reproducible and actionable the information produced by these studies was.METHODSTo improve the functional use of mathematical modeling in support of future infectious disease outbreaks, we conducted a systematic review of all ZIKV prediction studies published during the recent ZIKV pandemic using the PRISMA guidelines. Using MEDLINE, EMBASE and grey literature review, we identified studies that forecasted, predicted or simulated ecological or epidemiological phenomenon related to the Zika pandemic that were published as of March 01, 2017. Eligible studies underwent evaluation of objectives, data sources, methods, timeliness, reproducibility, accessibility and clarity by independent reviewers.RESULTS2034 studies were identified, of which n = 73 met eligibility criteria. Spatial spread, R0 (basic reproductive number) and epidemic dynamics were most commonly predicted, with few studies predicting Guillain-Barr\u00e9 Syndrome burden (4%), sexual transmission risk (4%) and intervention impact (4%). Most studies specifically examined populations in the Americas (52%), with few African-specific studies (4%). Case count (67%), vector (41%) and demographic data (37%) were the most common data sources. Real-time internet data and pathogen genomic information were used in 7% and 0% of studies, respectively, and social science and behavioral data were typically absent in modeling efforts. Deterministic models were favored over stochastic approaches. Forty percent of studies made model data entirely available, 29% provided all relevant model code, 43% presented uncertainty in all predictions and 54% provided sufficient methodological detail allowing complete reproducibility. Fifty-one percent of predictions were published after the epidemic peak in the Americas. While the use of preprints improved the accessibility of ZIKV predictions by a median 119 days sooner than journal publication dates, they were used in only 30% of studies.CONCLUSIONSMany ZIKV predictions were published during the 2016-2017 pandemic. The accessibility, reproducibility, timeliness, and incorporation of uncertainty in these published predictions varied and indicates that there is substantial room for improvement. To enhance the utility of analytical tools for outbreak response, it is essential to improve the sharing of model data, code, and preprints for future outbreaks, epidemics and pandemics.Author summaryResearchers published many studies which sought to predict and forecast important features of Zika virus (ZIKV) infections and their spread during the 2016-2017 ZIKV pandemic. We conducted a comprehensive review of such ZIKV prediction studies and evaluated their aims, the data sources they used, which methods were used, how timely they were published, and whether they provided sufficient information to be used or reproduced by others. Of the 73 studies evaluated, we found that the accessibility, reproducibility, timeliness, and incorporation of uncertainty in these published predictions varied and indicates that there is substantial room for improvement. We identified that the release of study findings before formal journal publication (\u2018pre-prints\u2019) increased the timeliness of Zika prediction studies, but note they were infrequently used during this public health emergency. Addressing these areas can improve our understanding of Zika and other outbreaks and ensure that forecasts can inform preparedness and response to future outbreaks, epidemics and pandemics.","Novel SARS-Cov-2 enters human cells via interaction between the surface spike (S) glycoprotein and the cellular membrane receptor angiotensin-converting enzyme 2 (ACE2). Using a combination of comparative structural analyses of the binding surface of the S protein to ACE2, docking experiments, and molecular dynamics simulations we computationally identified a minimal, stable fragment of ACE2. This fragment binds to the S protein, is soluble, and appears not to bind to the physiological ligand angiotensinII. These results suggest a possible use of the ACE2 fragment as a decoy that could interfere with viral binding by competition.","AbstractThe human betacoronaviruses HKU1 and OC43 (subgenus Embecovirus) arose from separate zoonotic introductions, OC43 relatively recently and HKU1 apparently much longer ago. Embecovirus particles are studded with two types of surface projections called S (for spike) and HE (for haemagglutinin-esterase), with S mediating receptor-binding and membrane fusion and HE acting as a receptor-destroying enzyme. Together, they promote dynamic virion attachment to glycan-based receptors with 9-O-acetylated sialic acid as main constituent. We recently showed that adaptation of HKU1 and OC43 to replication in the human respiratory tract involved loss-of-function mutations in the lectin domain of HE. Here we present the cryo-EM structure of the \u223c80 kDa, heavily glycosylated HKU1 HE at a global resolution of 3.4 \u00c5. Comparison to existing HE structures reveals a drastically truncated lectin domain, incompatible with sialic acid binding, but with the structure and function of the HE esterase domain left intact. Our cryo-EM structure, in combination with mass spectrometry analysis, also describes the extent of glycosylation on the now redundant lectin domain, which forms a putative glycan shield. The findings further our insight into the evolution and host adaptation of human embecoviruses and also demonstrate the utility of cryo-EM for studying small, heavily glycosylated proteins which are intractable to X-ray crystallography.","AbstractThe recent reclassification of the Riboviria, and the introduction of multiple new taxonomic categories including both subfamilies and subgenera for coronaviruses (family Coronaviridae, subfamily Orthocoronavirinae) represents a major shift in how official classifications are used to designate specific viral lineages. While the newly defined subgenera provide much-needed standardisation for commonly cited viruses of public health importance, no method has been proposed for the assignment of subgenus based on partial sequence data, or for sequences that are divergent from the designated holotype reference genomes. Here, we describe the genetic variation of a partial region of the coronavirus RNA-dependent RNA polymerase (RdRp), which is one of the most used partial sequence loci for both detection and classification of coronaviruses in molecular epidemiology. We infer Bayesian phylogenies from more than 7000 publicly available coronavirus sequences and examine clade groupings relative to all subgenus holotype sequences. Our phylogenetic analyses are largely coherent with genome-scale analyses based on designated holotype members for each subgenus. Distance measures between sequences form discrete clusters between taxa, offering logical threshold boundaries that can attribute subgenus or indicate sequences that are likely to belong to unclassified subgenera both accurately and robustly. We thus propose that partial RdRp sequence data of coronaviruses is sufficient for the attribution of subgenus-level taxonomic classifications and we supply the R package, \u201cMyCoV\u201d, which provides a method for attributing subgenus and assessing the reliability of the attribution.Importance StatementThe analysis of polymerase chain reaction amplicons derived from biological samples is the most common modern method for detection and classification of infecting viral agents, such as Coronaviruses. Recent updates to the official standard for taxonomic classification of Coronaviruses, however, may leave researchers unsure as to whether the viral sequences they obtain by these methods can be classified into specific viral taxa due to variations in the sequences when compared to type strains. Here, we present a plausible method for defining genetic dissimilarity cut-offs that will allow researchers to state which taxon their virus belongs to and with what level of certainty. To assist in this, we also provide the R package \u2018MyCoV\u2019 which classifies user generated sequences.","ABSTRACTAs the closest living relatives of animals, choanoflagellates offer unique insights into animal origins and core mechanisms underlying animal cell biology. However, unlike traditional model organisms, such as yeast, flies and worms, choanoflagellates have been refractory to DNA delivery methods for expressing foreign genes. Here we report the establishment of a robust method for expressing transgenes in the choanoflagellate Salpingoeca rosetta, overcoming barriers that have previously hampered DNA delivery and expression. To demonstrate how this method accelerates the study of S. rosetta cell biology, we engineered a panel of fluorescent protein markers that illuminate key features of choanoflagellate cells. We then investigated the localization of choanoflagellate septins, a family of GTP-binding cytoskeletal proteins that are hypothesized to regulate the multicellular rosette development in S. rosetta. Fluorescently tagged septins localized to the basal pole of S. rosetta single cells and rosettes in a pattern resembling septin localization in animal epithelia. The establishment of transfection in S. rosetta and its application to the study of septins represent critical advances in the growth of S. rosetta as an experimental model for investigating choanoflagellate cell biology, core mechanisms underlying animal cell biology, and the origin of animals.","AbstractUveitis is a visually-debilitating disorder that affects up to 30% of children with juvenile idiopathic arthritis (JIA). To identify genetic susceptibility loci for uveitis in JIA, we conducted a genome-wide association study comparing 192 JIA-associated uveitis cases with 330 JIA individuals without uveitis. Two cohorts of JIA patients underwent genotyping and quality control. We used an HLA-specific imputation panel to impute HLA-specific amino acids and HLA types, and identified the amino acid serine at position 11 (serine-11) in HLA- DRB1 as associated to increased risk of uveitis (OR = 2.60, p = 5.43\u00d710\u221210). The signal at serine-11 was female-specific (interaction of sex and serine-11, p = 0.0096). Serine-11 resides in the YST-motif (positions 10-12) in the peptide binding groove of HLA-DRB1. Quantitative binding affinity predictions revealed peptide-binding preferences that distinguish HLA-DRB1 allotypes with the YST-motif. Our findings highlight a genetically distinct, sexually-dimorphic feature of JIA-associated uveitis.","ABSTRACTClean freshwater lies at the heart of human society and monitoring its quality is paramount. In addition to chemical controls, traditional microbiological water tests focus on the detection of specific bacterial pathogens. The direct tracing of all aquatic DNA poses a more profound alternative. Yet, this has hitherto been underused due to challenges in cost and logistics. Here we present a simple, fast, inexpensive and comprehensive freshwater diagnostics workflow centred around portable nanopore DNA sequencing. Using defined bacterial compositions and spatiotemporal microbiota from surface water of an example river in Cambridge (UK), our study shows how nanopore sequencing can be readily integrated for the assessment of aquatic bacterial diversity and pollution. We provide a computational benchmark that features more than ten taxonomic classification tools to derive guidelines for bacterial DNA analyses with nanopore data. Through complementary physicochemical measurements, we find that nanopore metagenomics can depict fine temporal gradients along the main hydrological axis of an urban-rural interface and yield high-resolution pathogen maps that address concerns of public health.","AbstractBio-fluids contain various circulating cell-free RNA transcripts (ccfRNAs). The composition of these ccfRNAs varies between bio-fluids and constitute tantalizing biomarker candidates for several pathologies. ccfRNAs have also been demonstrated as mediators of cellular communication, yet little is known about their function in physiological and developmental settings and most works are limited to in-vitro studies. Here, we have developed iTAG-RNA, a novel method for the unbiased tagging of RNA transcripts in mice in-vivo. We used this method to isolate hepatocytes and kidney proximal epithelial cells-specific transcriptional response to a dietary challenge without interfering with the tissue architecture, and to identify multiple hepatocyte-secreted ccfRNAs in plasma. We also identified transfer of these hepatic derived ccfRNAs to adipose tissue, where they likely serve as a buffering mechanism to maintain cholesterol and lipid homeostasis. Our findings directly demonstrate in-vivo transfer of RNAs between tissues and highlight its implications for endocrine signaling and homeostasis.","ABSTRACTRATIONALEDespite improved diagnostics, pulmonary pathogens in immunocompromised children frequently evade detection, leading to significant morbidity and mortality.OBJECTIVESTo develop a highly sensitive metagenomic next generation sequencing (mNGS) assay capable of evaluating the pulmonary microbiome and identifying diverse pathogens in the lungs of immunocompromised children.METHODSWe collected 41 lower respiratory specimens from 34 immunocompromised children undergoing evaluation for pulmonary disease at 3 children\u2019s hospitals from 2014-2016. Samples underwent mechanical homogenization, paired RNA/DNA extraction, and metagenomic sequencing. Sequencing reads were aligned to the NCBI nucleotide reference database to determine taxonomic identities. Statistical outliers were determined based on abundance within each sample and relative to other samples in the cohort.MEASUREMENTS &amp; MAIN RESULTSWe identified a rich cross-domain pulmonary microbiome containing bacteria, fungi, RNA viruses, and DNA viruses in each patient. Potentially pathogenic bacteria were ubiquitous among samples but could be distinguished as possible causes of disease by parsing for outlier organisms. Samples with bacterial outliers had significantly depressed alpha-diversity (median 0.58, IQR 0.33-0.62 vs. median 0.94, IQR 0.93-0.95, p&lt;0.001). Potential pathogens were detected in half of samples previously negative by clinical diagnostics, demonstrating increased sensitivity for missed pulmonary pathogens (p&lt;0.001).CONCLUSIONSAn optimized mNGS assay for pulmonary microbes demonstrates significant inoculation of the lower airways of immunocompromised children with diverse bacteria, fungi, and viruses. Potential pathogens can be identified based on absolute and relative abundance. Ongoing investigation is needed to determine the pathogenic significance of outlier microbes in the lungs of immunocompromised children with pulmonary disease.","AbstractAimKidney impairment is observed in patients with COVID-19. We aimed to demonstrate the effect of anti-COVID-19 agent remdesivir on renal fibrosis.MethodsRemdesivir and its active nucleoside metabolite GS-441524 were used to treat TGF-\u03b2 stimulated renal fibroblasts (NRK-49F) and human renal epithelial cells (HK2). Cell viability was determined by CCK8 assay, and fibrotic markers were measured by Western blotting. Vehicle or remdesivir were given by intraperitoneal injection or renal injection through the left ureter in unilateral ureteral obstruction (UUO) mice. Serum and kidneys were harvested. The concentrations of remdesivir and GS-441524 were measured using LC-MS/MS. Renal and liver function were assessed. Renal fibrosis was evaluated by Masson\u2019s trichrome staining and Western blotting.ResultsRemdesivir and GS-441524 inhibited cell proliferation and the expression of fibrotic markers (fibronectin, pSmad3, and aSMA) in NRK-49F and HK2 cells. Intraperitoneal injection or renal injection of remdesivir attenuated renal fibrosis of UUO kidneys. Renal and liver function were not changed in remdesivir treated UUO mice. Remdesivir can not be detected, but two remdesivir metabolites were detected after injection.ConclusionRemdesivir inhibits renal fibrosis in obstructed kidneys.","AbstractMany detection methods have been used or reported for the diagnosis and/or surveillance of SARS-CoV-2. Among them, reverse transcription polymerase chain reaction (RT-PCR) is the most sensitive, claiming detection of about 5 copies of viruses. However, it has been reported that only 47-59% of the positive cases were identified by RT-PCR, probably due to loss or degradation of virus RNA in the sampling process, or even mutation of the virus genome. Therefore, developing highly sensitive methods is imperative to ensure robust detection capabilities. With the goal of improving sensitivity and accommodate various application settings, we developed a multiplex-PCR-based method comprised of 172 pairs of specific primers, and demonstrated its efficiency to detect SARS-CoV-2 at low copy numbers. The assay produced clean characteristic target peaks of defined sizes, which allowed for direct identification of positives by electrophoresis. In addition, optional sequencing can provide further confirmation as well as phylogenetic information of the identified virus(es) for specific strain discrimination, which will be of paramount importance for surveillance purposes that represent a global health imperative. Finally, we also developed in parallel a multiplex-PCR-based metagenomic method that is amenable to detect SARS-CoV-2, with the additional benefit of its potential for uncovering mutational diversity and novel pathogens at low sequencing depth.","AbstractThe novel coronavirus SARS-CoV-2 is the cause of Coronavirus Disease-2019 (COVID-19). The main receptor of SARS-CoV-2, angiotensin I converting enzyme 2 (ACE2), is now undergoing extensive scrutiny to understand the routes of transmission and sensitivity in different species. Here, we utilized a unique dataset of 410 vertebrates, including 252 mammals, to study cross-species conservation of ACE2 and its likelihood to function as a SARS-CoV-2 receptor. We designed a five-category ranking score based on the conservation properties of 25 amino acids important for the binding between receptor and virus, classifying all species from very high to very low. Only mammals fell into the medium to very high categories, and only catarrhine primates in the very high category, suggesting that they are at high risk for SARS-CoV-2 infection. We employed a protein structural analysis to qualitatively assess whether amino acid changes at variable residues would be likely to disrupt ACE2/SARS-CoV-2 binding, and found the number of predicted unfavorable changes significantly correlated with the binding score. Extending this analysis to human population data, we found only rare (&lt;0.1%) variants in 10/25 binding sites. In addition, we observed evidence of positive selection in ACE2 in multiple species, including bats. Utilized appropriately, our results may lead to the identification of intermediate host species for SARS-CoV-2, justify the selection of animal models of COVID-19, and assist the conservation of animals both in native habitats and in human care.","The pandemic from the Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) led to &gt;170,000 deaths, including &gt;14,500 in New York City (NYC) alone. This pandemic highlighted a pressing clinical and public health need for rapid, scalable diagnostics that can detect SARS-CoV-2 infection, interrogate strain evolution, and map host response in patients. To address these challenges, we designed a fast (30 minute) colorimetric test to identify SARS-CoV-2 infection and simultaneously developed a large-scale shotgun metatranscriptomic profiling platform for nasopharyngeal swabs. Both technologies were used to profile 338 clinical specimens tested for SARS-CoV-2 and 86 NYC subway samples, creating a broad molecular picture of the COVID-19 epidemic in NYC. Our results nominate a novel, NYC-enriched SARS-CoV-2 subclade, reveal specific host responses in ACE pathways, and find medication risks associated with SARS-CoV-2 infection and ACE inhibitors. Our findings have immediate applications to SARS-CoV-2 diagnostics, public health monitoring, and therapeutic development.","Recent history is punctuated by the emergence of highly pathogenic coronaviruses such as SARS- and MERS-CoV into human circulation. Upon infecting host cells, coronaviruses assemble a multi-subunit RNA-synthesis complex of viral non-structural proteins (NSP) responsible for the replication and transcription of the viral genome. Here, we present the 3.1 \u00c5 resolution structure of the SARS-CoV NSP12 polymerase bound to its essential co-factors, NSP7 and NSP8, using single particle cryo-electron microscopy. NSP12 possesses an architecture common to all viral polymerases as well as a large N-terminal extension containing a kinase-like fold and is unexpectedly bound by two NSP8 co-factors. This structure illuminates the assembly of the coronavirus core RNA-synthesis machinery, provides key insights into NSP12 polymerase catalysis and fidelity and acts as a template for the design of novel antiviral therapeutics.","Advances in single-cell technologies have enabled the investigation of T cell phenotypes and repertoires at unprecedented resolution and scale. Bioinformatic methods for the efficient analysis of these large-scale datasets are instrumental for advancing our understanding of adaptive immune responses in cancer, but also in infectious diseases like COVID-19. However, while well-established solutions are accessible for the processing of single-cell transcriptomes, no streamlined pipelines are available for the comprehensive characterization of T cell receptors. Here we propose Scirpy, a scalable Python toolkit that provides simplified access to the analysis and visualization of immune repertoires from single cells and seamless integration with transcriptomic data. Scirpy source code and documentation are available at https://github.com/icbi-lab/scirpy.","AbstractSevere acute respiratory syndrome (SARS) and Middle East respiratory syndrome (MERS) coronaviruses (CoVs) are zoonotic pathogens with high fatality rates and pandemic potential. Vaccine development has focussed on the principal target of the neutralizing humoral immune response, the spike (S) glycoprotein, which mediates receptor recognition and membrane fusion. Coronavirus S proteins are extensively glycosylated viral fusion proteins, encoding around 69-87 N-linked glycosylation sites per trimeric spike. Using a multifaceted structural approach, we reveal a specific area of high glycan density on MERS S that results in the formation of under-processed oligomannose-type glycan clusters, which was absent on SARS and HKU1 CoVs. We provide a comparison of the global glycan density of coronavirus spikes with other viral proteins including HIV-1 envelope, Lassa virus glycoprotein complex, and influenza hemagglutinin, where glycosylation plays a known role in shielding immunogenic epitopes. Consistent with the ability of the antibody-mediated immune response to effectively target and neutralize coronaviruses, we demonstrate that the glycans of coronavirus spikes are not able to form an efficacious high-density global shield to thwart the humoral immune response. Overall, our data reveal how differential organisation of viral glycosylation across class I viral fusion proteins influence not only individual glycan compositions but also the immunological pressure across the viral protein surface.","As the COVID-19 pandemic unfolds, researchers from all disciplines are coming together and contributing their expertise. CORD-19, a dataset of COVID-19 and coronavirus publications, has recently been published alongside calls to help mine the information it contains, and to create tools to search it more effectively. Here, we focus on the delineation of the publications included in CORD-19, and analyse this delineation from a scientometric perspective. We find that CORD-19 contains research not only on COVID-19 and coronaviruses, but on viruses in general. Publications from CORD-19 mostly focus on a few, well-defined areas, including: coronaviruses (primarily SARS, MERS, COVID-19); public health and viral epidemics; the molecular biology of viruses; influenza and other families of viruses; immunology and antivirals; methodology (testing, diagnosing, clinical trials). CORD-19 publications published in 2020, especially focused on topics of pressing relevance (spread, infection, efficacy of counter-measures), are disproportionately popular on social media. While we fully endorse the initiative that led to CORD-19, we also advise to consider its relatively broad content critically.","The Coronavirus disease -19 (COVID19) that started in Wuhan, China in December 2019 has spread worldwide emerging as a global pandemic. The severe respiratory pneumonia caused by the novel SARS-CoV-2 has so far claimed more than 14,500 lives and has impacted human lives worldwide. Development of universal vaccines against the novel SARS-CoV-2 holds utmost urgency to control COVID19 pandemic that appears to be more severe than any of the previous outbreaks of severe acute respiratory syndrome (SARS) and Middle-East respiratory syndrome (MERS). However, as the novel SARS-CoV-2 displays high transmission rates, the underlying severity hidden in the SARS-CoV2 genomes is required to be fully understood. We studied the complete genomes of 95 strains of SARS-CoV-2 reported from different geographical regions worldwide to uncover the pattern of spread of the novel SARS-CoV-2 across the globe. We show that there is no direct transmission pattern of the virus among neighbouring countries suggesting that the outbreak is a result of travel of infected humans to different countries. We revealed unique single nucleotide polymorphisms (SNPs) in nsp13, nsp14, nsp15, nsp16 (present in ORF1b polyprotein region) and S-Protein within 10 viral isolates from USA. These viral proteins are involved in RNA replication and processing, indicating highly evolved strains of the novel SARS-CoV-2 circulating in the population of USA than in other countries. Furthermore, we found an isolate from USA (MT188341) to carry frameshift mutation between positions 2540 and 2570 of nsp16 which functions as mRNA cap-1 methyltransferase (2\u2032-O-MTase). Thus, we reason that the replicative machinery of the novel SARS-CoV-2 is fast evolving to evade host challenges and survival. These mutations are needed to be considered, otherwise it will be difficult to develop effective treatment strategies. The two proteins also had dN/dS values approaching 1- ORF1ab polyprotein (dN/dS= 0.996, 0.575) and S protein (dN/dS= 0.88) and might confer selective advantage to the virus. Through the construction of SARS-CoV-2-human interactome, we further reveal multiple host proteins (PHB, PPP1CA, TGF-beta, JACK1, JACK2, SOCS3,STAT3, JAK1-2, SMAD3, BCL2, CAV1 &amp; SPECC1) which are manipulated by the virus proteins (nsp2, PL-PRO, N-protein, ORF7a, M-S-ORF3a complex, nsp7-nsp8-nsp9-RdRp complex) for mediating host immune mechanism for its survival.","AbstractRecently developed quantitative redox proteomic studies enable the direct identification of redox-sensing cysteine residues that regulate the functional behavior of target proteins in response to changing levels of reactive oxygen species (ROS). At the molecular level, redox regulation can directly modify the active sites of enzymes, although a growing number of examples indicate the importance of an additional underlying mechanism that involves conditionally disordered proteins. These proteins alter their functional behavior by undergoing a disorder-to-order transition in response to changing redox conditions. However, the extent to which this mechanism is used in various proteomes is currently unknown. Here, we use a recently developed sequence-based prediction tool incorporated into the IUPred2A web server to estimate redox-sensitive conditionally disordered regions on a large scale. We show that redox-sensitive conditional disorder is fairly widespread in various proteomes and that its presence strongly correlates with the expansion of specific domains in multicellular organisms that largely rely on extra stability provided by disulfide bonds or zinc ion binding. The analyses of yeast redox proteomes and human disease data further underlie the significance of this phenomenon in the regulation of a wide range of biological processes, as well as its biomedical importance.","ABSTRACTFor decades, cationic polymer nanoparticles have been investigated for nucleic acid delivery. Despite promising in vitro transfection results, most formulations have failed to translate into the clinic due to significant in vivo toxicity \u2013 especially when delivered intravenously. To address this significant problem, we investigated the detailed mechanisms that govern the complex in vivo systemic toxicity response to common polymeric nanoparticles. We determined that the toxicity response is material dependent. For branched polyethylenimine (bPEI) nanoparticles \u2013 toxicity is a function of multiple pathophysiological responses \u2013 triggering of innate immune sensors, induction of hepatic toxicity, and significant alteration of hematological properties. In contrast, for chitosan-based nanoparticles \u2013 systemic toxicity is primarily driven through innate immune activation. We further identified that modification of primary amines to secondary and tertiary amines using the small molecule imidazole-acetic-acid (IAA) ameliorates in vivo toxicity from both nanocarriers by different, material-specific mechanisms related to Toll-like receptor 4 activation (for bPEI) and complement activation driven neutrophil infiltration (for chitosan), respectively. Our results provide a detailed roadmap for evaluating in vivo toxicity of nanocarriers and identifies potential opportunities to reduce toxicity for eventual clinical translation.Graphical Abstract","AbstractMutation rates can evolve through genetic drift, indirect selection due to genetic hitchhiking, or direct selection on the physicochemical cost of high fidelity. However, for many systems, it has been difficult to disentangle the relative impact of these forces empirically. In RNA viruses, an observed correlation between mutation rate and virulence has led many to argue that their extremely high mutation rates are advantageous, because they may allow for increased adaptability. This argument has profound implications, as it suggests that pathogenesis in many viral infections depends on rare or de novo mutations. Here we present data for an alternative model whereby RNA viruses evolve high mutation rates as a byproduct of selection for increased replicative speed. We find that a poliovirus antimutator, 3DG64S, has a significant replication defect and that wild type and 3DG64S populations have similar adaptability in two distinct cellular environments. Experimental evolution of 3DG64S under r-selection led to reversion and compensation of the fidelity phenotype. Mice infected with 3DG64S exhibited delayed morbidity at doses well above the LD50, consistent with attenuation by slower growth as opposed to reduced mutational supply. Furthermore, compensation of the 3DG64S growth defect restored virulence, while compensation of the fidelity phenotype did not. Our data are consistent with the kinetic proofreading model for biosynthetic reactions and suggest that speed is more important than accuracy. In contrast to what has been suggested for many RNA viruses, we find that within host spread is associated with viral replicative speed and not standing genetic diversity.Author SummaryMutation rate evolution has long been a fundamental problem in evolutionary biology. The polymerases of RNA viruses generally lack proofreading activity and exhibit extremely high mutation rates. Since most mutations are deleterious and mutation rates are tuned by natural selection, we asked why hasn\u2019t the virus evolved to have a lower mutation rate? We used experimental evolution and a murine infection model to show that RNA virus mutation rates may actually be too high and are not necessarily adaptive. Rather, our data indicate that viral mutation rates are driven higher as a result of selection for viruses with faster replication kinetics. We suggest that viruses have high mutation rates, not because they facilitate adaption, but because it is hard to be both fast and accurate.","AbstractThe rigidity and flexibility of two recently reported crystal structures (PDB entries 6Y2E and 6LU7) of a protease from the SARS-CoV-2 virus, the infectious agent of the COVID-19 respiratory disease, has been investigated using pebble-game rigidity analysis, elastic network model normal mode analysis, and all-atom geometric simulations. This computational investigation of the viral protease follows protocols that have been effective in studying other homodimeric enzymes. The protease is predicted to display flexible motions in vivo which directly affect the geometry of a known inhibitor binding site and which open new potential binding sites elsewhere in the structure. A database of generated PDB files representing natural flexible variations on the crystal structures has been produced and made available for download from an institutional data archive. This information may inform structure-based drug design and fragment screening efforts aimed at identifying specific antiviral therapies for the treatment of COVID-19.","AbstractIn 2014, A novel tick-borne virus of the genus Flavivirus was first reported from the Mogiana region in Brazil. This virus was named the Mogiana tick virus (MGTV). Later, MGTV was also named as Jingmen tick virus (JMTV), Kindia tick virus (KDTV), Guangxi tick virus (GXTV) etc. In the present study, we used small RNA sequencing (sRNA-seq) to detect viruses in ticks and detected MGTV in Amblyomma testudinarium ticks, which had been captured in Yunnan province of China in the year of 2016. The full-length genome sequence of a new MGTV strain Yunnan2016 (GenBank: MT080097, MT080098, MT080099 and MT080100) was obtained and recommended to be included into the NCBI RefSeq database for the future studies of MGTV. Our phylogenetic analyses showed that viruses named MGTV, JMTV, KDTV and GXTV are monophyletic: the MGTV group (lineage) of viruses. We show, for the first time, that 5\u2032 and 3\u2032 sRNAs can be used to obtain full-length sequences of the 5\u2019 and 3\u2019 ends of, but not limited to genome sequences of RNA viruses. And we proved the feasibility of using the sRNA-seq based method for the detection of viruses in a sample containing miniscule RNA.","The surfaces of proteins are generally hydrophilic but there have been reports of sites that exhibit an exceptionally high affinity for individual water molecules. Not only do such molecules often fulfil critical biological functions, but also, they may alter the binding of newly designed drugs. In crystal structures, sites consistently occupied in each unit cell yield electron density clouds that represent water molecule presence. These are recorded in virtually all high-resolution structures obtained through X-ray diffraction. In this work, we utilized the wealth of data from the RCSB Protein Data Bank to train a residual deep learning model named \u2032hotWater\u2032 to identify sites on the surface of proteins that are most likely to bind water, the so-called water hot spots. The model can be used to score existing water molecules from a PDB file to provide their ranking according to the predicted binding strength or to scan the surface of a protein to determine the most likely water hot spots de novo. This is computationally much more efficient than currently used molecular dynamics simulations. Based on testing the model on three example proteins, which have been resolved using both high-resolution X-ray crystallography (providing accurate positions of trapped waters) as well as low-resolution X-ray diffraction, NMR or CryoEM (where structure refinement does not yield water positions), we were able to show that the hotWater method is able to recover in the \u2033water-free\u2033 structures many water binding sites known from the high-resolution structures. A blind test on a newly solved protein structure with waters removed from the PDB also showed good prediction of the crystal water positions. This was compared to two known algorithms that use electron density and was shown to have higher recall at resolutions &gt;2.6\u00c5. We also show that the algorithm can be applied to novel proteins such as the RNA polymerase complex from SARS-CoV-2, which could be of use in drug discovery. The hotWater model is freely available at (https://pypi.org/project/hotWater/).","ABSTRACTSummaryGenome Detective is a web-based, user-friendly software application to quickly and accurately assemble all known virus genomes from next generation sequencing datasets. This application allows the identification of phylogenetic clusters and genotypes from assembled genomes in FASTA format. Since its release in 2019, we have produced a number of typing tools for emergent viruses that have caused large outbreaks, such as Zika and Yellow Fever Virus in Brazil. Here, we present The Genome Detective Coronavirus Typing Tool that can accurately identify novel coronavirus (2019-nCoV) sequences isolated in China and around the world. The tool can accept up to 2,000 sequences per submission and the analysis of a new whole genome sequence will take approximately one minute. The tool has been tested and validated with hundreds of whole genomes from ten coronavirus species, and correctly classified all of the SARS-related coronavirus (SARSr-CoV) and all of the available public data for 2019-nCoV. The tool also allows tracking of new viral mutations as the outbreak expands globally, which may help to accelerate the development of novel diagnostics, drugs and vaccines.AvailabilityAvailable online: https://www.genomedetective.com/app/typingtool/cov*Contactkoen@emweb.be and deoliveira@ukzn.ac.zaSupplementary informationSupplementary data is available online.","SARS-CoV-2 has rapidly become a pandemic worldwide; therefore, an effective vaccine is urgently needed. Recently, messenger RNAs (mRNAs) have emerged as a promising platform for vaccination. Here, we systematically investigated the untranslated regions (UTRs) of mRNAs in order to enhance protein production. Through a comprehensive analysis of endogenous gene expression and de novo design of UTRs, we identified the optimal combination of 5\u2032 and 3\u2032 UTR, termed as NASAR, which was five to ten-fold more efficient than the tested endogenous UTRs. More importantly, NASAR mRNAs delivered by lipid-derived nanoparticles showed dramatic expression of potential SARS-CoV-2 antigens both in vitro and in vivo. These NASAR mRNAs merit further development as alternative SARS-CoV-2 vaccines.","ABSTRACTThe mechanisms underlying multicellular development in the animal stem lineage may be reconstructed through the study of choanoflagellates, the closest living relatives of animals. To determine the genetic underpinnings of multicellularity in the emerging model choanoflagellate S. rosetta, we performed a screen for mutants with defects in multicellular rosette development. In two of the mutants, Jumble and Couscous, single cells failed to develop into orderly rosettes but instead aggregated promiscuously into amorphous clumps of cells. Both mutants mapped to lesions in genes encoding glycosyltransferases and the mutations perturbed glycosylation patterns in the extracellular matrix (ECM). In animals, glycosyltransferases transfer activated sugars to donor molecules and thereby sculpt the polysaccharide-rich ECM, regulate integrin and cadherin activity, and, when disrupted, contribute to tumorigenesis. The finding that glycosyltransferases promote proper rosette development and prevent cell aggregation in S. rosetta suggests a pre-metazoan role for glycosyltransferases in regulating development and preventing abnormal tumor-like multicellularity.IMPACT STATEMENTA genetic screen reveals that glycosyltransferases are required for proper rosette development and the prevention of cell clumping in one of the closest living relatives of animals, the choanoflagellate S. rosetta.","ABSTRACTTRIM25 is a novel RNA-binding protein and a member of the Tripartite Motif (TRIM) family of E3 ubiquitin ligases, which plays a pivotal role in the innate immune response. Almost nothing is known about its RNA-related roles in cell biology. Furthermore, its RNA-binding domain has not been characterized. Here, we reveal that RNA-binding activity of TRIM25 is mediated by its PRY/SPRY domain, which we postulate to be a novel RNA-binding domain. Using CLIP-seq and SILAC-based co-immunoprecipitation assays, we uncover TRIM25\u2019s endogenous RNA targets and protein binding partners. Finally, we show that the RNA-binding activity of TRIM25 is important for its ubiquitin ligase function. These results reveal new insights into the molecular roles and characteristics of RNA-binding E3 ubiquitin ligases and demonstrate that RNA could be an essential factor for their biological functions.","AbstractThe infection of a novel coronavirus found in Wuhan of China (2019-nCoV) is rapidly spreading, and the incidence rate is increasing worldwide. Due to the lack of effective treatment options for 2019-nCoV, various strategies are being tested in China, including drug repurposing. In this study, we used our pretrained deep learning-based drug-target interaction model called Molecule Transformer-Drug Target Interaction (MT-DTI) to identify commercially available drugs that could act on viral proteins of 2019-nCoV. The result showed that atazanavir, an antiretroviral medication used to treat and prevent the human immunodeficiency virus (HIV), is the best chemical compound, showing a inhibitory potency with Kd of 94.94 nM against the 2019-nCoV 3C-like proteinase, followed by efavirenz (199.17 nM), ritonavir (204.05 nM), and dolutegravir (336.91 nM). Interestingly, lopinavir, ritonavir, and darunavir are all designed to target viral proteinases. However, in our prediction, they may also bind to the replication complex components of 2019-nCoV with an inhibitory potency with Kd &lt; 1000 nM. In addition, we also found that several antiviral agents, such as Kaletra, could be used for the treatment of 2019-nCoV, although there is no real-world evidence supporting the prediction. Overall, we suggest that the list of antiviral drugs identified by the MT-DTI model should be considered, when establishing effective treatment strategies for 2019-nCoV.","Rapid and widespread implementation of infectious disease surveillance is a critical component in the response to novel health threats. Molecular assays are the preferred method to detect a broad range of pathogens with high sensitivity and specificity. The implementation of molecular assay testing in a rapidly evolving public health emergency can be hindered by resource availability or technical constraints. In the context of the COVID-19 pandemic, the applicability of a pooled-sample testing protocol to screen large populations more rapidly and with limited resources is discussed. A Bayesian inference analysis in which hierarchical testing stages can have different sensitivities is implemented and benchmarked against early COVID-19 testing data. Optimal pool size and increases in throughput and case detection are calculated as a function of disease prevalence. Even for moderate losses in test sensitivity upon pooling, substantial increases in testing throughput and detection efficiency are predicted, suggesting that sample pooling is a viable avenue to circumvent current testing bottlenecks for COVID-19.","AbstractThe 2019 novel severe respiratory syndrome coronavirus-2 (SARS-CoV-2) outbreak has caused a large number of deaths with thousands of confirmed cases worldwide. The present study followed computational approaches to identify B- and T-cell epitopes for spike glycoprotein of SARS-CoV-2 by its interactions with the human leukocyte antigen alleles. We identified twenty-four peptide stretches on the SARS-CoV-2 spike protein that are well conserved among the reported strains. The S protein structure further validated the presence of predicted peptides on the surface. Out of which twenty are surface exposed and predicted to have reasonable epitope binding efficiency. The work could be useful for understanding the immunodominant regions in the surface protein of SARS-CoV-2 and could potentially help in designing some peptide-based diagnostics.","SUMMARYThe kinetochore is a dynamic multi-protein assembly that forms on each sister chromatid and interacts with microtubules of the mitotic spindle to drive chromosome segregation. In animals, kinetochores without attached microtubules expand their outermost layer into crescent and ring shapes to promote microtubule capture and spindle assembly checkpoint (SAC) signalling. Kinetochore expansion is an example of protein co-polymerization, but the mechanism is not understood. Here, we present evidence that kinetochore expansion is driven by oligomerization of the Rod-Zw10-Zwilch (RZZ) complex, an outer kinetochore component that recruits the motor dynein and the SAC proteins Mad1-Mad2. Depletion of ROD in human cells suppresses kinetochore expansion, as does depletion of Spindly, the adaptor that connects RZZ to dynein, while dynein itself is dispensable. Expansion is also suppressed by mutating ZWILCH residues implicated in Spindly binding. Conversely, supplying cells with excess ROD facilitates kinetochore expansion under otherwise prohibitive conditions. Using the C. elegans early embryo, we demonstrate that ROD-1 has a concentration-dependent propensity for oligomerizing into \u00b5m-scale filaments, and we identify the ROD-1 \u03b2-propeller as a key regulator of self-assembly. Finally, we show that a minimal ROD-1-Zw10 complex efficiently oligomerizes into filaments in vitro. Our results suggest that RZZ\u2019s capacity for oligomerization is harnessed by kinetochores to assemble the expanded outermost domain, in which RZZ filaments serve as recruitment platforms for SAC components and microtubule-binding proteins. Thus, we propose that RZZ self-assembly into filaments underlies the adaptive change in kinetochore size that contributes to chromosome segregation fidelity.","AbstractSevere fever with thrombocytopenia syndrome (SFTS) is an emerging tick-borne disease caused by SFTS virus (SFTSV) infection. Despite a gradual increase of SFTS cases and high mortality in endemic regions, no specific viral therapy nor vaccine is available. Here, we developed a single recombinant plasmid DNA encoding SFTSV genes, Gn and Gc together with NP-NS fusion antigen, as a vaccine candidate. The viral antigens were fused with Fms-like tyrosine kinase-3 ligand (Flt3L) and IL-12 gene was incorporated into the plasmid to enhance cell-mediated immunity. Vaccination with the DNA provides complete protection of IFNAR KO mice upon lethal SFTSV challenge, whereas immunization with a plasmid without IL-12 gene resulted in partial protection. Since we failed to detect antibodies against surface glycoproteins, Gn and Gc, in the immunized mice, antigen-specific cellular immunity, as confirmed by enhanced antigen-specific T cell responses, might play major role in protection. Finally, we evaluated the degree of protective immunity provided by protein immunization of the individual glycoprotein, Gn or Gc. Although both protein antigens induced a significant level of neutralizing activity against SFTSV, Gn vaccination resulted in relatively higher neutralizing activity and better protection than Gc vaccination. However, both antigens failed to provide complete protection. Given that DNA vaccines have failed to induce sufficient immunogenicity in human trials when compared to protein vaccines, optimal combinations of DNA and protein elements, proper selection of target antigens, and incorporation of efficient adjuvant, need to be further investigated for SFTSV vaccine development.Author summarySevere fever with thrombocytopenia syndrome (SFTS) is an emerging tick-borne infection endemic to East Asia including China, Korea, and Japan. Gradual rise of disease incidence and relatively high mortality have become a serious public health problem in the endemic countries. In this study, we developed a recombinant plasmid DNA encoding four antigens, Gn, Gc, NP, and NS, of SFTS virus (SFTSV) as a vaccine candidate. In order to enhance cell-mediated immunity, the viral antigens were fused with Flt3L and IL-2 gene was incorporated into the plasmid. Immunization with the DNA vaccine provides complete protection against lethal SFTSV infection in IFNAR KO mice. Antigen-specific T cell responses might play a major role in the protection since we observed enhanced T cell responses specific to the viral antigens but failed to detect neutralizing antibody in the immunized mice. When we immunized with either viral glycoprotein, Gn protein induced relatively higher neutralizing activity and better protection against SFTSV infection than Gc antigen, but neither generated complete protection. Therefore, an optimal combination of DNA and protein elements, as well as proper selection of target antigens, might be required to produce an effective SFTSV vaccine.","COVID-19 is an emerging infectious disease and was recently declared as a pandemic by WHO. Currently, there is no vaccine or therapeutic available for this disease. Drug repositioning represents the only feasible option to address this global challenge and a panel of fifty FDA-approved drugs that have been pre-selected by an assay of SARS-CoV was screened to identify potential antiviral drug candidates against SARS-CoV-2 infection. We found a total of 24 drugs which exhibited antiviral efficacy (0.1 \u03bcM &lt; IC50 &lt; 10 \u03bcM) against SARS-CoV-2. In particular, two FDA-approved drugs \u2013 niclosamide and ciclesonide \u2013 were notable in some respects. These drugs will be tested in an appropriate animal model for their antiviral activities. In near future, these already FDA-approved drugs could be further developed following clinical trials in order to provide additional therapeutic options for patients with COVID-19.","SARS-CoV-2 is a member of the coronaviridae family and is the etiological agent of the respiratory Coronavirus Disease 2019. The virus has spread rapidly around the world resulting in over two million cases and nearly 150,000 deaths as of April 17, 2020. Since no treatments or vaccines are available to treat COVID-19 and SARS-CoV-2, respiratory complications derived from the infections have overwhelmed healthcare systems around the world. This virus is related to SARS-CoV-1, the virus that caused the 2002-2004 outbreak of Severe Acute Respiratory Syndrome. In January 2020, the Center for Structural Genomics of Infectious Diseases implemented a structural genomics pipeline to solve the structures of proteins essential for coronavirus replication-transcription. Here we show the first structure of the SARS-CoV-2 nsp10-nsp16 2\u2032-O-methyltransferase complex with S-adenosylmethionine at a resolution of 1.8 \u00c5. This heterodimer complex is essential for capping viral mRNA transcripts for efficient translation and to evade immune surveillance.","Given the scale of the ongoing COVID-19 pandemic, the need for reliable, scalable testing, and the likelihood of reagent shortages, especially in resource-poor settings, we have developed a RT-qPCR assay that relies on an alternative to conventional viral reverse transcriptases, a thermostable reverse transcriptase / DNA polymerase (RTX)1. Here we show that RTX performs comparably to the other assays sanctioned by the CDC and validated in kit format. We demonstrate two modes of RTX use - (i) dye-based RT-qPCR assays that require only RTX polymerase, and (ii) TaqMan RT-qPCR assays that use a combination of RTX and Taq DNA polymerases (as the RTX exonuclease does not degrade a TaqMan probe). We also provide straightforward recipes for the purification of this alternative reagent. We anticipate that in low resource or point-of-need settings researchers could obtain the available constructs from Addgene or our lab and begin to develop their own assays, within whatever regulatory framework exists for them.","SUMMARYIn the analysis of genomic sequence data, so-called \u201calignment free\u201d approaches are often selected for their relative speed compared to alignment-based approaches, especially in the application of distance comparisons and taxonomic classification1,2,3,4. These methods are typically reliant on excising K-length substrings of the input sequence, called K-mers5. In the context of machine learning, K-mer based feature vectors have been used in applications ranging from amplicon sequencing classification to predictive modeling for antimicrobial resistance genes6,7,8. This can be seen as an analogy of the \u201cbag-of-words\u201d model successfully employed in natural language processing and computer vision for document and image classification9,10. Feature extraction techniques from natural language processing have previously been analogized to genomics data11; however, the \u201cbag-of-words\u201d approach is brittle in the RNA virus space due to the high intersequence variance and the exact matching requirement of K-mers. To reconcile the simplicity of \u201cbag-of-words\u201d methods with the complications presented by the intrinsic variance of RNA virus space, a method to resolve the fragility of extracted K-mers in a way that faithfully reflects an underlying biological phenomenon was devised. Our algorithm, Vorpal, allows the construction of interpretable linear models with clustered, representative \u2018degenerate\u2019 K-mers as the input vector and, through regularization, sparse predictors of binary phenotypes as the output. Here, we demonstrate the utility of Vorpal by identifying nucleotide-level genomic motif predictors for binary phenotypes in three separate RNA virus clades; human pathogen vs. non-human pathogen in Orthocoronavirinae, hemorrhagic fever causing vs. non-hemorrhagic fever causing in Ebolavirus, and human-host vs. non-human host in Influenza A. The capacity of this approach for in silico identification of hypotheses which can be validated by direct experimentation, as well as identification of genomic targets for preemptive biosurveillance of emerging viruses, is discussed. The code is available for download at https://github.com/mriglobal/vorpal.","ABSTRACTStrategies for containing an emerging infectious disease outbreak must be non-pharmaceutical when drugs or vaccines for the pathogen do not yet exist or are unavailable. The success of these non-pharmaceutical strategies will depend not only on the effectiveness of quarantine or other isolation measures but also on the epidemiological characteristics of the infection. However, there is currently no systematic framework to assess the relationship between different containment strategies and the natural history and epidemiological dynamics of the pathogen. Here, we compare the effectiveness of quarantine and symptom monitoring, implemented via contact tracing, in controlling epidemics using an agent-based branching model. We examine the relationship between epidemic containment and the disease dynamics of symptoms and infectiousness for seven case study diseases with diverse natural histories including Ebola, Influenza A, and Severe Acute Respiratory Syndrome (SARS). We show that the comparative effectiveness of symptom monitoring and quarantine depends critically on the natural history of the infectious disease, its inherent transmissibility, and the intervention feasibility in the particular healthcare setting. The benefit of quarantine over symptom monitoring is generally maximized for fast-course diseases, but we show the conditions under which symptom monitoring alone can control certain outbreaks. This quantitative framework can guide policy-makers on how best to use non-pharmaceutical interventions to contain emerging outbreaks and prioritize research during an outbreak of a novel pathogen.SIGNIFICANCEQuarantine and symptom monitoring of contacts with suspected exposure to an infectious disease are key interventions for the control of emerging epidemics; however, there does not yet exist a quantitative framework for comparing the control performance of each. Here, we use a mathematical model of seven case study diseases to show how the choice of intervention is influenced by the natural history of the infectious disease, its inherent transmissibility, and the intervention feasibility in the particular healthcare setting. We use this information to identify the most important characteristics of the disease and setting that need to be characterized for an emerging pathogen in order to make an informed decision between quarantine and symptom monitoring.","ABSTRACTHuman gremlin-1 is a physiologically versatile signaling molecule that has been associated with several human diseases including cancer. The ability of gremlin-1 to induce fibrosis in organs and transduce angiogenesis makes it a target for cancer therapy. RNAi-based therapy has proven to be very efficient and specific in tumor growth inhibition. The efficacy and specificity of siRNA-mediated gene silencing depends on the designing approaches. Here, empirical guidelines for siRNA design and comprehensive target site availability analysis were used to select effective siRNA from a plethora of potential candidates designed using several computation algorithms. Then, the selected siRNA candidates were subjected to stringent similarity searches in order to obtain siRNA candidates with reduced off-target effects (high specificity). The best candidates were compared to experimentally successful gremlin-1 siRNAs in order to predict the silencing potency of the selected siRNAs. siRNA-6 (sense strand: 5\u2019-CCAAGAAAUUCACUACCAU-3\u2019), siRNA-7 (sense strand: 5\u2019-CCAUGAUGGUCACACUCAA-3\u2019) and siRNA-47 (sense strand: 5\u2019-GGCCCAGCACAAUGACUCA-3\u2019) were predicted to be highly effective siRNA candidates for gremlin-1 silencing. These siRNAs can be considered for RNAi-based therapy because off-target effects are predicted to be minimal.","Single particle cryo-electron microscopy (cryoEM) is becoming widely adopted as a tool for structural characterization of biomolecules at near-atomic resolution. Vitrification of the sample to obtain a dense distribution of particles within a single field of view remains a major bottleneck for the success of such experiments. Here, we describe a simple and cost-effective method to increase the density of frozen-hydrated particles on grids with holey carbon support films. It relies on performing multiple rounds of sample application and blotting prior to plunge freezing in liquid ethane. We show that this approach is generally applicable and significantly increases particle density for a range of samples, such as small protein complexes, viruses and filamentous assemblies. The method is versatile, easy to implement, minimizes sample requirements and can enable characterization of samples that would otherwise resist structural studies using single particle cryoEM.","ABSTRACTZika virus (ZIKV) infection in humans has been associated with congenital malformations and other neurological disorders, such as Guillain-Barr\u00e9 syndrome. The mechanism(s) of ZIKV intrauterine transmission, the cell types involved, the most vulnerable period of pregnancy for severe outcomes from infection and other physiopathological aspects remain unknown. In this study, we analyzed placental samples obtained at the time of delivery from a group of twenty-four women diagnosed with ZIKV infection during the first, second or third trimesters of pregnancy. Villous immaturity was the main histological finding in the placental tissues, although placentas without alterations were also frequently observed. Significant enhancement of the number of syncytial sprouts was observed in the placentas of women infected during the third trimester, indicating the development of placental abnormalities after ZIKV infection. Hyperplasia of Hofbauer cells (HCs) was also observed in these third-trimester placental tissues, and remarkably, HCs were the only ZIKV-positive fetal cells found in the placentas studied that persisted until birth, as revealed by immunohistochemical (IHC) analysis. Thirty-three percent of women infected during pregnancy delivered infants with congenital abnormalities, although no pattern correlating the gestational stage at infection, the IHC positivity of HCs in placental tissues and the presence of congenital malformations at birth was observed. Placental tissue analysis enabled us to confirm maternal ZIKV infection in cases where serum from the acute infection phase was not available, which reinforces the importance of this technique in identifying possible causal factors of birth defects. The results we observed in the samples from naturally infected pregnant women may contribute to the understanding of some aspects of the pathophysiology of ZIKV.","AbstractInfectious bronchitis (IB) causes significant economic losses in the global poultry industry. Control of infectious bronchitis is hindered by the genetic diversity of the causative agent, infectious bronchitis virus (IBV), which has led to the emergence of several serotypes that lack complete serologic cross-protection. While serotyping by definition requires immunologic characterization, genotyping is an efficient means to identify IBVs detected in samples. Sanger sequencing of the S1 subunit of the spike gene is currently used to genotype IBV; however, the universal S1 PCR was created to work from cultured IBV and it is inefficient at detecting mixed isolates. This paper describes a MinION-based AmpSeq method that genetically typed IBV from clinical samples, including samples with multiple isolates. Total RNA was extracted from fifteen tracheal scrapings and choanal cleft swab samples, randomly reverse transcribed, and PCR amplified using modified S1-targeted primers. Amplicons were barcoded to allow for pooling of samples, processed per manufacturer\u2019s instructions into a 1D MinION sequencing library, and sequenced on the MinION. The AmpSeq method detected IBV in 13 of 14 IBV-positive samples. AmpSeq accurately detected and genotyped both IBV lineages in three of five samples containing two IBV lineages. Additionally, one sample contained three IBV lineages, and AmpSeq accurately detected two of the three. Strain identification, including detection of different strains from the same lineage, was also possible with this AmpSeq method. The results demonstrate the feasibility of using MinION-based AmpSeq for rapid and accurate identification and lineage typing of IBV from oral swab samples.","Because of the potential link between -1 programmed ribosomal frameshifting and response of a pseudoknot (PK) RNA to force, a number of single molecule pulling experiments have been performed on PKs to decipher the mechanism of programmed ribosomal frameshifting. Motivated in part by these experiments, we performed simulations using a coarse-grained model of RNA to describe the response of a PK over a range of mechanical forces (fs) and monovalent salt concentrations (Cs). The coarse-grained simulations quantitatively reproduce the multistep thermal melting observed in experiments, thus validating our model. The free energy changes obtained in simulations are in excellent agreement with experiments. By varying f and C, we calculated the phase diagram that shows a sequence of structural transitions, populating distinct intermediate states. As f and C are changed, the stem-loop tertiary interactions rupture first, followed by unfolding of the 3\u2019-end hairpin (I\u21ccF). Finally, the 5\u2019-end hairpin unravels, producing an extended state (E\u21ccI). A theoretical analysis of the phase boundaries shows that the critical force for rupture scales as (log Cm)\u03b1 with \u03b1 = 1 (0.5) for E\u21ccI (I\u21ccF) transition. This relation is used to obtain the preferential ion-RNA interaction coefficient, which can be quantitatively measured in single-molecule experiments, as done previously for DNA hairpins. A by-product of our work is the suggestion that the frameshift efficiency is likely determined by the stability of the 5\u2019 end hairpin that the ribosome first encounters during translation.","AbstractThe World Health Organization (WHO) has declared the 2019 novel coronavirus (2019-nCoV) infection outbreak a global health emergency. Currently, there is no effective anti-2019-nCoV medication. The sequence identity of the 3CL proteases of 2019-nCoV and SARS is 96%, which provides a sound foundation for structural-based drug repositioning (SBDR). Based on a SARS 3CL protease X-ray crystal structure, we construct a 3D homology structure of 2019-nCoV 3CL protease. Based on this structure and existing experimental datasets for SARS 3CL protease inhibitors, we develop an SBDR model based on machine learning and mathematics to screen 1465 drugs in the DrugBank that have been approved by the U.S. Food and Drug Administration (FDA). We found that many FDA approved drugs are potentially highly potent to 2019-nCoV.","Mutations in the E3 ubiquitin ligase parkin are the most common known cause of autosomal recessive parkinsonism. Multiple types of stress decrease parkin protein levels, an effect that may be relevant to sporadic Parkinson's disease (PD), but the mechanism(s) involved in this loss remain largely unclear. We sought to elucidate these mechanisms using a PD-relevant stressor, L-DOPA, the precursor to dopamine, which forms reactive oxygen species (ROS) as well as toxic quinones via auto-oxidation. We find that L-DOPA causes parkin loss through both an oxidative stress-independent and an oxidative stress-dependent pathway. Characterization of the latter reveals that it requires both the kinase PINK1 and parkin's interaction with phosphorylated ubiquitin (phospho-Ub) and is mediated by proteasomal degradation. Surprisingly, mitochondrial parkin activity and autoubiquitination as well as mitophagy are not required for such loss. During stress induced by the oxidative stressor hydrogen peroxide or the metabolic uncoupler CCCP, parkin degradation also requires its association with phospho-Ub, indicating that this mechanism is broadly generalizable. As oxidative stress, metabolic dysfunction and phospho-Ub levels are all elevated in PD patients, we suggest that these changes may lead to the loss of parkin expression in PD.","The coronavirus SARS-CoV-2 has infected more than 600,000 people and has overwhelmed hospital systems around the world. However, the factors mediating fatal SARS-CoV-2 infections are poorly understood. Here, we show that cigarette smoke causes a dose-dependent upregulation of Angiotensin Converting Enzyme 2 (ACE2), the SARS-CoV-2 receptor, in rodent and human lungs. Using single-cell sequencing data, we demonstrate that ACE2 is expressed in a subset of epithelial cells that line the respiratory tract, including goblet cells, club cells, and alveolar type 2 cells. Chronic smoke exposure triggers a protective expansion of mucus-secreting goblet cells and a concomitant increase in ACE2 expression. In contrast, quitting smoking causes a decrease in lung ACE2 levels. Taken together, these results may partially explain why smokers are particularly likely to develop severe SARS-CoV-2 infections, and they suggest that quitting smoking could lessen coronavirus susceptibility.","AbstractCellular membranes display an incredibly diverse range of shapes, both in the plasma membrane and at membrane bound organelles. These morphologies are intricately related to cellular functions, enabling and regulating fundamental membrane processes. However, the biophysical mechanisms at the origin of these complex geometries are not fully understood from the standpoint of membrane-protein coupling. In this work, we focused on a minimal model of helicoidal ramps representative of specialized endoplasmic reticulum compartments. Given a helicoidal membrane geometry, we asked what is the distribution of spontaneous curvature required to maintain this shape at mechanical equilibrium? Based on the Helfrich energy of elastic membranes with spontaneous curvature, we derived the shape equation for minimal surfaces, and applied it to helicoids. We showed the existence of switches in the sign of the spontaneous curvature associated with geometric variations of the membrane structures. Furthermore, for a prescribed gradient of spontaneous curvature along the exterior boundaries, we identified configurations of the helicoidal ramps that are confined between two infinitely large energy barriers. Overall our results suggest possible mechanisms for geometric control of helicoidal ramps in membrane organelles based on curvature-inducing proteins.","ABSTRACTMetals are a limiting resource for pathogenic bacteria and must be scavenged from host proteins. Hemoglobin provides the most abundant source of iron in the human body and is required by several pathogens to cause invasive disease. However, the consequences of hemoglobin evolution for bacterial nutrient acquisition remain unclear. Here we show that the \u03b1- and \u03b2-globin genes exhibit strikingly parallel signatures of adaptive evolution across simian primates. Rapidly evolving sites in hemoglobin correspond to binding interfaces of IsdB, a bacterial hemoglobin receptor encoded by pathogenic Staphylococcus aureus. Using an evolution-guided experimental approach, we demonstrate that divergence between primates and staphylococcal isolates governs hemoglobin recognition and bacterial growth. Reintroducing putative adaptive mutations in \u03b1- or \u03b2-globin proteins is sufficient to impair S. aureus binding, providing a mechanism for the evolution of disease resistance. These findings suggest that bacterial hemoprotein capture has driven repeated evolutionary conflicts with hemoglobin during primate descent.","An emerging disease is one infectious epidemic caused by a newly transmissible pathogen, which has either appeared for the first time or already existed in human populations, having the capacity to increase rapidly in incidence as well as geographic range. Adapting to human immune system, emerging diseases may trigger large-scale pandemic spreading, such as the transnational spreading of SARS, the global outbreak of A(H1N1), and the recent potential invasion of avian influenza A(H7N9). To study the dynamics mediating the transmission of emerging diseases, spatial epidemiology of networked metapopulation provides a valuable modeling framework, which takes spatially distributed factors into consideration. This review elaborates the latest progresses on the spatial metapopulation dynamics, discusses empirical and theoretical findings that verify the validity of networked metapopulations, and the application in evaluating the effectiveness of disease intervention strategies as well.","This study tests the hypothesis that activation of mitogen-activated protein kinase (MAPK) by physiologically-relevant concentrations of interleukin-33 (IL-33) contributes to enhanced cytokine expression by IL-12 stimulated human natural killer (NK) cells. While IL-33 canonically triggers type 2 cytokine responses, this cytokine can also synergize with type 1 cytokines like IL-12 to provoke interferon-gamma (IFN-\u03b3). We show that picogram concentrations of IL-12 and IL-33 are sufficient to promote robust secretion of IFN-\u03b3 by human NK cells that greatly exceeds responses to either cytokine alone. Nanogram doses of IL-33, potentially consistent with levels in tissue microenvironments, synergize with IL-12 to induce secretion of additional cytokines, including tumor necrosis factor (TNF) and granulocyte-macrophage colony-stimulating factor (GM-CSF). IL-33-induced activation of the p38 MAPK pathway in human NK cells is crucial for enhanced release of IFN-\u03b3 and TNF in response to IL-12. Mechanistically, IL-33-induced p38 MAPK signaling enhances stability of IFNG transcripts and triggers ADAM17-mediated cleavage of TNF from the cell surface. These data support our hypothesis and suggest that altered sensitivity of NK cells to IL-12 in the presence of IL-33 may have important consequences in diseases associated with mixed cytokine milieus, like asthma and chronic obstructive pulmonary disease.Graphical Abstract","AbstractRespiratory syncytial virus (RSV) is the leading cause of severe respiratory disease in infants. Other than age at the time of infection, the causes and correlates of severe illness in infants lacking known risk factors are poorly defined. We recruited a cohort of confirmed RSV-infected infants and simultaneously assayed the presence of resident airway microbiota and the molecular status of their airways using a novel method. Rigorous statistical analyses identified a molecular airway gene expression signature of severe illness dominated by excessive chemokine expression. Global 16S rRNA sequencing confirmed an association between H. influenzae and clinical severity. Interestingly, adjusting for H. influenzae in our gene expression analysis revealed an association between severity and airway lymphocyte accumulation. Exploring the relationship between airway gene expression and the time of onset of clinical symptoms revealed a robust, acute activation of interferon (IFN) signaling, which was absent in subjects with severe illness. Finally, we explored the relationship between IFN activity, airway gene expression and productive RSV infection using a novel in vitro model of bona fide pediatric human airway epithelial cells. Interestingly, blocking IFN signaling, but not IFN ligand production, in these cells leads to increased viral infection. Our data reveal that acute airway interferon responses are physiologically relevant in the context of infant RSV infection and may be a target for therapeutic intervention. Additionally, the airway gene expression signature we define may be useful as a biomarker for efficacy of intervention responses.","AbstractDomestication of plants and animals is the foundation for feeding the world population. We report that domestication of the model yeast S. cerevisiae reprogrammed its life cycle entirely. We tracked growth, gamete formation and cell survival across many environments for nearly 1000 genome sequenced isolates and found a remarkable dichotomy between domesticated and wild yeasts. Wild yeasts near uniformly trigger meiosis and sporulate when encountering nutrient depletions, whereas domestication relaxed selection on sexual reproduction and favoured survival as quiescent cells. Domestication also systematically enhanced fermentative over respiratory traits while decreasing stress tolerance. We show that this yeast domestication syndrome was driven by aneuploidies and gene function losses that emerged independently in multiple domesticated lineages during the specie\u2019s recent evolutionary history. We found domestication to be the most dramatic event in budding yeast evolution, raising questions on how much domestication has distorted our understanding of this key model species.","AbstractTaxonomic classification for the Coronaviridae can be challenging, due to the wide host tropism and highly variable genome of the viruses in this Family. Within the Alphacoronavirus genus, there is a single species Alphacoronavirus 1 that encompasses several biologically distinct viruses of distinct animal species. Here, we carried out phylogenetic analysis of members of the Alphacoronavirus genus, focusing on the viral spike gene, which is a primary driver of viral tropism and pathogenesis. We identify two distinct clades (A and B) within the Alphacoronavirus 1 species. Alphacoronavirus 1 clade A encompasses serotype I FCoV and CCoV, and Alphacoronavirus 1 clade B, encompasses serotype II FCoV and CCoV and TGEV-like viruses. We propose this clade designation, along with the newly proposed Alphacoronavirus 2 species, as an improved way to classify the diverse Alphacoronavirus genus.","AbstractDelays in case reporting are common to disease surveillance systems, making it difficult to track diseases in real-time. \u201cNowcast\u201d approaches attempt to estimate the complete case counts for a given reporting date, using a time series of case reports that is known to be incomplete due to reporting delays. Modeling the reporting delay distribution is a common feature of nowcast approaches. However, many nowcast approaches ignore a crucial feature of infectious disease transmission\u2014that future cases are intrinsically linked to past reported cases\u2014and are optimized to a single application, which may limit generalizability. Here, we present a Bayesian approach, NobBS (Nowcasting by Bayesian Smoothing) capable of producing smooth and accurate nowcasts in multiple disease settings. We test NobBS on dengue in Puerto Rico and influenza-like illness (ILI) in the United States to examine performance and robustness across settings exhibiting a range of common reporting delay characteristics (from stable to time-varying), and compare this approach with a published nowcasting package. We show that introducing a temporal relationship between cases considerably improves performance when the reporting delay distribution is time-varying, and we identify trade-offs in the role of moving windows to accurately capture changes in the delay. We present software implementing this new approach (R package \u201cNobBS\u201d) for widespread application.SignificanceAchieving accurate, real-time estimates of disease activity is challenged by delays in case reporting. However, approaches that seek to estimate cases in spite of reporting delays often do not consider the temporal relationship between cases during an outbreak, nor do they identify characteristics of robust approaches that generalize to a wide range of surveillance contexts with very different reporting delays. Here, we present a smooth Bayesian nowcasting approach that produces accurate estimates that capture the time evolution of the epidemic curve and outperform a previous approach in the literature. We assess the performance for two diseases to identify important features of the reporting delay distribution that contribute to the model\u2019s performance and robustness across surveillance settings.","AbstractHere we have generated 3D structures of glycoforms of the spike (S) protein SARS-CoV-2, based on reported 3D structures for the S protein and on reported glycomics data for the protein produced in HEK293 cells. We also report structures for glycoforms that represent those present in the nascent glycoproteins (prior to enzymatic modifications in the Golgi and ER), as well as those that are commonly observed on antigens present in other viruses.These models were subjected to MD simulation to take into account protein and glycan plasticity, and to determine the extent to which glycan microheterogeneity impacts antigenicity. Lastly, we have identified peptides in the S protein that are likely to be presented in human leukocyte antigen (HLA) complexes, and discuss the role of S protein glycosylation in potentially modulating the adaptive immune response to the SARS-CoV-2 virus or to a related vaccine.","AbstractCoronavirus disease 2019 (COVID-19) is newly emerging human infectious diseases, which is caused by Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2, also previously known as 2019-nCoV). Within two months of the outbreak, more than 80,000 cases of COVID-19 have been confirmed worldwide. Since the human to human transmission occurred easily and the human infection is rapidly increasing, the sensitive and early diagnosis is essential to prevent the global outbreak. Recently, World Health Organization (WHO) announced various primer and probe sets for SARS-CoV-2 previously developed in China, Germany, Hong Kong, Japan, Thailand, and USA. In this study, we compared the ability to detect SARS-CoV-2 RNA among the seven primer-probe sets for N gene and the three primer-probe sets for Orf1 gene. The result of the comparative analysis represented that the \u20182019-nCoV_N2, N3\u2019 of USA and the \u2018ORF1ab\u2019 of China are the most sensitive primer-probe sets for N and Orf1 genes, respectively. Therefore, the appropriate combination from ORF1ab (China), 2019-nCoV_N2, N3 (USA), and NIID_2019-nCOV_N (Japan) sets should be selected for the sensitive and reliable laboratory confirmation of SARS-CoV-2.","ABSTRACTCellular and viral factors participate in the replication cycle of rotavirus. We report that the guanine nucleotide exchange factor GBF1, which activates the small GTPase Arf1 to induce COPI transport processes, is required for rotavirus replication since knocking down GBF1 expression by RNA interference, or inhibiting its activity by treatment with Brefeldin A (BFA) or Golgicide A (GCA) significantly reduce the yield of infectious viral progeny. This reduction in virus yield was related to a block in virus assembly since in the presence of either BFA or GCA the assembly of infectious mature triple-layered virions was significantly prevented and only doubled layered-particles were detected. We report that the catalytic activity of GBF1, but not the activation of Arf1, is essential for the assembly of the outer capsid of rotavirus. We show that both BFA and GCA, as well as interfering with the synthesis of GBF1, alter the electrophoretic mobility of glycoproteins VP7 and NSP4 and block the trimerization of the virus surface VP7, a step required for its incorporation into virus particles. Although a post-translational modification of VP7 (other than glycosylation) could be related to the lack of trimerization, we found that NSP4 might also be involved in this process, since knocking-down its expression reduces VP7 trimerizarion. In support, recombinant VP7 protein overexpressed in transfected cells formed trimers only when co-transfected with NSP4.IMPORTANCERotavirus, a member of the family Reoviridae, is the major cause of severe diarrhea in children and young animals worldwide. Despite the significant advances in the characterization of the biology of this virus, the mechanisms involved in morphogenesis of the virus particle are still poorly understood. In this work, we show that the guanine nucleotide exchange factor GBF1, relevant for the COPI/Arf1-mediated cellular vesicular transport, participates in the replication cycle of the virus, influencing the correct processing of viral glycoproteins VP7 and NSP4, and the assembly of the virus surface proteins VP7 and VP4.","AbstractThe recent outbreak of pneumonia in Wuhan, China caused by the 2019 Novel Coronavirus (2019-nCoV) emphasizes the importance of detecting novel viruses and predicting their risks of infecting people. In this report, we introduced the VHP (Virus Host Prediction) to predict the potential hosts of viruses using deep learning algorithm. Our prediction suggests that 2019-nCoV has close infectivity with other human coronaviruses, especially the severe acute respiratory syndrome coronavirus (SARS-CoV), Bat SARS-like Coronaviruses and the Middle East respiratory syndrome coronavirus (MERS-CoV). Based on our prediction, compared to the Coronaviruses infecting other vertebrates, bat coronaviruses are assigned with more similar infectivity patterns with 2019-nCoVs. Furthermore, by comparing the infectivity patterns of all viruses hosted on vertebrates, we found mink viruses show a closer infectivity pattern to 2019-nCov. These consequences of infectivity pattern analysis illustrate that bat and mink may be two candidate reservoirs of 2019-nCov.These results warn us to beware of 2019-nCoV and guide us to further explore the properties and reservoir of it.One Sentence SummaryIt is of great value to identify whether a newly discovered virus has the risk of infecting human. Guo et al. proposed a virus host prediction method based on deep learning to detect what kind of host a virus can infect with DNA sequence as input. Applied to the Wuhan 2019 Novel Coronavirus, our prediction demonstrated that several vertebrate-infectious coronaviruses have strong potential to infect human. This method will be helpful in future viral analysis and early prevention and control of viral pathogens.","AbstractCoronaviruses (CoVs) are enveloped, positive-sense RNA viruses with an unusually large RNA genome and a unique replication strategy. They cause important diseases in mammals and birds ranging from enteritis in cows and pigs and upper respiratory disease in chickens, to potentially lethal human respiratory infections. Here, we apply ribosome profiling and parallel RNA sequencing to analyse global changes in host cell transcriptome and translatome upon infection with mouse hepatitis virus strain A59 (MHV-A59), a model murine coronavirus in the same genus as the human pathogens severe acute respiratory syndrome coronavirus (SARS-CoV) and Middle East respiratory syndrome coronavirus (MERS-CoV). Amongst differentially-regulated cellular genes, we observed up-regulation of all arms of the unfolded protein response (UPR), including translational activation of transcription factors ATF4, ATF5 and Chop. Polysome profiling of infected-cells revealed an accumulation of empty 80S ribosomes, consistent with increased phosphorylation of eIF2\u03b1 leading to translational shut-off via inhibited initiation. Ribosomal footprints on phosphorylated-eIF2\u03b1-resistant mRNAs revealed unambiguous upstream open reading frame (uORF) occupancy consistent with host maintenance of the UPR. Unexpectedly, an inhibitor of PERK that blocks the UPR and relieves translation inhibition was found to attenuate virus growth suggesting that MHV may subvert the UPR to its own advantage. This study sheds new light on the complex interactions between MHV and host during infection and provides new potential targets for antiviral intervention.","ABSTRACTOutbreak of coronavirus disease 2019 (COVID-19) occurred in Wuhan and has rapidly spread to almost all parts of world. In coronaviruses, the receptor binding domain (RBD) in the distal part of S1 subunit of SARS-CoV-2 spike protein can directly bind to angiotensin converting enzyme 2 (ACE2). RBD promote viral entry into the host cells and is an important therapeutic target. In this study, we discovered that theaflavin showed the lower idock score (idock score: \u22127.95 kcal/mol). To confirm the result, we discovered that theaflavin showed FullFitness score of \u2212991.21 kcal/mol and estimated \u0394G of \u22128.53 kcal/mol for the most favorable interaction with contact area of SARS-CoV-2 RBD by SwissDock service. Regarding contact modes, hydrophobic interactions contribute significantly in binding and additional hydrogen bonds were formed between theaflavin and Arg454, Phe456, Asn460, Cys480, Gln493, Asn501 and Val503 of SARS-CoV-2 RBD, near the direct contact area with ACE2. Our results suggest that theaflavin could be the candidate of SARS-CoV-2 entry inhibitor for further study.","AbstractBats are reservoirs for a number of highly pathogenic zoonotic viruses, yet they remain relatively asymptomatic during infection. Whether this viral resistance is due to a unique innate immune system is unknown. An evolutionarily conserved feature of vertebrate antiviral immunity is the interferon (IFN) response, which triggers cellular defenses through interferon-stimulated gene (ISG) expression. While bats encode an intact IFN system, global ISG expression patterns in bat cells are not well characterized. Here, we used RNA-Seq to assess the transcriptional response to IFN\u03b1 in cells derived from the bat Pteropus alecto (black flying fox). We show induction of more than 100 transcripts, most of which are canonical ISGs observed in other species. Kinetic gene profiling revealed that P. alecto ISGs fall into two unique temporal subclusters with similar early induction kinetics but distinct late-phase declines. In contrast to bat ISGs, human ISGs generally remained elevated for longer periods following IFN treatment, suggesting host-based differences in gene regulatory mechanisms. Notably, we also identified a small group of non-canonical bat ISGs, including an enzymatically active RNASEL that plays a role in controlling viral infection. These studies provide insight into the innate immune response of an important viral reservoir and lay a foundation for studies into the immunological features that may underlie unique virus-host relationship in bats.Significance StatementBats are considered unique in their ability to resist disease caused by viruses that are often pathogenic in humans. While the nature of this viral resistance is unknown, genomic data suggest bat innate immune systems may be specialized in controlling these disease-causing viruses. A critical cell intrinsic antiviral defense system in vertebrates is the interferon response, which suppresses viral infection through induction of hundreds of interferon-stimulated genes (ISGs). In this study, we report the repertoire of ISGs and several unique features of ISG induction kinetics in bat cells. We also characterize induction and antiviral activity of bat RNASEL, which is induced by IFN in bat, but not human cells. These studies lay the foundation for discovery of potentially new antiviral mechanisms in bats, which may spur research into development of therapies to combat viral infection.","AbstractThe emergence of intestinal organoids, as a stem cell-based self-renewable model system, has led to many studies on intestinal development and cell-cell signaling. However, potential issues regarding the phenotypic stability and reproducibility of the methodology during culture still needs to be addressed for different organoids. Here we investigated the transcriptomes of intestinal organoids derived from the same pig as well as batch-to-batch variation of organoids derived from different pigs over long-term passage. The set of genes expressed in organoids closely resembled that of the tissue of origin, including location specific functions, for at least 17 passages. Minor differences in gene expression were observed between individual organoid cultures. In contrast, most tissue-specific genes were not expressed in the transformed jejunum cell line IPECJ2, which also showed gene expression consistent with cancer phenotypes. We conclude that intestinal organoids provide a robust and stable model for translational research with clear advantages over transformed cells.","AbstractA newly identified coronavirus, 2019-nCoV, has been posing significant threats to public health since December 2019. ACE2, the host cell receptor for severe acute respiratory syndrome coronavirus (SARS), has recently been demonstrated in mediating 2019-nCoV infection. Interestingly, besides the respiratory system, substantial proportion of SARS and 2019-nCoV patients showed signs of various degrees of liver damage, the mechanism and implication of which have not yet been determined. Here, we performed an unbiased evaluation of cell type specific expression of ACE2 in healthy liver tissues using single cell RNA-seq data of two independent cohorts, and identified specific expression in cholangiocytes. The results indicated that virus might directly bind to ACE2 positive cholangiocytes but not necessarily hepatocytes. This finding suggested the liver abnormalities of SARS and 2019-nCoV patients may not be due to hepatocyte damage, but cholangiocyte dysfunction and other causes such as drug induced and systemic inflammatory response induced liver injury. Our findings indicate that special care of liver dysfunction should be installed in treating 2019-nCoV patients during the hospitalization and shortly after cure.","AbstractRapid production and publication of pathogen genome sequences during emerging disease outbreaks provide crucial public health information. In resource-limited settings, especially near an outbreak epicenter, conventional deep sequencing or bioinformatics are often challenging. Here we successfully used metagenomic next generation sequencing on an iSeq100 Illumina platform paired with an open-source bioinformatics pipeline to quickly characterize Cambodia\u2019s first case of COVID-2019.","AbstractThe newly identified SARS-CoV-2 has now been reported from around 183 countries with more than a million confirmed human cases including more than 68000 deaths. The genomes of SARS-COV-2 strains isolated from different parts of the world are now available and the unique features of constituent genes and proteins have gotten substantial attention recently. Spike glycoprotein is widely considered as a possible target to be explored because of its role during the entry of coronaviruses into host cells. We analyzed 320 whole-genome sequences and 320 spike protein sequences of SARS-CoV-2 using multiple sequence alignment tools. In this study, 483 unique variations have been identified among the genomes including 25 non-synonymous mutations and one deletion in the spike protein of SARS-CoV-2. Among the 26 variations detected, 12 variations were located at the N-terminal domain and 6 variations at the receptor-binding domain (RBD) which might alter the interaction with receptor molecules. In addition, 22 amino acid insertions were identified in the spike protein of SARS-CoV-2 in comparison with that of SARS-CoV. Phylogenetic analyses of spike protein revealed that Bat coronavirus have a close evolutionary relationship with circulating SARS-CoV-2. The genetic variation analysis data presented in this study can help a better understanding of SARS-CoV-2 pathogenesis. Based on our findings, potential inhibitors can be designed and tested targeting these proposed sites of variation.","AbstractSARS-CoV-2 is a novel coronavirus currently causing a pandemic. We show that the majority of amino acid positions, which differ between SARS-CoV-2 and the closely related SARS-CoV, are differentially conserved suggesting differences in biological behaviour. In agreement, novel cell culture models revealed differences between the tropism of SARS-CoV-2 and SARS-CoV. Moreover, cellular ACE2 (SARS-CoV-2 receptor) and TMPRSS2 (enables virus entry via S protein cleavage) levels did not reliably indicate cell susceptibility to SARS-CoV-2. SARS-CoV-2 and SARS-CoV further differed in their drug sensitivity profiles. Thus, only drug testing using SARS-CoV-2 reliably identifies therapy candidates. Therapeutic concentrations of the approved protease inhibitor aprotinin displayed anti-SARS-CoV-2 activity. The efficacy of aprotinin and of remdesivir (currently under clinical investigation against SARS-CoV-2) were further enhanced by therapeutic concentrations of the proton pump inhibitor omeprazole (aprotinin 2.7-fold, remdesivir 10-fold). Hence, our study has also identified anti-SARS-CoV-2 therapy candidates that can be readily tested in patients.","There are outstanding evolutionary questions on the recent emergence of coronavirus SARS-CoV-2/hCoV-19 in Hubei province that caused the COVID-19 pandemic, including (1) the relationship of the new virus to the SARS-related coronaviruses, (2) the role of bats as a reservoir species, (3) the potential role of other mammals in the emergence event, and (4) the role of recombination in viral emergence. Here, we address these questions and find that the sarbecoviruses -- the viral subgenus responsible for the emergence of SARS-CoV and SARS-CoV-2 -- exhibit frequent recombination, but the SARS-CoV-2 lineage itself is not a recombinant of any viruses detected to date. In order to employ phylogenetic methods to date the divergence events between SARS-CoV-2 and the bat sarbecovirus reservoir, recombinant regions of a 68-genome sarbecovirus alignment were removed with three independent methods. Bayesian evolutionary rate and divergence date estimates were consistent for all three recombination-free alignments and robust to two different prior specifications based on HCoV-OC43 and MERS-CoV evolutionary rates. Divergence dates between SARS-CoV-2 and the bat sarbecovirus reservoir were estimated as 1948 (95% HPD: 1879-1999), 1969 (95% HPD: 1930-2000), and 1982 (95% HPD: 1948-2009). Despite intensified characterization of sarbecoviruses since SARS, the lineage giving rise to SARS-CoV-2 has been circulating unnoticed for decades in bats and been transmitted to other hosts such as pangolins. The occurrence of a third significant coronavirus emergence in 17 years together with the high prevalence and virus diversity in bats implies that these viruses are likely to cross species boundaries again.","AbstractBackgroundNipah Virus (NiV) came into limelight recently due to an outbreak in Kerala, India. NiV causes severe disease and death in people with over 75% case fatality rate. It is a public health concern and has the potential to become a global pandemic. Lack of treatment has forced the containment methods to be restricted to isolation and surveillance. WHO\u2019s \u2018R&amp;D Blueprint list of priority diseases\u2019 (2018) indicates that there is an urgent need for accelerated research &amp; development for addressing NiV.Materials &amp; MethodsIn the quest for druglike NiV inhibitors (NVIs) a thorough literature search followed by systematic data curation was conducted. Rigorous data analysis was done with curated NVIs for prioritizing druglike compounds. For the same, more than 1800 descriptors of NVIs were computed and comparative analysis was performed with the FDA approved small molecules and antivirals. These compounds were further evaluated through PAINS filter to study their toxicity profile. Simultaneously, compounds were also prioritized based on the robustness of the assays through which they were identified.ResultsOur efforts lead to the creation of a well-curated structured knowledgebase of 182 NVIs with 98 small molecule inhibitors. The reported IC50/EC50 values for some of these inhibitors are in the nanomolar range \u2013 as low as 0.47 nM. In order to prioritize these inhibitors, we performed several tests and applied filters to identify drug-like non-toxic compounds. Of 98, a few compounds passed DruLito &amp; PAINS filters exhibiting drug-like properties and were also prioritized in an independent screen based only the assay robustness. The NVIs have diverse structural features and offer a wide spectrum of ways in which they can be developed further as druglike molecules.ConclusionWe report a knowledgebase for furthering the development of NVIs. The platform has a diverse set of 98 NVIs of which a few have been prioritized based on a combined evidence strategy. The platform has the provision to submit new inhibitors as and when reported by the community for further enhancement of NiV inhibitor landscape.","AbstractPrimary human airway epithelial cell (hAEC) cultures represent a universal platform to propagate respiratory viruses and characterize their host interactions in authentic target cells. To further elucidate specific interactions between human respiratory viruses and important host factors in airway epithelium, it is important to make hAEC cultures amenable to genetic modification. However, the short and finite lifespan of primary cells in cell culture creates a bottleneck for the genetic modification of these cultures. In the current study, we show that the incorporation of the Rho-associated protein kinase (ROCK) inhibitor (Y-27632) during cell propagation extends the life span of primary human cells in vitro and thereby facilitates the incorporation of lentivirus-based expression systems. Using fluorescent reporters for FACS-based sorting, we generated homogenously fluorescent hAEC cultures that differentiate normally after lentiviral transduction. As proof-of-principle, we demonstrate that host gene expression can be modulated post-differentiation via inducible short hairpin (sh)RNA-mediated knockdown. Importantly, functional characterization of these transgenic hAEC cultures with exogenous poly(I:C), as a proxy for virus infection, demonstrates that such modifications do not influence the host innate immune response. Moreover, the propagation kinetics of both human coronavirus 229E (HCoV-229E) and human respiratory syncytial virus (RSV) were not affected. Combined, these results validate our newly established protocol for the genetic modification of hAEC cultures thereby unlocking a unique potential for detailed molecular characterization of virus \u2013 host interactions in human respiratory epithelium.","AbstractThe genetic variability of Infectious bronchitis virus (IBV) is one of the main challenges for its control, hindering not only the development of effective vaccination strategies but also its classification and, consequently, epidemiology understanding. The 624/I and Q1 genotypes, now recognized to be part of the GI-16 lineage, represent an excellent example of the practical consequences of IBV molecular epidemiology limited knowledge. In fact, being their common origin unrecognized for a long time, independent epidemiological pictures were drawn for the two genotypes. To fix this misinterpretation, the present study reconstructs the history, population dynamics and spreading patterns of GI-16 lineage as a whole using a phylodynamic approach. A collection of worldwide available hypervariable region 1 and 2 (HVR12) and 3 (HVR3) sequences of the S1 protein was analysed together with 258 HVR3 sequences obtained from samples collected in Italy (the country where this genotype was initially identified) since 1963. The results demonstrate that after its emergence at the beginning of the XX century, GI-16 was able to persist until present days in Italy. Approximately in the late 1980s, it migrated to Asia, which became the main nucleus for further spreading to Middle East, Europe and especially South America, likely through multiple introduction events. A remarkable among-country diffusion was also demonstrated in Asia and South America. Interestingly, although most of the recent Italian GI-16 strains originated from ancestral viruses detected in the same country, a couple were closely related to Chinese ones, supporting a backward viral flow from China to Italy.Besides to the specific case-study results, this work highlights the misconceptions that originate from the lack of a unified nomenclature and poor molecular epidemiology data generation and sharing. This shortcoming appears particularly relevant since the described scenario could likely be shared by many other IBV genotypes and pathogens in general.","The ongoing historic outbreak of COVID-19 not only constitutes a global public health crisis, but also carries a devastating social and economic impact. The disease is caused by a newly identified coronavirus, Severe Acute Respiratory Syndrome coronavirus 2 (SARS-CoV-2). There is an urgent need to identify antivirals to curtail the COVID-19 pandemic. Herein, we report the remarkable sensitivity of SARS-CoV-2 to recombinant human interferons \u03b1 and \u03b2 (IFN\u03b1/\u03b2). Treatment with IFN-\u03b1 at a concentration of 50 international units (IU) per milliliter drastically reduces viral titers by 3.4 log or over 4 log, respectively, in Vero cells. The EC50 of IFN-\u03b1 and IFN-\u03b2 treatment is 1.35 IU/ml and 0.76 IU/ml, respectively, in Vero cells. These results suggest that SARS-CoV-2 is more sensitive than many other human pathogenic viruses, including SARS-CoV. Overall, our results demonstrate the potent efficacy of human Type I IFN in suppressing SARS-CoV-2 infection, a finding which could inform future treatment options for COVID-19.","AbstractAn outbreak of new SARS-like viral in Wuhan, China has been named 2019-nCoV. The current state of the epidemic is increasingly serious, and there has been the urgent necessity to develop an effective new drug. In previous studies, it was found that the conformation change in CTD1 was the region where SARS-CoV bound to human ACE2. Although there are mutations of the 2019-nCoV, the binding energy of ACE2 remains high. The surface glycoprotein of 2019-nCoV was coincident with the CTD1 region of the S-protein by comparing the I-TASSER prediction model with the actual SARS model, which suggests that 2019-nCoV may bind to the ACE2 receptor through conformational changes. Furthermore, site prediction on the surface glycoprotein of 2019-nCoV suggests some core amino acid area may be a novel drug target against 2019-nCoV.","ABSTRACTSilver nanoparticles (AgNPs) have wide-ranging applications, including as additives in consumer products and in medical diagnostics and therapy. Therefore understanding how AgNPs interact with biological systems is important for ascertaining any potential health risks due to the likelihood of high levels of human exposure. Besides any severe, acute effects, it is desirable to understand more subtle interactions that could lead to milder, chronic health impacts. Nanoparticles are small enough to be able to enter biological cells and interfere with their internal biochemistry. The initial contact between nanoparticle and cell is at the plasma membrane. To gain fundamental mechanistic insight into AgNP-membrane interactions, we investigate these phenomena in minimal model systems using a wide-range of biophysical approaches applied to lipid vesicles. We find a strong dependence on the medium composition, where colloidally stable AgNPs in a glucose buffer have negligible effect on the membrane. However, at a physiological salt concentrations, the AgNPs start to weakly aggregate and sporadic but significant membrane perturbation events are observed. Under these latter conditions, transient poration and structural remodelling of some vesicle membranes is observed. We observe that the fluidity of giant vesicle membranes universally decreases by an average of 16% across all vesicles. However, we observe a small population of vesicles display a significant change in mechanical properties with lower bending rigidity and higher membrane tension. Therefore we argue that the isolated occurrences of membrane perturbation by AgNPs are due to low probability mechanosensing events of AgNP aggregation at the membrane.GRAPHICAL ABSTRACT","AbstractEpidemics of Coronavirus Disease 2019 (COVID-19) now have more than 100,000 confirmed cases worldwide. Diagnosis of COVID-19 is currently performed by RT-qPCR methods, but the capacity of RT-qPCR methods is limited by its requirement of high-level facilities and instruments. Here, we developed and evaluated RT-LAMP assays to detect genomic RNA of SARS-CoV-2, the causative virus of COVID-19. RT-LAMP assays in this study can detect as low as 100 copies of SARS-CoV-2 RNA. Cross-reactivity of RT-LAMP assays to other human Coronaviruses was not observed. We also adapted a colorimetric detection method for our RT-LAMP assay so that the tests potentially performed in higher throughput.","AbstractB-cell lymphoma 2 (Bcl-2) has a dual function, acting both as an oncogene and an anti-tumor gene. It is well known that Bcl-2 exerts its tumor promoting function through the mitochondrial pathway. However, the mechanism by which Bcl-2 suppresses tumor formation is not well understood. We have previously shown that Bcl-2 inhibits cell cycle progression from the G0/G1 to the S phase after serum starvation, and that quiescent Bcl-2 expressing cells maintained a significant lower level of mitochondrial reactive oxygen species (ROS) than the control cells. Based on the fact that ROS mediate cell cycle progression, and are controlled by peroxisome proliferator-activated receptor-\u03b3 co-activator 1\u03b1 (PGC-1\u03b1), a key molecule induced by prolonged starvation and involved in mitochondrial metabolism, we hypothesized that PGC-1\u03b1 might be related with the cell cycle function of Bcl-2. Here, we showed that PGC-1\u03b1 was upregulated upon Bcl-2 overexpression and downregulated following Bcl-2 knockdown during serum starvation. Knockdown of PGC-1\u03b1 activated Bcl-2 expression. Taken together, our results suggest that after serum depletion, PGC-la might coordinate with Bcl-2 to reduce ROS, which in turn delay cell cycle progression.Summary statementPGC-1\u03b1 coordinate with Bcl-2 delay cell cycle progression to reduce ROS after serum depletion in human glioma U251 cells.","AbstractWe hypothesized that postnatal development of the vagina is impacted by early nutritional environment. Our objective was to determine if lipid profiles of vaginal swabs were different between gilts suckled by sow or fed milk replacer the first 48 h postpartum, with and without a lard-based fat supplement. Gilts (&gt;1.3 kg) were selected at birth across 8 litters and assigned to treatments: colostrum suckled (S, n=8); S plus fat supplement (SF, n=5); bottle-fed milk replacer (B, n=8); or B plus fat supplement (BF, n=7). At 48 h postnatal, vaginal swabs were taken with a cytology brush, immersed in ultrapure water to burst cells, and lipids extracted for analysis using multiple reaction monitoring (MRM)-profiling. Lipids extracted from serum collected at 48 h from gilts and milk collected from sows at 24 h were also analyzed with MRM-profiling. Receiver operating characteristic curve analysis found 18 lipids highly distinguished [area-under-the-curve (AUC) &gt; 0.9] between S and B gilts, including phosphatidylethanolamine with 34 carbon and four unsaturations in the fatty acyl residues [PE(34:4)]. Twelve lipids from vaginal swabs highly correlated (r&gt; 0.6;p&lt; 0.01) with nutrition source. Lipids more abundant in milk replacer drove association. For example, mean intensity of PE (34:4) was 149-fold higher in milk replacer than colostrum, with 1.6- and 2.12-fold higher levels in serum and vaginal swab samples (p&lt; 0.001), respectively, of B versus S gilts. Findings support that vaginal swabs can be used to noninvasively study effects of perinatal nutrition on tissue composition.Summary sentenceVaginal swab lipidome profiles at 48 h reflect the fat composition of neonatal diet during first two days postnatal.","AbstractRecent outbreaks of African swine fever virus (ASFV) in China severely influenced the swine industry of the country. Currently, there is no effective vaccine or drugs against ASFVs. How to effectively control the virus is challenging. In this study, we have analyzed all the publicly available ASFV genomes and demonstrated that there was a large genetic diversity of ASFV genomes. Interestingly, the genetic diversity was mainly caused by extensive genomic insertions and/or deletions (indels) instead of the point mutations. The genomic diversity of the virus resulted in proteome diversity. Over 250 types of proteins were inferred from the ASFV genomes, among which only 144 were observed in all analyzed viruses. Further analyses showed that the homologous recombination may contribute much to the indels, as supported by significant associations between the occurrence of extensive recombination events and the indels in the ASFV genomes. Repeated elements of dozens of nucleotides in length were observed to widely distribute and cluster in the adjacent positions of ASFV genomes, which may facilitate the occurrence of homologous recombination. Moreover, two enzymes, which were possibly related to the homologous recombination, i.e., a Lambda-like exonuclease with a YqaJ-like viral recombinase domain, and a DNA topoisomerase II, were found to be conservative in all the analyzed ASFVs. This work highlighted the importance of the homologous recombination in the evolution of the ASFVs, and helped with the strategy development of the prevention and control of the virus.","AbstractThe novel coronavirus SARS-CoV-2 (2019-nCoV) is a member of the family coronaviridae and contains a single-stranded RNA genome with positive-polarity. To reveal the evolution mechanism of SARS-CoV-2 genome, we performed comprehensive genomic analysis with newly sequenced SARS-CoV-2 strains and 20 closely related coronavirus strains. Among 98 nucleotide mutations at 93 sites of the genome among different SARS-CoV-2 strains, 58 of them caused amino acid change, indicating a result of neutral evolution. However, the ratio of nucleotide substitutions to amino acid substitutions of spike gene (9.07) between SARS-CoV-2 WIV04 and Bat-SARSr-CoV RaTG13 was extensively higher than those from comparisons between other coronaviruses (range 1.29 - 4.81). The elevated synonymous mutations between SARS-CoV-2 and RaTG13, suggesting they underwent stronger purifying selection. Moreover, their nucleotide substitutions are enriched with T:C transition, which is consistent with the mutation signature caused by deactivity of RNA 3\u2019-to-5\u2019 exoribonuclease (ExoN). The codon usage was similar between SARS-CoV-2 and other strains in beta-coronavirus lineage B, suggesting it had small impact on the mutation pattern. In comparison of SARS-CoV-2 WIV04 with Bat-SARSr-CoV RaTG13, the ratios of non-synonymous to synonymous substitution rates (dN/dS) was the lowest among all performed comparisons, reconfirming the evolution of SARS-CoV-2 under stringent selective pressure. Moreover, some sites of spike protein might be subjected to positive selection. Therefore, our results will help understanding the evolutionary mechanisms contribute to viral pathogenicity and its adaptation with hosts.","AbstractAs the predominant etiological agent of the common cold, human rhinovirus (HRV) is the leading cause of human infectious disease. Early studies showed monovalent formalin-inactivated HRV vaccine can be protective, and virus-neutralizing antibodies (nAb) correlated with protection. However, co-circulation of many HRV types discouraged further vaccine efforts. We approached this problem straightforwardly. We tested the hypothesis that increasing virus input titers in polyvalent inactivated HRV vaccine will result in broad nAb responses. Here, we show that serum nAb against many rhinovirus types can be induced by polyvalent, inactivated HRVs plus alhydrogel (alum) adjuvant. Using formulations up to 25-valent in mice and 50-valent in rhesus macaques, HRV vaccine immunogenicity was related to sufficient quantity of input antigens, and valency was not a major factor for potency or breadth of the response. We for the first time generated a vaccine capable of inducing nAb responses to numerous and diverse HRV types.","AbstractBackgroundLassa fever (LF) is increasingly recognized as an important rodent-borne viral hemorrhagic fever presenting a severe public health threat to sub-Saharan West Africa. In 2018, LF caused an unprecedented outbreak in Nigeria, and the situation was worse in 2019. This work aims to study the epidemiological features of outbreaks in different Nigerian regions and quantify the association between reproduction number (R) and local rainfall by using modeling analysis.MethodsWe quantify the infectivity of LF by the reproduction numbers estimated from four different growth models: the Richards, three-parameter logistic, Gompertz, and Weibull growth models. LF surveillance data are used to fit the growth models and estimate theRs and epidemic turning points (\u03c4) in different regions at different time periods. Cochran\u2019s Q test is further applied to test the spatial heterogeneity of the LF epidemics. A linear random-effect regression model is adopted to quantify the association betweenRand local rainfall with various lag terms.FindingsOur estimatedRs for 2017-18 (1.33 with 95% CI: [1.29, 1.37]) and 2018-19 (1.29 with 95% CI: [1.27, 1.32]) are significantly higher than those for 2016-17 (1.23 with 95% CI: [1.22, 1.24]). We report spatial heterogeneity in theRs for outbreaks in different Nigerian regions. For the association between rainfall andR, we find that a one unit (mm) increase in average rainfall over the past 7 months could cause a 0.62% (95% CI: [0.20%, 1.05%]) rise inR.ConclusionThere is significant spatial heterogeneity in the LF epidemics in different Nigerian regions. We report clear evidence of rainfall impacts on LF outbreaks in Nigeria and quantify the impact.","The novel coronavirus (2019-nCoV) spike protein is a smart molecular machine that instigates the entry of coronavirus to the host cell causing the COVID-19 pandemic. In this study, a structural-topology based model Hamiltonian of C3 symmetric trimeric spike is developed to explore its complete conformational energy landscape using molecular dynamic simulations. The study finds 2019-nCoV to adopt a unique strategy by undertaking a dynamic conformational asymmetry induced by a few unique inter-chain interactions. This results in two prevalent asymmetric structures of spike where one or two spike heads lifted up undergoing a dynamic transition likely to enhance rapid recognition of the host-cell receptor turning on its high-infectivity. The crucial interactions identified in this study are anticipated to potentially affect the efficacy of therapeutic targets.","ABSTRACTIL-1\u03b2 has emerged as a key mediator of the cytokine storm linked to high morbidity and mortality from COVID-19 and blockade of the IL-1 receptor (IL-1R) with Anakinra has entered clinical trials in COVID-19 subjects. Yet, knowledge of the specific immune cell subsets targeted by IL-1\u03b2 and IL-1\u03b2-induced signaling pathways in humans is limited. Utilizing mass cytometry (CyTOF) of human peripheral blood mononuclear cells, we identified effector memory CD4 T cells and CD4\u2212CD8low/-CD161+ T cells as the circulating immune subtypes with the greatest expression of p-NF-\u03baB in response to IL-1\u03b2 stimulation. Notably, CCR6 distinctly identified T cells most responsive to IL-1\u03b2. Other subsets including CD11c myeloid dendritic cells (mDCs), classical monocytes (CM), two subsets of natural killer cells (CD16\u2212CD56brightCD161\u2212 and CD16\u2212CD56dimCD161+) and a population of lineage\u2212(Lin-) cells expressing CD161 and CD25 also showed IL-1\u03b2-induced expression of p-NF-kB. The IL-1R antagonist, Anakinra significantly inhibited IL-1\u03b2-induced p-NF-kB in the CCR6+ T cells and CD11c mDCs with a trending inhibition in CD14 monocytes and Lin\u2212CD161+CD25+ cells. IL-1\u03b2 also induced a rapid but much less robust increase in p-p38 expression as compared to p-NF-kB in the majority of these same immune cell subsets. Prolonged IL-1\u03b2 stimulation greatly increased p-STAT3 and to a much lesser extent p-STAT1 and p-STAT5 in T cell subsets, monocytes, DCs and the Lin\u2212CD161+CD25+ cells suggesting IL-1\u03b2-induced production of downstream STAT-activating cytokines, consistent with its role in cytokine storm. Interindividual heterogeneity and inhibition of this activation by Anakinra raises the intriguing possibility that assays to measure IL-1\u03b2-induced p-NF-kB in CCR6+ T cell subtypes could identify those at higher risk of cytokine storm and those most likely to benefit from Anakinra therapy.","AbstractThe severity and outcome of respiratory viral infections is partially determined by the cellular response mounted by infected lung epithelial cells. Disease prevention and treatment is dependent on our understanding of the shared and unique responses elicited by diverse viruses, yet few studies compare host responses to different viruses while controlling other experimental parameters. We compared changes in gene expression of murine lung epithelial cells infected individually by three respiratory viruses causing mild (rhinovirus, RV1B), moderate (coronavirus, MHV-1), and severe (influenza A virus, PR8) disease in mice. RV1B infection caused numerous gene expression changes, but the differential effect peaked at 12 hours post-infection. PR8 altered an intermediate number of genes whose expression continued to change through 24 hours. MHV-1 had comparatively few effects on host gene expression. The viruses elicited highly overlapping responses in antiviral genes, though MHV-1 induced a lower type I interferon response than the other two viruses. Signature genes were identified for each virus and included host defense genes for PR8, tissue remodeling genes for RV1B, and transcription factors for MHV-1. Our comparative approach identified universal and specific transcriptional signatures of virus infection that can be used to discover mechanisms of pathogenesis in the respiratory tract.","There appears to be large regional variations for susceptibility, severity and mortality for Covid-19 infections. We set out to examine genetic differences in the human angiotensin-converting enzyme 2 (hACE2) gene, as its receptor serves as a cellular entry for SARS-CoV-2. By comparing 56,885 Non-Finnish European and 9,197 East Asians (including 1,909 Koreans) four missense mutations were noted in the hACE2 gene. Molecular dynamic demonstrated that two of these variants (K26R and I468V) may affect binding characteristics between S protein of the virus and hACE2 receptor. We also examined hACE2 gene expression in eight global populations from the HapMap3 and noted marginal differences in expression for some populations as compared to the Chinese population. However, for both of our studies, the magnitude of the difference was small and the significance is not clear in the absence of further in vitro and functional studies.","Pooling of samples can increase lab capacity when using Polymerase chain reaction (PCR) to detect infections such as COVID-19. However, pool testing is typically performed via an adaptive testing strategy which requires a feedback loop in the lab and at least two PCR runs to confirm positive results. This can cost precious time. We discuss a non-adaptive testing method where each sample is distributed in a prescribed manner over several pools, and which yields reliable results after one round of testing. More precisely, assuming knowledge about the overall infection incidence rate, we calculate explicit error bounds on the number of false positives which scale very favourably with pool size and sample multiplicity. This allows for hugely streamlined PCR testing and cuts in detection times for a large-scale testing scenario. A viable consequence of this method could be real-time screening of entire communities, frontline healthcare workers and international flight passengers, for example, using the PCR machines currently in operation.","ABSTRACTHuman astroviruses are small nonenveloped viruses with positive-sense single-stranded RNA genomes that contain three main open reading frames: ORF1a, ORF1b and ORF2. Astroviruses cause acute gastroenteritis in children worldwide and have been associated with encephalitis and meningitis in immunocompromised individuals. Through comparative genomic analysis of &gt;400 astrovirus sequences, we identified a conserved \u201cORFX\u201d overlapping the capsid-encoding ORF2 in genogroup I, III and IV astroviruses. ORFX appears to be subject to purifying selection, consistent with it encoding a functional protein product, termed XP. Using ribosome profiling of cells infected with human astrovirus 1, we confirm initiation at the ORFX AUG. XP-knockout astroviruses are strongly attenuated and after passaging can partly restore viral titer via pseudo-reversions, thus demonstrating that XP plays an important role in virus growth. To further investigate XP, we developed an astrovirus replicon system. We demonstrate that XP has only minor effects on RNA replication and structural protein production. Instead, XP associates with the plasma membrane with an extracellular N-terminus topology and promotes efficient virus release. Using two different assays, we show that expression of human or related astrovirus XPs leads to cell permeabilization, suggesting a viroporin-like activity. The discovery of XP advances our knowledge of these important human viruses and opens a new direction of research into astrovirus replication and pathogenesis.","AbstractThe recent outbreak by SARS-CoV-2 has generated a chaos in global health and economy and claimed/infected a large number of lives. Closely resembling with SARS CoV, the present strain has manifested exceptionally higher degree of spreadability, virulence and stability possibly due to some unidentified mutations. The viral spike glycoprotein is very likely to interact with host Angiotensin-Converting Enzyme 2 (ACE2) and transmits its genetic materials and hijacks host machinery with extreme fidelity for self propagation. Few attempts have been made to develop a suitable vaccine or ACE2 blocker or virus-receptor inhibitor within this short period of time. Here, attempt was taken to develop some therapeutic and vaccination strategies with a comparison of spike glycoproteins among SARS-CoV, MERS-CoV and the SARS-CoV-2. We verified their structure quality (SWISS-MODEL, Phyre2, Pymol) topology (ProFunc), motifs (MEME Suite, GLAM2Scan), gene ontology based conserved domain (InterPro database) and screened several epitopes (SVMTrip) of SARS CoV-2 based on their energetics, IC50 and antigenicity with regard to their possible glycosylation and MHC/paratopic binding (Vaxigen v2.0, HawkDock, ZDOCK Server) effects. We screened here few pairs of spike protein epitopic regions and selected their energetic, IC50, MHC II reactivity and found some of those to be very good target for vaccination. A possible role of glycosylation on epitopic region showed profound effects on epitopic recognition. The present work might be helpful for the urgent development of a suitable vaccination regimen against SARS CoV-2.","AbstractThe ongoing COVID-19 pandemic is caused by the novel coronavirus SARS-CoV-2. Age, smoking, obesity, and chronic diseases such as cardiovascular disease and diabetes have been described as risk factors for severe complications and mortality in COVID-19. Obesity and diabetes are usually associated with dysregulated lipid synthesis and clearance which can initiate or aggravate pulmonary inflammation and injury. It has been shown that for viral entry into the host cell, SARS-CoV-2 utilizes the angiotensin converting enzyme 2 (ACE2) receptors present on the cells. We aimed to characterize how SARS-CoV-2 dysregulates lipid metabolism pathways in the host and the effect of dysregulated lipogenesis on the regulation of ACE2, specifically in obesity. In our study, through the re-analysis of publicly available transcriptomic data, we first found that lung epithelial cells infected with SARS-CoV-2 showed upregulation of genes associated with lipid metabolism, including the SOC3 gene which is involved in regulation of inflammation and inhibition of leptin signaling. This is of interest as viruses may hijack host lipid metabolism to allow completion of their viral replication cycles. Furthermore, a mouse model of diet-induced obesity showed a significant increase in Ace2 expression in the lungs which negatively correlated with the expression of genes that code for sterol response element binding proteins 1 and 2 (SREBP). Suppression of Srebp1 showed a significant increase in Ace2 expression in the lung. Together our results suggest that the dysregulated lipogenesis and the subsequently high ACE2 expression in obese patients might be the mechanism underlying the increased risk for severe complications in those patients when infected by SARS-CoV-2.","AbstractSevere acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is the cause of coronavirus disease (COVID-19) that has resulted in a global pandemic. It is a highly contagious positive strand RNA virus and its clinical presentation includes severe to critical respiratory disease that appears to be fatal in \u223c3-5% of the cases. The viral spike (S) coat protein engages the human angiotensin-converting enzyme2 (ACE2) cell surface protein to invade the host cell. The SARS-CoV-2 S-protein has acquired mutations that increase its affinity to human ACE2 by \u223c10-15-fold compared to SARS-CoV S-protein, making it highly infectious. In this study, we assessed if ACE2 polymorphisms might alter host susceptibility to SARS-CoV-2 by affecting the ACE2 S-protein interaction. Our comprehensive analysis of several large genomic datasets that included over 290,000 samples representing &gt;400 population groups identified multiple ACE2 protein-altering variants, some of which mapped to the S-protein-interacting ACE2 surface. Using recently reported structural data and a recent S-protein-interacting synthetic mutant map of ACE2, we have identified natural ACE2 variants that are predicted to alter the virus-host interaction and thereby potentially alter host susceptibility. In particular, human ACE2 variants S19P, I21V, E23K, K26R, T27A, N64K, T92I, Q102P and H378R are predicted to increase susceptibility. The T92I variant, part of a consensus NxS/T N-glycosylation motif, confirmed the role of N90 glycosylation in immunity from non-human CoVs. Other ACE2 variants K31R, N33I, H34R, E35K, E37K, D38V, Y50F, N51S, M62V, K68E, F72V, Y83H, G326E, G352V, D355N, Q388L and D509Y are putative protective variants predicted to show decreased binding to SARS-CoV-2 S-protein. Overall, ACE2 variants are rare, consistent with the lack of selection pressure given the recent history of SARS-CoV epidemics, however, are likely to play an important role in altering susceptibility to CoVs.","Isothermal nucleic acid amplification tests (iNAT), such as loop-mediated isothermal amplification (LAMP), are good alternatives to polymerase chain reaction (PCR)-based amplification assays, especially for point-of-care and low resource use, in part because they can be carried out with relatively simple instrumentation. However, iNATs can generate spurious amplicons, especially in the absence of target sequences, resulting in false positive results. This is especially true if signals are based on non-sequence-specific probes, such as intercalating dyes or pH changes. In addition, pathogens often prove to be moving, evolving targets, and can accumulate mutations that will lead to inefficient primer binding and thus false negative results. Internally redundant assays targeting different regions of the target sequence can help to reduce such false negatives. Here we describe rapid conversion of three previously described SARS-CoV-2 LAMP assays that relied on non-sequence-specific readout into assays that can be visually read using sequence-specific fluorogenic oligonucleotide strand exchange (OSD) probes. We evaluate one-pot operation of both individual and multiplex LAMP-OSD assays and demonstrate detection of SARS-CoV-2 virions in crude human saliva.","Coronavirus disease 2019 (COVID-19) is a viral pandemic caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). COVID-19 is predominantly defined by respiratory symptoms, but cardiac complications including arrhythmias, heart failure, and viral myocarditis are also prevalent. Although the systemic ischemic and inflammatory responses caused by COVID-19 can detrimentally affect cardiac function, the direct impact of SARS-CoV-2 infection on human cardiomyocytes is not well-understood. We used human induced pluripotent stem cell-derived cardiomyocytes (hiPSC-CMs) as a model system to examine the mechanisms of cardiomyocyte-specific infection by SARS-CoV-2. Microscopy and immunofluorescence demonstrated that SARS-CoV-2 can enter and replicate within hiPSC-CMs, localizing at perinuclear locations within the cytoplasm. Viral cytopathic effect induced hiPSC-CM apoptosis and cessation of beating after 72 hours of infection. These studies show that SARS-CoV-2 can infect hiPSC-CMs in vitro, establishing a model for elucidating the mechanisms of infection and potentially a cardiac-specific antiviral drug screening platform.","AbstractViruses are known to have some of the highest and most diverse mutation rates found in any biological replicator, topped by single-stranded (ss) RNA viruses, while double-stranded (ds) DNA viruses have rates approaching those of bacteria. As mutation rates are tightly and negatively correlated with genome size, selection is a clear driver of viral evolution. However, the role of intragenomic interactions as drivers of viral evolution is less well documented. To understand how these two processes affect viral evolution, we systematically surveyed ssRNA, ssDNA, dsRNA, and dsDNA viruses, to find which virus type and which functions show evidence for episodic diversifying selection and correlated evolution. We show that while evidence for selection is mostly found in single stranded viruses, and correlated evolution is more prevalent in DNA viruses, the genes that are affected by both processes are involved in key aspects of their life cycle, favoring viral stability over proliferation. We further show that both evolutionary processes are intimately linked at the amino acid level, which suggests that selection alone does not explain the whole evolutionary \u2014and epidemiological\u2014 potential of viruses.","AbstractMetagenomic sequencing has the potential to transform microbial detection and characterization, but new tools are needed to improve its sensitivity. We developed CATCH (Compact Aggregation of Targets for Comprehensive Hybridization), a computational method to enhance nucleic acid capture for enrichment of diverse microbial taxa. CATCH designs compact probe sets that achieve full coverage of known sequence diversity and that scale well with this diversity. To illustrate applications of CATCH, we focused on capturing viral genomes. We designed, synthesized, and validated multiple probe sets, including one that targets whole genomes of the 356 viral species known to infect humans. Capture with these probe sets enriched unique viral content on average 18\u00d7 and allowed us to assemble genomes that we could not otherwise recover, while accurately preserving within-sample diversity. We used this approach to recover genomes from the 2018 Lassa fever outbreak in Nigeria and to improve detection of viral infections in samples with unknown content. Together, this work demonstrates a path toward more sensitive, cost-effective metagenomic sequencing.","AbstractThe effective reproduction number, Rt, is an important prognostic for infectious disease epidemics. Significant changes in Rt can forewarn about new transmissions or predict the efficacy of interventions. The renewal model infers Rt from incidence data and has been applied to Ebola virus disease and pandemic influenza outbreaks, among others. This model estimates Rt using a sliding window of length k. While this facilitates real-time detection of statistically significant Rt fluctuations, inference is highly k -sensitive. Models with too large or small k might ignore meaningful changes or over-interpret noise-induced ones. No principled k -selection scheme exists. We develop a practical yet rigorous scheme using the accumulated prediction error (APE) metric from information theory. We derive exact incidence prediction distributions and integrate these within an APE framework to identify the k best supported by available data. We find that this k optimises short-term prediction accuracy and expose how common, heuristic k -choices, which seem sensible, could be misleading.","AbstractCOVID-19 has become a global pandemic that threatens millions of people worldwide. There is an urgent call for developing effective drugs against the virus (SARS-CoV-2) causing this disease. The main protease of SARS-CoV-2, 3C-like protease (3CLpro), is highly conserved across coronaviruses and is essential for the maturation process of viral polyprotein. Scutellariae radix (Huangqin in Chinese), the root of Scutellaria baicalensis has been widely used in traditional Chinese medicine to treat viral infection related symptoms. The extracts of S. baicalensis have exhibited broad spectrum antiviral activities. We studied the anti-SARS-CoV-2 activity of S. baicalensis and its ingredient compounds. We found that the ethanol extract of S. baicalensis inhibits SARS-CoV-2 3CLpro activity in vitro and the replication of SARS-CoV-2 in Vero cells with an EC50 of 0.74 \u03bcg/ml. Among the major components of S. baicalensis, baicalein strongly inhibits SARS-CoV-2 3CLpro activity with an IC50 of 0.39 \u03bcM. We further identified four baicalein analogue compounds from other herbs that inhibit SARS-CoV-2 3CLpro activity at microM concentration. Our study demonstrates that the extract of S. baicalensis has effective anti-SARS-CoV-2 activity and baicalein and analogue compounds are strong SARS-CoV-2 3CLpro inhibitors.","AbstractWhether a pathogen entering a new host species results in a single infection or in onward transmission, and potentially an outbreak, depends upon the progression of infection in the index case. Although index infections are rarely observable in nature, experimental inoculations of pathogens into novel host species have a long history in biomedical research. This provides a rich and largely unexploited data source for meta-analyses to identify the host and pathogen determinants of variability in infection outcomes. Here, we analysed the progressions of 514 experimental cross-species inoculations of rabies virus, a widespread zoonotic pathogen which in nature exhibits both dead end infections and varying levels of sustained transmission in novel hosts. Inoculations originating from bats rather than carnivores, and from warmer to cooler-bodied species caused infections with shorter incubation periods that were associated with diminished virus excretion. Inoculations between distantly related hosts tended to result in shorter clinical disease periods, which will also impede transmission. All effects were modulated by infection dose and together suggest that increased virulence as host species become more dissimilar is the limiting factor preventing onward transmission. These results explain observed constraints on rabies virus host shifts, allow us to evaluate the risk of novel reservoirs establishing, and give mechanistic insights into why host shifts are less likely between genetically distant species. More generally, our study highlights meta-analyses of experimental infections as a tractable approach to quantify the complex interactions between virus, reservoir, and novel host that shape the outcome of cross-species transmission.Significance statementEmerging disease epidemics often result from a pathogen establishing transmission in a novel host species. However, most cross-species transmissions fail to establish in the newly infected species for reasons that remain poorly understood. Examining cross-species inoculations involving rabies, a widespread viral zoonosis, we show that mismatches in virulence, which are predictable from host and viral factors, make sustained transmission in the novel host less likely. In particular, disease progression was accelerated and virus excretion decreased when the reservoir and novel host were physiologically or genetically more dissimilar. These mechanistic insights help to explain and predict host shift events and highlight meta-analyses of existing experimental inoculation data as a powerful and generalisable approach for understanding the dynamics of index infections in novel species.","AbstractThe host antiviral response involves the induction of interferons and proinflammatory cytokines, but also the activation of cell death pathways, including apoptosis, to limit viral replication and spreading. This host defense is strictly regulated to eliminate the infection while limiting tissue damage that is associated with virus pathogenesis. Post-translational modifications, most notably phosphorylation, are key regulators of the antiviral defense implying an important role of protein phosphatases. Here, we investigated the role of the dual-specificity phosphatase 1 (DUSP1) in the host defense against human respiratory syncytial virus (RSV), a pathogenic virus of the Pneumoviridae family, and Sendai virus (SeV), a model virus being developed as a vector for anti-RSV vaccine. We found that DUSP1 is upregulated before being subjected to proteasomal degradation. DUSP1 does not inhibit the antiviral response, but negatively regulates virus-induced JNK/p38 MAPK phosphorylation. Interaction with the JNK-interacting protein 1 scaffold protein prevents dephosphorylation of JNK by DUSP1, likely explaining that AP-1 activation and downstream cytokine production are protected from DUSP1 inhibition. Importantly, DUSP1 promotes SeV-induced apoptosis and suppresses cell migration in RSV-infected cells. Collectively, our data unveil a previously unrecognized selective role of DUSP1 in the regulation of tissue damage and repair during infections by RSV and SeV.","Rapidly spreading viral pandemics, such as those caused by influenza and SAR-CoV-2 (COVID19), require rapid action and the fastest way to combat this challenge is by repurposing existing drugs as anti-viral therapeutics. Here we first show that human organ-on-a-chip (Organ Chip) microfluidic culture devices lined by a highly differentiated, primary, human lung airway epithelium cultured under an air-liquid interface and fed by continuous medium flow can be used to model virus entry, replication, strain-dependent virulence, host cytokine production, and recruitment of circulating immune cells in response to infection by influenza, as well as effects of existing and novel therapeutics. These Airway Chips, which contain human lung epithelial cells that express high levels of ACE2 and TMPRSS2, were then used to assess the inhibitory activities of 7 clinically approved drugs (chloroquine, arbidol, toremifene, clomiphene, amodiaquine, verapamil, and amiodarone) that we found inhibit infection by viral pseudoparticles expressing SARS-CoV-2 spike protein in human Huh-7 cells, and others recently showed suppress infection by native SARS-CoV-2 in Vero cells. However, when these drugs were administered under flow at the maximal concentration in blood reported in clinical studies in human Airway Chips, only two of these drugs - amodiaquine and toremifene - significantly inhibited entry of the pseudotyped SARS-CoV-2 virus. This work suggests that human Organ Chip technology may be used in conjunction with existing rapid cell-based screening assays to study human disease pathogenesis and expedite drug repurposing in biothreat crises caused by pandemic viruses.","AbstractOne striking feature of the large KRAB-containing zinc finger protein (KZFP) family is its rapid expansion and divergence since its formation about 400 million years ago. However, the evolutionary characteristics of KRAB domains, C2H2 zinc fingers and the full protein of KZFPs have not been fully analyzed. As for the drivers of the rapid evolution, it\u2019s partly due to their coevolution with transposable elements (TEs). But their diverse functions besides inhibiting TEs suggest other reasons exist. Here we address these two issues by the systematic analysis of the divergence time and diversification pattern of KZFPs at three aspects and the functional analysis of the potential target genes besides TEs. We found that old-zinc-finger-containing KZFPs tend to have varied and disordered KRAB domains, indicating there are two ways of the evolution of KZFPs, including the variation of KRAB domains and the diversification of zinc fingers. Among them, the divergence of zinc fingers mainly contributes to the rapid evolution of KZFPs. Thus, we mainly focused on the functional requirements of the evolution of zinc fingers. Different from the classical regulation pattern of this family, we found and experimentally confirmed that KZFPs tend to bind to non-TE regions and can positively regulate target genes. Although most young genes tend to be with a low expression level, young-zinc-finger-containing KZFPs tend to be highly expressed in early embryonic development or early mesoderm differentiation, indicating their particular evolutionarily novel functional roles in these two processes. We further validated a young KZFP, ZNF611, can bind to non-TE region of STK38 gene and positively regulates its expression in ESCs. The emergence of new sequence in STK38 promoter may drive the evolution of zinc fingers in ZNF611. Finally, we proposed a \u2018two-way evolution model\u2019 of KZFP family.","AbstractDengue virus (DENV) represents a public health and economic burden in affected countries. The availability of genomic data is key to understand viral evolution and dynamics, supporting improved control strategies. Currently, the use of High Throughput Sequencing (HTS) technologies, which can be applied both directly to patient samples (shotgun metagenomics) and to PCR amplified viral sequences (targeted metagenomics), is the most informative approach to monitor the viral dissemination and genetic diversity.Despite many advantages, these technologies require bioinformatics expertise and appropriate infrastructure for the analysis and interpretation of the resulting data. In addition, the many software solutions available can hamper reproducibility and comparison of results.Here we present DEN-IM, a one-stop, user-friendly, containerised and reproducible workflow for the analysis of DENV sequencing data, both from shotgun and targeted metagenomics approaches. It is able to infer DENV coding sequence (CDS), identify serotype and genotype, and generate a phylogenetic tree. It can easily be run on any UNIX-like system, from local machines to high-performance computing clusters, performing a comprehensive analysis without the requirement of extensive bioinformatics expertise.Using DEN-IM, we successfully analysed two DENV datasets. The first comprised 25 shotgun metagenomic sequencing samples of varying serotype and genotype, including a spiked sample containing the existing four serotypes. The second dataset consisted of 106 targeted metagenomics samples of DENV 3 genotype III where DEN-IM allowed detection of the intra-genotype diversity.The DEN-IM workflow, parameters and execution configuration files, and documentation are freely available at https://github.com/B-UMMI/DEN-IM.","Enterovirus G (EV-G) belongs to the family of Picornaviridae. Two types of recombinant porcine EV-Gs carrying papain-like cysteine protease (PLCP) gene of porcine torovirus, a virus in Coronaviridae, are reported. Type 1 recombinant EV-Gs are detected in pig feces in Japan, USA, and Belgium and carry the PLPC gene at the junction site of 2C/3A genes, while PLPC gene replaces the viral structural genes in type 2 recombinant EV-G detected in pig feces in a Chinese farm. We identified a novel type 2 recombinant EV-G carrying the PLCP gene with flanking sequences in place of the viral structural genes in pig feces in Japan. The ~0.3 kb-long upstream flanking sequence had no sequence homology with any proteins deposited in GenBank, while the downstream ~0.9 kb-long flanking sequence included a domain having high amino acid sequence homology with a baculoviral inhibitor of apoptosis repeat superfamily. The pig feces, where the novel type 2 recombinant EV-G was detected, also carried type 1 recombinant EV-G. Although the phylogenetic analysis suggested that these two recombinant EV-Gs have independently evolved, type 1 recombinant EV-G might have served as a helper virus by providing viral structural proteins for dissemination of the type 2 recombinant EV-G.","AbstractThe role of microRNAs (miRNAs) in cellular processes captured the attention of many researchers, since their dysregulation is shown to affect the cancer disease landscape by sustaining proliferative signaling, evading program cell death, and inhibiting growth suppressors. Thus, miRNAs have been considered important diagnostic and prognostic biomarkers for several types of tumors. Machine learning algorithms have proven to be able to exploit the information contained in thousands of miRNAs to accurately predict and classify cancer types. Nevertheless, extracting the most relevant miRNA expressions is fundamental to allow human experts to validate and make sense of the results obtained by automatic algorithms. We propose a novel feature selection approach, able to identify the most important miRNAs for tumor classification, based on consensus on feature relevance from high-accuracy classifiers of different typologies. The proposed methodology is tested on a real-world dataset featuring 8,129 patients, 29 different types of tumors, and 1,046 miRNAs per patient, taken from The Cancer Genome Atlas (TCGA) database. A new miRNA signature is suggested, containing the 100 most important oncogenic miRNAs identified by the presented approach. Such a signature is proved to be sufficient to identify all 29 types of cancer considered in the study, with results nearly identical to those obtained using all 1,046 features in the original dataset. Subsequently, a meta-analysis of the medical literature is performed to find references to the most important biomarkers extracted by the methodology. Besides known oncomarkers, 15 new miRNAs previously not ranked as important biomarkers for diagnosis and prognosis in cancer pathologies are uncovered. Such miRNAs, considered relevant by the machine learning algorithms, but still relatively unexplored by specialized literature, could provide further insights in the biology of cancer.","AbstractObjectiveHealth system resilience and resilience of a country include the capacity of health personnel, institutions, and populations to prepare for and effectively respond to crises. This study investigates the knowledge and attitudes of the public concerning Ebola Virus Disease in Trinidad and Tobago.Design and MethodsA cross sectional study whereby respondents (n = 920) were sampled from public places. Data were collected via interviewer administered questionnaires. Data were analysed using SPSS version 23.ResultsThe response rate was 67.6 % (622/920). The main age category of responders was the 20 to 30 year age category (40.5%); responders were mostly female (58.0 %). Regarding knowledge, there were significant differences among occupational categories (F = 2.811, df1 = 6, df2 = 571, p-value = 0.011). Tukey\u2019s HSD post hoc test revealed that the mean knowledge scores for professional and sales occupations differed significantly (p-value = 0.003). There was a significant association between being afraid to go for treatment and age (p-value = 0.001). Significant associations were also found between occupational grouping and education attainment with opinion about the preparedness of private medical facilities, likelihood to shun family members with Ebola, being afraid to go for treatment and preference for traditional medicine (p-value &lt;0.05).ConclusionThis study highlights opportunities for community engagement to enhance health system resilience during outbreaks which would maximise national and global health security.","AbstractBACKGROUNDAcute gastroenteritis (AGE) is a leading cause of morbidity and mortality in young children worldwide. Human Bocavirus (HBoV) is an emerging virus globally associated with diarrhea. The aim of this study was to demonstrate the prevalence of HBoV genotypes in children (\u22645 years) from rural communities in South Africa (SA) suffering from AGE.MATERIAL AND METHODA total of 141 fecal samples of children \u22645 years with acute gastroenteritis (AGE) were collected from rural Primary Health Care facilities in the Vhembe district of SA between June 2017 and July 2018. Clinical symptoms and demographic data were also recorded. A total of 102 (72%) were outpatients and 39 (28%) were hospitalized patients. Human Bocavirus (HBoV) genotypes were determined using Real-Time Multiplex PCR. DNA extracts of positive samples were confirmed by conventional PCR targeting the NS1 gene. Co-infection with other enteric viruses were determined in HBoV positive samples using Real-Time PCR.RESULTSHBoV was detected in 8 (5.7%) children with AGE. Children were in the age group between 1-24 months. HBoV1 and HBoV3 genotypes were each detected in 3 (37.5%) stool samples and HBoV2 in 2 (25%) stool samples. Co-infection with other enteric viruses included Rotavirus (37.5%); Adenovirus (37.5%); Norovirus (25%) and Astrovirus (12.5%).CONCLUSIONHBoV infections could be seen as a potential emerging diarrheal pathogen in South Africa. Further studies are required to understand the role of HBoV infections in children and adults with acute gastroenteritis.Author summaryAcute gastroenteritis (AGE) is recognized as a major cause for mortality in children \u22645 years of age in Africa and other developing countries. Viruses known to be involved in AGE includes Rotavirus, Norovirus, Astrovirus and Adenovirus and have been reported globally. Recently the Human Bocavirus (HBoV) have been reported in numerous studies globally as a potential cause of diarrhea. In this study, the prevalence and genetic diversity of human Bocavirus in children with AGE from rural communities in Limpopo, South Africa were investigated. In total, 141 stool samples from children \u2264 5 years with AGE were assessed for the presence of HBoV using Real-Time PCR. HBoV were detected in 8 (5.7%) patients and included 3 positive samples for HBoV1 and HBoV3 respectively and 2 positive for HBoV2. No HBoV4 were detected. Among the 8 positive HBoV samples, co-infection with other enteric viruses were found in 7 (87.5%) samples, while mono infection with HBoV alone was detected in 1 (12.5%) patient. HBoV mixed infection with Rotavirus (3/8; 37.5%); Adenovirus (3/8; 37.5%); Norovirus (2/8; 25%) and Astrovirus (1/8; 12.5%) were observed in this study. This study reported for the first time on the prevalence of human Bocavirus in children with AGE from rural communities in South Africa.","Respiratory failure is the leading cause of COVID-19 death and disproportionately impacts adults more than children. Here, we present a large-scale snATAC-seq dataset (90,980 nuclei) of the human lung, generated in parallel with snRNA-seq (46,500 nuclei), from healthy donors of ~30 weeks, ~3 years and ~30 years of age. Focusing on genes implicated in SARS-CoV-2 cell entry, we observed an increase in the proportion of alveolar epithelial cells expressing ACE2 and TMPRSS2 in adult compared to young lungs. Consistent with expression dynamics, 10 chromatin peaks linked to TMPRSS2 exhibited significantly increased activity with age and harbored IRF and STAT binding sites. Furthermore, we identified 14 common sequence variants in age-increasing peaks with predicted regulatory function, including several associated with respiratory traits and TMPRSS2 expression. Our findings reveal a plausible contributor to why children are more resistant to COVID-19 and provide an epigenomic basis for transferring this resistance to older populations.","AbstractSingle particle cryo-EM is a powerful method for studying proteins and other biological macromolecules. Many of these molecules comprise regions with varying structural properties including disorder, flexibility, and partial occupancy. These traits make computational 3D reconstruction from 2D images challenging. Detergent micelles and lipid nanodiscs, used to keep membrane proteins in solution, are common examples of locally disordered structures that can negatively affect existing iterative refinement algorithms which assume rigidity (or spatial uniformity). We introduce a cross-validation approach to derive non-uniform refinement, an algorithm that automatically regularizes 3D density maps during iterative refinement to account for spatial variability, yielding dramatically improved resolution and 3D map quality. We find that in common iterative refinement methods, regularization using spatially uniform filtering operations can simultaneously over- and under-regularize local regions of a 3D map. In contrast, non-uniform refinement removes noise in disordered regions while retaining signal useful for aligning particle images. Our results include state-of-the-art resolution 3D reconstructions of multiple membrane proteins with molecular weight as low as 90kDa. These results demonstrate that higher resolutions and improved 3D density map quality can be achieved even for small membrane proteins, an important use case for single particle cryo-EM, both in structural biology and drug discovery. Non-uniform refinement is implemented in the cryoSPARC software package and has already been used successfully in several notable structural studies.","AbstractManipulation of natural mosquito populations using the endosymbiotic bacteria Wolbachia is being investigated as a novel strategy to reduce the burden of mosquito-borne viruses. To evaluate the efficacy of these interventions, it will be critical to determine Wolbachia infection frequencies in Aedes aegypti mosquito populations. However, current diagnostic tools are not well-suited to fit this need. Morphological methods cannot identify Wolbachia, immunoassays often suffer from low sensitivity and poor throughput, while PCR and spectroscopy require complex instruments and technical expertise, which restrict their use to centralized laboratories. To address this unmet need, we have used loop-mediated isothermal amplification (LAMP) and oligonucleotide strand displacement (OSD) probes to create a one-pot sample-to-answer nucleic acid diagnostic platform for vector and symbiont surveillance. LAMP-OSD assays can directly amplify target nucleic acids from macerated mosquitoes without requiring nucleic acid purification and yield specific single endpoint yes/no fluorescence signals that are observable to eye or by cellphone camera. We demonstrate cellphone-imaged LAMP-OSD tests for two targets, the Aedes aegypti cytochrome oxidase I (coi) gene and the Wolbachia surface protein (wsp) gene, and show a limit of detection of 4 and 40 target DNA copies, respectively. In a blinded test of 90 field-caught mosquitoes, the coi LAMP-OSD assay demonstrated 98% specificity and 97% sensitivity in identifying Ae. aegypti mosquitoes even after 3 weeks of storage without desiccant at 37 \u00b0C. Similarly, the wsp LAMP-OSD assay readily identified the wAlbB Wolbachia strain in field-collected Aedes albopictus mosquitoes without generating any false positive signals. Modest technology requirements, minimal execution steps, simple binary readout, and robust accuracy make the LAMP-OSD-to-cellphone assay platform well suited for field vector surveillance in austere or resource-limited conditions.Author summaryMosquitoes spread many human pathogens and novel approaches are required to reduce the burden of mosquito-borne disease. One promising approach is transferring Wolbachia into Aedes aegypti mosquitoes where it blocks transmission of arboviruses like dengue, Zika and Yellow fever viruses and spreads through mosquito populations. For effective evaluation of this approach, regular surveillance of Wolbachia infections in Ae. aegypti is required, but current diagnostic tools are not well suited to support these critical surveillance needs. To fill this need we developed a simple, robust and inexpensive assay to identify Ae. aegypti mosquitoes and Wolbachia using our unique one-pot assay platform, LAMP-OSD, which uses loop-mediated isothermal amplification to amplify nucleic acid targets at a single temperature. Unlike other LAMP-based tests, our assays assure accuracy by coupling amplification with novel nucleic acid strand displacement (OSD) probes that hybridize to specific sequences in LAMP amplification products and thereby generate simple yes/no readout of fluorescence readable by human eye and by off-the-shelf cellphones. To facilitate field use, we developed our assays so they are compatible with crushed mosquito homogenate as the template, meaning no nucleic acid extraction is required. In blinded tests using field collected mosquitoes, LAMP-OSD-cellphone tests performed robustly to identify 29 of 30 Ae. aegypti even after 3 weeks of storage at 37 \u00b0C while producing only one false positive out of 60 non-specific mosquitoes. Similarly, our assay could identify Wolbachia in field-caught Aedes albopictus without producing any false positives. Our easy to use and easy to interpret assays should facilitate widespread field mosquito surveillance with minimal instrumentation and high accuracy.","AbstractCoronavirus disease 2019 (COVID-19) outcomes vary from asymptomatic infection to death. This disparity may reflect different airway levels of the SARS-CoV-2 receptor, ACE2, and the spike protein activator, TMPRSS2. Here we explore the role of genetics and co-expression networks in regulating these genes in the airway, through the analysis of nasal airway transcriptome data from 695 children. We identify expression quantitative trait loci (eQTL) for both ACE2 and TMPRSS2, that vary in frequency across world populations. Importantly, we find TMPRSS2 is part of a mucus secretory network, highly upregulated by T2 inflammation through the action of interleukin-13, and that interferon response to respiratory viruses highly upregulates ACE2 expression. Finally, we define airway responses to coronavirus infections in children, finding that these infections upregulate IL6 while also stimulating a more pronounced cytotoxic immune response relative to other respiratory viruses. Our results reveal mechanisms likely influencing SARS-CoV-2 infectivity and COVID-19 clinical outcomes.","AbstractThe incidence of locally acquired dengue infections increased during the last decade in the United States, compelling a sustained research effort on the dengue mosquito vector, Aedes aegypti, and its microbiome, which has been shown to influence virus transmission success. We examined the \u2018metavirome\u2019 of four populations of Ae. aegypti mosquitoes collected in 2016-2017 from Manatee County, Florida. Unexpectedly, we discovered that dengue virus serotype 4 (DENV4) was circulating in these mosquito populations, representing the first documented case of such a phenomenon in the absence of a local DENV4 human case in this county over a two-year period. We confirmed that all of the mosquito populations carried the same DENV4 strain, assembled its full genome, validated infection orthogonally by reverse transcriptase PCR, traced the virus origin, estimated the time period of its introduction to the Caribbean region, as well as explored the viral genetic signatures and mosquito-specific virome associations that potentially mediated DENV4 persistence in mosquitoes. We discuss the significance of prolonged maintenance of these DENV4 infections in Ae. aegypti that occurred in the absence of a DENV4 human index case in Manatee County with respect to the inability of current surveillance paradigms to detect mosquito vector infections prior to a potential local outbreak.ImportanceSince 1999, dengue outbreaks in the continental United States (U.S.) involving local transmission have occurred episodically and only in Florida and Texas. In Florida, these episodes appear to be coincident with increased introductions of dengue virus into the region through human travel and migration from endemic countries. To date, the U.S. public health response to dengue outbreaks is largely reactive, and implementation of comprehensive arbovirus surveillance in advance of predictable transmission seasons, which would enable proactive preventative efforts, remains unsupported. The significance of our finding is that it is the first documented report of non-outbreak DENV4 transmission and maintenance within a local mosquito vector population in the continental U.S.in the absence of a human case during a two-year time period. Our data suggest that molecular surveillance of mosquito populations in high-risk, high tourism areas of the U.S., may allow for proactive, targeted vector control before potential arbovirus outbreaks.","AbstractThe World Health Organization has recently declared the ongoing outbreak of COVID-19, which is caused by a novel coronavirus SARS-CoV-2, as pandemic. There is currently a lack of knowledge in the antibody response elicited from SARS-CoV-2 infection. One major immunological question is concerning the antigenic differences between SARS-CoV-2 and SARS-CoV. We address this question by using plasma from patients infected by SARS-CoV-2 or SARS-CoV, and plasma obtained from infected or immunized mice. Our results show that while cross-reactivity in antibody binding to the spike protein is common, cross-neutralization of the live viruses is rare, indicating the presence of non-neutralizing antibody response to conserved epitopes in the spike. Whether these non-neutralizing antibody responses will lead to antibody-dependent disease enhancement needs to be addressed in the future. Overall, this study not only addresses a fundamental question regarding the antigenicity differences between SARS-CoV-2 and SARS-CoV, but also has important implications in vaccine development.","ABSTRACTA fully Integrated Micropillar Polydimethylsiloxane Accurate CRISPR Detection (IMPACT) system is developed for viral DNA detection. This powerful system is patterned with high-aspect ratio micropillars to enhance reporter probe binding. After surface modification and probe immobilization, CRISPR Cas12a/crRNA complex is injected into the fully enclosed system. With the presence of double-stranded DNA target, the CRISPR enzyme is activated and non-specifically cleaves the ssDNA reporters initially immobilized on the micropillars. This collateral cleavage releases fluorescence dyes into the assay, and the intensity is linearly proportional to the target DNA concentration ranging from 0.1 to 10 nM. Importantly, this system does not rely on traditional dye-quencher labeled probe thus eliminating the fluorescence background presented in the assay. Furthermore, our one-step detection protocol is performed at isothermal conditions (37\u00b0C) without using complicated and time-consuming off-chip probe hybridization and denaturation. This miniaturized and fully packed IMPACT chip demonstrates rapid, sensitive, and simple nucleic acid detection and is an ideal candidate for the next generation molecular diagnostic platform for point-of-care (POC) applications, responding to emerging and deadly pathogen outbreaks.","AbstractRNA viruses are the only known RNA-protein (RNP) entities capable of autonomous replication (albeit within a permissive environment). A 33.5-kb nidovirus has been considered close to the upper size limit for such entities; conversely, the minimal cellular DNA genome is ~200 kb. This large difference presents a daunting gap for the transition from primordial RNP to contemporary DNA-RNP-based life. Whether or not RNA viruses represent transitional steps on the road to DNA-based life, studies of larger RNA viruses advance our understanding of size constraints on RNP entities. For example, emergence of the largest previously known RNA genomes (20-34 kb in positive-stranded nidoviruses, including coronaviruses) is associated with a proofreading exoribonuclease encoded in the nidoviral open reading frame 1b (ORF1b). However, apparent constraints on the size of ORF1b, which encodes this and other key replicative enzymes, have been hypothesized to limit further expansion of viral RNA genomes. Here, we characterize a novel nidovirus (planarian secretory cell nidovirus; PSCNV) whose disproportionately large ORF1b-like region, and overall 41.1 kb genome, substantially extend the presumed limits on RNA genome size. This genome encodes a predicted 13,556-aa polyprotein in an unconventional single ORF, yet retains canonical nidoviral genome organization and expression, and key replicative domains. Our evolutionary analysis suggests that PSCNV diverged early from multi-ORF nidoviruses, and subsequently acquired additional genes, including those typical of large DNA viruses or hosts. PSCNV\u2019s greatly expanded genome, proteomic complexity, and unique features \u2013 impressive in themselves \u2013 attest to the likelihood of still-larger RNA genomes awaiting discovery.Significance StatementRNA viruses are the only known RNA-protein (RNP) entities capable of autonomous replication. The upper genome size for such entities was assumed to be &lt;35 kb; conversely, the minimal cellular DNA genome is ~200 kb. This large difference presents a daunting gap for the proposed evolution of contemporary DNA-RNP-based life from primordial RNP entities. Here, we describe a nidovirus from planarians, whose 41.1 kb genome is 23% larger than the largest known of RNA virus. The planarian secretory cell nidovirus has broken apparent constraints on the size of the genomic subregion that encodes core replication machinery, and has acquired genes not previously observed in RNA viruses. This virus challenges and advances our understanding of the limits to RNA genome size.","The nasopharynx (NP) is a reservoir for microbes associated with acute respiratory illnesses (ARI). The development of asthma is initiated during infancy, driven by airway inflammation associated with infections. Here, we report viral and bacterial community profiling of NP aspirates across a birth cohort, capturing all lower respiratory illnesses during their first year. Most infants were initially colonized with Staphylococcus or Corynebacterium before stable colonization with Alloiococcus or Moraxella, with transient incursions of Streptococcus, Moraxella or Haemophilus marking virus-associated ARIs. Our data identify the NP microbiome as a determinant for infection spread to the lower airways, severity of accompanying inflammatory symptoms, and risk for future asthma development. Early asymptomatic colonization with Streptococcus was a strong asthma predictor, and antibiotic usage disrupted asymptomatic colonization patterns.","The pathogenesis of infectious diarrheal diseases is largely attributed to enterotoxin proteins that disrupt intestinal water absorption, causing severe dehydration. Despite profound health consequences, the impacts of diarrhea-causing microbes on the evolutionary history of host species are largely unknown. We investigated patterns of genetic variation in mammalian Guanylate Cyclase-C (GC-C), an intestinal receptor frequently targeted by bacterial enterotoxins, to determine how hosts might adapt in response to diarrheal infections. Under normal conditions, GC-C interacts with endogenous guanylin peptides to promote water secretion in the intestine, but signaling can be hijacked by bacterially-encoded heat-stable enterotoxins (STa) during infection, which leads to overstimulation of GC-C and diarrhea. Phylogenetic analysis in mammals revealed evidence of recurrent positive selection in the GC-C ligand-binding domain in primates and bats, consistent with selective pressures to evade interactions with STa. Using in vitro assays and transgenic intestinal organoids to model STa-mediated diarrhea, we show that GC-C diversification in these lineages results in substantial variation in toxin susceptibility. In bats, we observe a unique pattern of compensatory coevolution in the endogenous GC-C ligand uroguanylin, reflecting intense bouts of positive selection at the receptor-ligand interface. These findings demonstrate control of water physiology as a previously unrecognized interface for genetic conflict and reveal diarrheal pathogens as a source of selective pressure among diverse mammals.","ABSTRACTA novel coronavirus (SARS-CoV-2) is the causative agent of an emergent severe respiratory disease (COVID-19) in humans that is threatening to result in a global health crisis. By using genomic, sequence, structural and evolutionary analysis, we show that Alpha- and Beta-CoVs possess several novel families of immunoglobulin (Ig) domain proteins, including ORF8 and ORF7a from SARS-related coronaviruses and two protein groups from certain Alpha-CoVs. Among them, ORF8 is distinguished in being rapidly evolving, possessing a unique insert and a hypervariable position among SARS-CoV-2 genomes in its predicted ligand-binding groove. We also uncover many Ig proteins from several metazoan viruses which are distinct in sequence and structure but share an architecture comparable to that of CoV Ig domain proteins. Hence, we propose that deployment of Ig domain proteins is a widely-used strategy by viruses, and SARS-CoV-2 ORF8 is a potential pathogenicity factor which evolves rapidly to counter the immune response and facilitate the transmission between hosts.","Lower respiratory viral infections, such as influenza virus and severe acute respiratory syndrome coronavirus 2 (SARS-CoV2) infections, often cause severe viral pneumonia in aged individuals. Here, we report that influenza viral pneumonia leads to chronic non-resolving lung pathology and exaggerated accumulation of CD8+ tissue-resident memory T cells (TRM) in the respiratory tract of aged hosts. TRM accumulation relies on elevated TGF-b; present in aged tissues. Further, we show that TRM isolated from aged lungs lack a subpopulation characterized by expression of molecules involved in TCR signaling and effector function. Consequently, TRM cells from aged lungs were insufficient to provide heterologous protective immunity. Strikingly, the depletion of CD8+ TRM cells dampens persistent chronic lung inflammation and ameliorates tissue fibrosis in aged, but not young, animals. Collectively, our data demonstrate that age-associated TRM cell malfunction supports chronic lung inflammatory and fibrotic sequelae following viral pneumonia in aged hosts.","AbstractBees, important pollinators, have declined significantly in recent decades, and human- induced changes to nutritional landscapes are partly responsible. Changes to nutritional quality rather than quantity have been overlooked as a threat to bee health. Yet knowledge of bee nutrition is currently largely restricted to adults of social species. Larval stages, where most growth occurs, are relatively understudied - perhaps because most social bees provision progressively and collectively, making nutrition difficult to trace.In mass-provisioning solitary bees (Osmia bicornis L.), we can manipulate and follow larval nutrition, and thereby determine effects of changes in diet quality. Under the Geometric Framework for Nutrition, we restricted larvae to 6 diets: 3 protein:carbohydrate ratios and 2 nutrient concentrations. We asked: (a) which diets maximise body size and survival, (b) what consumption rules do larvae follow when nutrients are imbalanced? Finally, (c) given a choice of complementary diets, are larvae able to select a dietary balance?Larvae pupated after consuming a fixed carbohydrate amount, but tolerated a wide range of protein. Body size and survival were maximised on our lowest P:C ratio diet, and having consumed the most carbohydrate. When eating freely from two diets, larvae converged on a P:C ratio of 1:1.8, but not an overall nutrient intake target. Nevertheless, larvae maintained stable carbohydrate intake, while protein intake varied with the available diet.Our results suggest solitary bee larvae regulate carbohydrate most closely, but that excessive indigestible material may limit their actual nutrient intake. Carbohydrate may be critical to overwinter survival, and/or may be more limiting than protein. The large variation in protein tolerated, despite its importance, suggests bee larvae may be vulnerable to landscape changes - and therefore reliant on parents to regulate protein. Given the mixed evidence on whether parents can sense pollen protein content, our results highlight bees\u2019 potential vulnerability to a \u201cnutritional trap\u201d, i.e. where rapid changes in their nutritional environment outstrip their evolved capacity to detect those changes, impairing their fitness.","1.1.AbstractSevere acute respiratory syndrome coronavirus (SARS-CoV) is the causative agent of a respiratory disease with a high case fatality rate. During the formation of the coronaviral replication/transcription complex (RTC), essential steps include processing of the conserved polyprotein nsp7-10 region by the main protease Mpro and subsequent complex formation of the released nsp\u2019s. Here, we analyzed processing of the coronavirus nsp7-10 region using native mass spectrometry showing consumption of substrate, rise and fall of intermediate products and complexation. Importantly, there is a clear order of cleavage efficiencies, which is influenced by the polyprotein tertiary structure. Furthermore, the predominant product is an nsp7+8(2:2) hetero-tetramer with nsp8 scaffold. In conclusion, native MS, opposed to other methods, can expose the processing dynamics of viral polyproteins and the landscape of protein interactions in one set of experiments. Thereby, new insights into protein interactions, essential for generation of viral progeny, were provided, with relevance for development of antivirals.","The recent emergence of SARS-CoV-2 and the subsequent COVID-19 pandemic have posed a public health crisis. Higher morbidity and mortality of men with COVID-19 may be explained by androgen-driven mechanisms. One such proposed mechanism is androgen regulation of pulmonary TMPRSS2, the host co-receptor for SARS-CoV-2. We find no evidence for increased TMPRSS2 mRNA expression in the lungs of males compared to females in humans or mice. Furthermore, in male mice, treatment with the androgen receptor antagonist enzalutamide does not decrease pulmonary TMPRSS2 expression. Nevertheless, regardless of sex, smoking significantly increases the expression of TMPRSS2, which reverts back to never-smoker levels in former smokers. Finally, we show that in mouse models, despite equivalent AR transcript levels, males express markedly higher amounts of AR protein. If a similar sex-specific regulation of AR protein occurs in human lung, androgens could play important roles in clinical outcome of COVID-19 through mechanisms other than TMPRSS2 regulation.","Many enveloped viruses infect cells within endocytic compartments. The drop in pH that accompanies endosomal maturation, often in conjunction with proteolytic factors, serves as a trigger for viral fusion proteins to insert into the endosomal membrane and drive fusion. The dynamics of this process has been studied by tracking viruses within living cells, which limits the precision with which fusion can be synchronized and controlled, and by reconstituting viral fusion to synthetic membranes, which introduces non-physiological membrane curvature and composition. To overcome these limitations, we have engineered the chemically controllable triggering of single-virus fusion within endosomes. We isolate influenza virus:endosome conjugates from cells prior to fusion, immobilize them in a microfluidic flow cell, and then rapidly and controllably trigger fusion. This platform demonstrates lipid-mixing kinetics that are grossly similar to influenza fusion with model membranes but display some subtle differences. Because it preserves endosomal membrane asymmetry and protein composition, it also provides a means to test how perturbations to endosomal trafficking and cellular restriction factors affect viral membrane fusion.","AbstractBackgroundLeading to a sustained epidemic spread with &gt;40,000 confirmed human infections, including &gt;10,000 deaths, COVID-19 was caused by 2019-nCov and resulted in acute respiratory distress syndrome (ARDS) and sepsis, which brought more challenges to the patient\u2019s treatment. The S-glycoprotein, which recognized as the key factor for the entry of 2019-nCov into the cell, contains two functional domains: an ACE2 receptor binding domain and a second domain necessary for fusion of the coronavirus and cell membranes. FURIN activity, exposes the binding and fusion domains, is essential for the zoonotic transmission of 2019-nCov. Moreover, it has been reported that ACE2 is likely to be the receptor for 2019-nCoV. In addition, FURIN enzyme and ACE2 receptor were expressed in airway epithelia, cardiac tissue, and enteric canals, which considered as the potential target organ of the virus. However, report about the expression of FURIN and ACE2 in oral tissues was limited.MethodsIn order to investigate the potential infective channel of new coronavirus in oral cavity, we analyze the expression of ACE2 and FURIN that mediate the new coronavirus entry into host cells in oral mucosa using the public single-cell sequence datasets. Furthermore, immunohistochemical staining experiment was performed to confirm the expression of ACE2 and FURIN in the protein level.ResultsThe bioinformatics results indicated the differential expression of ACE2 and FURIN on epithelial cells of different oral mucosal tissues and the proportion of FURIN-positive cells was obviously higher than that of ACE2-positive cells. IHC experiments revealed that both the ACE2-positive and FURIN-positive cells in the target tissues were mainly positioned in the epithelial layers, partly expressed in fibroblasts, which further confirm the bioinformatics results.ConclusionsBased on these findings, we speculated that 2019-nCov could effectively invade oral mucosal cells though two possible routes: binding to the ACE2 receptor and fusion with cell membrane activated by FURIN protease. Our results indicated that oral mucosa tissues are susceptible to 2019-nCov, which provides valuable information for virus-prevention strategy in clinical care as well as daily life.","AbstractWe have developed a generalizable \u2018smart molecular diagnostic\u2019 capable of accurate point-of-care (POC) detection of variable nucleic acid targets. Our one-pot isothermal assay relies on multiplex execution of four loop-mediated isothermal amplification reactions, with primers that are degenerate and redundant, thereby increasing the breadth of targets while reducing the probability of amplification failure. An easy-to-read visual answer is computed directly by a multi-input Boolean OR gate signal transducer that uses degenerate strand exchange probes to assess any combination of amplicons. We demonstrate our platform by using the same assay to detect divergent Asian and African lineages of the evolving Zika virus (ZIKV), while maintaining selectivity against non-target viruses. Direct analysis of biological specimens proved possible, with 20 virions / \u00b5l being directly detected in human saliva within 90 minutes, and crudely macerated ZIKV-infected Aedes aegypti mosquitoes being identified with 100% specificity and sensitivity. The ease-of-use with minimal instrumentation, broad programmability, and built-in fail-safe reliability make our smart molecular diagnostic attractive for POC use.","AbstractBackgroundNew endemic disease has been spread across Wuhan City, China on December 2019. Within few weeks, the World Health Organization (WHO) announced a novel coronavirus designated as coronavirus disease 2019 (COVID-19). In late January 2020, WHO declared the outbreak of a \u201cpublic-health emergency of international concern\u201d due to the rapid and increasing spread of the disease worldwide. Currently, there is no vaccine or approved treatment for this emerging infection; thus the objective of this study is to design a multi epitope peptide vaccine against COVID-19 using immunoinformatics approach.MethodSeveral techniques facilitating the combination of immunoinformatics approach and comparative genomic approach were used in order to determine the potential peptides for designing the T cell epitopes-based peptide vaccine using the envelope protein of 2019-nCoV as a target.ResultsExtensive mutations, insertion and deletion were discovered with comparative sequencing in COVID-19 strain. Additionally, ten peptides binding to MHC class I and MHC class II were found to be promising candidates for vaccine design with adequate world population coverage of 88.5% and 99.99%, respectively.ConclusionT cell epitopes-based peptide vaccine was designed for COVID-19 using envelope protein as an immunogenic target. Nevertheless, the proposed vaccine is rapidly needed to be validated clinically in order to ensure its safety, immunogenic profile and to help on stopping this epidemic before it leads to devastating global outbreaks.","AbstractThe ongoing outbreak of viral pneumonia in China and beyond is associated with a novel coronavirus, provisionally termed 2019-nCoV. This outbreak has been tentatively associated with a seafood market in Wuhan, China, where the sale of wild animals may be the source of zoonotic infection. Although bats are likely reservoir hosts for 2019-nCoV, the identity of any intermediate host facilitating transfer to humans is unknown. Here, we report the identification of 2019-nCoV related coronaviruses in pangolins (Manis javanica) seized in anti-smuggling operations in southern China. Metagenomic sequencing identified pangolin associated CoVs that belong to two sub-lineages of 2019-nCoV related coronaviruses, including one very closely related to 2019-nCoV in the receptor-binding domain. The discovery of multiple lineages of pangolin coronavirus and their similarity to 2019-nCoV suggests that pangolins should be considered as possible intermediate hosts for this novel human virus and should be removed from wet markets to prevent zoonotic transmission.","ABSTRACTZika virus can be passed from a pregnant woman to her fetus, thus leading to birth defects including more than microcephaly. It has been recently estimated that one-third of the world population will be infected by Zika, but unfortunately no vaccine or medicine is available so far. Zika NS2B-NS3pro is essential for its replication and thus represents an attractive target for drug discovery/design. Here we characterized conformation, catalysis, inhibition and dynamics of linked and unlinked Zika NS2B-NS3pro complexes by both experiments and MD simulations. The results unveil the unique properties of Zika NS2B-NS3pro which are very different from Dengue one. Particularly, CD and NMR studies indicate that unlike Dengue, the C-terminal region of Zika NS2B with a significant sequence variation is highly disordered in the open conformation. Indeed, MD simulations reveal that up to 100 ns, the Dengue NS2B C-terminus constantly has close contacts with its NS3pro domain. By a sharp contrast, the Zika NS2B C-terminus loses the contacts with its NS3pro domain after 10 ns, further forming a short \u03b2-sheet characteristic of the closed conformation at 30 ns. Furthermore, we found that a small molecule, previously identified as an active site inhibitor for other flaviviral NS2B-NS3pro, inhibited Zika NS2B-NS3pro potently in an allosteric manner. Our study provides the first insight into the dynamics of Zika NS2B-NS3pro and further deciphers that it is susceptible to allosteric inhibition, which thus bears critical implications for the future development of therapeutic allosteric inhibitors.","SummaryRapidly responding to novel pathogens, such as SARS-CoV-2, represents an extremely challenging and complex endeavor. Numerous promising therapeutic and vaccine research efforts to mitigate the catastrophic effects of COVID-19 pandemic are underway, yet an efficacious countermeasure is still not available. To support these global research efforts, we have used a novel computational pipeline combining machine learning, bioinformatics, and supercomputing to predict antibody structures capable of targeting the SARS-CoV-2 receptor binding domain (RBD). In 22 days, using just the SARS-CoV-2 sequence and previously published neutralizing antibody structures for SARS-CoV-1, we generated 20 initial antibody sequences predicted to target the SARS-CoV-2 RBD. As a first step in this process, we predicted (and publicly released) structures of the SARS-CoV-2 spike protein using homology-based structural modeling. The predicted structures proved to be accurate within the targeted RBD region when compared to experimentally derived structures published weeks later. Next we used our in silico design platform to iteratively propose mutations to SARS-CoV-1 neutralizing antibodies (known not to bind SARS-Cov-2) to enable and optimize binding within the RBD of SARS-CoV-2. Starting from a calculated baseline free energy of \u221248.1 kcal/mol (\u00b1 8.3), our 20 selected first round antibody structures are predicted to have improved interaction with the SARS-CoV-2 RBD with free energies as low as \u221282.0 kcal/mole. The baseline SARS-CoV-1 antibody in complex with the SARS-CoV-1 RBD has a calculated interaction energy of \u221252.2 kcal/mole and neutralizes the virus by preventing it from binding and entering the human ACE2 receptor. These results suggest that our predicted antibody mutants may bind the SARS-CoV-2 RBD and potentially neutralize the virus. Additionally, our selected antibody mutants score well according to multiple antibody developability metrics. These antibody designs are being expressed and experimentally tested for binding to COVID-19 viral proteins, which will provide invaluable feedback to further improve the machine learning\u2013driven designs. This technical report is a high-level description of that effort; the Supplementary Materials includes the homology-based structural models we developed and 178,856 in silico free energy calculations for 89,263 mutant antibodies derived from known SARS-CoV-1 neutralizing antibodies.","ABSTRACTOn December 31, 2019, the World Health Organization was notified about a cluster of pneumonia of unknown aetiology in the city of Wuhan, China. Chinese authorities later identified a new coronavirus (2019-nCoV) as the causative agent of the outbreak. As of January 23, 2020, 655 cases have been confirmed in China and several other countries. Understanding the transmission characteristics and the potential for sustained human-to-human transmission of 2019-nCoV is critically important for coordinating current screening and containment strategies, and determining whether the outbreak constitutes a public health emergency of international concern (PHEIC). We performed stochastic simulations of early outbreak trajectories that are consistent with the epidemiological findings to date. We found the basic reproduction number, R0, to be around 2.2 (90% high density interval 1.4\u20143.8), indicating the potential for sustained human-to-human transmission. Transmission characteristics appear to be of a similar magnitude to severe acute respiratory syndrome-related coronavirus (SARS-CoV) and the 1918 pandemic influenza. These findings underline the importance of heightened screening, surveillance and control efforts, particularly at airports and other travel hubs, in order to prevent further international spread of 2019-nCoV.","ABSTRACTHost interferon-induced transmembrane proteins (IFITMs) are broad-spectrum antiviral restriction factors. Of these, IFITM3 potently inhibits viruses that enter cells through acidic endosomes, many of which are zoonotic and emerging viruses with bats (order Chiroptera) as natural hosts. We previously demonstrated that microbat IFITM3 is antiviral. Here we show that bat IFITMs are characterized by strong adaptive evolution and identify a highly variable and functionally important site - codon 70 - within the conserved CD225 domain of IFITMs. Mutation of this residue in microbat IFITM3 impairs restriction of four different virus families that enter cells via endosomes. This mutant shows altered subcellular localization and reduced S-palmitoylation, a phenotype copied by mutation of conserved cysteine residues in microbat IFITM3. Furthermore, we show that microbat IFITM3 is S-palmitoylated on cysteine residues C71, C72 and C105, mutation of each cysteine residue individually impairs virus restriction, and a triple C71-C72-C105 mutant loses all restriction, concomitant with subcellular re-localization of microbat IFITM3 to Golgi-associated sites. Thus, we propose that S-palmitoylation is critical for Chiropteran IFITM3 function and identify a key molecular determinant of IFITM3 S-palmitoylation.","ABSTRACTPrevious findings of Middle East Respiratory Syndrome coronavirus (MERS-CoV)-related viruses in bats, and the ability of Tylonycteris-BatCoV HKU4 spike protein to utilize MERS-CoV receptor, human dipeptidyl peptidase 4 hDPP4, suggest a bat ancestral origin of MERS-CoV. We developed 12 primary bat cell lines from seven bat species, including Tylonycteris pachypus, Pipistrellus abramus and Rhinolophus sinicus (hosts of Tylonycteris-BatCoV HKU4, Pipistrellus-BatCoV HKU5 and SARS-related-CoV respectively), and tested their susceptibilities to MERS-CoVs, SARS-CoV and human coronavirus 229E (HCoV-229E). Five cell lines, including P. abramus and R. sinicus but not T. pachypus cells, were susceptible to human MERS-CoV EMC/2012. However, three tested camel MERS-CoV strains showed different infectivities, with only two strains capable of infecting three and one cell lines respectively. SARS-CoV can only replicate in R. sinicus cells, while HCoV-229E cannot replicate in any bat cells. Bat dipeptidyl peptidase 4 (DPP4) sequences were closely related to those of human and non-human primates but distinct from dromedary DPP4 sequence. Critical residues for binding to MERS-CoV spike protein were mostly conserved in bat DPP4. DPP4 was expressed in the five bat cells susceptible to MERS-CoV, with significantly higher mRNA expression levels than those in non-susceptible cells (P=0.0174), supporting that DPP4 expression is critical for MERS-CoV infection in bats. However, overexpression of T. pachypus DPP4 failed to confer MERS-CoV susceptibility in T. pachypus cells, suggesting other cellular factors in determining viral replication. The broad cellular tropism of MERS-CoV should prompt further exploration of host diversity of related viruses to identify its ancestral origin.","In social species where offspring are reared together in communal burrows or similar structures, young animals typically do not engage in between-group contact during their development, a behavioural trait we call offspring with restricted between-group contact (ORC). The impact of this trait on the persistence of contagious pathogens that generate lifelong immunity in their hosts is currently unclear. We hypothesize that in populations with ORC, the formation, in groups, of a protective barrier of only recovered adults, prevents the transmission of this type of pathogens to the new susceptible hosts (i.e. young animals), thereby increasing the probability of epidemic fade-out. We implement a spatially implicit individual-based Susceptible-Infected-Recovered (SIR) model for a large range of host and pathogen traits and show that the epidemic fade-out probability is consistently higher in populations with ORC, especially when disease spread is fast (high basic reproduction number R0). We also show that ORC can counteract the cost of group-living in terms of disease risk to a greater extent than variation in other traits. We discuss our findings in relation to herd immunity and outline how they could be used to implement efficient management measures such as vaccinations.","Summary / AbstractCodon usage patterns of viruses reflect a series of evolutionary changes that enable viruses to shape their survival rates and fitness toward the external environment and, most importantly, their hosts. In the present study, we employed multiple codon usage analysis indices to determine genotype specific codon usage patterns of Zika virus (ZIKV) strains from the current outbreak and those reported previously. Several genotype specific and common codon usage traits were noted in ZIKV coding sequences, indicative of independent evolutionary origins from a common ancestor. The overall influence of natural selection was found to be more profound than that of mutation pressure and acting on specific set of viral genes belonging to ZIKV strains of Asian genotype from the recent outbreak. Furthermore, an interplay of codon adaptation and deoptimization have been observed in ZIKV genomes. The collective findings of codon analysis in association with the geographical data of Aedes populations in the Americas suggests that ZIKV have evolved a dynamic set of codon usage patterns in order to maintain a successful replication and transmission chain within multiple hosts and vectors.","Vascular leak is a key driver of organ injury in diseases such as Acute Respiratory Distress Syndrome caused by viruses, including COVID-19. Strategies that reduce enhanced permeability and vascular inflammation are promising therapeutic targets. Activation of the Angiopoietin-1 (Angpt1)-Tie2 tyrosine kinase signaling pathway is an important regulator of vascular quiescence. Here we describe the design and construction of a new soluble ANGPT1 mimetic that is a potent activator of endothelial Tie2 in vitro and in vivo. Using a chimeric fusion strategy, we replaced the extracellular matrix (ECM) binding and oligomerization domain of ANGPT1 with a heptameric scaffold derived from the C-terminus of serum complement protein C4-binding protein \u03b1 (C4BP). We refer to this new fusion protein biologic as C4BP-ANG1, which forms a stable heptamer and induces TIE2 phosphorylation in cultured cells, and in the lung following i.v. injection of mice. Injection of C4BP-ANG1 ameliorates VEGF- and lipopolysaccharide-induced vascular leakage, in keeping with the known functions of Angpt1-Tie2 in maintaining quiescent vascular stability, and therefore is a promising candidate treatment for inflammatory endothelial dysfunction.","ABSTRACTNew anti-AIDS treatments must be continually developed in order to overcome resistance mutations including those emerging in the newest therapeutic target, the viral integrase (IN). Multimerization of IN is functionally imperative and provides a forthcoming therapeutic target. Allosteric inhibitors of IN bind to non-catalytic sites and prevent correct multimerization not only restricting viral integration but also the assembly and maturation of viral particles. Here, we report an allosteric inhibitor peptide targeting an unexploited SH3-docking platform of retroviral IN. The crystal structure of the peptide in complex with the HIV-1 IN core domain reveals a steric interference that would inhibit conserved docking of SH3-containing domain with the core domain vital for IN multimerization, providing a template for the development of novel anti-IN allosteric inhibitors.","SummaryFasting paradigms elicit a wide-range of health benefits including suppressing inflammation. Exploring the molecular mechanisms that prevent inflammation during caloric restriction may yield promising new therapeutic targets. During fasting, activation of the nuclear receptor peroxisome proliferator-activated receptor alpha (PPARA) promotes the utilization of lipids as an energy source. Herein, we show that ligand activation of PPARA directly upregulates the long non-coding RNA gene Gm15441 through binding sites within its promoter. Gm15441 expression suppresses its antisense transcript, encoding thioredoxin interacting protein (TXNIP). This, in turn, decreases TXNIP-stimulated NLRP3 inflammasome activation, caspase-1 (CASP1) cleavage, and proinflammatory interleukin 1 beta (IL1B) maturation. Gm15441-null mice were developed and shown to be more susceptible to NLRP3 inflammasome activation and to exhibit elevated CASP1 and IL1B cleavage in response to metabolic and inflammatory stimuli. These findings provide evidence for a novel mechanism by which PPARA attenuates hepatic inflammasome activation in response to metabolic stress through lncRNA Gm15441 induction. Graphical abstract","AbstractBackgroundAs of May 27, 2018, 54 cases of Ebola virus disease (EVD) were reported in \u00c9quateur Province, Democratic Republic of Congo. We used reported case counts and time series from prior outbreaks to estimate the current outbreak size and duration with and without vaccine use.MethodsWe modeled Ebola virus transmission using a stochastic branching process model with a negative binomial distribution, using both estimates of reproduction number R declining from supercritical to subcritical derived from past Ebola outbreaks, as well as a particle filtering method to generate a probabilistic projection of the future course of the outbreak conditioned on its reported trajectory to date; modeled using 0%, 44%, and 62% estimates of vaccination coverage. Additionally, we used the time series for 18 prior Ebola outbreaks from 1976 to 2016 to parameterize a regression model predicting the outbreak size from the number of observed cases from April 4 to May 27.ResultsWith the stochastic transmission model, we projected a median outbreak size of 78 EVD cases (95% credible interval: 52, 125.4), 86 cases (95% credible interval: 53, 174.3), and 91 cases (95% credible interval: 52, 843.5), using 62%, 44%, and 0% estimates of vaccination coverage. With the regression model, we estimated a median size of 85.0 cases (95% prediction interval: 53.5, 216.6).ConclusionsThis outbreak has the potential to be the largest outbreak in DRC since 2007. Vaccines are projected to limit outbreak size and duration but are only part of prevention, control, and care strategies.","AbstractBackgroundThe 2019 novel coronavirus (2019-nCoV or SARS-CoV-2) has spread more rapidly than any other betacoronavirus including SARS-CoV and MERS-CoV. However, the mechanisms responsible for infection and molecular evolution of this virus remained unclear.MethodsWe collected and analyzed 120 genomic sequences of 2019-nCoV including 11 novel genomes from patients in China. Through comprehensive analysis of the available genome sequences of 2019-nCoV strains, we have tracked multiple inheritable SNPs and determined the evolution of 2019-nCoV relative to other coronaviruses.ResultsSystematic analysis of 120 genomic sequences of 2019-nCoV revealed co-circulation of two genetic subgroups with distinct SNPs markers, which can be used to trace the 2019-nCoV spreading pathways to different regions and countries. Although 2019-nCoV, human and bat SARS-CoV share high homologous in overall genome structures, they evolved into two distinct groups with different receptor entry specificities through potential recombination in the receptor binding regions. In addition, 2019-nCoV has a unique four amino acid insertion between S1 and S2 domains of the spike protein, which created a potential furin or TMPRSS2 cleavage site.ConclusionsOur studies provided comprehensive insights into the evolution and spread of the 2019-nCoV. Our results provided evidence suggesting that 2019-nCoV may increase its infectivity through the receptor binding domain recombination and a cleavage site insertion.One Sentence SummaryNovel 2019-nCoV sequences revealed the evolution and specificity of betacoronavirus with possible mechanisms of enhanced infectivity.","The current state of much of the Wuhan pneumonia virus (COVID-19) research shows a regrettable lack of data sharing and considerable analytical obfuscation. This impedes global research cooperation, which is essential for tackling public health emergencies, and requires unimpeded access to data, analysis tools, and computational infrastructure. Here we show that community efforts in developing open analytical software tools over the past ten years, combined with national investments into scientific computational infrastructure, can overcome these deficiencies and provide an accessible platform for tackling global health emergencies in an open and transparent manner. Specifically, we use all COVID-19 genomic data available in the public domain so far to (1) underscore the importance of access to raw data and to (2) demonstrate that existing community efforts in curation and deployment of biomedical software can reliably support rapid, reproducible research during global health crises. All our analyses are fully documented at https://github.com/galaxyproject/SARS-CoV-2.","AbstractThe large magnitude of protein-protein interaction (PPI) pairs within the human interactome necessitates the development of predictive models and screening tools to better understand this fundamental molecular communication. However, despite enormous efforts from various groups to develop predictive techniques in the last decade, PPI complex structures are in general still very challenging to predict due to the large number of degrees of freedom. In this study, we use the binding complex of human profilin (PFN1) and polyproline-10 (P10) as a model system to examine various approaches, with the aim of going beyond normal protein docking for PPI prediction and evaluation. The potential of mean force (PMF) was first obtained from the timeconsuming umbrella sampling, which confirmed that the most stable binding structure identified by the maximal PMF difference is indeed the crystallographic binding structure. Moreover, crucial residues previously identified in experimental studies, W3, H133 and S137 of PFN1, were found to form favorable hydrogen bonds with P10, suggesting a zipping process during the binding between PFN1 and P10. We then explored both regular molecular dynamics (MD) and steered molecular dynamics (SMD) simulations, seeking for better criteria of ranking the PPI prediction. Despite valuable information obtained from conventional MD simulations, neither the commonly used interaction energy between the two binding parties nor the long-term root mean square displacement (RMSD) correlates well with the PMF results. On the other hand, with a sizable collection of trajectories, we demonstrated that the average rupture work calculated from SMD simulations correlates fairly well with the PMFs (R2 = 0.67), making it a promising PPI screening method.","The COVID-19 global pandemic is an unprecedented health emergency. Insufficient access to testing has hampered effective public health interventions and patient care management in a number of countries. Furthermore, the availability of regulatory-cleared reagents has challenged widespread implementation of testing. We rapidly developed a qRT-PCR SARS-CoV-2 detection assay using a 384-well format and tested its analytic performance across multiple nucleic acid extraction kits. Our data shows robust analytic accuracy on residual clinical biospecimens. Limit of detection sensitivity and specificity was confirmed with currently available commercial reagents. Our methods and results provide valuable information for other high-complexity laboratories seeking to develop effective, local, laboratory-developed procedures with high-throughput capability to detect SARS-CoV-2.","The literature reports several symptoms of SARS-CoV-2 in humans such as fever, cough, fatigue, pneumonia, and headache. Furthermore, patients infected with similar strains (SARS-CoV and MERS-CoV) suffered testis, liver, or thyroid damage. Angiotensin-converting enzyme 2 (ACE2) serves as an entry point into cells for some strains of coronavirus (SARS-CoV, MERS-CoV, SARS-CoV-2). Our hypothesis was that as ACE2 is essential to the SARS-CoV-2 virus invasion, then brain regions where ACE2 is the most expressed are more likely to be disturbed by the infection. Thus, the expression of other genes which are also over-expressed in those damaged areas could be affected. We used mRNA expression levels data of genes provided by the Allen Human Brain Atlas (ABA), and computed spatial correlations with the LinkRbrain platform. Genes whose co-expression is spatially correlated to that of ACE2 were then clustered into 16 groups, depending on the organ in which they are the most expressed (as described by the NCBI genes database). The list of organs where genes sharing local over-expression with the ACE2 gene are the most expressed is astonishingly similar to the organs affected by Covid-19.","AbstractThe application of small angle X-ray scattering (SAXS) to the structural characterization of transmembrane proteins (MPs) in detergent solutions has become a routine procedure at the most synchrotron BioSAXS beamlines around the world. SAXS provides overall parameters and low resolution shapes of solubilized MPs, but is also meaningfully employed in hybrid modeling procedures that combine scattering data with information provided by high-resolution techniques (eg. macromolecular crystallography, nuclear magnetic resonance and cryo-electron microscopy). Structural modeling of MPs from SAXS data is non-trivial, and the necessary computational procedures require further formalization and facilitation. We propose an automated pipeline integrated with the laboratory-information management system ISPyB, aimed at preliminary SAXS analysis and the first-step reconstruction of MPs in detergent solutions, in order to streamline high-throughput studies, especially at synchrotron beamlines. The pipeline queries an ISPyB database for available a priori information via dedicated services, estimates model-free SAXS parameters and generates preliminary models utilizing either ab initio, high-resolution-based, or mixed/hybrid methods. The results of the automated analysis can be inspected online using the standard ISPyB interface and the estimated modeling parameters may be utilized for further in-depth modeling beyond the pipeline. Examples of the pipeline results for the modelling of the tetrameric alpha-helical membrane channel Aquaporin0 and mechanosensitive channel T2, solubilized by n-Dodecyl \u03b2-D-maltoside are presented. We demonstrate how the increasing amount a priori information improves the model resolution and enables deeper insights into the molecular structure of protein-detergent complexes.STATEMENT OF SIGNIFICANCESmall angle X-ray scattering (SAXS) using synchrotron radiation is a powerful technique for the structural characterization of transmembrane proteins (MPs) in detergent solutions Overall structural characterization and modeling of MPs from SAXS data is non-trivial, and the necessary computational procedures require further formalization and facilitation. We propose an automated pipeline integrated with the laboratory-information management system ISPyB, aimed at preliminary SAXS analysis and modelling of MPs in detergent solutions, in order to streamline high-throughput studies, especially at synchrotron beamlines.","SummaryPlants have a highly sophisticated endomembrane system that is targeted by plant viruses for cell-to-cell movement. The movement protein (MP) of Ourmia melon virus (OuMV) is targeted to plasmodesmata (PD) and forms tubules to facilitate cell-to-cell movement. Despite a number of functionally important regions for correct subcellular localization of OuMV MP has been identified, little is known about the pathways OuMV MP hijacks to reach PD. Here, we demonstrate that OuMV MP localizes to the trans-Golgi network (TGN), but not to the multivesicular body/prevacuolar compartment or Golgi, and carries two putative sorting motifs, a tyrosine (Y) and a dileucine (LL) motif, near its N-terminus. Introducing glycine substitutions in these motifs results in loss of OuMV infectivity in Nicotiana benthamiana and Arabidopsis (Arabidopsis thaliana). Live cell imaging of GFP-labeled sorting motif mutants shows that Y motif mutants fail to localize to the TGN, plasma membrane, and PD. Mutations in the LL motif do not impair plasma membrane targeting of MP, but affect its ability to associate with callose deposits at PD. Taken together, these data suggest that both Y and LL motifs are indispensable for targeting of OuMV MP to PD and for efficient systemic infection, but show differences in functionality. This study provides new insights into the role of sorting motifs in intracellular targeting of MPs and vesicle trafficking pathways that plant viruses hijack for cell-to-cell movement.","SUMMARYA new coronavirus (CoV) identified as COVID-19 virus is the etiological agent responsible for the 2019-2020 viral pneumonia outbreak that commenced in Wuhan1-4. Currently there is no targeted therapeutics and effective treatment options remain very limited. In order to rapidly discover lead compounds for clinical use, we initiated a program of combined structure-assisted drug design, virtual drug screening and high-throughput screening to identify new drug leads that target the COVID-19 virus main protease (Mpro). Mpro is a key CoV enzyme, which plays a pivotal role in mediating viral replication and transcription, making it an attractive drug target for this virus5,6. Here, we identified a mechanism-based inhibitor, N3, by computer-aided drug design and subsequently determined the crystal structure of COVID-19 virus Mpro in complex with this compound. Next, through a combination of structure-based virtual and high-throughput screening, we assayed over 10,000 compounds including approved drugs, drug candidates in clinical trials, and other pharmacologically active compounds as inhibitors of Mpro. Six of these inhibit Mpro with IC50 values ranging from 0.67 to 21.4 \u03bcM. Ebselen also exhibited strong antiviral activity in cell-based assays. Our results demonstrate the efficacy of this screening strategy, which can lead to the rapid discovery of drug leads with clinical potential in response to new infectious diseases where no specific drugs or vaccines are available.","AbstractObjectiveHIV infection and aging are both associated with neurodegeneration. However, whether the aging process alone or other factors associated with advanced age account for the progression of neurodegeneration in the aging HIV-positive (HIV+) population remains unclear.MethodsHIV+ (n=70) and HIV-negative (HIV-, n=34) participants underwent diffusion tensor imaging (DTI) and metrics of microstructural properties were extracted from regions of interest (ROIs). A support vector regression model was trained on two independent datasets of healthy adults across the adult life-span (n=765, Cam-CAN = 588; UiO = 177) to predict participant age from DTI metrics, and applied to the HIV dataset. Predicted brain age gap (BAG) was computed as the difference between predicted age and chronological age, and statistically compared between HIV groups. Regressions assessed the relationship between BAG and HIV severity/medical comorbidities. Finally, correlation analyses tested for associations between BAG and cognitive performance.ResultsBAG was significantly higher in the HIV+ group than the HIV-group F (1, 103) = 12.408, p = 0.001). HIV RNA viral load was significantly associated with BAG, particularly in older HIV+ individuals (R2 = 0.29, F(7, 70) = 2.66, p = 0.021). Further, BAG was negatively correlated with domain-level cognitive function (learning: r = \u22120.26, p = 0.008; memory: r = \u22120.21, p = 0.034).ConclusionsHIV infection is associated with augmented white matter aging, and greater brain aging is associated with worse cognitive performance in multiple domains.","As of early March, 2019-nCoV has infected more than one hundred thousand people and claimed thousands of lives. 2019-nCoV is a novel form of coronavirus that causes COVID-19 and has high similarity with SARS-CoV. No approved vaccine yet exists for any form of coronavirus. Here we use computational tools from structural biology and machine learning to identify 2019-nCoV T-cell and B-cell epitopes based on viral protein antigen presentation and antibody binding properties. These epitopes can be used to develop more effective vaccines and identify neutralizing antibodies. We identified 405 viral peptides with good antigen presentation scores for both human MHC-I and MHC-II alleles, and two potential neutralizing B-cell epitopes near the 2019-nCoV spike protein receptor binding domain (440-460 and 494-506). Analyzing mutation profiles of 68 viral genomes from four continents, we identified 96 coding-change mutations. These mutations are more likely to occur in regions with good MHC-I presentation scores (p=0.02). No mutations are present near the spike protein receptor binding domain. Based on these findings, the spike protein is likely immunogenic and a potential vaccine candidate. We validated our computational pipeline with SARS-CoV experimental data.Significance StatementThe novel coronavirus 2019-nCoV has affected more than 100 countries and continues to spread. There is an immediate need for effective vaccines that contain antigens which trigger responses from human T-cells and B-cells (known as epitopes). Here we identify potential T-cell epitopes through an analysis of human antigen presentation, as well as B-cell epitopes through an analysis of protein structure. We identify a list of top candidates, including an epitope located on 2019-nCoV spike protein that potentially triggers both T-cell and B-cell responses. Analyzing 68 samples, we observe that viral mutations are more likely to happen in regions with strong antigen presentation, a potential form of immune evasion. Our computational pipeline is validated with experimental data from SARS-CoV.","AbstractThe interferon-induced transmembrane protein, IFITM3, has been shown to restrict influenza virus infection in murine and in vitro settings for ten years, but no explanation has been found to explain why this virus infection is so highly contagious and infects most individuals it comes in contact with. We confirm that the expression level of IFITM3 plays a role in determining the level of viral infection through manipulation of IFITM3 levels with interferon (IFN) stimulation and overexpression systems. Low basal expression may put some immune cells, including lymphocytes and lung-resident macrophages, at risk of influenza virus infection. Investigating the induction of IFITM3 by IFN, we find a strong preference for Type I IFN in IFITM3 induction in both cell lines and primary human cells. While myeloid cells can increase expression following stimulation by Type I IFN, lymphocytes show minimal induction of IFITM3 following IFN stimulation, suggesting that they are always at risk of viral infection. Surprisingly, we found that the time it takes for maximal induction of IFITM3 is relatively slow for an interferon-stimulated gene at around 36 hours. Low basal expression and slow induction of IFITM3 could increase the risk of influenza virus infection in selected immune cells.ImportanceInfluenza virus infection remains one of the top ten threats to global health, causing significant deaths and hospitalisations across the world each year. Understanding mechanisms for controlling influenza virus infection remain a priority. The interferon-induced transmembrane protein IFITM3 can restrict influenza infection by limiting replication of the virus. The precise mechanisms of how IFITM3 reduced replication of influenza are unknown, although it is predicted to prevent release of viral contents into the cytosol by preventing pore formation on the endosomal compartments where it is suggested to reside. Here we have shown that the expression level of IFITM3 is important in determining the control of influenza virus infection. We find an expression pattern for IFITM3 that varies based on cell type, tissue locality, differentiation state and cell naivety, all of which highlights cells that may be at the highest risk of influenza infection.","AbstractMotivationThe simultaneous consideration of sequence alignment and RNA secondary structure, or structural alignment, is known to help predict more accurate secondary structures of homologs. However, the consideration is heavy and can be done only roughly to decompose structural alignments.ResultsThe PhyloFold method, which predicts secondary structures of homologs considering likely pairwise structural alignments, was developed in this study. The method shows the best prediction accuracy while demanding comparable running time compared to conventional methods.AvailabilityThe source code of the programs implemented in this study is available on \u201chttps://github.com/heartsh/phylofold\u201d and \u201chttps://github.com/heartsh/phyloalifold\u201c.Contact\u201ctagashira_masaki_17@stu-cbms.k.u-tokyo.ac.jp\u201d.Supplementary informationSupplementary data are available.","AbstractSince the beginning of the 2019-20 coronavirus pandemic, a large number of relevant articles has been published or become available in preprint servers. These articles, along with earlier related literature, compose a valuable knowledge base affecting contemporary research studies or, even, government actions to limit the spread of the disease and treatment decisions taken by physicians. However, the number of such articles is increasing at an intense rate making the exploration of the relevant literature and the identification of useful knowledge in it challenging. In this work, we describe BIP4COVID19, an open dataset compiled to facilitate the coronavirus-related literature exploration, by providing various indicators of scientific impact for the relevant articles. Finally, we provide a publicly accessible web interface on top of our data, allowing the exploration of the publications based on the computed indicators.","AbstractOne striking characteristic of certain herpesviruses is their ability to induce rapid and widespread RNA decay in order to gain access to host resources. This phenotype is induced by viral endoribonucleases, including SOX in KSHV, muSOX in MHV68, BGLF5 in EBV and vhs in HSV-1. Here, we performed comparative RNA-seq upon expression of these herpesviral endonucleases in order to characterize their effect on the host transcriptome. Consistent with previous reports, we found that approximately two thirds of transcripts are downregulated in cells expressing any of these viral endonucleases. Among transcripts spared from degradation, we uncovered a cluster of transcripts that systematically escape degradation from all tested endonucleases. Among these escapees, we identified C19ORF66 and reveal that like the previously identified escapees, this transcript is protected from degradation by its 3\u2019UTR. We then show that C19ORF66, a known anti-viral protein, is a potent KSHV restriction factor, suggesting that its ability to escape viral cleavage may be an important component of the host response to viral infection. Collectively, our comparative approach is a powerful tool to pinpoint key regulators of the viral-host interplay and led us to uncover a novel KSHV regulator.","Human infections with severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) cause coronavirus disease 19 (COVID-19) and there is currently no cure. The 3C-like protease (3CLpro), a highly conserved protease indispensable for replication of coronaviruses, is a promising target for development of broad-spectrum antiviral drugs. To advance the speed of drug discovery and development, we investigated the inhibition of SARS-CoV-2 3CLpro by natural products derived from Chinese traditional medicines. Baicalin and baicalein were identified as the first non-covalent, non-peptidomimetic inhibitors of SARS-CoV-2 3CLpro and exhibited potent antiviral activities in a cell-based system. Remarkably, the binding mode of baicalein with SARS-CoV-2 3CLpro determined by X-ray protein crystallography is distinctly different from those of known inhibitors. Baicalein is perfectly ensconced in the core of the substrate-binding pocket by interacting with two catalytic residues, the crucial S1/S2 subsites and the oxyanion loop, acting as a 'shield' in front of the catalytic dyad to prevent the peptide substrate approaching the active site. The simple chemical structure, unique mode of action, and potent antiviral activities in vitro, coupled with the favorable safety data from clinical trials, emphasize that baicalein provides a great opportunity for the development of critically needed anti-coronaviral drugs.","AbstractChina reported zero locally-acquired malaria cases in 2017 and 2018. Understanding the spatio-temporal pattern underlying this decline, especially the relationship between locally-acquired and imported cases, can inform efforts to maintain elimination and prevent re-emergence. This is particularly pertinent in Yunnan province, where the potential for local transmission is highest. Using a geo-located individual-level dataset of cases recorded in Yunnan province between 2011 and 2016, we jointly estimate the case reproduction number, Rc, and the number of unobserved sources of infection. We use these estimates within spatio-temporal geostatistical models to map how transmission varied over time and space, estimate the timeline to elimination and the risk of resurgence. Our estimates suggest that, maintaining current intervention efforts, Yunnan is unlikely to experience sustained local transmission up to 2020. However, even with a mean Rc of 0.005 projected for the year 2019, locally-acquired cases are possible due to high levels of importation.","AbstractSecondary bacterial pneumonia following influenza infection is a significant cause of mortality worldwide. Upper respiratory tract pneumococcal carriage is important as both determinants of disease and population transmission. The immunological mechanisms that contain pneumococcal carriage are well-studied in mice but remain unclear in humans. Loss of this control of carriage following influenza infection is associated with secondary bacterial pneumonia during seasonal and pandemic outbreaks. We used a human type 6B pneumococcal challenge model to show that carriage acquisition induces early degranulation of resident neutrophils and recruitment of monocytes to the nose. Monocyte function associated with clearance of pneumococcal carriage. Prior nasal infection with live attenuated influenza virus induced inflammation, impaired innate function and altered genome-wide nasal gene responses to pneumococcal carriage. Levels of the cytokine IP-10 promoted by viral infection at the time of pneumococcal encounter was positively associated with bacterial density. These findings provide novel insights in nasal immunity to pneumococcus and viral-bacterial interactions during co-infection.","AbstractDelivery of native or chemically modified recombinant proteins into mammalian cells shows promise for functional investigations and various technological applications, but concerns that sub-cellular localization and functional integrity of delivered proteins may be affected remain high. Here, we surveyed batch electroporation as a delivery tool for single polypeptides and multi-subunit protein assemblies of kinetochores, a spatially confined and well-studied subcellular structures. After electroporation in human cells, recombinant fluorescent Ndc80 and Mis12 multi-subunit complexes displayed native localization, physically interacted with endogenous binding partners, and functionally complemented depleted endogenous counterparts to promote mitotic checkpoint signaling and chromosome segregation. Farnesylation is required for kinetochore localization of the Dynein adaptor Spindly. In cells with chronically inhibited farnesyl transferase activity, in vitro farnesylation and electroporation reconstituted robust kinetochore localization of Spindly. Thus, electroporation is uniquely versatile for delivering synthetic and, as required, chemically modified functional mimics of endogenous proteins, and is therefore a promising tool for chemical and synthetic biology.","AbstractPompe disease results from a defect in human acid \u03b1-glucosidase (GAA), a lysosomal enzyme that cleaves terminal \u03b11-4 and \u03b11-6 glucose from glycogen. In Pompe disease (also known as Glycogen Storage Disorder type II), the accumulation of undegraded glycogen in lysosomes leads to cellular dysfunction, primarily in muscle and heart tissues. Pompe disease is an active candidate of clinical research, with pharmacological chaperone therapy tested and enzyme replacement therapy approved. Despite production of large amounts of recombinant GAA annually, the structure of GAA has not been reported until now. Here, we describe the first structure of GAA, at 1.7\u00c5 resolution. Three structures of GAA complexes reveal the molecular basis for the hundreds of mutations that lead to Pompe disease and for pharmacological chaperoning in the protein. The GAA structure reveals a surprising second sugar-binding site 34\u00c5 from the active site, suggesting a possible mechanism for processing of large glycogen substrates. Overall, the structure will assist in the design of next-generation treatments for Pompe disease.","ABSTRACTDue to the shortage of masks during the pandemic, we recently demonstrated that household textiles are effective environmental droplet barriers (EDBs) with identical droplet retention potential as medical masks. To further promote the implementation of a universal community droplet reduction solution based on a synchronized encouragement/enforcement of mask utilization by the public based on widely available textiles (mask fabrication without the need for sewing machines), here we conducted a study using germ-free mice to determine to what extent textiles were effective in vivo. Using a bacterial-suspension spray simulation model of droplet ejection (mimicking a sneeze), we quantified the extent by which 100% cotton textile prevented the contamination of germ-free animals on the other side of the textile-barrier (simulating a properly worn mask). Of relevance, all mice protected with textiles remained germ-free after two sprays (inoculation dose: &gt;600 bacterial droplet units per 56.75cm2) compared to the contamination of mice not protected by a textile (0/12 vs 6/6, Fisher\u2019s exact, p&lt;0.0001). In a second phase of the experiment with 12 germ-free mice exposed again to 10-fold more droplets remained germ-free, while 100% of mice at 180cm became colonized with a single spray (0/8 vs 4/4, Fisher exact, p=0.002). Collectively, barriers protected all mice (even with low-density textiles, heavy vs. light fabric, T-test, p=0.0028) when using textile-EDB to cover the cages (0/20 vs 10/10, Fisher exact, p&lt;0.0001). This study demonstrated, in vivo, that widely available household textiles are 100% effective at preventing contamination of the environment and the exposed animals by microbe-carrying droplets.","AbstractMotivation: Detecting protein domains sharing no similarity to known domains, as stored in domain databases, is a challenging problem, particularly for unannotated proteomes, domains emerged recently, fast diverging proteins or domains with intrinsically disordered regions.Results: We developed pyHCA and HCAtk, a python API and standalone tool gathering together improved versions of previously developed methodologies, with new functionalities. The developed tools can be either used from command line or from a python API.Availability: HCAtk and pyHCA are available at https://github.com/T-B-F/pyHCA under the CeCILL-C license.","AbstractPorcine epidemic diarrhea virus (PEDV) is a highly contagious pathogenic virus that causes severe diarrhea and dehydration in pigs of all ages. Deoxynivalenol (DON), the most abundant trichothecene in food and feed, causes vomit and diarrhea in animals and human. However, whether DON exposure could affect PEDV infection remains unknown. Herein, we investigated the impacts of DON on entry and replication of PEDV, morbidity situation of piglets and the mechanisms involved. In vivo, twenty-seven piglets infected naturally with PEDV were randomly divided into three groups, receiving the basal diet containing 0, 750 and 1500 \u03bcg/kg DON, respectively. We observed significant increases in the diarrhea rates, the villous injury of jejunums and the PEDV proliferation of duodenum, jejunum, ileum and mesenterium of piglets in experimental groups compared with control. Additionally, the autophagosome-like vesicles and the autophagy-related protein expressions were also increased in experimental groups. In vitro, we observed that, approximately 2 hrs post-infection, 0.1, 0.5 and 1.0 \u03bcM DON promoted PEDV entry (P &lt; 0.05) in IPEC-J2s and resulted in tight junction protein occludin internalization. Knockdown of occludin and CRISPR-Cas9-mediated knockout of LC3B indicated a vital role of autophagy-induced occludin internalization in DON-promoted PEDV entry. We also observed that, 24 hrs post-infection, a significant increase in PEDV replication after 0.1, 0.5 and 1.0 \u03bcM DON treatment, along with the induction of a complete autophagy. Specifically, deletion of LC3B indicated a crucial role of autophagy in DON-promoted PEDV replication. Pretreatment with SB202190, a p38 signaling inhibitor, abolished the induction of autophagy. Furthermore, downregulation of type I interferon revealed that DON contributed PEDV to escape innate immune. Mechanistically, DON-caused innate immune escape was related to the upregulation of LC3B, which further inhibited STING phosphorylation. Taken together, DON could promote PEDV infection by inducing occludin internalization and innate immune escape via triggering p38-mediated autophagy.Author summaryPorcine epidemic diarrhea (PED), a devastating enteric disease, leads to catastrophic economic loss to the global pig industry. Its primary pathogen is the coronavirus PED virus (PEDV). Growing evidence indicates that pathogen infection is not the only factor of PED outbreaks, other non-infectious factors is also related to this disease. We guessed some ubiquitous substances, such as deoxynivalenol (DON), that lead to pig intestinal epithelial cell stress might encourage the progress and spread of PED. In the present study, the weaning piglets infected naturally with PEDV and the IPEC-J2 cell line were selected as models to explore the effects of DON on PEDV infection, morbidity and gut barrier. Our results showed that DON exposure can promote PEDV infection in vitro and in vivo, and the underlying mechanism might be related to LC3B-mediated autophagy. Our findings reveal new pathways for developing potential novel antiviral strategies against PEDV infection.","AbstractRecently emerged Wuhan\u2019s novel coronavirus designated as SARS-CoV-2, a causative agent of coronavirus disease 2019 (COVID-19) is rapidly spreading its pathogenicity throughout the world now. More than 4000 mortalities have occurred worldwide till the writing of this article and this number is increasing every passing hour. World Health Organization (WHO) has declared it as a global public health emergency. The multiple sequence alignment data correlated with already published reports on SARS-CoV-2 indicated that it is closely related to Bat-Severe Acute Respiratory Syndrome like coronavirus (Bat CoV SARS-like) and well-studied Human SARS. In this study, we have exploited the complementary approach to examine the intrinsically disordered regions in proteome of SARS-CoV-2 using Bat SARS-like and Human SARS CoVs as comparative models. According to our findings, SARS-CoV-2 proteome contains a significant amount of ordered proteins except Nucleocapsid, Nsp8, and ORF6. Further, cleavage sites in replicase 1ab polyprotein are found to be highly disordered. We have extensively investigated the dark proteome in SARS-CoV-2 which will have implications for the structured and unstructured biology of SARS or SARS-like coronaviruses.SignificanceThe infection caused by Novel Coronavirus (SARS-CoV-2) is responsible for the current pandemic that cause severe respiratory disease and pneumonia-like infection in humans. Currently, there is no such in-depth information on protein structure and function available in public domain and no effective anti-viral or vaccines are available for the treatment of this infection. Our study provides comparative order and disorder-based proteome information with Human SARS and Bat CoV that may be useful for structure-based drug discovery.","The global pandemic of Coronavirus disease 2019 (COVID-19) is a disaster for human society. A convenient and reliable in vitro neutralization assay is very important for the development of neutralizing antibodies, vaccines and other inhibitors. In this study, G protein-deficient vesicular stomatitis virus (VSVdG) bearing full-length and truncated spike (S) protein of SARS-CoV-2 were evaluated. The virus packaging efficiency of VSV-SARS-CoV-2-Sdel18 (S with C-terminal 18 amino acid truncation) is much higher than VSV-SARS-CoV-2-S. A neutralization assay for antibody screening and serum neutralizing titer quantification was established based on VSV-SARS-CoV-2-Sdel18 pseudovirus and human angiotensin-converting enzyme 2 (ACE2) overexpressed BHK21 cell (BHK21-hACE2). The experimental results can be obtained by automatically counting EGFP positive cell number at 12 hours after infection, making the assay convenient and high-throughput. The serum neutralizing titer of COVID-19 convalescent patients measured by VSV-SARS-CoV-2-Sdel18 pseudovirus assay has a good correlation with live SARS-CoV-2 assay. Seven neutralizing monoclonal antibodies targeting receptor binding domain (RBD) of SARS-CoV-2-S were obtained. This efficient and reliable pseudovirus assay model could facilitate the development of new drugs and vaccines.","AbstractMost emerging pathogens can infect multiple species, underscoring the importance of understanding the ecological and evolutionary factors that allow some hosts to harbor greater infection prevalence and share pathogens with other species. However, our understanding of pathogen jumps is primarily based around viruses, despite bacteria accounting for the greatest proportion of zoonoses. Because bacterial pathogens in bats (Order: Chiroptera) can have conservation and human health consequences, studies that examine the ecological and evolutionary drivers of bacterial prevalence and barriers to pathogen sharing are crucially needed. We here studied hemotropic Mycoplasma spp. (i.e., hemoplasmas) across a species-rich bat community in Belize over two years. Across 469 bats spanning 33 species, half of individuals and two-thirds of species were hemoplasma positive. Infection prevalence was higher for males and for species with larger body mass and colony sizes. Hemoplasmas displayed high genetic diversity (21 novel genotypes) and strong host specificity. Evolutionary patterns supported co-divergence of bats and bacterial genotypes alongside phylogenetically constrained host shifts. Bat species centrality to the network of shared hemoplasma genotypes was phylogenetically clustered and unrelated to prevalence, further suggesting rare\u2014but detectable\u2014bacterial sharing between species. Our study highlights the importance of using fine phylogenetic scales when assessing host specificity and suggests phylogenetic similarity may play a key role in host shifts for not only viruses but also bacteria. Such work more broadly contributes to increasing efforts to understand cross-species transmission and epidemiological consequences of bacterial pathogens.","ABSTRACTThe virus that causes COVID-19, SARS-CoV-2, has a large RNA genome that encodes numerous proteins that might be targets for antiviral drugs. Some of these proteins, such as the RNA-dependent RNA polymers, helicase and main protease, are well conserved between SARS-CoV-2 and the original SARS virus, but several others are not. This study examines one of the most novel proteins encoded by SARS-CoV-2, a macrodomain of nonstructural protein 3 (nsp3). Although 26% of the amino acids in this SARS-CoV-2 macrodomain differ from those seen in other corona-viruses, the protein retains the ability to bind ADP-ribose, which is an important characteristic of beta coronaviruses, and potential therapeutic target.","ABSTRACTStopping COVID-19 is a priority worldwide. Understanding which cell types are targeted by SARS-CoV-2 virus, whether interspecies differences exist, and how variations in cell state influence viral entry is fundamental for accelerating therapeutic and preventative approaches. In this endeavor, we profiled the transcriptome at single-cell resolution of nine tissues from a Macaca fascicularis monkey. The distribution of SARS-CoV-2 facilitators, ACE2 and TMRPSS2, in different cell subtypes showed substantial heterogeneity across lung, kidney, thyroid and liver. Co-expression analysis identified immunomodulatory proteins such as IDO2 and ANPEP as potential SARS-CoV-2 targets responsible for immune cell exhaustion. Furthermore, single-cell chromatin accessibility analysis of the kidney unveiled a plausible link between IL6-mediated innate immune responses aiming to protect tissue and enhanced ACE2 expression that could promote viral entry. Our work constitutes a unique resource for understanding SARS-CoV-2 pathophysiology in two phylogenetically close species, which might guide in the development of effective treatments in humans.Bullet pointsWe used a single-cell transcriptome atlas of 9 monkey tissues to study COVID-19.ACE2+TMPRSS2+ epithelial cells of lung, kidney and liver are targets for SARS-CoV-2.ACE2 correlation analysis shows IDO2 and ANPEP as potential therapeutic opportunities.We unveil a link between IL6, STAT transcription factors and boosted SARS-CoV-2 entry.","AbstractLectins are glycan-binding proteins with no catalytic activity and ubiquitously expressed in nature. Numerous bacteria employ lectins to efficiently bind to epithelia, thus facilitating tissue colonisation. Wounded skin is one of the preferred niches for Pseudomonas aeruginosa, which has developed diverse strategies to impair tissue repair processes and promote infection.Here, we analyse the effect of the P. aeruginosa fucose-binding lectin LecB on human keratinocytes and demonstrate that it triggers events in the host, upon binding to fucosylated residues on cell membrane receptors, that extend beyond its role as an adhesion molecule. We found that LecB associates with several growth factor receptors and dampens their signalling pathways, leading to the arrest of cell cycle. Additionally, we describe a novel LecB-triggered mechanism to downregulate host cell receptors by showing that LecB leads to insulin-like growth factor receptor 1 internalisation, without receptor activation, and subsequent missorting towards intracellular endosomal compartments.Overall, these data highlight that LecB is a multitask virulence factor that, through subversion of several host pathways, has a profound impact on keratinocyte proliferation and survival.","There is an ongoing worldwide coronavirus disease 2019 (Covid-19) pandemic caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). At present, confirmatory diagnosis is by reverse transcription polymerase chain reaction (RT-PCR), typically taking several hours and requiring a molecular laboratory to perform. There is an urgent need for rapid, simplified and cost-effective detection methods. We have developed and analytically validated a protocol for direct rapid extraction-free PCR (DIRECT-PCR) detection of SARS-CoV-2 without the need for nucleic acid purification. As few as 6 RNA copies per reaction of viral nucleocapsid (N) gene from respiratory samples such as sputum and nasal exudate can be detected directly using our one-step inhibitor-resistant assay. The performance of this assay was validated on a commercially available portable PCR thermocycler. Viral lysis, reverse transcription, amplification and detection are achieved in a single-tube homogeneous reaction within 36 minutes. This minimized hands-on time, reduces turnaround-time for sample-to-result and obviates the need for RNA purification reagents. It could enable wider use of Covid-19 testing for diagnosis, screening and research in countries and regions where laboratory capabilities are limiting.","Pathogenic human coronavirus infections, such as severe acute respiratory syndrome CoV (SARS-CoV) and Middle East respiratory syndrome CoV (MERS-CoV), cause high morbidity and mortality 1,2. Recently, a severe pneumonia-associated respiratory syndrome caused by a new coronavirus was reported at December 2019 (2019-nCoV) in the city Wuhan, Hubei province, China3\u20135, which was also named as pneumonia-associated respiratory syndrome (PARS)6. Up to 9th of February 2020, at least 37, 251 cases have been reported with 812 fatal cases according to the report from China CDC. However, the immune mechanism that potential orchestrated acute mortality from patients of 2019-nCoV is still unknown. Here we show that after the 2019-nCoV infection, CD4+T lymphocytes are rapidly activated to become pathogenic T helper (Th) 1 cells and generate GM-CSF etc. The cytokines environment induces inflammatory CD14+CD16+ monocytes with high expression of IL-6 and accelerates the inflammation. These aberrant and excessive immune cells may enter the pulmonary circulation in huge numbers and play an immune damaging role to causing lung functional disability and quick mortality. Our results demonstrate that excessive non-effective host immune responses by pathogenic T cells and inflammatory monocytes may associate with severe lung pathology. Therefore, we suggest that monoclonal antibody that targets the GM-CSF or interleukin 6 receptor may potentially curb immunopathology caused by 2019-nCoV and consequently win more time for virus clearance.","AbstractThe rapid sharing of sequence information as seen throughout the current SARS-CoV-2 epidemic, represents an inflection point for genomic epidemiology. Here we describe aspects of coronavirus evolutionary genetics revealed from these data, and provide the first direct RNA sequence of SARS-CoV-2, detailing coronaviral subgenome-length mRNA architecture.","AbstractSince December 2019, the outbreak of Coronavirus Disease 2019 (COVID-19) spread from Wuhan, China to the world, it has caused more than 87,000 diagnosed cases and more than 3,000 deaths globally. To fight against COVID-19, we carried out research for the near native SARS-CoV-2 and report here our preliminary results obtained. The pathogen of the COVID-19, the native SARS-CoV-2, was isolated, amplified and purified in a BSL-3 laboratory. The whole viral architecture of SARS-CoV-2 was examined by transmission electron microscopy (both negative staining and cryo-EM). We observed that the virion particles are roughly spherical or moderately pleiomorphic. Spikes have nail-like shape towards outside with a long body embedded in the envelope. The morphology of virion observed in our result indicates that the S protein of SARS-CoV-2 is in post-fusion state, with S1 disassociated. This state revealed by cryo-EM first time could provide an important information for the identification and relevant clinical research of this new coronavirus.","AbstractInfectious-disease outbreaks are often characterized by the reproductive number and \ud835\udce1 exponential rate of growth r. \ud835\udce1 provides information about out-break control and predicted final size. Directly estimating \ud835\udce1 is difficult, while r can often be estimated from incidence data. These quantities are linked by the generation interval \u2013 the time between when an individual is infected by an infector, and when that infector was infected. It is often infeasible to ob-tain the exact shape of a generation-interval distribution, and to understand how this shape affects estimates of \ud835\udce1. We show that estimating generation interval mean and variance provides insight into the relationship between and \ud835\udce1 r. We use examples based on Ebola, rabies and measles to explore approximations based on gamma-distributed generation intervals, and find that use of these simple approximations are often sufficient to capture the r\u2013\ud835\udce1 relationship and provide robust estimates of \ud835\udce1.","AbstractIn this work, we use a within-host viral dynamic model to describe the SARS-CoV-2 kinetics in host. Chest radiograph score data are used to estimate the parameters of that model. Our result shows that the basic reproductive number of SARS-CoV-2 in host growth is around 3.79. Using the same method we also estimate the basic reproductive number of MERS virus is 8.16 which is higher than SARS-CoV-2. The PRCC method is used to analyze the sensitivities of model parameters and the drug effects on virus growth are also implemented to analyze the model.","AbstractSince December 2019, coronavirus disease 2019 (COVID-19) emerged in Wuhan, Central China and rapidly spread throughout China. Up to March 3, 2020, SARS-CoV-2 has infected more than 89,000 people in China and other 66 countries across six continents. In this study, we used 10 new sequenced genomes of SARS-CoV-2 and combined 136 genomes from GISAID database to investigate the genetic variation and population demography through different analysis approaches (e.g. Network, EBSP, Mismatch, and neutrality tests). The results showed that 80 haplotypes had 183 substitution sites, including 27 parsimony-informative and 156 singletons. Sliding window analyses of genetic diversity suggested a certain mutations abundance in the genomes of SARS-CoV-2, which may be explaining the existing widespread and high adaptation of the deadly virus. Phylogenetic analysis showed that the view, pangolin acted as an intermediate host, may be controversial. The network indicated that, in the original haplotype (H14), one patient sample lived near the Huanan seafood market (approximate 2 km), indicating high possibility of the patient having a history of unconscious contact with this market. However, based on this clue, we cannot accurately concluded that whether this market was the origin center of SARS-CoV-2. Additionally, 16 genomes, collected from this market, assigned to 10 haplotypes, indicated a circulating infection within the market in a short term and then leading to the outbreak of SARS-CoV-2 in Wuhan and other areas. The EBSP results showed that the first estimated expansion date of SARS-CoV-2 began from 7 December 2019, which may indicated that the transmission could have begun from person to person in mid to late November.","Compound heterozygotes occur when different mutations at the same locus on both maternal and paternal chromosomes produce a recessive trait. Here we present the tool VarCount for the quantification of mutations at the individual level. We used VarCount to characterize compound heterozygous coding variants in patients with epileptic encephalopathy and in the 1000 genomes participants. The Epi4k data contains variants identified by whole exome sequencing in patients with either Lennox-Gastaut Syndrome (LGS) or Infantile Spasms (IS), as well as their parents. We queried the Epi4k dataset (264 trios) and the phased 1000 genomes data (2504 participants) for recessive variants. To assess enrichment, transcript counts were compared between the Epi4k and 1000 genomes participants using minor allele frequency (MAF) cutoffs of 0.5% and 1.0%, and including all ancestries or only probands of European ancestry. In the Epi4k participants, we found enrichment for rare, compound heterozygous mutations in six genes, including three involved in neuronal growth and development; PRTG (p=0.00086, 1% MAF, combined ancestries), TNC (p=0.0221% MAF, combined ancestries), and MACF1 (p=0.0245, 0.5% MAF, EU ancestry). Due the total number of transcripts considered in these analyses, the enrichment detected was not significant after correction for multiple testing and higher powered or prospective studies are necessary to validate the candidacy of these genes. However, PRTG, TNC, and MACF1 are potential novel recessive epilepsy genes and our results highlight that compound heterozygous mutations should be considered in sporadic epilepsy.","ABSTRACTThe novel betacoronavirus (SARS-CoV-2) is highly contagious and can cause serious acute respiratory illness syndromes, often fatal, called covid-19. It is an urgent priority to better understand SARS-CoV-2 infection mechanisms that will help in the development of prophylactic vaccines and therapeutics that are very important to people health and socioeconomic stability around the world. The surface coronavirus spike (S) glycoprotein is considered as a key factor in host specificity because it mediates infection by receptor-recognition and membrane fusion. Here the analysis of CoV-2 S protein revealed in S1subunit a B56-binding LxxIxE-like motif that could recruit the host protein phosphatase 2A (PP2A). This motif is absent in SARS-CoV and MERS-CoV. PP2A is a major family of serine/threonine phosphatases in eukaryotic cells. Phosphatases and kinases are big players in the regulation of pro-inflammatory responses during pathogenic infections. Moreover, studies have shown that viruses use multiple strategies to target PP2A in order to manipulate host\u2019s antiviral responses. The latest studies have indicated that SARS-CoV-2 is involved in sustained inflammation in the host. Therefore, by controlling acute inflammation; it is possible to eliminate its dangerous effects on the host. Among efforts to fight covid-19, the interaction between LxxIxE-like motif and PP2A-B56-binding pocket could be a target for the development of a bioactive peptide and ligand inhibitors for therapeutic purposes.","ABSTRACTThe cutting-edge technology vaccinomics is the combination of two topics immunogenetics and immunogenomics with the knowledge of systems biology and immune profiling for designing vaccine against infectious disease. In our present study, an epitope-based peptide vaccine against nonstructural protein 4 of beta coronavirus, using a combination of B cell and T cell epitope predictions, followed by molecular docking methods are performed. Here, protein sequences of homologous nonstructural protein 4 of beta coronavirus are collected and conserved regions present in them are investigated via phylogenetic study to determine the most immunogenic part of protein. From the identified region of the target protein, the peptide sequence IRNTTNPSAR from the region ranging from 38-47 and the sequence PTDTYTSVYLGKFRG from the positions of 76-90 are considered as the most potential B cell and T cell epitopes respectively. Furthermore, this predicted T cell epitopes PTDTYTSVY and PTDTYTSVYLGKFRG interacted with MHC allelic proteins HLA-A*01:01 and HLA-DRB5*01:01 respectively with the low IC50 values. These epitopes are perfectly fitted into the epitope binding grooves of alpha helix of MHC I molecule and MHC II molecule with binding energy scores \u2212725.0 Kcal/mole and \u2212786.0 Kcal/mole respectively, showing stability in MHC molecules binding. This MHC restricted epitope PTDTYTSVY also showed a good conservancy of 50.16% in world population coverage. This MHC I HLA-A*01:01 allele is present among 58.87% of Chinese population also. Therefore, the epitopes IRNTTNPSAR and PTDTYTSVYLGKFRG may be considered as potential peptides for peptide-based vaccine for coronavirus after further experimental study.","Direct RNA sequencing using an Oxford Nanopore MinION characterised the transcriptome of SARS-CoV-2 grown in Vero E6 cells. This cell line is being widely used to propagate the novel coronavirus. The viral transcriptome was analysed using a recently developed ORF-centric pipeline. This revealed the pattern of viral transcripts, (i.e. subgenomic mRNAs), generally fitted the predicted replication and transcription model for coronaviruses. A 24 nt in-frame deletion was detected in subgenomic mRNAs encoding the spike (S) glycoprotein. This feature was identified in over half of the mapped transcripts and was predicted to remove a proposed furin cleavage site from the S glycoprotein. This motif directs cleavage of the S glycoprotein into functional subunits during virus entry or exit. Cleavage of the S glycoprotein can be a barrier to zoonotic coronavirus transmission and affect viral pathogenicity. Allied to this transcriptome analysis, tandem mass spectrometry was used to identify over 500 viral peptides and 44 phosphopeptides, covering almost all of the proteins predicted to be encoded by the SARS-CoV-2 genome, including peptides unique to the deleted variant of the S glycoprotein. Detection of an apparently viable deletion in the furin cleavage site of the S glycoprotein reinforces the point that this and other regions of SARS-CoV-2 proteins may readily mutate. This is of clear significance given the interest in the S glycoprotein as a potential vaccine target and the observation that the furin cleavage site likely contributes strongly to the pathogenesis and zoonosis of this virus. The viral genome sequence should be carefully monitored during the growth of viral stocks for research, animal challenge models and, potentially, in clinical samples. Such variations may result in different levels of virulence and morbidity and mortality.","C3A is a sub-clone of human hepatoblastoma HepG2 cell line with the strong contact inhibition of growth. We fortuitously found that C3A was more susceptible to human coronavirus HCoV-OC43 infection than HepG2, which was attributed to the increased efficiency of virus entry into C3A cells. In an effort to search for the host cellular protein(s) mediating the differential susceptibility of the two cell lines to HCoV-OC43 infection, we found that ADAP2, GILT and LY6E, three cellular proteins with known activity of interfering virus entry, expressed at significantly higher levels in HepG2 cells. Functional analyses revealed that ectopic expression of LY6E, but not GILT or ADAP2, in HEK 293 cells inhibited the entry of HCoV-OC43. While overexpression of LY6E in C3A and A549 cells efficiently inhibited the infection of HCoV-OC43, knockdown of LY6E expression in HepG2 significantly increased its susceptibility to HCoV-OC43 infection. Moreover, we found that LY6E also efficiently restricted the entry mediated by the envelope spike proteins of other human coronaviruses, including the currently pandemic SARS-CoV-2. Interestingly, overexpression of serine protease TMPRSS2 or amphotericin treatment significantly neutralized the IFITM3 restriction of human coronavirus entry, but did not compromise the effect of LY6E on the entry of human coronaviruses. The work reported herein thus demonstrates that LY6E is a critical antiviral immune effector that controls CoV infection and pathogenesis via a distinct mechanism.","In an outbreak of an emerging disease the epidemiological characteristics of the pathogen may be largely unknown. A key determinant of ability to control the outbreak is the relative timing of infectiousness and symptom onset. We provide a method for identifying this relationship with high accuracy based on data from household-stratified symptom-onset data. Further, this can be achieved with observations taken on only a few specific days, chosen optimally, within each household. This constitutes an important tool for outbreak response. An accurate and computationally-efficient heuristic for determining the optimal surveillance scheme is introduced. This heuristic provides a novel approach to optimal design for Bayesian model discrimination.","The discovery of epitopes is helpful to the development of SARS-CoV-2 vaccine. The sequences of the surface protein of SARS-CoV-2 and its proximal sequences were obtained by BLAST, the sequences of the whole genome of SARS-CoV-2 were obtained from the GenBank. Based on the NCBI Reference Sequence: NC_045512.2, the conformational and linear B cell epitopes of the surface protein were predicted separately by various prediction methods. Furthermore, the conservation of the epitopes, the adaptability and other evolutionary characteristics were also analyzed. 7 epitopes were predicted, including 5 linear epitopes and 2 conformational epitopes, one of the linear and one of the conformational were coincide. The epitope D mutated easily, but the other epitopes were very conservative and the epitope C was the most conservative. It is worth mentioning that all of the 6 dominated epitopes were absolutely conservative in nearly 1000 SARS-CoV-2 genomes, and they deserved further study. The findings would facilitate the vaccine development, had the potential to be directly applied on the treatment in this disease, but also have the potential to prevent the possible threats caused by other types of coronavirus.","Detection of pathogenic sequences or variants in DNA and RNA through a point-of-care diagnostic approach is valuable for rapid clinical prognosis. In recent times, CRISPR based detection of nucleic acids has provided an economical and quicker alternative to sequencing-based platforms which are often difficult to implement in the field. Here, we present FnCas9 Editor Linked Uniform Detection Assay (FELUDA) that employs a highly accurate enzymatic readout for detecting nucleotide sequences, identifying nucleobase identity and inferring zygosity with precision. We demonstrate that FELUDA output can be adapted to multiple signal detection platforms and can be quickly designed and deployed for versatile applications including rapid diagnosis during infectious disease outbreaks like COVID-19.","AbstractBackgroundThe ongoing outbreak of COVID-19 has spread rapidly and sparked global concern. While the transmission of SARS-CoV-2 through human respiratory droplets and contact with infected persons is clear, the aerosol transmission of SARS-CoV-2 has been little studied.MethodsThirty-five aerosol samples of three different types (total suspended particle, size segregated and deposition aerosol) were collected in Patient Areas (PAA) and Medical Staff Areas (MSA) of Renmin Hospital of Wuhan University (Renmin) and Wuchang Fangcang Field Hospital (Fangcang), and Public Areas (PUA) in Wuhan, China during COVID-19 outbreak. A robust droplet digital polymerase chain reaction (ddPCR) method was employed to quantitate the viral SARS-CoV-2 RNA genome and determine aerosol RNA concentration.ResultsThe ICU, CCU and general patient rooms inside Renmin, patient hall inside Fangcang had undetectable or low airborne SARS-CoV-2 concentration but deposition samples inside ICU and air sample in Fangcang patient toilet tested positive. The airborne SARS-CoV-2 in Fangcang MSA had bimodal distribution with higher concentration than those in Renmin during the outbreak but turned negative after patients number reduced and rigorous sanitization implemented. PUA had undetectable airborne SARS-CoV-2 concentration but obviously increased with accumulating crowd flow.ConclusionsRoom ventilation, open space, proper use and disinfection of toilet can effectively limit aerosol transmission of SARS-CoV-2. Gathering of crowds with asymptomatic carriers is a potential source of airborne SARS-CoV-2. The virus aerosol deposition on protective apparel or floor surface and their subsequent resuspension is a potential transmission pathway and effective sanitization is critical in minimizing aerosol transmission of SARS-CoV-2.","Cardiovascular disease is the leading cause of death worldwide. Advanced insights into disease mechanisms and strategies to improve therapeutic opportunities require deeper understanding of the molecular processes of the normal heart. Knowledge of the full repertoire of cardiac cells and their gene expression profiles is a fundamental first step in this endeavor. Here, using large-scale single cell and nuclei transcriptomic profiling together with state-of-the-art analytical techniques, we characterise the adult human heart cellular landscape covering six anatomical cardiac regions (left and right atria and ventricles, apex and interventricular septum). Our results highlight the cellular heterogeneity of cardiomyocytes, pericytes and fibroblasts, revealing distinct subsets in the atria and ventricles indicative of diverse developmental origins and specialized properties. Further we define the complexity of the cardiac vascular network which includes clusters of arterial, capillary, venous, lymphatic endothelial cells and an atrial-enriched population. By comparing cardiac cells to skeletal muscle and kidney, we identify cardiac tissue resident macrophage subsets with transcriptional signatures indicative of both inflammatory and reparative phenotypes. Further, inference of cell-cell interactions highlight a macrophage-fibroblast-cardiomyocyte network that differs between atria and ventricles, and compared to skeletal muscle. We expect this reference human cardiac cell atlas to advance mechanistic studies of heart homeostasis and disease.","AbstractSevere fever with thrombocytopenia syndrome (SFTS) is an emerging infectious disease localized to China, Japan, and Korea that is characterized by severe hemorrhage and a high fatality rate. Currently, no specific vaccine or treatment has been approved for this disease. To develop a therapeutic agent for SFTS, we isolated antibodies from a phage-displayed antibody library that was constructed from a patient who recovered from SFTS virus (SFTSV) infection. One antibody, designated as Ab10, was reactive to the Gn envelope glycoprotein of SFTSV and protected host cells and A129 mice from infection in both in vitro and in vivo experiments. Notably, Ab10 protected 80% of mice, even when injected 5 days after inoculation with a lethal dose of SFTSV. Using cross-linker assisted mass spectrometry and alanine scanning, we located the non-linear epitope of Ab10 on the Gn glycoprotein domain II and an unstructured stem region, suggesting that Ab10 may inhibit a conformational alteration that is critical for cell membrane fusion between the virus and host cell. Ab10 reacted to recombinant Gn glycoprotein in Gangwon/Korea/2012, HB28, and SD4 strains. Additionally, based on its epitope, we predict that Ab10 binds the Gn glycoprotein in 247 of 272 reported SFTSV isolates previously reported. Together, these data suggest that Ab10 has potential to be developed into a therapeutic agent that could protect against more than 90% of reported SFTSV isolates.Author summarySevere fever with thrombocytopenia syndrome (SFTS) is an emerging infectious disease localized to China, Japan, and Korea. This tick-borne virus has infected more than 5,000 humans with a 6.4% to 20.9% fatality rate. Currently, there are no prophylactic or therapeutic measures against this virus. Historically, antibodies from patients who recovered from viral infection have been used to treat new patients. Until now, one recombinant monoclonal antibody was approved for the prophylaxis of respiratory syntial virus infection. We selected 10 antibodies from a patient who recovered from SFTS and found that one antibody potently inhibited SFTS viral infection in both test tube and animal studies. We determined the binding site of this antibody to SFTS virus, which allowed us to predict that this antibody could bind 247 out of 272 SFTS virus isolates reported up to now. We anticipate that this antibody could be developed into a therapeutic measure against SFTS.","When emerging pathogens encounter new host species for which they are poorly adapted, they must evolve to escape extinction. Pathogens experience selection on traits at multiple scales, including replication rates within host individuals and transmissibility between hosts. We analyze a stochastic model linking pathogen growth and competition within individuals to transmission between individuals. Our analysis reveals a new factor, the cross-scale reproductive number of a mutant virion, that quantifies how quickly mutant strains increase in frequency when they initially appear in the infected host population. This cross-scale reproductive number combines with viral mutation rates, single-strain reproductive numbers, and transmission bottleneck width to determine the likelihood of evolutionary emergence, and whether evolution occurs swiftly or gradually within chains of transmission. We find that wider transmission bottlenecks facilitate emergence of pathogens with short-term infections, but hinder emergence of pathogens exhibiting cross-scale selective conflict and long-term infections. Our results provide a framework to advance the integration of laboratory, clinical and field data in the context of evolutionary theory, laying the foundation for a new generation of evidence-based risk assessment of emergence threats.","Severe Acute Respiratory Syndrome coronavirus 2 (SARS-CoV-2) causes Coronavirus disease 2019 (COVID-19), a respiratory tract infection. The standard molecular diagnostic test is a multistep process involving viral RNA extraction and real-time quantitative reverse transcriptase PCR (qRT-PCR). Laboratories across the globe face constraints on equipment and reagents during the COVID-19 pandemic. We have developed a simplified qRT-PCR assay that removes the need for an RNA extraction process and can be run on a real-time thermal cycler. The assay uses custom primers and probes, and maintains diagnostic sensitivity within 98.0% compared to the assay run on a high-throughput, random-access automated platform, the Panther Fusion (Hologic). This assay can be used to increase capacity for COVID-19 testing for national programmes worldwide.","RNA viruses are responsible for a large variety of animal infections. Equine Arteritis Virus (EAV) is a positive single-stranded RNA virus member of the family Arteriviridae from the order Nidovirales like the Coronaviridae. EAV causes respiratory and reproductive diseases in equids. Although two vaccines are available, the vaccination coverage of the equine population is largely insufficient to prevent new EAV outbreaks around the world. In this study, we present a high-throughput in vitro assay suitable for testing candidate antiviral molecules on equine dermal cells infected by EAV. Using this assay, we identified three molecules that impair EAV infection in equine cells: the broad-spectrum antiviral and nucleoside analog ribavirin, and two compounds previously described as inhibitors of dihydroorotate dehydrogenase (DHODH), the fourth enzyme of the pyrimidine biosynthesis pathway. These molecules effectively suppressed cytopathic effects associated to EAV infection, and strongly inhibited viral replication and production of infectious particles. Since ribavirin is already approved in human and small animal, and that several DHODH inhibitors are in advanced clinical trials, our results open new perspectives for the management of EAV outbreaks.","AbstractHuman cytomegalovirus (HCMV) is the most common congenital infection worldwide, frequently causing hearing loss and brain damage in afflicted infants. A vaccine to prevent maternal acquisition of HCMV during pregnancy is necessary to reduce the incidence of infant disease. The glycoprotein B (gB) + MF59 adjuvant subunit vaccine platform is the most successful HCMV vaccine tested to-date, demonstrating approximately 50% efficacy in preventing HCMV acquisition in phase II trials. However, the mechanism of vaccine protection remains unknown. Plasma from 33 gB/MF59 vaccinees at peak immunogenicity was tested for gB epitope specificity as well as neutralizing and non-neutralizing anti-HCMV effector functions, and compared to an HCMV-seropositive cohort. gB/MF59 vaccination elicited IgG responses with gB-binding magnitude and avidity comparable to natural infection. Additionally, IgG subclass distribution was similar with predominant IgG1 and IgG3 responses induced by gB vaccination and HCMV infection. However, vaccine-elicited antibodies exhibited limited neutralization of the autologous virus, negligible neutralization of multiple heterologous strains, and limited binding responses against gB structural motifs targeted by neutralizing antibodies including AD-1, AD-2, and Domain I. Interestingly, vaccinees had high-magnitude IgG responses against AD-3 linear epitopes, demonstrating immunodominance against this non-neutralizing, cytosolic region. Finally, vaccine-elicited IgG robustly bound trimeric, membrane-associated gB on the surface of transfected or HCMV-infected cells and mediated virion phagocytosis, though were poor mediators of NK cell activation. Altogether, these data suggest that non-neutralizing antibody functions, including virion phagocytosis, likely played a role in the observed 50% vaccine-mediated protection against HCMV acquisition.SignificanceThe CDC estimates that every hour, a child is born in the United States with permanent neurologic disability resulting from human cytomegalovirus (HCMV) infection \u2013 more than is caused by Down syndrome, fetal alcohol syndrome, and neural tube defects combined. A maternal vaccine to block transmission of HCMV to the developing fetus is a necessary intervention to prevent these adverse outcomes. The gB/MF59 vaccine is the most successful tested clinically to-date, achieving 50% reduction in HCMV acquisition. This manuscript establishes the function and epitope specificity of the humoral response stimulated by this vaccine that may explain the partial vaccine efficacy. Understanding the mechanism of gB/MF59-elicited protective immune responses will guide rational design and evaluation of the next generation of HCMV vaccines.","AbstractInterferon-induced proteins with tetratricopeptide repeats (IFITs) are highly expressed during the cell-intrinsic immune response to viral infection. IFIT1 inhibits translation by binding directly to the 5\u2032 end of foreign RNAs, particularly those with non-self cap structures, precluding the recruitment of the cap-binding eukaryotic translation initiation factor 4F and subsequent 40S recruitment. Interaction of different IFIT family members is well described, but little is known of the molecular basis of IFIT association or its impact on function. Here, we reconstituted different complexes of IFIT1, IFIT2 and IFIT3 in vitro, which enabled us to reveal critical aspects of IFIT complex assembly. IFIT1 interacts rapidly and strongly with IFIT3 forming a stable heterotetramer. IFIT2 and IFIT3 homodimers dissociate to form a more stable heterodimer that associates with IFIT1, forming an IFIT1:IFIT2:IFIT3 trimer. Site-directed mutagenesis revealed a C-terminal \u2018YxxxL\u2019 motif in IFIT1 that mediates its association with IFIT3. Using various reporter mRNAs, we demonstrate for the first time that IFIT3 stabilises IFIT1 binding to cap0-mRNA and enhances its translation inhibition activity. Disrupting the binding interface between IFIT1 and IFIT3 abrogated this enhancement. This work reveals molecular aspects of IFIT assembly and provides an important \u2018missing link\u2019 between IFIT interaction and function.","Polyadenylation at the 3\u2019-end is a major regulator of messenger RNA and its length is known to affect nuclear export, stability and translation, among others. Only recently, strategies have emerged that allow for genome-wide poly(A) length assessment. These methods identify genes connected to poly(A) tail measurements indirectly by short-read alignment to genetic 3\u2019-ends. Concurrently Oxford Nanopore Technologies (ONT) established full-length isoform RNA sequencing containing the entire poly(A) tail. However, assessing poly(A) length through basecalling has so far not been possible due the inability to resolve long homopolymeric stretches in ONT sequencing.Here we presenttailfindr, an R package to estimate poly(A) tail length on ONT long-read sequencing data.tailfindroperates on unaligned, basecalled data. It measures poly(A) tail length from both native RNA and DNA sequencing, which makes poly(A) tail studies by full-length cDNA approaches possible for the first time. We assesstailfindr\u2019sperformance across different poly(A) lengths, demonstrating thattailfindris a versatile tool providing poly(A) tail estimates across a wide range of sequencing conditions.","We investigate the accumulated microbial and autoantigen antibody repertoire in adult-onset dermatomyositis patients sero-positive for TIF1\u03b3 (TRIM33) autoantibodies. We use an untargeted high-throughput approach which combines immunoglobulin disease-specific epitope-enrichment and identification of microbial and human antigens. Increased microbial diversity was observed in dermatomyositis. Viruses were over-represented and species of the Poxviridae family were significantly enriched. The autoantibodies identified recognised a large portion of the human proteome, including interferon regulated proteins; these proteins were clustered in specific biological processes. Apart from TRIM33, autoantibodies against eleven further TRIM proteins, including TRIM21, were identified. Some of these TRIM proteins shared epitope homology with specific viral species including poxviruses. Our data suggest antibody accumulation in dermatomyositis against an expanded diversity of microbial and human proteins and evidence of non-random targeting of specific signalling pathways. Our findings indicate that molecular mimicry and epitope spreading events may play a significant role in the pathogenesis of dermatomyositis.","AbstractThis work aimed to evaluate the cytotoxic potential against cancer cells of Passiflora genus plant species cultivated in Brazil and identify the mechanism of cytotoxicity induced by the most promising extract. Leaf extracts from 14 Passiflora (P.) species were obtained ASE and in vitro cytotoxicity evaluated against cancer cell lines using MTT assay at a single concentration of 50 \u03bcg/mL. Additionally, the IC50 of the P. alata (ELPA) leaf extracts was determined against both tumor (HCT-116, SF-295, OVACAR-8, and HL-60), and non-tumor cells (PBMC). The ELPA flavonoids were identified by HPLC-DAD and UHPLC-MS/MS. The morphological analyses used light and fluorescence microscopy, and cell cycle and DNA fragmentation analyses used flow cytometry to determine the mechanism of cell death induced by ELPA in HL-60. Among the Passiflora leaf extracts evaluated; ELPA stood out with high cytotoxic activity, followed by P. capsularis and P. quadrangulares with varying high and low cytotoxic activity. ELPA presented high cytotoxic potency in HL-60 (IC50 19.37 \u03bcg/mL), yet without cytotoxic activity against PBMC, suggesting selectivity for tumor cells. The cytotoxic activity of ELPA may well be linked to the presence of ten identified flavonoids. Cells treated with ELPA presented the hallmarks typical of apoptosis and necrosis, with cell cycle arrest in the G2/M phase. Conclusion: From among the studied species, ELPA presented greater cytotoxic activity, possibly a consequence of synergistic flavonoid action which induces cell death by apoptosis and necrosis.","The infection of the novel coronavirus SARS-CoV-2 have caused more than 150,000 deaths, but no vaccine or specific therapeutic antibody is currently available. SARS-CoV-2 relies on its spike protein, in particular the receptor binding domain (RBD), to bind human cell receptor angiotensin-converting enzyme 2 (ACE2) for viral entry, and thus targeting RBD holds the promise for preventing SARS-CoV-2 infection. In this work, a competitive biopanning strategy of a phage display antibody library was applied to screen blocking antibodies against RBD. High-affinity antibodies were enriched after the first round using a standard panning process in which RBD-His recombinant protein was immobilized as a bait. At the next two rounds, immobilized ACE2-Fc and free RBD-His proteins were mixed with the enriched phage antibodies. Antibodies binding to RBD at epitopes different from ACE2-binding site were captured by the immobilized ACE2-Fc, forming a \"sandwich\" complex. Only antibodies competed with ACE2 for recognizing RBD at the same or similar epitopes can bind to the free RBD-His in the supernatant and be subsequently separated by the Ni-NTA magnetic beads. Top 1 lead from the competitive biopanning of a synthetic antibody library, Lib AB1, was produced as the full-length IgG1 format. It was proved to competitively block the binding of RBD to ACE2 protein, and potently inhibit SARS-CoV-2 pseudovirus infection of ACE2-overexpressing Hela cells with IC50 values of 12nM. Nevertheless, top 1 lead from the standard biopanning of Lib AB1, can only bind to RBD in vitro but not have the blocking or neutralization activity. Our strategy can efficiently isolate the blocking antibodies of RBD, and it would speed up the discovery of neutralizing antibodies against SARS-CoV-2.","ABSTRACT9-O-acetylation of sialic acid is a common modification that plays important roles in host-pathogen interactions. CASD1 has been described as a sialate-O-acetyltransferase and has been shown to be essential for 9-O-acetylation of sialic acid in some cell lines in vitro. In this study, we used knockout mice to confirm that CASD1 is indeed responsible for 9-O-acetylation of sialic acids in vivo. We observed a complete loss of 9-O-acetylation of sialic acids on the surface of myeloid, erythroid and CD4+ T cells in Casd1-deficient mice. Although 9-O-acetylation of sialic acids on multiple hematopoietic lineages was lost, there were no obvious defects in hematopoiesis. Interestingly, red blood cells from Casd1-deficient mice also lost reactivity to TER-119, a rat monoclonal antibody that is widely used to mark the murine erythroid lineage. The sialic acid glyco-epitope recognized by TER-119 on red blood cells was sensitive to the sialic acid O-acetyl esterase activity of the hemagglutinin esterase from bovine coronavirus but not to the corresponding enzyme from the influenza C virus. During erythrocyte development TER-119+ Ery-A and Ery-B cells could be stained by catalytically inactive bovine coronavirus hemaggutinin-esterase but not by the inactive influenza C hemagglutinin esterase, while TER-119+ Ery-C stage cells and mature erythrocytes were recognized by both virolectins. These results suggest that throughout murine erythrocyte development, cells of the erythroid lineage express a glycoconjugate bearing a modified 7,9-di-O-acetyl form of sialic acid, that is recognized specifically by the bovine coronavirus lectin and not by the influenza C hemagglutinin, and this modified sialic acid moiety is a component of the TER-119 epitope. As erythrocytes mature, the surface of Ery-C cells and mature erythrocytes also acquires a distinct CASD1-dependent 9-O-acetyl sialic acid moiety that can be recognized by virolectins from both influenza C and bovine coronavirus that are specific for 9-O-acetyl sialic acid.","AbstractThe outbreak of a novel betacoronavirus (2019-nCov) represents a pandemic threat that has been declared a public health emergency of international concern. The CoV spike (S) glycoprotein is a key target for urgently needed vaccines, therapeutic antibodies, and diagnostics. To facilitate medical countermeasure (MCM) development we determined a 3.5 \u00c5-resolution cryo-EM structure of the 2019-nCoV S trimer in the prefusion conformation. The predominant state of the trimer has one of the three receptor-binding domains (RBDs) rotated up in a receptor-accessible conformation. We also show biophysical and structural evidence that the 2019-nCoV S binds ACE2 with higher affinity than SARS-CoV S. Additionally we tested several published SARS-CoV RBD-specific monoclonal antibodies and found that they do not have appreciable binding to nCoV-2019 S, suggesting antibody cross-reactivity may be limited between the two virus RBDs. The atomic-resolution structure of 2019-nCoV S should enable rapid development and evaluation of MCMs to address the ongoing public health crisis.","AbstractInherited retinal degenerations (IRDs) are at the focus of current genetic therapeutic advancements. For a genetic treatment such as gene therapy to be successful an accurate genetic diagnostic is required. Genetic diagnostics relies on the assessment of the probability that a given DNA variant is pathogenic. Non-coding variants present a unique challenge for such assessments as compared to coding variants. For one, non-coding variants are present at much higher number in the genome than coding variants. In addition, our understanding of the rules that govern the non-coding regions of the genome is less complete than our understanding of the coding regions. Methods that allow for both the identification of candidate non-coding pathogenic variants and their functional validation may help overcome these caveats allowing for a greater number of patients to benefit from advancements in genetic therapeutics. We present here an unbiased approach combining whole genome sequencing (WGS) with patient induced pluripotent stem cell (iPSC) derived retinal organoids (ROs) transcriptome analysis. With this approach we identified and functionally validated a novel pathogenic non-coding variant in a small family with a previously unresolved genetic diagnosis.","SARS-CoV-2, a member of the coronavirus family, is responsible for the current COVID-19 pandemic. We previously demonstrated that four nucleotide analogues (specifically, the active triphosphate forms of Sofosbuvir, Alovudine, AZT and Tenofovir alafenamide) inhibit the SARS-CoV-2 RNA-dependent RNA polymerase (RdRp). Tenofovir and emtricitabine are the two components in DESCOVY and TRUVADA, the two FDA-approved medications for use as pre-exposure prophylaxis (PrEP) to prevent HIV infection. This is a preventative method in which individuals who are HIV negative (but at high-risk of contracting the virus) take the combination drug daily to reduce the chance of becoming infected with HIV. PrEP can stop HIV from replicating and spreading throughout the body. We report here that the triphosphates of tenofovir and emtricitabine, the two components in DESCOVY and TRUVADA, act as terminators for the SARS-CoV-2 RdRp catalyzed reaction. These results provide a molecular basis to evaluate the potential of DESCOVY and TRUVADA as PrEP for COVID-19.","Environmental DNA (eDNA) and its subdiscipline, invertebrate-derived DNA (iDNA) have been used to survey biodiversity non-invasively [1,2]. Water is ubiquitous in most ecosystems, and, among invertebrates, terrestrial haematophagous leeches are abundant and can be easily collected in many tropical rainforests [3,4]. Such non-invasive nucleic acid sources can mitigate difficulties of obtaining wildlife samples, particularly in remote areas or for rare species. Recently, eDNA/iDNA sources have been applied to monitoring specific wildlife pathogens [5,6]. However, previous studies have focused on known pathogens, whereas most wildlife pathogens are uncharacterized and unknown. Non-invasive approaches to monitoring known and novel pathogens may be of particular benefit in ecosystems prone to viral emergence, many of which occur in areas where invasive sampling is challenging, such as tropical rainforests. Here, we show that both eDNA from natural waterholes, and iDNA from terrestrial haematophagous leeches, can be used to detect unknown viruses circulating in mammalian hosts (Figure 1). Using a curated set of RNA oligonucleotides based on the ViroChip microarray assay [7] as baits in a hybridization capture system, multiple mammalian RNA and DNA viruses were detected from both eDNA and iDNA samples. Congruence was found between host DNA assignment and viruses identified in leeches, and between animals observed visiting the waterholes and the viruses detected. Our results demonstrate that eDNA/iDNA samples may represent an effective non-invasive resource for studying wildlife viral diversity. Several of the detected viruses were novel, highlighting the potential of eDNA/iDNA for epidemiological analysis of emerging viruses prior to their emergence.","AbstractTheColonyCollapseDisorder or CCD is a multi-faceted syndrome decimating bee populations worldwide[1]. A group of viruses of the widely distributedDicistroviridaefamily have been identified as a causing agent of CCD[2]. This family of viruses employ non-coding RNA sequences, calledInternalRibosomalEntrySite (IRES), to precisely exploit the host machinery for protein production. Using single-particle cryo-electron microscopy (cryo-EM) we have characterized at high resolution how the IRES of the intergenic region of theIsraeliAcuteParalysisVirus (IAPV) captures and redirects translating ribosomes towards viral messengers. Through a series of six structures at nominal resolutions close to 3\u00c5, we could reconstruct the trajectory of IAPV-IRES from an early small subunit recruitment to a final post-translocated state in the ribosome. An early commitment of IRES/ribosome complexes for global pre-translocation mimicry explains the high efficiency observed for this IRES. The presented structures will help guide on-going efforts directed towards fighting CCD through RNA-interference technology [3].","AbstractMiddle East respiratory syndrome coronavirus (MERS-CoV) is a major emerging zoonotic infectious disease. Since its first outbreak in 2012, the virus has repeatedly transmitted from camels to humans with 2,468 confirmed cases, causing 851 deaths. To date, there are no efficacious drugs and vaccines against MERS-CoV, increasing its potential to cause a public health emergency. A critical step in the life cycle of MERS-CoV is the fusion with the host cell with its spike (S) protein as main determinant of viral entry. Proteolytic cleavage of S exposes its fusion peptide (FP), which initiates membrane fusion. Previous studies on the related severe acute respiratory syndrome coronavirus (SARS-CoV) FP have shown that calcium (Ca2+) plays an important role for fusogenic activity via a Ca2+ binding pocket with conserved glutamic acid (E) and aspartic acid (D) residues. SARS-CoV and MERS-CoV FP share a high sequence homology and here, we investigated whether Ca2+ is required for MERS-CoV fusion by substituting E and D residues in the MERS-CoV FP with neutrally charged alanines. Upon verifying mutant cell surface expression and proteolytic cleavage, we tested the mutants ability to mediate infection of pseudo-particles (PPs) on host cells without and with Ca2+. Our results demonstrate that intracellular Ca2+ enhances MERS-CoV WT PPs infection by approximately two-fold and that E891 is a crucial residue for Ca2+ interaction. Electron spin resonance revealed that this enhancement could be attributed to Ca2+ increasing MERS-CoV FP fusion-relevant membrane ordering. Intriguingly, isothermal calorimetry titration showed that MERS-CoV FP binds one Ca2+, as opposed to SARS-CoV FP which binds two. Our data suggests that there are significant differences in FP-Ca2+ interactions of MERS-CoV and SARS-CoV FP despite their high sequence similarity and that the number of Ca2+ ions interacting with the FP has implications on the fusion dynamics of the virus.","Entry of SARS-CoV-2, etiological agent of COVID-19, in the host cell is driven by the interaction of its spike protein with human ACE2 receptor and a serine protease, TMPRSS2. Although complex between SARS-CoV-2 spike protein and ACE2 has been structurally resolved, the molecular details of the SARS-CoV-2 and TMPRSS2 complex are still elusive. TMPRSS2 is responsible for priming of the viral spike protein that entails cleavage of the spike protein at two potential sites, Arg685/Ser686 and Arg815/Ser816. The present study aims to investigate the conformational details of complex between TMPRSS2 and SARS-CoV-2 spike protein, in order to discern the finer details of the priming of viral spike and to point candidate drug targets. Briefly, full length structural model of TMPRSS2 was developed and docked against the resolved structure of SARS-CoV-2 spike protein with directional restraints of both cleavage sites. The docking simulations showed that TMPRSS2 interacts with the two different loops of SARS-CoV-2 spike protein, each containing different cleavage sites. Key functional residues of TMPRSS2 (His296, Ser441 and Ser460) were found to interact with immediate flanking residues of cleavage sites of SARS-CoV-2 spike protein. Compared to the N-terminal cleavage site (Arg685/Ser686), TMPRSS2 region that interact with C-terminal cleavage site (Arg815/Ser816) of the SARS-CoV-2 spike protein was predicted as relatively more druggable. In summary, the present study provide structural characteristics of molecular complex between human TMPRSS2 and SARS-CoV-2 spike protein and points to the candidate drug targets that could further be exploited to direct structure base drug designing.","cis-acting RNA motifs play a major role in regulating many aspects of RNA biology including posttranscriptional processing, nuclear export, RNA localization, translation and degradation. Here we analyzed the genomes of SARS-CoV-2 and other single-strand RNA (ssRNA) viruses for the presence of a unique cis RNA element called SECReTE. This motif consists of 10 or more consecutive triplet nucleotide repeats where a pyrimidine nucleotide (C or U) in present every third base, and which we identified in mRNAs encoding secreted proteins in bacteria, yeast, and humans. This motif facilitates mRNA localization to the endoplasmic reticulum (ER), along with the enhanced translation and secretion of translated protein. We now examined for SECReTE presence in Group IV and V RNA viruses, the former including the Coronaviridae, like SARS-CoV-2 and other positive (+)ssRNA viruses, and the latter consisting of negative (-) ssRNA viruses. Interestingly, the SARS-CoV-2 genome contains 40 SECReTE motifs at an abundance of ~1.3 SECReTEs/kilobase (kb). Moreover, all ssRNA viruses we examined contain multiple copies of this motif and appears in (+)ssRNA viruses as non-random in occurrence and independent of genome length. Importantly, (+)ssRNA viruses (e.g. Coronaviruses and Hepaciviruses), which utilize ER membranes to create double membrane vesicles to serve as viral replication centers (VRCs), contain more SECReTE motifs per kb as compared to (-)ssRNA viruses (e.g. Rabies, Mumps, and Influenza), that replicate in the nucleus or the cytoplasm, or other (+)ssRNA viruses (e.g. Enteroviruses and Flaviviruses) which employ different organellar membranes. As predicted by our earlier work, SECReTE sequences are mostly found in membranal or ER-associated/secreted proteins. Thus, we propose that SECReTE motifs could be important for the efficient translation and secretion of secreted viral proteins, as well as for VRC formation. Future studies of SECReTE function and identification of SECReTE-binding proteins could provide new drug targets to treat COVID-19 and other (+)ssRNA related diseases.","AbstractThe recent emergence of the pathogenic SARS-CoV-2 initiated a worldwide health crisis. The entry of the virus into cells is mediated by the binding of the viral Spike protein to the angiotensin-converting enzyme-2 (ACE2), followed by its priming by the TMPRSS2 serine protease, both present on the cellular membrane of the target cells. In the respiratory tract, these targets are ciliated cells. Interestingly, various reports indicate an association between SARS-CoV-2 infection and anosmia, suggesting an alteration not restricted to the respiratory tissue, but that might also include the olfactory sensory epithelium. We explored this possibility by generating RNA-seq libraries from human neuroepithelium, in which we found significant expression of ACE2 and TMPRSS2. To determine whether specific cell types of this chemosensory tissue may coexpress both of the virus entry genes, we analyzed a scRNA-seq dataset. We determined that sustentacular cells, which are in direct contact with the external world and maintain the integrity of olfactory sensory neurons, represents a prime candidate for SARS-CoV-2 infection via the nose, and possibly for SARS-CoV-2-induced anosmia.","Community transmission of the new coronavirus SARS-CoV-2 is a major public health concern that remains difficult to assess. We present a genomic survey of SARS-CoV-2 from a during the first 10 weeks of COVID-19 activity in New South Wales, Australia. Transmission events were monitored prospectively during the critical period of implementation of national control measures. SARS-CoV-2 genomes were sequenced from 209 patients diagnosed with COVID-19 infection between January and March 2020. Only a quarter of cases appeared to be locally acquired and genomic-based estimates of local transmission rates were concordant with predictions from a computational agent-based model. This convergent assessment indicates that genome sequencing provides key information to inform public health action and has improved our understanding of the COVID-19 evolution from outbreak to epidemic.","ABSTRACTDengue virus (DENV) is a mosquito-borne virus of the family Flaviviridae. The RNA viral genome encodes for a polyprotein that is co-translationally processed into three structural proteins and seven non-structural proteins. The non-structural protein 1 (NS1) is a multifunctional viral protein actively secreted in vertebrate and mosquito cells during DENV infection. In mosquito cells, NS1 is secreted in a caveolin-1 (CAV-1) dependent manner by an unconventional pathway. The caveolin chaperone complex (CCC) is a cytoplasmic complex formed by caveolin-1 and the chaperones FKBP52, Cy40 and CyA which is responsible for cholesterol traffic inside the cell. In this work, we demonstrate that in infected mosquito cells, DENV NS1 is secreted by an early and unconventional route that bypasses the Golgi apparatus in close association with the CCC. Treatment of mosquito cells with classic secretion inhibitors such as brefeldin A, golgicide A and Fli-06 showed no effect on NS1 secretion, but significant reductions in recombinant luciferase secretion and virion release. Silencing the expression of CAV1, FKBP52 with siRNAs or the inhibition of CyA by cyclosporine A resulted in significant decrease in NS1 secretion without affecting virion release. Using co-localization, co-inmunoprecipitation and proximity ligation assays, NS1 was found to co-localize and interact with all the protein components of the CCC in mosquito infected cells. In addition, CAV-1 and FKBP52 expression was found augmented in DENV infected cells. Finally, the treatment of ZIKV infected mosquito cells with brefeldin A and golgicide A showed no effect on NS1 secretion, while affecting virion release. ZIKV NS1 was also found to co-localize with CAV-1 in infected mosquito cells. These results suggest that in mosquito cells, ZIKV NS1 follows the same secretory pathway observed for DENV NS1. The association of NS1 with the cholesterol transporter CCC agrees with the lipoprotein nature of secreted hexameric NS1.AUTHOR SUMMARYDengue protein NS1 is secreted in infected mosquito and vertebrate cells. In humans, secreted NS1 have been associated with pathogenesis. In mosquito cells, NS1 follows an unconventional secretion pathway that is dependent on Caveolin-1. This work shows that in mosquito cells, NS1 secretion is associated to the chaperone caveolin complex, a complex formed by caveolin-1 and several chaperones, in charge of cholesterol transport within the cells. Reduction of the expression or the activity of chaperone caveolin complex in mosquito infected cells, diminished the secretion of NS1 without affecting virion release. Direct interaction between NS1 and the chaperone caveolin complex proteins was demonstrated by several assays. Moreover, increased expression of the caveolin-1 and co-chaperone FKBP52 during dengue infection was found, presumably in response to the higher requirements of these proteins during dengue virus infection. Results obtained with ZIKV infected mosquito cells suggest that also ZIKV NS1 is released following an unconventional secretory route in association with the chaperone caveolin complex. The functions of secreted NS1 within mosquito are unclear. However, giving the importance of the soluble NS1 in the vertebrate host, manipulation of the NS1 secretory route may prove a valuable strategy for dengue mosquito control and patient treatment.","AbstractRationalePneumonia is a leading cause of mortality in infants and young children. The mechanisms that lead to mortality in these children are poorly understood. Studies of the cellular immunology of the infant airway have traditionally been hindered by the limited sample volumes available from the young, frail children who are admitted to hospital with pneumonia. This is further compounded by the relatively low frequencies of certain immune cell phenotypes that are thought to be critical to the clinical outcome of pneumonia. To address this, we developed a novel in-silico deconvolution method for inferring the frequencies of immune cell phenotypes in the airway of children with different survival outcomes using proteomic data.MethodsUsing high-resolution mass spectrometry, we identified &gt; 1,000 proteins expressed in the airways of children who were admitted to hospital with clinical pneumonia. 61 of these children were discharged from hospital and survived for more than 365 days after discharge, while 19 died during admission. We used machine learning by random forest to derive protein features that could be used to deconvolve immune cell phenotypes in paediatric airway samples. We applied these phenotype-specific signatures to identify airway-resident immune cell phenotypes that were differentially enriched by survival status and validated the findings using a large retrospective pneumonia cohort.Main ResultsWe identified immune-cell phenotype classification features for 33 immune cell types. Eosinophil-associated features were significantly elevated in airway samples obtained from pneumonia survivors and were downregulated in children who subsequently died. To confirm these results, we analyzed clinical parameters from &gt;10,000 children who had been admitted with pneumonia in the previous 10 years. The results of this retrospective analysis mirrored airway deconvolution data and showed that survivors had significantly elevated eosinophils at admission compared to fatal pneumonia.ConclusionsUsing a proteomics bioinformatics approach, we identify airway eosinophils as a critical factor for pneumonia survival in infants and young children.","AbstractIn mammalian cells, widespread acceleration of cytoplasmic mRNA degradation is linked to impaired RNA polymerase II (Pol II) transcription. This mRNA decay-induced transcriptional repression occurs during infection with gammaherpesviruses including Kaposi\u2019s sarcoma-associated herpesvirus (KSHV) and murine gammaherpesvirus 68 (MHV68), which encode an mRNA endonuclease that initiates widespread RNA decay. Here, we show that MHV68-induced mRNA decay leads to a genome-wide reduction of Pol II occupancy at mammalian promoters. Viral genes, despite the fact that they require Pol II for transcription, escape this transcriptional repression. Protection is not governed by viral promoter sequences; instead, location on the viral genome is both necessary and sufficient to escape the transcriptional repression effects of mRNA decay. We hypothesize that the ability to escape from transcriptional repression is linked to the localization of viral DNA in replication compartments, providing a means for these viruses to counteract decay-induced viral transcript loss.","SARS-coronavirus 2 is the causal agent of the COVID-19 outbreak. SARS-Cov-2 entry into a cell is dependent upon binding of the viral spike (S) protein to cellular receptor and on cleavage of the spike protein by the host cell proteases such as Cathepsin L and Cathepsin B. CTSL/B are crucial elements of lysosomal pathway and both enzymes are almost exclusively located in the lysosomes.CTSL disruption offers potential for CoVID-19 therapies. The mechanisms of disruption include: decreasing expression of CTSL, direct inhibition of CTSL activity and affecting the conditions of CTSL environment (increase pH in lysosomes). We have conducted a high throughput drug screen gene expression analysis to identify compounds that would downregulate the expression of CTSL/CTSB. One of the top significant results shown to downregulate the expression of the CTSL gene is Amantadine. Amantadine was approved by the US Food and Drug Administration in 1968 as a prophylactic agent for influenza and later for Parkinsons disease. It is available as a generic drug. Amantadine in addition to downregulating CTSL appears to further disrupt lysosomal pathway , hence interfering with the capacity of the virus to replicate. It acts as a lysosomotropic agent altering the CTSL functional environment. We hypothesize that Amantadine could decrease the viral load in SARS-CoV-2 positive patients and as such it may serve as a potent therapeutic decreasing the replication and infectivity of the virus likely leading to better clinical outcomes. Clinical studies will be needed to examine the therapeutic utility of amantadine in COVID-19 infection.","Severe acute respiratory syndrome coronavirus 2 (2019-nCoV), which first broke out in Wuhan (China) in December of 2019, causes a severe acute respiratory illness with a mortality ranging from 3% to 6%. To better understand the evolution of the newly emerging 2019-nCoV, in this paper, we analyze the codon usage pattern of 2019-nCoV. For this purpose, we compare the codon usage of 2019-nCoV with that of other 30 viruses belonging to the subfamily of orthocoronavirinae. We found that 2019-nCoV shows a rich composition of AT nucleotides that strongly influences its codon usage, which appears to be not optimized to human host. Then, we study the evolutionary pressures influencing the codon usage and evolutionary rates of the sequences of five conserved genes that encode the corresponding proteins (viral replicase, spike, envelope, membrane and nucleocapsid) characteristic of coronaviruses. We found different patterns of both mutational bias and nature selection that affect the codon usage of these genes at different extents. Moreover, we show that the two integral membrane proteins proteins (matrix and envelope) tend to evolve slowly by accumulating nucleotide mutations on their genes. Conversely, genes encoding nucleocapsid (N), viral replicase and spike proteins are important targets for the development of vaccines and antiviral drugs, tend to evolve faster as compared to other ones. Taken together, our results suggest that the higher evolutionary rate observed for these two genes could represent a major barrier in the development of antiviral therapeutics 2019-nCoV.","The global population is at present suffering from a pandemic of Coronavirus disease 2019 (COVID-19), caused by the novel coronavirus Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2). The goals of this study were to use artificial intelligence (AI) to predict blueprints for designing universal vaccines against SARS-CoV-2, that contain a sufficiently broad repertoire of T-cell epitopes capable of providing coverage and protection across the global population. To help achieve these aims, we profiled the entire SARS-CoV-2 proteome across the most frequent 100 HLA-A, HLA-B and HLA-DR alleles in the human population, using host-infected cell surface antigen presentation and immunogenicity predictors from the NEC Immune Profiler suite of tools, and generated comprehensive epitope maps. We then used these epitope maps as input for a Monte Carlo simulation designed to identify statistically significant epitope hotspot regions in the virus that are most likely to be immunogenic across a broad spectrum of HLA types. We then removed epitope hotspots that shared significant homology with proteins in the human proteome to reduce the chance of inducing off-target autoimmune responses. We also analyzed the antigen presentation and immunogenic landscape of all the nonsynonymous mutations across 3400 different sequences of the virus, to identify a trend whereby SARS-COV-2 mutations are predicted to have reduced potential to be presented by host-infected cells, and consequently detected by the host immune system. A sequence conservation analysis then removed epitope hotspots that occurred in less-conserved regions of the viral proteome. Finally, we used a database of the HLA genotypes of approximately 22 000 individuals to develop a digital twin type simulation to model how effective different combinations of hotspots would work in a diverse human population, and used the approach to identify an optimal constellation of epitopes hotspots that could provide maximum coverage in the global population. By combining the antigen presentation to the infected-host cell surface and immunogenicity predictions of the NEC Immune Profiler with a robust Monte Carlo and digital twin simulation, we have managed to profile the entire SARS-CoV-2 proteome and identify a subset of epitope hotspots that could be harnessed in a vaccine formulation to provide a broad coverage across the global population.","ABSTRACTNovel Coronavirus (nCoV) outbreak in the city of Wuhan, China during December 2019, has now spread to various countries across the globe triggering a heightened containment effort. This human pathogen is a member of betacoronavirus genus carrying 30 kilobase of single positive-sense RNA genome. Understanding the evolution, zoonotic transmission, and source of this novel virus would help accelerating containment and prevention efforts. The present study reported detailed analysis of 2019-nCoV genome evolution and potential candidate peptides for vaccine development. This nCoV genotype might have been evolved from a bat-CoV by accumulating non-synonymous mutations, indels, and recombination events. Structural proteins Spike (S), and Membrane (M) had extensive mutational changes, whereas Envelope (E) and Nucleocapsid (N) proteins were very conserved suggesting differential selection pressures exerted on 2019-nCoV during evolution. Interestingly, 2019-nCoV Spike protein contains a 39 nucleotide sequence insertion relative to SARS-like bat-SL-CoVZC45/2017. Furthermore, we identified eight high binding affinity (HBA) CD4 T-cell epitopes in the S, E, M and N proteins, which can be commonly recognized by HLA-DR alleles of Asia and Asia-Pacific Region population. These immunodominant epitopes can be incorporated in universal subunit CoV vaccine. Diverse HLA types and variations in the epitope binding affinity may contribute to the wide range of immunopathological outcomes of circulating virus in humans. Our findings emphasize the requirement for continuous surveillance of CoV strains in live animal markets to better understand the viral adaptation to human host and to develop practical solutions to prevent the emergence of novel pathogenic CoV strains.","AbstractXylella fastidiosa (Xf) is a quarantine plant pathogen bacterium originating from the Americas and that has emerged in Europe in 2013. Xf can be detected directly on plant macerate using molecular methods such as real-time PCR, which is a sensitive technique. However, some plants may contain components that can act as PCR reaction inhibitors, which can lead to false negative results or an underestimation of the bacterial concentration present in the analyzed plant sample. Droplet digital PCR (ddPCR) is an innovative tool based on the partitioning of the PCR reagents and the DNA sample into thousands of droplets, allowing the quantification of the absolute number of target DNA molecules present in a reaction mixture, or an increase of the detection sensitivity. In this study, a real-time PCR protocol, already used for Xf detection in the framework of official surveys in the European Union, was transferred and optimized for Xf detection using ddPCR. This new assay was evaluated and compared to the initial real-time PCR on five plant matrices artificially inoculated and on naturally infected plants. In our conditions, this new ddPCR enabled the detection of Xf on all artificially inoculated plant macerates with a similar limit of detection, or a slight benefit for Quercus ilex. Moreover, ddPCR improved diagnostic sensitivity as it enabled detection of Xf in samples of Polygala myrtifolia or Q. ilex that were categorized as negative or close to the limit of detection using the real-time PCR. Here, we report for the first time a ddPCR assay for the detection of the bacterium Xf.","AbstractThe response of pathophysiological research to emerging epidemics often occurs after the epidemic and, as a consequence, has little to no impact on improving patient outcomes or on developing high-quality evidence to inform clinical management strategies during the epidemic. Rapid and informed guidance of epidemic (research) responses to severe infectious disease outbreaks requires quick compilation and integration of existing pathophysiological knowledge. As a case study we chose the Zika virus (ZIKV) outbreak that started in 2015 to develop a proof-of-concept knowledge repository. To extract data from available sources and build a computationally tractable and comprehensive molecular interaction map we applied generic knowledge management software for literature mining, expert knowledge curation, data integration, reporting and visualisation. A multi-disciplinary team of experts, including clinicians, virologists, bioinformaticians and knowledge management specialists, followed a pre-defined workflow for rapid integration and evaluation of available evidence. While conventional approaches usually require months to comb through the existing literature, the initial ZIKV KnowledgeBase (ZIKA KB) was completed within a few weeks. Recently we updated the ZIKA KB with additional curated data from the large amount of literature published since 2016 and made it publicly available through a web interface together with a step-by-step guide to ensure reproducibility of the described use case (S4). In addition, a detailed online user manual is provided to enable the ZIKV research community to generate hypotheses, share knowledge, identify knowledge gaps, and interactively explore and interpret data (S5). A workflow for rapid response during outbreaks was generated, validated and refined and is also made available. The process described here can be used for timely structuring of pathophysiological knowledge for future threats. The resulting structured biological knowledge is a helpful tool for computational data analysis and generation of predictive models and opens new avenues for infectious disease research.Availabilitywww.zikaknowledgebase.euFundingEuropean Commission\u2019s Seventh Framework Research Programme project PREPARE (FP7-Health n\u00b0602525) and ZIKALLIANCE (MK, H2020; No 734548).Author summaryDuring the recent ZIKV outbreak there was little information about the interactions between Zika virus and the host, however, the massive research response lead to a steep increase in the number of relevant publications within a very short period of time. At the time, there was no structured and comprehensive database available for integrated molecular and physiological data and knowledge about ZIKV infection. Researchers had to manually review the literature (amounting to over 5000 articles on ZIKV during our last update of the ZIKA KB in September 2018) to extract information about host\u2013pathogen interaction and affected molecular, cellular and organ pathways. We explored the use of automated literature analysis and a defined cooperative effort between experts from various scientific, biomedical and information-technology domains to rapidly compile existing pathophysiological knowledge as a potential tool to support investigations during an emergency. This tool is contrasted with conventional approaches that would take months to comb through the massive amount of existing literature. In addition to providing background information for research, scientific publications can be processed to transform textual information into complex networks, which can be integrated with existing knowledge resources to suggest novel hypotheses that potentially contribute to innovative infectious disease research approaches. This study shows that the knowledge extraction and mapping process required to inform clinical and research responses to an emerging epidemic can be efficiently and effectively executed with a dedicated and trained group of experts, a validated process and the necessary tools. Our results further provide an overview of ZIKV biology, allow prediction of drug efficacy and indentify specific host factors and signalling pathways affected by ZIKV.","Taiwan experienced two waves of imported cases of coronavirus disease 2019 (COVID-19), first from China in January to late February, followed by those from other countries starting in early March. Additionally, several cases could not be traced to any imported cases and were suspected as sporadic local transmission. Twelve full viral genomes were determined in this study by Illumina sequencing either from virus isolates or directly from specimens, among which 5 originated from clustered infections. Phylogenetic tree analysis revealed that these sequences were in different clades, indicating that no major strain has been circulating in Taiwan. A deletion in open reading frame 8 was found in one isolate. Only a 4-nucleotide difference was observed among the 5 genomes from clustered infections.","AbstractAimsDickeya species are high consequence plant pathogenic bacteria listed among the quarantine pathogens of the European Union; associated with potato disease outbreaks and subsequent economic losses worldwide. Early, accurate, and reliable detection of Dickeya spp. is needed to prevent establishment and further dissemination of this pathogen. Therefore, a multiplex TaqMan qPCR was developed for sensitive detection of Dickeya spp. and specifically, D. dianthicola.Methods and ResultsA signature genomic region for the genus Dickeya (mglA/mglC) and unique genomic region for D. dianthicola (alcohol dehydrogenase) were identified using a whole genome based comparative genomics approach. The developed multiplex TaqMan qPCR was validated using extensive inclusivity and exclusivity panels, and naturally/artificially infected samples to confirm broad range detection capability and specificity. Both sensitivity and spiked assays showed detection limit of 10 fg DNA.ConclusionThe developed multiplex assay is sensitive and reliable to detect Dickeya spp. and D. dianthicola with no false positives or false negatives. It was able to detect mixed infection from naturally and artificially infected plant materials.Significance and ImpactThe developed assay will serve as a practical tool for screening of propagative material, monitoring the presence and distribution, and quantification of target pathogens in a breeding program. The assay also has applications in routine diagnostics, biosecurity and microbial forensics.","Here we aim to describe early mutational events across samples from publicly available SARS-CoV-2 sequences from the sequence read archive repository. Up until March 27, 2020, we downloaded 53 illumina datasets, mostly from China, USA (Washington DC) and Australia (Victoria). Of 30 high quality datasets, 27 datasets (90%) contain at least a single founder mutation and most of the variants are missense (over 63%). Five-point mutations with clonal (founder) effect were found in USA sequencing samples. Sequencing samples from USA in GenBank present this signature with 50% allele frequencies among samples. Australian mutation signatures were more diverse than USA samples, but still, clonal events were found in those samples. Mutations in the helicase and orf1a coding regions from SARS-CoV-2 were predominant, among others, suggesting that these proteins are prone to evolve by natural selection. Finally, we firmly urge that primer sets for diagnosis be carefully designed, since rapidly occurring variants would affect the performance of the reverse transcribed quantitative PCR (RT-qPCR) based viral testing.","Here we present a novel protocol for the construction of saturation single-site\u2014and massive multi-site\u2014mutant libraries of a bacteriophage. We segmented the \u03a6X174 genome into 14 non-toxic and non-replicative fragments compatible with golden gate assembly. We next used nicking mutagenesis with oligonucleotides prepared from unamplified oligo pools with individual segments as templates to prepare near-comprehensive single-site mutagenesis libraries of genes encoding the F capsid protein (421 amino acids scanned) and G spike protein (172 amino acids scanned). Libraries possessed greater than 99% of all 11,860 programmed mutations. Golden Gate cloning was then used to assemble the complete \u03a6X174 mutant genome and generate libraries of infective viruses. This protocol will enable reverse genetics experiments for studying viral evolution and, with some modifications, can be applied for engineering of therapeutically relevant bacteriophages with larger genomes.","ABSTRACTOver two-thirds of a century ago, Hodgkin and Huxley proposed the existence of voltage gated ion channels (VGIC) to carry Na+ and K+ ions across the cell membrane to create the nerve impulse, in response to depolarization of the membrane. The channels have multiple physiological roles, and play a central role in a wide variety of diseases when they malfunction. The first channel structure was found by MacKinnon and coworkers in 1998. Subsequently the structure of a number of VGIC was determined in the open (ion conducting) state. This type of channel consists of four voltage sensing domains (VSD), each formed from four transmembrane (TM) segments, plus a pore domain through which ions move. Understanding the gating mechanism (how the channel opens and closes) requires structures. One TM segment (S4) has an arginine in every third position, with one such segment per domain. It is usually assumed that these arginines are all ionized, and in the resting state are held toward the intracellular side of the membrane by voltage across the membrane. They are assumed to move outward (extracellular direction) when released by depolarization of this voltage, producing a capacitive gating current and opening the channel. We suggest alternate interpretations of the evidence that led to these models. Measured gating current is the total charge displacement of all atoms in the VSD; we propose that the prime, but not sole, contributor is proton motion, not displacement of the charges on the arginines of S4. It is known that the VSD can conduct protons. Quantum calculations on the Kv1.2 potassium channel VSD show how; the key is the amphoteric nature of the arginine side chain, which allows it to transfer a proton; this appears to be the first time the arginine side chain has had its amphoteric character considered. We have calculated one such proton transfer in detail: this proton starts from a tyrosine that can ionize, transferring to the NE of the third arginine on S4; that arginine\u2019s NH then transfers a proton to a glutamate. The backbone remains static. A mutation predicted to affect the proton transfer has been qualitatively confirmed experimentally, from the change in the gating current-voltage curve. The total charge displacement in going from a normal closed potential of \u221270 mV across the membrane to 0 mV (open), is calculated to be approximately consistent with measured values, although the error limits on the calculation require caution in interpretation.","AbstractA novel coronavirus (2019-nCoV) outbreak has caused a global pandemic resulting in tens of thousands of infections and thousands of deaths worldwide. The RNA-dependent RNA polymerase (RdRp, also named nsp12), which catalyzes the synthesis of viral RNA, is a key component of coronaviral replication/transcription machinery and appears to be a primary target for the antiviral drug, remdesivir. Here we report the cryo-EM structure of 2019-nCoV full-length nsp12 in complex with cofactors nsp7 and nsp8 at a resolution of 2.9-\u00c5. Additional to the conserved architecture of the polymerase core of the viral polymerase family and a nidovirus RdRp-associated nucleotidyltransferase (NiRAN) domain featured in coronaviral RdRp, nsp12 possesses a newly identified \u03b2-hairpin domain at its N-terminal. Key residues for viral replication and transcription are observed. A comparative analysis to show how remdesivir binds to this polymerase is also provided. This structure provides insight into the central component of coronaviral replication/transcription machinery and sheds light on the design of new antiviral therapeutics targeting viral RdRp.One Sentence SummaryStructure of 2019-nCov RNA polymerase.","Background: Rapid and reliable diagnostic assays were critical for prevention and control of the coronavirus pneumonia caused by COVID-19. Objective: This study was to establish one-pot real-time reverse-transcription loop-mediated isothermal amplification (RT-LAMP) assay and one-pot visual RT-LAMP assay for the detection of COVID-19. Methods: Six specific LAMP primers targeting the N gene of COVID-19 were designed, the RT-LAMP reaction system was optimized with plasmid pUC57 containing N gene sequence, the detection limit was determined with a serial dilution of the plasmid pUC57 containing N gene sequence, and the one-pot real-time RT-LAMP assay and one-pot visual RT-LAMP assay for the detection of COVID-19 were established. Results: Our results showed that the one-pot RT-LAMP assays can detect COVID-19 with a limit of \u2265 6 copies per \u03bcl\u22121 of pUC57 containing N gene sequence. Conclusion: This study provides rapid, reliable and sensitive tools for facilitating preliminary and cost-effective prevention and control of COVID-19.","AbstractThe current outbreak of Coronavirus Disease 2019 (COVID-19) by a novel betacoronavirus severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has aroused great public health concern. Coronavirus has a history of causing epidemics in human and animals. In 2017 an outbreak in piglets by a novel coronavirus was emerged designated as swine acute diarrhea syndrome coronavirus (SADS-CoV) which is originated from the same genus of horseshoe bats (Rhinolophus) as Severe Acute Respiratory Syndrome CoV (SARS-CoV) having a broad species tropism. In addition to human cells, it can also infect cell lines from diverse species. Coronavirus host range is determined by its spike glycoprotein (S). Given the importance of S protein in viral entry to cells and host immune responses, here we report the cryo-EM structure of the SADS-CoV S in the prefusion conformation at a resolution of 3.55 \u00c5. Our study reveals that SADS-CoV S structure takes an intra-subunit quaternary packing mode where the NTD and CTD from the same subunit pack together by facing each other. The comparison of NTD and CTD with that of the other four genera suggests the evolutionary process of the SADS-CoV S. Moreover, SADS-CoV S has several characteristic structural features, such as more compact architecture of S trimer, and masking of epitopes by glycan shielding, which may facilitate viral immune evasion. These data provide new insights into the evolutionary relationships of SADS-CoV S and would extend our understanding of structural and functional diversity, which will facilitate to vaccine development.","The outbreak of COVID-19 has now become a global pandemic and it continues to spread rapidly worldwide, severely threatening lives and economic stability. Making the problem worse, there is no specific antiviral drug that can be used to treat COVID-19 to date. SARS-CoV-2 initiates its entry into human cells by binding to angiotensin-converting enzyme 2 (hACE2) via the receptor binding domain (RBD) of its spike protein. Therefore, molecules that can block SARS-CoV-2 from binding to hACE2 may potentially prevent the virus from entering human cells and serve as an effective antiviral drug. Based on this idea, we designed a series of peptides that can strongly bind to SARS-CoV-2 RBD in computational experiments. Specifically, we first constructed a 31-mer peptidic scaffold by linking two fragments grafted from hACE2 (a.a. 22-44 and 351-357) with a linker glycine, and then redesigned the peptide sequence to enhance its binding affinity to SARS-CoV-2 RBD. Compare with several computational studies that failed to identify that SARS-CoV-2 shows higher binding affinity for hACE2 than SARS-CoV, our protein design scoring function, EvoEF2, makes a correct identification, which is consistent with the recently reported experimental data, implying its high accuracy. The top designed peptide binders exhibited much stronger binding potency to hACE2 than the wild-type (-53.35 vs. -46.46 EvoEF2 energy unit for design and wild-type, respectively). The extensive and detailed computational analyses support the high reasonability of the designed binders, which not only recapitulated the critical native binding interactions but also introduced new favorable interactions to enhance binding. Due to the urgent situation created by COVID-19, we share these computational data to the community, which should be helpful to develop potential antiviral peptide drugs to combat this pandemic.","AbstractSARS-CoV-2 is a betacoronavirus that is responsible for the COVID-19 pandemic. The genome of SARS-CoV-2 was reported recently, but its transcriptomic architecture is unknown. Utilizing two complementary sequencing techniques, we here present a high-resolution map of the SARS-CoV-2 transcriptome and epitranscriptome. DNA nanoball sequencing shows that the transcriptome is highly complex owing to numerous recombination events, both canonical and noncanonical. In addition to the genomic RNA and subgenomic RNAs common in all coronaviruses, SARS-CoV-2 produces a large number of transcripts encoding unknown ORFs with fusion, deletion, and/or frameshift. Using nanopore direct RNA sequencing, we further find at least 41 RNA modification sites on viral transcripts, with the most frequent motif being AAGAA. Modified RNAs have shorter poly(A) tails than unmodified RNAs, suggesting a link between the internal modification and the 3\u2032 tail. Functional investigation of the unknown ORFs and RNA modifications discovered in this study will open new directions to our understanding of the life cycle and pathogenicity of SARS-CoV-2.HighlightsWe provide a high-resolution map of SARS-CoV-2 transcriptome and epitranscriptome using nanopore direct RNA sequencing and DNA nanoball sequencing.The transcriptome is highly complex owing to numerous recombination events, both canonical and noncanonical.In addition to the genomic and subgenomic RNAs common in all coronaviruses, SARS-CoV-2 produces transcripts encoding unknown ORFs.We discover at least 41 potential RNA modification sites with an AAGAA motif.","AbstractMany pathogens take advantage of the dependence of the host on the interaction of hundreds of extracellular proteins with the glycosaminoglycans heparan sulfate to regulate homeostasis and use heparan sulfate as a means to adhere and gain access to cells. Moreover, mucosal epithelia such as that of the respiratory tract are protected by a layer of mucin polysaccharides, which are usually sulfated. Consequently, the polydisperse, natural products of heparan sulfate and the allied polysaccharide, heparin have been found to be involved and prevent infection by a range of viruses including S-associated coronavirus strain HSR1. Here we use surface plasmon resonance and circular dichroism to measure the interaction between the SARS-CoV-2 Spike S1 protein receptor binding domain (SARS-CoV-2 S1 RBD) and heparin. The data demonstrate an interaction between the recombinant surface receptor binding domain and the polysaccharide. This has implications for the rapid development of a first-line therapeutic by repurposing heparin and for next-generation, tailor-made, GAG-based antivirals.","AbstractInfluenza A virus (IAV) and influenza B virus (IBV) cause yearly epidemics with significant morbidity and mortality. When zoonotic IAVs enter the human population, the viral hemagglutinin (HA) requires adaptation to achieve sustained virus transmission. In contrast, IBV has been circulating in humans, its only host, for a long period of time. Whether this entailed adaptation of IBV HA to the human airways is unknown. To address this question, we compared seasonal IAV (A/H1N1 and A/H3N2) and IBV viruses (B/Victoria and B/Yamagata lineage) with regard to host-dependent activity of HA as the mediator of membrane fusion during viral entry. We first investigated proteolytic activation of HA, by covering all type II transmembrane serine protease (TTSP) and kallikrein enzymes, many of which proved present in human respiratory epithelium. Compared to IAV, the IBV HA0 precursor is cleaved by a broader panel of TTSPs and activated with much higher efficiency. Accordingly, knockdown of a single protease, TMPRSS2, was sufficient to abrogate spread of IAV but not IBV in human respiratory epithelial cells. Second, the HA fusion pH proved similar for IBV and human-adapted IAVs (one exception being HA of 1918 IAV). Third, IBV HA exhibited higher expression at 33\u00b0C, a temperature required for membrane fusion by B/Victoria HA. This indicates pronounced adaptation of IBV HA to the mildly acidic pH and cooler temperature of human upper airways. These distinct and intrinsic features of IBV HA are compatible with extensive host-adaptation during prolonged circulation of this respiratory virus in the human population.ImportanceInfluenza epidemics are caused by influenza A (IAV) and influenza B (IBV) viruses. IBV causes substantial disease, however it is far less studied than IAV. While IAV originates from animal reservoirs, IBV circulates in humans only. Virus spread requires that the viral hemagglutinin (HA) is active and sufficiently stable in human airways. We here resolve how these mechanisms differ between IBV and IAV. Whereas human IAVs rely on one particular protease for HA activation, this is not the case for IBV. Superior activation of IBV by several proteases should enhance shedding of infectious particles. IBV HA exhibits acid-stability and a preference for 33\u00b0C, indicating pronounced adaptation to the human upper airways, where the pH is mildly acidic and a cooler temperature exists. These adaptive features are rationalized by the long existence of IBV in humans, and may have broader relevance for understanding the biology and evolution of respiratory viruses.","AbstractTwo notable features have been identified in the SARS-CoV-2 genome: (1) the receptor binding domain of SARS-CoV-2; (2) a unique insertion of twelve nucleotide or four amino acids (PRRA) at the S1 and S2 boundary. For the first feature, the similar RBD identified in SARs-like virus from pangolin suggests the RBD in SARS-CoV-2 may already exist in animal host(s) before it transmitted into human. The left puzzle is the history and function of the insertion at S1/S2 boundary, which is uniquely identified in SARS-CoV-2. In this study, we identified two variants from the first Guangdong SARS-CoV-2 cell strain, with deletion mutations on polybasic cleavage site (PRRAR) and its flank sites. More extensive screening indicates the deletion at the flank sites of PRRAR could be detected in 3 of 68 clinical samples and half of 22 in vitro isolated viral strains. These data indicate (1) the deletion of QTQTN, at the flank of polybasic cleavage site, is likely benefit the SARS-CoV-2 replication or infection in vitro but under strong purification selection in vivo since it is rarely identified in clinical samples; (2) there could be a very efficient mechanism for deleting this region from viral genome as the variants losing 23585-23599 is commonly detected after two rounds of cell passage. The mechanistic explanation for this in vitro adaptation and in vivo purification processes (or reverse) that led to such genomic changes in SARS-CoV-2 requires further work. Nonetheless, this study has provided valuable clues to aid further investigation of spike protein function and virus evolution. The deletion mutation identified in vitro isolation should be also noted for current vaccine development.","AbstractAn effective vaccine for hepatitis C virus (HCV) is a major unmet need, and it requires an antigen that elicits immune responses to key conserved epitopes. Based on structures of antibodies targeting HCV envelope glycoprotein E2, we designed immunogens to modulate the structure and dynamics of E2 and favor induction of bNAbs in the context of a vaccine. These designs include a point mutation in a key conserved antigenic site to stabilize its conformation, as well as redesigns of an immunogenic region to add a new N-glycosylation site and mask it from antibody binding. Designs were experimentally characterized for binding to a panel of human monoclonal antibodies (HMAbs) and the coreceptor CD81 to confirm preservation of epitope structure and preferred antigenicity profile. Selected E2 designs were tested for immunogenicity in mice, with and without hypervariable region 1, which is an immunogenic region associated with viral escape. One of these designs showed improvement in polyclonal immune serum binding to HCV pseudoparticles and neutralization of isolates associated with antibody resistance. These results indicate that antigen optimization through structure-based design of the envelope glycoproteins is a promising route to an effective vaccine for HCV.","COVID-19 pandemic has caused tens of thousands of deaths and is now a severe threat to global health. Clinical practice has demonstrated that the SARS-CoV-2 S1 specific antibodies and viral antigens can be used as diagnostic and prognostic markers of COVID-19. However, the popular point-of-care biomarker detection technologies, such as the lateral-flow test strips, provide only yes/no information and have very limited sensitivities. Thus, it has a high false-negative rate and cannot be used for the quantitative evaluation of the patient's immune response. Conventional ELISA (enzyme-linked immunosorbent assay), on the other hand, can provide quantitative, accurate, and sensitive results, but it involves complicated and expensive instruments and long assay time. In addition, samples need to be sent to centralized labs, which significantly increases the turn-around time. Here, we present a microfluidic ELISA technology for rapid (15-20 minutes), quantitative, sensitive detection of SARS-CoV-2 biomarkers using SARS-CoV-2 specific IgG and viral antigen - S protein in serum. We also characterized various humanized monoclonal IgG, and identified a candidate with a high binding affinity towards SARS-CoV-2 S1 protein that can serve as the calibration standard of anti-SARS-CoV-2 S1 IgG in serological analyses. Furthermore, we demonstrated that our microfluidic ELISA platform can be used for rapid affinity evaluation of monoclonal anti-S1 antibodies. The microfluidic ELISA device is highly portable and requires less than 10 \u03bcL of samples for each channel. Therefore, our technology will greatly facilitate rapid and quantitative analysis of COVID-19 patients and vaccine recipients at point-of-care.","AbstractAdult honey bees harbor a specialized gut microbiota of relatively low complexity. While seasonal differences in community composition have been reported, previous studies have focused on compositional changes rather than differences in absolute bacterial loads. Moreover, little is known about the gut microbiota of winter bees, which live much longer than bees during the foraging season, and which are critical for colony survival. We quantified seven core members of the bee gut microbiota in a single colony over two years and characterized the community composition in 14 colonies during summer and winter. Our data shows that total bacterial loads substantially differ between foragers, nurses, and winter bees. Long-lived winter bees had the highest bacterial loads and the lowest community \u03b1-diversity, with a characteristic shift towards high levels of Bartonella and Commensalibacter, and a reduction of opportunistic colonizers. Using gnotobiotic bee experiments, we show that diet is a major contributor to the observed differences in bacterial loads. Overall, our study reveals that the gut microbiota of winter bees is remarkably different from foragers and nurses. Considering the importance of winter bees for colony survival, future work should focus on the role of the gut microbiota in winter bee health and disease.","AbstractOne central goal of genome biology is to understand how the usage of the genome differs between organisms. Our knowledge of genome composition, needed for downstream inferences, is critically dependent on gene annotations, yet problems associated with gene annotation and assembly errors are usually ignored in comparative genomics. Here we analyze the genomes of 68 species across 12 animal phyla and some single-cell eukaryotes for general trends in genome composition and transcription, taking into account problems of gene annotation. We show that, regardless of genome size, the ratio of introns to intergenic sequence is comparable across essentially all animals, with nearly all deviations dominated by increased intergenic sequence. Genomes of model organisms have ratios much closer to 1:1, suggesting that the majority of published genomes of non-model organisms are underannotated and consequently omit substantial numbers of genes, with likely negative impact on evolutionary interpretations. Finally, our results also indicate that most animals transcribe half or more of their genomes arguing against differences in genome usage between animal groups, and also suggesting that the transcribed portion is more dependent on genome size than previously thought.","AbstractBackgroundAll types of facioscapulohumeral muscular dystrophy (FSHD) are caused by the aberrant myogenic activation of the somatically silent DUX4 gene, which initiates a cascade of cellular events ultimately leading to FSHD pathophysiology. Therefore, FSHD is a dominant gain-of-function disease that is amenable to modeling by DUX4 overexpression. However, there is large variability in the patient population. Typically, progressive skeletal muscle weakness becomes noticeable in the second or third decade of life, yet there are many genetically FSHD individuals who develop symptoms much later in life or remain relatively asymptomatic throughout their lives. Conversely, in rare cases, FSHD may present clinically prior to 5-10 yrs of age, ultimately manifesting as a very severe early onset form of the disease. Thus, there is a need to control the timing and severity of pathology in FSHD-like models.MethodsWe have recently described a line of conditional DUX4 transgenic mice, FLExDUX4, that develop a myopathy upon induction of human DUX4-fl expression in skeletal muscle. Here, we use the FLExDUX4 mouse crossed with the skeletal muscle-specific and tamoxifen inducible line ACTAl-MerCreMer to generate a highly versatile bi-transgenic mouse model with chronic, low-level DUX4-fl expression and mild pathology, that can be induced to develop more severe FSHD-like pathology in a dose-dependent response to tamoxifen. We identified conditions to reproducibly generate models exhibiting mild, moderate, or severe DUX4-dependent pathophysiology, and characterized their progression.ResultsWe assayed DUX4-fl mRNA and protein levels, fitness, strength, global gene expression, histopathology, and immune response, all of which are consistent with an FSHD-like myopathic phenotype. Importantly, we identified sex-specific and muscle-specific differences that should be considered when using these models for preclinical studies.ConclusionsThe ACTA1-MCM;FLExDUX4 bi-transgenic mouse model expresses a chronic low level of DUX4-fl and has mild pathology and detectable muscle weakness. The onset and progression of moderate to severe pathology can be controlled via tamoxifen injection to provide consistent and readily screenable phenotypes for assessing therapies targeting DUX4-fl mRNA and protein. Thus, these FSHD-like mouse models can be used to study a range of DUX4-fl expression and pathology dependent upon investigator need, through controlled mosaic expression of DUX4.","ABSTRACTThe outbreak of pneumonia originating in Wuhan, China, has generated 830 confirmed cases, including 26 deaths, as of 24 January 2020. The virus (2019-nCoV) has spread elsewhere in China and to other countries, including South Korea, Thailand, Japan and USA. Fortunately, there has not yet been evidence of sustained human-to-human transmission outside of China. Here we assess the risk of sustained transmission whenever the coronavirus arrives in other countries. Data describing the times from symptom onset to hospitalisation for 47 patients infected in the current outbreak are used to generate an estimate for the probability that an imported case is followed by sustained human-to-human transmission. Under the assumptions that the imported case is representative of the patients in China, and that the 2019-nCoV is similarly transmissible to the SARS coronavirus, the probability that an imported case is followed by sustained human-to-human transmission is 0.37. However, if the mean time from symptom onset to hospitalisation can be halved by intense surveillance, then the probability that an imported case leads to sustained transmission is only 0.005. This emphasises the importance of current surveillance efforts in countries around the world, to ensure that the ongoing outbreak will not become a large global epidemic.","Chloroquine is an anti-malarial and immunosuppressant drug that has cationic amphipathic chemical properties. We performed genome-wide screens in human cells with chloroquine and several other widely used cationic amphipathic drugs (CADs) including the anti-depressants, sertraline (Zoloft) and fluoxetine (Prozac), the analgesic nortriptyline (Pamelor), the anti-arrhythmic amiodarone (Cordarone), and the anti-hypertensive verapamil (Calan) to characterize their molecular similarities and differences. Despite CADs having different disease indications but consistent with them sharing key chemical properties, we found CADs to have remarkably similar phenotypic profiles compared with non-CADs we and others have previously screened. The most significant genetic interaction for all CADs was the initiating step in sphingolipid biosynthesis catalyzed by serine palmitoyltransferase (SPT). A comparison of genome-wide screens performed with diverse pathogens from viruses, bacteria, plants, and parasites including Ebola, adeno-associated virus AAV2, HIV, Rotavirus, Influenza A, Zika virus, Picornavirus, Exotoxin A, Cholera toxin, Type III secretion system and Shiga toxin, Ricin toxin, and Toxoplasma gondii showed SPT as a top common host factor and 80% overlap overall in top hits specifically with CADs. Potential sphingolipid-mediated mechanisms for the host response- and virulence-modulating effects of CADs involve autophagy and SERPINE1/PAI-1 (plasminogen activator inhibitor-1). Chloroquine has recently shown potential as an anti-viral agent for the novel coronavirus SARS-CoV-2, the causative agent of COVID-19 respiratory disease. Our study demonstrates that numerous readily available drugs molecularly function highly similar to chloroquine, which suggests they might be considered for further pre-clinical investigation in the context of SARS-CoV-2. More generally, our work suggests the diverse pathogen mitigating potential of drugs that inhibit host sphingolipid biosynthesis such as CADs.","AbstractMicroglia express muscarinic G protein-coupled receptors (GPCRs) that sense cholinergic activity and are activated by acetylcholine to potentially regulate microglial functions. Knowledge about how distinct types of muscarinic GPCR signaling regulate microglia function in vitro and in vivo is still poor, partly due to the fact that some of these receptors are also present in astrocytes and neurons. We generated mice expressing the hM3Dq Designer Receptor Exclusively Activated by Designer Drugs (DREADD) selectively in microglia to investigate the role of muscarinic M3Gq-linked signaling. We show that activation of hM3Dq using clozapine N-oxide (CNO) elevated intracellular calcium levels and increased phagocytosis of FluoSpheres in vitro. Acute treatment with CNO in vivo did not affect male mouse behavior, however chronic CNO treatment decreased sickness behavior triggered by lipopolysaccharide (LPS) treatment. Interestingly, whereas acute treatment with CNO increased synthesis of cytokine mRNA, chronic treatment attenuated LPS-induced cytokine mRNA changes in the brain, likely explaining the improvement in sickness behavior by chronic hM3Dq activation. No effect of CNO was observed in DREADD-negative mice. These results suggest that chronic activation of M3 muscarinic receptors (the hM3Dq progenitor) in microglia, and potentially other Gq-coupled GPCRs, preconditions microglia to decrease their response to further immunological challenges. Our results indicate that hM3Dq can be a useful tool to modulate neuroinflammation and study microglial immunological memory in vivo, which may be applicable for manipulations of neuroinflammation in neurodegenerative and psychiatric diseases.HighlightsMicroglial function was manipulated by specific activation of hM3Dq signaling.Chronic hM3Dq activation prevented LPS-induced sickness behavior in mice.Microglial hM3Dq signaling modulated expression of pro-inflammatory cytokines.","AbstractDuring lytic replication of Kaposi\u2019s sarcoma-associated herpesvirus (KSHV), the gene expression landscape of a cell is remodeled to evade the immune response and create an environment favorable to viral replication. A major driver of these gene expression changes is a virally encoded, messenger RNA (mRNA)-specific endonuclease termed SOX. SOX cleaves the majority of cytoplasmic mRNAs, but does so at specific internal sites loosely defined by a degenerate sequence motif. If and how RNA sequence directs SOX targeting remained unknown. To address these questions, we used recombinant, highly purified SOX endonuclease in a series of biochemical assays to reconstitute the cleavage reaction in vitro and gain significant insight into the biochemical mechanism of both SOX target recognition and endonucleolytic cleavage. Using this system, we determined that cut site specificity is preserved with purified SOX and a validated target RNA and thus does not require additional cellular cofactors. Furthermore, we showed that SOX displays robust, sequence-specific RNA binding to residues proximal to the cleavage site, which must be presented in a particular structural context. The strength of SOX binding dictates cleavage efficiency, providing an explanation for the breadth of target RNA susceptibility observed in cells.Significance StatementKaposi\u2019s sarcoma-associated herpesvirus (KSHV) is an oncogenic human virus that causes Kaposi\u2019s sarcoma, primary effusion lymphoma, and multicentric Castleman disease. During viral replication, KSHV expresses an enzyme called SOX that cuts and inactivates the majority of cellular messenger RNAs, preventing their translation into proteins. Some mRNAs are efficiently cleaved by SOX, while others are poorly cleaved, but the mechanistic basis underlying this selectivity has remained largely unknown. Here, we reveal that the efficiency of RNA cleavage is heavily impacted by RNA sequences proximal to the cleavage site, which serve as a SOX binding platform. This helps explain both the range of RNA cleavage efficiency observed in SOX-expressing cells as well as the sequence specificity underlying SOX targeting.","AbstractSevere acute respiratory syndrome-related coronavirus 2 (SARS-CoV-2) has resulted in a pandemic and continues to spread quickly around the globe. Currently, no effective vaccine is available to prevent COVID-19 and an intense global development activity is in progress. In this context, the different technology platforms face several challenges resulting from the involvement of a new virus still not fully characterised. Finding of the right conditions for virus amplification for the development of vaccines based on inactivated or attenuated whole viral particles is among them. Here, we describe the establishment of a workflow based on shotgun tandem mass spectrometry data to guide the optimisation of the conditions for viral amplification. In parallel, we analysed the dynamic of the host cell proteome following SARS-CoV-2 infection providing a global overview of biological processes modulated by the virus and that could be further explored to identify drug targets to address the pandemic.","Since the first reports of pneumonia associated with a novel coronavirus (COVID-19) emerged in Wuhan, Hubei province, China, there have been considerable efforts to sequence the causative virus, SARS-CoV-2 (also referred to as hCoV-19) and to make viral genomic information available quickly on shared repositories. As of 30 March 2020, 7,680 consensus sequences have been shared on GISAID, the principal repository for SARS-CoV-2 genetic information. These sequences are primarily consensus sequences from clinical and passaged samples, but few reports have looked at diversity of virus populations within individual hosts or cultures. Understanding such diversity is essential to understanding viral evolutionary dynamics. Here, we characterize within-host viral diversity from a primary isolate and passaged samples, all originally deriving from an individual returning from Wuhan, China, who was diagnosed with COVID-19 and subsequently sampled in Wisconsin, United States. We use a metagenomic approach with Oxford Nanopore Technologies (ONT) GridION in combination with Illumina MiSeq to capture minor within-host frequency variants \u22651%. In a clinical swab obtained from the day of hospital presentation, we identify 15 single nucleotide variants (SNVs) \u22651% frequency, primarily located in the largest gene - ORF1a. While viral diversity is low overall, the dominant genetic signatures are likely secondary to population size changes, with some evidence for mild purifying selection throughout the genome. We see little to no evidence for positive selection or ongoing adaptation of SARS-CoV-2 within cell culture or in the primary isolate evaluated in this study.","AbstractBacterial blight in pomegranate caused by Xanthomonas axonopodis pv. punicae (Xap) is an increasing threat for pomegranate cultivation in India. To prevent the economic losses, it is pivotal to detect the infection in latent stages rather than in later stages. We have developed an enhanced method termed as loop-mediated isothermal amplification (LAMP) technique to evaluate for the latent detection of Xap in pomegranate using six set of specific primers. Three DNA intercalating dyes were used, such as Ethidium bromide, hydroxynaphthol blue (HNB) and SYBR Green resulted in visualising the positivity for LAMP assay. The reaction time and temperature were to be 65\u00b0C from 30 min onwards, for the dyes and its sensitivity was observed up to 10\u22127 ng in the LAMP assay. For field applicability, LAMP assay detected Xap on 7th day post infection while the PCR amplified Xap after 11th day post infection. Finally, the specificity of LAMP assay was validated to be positive with ten Xap isolates for its accuracy and 29 non-Xap bacterial isolates showed negative results. Moreover, this method could be used as a better alternative to PCR based methods, for early detection of the pathogens.","AbstractCoronavirus disease 2019 (COVID-19) is a new emerging human infectious disease caused by Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2, also previously known as 2019-nCoV), originated in Wuhan seafood and animal market, China. Since December 2019, more than 69,000 cases of COVID-19 have been confirmed in China and quickly spreads to other counties. Currently, researchers put their best efforts to identify effective drugs for COVID-19. The neutralizing antibody, which binds to viral capsid in a manner that inhibits cellular entry of virus and uncoating of the genome, is the specific defense against viral invaders. In this study, we investigate to identify neutralizing antibodies that can bind to SARS-CoV-2 Sipke (S) protein and interfere with the interaction between viral S protein and a host receptor by bioinformatic methods. The sequence analysis of S protein showed two major differences in the RBD region of the SARS-CoV-2 S protein compared to SARS-CoV and SARS-CoV related bat viruses (btSARS-CoV). The insertion regions were close to interacting residues with the human ACE2 receptor. Epitope analysis of neutralizing antibodies revealed that SARS-CoV neutralizing antibodies used conformational epitopes, whereas MERS-CoV neutralizing antibodies used a common linear epitope region, which contributes to form the \u03b2-sheet structure in MERS-CoV S protein and deleted in SARS-CoV-2 S protein. To identify effective neutralizing antibodies for SARS-CoV-2, the binding affinities of neutralizing antibodies with SARS-CoV-2 S protein were predicted and compared by antibody-antigen docking simulation. The result showed that CR3022 neutralizing antibody from human may have higher binding affinity with SARS-CoV-2 S protein than SARS-CoV S protein. We also found that F26G19 and D12 mouse antibodies could bind to SARS-CoV S protein with high affinity. Our findings provide crucial clues towards the development of antigen diagnosis, therapeutic antibody, and the vaccine against SARS-CoV-2.","ABSTRACTThe amphipod crustacean Parhyale hawaiensis is a blossoming model system for studies of developmental mechanisms and more recently regeneration. We have sequenced the genome allowing annotation of all key signaling pathways, transcription factors, and non-coding RNAs that will enhance ongoing functional studies. Parhyale is a member of the Malacostraca clade, which includes crustacean food crop species. We analysed the immunity related genes of Parhyale as an important comparative system for these species, where immunity related aquaculture problems have increased as farming has intensified. We also find that Parhyale and other species within Multicrustacea contain the enzyme sets necessary to perform lignocellulose digestion (\u201cwood eating\u201d), suggesting this ability may predate the diversification of this lineage. Our data provide an essential resource for further development of Parhyale as an experimental model. The first malacostracan genome will underpin ongoing comparative work in food crop species and research investigating lignocellulose as an energy source.","Pandemics originating from pathogen transmission between animals and humans highlight the broader need to understand how natural hosts have evolved in response to emerging human pathogens and which groups may be susceptible to infection. Here, we investigate angiotensin-converting enzyme 2 (ACE2), the host protein bound by SARS-CoV and SARS-CoV-2. We find that the ACE2 gene is under strong selection pressure in bats, the group in which the progenitors of SARS-CoV and SARS-CoV-2 are hypothesized to have evolved, particularly in residues that contact SARS-CoV and SARS-CoV-2. We detect positive selection in non-bat mammals in ACE2 but in a smaller proportion of branches than in bats, without enrichment of selection in residues that contact SARS-CoV or SARS-CoV-2. Additionally, we evaluate similarity between humans and other species in residues that contact SARS-CoV or SARS-CoV-2, revealing potential susceptible species but also highlighting the difficulties of predicting spillover events. This work increases our understanding of the relationship between mammals, particularly bats, and coronaviruses, and provides data that can be used in functional studies of how host proteins are bound by SARS-CoV and SARS-CoV-2 strains.","AbstractAntibiotic resistance often generates a fitness cost to bacteria in drug-free environments. Understanding the causes of the cost is considered the Holy Grail in the antibiotic resistance field, as it is the main determinant of the prevalence of resistances upon reducing antibiotics use. We show that DNA breaks can explain most of the variation in the cost of resistances common in pathogens. Here we demonstrate that targeting the RNase that degrades R-loops, which cause DNA breaks, exacerbates the cost of resistance. Consequently, lack of RNase HI function drives resistant clones to extinction in populations with high initial frequency of resistance, both in laboratory conditions and in a mouse model of gut colonization. Thus, RNase HI provides a target specific against resistant bacteria, which we validate using a repurposed drug. In summary, we revealed key mechanisms underlying the cost of antibiotic resistance that can be exploited to specifically eliminate resistant bacteria.","ABSTRACTAs a sensitive signaling system, the mitotic checkpoint ensures faithful chromosome segregation by delaying anaphase onset when even a single kinetochore is unattached. The key signal amplification reaction for the checkpoint is the conformational conversion of open MAD2 (O-MAD2) into closed MAD2 (C-MAD2). The reaction was suggested to be catalyzed by an unusual catalyst, a MAD1:C-MAD2 tetramer, but how the catalysis is executed and regulated remains elusive. Here we report that in addition to the well-characterized middle region (MIM), both amino- and carboxyl-terminal domains (NTD and CTD) of MAD1 also contribute to the mitotic checkpoint. In contrast to MIM that stably associates with C-MAD2, MAD1-NTD and CTD surprisingly bind to both O-MAD2 and C-MAD2, suggesting their interactions with substrates and products of the O-C conversion. MAD1-NTD also interacts with CTD. MPS1 kinase interacts with and phosphorylates both NTD and CTD. The phosphorylation reduces the NTD:CTD interaction and CTD interaction with MPS1. Mutating CTD phosphorylation sites including Thr716 compromises MAD2 binding and the checkpoint responses. Ser610 and Tyr634 also contribute to the checkpoint. Our results have uncovered previously unknown interactions of MAD1-NTD and CTD with MAD2 conformers and their regulation by MPS1 kinase, providing novel insights into the mitotic checkpoint signaling.","AbstractThe global spread of SARS-CoV-2 requires an urgent need to find effective therapeutics for the treatment of COVID-19. We developed a data-driven drug repositioning framework, which applies both machine learning and statistical analysis approaches to systematically integrate and mine large-scale knowledge graph, literature and transcriptome data to discover the potential drug candidates against SARS-CoV-2. The retrospective study using the past SARS-CoV and MERS-CoV data demonstrated that our machine learning based method can successfully predict effective drug candidates against a specific coronavirus. Our in silico screening followed by wet-lab validation indicated that a poly-ADP-ribose polymerase 1 (PARP1) inhibitor, CVL218, currently in Phase I clinical trial, may be repurposed to treat COVID-19. Our in vitro assays revealed that CVL218 can exhibit effective inhibitory activity against SARS-CoV-2 replication without obvious cytopathic effect. In addition, we showed that CVL218 is able to suppress the CpG-induced IL-6 production in peripheral blood mononuclear cells, suggesting that it may also have anti-inflammatory effect that is highly relevant to the prevention immunopathology induced by SARS-CoV-2 infection. Further pharmacokinetic and toxicokinetic evaluation in rats and monkeys showed a high concentration of CVL218 in lung and observed no apparent signs of toxicity, indicating the appealing potential of this drug for the treatment of the pneumonia caused by SARS-CoV-2 infection. Moreover, molecular docking simulation suggested that CVL218 may bind to the N-terminal domain of nucleocapsid (N) protein of SARS-CoV-2, providing a possible model to explain its antiviral action. We also proposed several possible mechanisms to explain the antiviral activities of PARP1 inhibitors against SARS-CoV-2, based on the data present in this study and previous evidences reported in the literature. In summary, the PARP1 inhibitor CVL218 discovered by our data-driven drug repositioning framework can serve as a potential therapeutic agent for the treatment of COVID-19.","AbstractOver the past several decades, multiple coronaviruses (CoVs) have emerged as highly infectious, lethal viruses in humans, most notably in the pandemic outbreak of COVID-19, the disease caused by SARS-CoV-2. To date, there are no known therapeutic or preventative agents to target CoVs. Though age and comorbidities severely increase case fatality rates, the host factors that influence resistance or susceptibility to infection with highly pathogenic human CoVs are unknown. Innate immune responses to CoVs are initiated by recognition of double-stranded (ds) RNA and induction of interferon, which turns on a gene expression program that inhibits viral replication. SARS-CoV-2 conserves an ADP-ribosylhydrolase domain previously shown to counteract innate immunity to both mouse hepatitis virus (MHV), a model CoV, and SARS-CoV. Here we show that SARS-CoV-2 infection of cell lines, infected ferrets, and a deceased patient\u2019s lung consistently and strikingly dysregulates the nicotinamide adenine dinucleotide (NAD+) gene set with respect to NAD+ synthesis and utilization. SARS-CoV-2 induces a set of poly(ADP-ribose) polymerase (PARP) family members; these PARPs include enzymes required for the innate immune response to MHV. Further, we show that MHV infection induces an attack on host cell nicotinamide adenine dinucleotide (NAD+) and nicotinamide adenine dinucleotide phosphate (NADP+). The data indicate that overexpression of a virally induced PARP, PARP10, is sufficient to depress host cell NAD metabolism and that NAD+ boosting strategies differ in their efficacy to restore PARP10 function. Gene expression and pharmacological data suggest that boosting NAD+ through the nicotinamide and nicotinamide riboside kinase pathways may restore antiviral PARP functions to support innate immunity to SARS-CoV-2, whereas PARP1,2 inhibition may be less likely to restore antiviral PARP functions.","AbstractEmerging and re-emerging RNA viruses occasionally cause epidemics and pandemics worldwide, such as the on-going outbreak of coronavirus SARS-CoV-2. Existing direct-acting antiviral (DAA) drugs cannot be applied immediately to new viruses because of virus-specificity, and the development of new DAA drugs from the beginning is not timely for outbreaks. Thus, host-targeting antiviral (HTA) drugs have many advantages to fight against a broad spectrum of viruses, by blocking the viral replication and overcoming the potential viral mutagenesis simultaneously. Herein, we identified two potent inhibitors of DHODH, S312 and S416, with favorable drug-like and pharmacokinetic profiles, which all showed broad-spectrum antiviral effects against various RNA viruses, including influenza A virus (H1N1, H3N2, H9N2), Zika virus, Ebola virus, and particularly against the recent novel coronavirus SARS-CoV-2. Our results are the first to validate that DHODH is an attractive host target through high antiviral efficacy in vivo and low virus replication in DHODH knocking-out cells. We also proposed the drug combination of DAA and HTA was a promising strategy for anti-virus treatment and proved that S312 showed more advantageous than Oseltamivir to treat advanced influenza diseases in severely infected animals. Notably, S416 is reported to be the most potent inhibitor with an EC50 of 17nM and SI value &gt;5882 in SARS-CoV-2-infected cells so far. This work demonstrates that both our self-designed candidates and old drugs (Leflunomide/Teriflunomide) with dual actions of antiviral and immuno-repression may have clinical potentials not only to influenza but also to COVID-19 circulating worldwide, no matter such viruses mutate or not.","Hydroxychloroquine (HCQ) has emerged as a potential and controversial antiviral candidate therapy for COVID-19. While many clinical trials are underway to test the efficacy of HCQ as a treatment for COVID-19, underlying mechanisms of HCQ in the setting of COVID-19 remain unclear. Hence, we examined differential gene expression signatures of HCQ exposure, in vitro SARS-CoV-2 infection, and host signatures of COVID-19 in blood, bronchoalveolar lavage, and postmortem lung to evaluate whether HCQ transcriptome signatures associate with restoration of SARS-CoV-2-related host transcriptional responses. Here, we show that 24 hours of in vitro treatment of peripheral blood mononuclear cells (PBMC) with HCQ significantly impacted transcription of 16 genes involved in immune regulation and lipid metabolism. Using transcriptome data from in vitro SARS-CoV-2 infected NHBE and A549 cells and PBMC derived from confirmed COVID-19 infected patients, we determined that only 0.24% of the COVID-19 PBMC differentially expressed gene set and 0.39% of the in vitro SARS-CoV-2 cells differentially expressed gene set overlapped with HCQ-related differentially expressed genes. Moreover, we observed that HCQ treatment significantly impacted transcription of 159 genes in human primary monocyte-derived macrophages involved in cholesterol biosynthetic process and chemokine activity. Notably, when we compared the macrophage HCQ-related gene lists with genes transcriptionally altered during SARS-CoV-2 infection and in bronchoalveolar lavage of COVID-19+ patients, the CXCL6 gene was impacted in all three transcriptional signatures revealing evidence in favor of chemokine modulation. HCQ-related transcriptional changes minimally overlapped with host genes altered in postmortem lung biopsies from COVID-19 participants. These results may provide insight into the immunomodulation mechanisms of HCQ treatment in the setting of COVID-19 and suggest HCQ is not a panacea to SARS-CoV-2 infection.","AbstractMass spectrometry (MS) can deliver valuable diagnostic data that complements genomic information and allows us to increase our current knowledge of the COVID-19 disease caused by the SARS-CoV-2 virus. We developed a simple, MS-based method to specifically detect SARS-CoV-2 proteins from gargle solution samples of COVID-19 patients. Our protocol consists of an acetone precipitation and tryptic digestion of proteins contained within the gargle solution, followed by a targeted MS analysis. Our methodology identifies unique peptides originating from SARS-CoV-2 nucleoprotein. Building on these promising initial results, faster MS protocols can now be developed as routine diagnostic tools for COVID-19 patients.Graphical AbstractImage credit (left): Gerd Altmann, Pixabay License, https://pixabay.com/illustrations/corona-coronavirus-virus-covid-19-4959447","ABSTRACTExtracellular \u03b2-amyloid (A\u03b2) plaque deposits and inflammatory immune activation are thought to alter various aspects of tissue microstructure, such as extracellular free water, fractional anisotropy and diffusivity, as well as the density and geometric arrangement of axonal processes. Quantifying these microstructural changes in Alzheimer\u2019s disease and related neurodegenerative dementias could serve to accurately monitor or predict disease course. In the present study we used high-field diffusion magnetic resonance imaging (dMRI) to determine how A\u03b2 and inflammatory interleukin-6 (IL6), alone or in combination, affect in vivo tissue microstructure in the TgCRND8 mouse model of Alzheimer\u2019s-type A\u03b2 deposition. TgCRND8 and non-transgenic (nTg) mice expressing brain-targeted IL6 or enhanced glial fibrillary protein (EGFP controls) were scanned at 8 months of age using a 2-shell, 54-gradient direction dMRI sequence at 11.1 Tesla. Images were processed using the free water elimination method and the neurite orientation dispersion and density imaging (NODDI) model. DTI and NODDI processing in TgCRND8 mice revealed a microstructure pattern consistent with reduced white matter integrity along with an increase in density and geometric complexity of axonal and dendritic processes. This included reduced FA, mean diffusivity (MD), and free water (FW), and increased \u2018neurite\u2019 density (NDI) and orientation dispersion (ODI). IL6 produced a \u2018protective-like\u2019 effect on FA in TgCRND8 mice, although there were minimal microstructure changes in these mice compared IL6 expressing nTg mice. In addition, we found that NDI and ODI had an inverse relationship with the functional connectome clustering coefficient, which was affected by A\u03b2 and IL6. The relationship between NODDI and graph theory metrics suggests that increasing the density and orientation dispersion of neurites may relate to diminished functional network organization in the brain.","AbstractBackgroundFebrile illness is a major burden in African children, and non-malarial causes of fever are uncertain. We built and employed IDseq, a cloud-based, open access, bioinformatics platform and service to identify microbes from metagenomic next-generation sequencing of tissue samples. In this pilot study, we evaluated blood, nasopharyngeal, and stool specimens from 94 children (aged 2-54 months) with febrile illness admitted to Tororo District Hospital, Uganda.ResultsThe most common pathogens identified were Plasmodium falciparum (51.1% of samples) and parvovirus B19 (4.4%) from blood; human rhinoviruses A and C (40%), respiratory syncytial virus (10%), and human herpesvirus 5 (10%) from nasopharyngeal swabs; and rotavirus A (50% of those with diarrhea) from stool. Among other potential pathogens, we identified one novel orthobunyavirus, tentatively named Nyangole virus, from the blood of a child diagnosed with malaria and pneumonia, and Bwamba orthobunyavirus in the nasopharynx of a child with rash and sepsis. We also identified two novel human rhinovirus C species.ConclusionsThis exploratory pilot study demonstrates the utility of mNGS and the IDseq platform for defining the molecular landscape of febrile infectious diseases in resource limited areas. These methods, supported by a robust data analysis and sharing platform, offer a new tool for the surveillance, diagnosis, and ultimately treatment and prevention of infectious diseases.","ABSTRACTBackgroundEstimating infectious disease parameters such as the serial interval (time between symptom onset in primary and secondary cases) and reproductive number (average number of secondary cases produced by a primary case) are important to understand infectious disease dynamics. Many estimation methods require linking cases by direct transmission, a difficult task for most diseases.MethodsUsing a subset of cases with detailed genetic or contact investigation data to develop a training set of probable transmission events, we build a model to estimate the relative transmission probability for all case-pairs from demographic, spatial and clinical data. Our method is based on naive Bayes, a machine learning classification algorithm which uses the observed frequencies in the training dataset to estimate the probability that a pair is linked given a set of covariates.ResultsIn simulations we find that the probabilities estimated using genetic distance between cases to define training transmission events are able to distinguish between truly linked and unlinked pairs with high accuracy (area under the receiver operating curve value of 95%). Additionally only a subset of the cases, 10-50% depending on sample size, need to have detailed genetic data for our method to perform well. We show how these probabilities can be used to estimate the average effective reproductive number and apply our method to a tuberculosis outbreak in Hamburg, Germany.ConclusionsOur method is a novel way to infer transmission dynamics in any dataset when only a subset of cases has rich contact investigation and/or genetic data.KEY MESSAGESThis method provides a way to calculate the relative probability that two infectious disease patients are connected by direct transmission using clinical, demographic, geographic, and genetic characteristics.We use a na\u00efve Bayes, a machine learning technique to estimate these probabilities using a training set of probable links defined by contact investigation or pathogen WGS data on a subset of cases.These probabilities can be used to explore possible transmission chains, rule out transmission events, and estimate the reproductive number.","AbstractAntibiotics are widely used to treat infections in humans. However, the impact of antibiotic use on host cells is understudied. We have identified a novel antiviral effect of commonly used aminoglycoside antibiotics. We show that mucosal application of aminoglycosides increased host resistance to a broad range of viral infections including herpes simplex viruses, influenza A virus and Zika virus. Aminoglycoside treatment also reduced viral replication in primary human cells. This antiviral activity was independent of the microbiota as aminoglycoside treatment protected germ-free mice. Microarray analysis uncovered a marked upregulation of transcripts for interferon-stimulated genes (ISGs) following aminoglycoside application. ISG induction was mediated by TLR3, and required TIR-domain-containing adapter-inducing interferon-\u03b2 (TRIF), signaling adaptor, and interferon regulatory factors 3 (IRF3) and IRF7, transcription factors that promote ISG expression. XCR1+ dendritic cells, which uniquely express TLR3, were recruited to the vaginal mucosa upon aminoglycoside treatment and were required for ISG induction. These results highlight an unexpected ability of aminoglycoside antibiotics to confer broad antiviral resistance in vivo.","AbstractThe current emergence of the novel coronavirus pandemic caused by SARS-CoV-2 demands the development of new therapeutic strategies to prevent rapid progress of mortalities. The coronavirus spike (S) protein, which facilitates viral attachment, entry and membrane fusion is heavily glycosylated and plays a critical role in the elicitation of host immune response. The spike protein is comprised of two protein subunits (S1 and S2), which together possess 22 potential N-glycosylation sites. Herein, we report the mapping of glycosylation on spike protein subunits S1 and S2 expressed on human cells through high resolution mass spectrometry. We have characterized the quantitative N-glycosylation profile on spike protein and interestingly, observed two unexpected O-glycosylation modifications on the receptor binding domain (RBD) of spike protein subunit S1. Even though O-glycosylation has been predicted on the spike protein of SARS-Cov-2, this is the first report of experimental data for both the site of O-glycosylation and identity of the O-glycans attached on the subunit S1. Our data on the N- and O-glycosylation is strengthened by extensive manual interpretation of each glycopeptide spectra in addition to using bioinformatics tools to confirm the complexity of glycosylation in the spike protein. The elucidation of the glycan repertoire on the spike protein provides insights into the viral binding studies and more importantly, propels research towards the development of a suitable vaccine candidate.","AbstractThe African swine fever virus (ASFV) has severely influenced the swine industry of the world. Unfortunately, there is no effective antiviral drug or vaccine against the virus until now. Identification of new anti-ASFV drugs is urgently needed. Here, an up-to-date set of protein-protein interactions (PPIs) between ASFV and swine were curated by integration of PPIs from multiple sources. Thirty-two swine proteins were observed to interact with ASFVs and were defined as AIPs. They were found to play a central role in the swine PPI network, with significant larger degree, betweenness and smaller shortest path length than other swine proteins. Some of AIPs also interacted with several other viruses and could be taken as potential targets of drugs for broad-spectrum effect, such as HSP90AB1. Finally, the antiviral drugs which targeted AIPs and ASFV proteins were predicted. Several drugs with either broad-spectrum effect or high specificity on AIPs were identified, such as Polaprezinc. This work could not only deepen our understanding towards the ASFV-swine interactions, but also help for the development of effective antiviral drugs against the ASFVs.","Determining the fitness of viral genotypes has become a standard practice in virology as it is essential to evaluate their evolutionary potential. Darwinian fitness, defined as the advantage of a given genotype with respect to a reference one, is a mesoscopic property that captures into a single figure differences in performance at every stage of viral infection. But to which extent viral fitness results from particular molecular interactions with host factors and regulatory networks during infection? Can we identify host genes, and then functional classes, whose expression depends on viral fitness? Here, we compared the transcriptomes of tobacco plants infected with seven genotypes of tobacco etch potyvirus (TEV) that differ in fitness. We found that the larger the fitness differences among genotypes, the more dissimilar the transcriptomic profiles are. Consistently, two different mutations, one in the viral RNA polymerase and another in the viral suppressor of RNA silencing, that led to close fitness values, also resulted in significantly similar gene expression profiles. Moreover, we identified host genes whose expression showed a significant correlation, positive or negative, with TEV fitness. Over-expression of genes with positive correlation activates hormone-and RNA silencing-mediated pathways of plant defense. By contrast, under-expression of genes negatively correlated reduces metabolism, growth, and development. Overall, these results reveal the high information content of viral fitness, and suggest its potential use to predict differences in genomic profiles of infected hosts.","Fast testing can help mitigate the coronavirus disease 2019 (COVID-19) pandemic. Despite their accuracy for single sample analysis, infectious diseases diagnostic tools, like RT-PCR, require substantial resources to test large populations. We develop a scalable approach for determining the viral status of pooled patient samples. Our approach converts group testing to a linear inverse problem, where false positives and negatives are interpreted as generated by a noisy communication channel, and a message passing algorithm estimates the illness status of patients. Numerical results reveal that our approach estimates patient illness using fewer pooled measurements than existing noisy group testing algorithms. Our approach can easily be extended to various applications, including where false negatives must be minimized. Finally, in a Utopian world we would have collaborated with RT-PCR experts; it is difficult to form such connections during a pandemic. We welcome new collaborators to reach out and help improve this work!","AbstractIn this study, we used pan RNA-seq analysis to reveal the ubiquitous existence of 5\u2019 end and 3\u2019 end small RNAs. 5\u2019 and 3\u2019 sRNAs alone can be used to annotate mitochondrial with 1-bp resolution and nuclear non-coding genes and identify new steady-state RNAs, which are usually from functional genes. Using 5\u2019, 3\u2019 and intronic sRNAs, we revealed that the enzymatic dsRNA cleavage and RNAi could involve in the RNA degradation and gene expression regulation of U1 snRNA in human. The further study of 5\u2019, 3\u2019 and intronic sRNAs help rediscover double-stranded RNA (dsRNA) cleavage, RNA interference (RNAi) and the regulation of gene expression, which challenges the classical theories. In this study, we provided a simple and cost effective way for the annotation of mitochondrial and nuclear non-coding genes and the identification of new steady-state RNAs, particularly long non-coding RNAs (lncRNAs). We also provided a different point of view for cancer and virus, based on the new discoveries of dsRNA cleavage, RNAi and the regulation of gene expression.","Viruses are one of the major causes of various acute and chronic infectious diseases and thus a major contributor to the global burden of disease. Several studies have shown how viruses have evolved to hijack basic cellular pathways and evade innate immune response by modulating key host factors and signalling pathways. A collective view of these multiple studies could advance our understanding of viral evasion mechanisms and provide new therapeutic perspectives for the treatment of viral diseases. Here, we performed an integrative meta-analysis to elucidate the 17 different host-virus interactomes. Network and bioinformatics analyses showed how viruses with small genomes efficiently achieve the maximal effect by targeting multifunctional and highly connected host proteins with a high occurrence of disordered regions. We also identified the core cellular process subnetworks that are targeted by all the viruses. Integration with functional RNA interference (RNAi) datasets showed that a large proportion of the targets are required for viral replication. Furthermore, we performed an interactome-informed drug re-purposing screen and identified novel activities for broad-spectrum antiviral agents against hepatitis C virus and human metapneumovirus. Altogether, these orthogonal datasets could serve as a platform for hypothesis generation and follow-up studies to broaden our understanding of the viral evasion landscape.","ABSTRACTFormation of stress granules (SGs), cytoplasmic condensates of stalled translation initiation complexes, is regulated by post-translational protein modification. Alphaviruses interfere with SG formation in response to inhibition of host protein synthesis through the activities of nonstructural protein 3 (nsP3). nsP3 has a conserved N-terminal macrodomain that binds and can remove ADP-ribose from ADP-ribosylated proteins and a C-terminal hypervariable domain that binds essential SG component G3BP1. We showed that the hydrolase activity of chikungunya virus nsP3 macrodomain removed ADP-ribosylation of G3BP1 and suppressed SG formation. ADP-ribosylhydrolase-deficient nsP3 mutants allowed stress-induced cytoplasmic condensation of translation initiation factors. nsP3 also disassembled SG-like aggregates enriched with translation initiation factors that are induced by the expression of FUS mutant R495X linked to amyotrophic lateral sclerosis. Therefore, our data indicate that regulation of ADP-ribosylation controls the localization of translation initiation factors during virus infection and other pathological conditions.","AbstractBats are recognized as important reservoirs of viruses deadly to other mammals, including humans. These infections are typically nonpathogenic in bats raising questions about innate immune differences that might exist between bats and other mammals. Tetherin is an important restriction factor which inhibits the release of a diverse range of viruses from host cells, including retroviruses, coronaviruses, filoviruses, and paramyxoviruses, some of which are deadly to humans and transmitted by bats. Here we characterise the tetherin genes from 27 species of bats, revealing that they are under strong selective pressure, and that fruit bats and vesper bats express unique structural variants of the tetherin protein. The expression of two computationally predicted splice isoforms of fruit bat tetherin is verified here, and we identify an additional third unique splice isoform which includes a C-terminal region that is not homologous to known mammalian tetherin variants. We also report that vesper bats possess and express at least five tetherin genes, including structural variants, a greater number than any other mammal reported to date. These findings support the hypothesis of differential antiviral gene evolution in bats relative to other mammals.","AbstractThe outbreak of 2019-nCoV pneumonia (COVID-19) in the city of Wuhan, China has resulted in more than 70,000 laboratory confirmed cases, and recent studies showed that 2019-nCoV (SARS-CoV-2) could be of bat origin but involve other potential intermediate hosts. In this study, we assembled the genomes of coronaviruses identified in sick pangolins. The molecular and phylogenetic analyses showed that pangolin Coronaviruses (pangolin-CoV) are genetically related to both the 2019-nCoV and bat Coronaviruses but do not support the 2019-nCoV arose directly from the pangolin-CoV. Our study also suggested that pangolin be natural host of Betacoronavirus, with a potential to infect humans. Large surveillance of coronaviruses in pangolins could improve our understanding of the spectrum of coronaviruses in pangolins. Conservation of wildlife and limits of the exposures of humans to wildlife will be important to minimize the spillover risks of coronaviruses from wild animals to humans.","The development of molecular syndrome-based kits for the diagnosis of respiratory infections offers rapid and sensitive detection of common respiratory pathogens and will have a significant impact on the care of patients. In this study, we present the results obtained after the introduction of the FilmArray respiratory panel in a field Point of care (POC) for the diagnosis of virus and bacteria responsible of respiratory tract infections in Senegal rural area. From February to August 2017, we collected nasal swabs from febrile patients that presented symptoms of respiratory tract infections in three health posts located in Niakhar. Specimens were tested on site by multiplex Polymerase Chain Reaction (PCR), using the FilmArray respiratory panel\u00ae that targets 20 pathogens, including 17 virus and 3 bacteria (bioM\u00e9rieux). Nasal swabs were collected from 113 patients. The median age was 4 years (ranging from 4 months to 60 years) and 51 (45%) were males. The prevalence of respiratory pathogens was 37.5% (12/32) during the dry season and 54.3% (44/81) in the rainy season (p=0.16). The prevalence of respiratory pathogen carriage was higher in children under 5 years of age (38/55, 69.1%). The most prevalent micro-organisms detected were influenza B virus (16/113, 14%), human rhinovirus/enterovirus (10/113, 9%), parainfluenzae virus (9/113, 8%), respiratory syncytial virus (8/113, 7%), adenovirus (5/113, 4%), human metapneumovirus (3/113, 3%), Chlamydia pneumoniae (2/113, 2%) and Coronavirus (2/113, 2%). The study has demonstrated that the integration of the FilmArray respiratory panel into a field POC could significantly improve the management of respiratory tract infections in rural areas.","ABSTRACTFollowing reovirus infection, cells activate stress responses that repress canonical cellular translation as a mechanism to limit production of progeny virions. This includes the formation of stress granules (SG) that sequester translationally-stalled cellular transcripts, translation initiation factors, ribosomal proteins, and RNA binding proteins until conditions improve and translation can resume. Work by others suggests that these cellular stress responses, which are part of the integrated stress response, may benefit rather than repress reovirus replication. In agreement with this, we report that stressing cells prior to infection with sodium arsenite (SA), a robust inducer of SG and activator of eIF2\u03b1 kinases, enhanced viral protein expression, percent infectivity and viral titer in SA-treated cells compared to untreated cells. SA-mediated enhancement of reovirus replication was not strain-specific, but was cell-type specific. While pre-treatment of cells with SA offered the greatest enhancement, treatment of infected cultures as late as 4 h post infection resulted in an increase in the percent of cells infected. SA activates the HRI kinase, which phosphorylates eIF2\u03b1 and subsequently induces SG formation. Other stresses, such as heat shock (HS) and osmotic shock also activate HRI. Heat shock of cells prior to reovirus infection readily induced SG in greater than 85% of cells. Although HS pre-treatment had no effect on the percentage of infected cells or viral yield, it did enhance viral protein expression. These data suggest that SA pre-treatment perturbs the cell in a way that is beneficial for reovirus and that neither HRI activation nor SG induction is sufficient for reovirus infection enhancement.SIGNIFICANCEAll viruses rely on the host translational machinery for the synthesis of viral proteins. In response to viral infection, cells activate the integrated stress response resulting in the phosphorylation of eIF2\u03b1 and translation shutoff. Despite this, reovirus replicates to reduced titers in the absence of this response. In this work, we report that sodium arsenite activation of the integrated stress response prior to virus inoculation enhances virus infectivity, protein expression and titer. Together, these data suggest that modulation of conserved cellular stress responses can alter reovirus replication.","AbstractFinding novel biomarkers for human pathologies and predicting clinical outcomes for patients is rather challenging. This stems from the heterogenous response of individuals to disease which is also reflected in the inter-individual variability of gene expression responses. This in turn obscures differential gene expression analysis (DGE). In the midst of the COVID-19 pandemic, we wondered whether an alternative to DGE approaches could be applied to dissect the molecular nature of a host-response to infection exemplified here by an analysis of H1N1 influenza, community/hospital acquired pneumonia (CAP) and sepsis. To this end, we turned to the analysis of ensemble gene noise. Ensemble gene noise, as we defined it here, represents a variance within an individual for a collection of genes encoding for either members of known biological pathways or subunits of annotated protein complexes. From the law of total variance, ensemble gene noise depends on the stoichiometry of the ensemble genes\u2019 expression and on their average noise (variance). Thus, rather than focusing on specific genes, ensemble gene noise allows for the holistic identification and interpretation of gene expression disbalance on the level of gene networks and systems. Comparing H1N1, CAP and sepsis patients we spotted common disturbances in a number of pathways/protein complexes relevant to the sepsis pathology which lead to an increase in the ensemble gene noise. Among others, these include mitochondrial respiratory chain complex I and peroxisomes which could be readily targeted for adjuvant treatment by methylene blue and 4-phenylbutyrate respectively. Finally, we showed that ensemble gene noise could be successfully applied for the prediction of clinical outcome, namely mortality, of CAP and sepsis patients. Thus, we conclude that ensemble gene noise represents a promising approach for the investigation of molecular mechanisms of a pathology through a prism of alterations in coherent expression of gene circuits.","Cryo-EM single-particle analysis has proven powerful in determining the structures of rigid macromolecules. However, many protein complexes are flexible and can change conformation and composition as a result of functionally-associated dynamics. Such dynamics are poorly captured by current analysis methods. Here, we present cryoDRGN, an algorithm that for the first time leverages the representation power of deep neural networks to efficiently reconstruct highly heterogeneous complexes and continuous trajectories of protein motion. We apply this tool to two synthetic and three publicly available cryo-EM datasets, and we show that cryoDRGN provides an interpretable representation of structural heterogeneity that can be used to identify discrete states as well as continuous conformational changes. This ability enables cryoDRGN to discover previously overlooked structural states and to visualize molecules in motion.","Lay summaryOur researchers took a look at a sequence of DNA known as the ACE2 gene. This gene is most well known for its role in regulating blood pressure. But in recent times, it\u2019s drawn a lot of attention from the scientific community because it may also serve as a doorway of sorts, enabling viruses like SARS-CoV-2 to infect cells. Our researchers looked at the ACE2 gene in more than 200,000 people, comparing their exact DNA sequences to see where there are differences among people. Variation in the DNA sequence of a gene is common and is sometimes meaningless. But other times, small changes in the DNA sequence can alter the protein that is made from that gene. In this case the ACE2 gene makes the ACE2 protein, which is what the SARS-CoV-2 virus interacts with. We found a lot of variation between individuals and checked to see if that variation coincided with any traits (i.e., people with variant X tend to have high blood pressure more often than people without variant X). All of the traits we looked at were non-COVID-19-related traits, meaning we haven\u2019t asked these people anything about COVID-19 yet (this is because these DNA sequences were collected before the pandemic).We found that there are a number of variations observed among people in a specific part of the ACE2 gene. These variations are expected to alter the shape or functionality of a specific part of the ACE2 protein: The part that interacts with the SARS-CoV-2 virus. We don\u2019t yet know what the real-life significance of this variation is, but it\u2019s possible that these variants decrease the protein\u2019s ability to interact with the SARS-CoV-2 virus, thus decreasing the person\u2019s likelihood of being infected. We can speculate that there will be a spectrum of vulnerability to COVID-19 among people, where some people are more vulnerable than others, and that variants in this part of the ACE2 gene may be one of the reasons. The research we presented here shines a light on this part of the ACE2 gene and may give future researchers a direction to go in as they try to figure out what makes people vulnerable to COVID-19 and similar viruses.","AbstractSARS-COV-2 and all other coronaviruses express its 3 prime genes by forming sub-genomic RNA. As the genome of these virus exist in RNA form, only by profiling the relative abundance of these sgRNAs, can the viral transcriptome be revealed. Utilizing publically available meta-transcriptomic data generated from patient samples, we were able to infer the viral transcriptome in vivo, which is distinct from the in vitro one derived from cell culture. Inter-sample diversity was also observed and a sample specific transcript was identified. By doing the same analysis to MERS and SARS data, we were able to compare the three in terms of transcription. Among the differences, SARS-COV-2 has significantly elevated expression of the Spike gene, which may contribute to its high transmissibility.HighlightsThe in vivo transcriptome of SARS-CoV-2 revealed by sgRNA profiling, for 25 patient samples around the globe.The Spike protein expression is an order of magnitude higher in SARS-CoV-2 than MERS-CoV or SARS-CoV, possibly contributing to the virus\u2019 elevated transmissibility.The in vivo SARS-CoV-2 transcriptomes, as inferred from human patient data was distinct from the in vitro one derived from cell line culture, all the accessory genes were up-regulated in vivo, suggesting intricate expression regulation mechanism for the small viral genome.","The outbreak of the 2019 Novel Coronavirus (SARS-CoV-2) rapidly spread from Wuhan, China to more than 150 countries, areas or territories, causing staggering number of infections and deaths. A systematic profiling of the immune vulnerability landscape of SARS-CoV-2, which can bring critical insights into the immune clearance mechanism, peptide vaccine development, and antiviral antibody development, is lacking. In this study, we investigated the potential of the SARS-CoV-2 viral proteins to induce class I and II MHC presentation and to form linear antibody epitopes. We created an online database to broadly share the predictions as a resource for the research community. Using this resource, we showed that genetic variations in SARS-CoV-2, though still few for the moment, already follow the pattern of mutations in related coronaviruses, and could alter the immune vulnerability landscape of this virus. Importantly, we discovered evidence that SARS-CoV-2, along with related coronaviruses, used mutations to evade attack from the human immune system. Overall, we present an immunological resource for SARS-CoV-2 that could promote both therapeutic development and mechanistic research.","An outbreak of the novel coronavirus SARS-CoV-2, the causative agent of COVID-19 respiratory disease, has infected over 290,000 people since the end of 2019, killed over 12,000, and caused worldwide social and economic disruption. There are currently no antiviral drugs with proven efficacy nor are there vaccines for its prevention. Unfortunately, the scientific community has little knowledge of the molecular details of SARS-CoV-2 infection. To illuminate this, we cloned, tagged and expressed 26 of the 29 viral proteins in human cells and identified the human proteins physically associated with each using affinity-purification mass spectrometry (AP-MS), which identified 332 high confidence SARS-CoV-2-human protein-protein interactions (PPIs). Among these, we identify 67 druggable human proteins or host factors targeted by 69 existing FDA-approved drugs, drugs in clinical trials and/or preclinical compounds, that we are currently evaluating for efficacy in live SARS-CoV-2 infection assays. The identification of host dependency factors mediating virus infection may provide key insights into effective molecular targets for developing broadly acting antiviral therapeutics against SARS-CoV-2 and other deadly coronavirus strains.","AbstractScientific conferences provide valuable opportunities for researchers across career stages and disciplines to present their latest work and to network with their peers. The advent of the internet has opened new possibilities for interaction, collaboration and networking, yet the uptake of tools enabling remote participation at scientific meetings has been slow. Academic conferences have become more international with high proliferation in the number of meetings and attendees, but the format and quality of their organization lags behind what is possible. As a result, the experience of attending conferences in many disciplines today is not appreciably different in many respects. A few conferences in some disciplines have recently implemented valuable changes for the community and have become more receptive to attendees with families. Many meetings could still be improved significantly in terms of diversity, inclusivity, promoting early career researcher (ECR) networking and career development, venue accessibility, and more importantly, reducing the meetings\u2019 carbon footprint. These issues are openly visible and to some extent discussed among researchers on social media. It is important to accelerate and mandate these efforts so that researchers in all disciplines, in particular ECRs, consistently benefit from scientific gatherings, for years to come. We examined the current state of over 200 national and international academic meetings in various disciplines for key features of inclusivity and sustainability and propose solutions to make conferences more modern, effective, equitable and intellectually productive for the research community and environmentally sustainable for our planet. \u201cThey always say time changes things, but you actually have to change them yourself.\u201d\u2014 Andy Warhol","AbstractSince 2002, beta coronaviruses (CoV) have caused three zoonotic outbreaks, SARS-CoV in 2002, MERS-CoV in 2012, and the recent outbreak of SARS-CoV-2 late in 2019 (also named as COVID-19 or novel coronavirus 2019 or nCoV2019. Spike(S) protein, one of the structural proteins of this virus plays key role in receptor (ACE2) binding and thus virus entry. Thus, this protein has attracted scientists for detailed study and therapeutic targeting. As the 2019 novel coronavirus takes its course throughout the world, more and more sequence analyses are been done and genome sequences getting deposited in various databases. From India two clinical isolates have been sequenced and the full genome deposited in GenBank. We have performed sequence analyses of the spike protein of the Indian isolates and compared with that of the Wuhan, China (where the outbreak was first reported). While all the sequences of Wuhan isolates are identical, we found point mutations in the Indian isolates. Out of the two isolates one was found to harbour a mutation in its Receptor binding domain (RBD) at position 407. At this site arginine (a positively charged amino acid) was replaced by isoleucine (a hydrophobic amino acid that is also a C-beta branched amino acid). This mutation has been seen to change the secondary structure of the protein at that region and this can potentially alter receptor ding of the virus. Although this finding needs further validation and more sequencing, the information might be useful in rational drug designing and vaccine engineering.","AbstractThe study of pulmonary samples from individuals who have died as a direct result of COVID-19 infection is vital to our understanding of the pathogenesis of this disease. Histopathologic studies of lung tissue from autopsy of patients with COVID-19 specific mortality are only just emerging. All existing reports have relied on traditional 2-dimensional slide-based histological methods for specimen preparation. However, emerging methods for high-resolution, massively multiscale imaging of tissue microstructure using fluorescence labeling and tissue clearing methods enable the acquisition of tissue histology in 3-dimensions, that could open new insights into the nature of SARS-Cov-2 infection and COVID-19 disease processes. In this article, we present the first 3-dimensional images of lung autopsy tissues taken from a COVID-19 patient, including 3D \u201cvirtual histology\u201d of cubic-millimeter volumes of the diseased lung, providing unique insights into disease processes contributing to mortality that could inform frontline treatment decisions.","AbstractIn this study, we reported for the first time the existence of complemented palindrome small RNAs (cpsRNAs) and proposed cpsRNAs and palindrome small RNAs (psRNAs) as a novel class of small RNAs. The first discovered cpsRNA UCUUUAACAAGCUUGUUAAAGA from SARS coronavirus named SARS-CoV-cpsR-22 contained 22 nucleotides perfectly matching its reverse complementary sequence. Further sequence analysis supported that SARS-CoV-cpsR-22 originated from bat betacoronavirus. The results of RNAi experiments showed that one 19-nt segment of SARS-CoV-cpsR-22 significantly induced cell apoptosis. These results suggested that SARS-CoV-cpsR-22 could play a role in SARS-CoV infection or pathogenicity. The discovery of psRNAs and cpsRNAs paved the way to find new markers for pathogen detection and reveal the mechanisms in the infection or pathogenicity from a different point of view. The discovery of psRNAs and cpsRNAs also broaden the understanding of palindrome motifs in animal of plant genomes.","ABSTRACTThe likelihood of experiencing social isolation increases later in life, particularly for females. It remains unknown how late-life social isolation impacts cognition and affective behavior in aged mice. We assessed the impact of late-life social isolation in 18-month old female mice. One month of single-housing did not lead to robust depressive-like symptomology, altered social interaction behavior, or sensitivity to context fear acquisition or memory. Rather, isolation increased hyperactivity and exploration, and reduced anxiety-like behavior in the open field and elevated plus maze, findings that have been similarly observed in young female and male mice following early-life isolation. These findings suggest that hyperactivity is a robust behavior following social isolation across the lifespan.","ABSTRACTThe respiratory syndrome caused by a new type of coronavirus has been emerging from China and caused more than 1000 death globally since December 2019. This new virus, called 2019 novel coronavirus (2019-nCoV) uses the same receptor called Angiotensinconverting enzyme 2 (ACE2) to attack humans as the coronavirus that caused the severe acute respiratory syndrome (SARS) seventeen years ago. Both viruses recognize ACE2 through the spike proteins (S-protein) on their surfaces. It was found that the S-protein from the SARS coronavirus (SARS-CoV) bind stronger to ACE2 than 2019-nCoV. However, function of a bio-system is often under kinetic, rather than thermodynamic, control. To address this issue, we constructed a structural model for complex formed between ACE2 and the S-protein from 2019-nCoV, so that the rate of their association can be estimated and compared with the binding of S-protein from SARS-CoV by a multiscale simulation method. Our simulation results suggest that the association of new virus to the receptor is slower than SARS, which is consistent with the experimental data obtained very recently. We further integrated this difference of association rate between virus and receptor into a mathematical model which describes the life cycle of virus in host cells and its interplay with the innate immune system. Interestingly, we found that the slower association between virus and receptor can result in longer incubation period, while still maintaining a relatively higher level of viral concentration in human body. Our computational study therefore provides, from the molecular level, one possible explanation that the new disease by far spread much faster than SARS.","AbstractWith two-thirds of the global population projected to be living in urban areas by 2050, understanding the transmission patterns of viral pathogens within cities is crucial for effective prevention strategies. Here, in unprecedented spatial resolution, we analysed the socioeconomic determinants of influenza transmission in a European city. We combined geographical and epidemiological data with whole genome sequencing of influenza viruses at the scale of urban quarters and statistical blocks, the smallest geographic subdivisions within a city. We observed annually re-occurring geographic clusters of influenza incidences, mainly associated with net income, and independent of population density and living space. Vaccination against influenza was also mainly associated with household income and was linked to the likelihood of influenza-like illness within an urban quarter. Transmissions patterns within and between quarters were complex. High-resolution city-level epidemiological studies combined with social science surveys such as this will be essential for understanding seasonal and pandemic transmission chains and delivering tailored public health information and vaccination programs at the municipal level.","AbstractThe 2019 novel coronavirus (2019-nCoV) is currently causing a widespread outbreak centered on Hubei province, China and is a major public health concern. Taxonomically 2019-nCoV is closely related to SARS-CoV and SARS-related bat coronaviruses, and it appears to share a common receptor with SARS-CoV (ACE-2). Here, we perform structural modeling of the 2019-nCoV spike glycoprotein. Our data provide support for the similar receptor utilization between 2019-nCoV and SARS-CoV, despite a relatively low amino acid similarity in the receptor binding module. Compared to SARS-CoV, we identify an extended structural loop containing basic amino acids at the interface of the receptor binding (S1) and fusion (S2) domains, which we predict to be proteolytically-sensitive. We suggest this loop confers fusion activation and entry properties more in line with MERS-CoV and other coronaviruses, and that the presence of this structural loop in 2019-nCoV may affect virus stability and transmission.","AbstractMembers of the Nidovirales order have (+)RNA genomes amongst the largest in size in the RNA virus world. Expression of their genes is promoted through reading of genomic RNA and mRNA transcripts by the ribosome of the infected cell. The 5\u2019-end of these RNAs is supposedly protected by an RNA-cap structure (m7GpppNm) whose most synthesis steps remain elusive. In Eukaryotes, the RNA-cap structure is methylated by RNA methyltransferases (MTases) at the RNA-cap N7-guanine position as well as the 2\u2019-O methyl position of the first transcribed nucleotide. In Coronaviridae, two separate enzymes (nsp14 and nsp16) perform the N7-guanine and the 2\u2019-OH methylation, respectively. One salient feature of the Nidovirales N7-guanine MTase nsp14 is that it is the only example of non-Rossman fold viral MTase known so far. Conversely, all other Nidovirales nsp16-like MTases have a canonical Rossman fold. Many Nidovirales members lack either any RNA MTase signature sequence (e.g., Arteriviridae), or lack a N7-guanine MTase signature sequence (e.g., Tobaniviridae, Euroniviridae, Roniviridae, Medioniviridae). Both nsp14-and nsp16-like enzyme genes are usually located in Orf1b encoding for the replication machinery. Here, we report the discovery of a putative Rossman fold RNA MTase in the Orf1a of ten Tobaniviridae members. Multiple sequence alignments and structural analyses identify this novel gene as a typical RNA-cap N7-guanine MTase with substrate specificity and active-site organization similar to the canonical eukaryotic RNA-cap N7-guanine MTase.","AbstractA discrete dynamic model for human epidemics was developed in present study. The model included major parameters as transmission strength and its decline parameters, mean incubation period, hospitalization time, non-hospitalization daily mortality, non-hospitalization daily recovery rate, and hospitalization proportion, etc. Sensitivity analysis of the model indicated the total cumulative cases significantly increased with initial transmission strength, hospitalization time. The total cumulative cases significantly decreased with transmission strength\u2019s decline and hospitalization proportion, and linearly decreased with non-hospitalization daily mortality and non-hospitalization daily recovery rate. In a certain range, the total cumulative cases significantly increased with mean incubation period. Sensitivity analysis demonstrated that dynamic change of transmission strength is one of the most important and controllable factors. In addition, reducing the delay for hospitalization is much effective in weakening disease epidemic. Non-hospitalization recovery rate is of importance for enhancing immunity to recover from the disease.","The disease known as coronavirus disease 19 (COVID-19), potentially caused by an outbreak of the severe acute respiratory syndrome-coronavirus 2 (SARS-CoV-2) in Wuhan, China, has hit the world hard, and has led to an unprecedent health and economic crisis. In order to develop treatment options able to stop or ameliorate SARS-CoV-2 effects, we need to understand the biology of the virus inside cells, but this kind of studies are still scarce. A recent study investigated translatome and proteome host cell changes induced in vitro by SARS-CoV-2. In the present study, we use the publicly available proteomics data from this study to re-analyze the mechanisms altered by the virus infection by impact pathways analysis and network analysis. Proteins linked to inflammatory response, but also proteins related to chromosome segregation during mitosis, were found to be regulated. The up-regulation of the inflammatory-related proteins observed could be linked to the propagation of inflammatory reaction and lung injury that is observed in advanced stages of COVID-19 patients.","SUMMARYThe rapid and escalating spread of SARS coronavirus 2 (SARS-CoV-2) poses an immediate public health emergency, and no approved therapeutics or vaccines are currently available. The viral spike protein S binds ACE2 on host cells to initiate molecular events that release the viral genome intracellularly. Soluble ACE2 inhibits entry of both SARS and SARS-2 coronaviruses by acting as a decoy for S binding sites, and is a candidate for therapeutic and prophylactic development. Using deep mutagenesis, variants of ACE2 are identified with increased binding to the receptor binding domain of S at a cell surface. Mutations are found across the interface and also at buried sites where they are predicted to enhance folding and presentation of the interaction epitope. The N90-glycan on ACE2 hinders association. The mutational landscape offers a blueprint for engineering high affinity ACE2 receptors to meet this unprecedented challenge.","The global outbreak of coronavirus disease 2019 (COVID-19) has placed an unprecedented burden on healthcare systems as the virus spread from the initial 27 reported cases in the city of Wuhan, China to a global pandemic in under three months1. Resources essential to monitoring virus transmission have been challenged with a demand for expanded surveillance. The CDC 2019-nCoV Real-Time Diagnostic Panel uses a real-time reverse transcription quantitative polymerase chain reaction (RT-qPCR) consisting of two TaqMan probe and primer sets specific for the 2019-nCoV N gene, which codes for the nucleocapsid structural protein that encapsulates viral RNA, for the qualitative detection of 2019-nCoV viral RNA in respiratory samples. To isolate RNA from respiratory samples, the CDC lists RNA extraction kits from three manufacturers. In anticipation of a limited supply chain of RNA extraction kits and the need for test scalability, we sought to identify alternative RNA extraction methods. Here we show that direct lysis of respiratory samples can be used in place of RNA extraction kits to run the CDC 2019-nCoV Real-Time Diagnostic assay with the additional benefits of higher throughput, lower cost, faster turnaround and possibly higher senitivity and improved saftey.","AbstractEbolaviruses are non-segmented, negative-sense RNA viruses (NNSVs) within the order Mononegavirales that possess the multifunctional virion protein 35 (VP35), a major determinant of virulence and pathogenesis that is indispensable for viral replication and host innate immune evasion. VP35 is functionally equivalent to the phosphoprotein (P) of other mononegaviruses such as rhabdoviruses and paramyxoviruses. Phosphorylation of the P protein is universally regarded as functionally important however, a regulatory role(s) of phosphorylation on VP35 function remains unexplored. Here, we identified a highly conserved Ser129 residue near the homo-oligomerization coiled coil motif, which is essential for VP35 functions. Affinity-purification MS followed by post-translational modification (PTM) analysis predicted phosphorylation of Ser129. Co-immunoprecipitation, cross-linking, and biochemical characterization studies revealed a moderately decreased capacity of VP35-S129A to oligomerize. Functional analysis showed that Ser-to-Ala substitution of Ebola virus (EBOV) VP35 did not affect IFN inhibitory activity but nearly abolished EBOV minigenome activity. Further coimmunoprecipitation studies demonstrated a lost interaction between VP35-S129A and the amino terminus of the viral polymerase but not between viral nucleoprotein (NP) or VP35-WT. Taken together, our findings provide evidence that phosphorylation modulates VP35 function, supporting VP35 as a NNSV P protein and providing a potentially valuable therapeutic target.ImportanceEbola virus (EBOV) can cause severe disease in humans. The 2013-2016 West African epidemic and the two recent outbreaks in the Democratic Republic of the Congo underscore the urgent need for effective countermeasures, which remain lacking. A better understanding of EBOV biology and the modulation of multifunctional viral proteins is desperately needed to develop improved therapeutics. We provide evidence here that function of virion protein 35 (VP35) is modulated by phosphorylation of Ser129, a conserved residue among other ebolavirus species. These findings shed light on EBOV biology and present a potential target for broad acting anti-ebolavirus therapeutics.","AbstractDuring its first month, the recently emerged 2019 Wuhan novel coronavirus (2019-nCoV) has already infected many thousands of people in mainland China and worldwide and took hundreds of lives. However, the swiftly spreading virus also caused an unprecedentedly rapid response from the research community facing the unknown health challenge of potentially enormous proportions. Unfortunately, the experimental research to understand the molecular mechanisms behind the viral infection and to design a vaccine or antivirals is costly and takes months to develop. To expedite the advancement of our knowledge we leverage the data about the related coronaviruses that is readily available in public databases, and integrate these data into a single computational pipeline. As a result, we provide a comprehensive structural genomics and interactomics road-maps of 2019-nCoV and use these information to infer the possible functional differences and similarities with the related SARS coronavirus. All data are made publicly available to the research community at http://korkinlab.org/wuhan","AbstractThe present outbreak of lower respiratory tract infections, including respiratory distress syndrome, is the third spillover, in only two decades, of an animal coronavirus to humans resulting in a major epidemic. Here, the Coronavirus Study Group (CSG) of the International Committee on Taxonomy of Viruses, which is responsible for developing the official classification of viruses and taxa naming (taxonomy) of the Coronaviridae family, assessed the novelty of the human pathogen tentatively named 2019-nCoV. Based on phylogeny, taxonomy and established practice, the CSG formally recognizes this virus as a sister to severe acute respiratory syndrome coronaviruses (SARS-CoVs) of the species Severe acute respiratory syndrome-related coronavirus and designates it as severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). To facilitate communication, the CSG further proposes to use the following naming convention for individual isolates: SARS-CoV-2/Isolate/Host/Date/Location. The spectrum of clinical manifestations associated with SARS-CoV-2 infections in humans remains to be determined. The independent zoonotic transmission of SARS-CoV and SARS-CoV-2 highlights the need for studying the entire (virus) species to complement research focused on individual pathogenic viruses of immediate significance. This research will improve our understanding of virus-host interactions in an ever-changing environment and enhance our preparedness for future outbreaks.","Viruses and bacteria are the cause of a large number of different human diseases. It is believed that some of them may even contribute to the development of cancer. The present work is dedicated to the identification of mycoplasmas in patients with breast cancer. Mycoplasmas may participate in the development of several human diseases including chronic fatigue syndrome, acquired immunodeficiency syndrome, atypical pneumonia, etc. Moreover, there is a reason to believe that mycoplasma can participate in the development of cancer, leukemia and lymphoma. DNA samples from blood, saliva and tumor tissues of the Oncology Institute of Moldova patients diagnosed with breast cancer were analyzed. Mycoplasma testing was performed using nested PCR method. For Mycoplasma spp. detection, we used primers from the region of the 16S-23S RNA genes. The identification of Mycoplasma faucium, Mycoplasma salivarium and Mycoplasma orale was performed by nested PCR with primers for RNA polymerase beta subunit gene corresponding to mycoplasma. M.faucium and M.salivarius was found in saliva at about 100%, and M.orale at a frequency of about 50%. Only M.faucium was found with the frequency of about 60% in the tissue of the patients. Moreover, a fairly high rate of detection of mycoplasma is observed both in the cases when primers for RNA polymerase gene and primers for 16S-23S RNA were used. We found M.faucium in tumor tissues of patients diagnosed with breast cancer. It is known that mycoplasmas are able to stimulate the synthesis of certain cytokines, which act as mitogenes on the cell. We assume that M.faucium can stimulate the mitogenes synthesis in breast tissues (e.g., cytokines) which, in turn, stimulate cell division and thus participate in the initiation of breast cancer.","ABSTRACTCoronavirus disease 2019 (COVID-19) has caused more than 40,000 deaths worldwide1. Approximately 14% of patients with COVID-19 experienced severe disease and 5% were critically ill2. Studies have shown that dysregulation of the COVID-19 patients\u2019 immune system may lead to inflammatory storm and cause severe illness and even death3,4. Tocilizumab treatment targeting interleukin 6 receptor has shown inspiring clinical results of severe COVID-19 patients5. However, the immune network with Tocilizumab treatment at single cell resolution has not been uncovered. Here, we profiled the single-cell transcriptomes of 13,289 peripheral blood mononuclear cells isolated at three longitudinal stages from two severe COVID-19 patients treated with Tocilizumab. We identified a severe stage-specific monocyte subpopulation and these cells centric immune cell interaction network connected by the inflammatory cytokines and their receptors. The over-activated inflammatory immune response was attenuated after Tocilizumab treatment, yet immune cells including plasma B cells and CD8+ T cells still exhibited an intense humoral and cell-mediated anti-virus immune response in recovered COVID-19 patients. These results provided critical insights into the immunopathogenesis of severe COVID-19 and revealed fundamentals of effectiveness in Tocilizumab treatment.","AbstractMotivationInfections by RNA viruses such as Influenza, HIV still pose a serious threat to human health despite extensive research on viral diseases. One challenge for producing effective prevention and treatment strategies is high intra-species genetic diversity. As different strains may have different biological properties, characterizing the genetic diversity is thus important to vaccine and drug design. Next-generation sequencing technology enables comprehensive characterization of both known and novel strains and has been widely adopted for sequencing viral populations. However, genome-scale reconstruction of haplotypes is still a challenging problem. In particular, haplotype assembly programs often produce contigs rather than full genomes. As a mutation in one gene can mask the phenotypic effects of a mutation at another locus, clustering these contigs into genome-scale haplotypes is still needed.ResultsWe developed a contig binning tool, VirBin, which clusters contigs into different groups so that each group represents a haplotype. Commonly used features based on sequence composition and contig coverage cannot effectively distinguish viral haplotypes because of their high sequence similarity and heterogeneous sequencing coverage for RNA viruses. VirBin applied prototype-based clustering to cluster regions that are more likely to contain mutations specific to a haplotype. The tool was tested on multiple simulated sequencing data with different haplotype abundance distributions and contig sizes, and also on mock quasispecies sequencing data. The benchmark results with other contig binning tools demonstrated the superior sensitivity and precision of VirBin in contig binning for viral haplotype reconstruction.Availabilityhttps://github.com/chjiao/VirBinContactyannisun@cityu.edu.hk","Zika virus is spread mainly by the bite of an infected mosquito, which can be passed from a pregnant woman to her fetus, thus leading to birth defects including more than microcephaly. It has been recently estimated that one-third of the world population will be infected by Zika in the near future, but unfortunately so far there is no vaccine or medicine for Zika. In particular, the special concern on the vaccine treatment to Zika and Dengue arising from antibody-dependent enhancement strongly emphasizes the key role of its NS2B-NS3 protease (NS2B-NS3pro) as a target for anti-Zika drug discovery/design due to its absolutely-essential role in viral replication. In response to the current global health emergency triggered by the Zika outbreak, we successfully obtained several active forms of Zika NS2B-NS3pro and further attempted to discover its inhibitors from eatable plants and traditional herbal medicines to immediately fight Zika. Here, for the first time, we discovered that quercetin, a flavonoid extensively existing in many fruits and vegetables, effectively inhibits Zika NS2B-NS3pro. We further quantify its inhibitory activity with IC50 of 26 uM; and Ki of 23 uM. As quercetin has been extensively found in fruits, vegetables, leaves and grains, our discovery would benefit the public to immediately fight Zika.","AbstractAs reported by the World Health Organization, a novel coronavirus (2019-nCoV) was identified as the causative virus of Wuhan pneumonia of unknown etiology by Chinese authorities on 7 January, 2020. In this study, we developed a Bats-Hosts-Reservoir-People transmission network model for simulating the potential transmission from the infection source (probable be bats) to the human infection. Since the Bats-Hosts-Reservoir network was hard to explore clearly and public concerns were focusing on the transmission from a seafood market (reservoir) to people, we simplified the model as Reservoir-People transmission network model. The basic reproduction number (R0) was calculated from the RP model to assess the transmissibility of the 2019-nCoV.","AbstractThree groups recently identified essential genes in human cancer cell lines using wet experiments, and these genes are of high values. Herein, we improved the widely used Z curve method by creating a \u03bb-interval Z curve, which considered interval association information. With this method and recursive feature elimination technology, a computational model was developed to predict human gene essentiality. The 5-fold cross-validation test based on our benchmark dataset obtained an area under the receiver operating characteristic curve (AUC) of 0.8814. For the rigorous jackknife test, the AUC score was 0.8854. These results demonstrated that the essentiality of human genes could be reliably reflected by only sequence information. However, previous classifiers in three eukaryotes can gave satisfactory prediction only combining sequence with other features. It is also demonstrated that although the information contributed by interval association is less than adjacent nucleotides, this information can still play an independent role. Integrating the interval information into adjacent ones can significantly improve our classifier\u2019s prediction capacity. We re-predicted the benchmark negative dataset by Pheg server (https://cefg.uestc.edu.cn/Pheg), and 118 genes were additionally predicted as essential. Among them, 21 were found to be homologues in mouse essential genes, indicating that at least a part of the 118 genes were indeed essential, however previous experiments overlooked them. As the first available server, Pheg could predict essentiality for anonymous gene sequences of human. It is also hoped the \u03bb-interval Z curve method could be effectively extended to classification issues of other DNA elements.","AbstractPublic health concerns about recent viral epidemics have motivated researchers to seek transdisciplinary understanding of infection in wildlife hosts. With its deep history devoted to explaining the abundance and distribution of organisms, ecology can augment current methods for studying viral dynamics. However, datasets allowing ecological explorations of viral communities are lacking, and common methods for delineating viral operational taxonomic units (OTUs), or \u201cspecies\u201d, are subjective. Here, we comprehensively sampled 1,086 bats from two Puerto Rican caves and tested them for infection with herpesviruses. Using percent identity of nucleotides and a machine learning algorithm, we categorized herpesviruses into 41 OTUs, representing approximately 80% of all herpesviruses in the host community. Although 13 OTUs were detected in multiple host species, OTUs generally exhibited host specificity by infecting a core host species at a significantly higher prevalence than in all other species combined. Only two OTUs showed significantly different prevalence between host sexes. This work is the first exploration of viral community ecology in a community of wildlife hosts.","Deep sequencing of clinical samples is now an established tool for the detection of infectious pathogens, with direct medical applications. The large amount of data generated provides an opportunity to detect species even at very low levels, provided that computational tools can effectively interpret potentially complex metagenomic mixtures. Data interpretation is complicated by the fact that short sequencing reads can match multiple organisms and by the lack of completeness of existing databases, in particular for viral pathogens. This interpretation problem can be formulated statistically as a mixture model, where the species of origin of each read is missing, but the complete knowledge of all species present in the mixture helps with the individual reads assignment. Several analytical tools have been proposed to approximately solve this computational problem. Here, we show that the use of parallel Monte Carlo Markov chains (MCMC) for the exploration of the species space enables the identification of the set of species most likely to contribute to the mixture. The added accuracy comes at a cost of increased computation time. Our approach is useful for solving complex mixtures involving several related species. We designed our method specifically for the analysis of deep transcriptome sequencing datasets and with a particular focus on viral pathogen detection, but the principles are applicable more generally to all types of metagenomics mixtures. The work is implemented as a user friendly R package, available from CRAN: http://cran.r-project.org/web/packages/metaMix","ABSTRACTHand hygiene is considered as an efficient and cost-effective way to limit the spread of diseases and, as such, it is recommended by both the World Health Organization (WHO) and the Centres for Disease Control and Prevention (CDC). While the effect of hand washing on individual transmissibility of a disease has been studied through medical and public-health research, its potential as a mitigation strategy against a global pandemic has not been fully explored yet. In this study, we investigate contagion dynamics through the world air transportation network and analyze the impact of hand-hygiene behavioural changes of airport population against the spread of infectious diseases worldwide. Using a granular dataset of the world air transportation traffic, we build a detailed individual mobility model that controls for the correlated and recurrent nature of human travel and the waiting-time distributions of individuals at different locations. We perform a Monte-Carlo simulation study to assess the impact of different hand-washing mitigation strategies at the early stages of a global epidemic. From the simulation results we find that increasing the hand cleanliness homogeneously at all airports in the world can inhibit the impact of a potential pandemic by 24 to 69%. By quantifying and ranking the contribution of the different airports to the mitigation of an epidemic outbreak, we identify ten key airports at the core of a cost-optimal deployment of the hand-washing strategy: increasing the engagement rate at those locations alone could potentially reduce a world pandemic by 8 to 37%. This research provides evidence of the effectiveness of hand hygiene in airports on the global spread of infectious diseases, and has important implications for the way public-health policymakers may design new effective strategies to enhance hand hygiene in airports through behavioral changes.","AbstractBackgroundThere is a critical need for effective therapies that are immediately available to control the spread of COVID-19 disease. In this study, we assessed currently marketed intravenous immunoglobulin (IVIG) products for antibodies against human common coronaviruses that may cross-react with the SARS-CoV-2 virus.MethodsGamunex\u00ae-C and Flebogamma\u00ae DIF (Grifols) IVIG were tested against several betacoronaviruses antigens using ELISA techniques: HCoV (undetermined antigen), HCoV-HKU1 (N protein), SARS-CoV (culture lysate), MERS-CoV (N protein; S1 protein/RBD; S protein), and SARS-CoV-2 (S1 protein).ResultsBoth IVIG products showed consistent reactivity to components of the tested viruses. Positive cross-reactivity was seen in SARS-CoV, MERS-CoV, and SARS-CoV-2. For SARS-CoV-2, positive reactivity was observed at IVIG concentrations ranging from 100 \u03bcg/mL with Gamunex-C to 1 mg/mL with Flebogamma 5% DIF.ConclusionGamunex-C and Flebogamma DIF IVIG contain antibodies reacting against SARS-CoV-2 antigens. These preparations may be useful for immediate treatment of COVID-19 disease.","ABSTRACTSialic acids (Sia) are the primary receptors for influenza viruses, and are widely displayed on cell surfaces and in secreted mucus. Sia may be present in variant forms that include O-acetyl modifications at C4, C7, C8, and C9 positions, and N-acetyl or N-glycolyl at C5. They can also vary in their linkages, including \u03b12-3 or \u03b12-6-linkages. Here, we analyzed the distribution of modified Sia in cells and tissues of wild-type mice, or in mice lacking cytidine 5\u2019-monophosphate-N-acetylneuraminic acid hydroxylase (CMAH) enzyme that synthesizes N-glycolyl modifications (Neu5Gc). We also examined the variation of Sia forms on erythrocytes and saliva from different animals. To determine the effect of Sia modifications on influenza A virus (IAV) infection, we tested for effects on hemagglutinin (HA) binding and neuraminidase (NA) cleavage. We confirmed that 9-O-acetyl, 7,9-O-acetyl, 4-O-acetyl, and Neu5Gc modifications are widely but variably expressed in mouse tissues, with the highest levels detected in the respiratory and gastrointestinal tracts. Secreted mucins in saliva and surface proteins of erythrocytes showed a great degree of variability in display of modified Sia between different species. IAV HA from different virus strains showed consistently reduced binding to both Neu5Gc and O-acetyl modified Sia; however, while IAV NA were inhibited by Neu5Gc and O-acetyl modifications, there was significant variability between NA types. The modifications of Sia in mucus may therefore have potent effects on the functions of IAV, and may affect both pathogens and the normal flora of different mucosal sites.IMPORTANCESialic acids (Sia) are involved in many different cellular functions and are receptors for many pathogens. Sia come in many chemically modified forms but we lack a clear understanding of how they alter the interactions with microbes. Here we examine the expression of modified Sia in mouse tissues, on secreted mucus in saliva, and on erythrocytes, including those from IAV host species and animals used in IAV research. These Sia forms varied considerably between different animals, and their inhibitory effects on IAV NA and HA activities and on bacterial sialidases (neuraminidases) suggest a host-variable protective role in secreted mucus.","AbstractThe SARS-CoV-2 epidemic has rapidly spread outside China with major outbreaks occurring in Italy, South Korea and Iran. Phylogenetic analyses of whole genome sequencing data identified a distinct SARS-CoV-2 clade linked to travellers returning from Iran to Australia and New Zealand. This study highlights potential viral diversity driving the epidemic in Iran, and underscores the power of rapid genome sequencing and public data sharing to improve the detection and management of emerging infectious diseases.","AbstractCoronaviruses (CoVs) traffic frequently between species resulting in novel disease outbreaks, most recently exemplified by the newly emerged SARS-CoV-2. Herein, we show that the ribonucleoside analog \u03b2-D-N4-hydroxycytidine (NHC, EIDD-1931) has broad spectrum antiviral activity against SARS-CoV 2, MERS-CoV, SARS-CoV, and related zoonotic group 2b or 2c Bat-CoVs, as well as increased potency against a coronavirus bearing resistance mutations to another nucleoside analog inhibitor. In mice infected with SARS-CoV or MERS-CoV, both prophylactic and therapeutic administration of EIDD-2801, an orally bioavailable NHC-prodrug (b-D-N4-hydroxycytidine-5\u2019-isopropyl ester), improved pulmonary function, and reduced virus titer and body weight loss. Decreased MERS-CoV yields in vitro and in vivo were associated with increased transition mutation frequency in viral but not host cell RNA, supporting a mechanism of lethal mutagenesis. The potency of NHC/EIDD-2801 against multiple coronaviruses, its therapeutic efficacy, and oral bioavailability in vivo, all highlight its potential utility as an effective antiviral against SARS-CoV-2 and other future zoonotic coronaviruses.","AbstractRecent advances in identification and characterization of human-specific regulatory DNA sequences set the stage for the assessment of their global impact on physiology and pathology of Modern Humans. Gene set enrichment analyses (GSEA) of 8,405 genes linked with 35,074 human-specific neuro-regulatory single-nucleotide changes (hsSNCs) revealed a staggering breadth of significant associations with morphological structures, physiological processes, and pathological conditions of Modern Humans. Significantly enriched traits include more than 1,000 anatomically-distinct regions of the adult human brain, many different types of cells and tissues, more than 200 common human disorders and more than 1,000 records of rare diseases. Thousands of genes connected with neuro-regulatory hsSNCs have been identified, which represent essential genetic elements of the autosomal inheritance and offspring survival phenotypes. A total of 1,494 hsSNC- linked genes are associated with either autosomal dominant or recessive inheritance and 2,273 hsSNC-linked genes have been associated with premature death, embryonic lethality, as well as pre-, peri-, neo-, and post-natal lethality phenotypes of both complete and incomplete penetrance. Differential GSEA implemented on hsSNC-linked loci and associated genes identify 7,990 genes linked to evolutionary distinct classes of human-specific regulatory sequences (HSRS), expression of a majority of which (5,389 genes; 67%) is regulated by stem cell-associated retroviral sequences (SCARS). Interrogations of the MGI database revealed readily available mouse models tailored for precise experimental definitions of functional effects of hsSNCs and SCARS on genes causally affecting thousands of mammalian phenotypes and implicated in hundreds of common and rare human disorders. These observations suggest that a preponderance of human-specific traits evolved under a combinatorial regulatory control of HSRS and neuro-regulatory loci harboring hsSNCs that are fixed in humans, distinct from other primates, and located in differentially-accessible chromatin regions during brain development.","ABSTRACTBackgroundProtease-activated receptor 2 (PAR-2) is activated by proteases involved in allergy and triggers airway epithelial secretion and inflammation. PAR-2 is normally expressed basolaterally in differentiated nasal ciliated cells.ObjectiveWe tested if epithelial remodeling during diseases characterized by loss of cilia and squamous metaplasia may alter PAR-2 polarization.MethodsEndogenous PAR-2 responses were measured by live cell calcium and cilia imaging, measurement of fluid secretion, and quantification of cytokines. We utilized airway squamous cell lines, primary differentiated air-liquid interface cultures, and tissue explants. Cells were exposed to disease-related modifiers that alter epithelial morphology, including IL-13, cigarette smoke condensate, and retinoic acid deficiency. We used concentrations and exposure times that altered epithelial morphology without causing breakdown of the epithelial barrier, likely reflecting early disease states.ResultsPAR-2 signaling in airway squamous cells activated calcium and inflammatory responses. Squamous cells cultured at air liquid interface (ALI) responded to PAR-2 agonists applied both apically and basolaterally. Primary well-differentiated nasal epithelial ALI cultures responded only to basolateral PAR-2 stimulation. Primary cultures exposed to IL-13, cigarette smoke condensate, or reduced retinoic acid responded to both apical and basolateral PAR-2 stimulation. Nasal polyp tissue, but not control middle turbinate, exhibited apical calcium responses to PAR-2 stimulation. However, isolated ciliated cells from both polyp and turbinate maintained basolateral PAR-2 polarization.ConclusionsSquamous metaplasia and/or loss of cilia enhances apical PAR-2 responses. Altered PAR-2 polarization in dedifferentiated or remodeled epithelia may contribute to increased sensitivity to inhaled protease allergens in inflammatory airway diseases.","Sequence analyses of RNA virus genomes remain challenging due to the exceptional genetic plasticity of these viruses. Because of high mutation and recombination rates, genome replication by viral RNA-dependent RNA polymerases leads to populations of closely related viruses, so-called 'quasispecies'. Standard (short-read) sequencing technologies are ill-suited to reconstruct large numbers of full-length haplotypes of (i) RNA virus genomes and (ii) subgenome-length (sg) RNAs comprised of noncontiguous genome regions. Here, we used a full-length, direct RNA sequencing (DRS) approach based on nanopores to characterize viral RNAs produced in cells infected with a human coronavirus. Using DRS, we were able to map the longest (~26 kb) contiguous read to the viral reference genome. By combining Illumina and nanopore sequencing, we reconstructed a highly accurate consensus sequence of the human coronavirus (HCoV) 229E genome (27.3 kb). Furthermore, using long reads that did not require an assembly step, we were able to identify, in infected cells, diverse and novel HCoV-229E sg RNAs that remain to be characterized. Also, the DRS approach, which circumvents reverse transcription and amplification of RNA, allowed us to detect methylation sites in viral RNAs. Our work paves the way for haplotype-based analyses of viral quasispecies by demonstrating the feasibility of intra-sample haplotype separation. Even though several technical challenges remain to be addressed to exploit the potential of the nanopore technology fully, our work illustrates that direct RNA sequencing may significantly advance genomic studies of complex virus populations, including predictions on long-range interactions in individual full-length viral RNA haplotypes.","AbstractThe emergence of the novel coronavirus SARS-CoV-2, which in humans is highly infectious and leads to the potentially fatal disease COVID-19, has caused tens of thousands of deaths and huge global disruption. The viral infection may also represent an existential threat to our closest living relatives, the nonhuman primates, many of which have already been reduced to small and endangered populations. The virus engages the host cell receptor, angiotensin\u2010converting enzyme\u20102 (ACE2), through the receptor binding domain (RBD) on the spike protein. The contact surface of ACE2 displays amino acid residues that are critical for virus recognition, and variations at these critical residues are likely to modulate infection susceptibility across species. While infection studies have shown that rhesus macaques exposed to the virus develop COVID-19-like symptoms, the susceptibility of other nonhuman primates is unknown. Here, we show that all apes, including chimpanzees, bonobos, gorillas, and orangutans, and all African and Asian monkeys, exhibit the same set of twelve key amino acid residues as human ACE2. Monkeys in the Americas, and some tarsiers, lemurs and lorisoids, differ at significant contact residues, and protein modeling predicts that these differences should greatly reduce the binding affinity of the ACE2 for the virus, hence moderating their susceptibility for infection. Our study suggests that apes and African and Asian monkeys are all likely to be highly susceptible to SARS-CoV-2, representing a critical threat to their survival. Urgent actions may be necessary to limit their exposure to humans.","ABSTRACTBinge eating (BE) is a heritable symptom of eating disorders with an unknown genetic etiology. Rodent models for BE of palatable food permit the study of genetic and biological mechanisms. We previously used genetic mapping and transcriptome analysis to map a coding mutation in Cyfip2 associated with increased BE in the BE-prone C57BL/6NJ substrain compared to the BE-resistant C57BL/6J substrain. The increase in BE in C57BL/6NJ mice was associated with a decrease in transcription of genes enriched for myelination in the striatum. Here, we tested the hypothesis that decreasing myelin levels with the demyelinating agent cuprizone would enhance BE. Mice were treated with a 0.3% cuprizone home cage diet for two weeks. Following a three-week recovery period, mice were trained for BE in an intermittent, limited access procedure. Cuprizone induced similar weight loss in both substrains and sexes that recovered within 48 h after removal of the cuprizone diet. Surprisingly, cuprizone reduced BE in male but not female C57BL/6NJ mice while having no effect in C57BL/6J mice. Cuprizone also reduced myelin basic protein (MBP) at seven weeks post-cuprizone removal while having no effect on myelin-associated glycoprotein (MAG) at this time point. C57BL/6N mice also showed less MBP than C57BL/6J mice. There were no statistical interactions of Treatment with Sex on MBP levels, indicating that differences in MBP are unlikely to account for sex differences in BE. To summarize, cuprizone induced an unexpected, male-specific reduction in BE which could indicate sex-specific biological mechanisms that depend on genetic background.","SummaryBackgroundThe 2019 new coronavirus, \u201c2019-nCoV\u201d, was discovered from Wuhan Viral Pneumonia cases in December 2019, and was named by the World Health Organization on January 12, 2020. In the early stage, people knows little about the 2019-nCoV virus was not clear, and the spread period was encountering China\u2019s annual spring migration, which made the epidemic spread rapidly from Wuhan to almost all provinces in China.MethodsThis study builds a SEIRD model that considers the movement of people across regions, revealing the effects of three measures on controlling the spread of the epidemic.Based on MATLAB R2017a, computational experiments were performed to simulate the epidemic prevention and control measures.FindingsThe research results show that current prevention and control measures in China are very necessary. This study further validates the concerns of international and domestic experts regarding asymptomatic transmission (E-status).InterpretationThe results of this study are applicable to explore the impact of the implementation of relevant measures on the prevention and control of epidemic spread, and to identify key individuals that may exist during the spread of the epidemic.","AbstractAccumulating evidence has shown that long noncoding RNAs (lncRNAs) are involved in several biological processes, including immune responses. However, the role of lncRNAs in antiviral innate immune responses remains largely unexplored. Here, we identify an uncharacterized human lncRNA from influenza A virus (IAV) patients, antivirus and activate neutrophil (AVAN), that is significantly up-regulated upon virus infection. Mechanistically, nuclear lncRNA-AVAN positively regulates the transcription of forkhead box O3A (FOXO3a) by associating with its promoter and inducing chromatin remodeling to promote neutrophil chemotaxis. Furthermore, we also found that cytoplasmic lncRNA-AVAN directly binds tripartite motif containing 25 (TRIM25) and enhances the association of TRIM25 and Retinoic acid inducible gene-1 proteins (RIG-I) and the ubiquitylation of RIG-I, thereby promoting TRIM25- and RIG-I-mediated antiviral innate immune signaling. More importantly, we enforced the expression of AVAN in transgenic mice and found that it significantly alleviated IAV virulence and virus production. Collectively, these findings highlight the potential clinical implications of lncRNA-AVAN as a key positive regulator of the antiviral innate immune response and a promising target for developing broad antiviral therapeutics.","AbstractThe focused drug repurposing of known approved drugs (such as lopinavir/ritonavir) has been reported failed for curing SARS-CoV-2 infected patients. It is urgent to generate new chemical entities against this virus. As a key enzyme in the life-cycle of coronavirus, the 3C-like main protease (3CLpro or Mpro) is the most attractive for antiviral drug design. Based on a recently solved structure (PDB ID: 6LU7), we developed a novel advanced deep Q-learning network with the fragment-based drug design (ADQN-FBDD) for generating potential lead compounds targeting SARS-CoV-2 3CLpro. We obtained a series of derivatives from those lead compounds by our structure-based optimization policy (SBOP). All the 47 lead compounds directly from our AI-model and related derivatives based on SBOP are accessible in our molecular library at https://github.com/tbwxmu/2019-nCov. These compounds can be used as potential candidates for researchers in their development of drugs against SARS-CoV-2.","SummaryPharmacological interventions that target human ageing would extend individual healthspan and result in dramatic economic benefits to rapidly ageing societies worldwide. For such interventions to be contemplated they need to comprise drugs that are efficacious when given to adults and for which extensive human safety data are available. Here we show that dramatic lifespan extension can be achieved in C.elegans by targeting multiple, evolutionarily conserved ageing pathways using drugs that are already in human use. By targeting multiple synergistic ageing pathways, we are able to slow ageing rate, double lifespan and improves healthspan while minimize developmental and fitness trade-offs. Moreover, we established that there is no synergistic benefit in a daf-2 or daf-7 background, implying the involvement of the TGF\u03b2 and IGF pathways in this synergy. Employing lipidomics and transcriptomics analysis we found lipid metabolism to be affected resulting in increased monounsaturated fatty acids (MUFA) and decrease membrane peroxidation index. Our best drug combination showed a conserved lifespan extension in fruit flies. To the best of our knowledge, this is the largest lifespan effect ever reported for any adult-onset drug treatment in C. elegans. This drug-repurposing approach, using drugs already approved for humans to target multiple conserved aging pathways simultaneously, could lead to interventions that prevent age-related diseases and overall frailty in a rapidly ageing population.","AbstractWe study an epidemic model with nonlinear incidence rate, describing the saturated mass action as well as the psychological effect of certain serious diseases on the community. Firstly, the existence and local stability of disease-free and endemic equilibria are investigated. Then, we prove the occurrence of backward bifurcation, saddle-node bifurcation, Hopf bifurcation and cusp type Bogdanov-Takens bifurcation of codimension 3. Finally, numerical simulations, including one limit cycle, two limit cycles, unstable homoclinic loop and many other phase portraits are presented. These results show that the psychological effect of diseases and the behavior change of the susceptible individuals may affect the final spread level of an epidemic.","Bats are responsible for the zoonotic transmission of several major viral diseases including the 2003 SARS outbreak and the ongoing COVID-19 pandemic. While bat genomic sequencing studies have revealed characteristic adaptations of the innate immune system, functional genomic studies are urgently needed to provide a foundation for the molecular dissection of the tolerance of viral infections in bats. Here we report the establishment and screening of genome-wide RNAi library and CRISPR library for the model megabat, Pteropus Alecto. We used the complementary RNAi and CRISPR libraries to interrogate Pteropus Alecto cells for infection with two different viruses, mumps virus and Influenza A virus, respectively. Screening results converged on the endocytosis pathway and the protein secretory pathway as required for both viral infections. Additionally, we revealed a general dependence of the C-1-tetrahydrofolate synthase gene, MTHFD1, for viral replication in bat cells as well as in human cells. MTHFD1 inhibitor carolacton potently blocked replication of several RNA viruses including SARS-CoV-2. Our studies provide a resource for systematic inquiry into the genetic underpinnings of bat biology and a potential target for developing broad spectrum antiviral therapy.","In this study, we aimed to detect viruses in hard ticks using the small RNA sequencing based method. A 235-bp DNA segment was detected in Dermacentor nuttalli (hard ticks) and D. silvarum (hard ticks) from sheep and bovine, respectively. The detected 235-bp segment had an identity of 99% to a 235-bp DNA segment of African Swine Fever Virus (ASFV) and contained three single nucleotide mutations (C38T, C76T and A108C). C38T, resulting in an single amino acid mutation G66D, suggests the existence of a new ASFV strain, which is different from all reported ASFV strains in NCBI GenBank database. These results also suggest that ASFV could have a wide range of hosts or vectors, beyond the well known Suidae family and soft ticks. Our findings pave the way toward further studies of ASFV transmission and development of prevention and control measures.","ABSTRACTThe main protease of coronaviruses and the 3C protease of enteroviruses share a similar active-site architecture and a unique requirement for glutamine in the P1 position of the substrate. Because of their unique specificity and essential role in viral polyprotein processing, these proteases are suitable targets for the development of antiviral drugs. In order to obtain near-equipotent, broad-spectrum antivirals against alphacoronaviruses, betacoronaviruses, and enteroviruses, we pursued structure-based design of peptidomimetic \u03b1-ketoamides as inhibitors of main and 3C proteases. Six crystal structures of protease:inhibitor complexes were determined as part of this study. Compounds synthesized were tested against the recombinant proteases as well as in viral replicons and virus-infected cell cultures; most of them were not cell-toxic. Optimization of the P2 substituent of the \u03b1-ketoamides proved crucial for achieving near-equipotency against the three virus genera. The best near-equipotent inhibitors, 11u (P2 = cyclopentylmethyl) and 11r (P2 = cyclohexylmethyl), display low-micromolar EC50 values against enteroviruses, alphacoronaviruses, and betacoronaviruses in cell cultures. In Huh7 cells, 11r exhibits three-digit picomolar activity against Middle East Respiratory Syndrome coronavirus.","Some free fatty acids derived from milk and vegetable oils are known to have potent antiviral and antibacterial properties. However, therapeutic applications of short to medium chain fatty acids are limited by physical characteristics such as immiscibility in aqueous solutions. We evaluated a novel proprietary formulation based on an emulsion of short chain caprylic acid, ViroSAL, for its ability to inhibit a range of viral infections in vitro and in vivo. In vitro, ViroSAL inhibited the enveloped viruses Epstein-Barr, measles, herpes simplex, Zika and orf parapoxvirus, together with Ebola, Lassa, vesicular stomatitis and SARS-CoV-1 pseudoviruses, in a concentration- and time-dependent manner. Evaluation of the components of ViroSAL revealed that caprylic acid was the main antiviral component; however, the ViroSAL formulation significantly inhibited viral entry compared with caprylic acid alone. In vivo, VIROSAL significantly inhibited Zika and Semliki Forest Virus replication in mice following the inoculation of these viruses into mosquito bite sites. In agreement with studies investigating other free fatty acids, ViroSAL had no effect on norovirus, a non-enveloped virus, indicating that its mechanism of action may be via surfactant disruption of the viral envelope. We have identified a novel antiviral formulation that may have applications for prevention and/or treatment of a broad range of enveloped viruses.","ABSTRACTBackgroundMetagenomic next generation sequencing (mNGS) has enabled the rapid, unbiased detection and identification of microbes without pathogen-specific reagents, culturing, or a priori knowledge of the microbial landscape. mNGS data analysis requires a series of computationally intensive processing steps to accurately determine the microbial composition of a sample. Existing mNGS data analysis tools typically require bioinformatics expertise and access to local server-class hardware resources. For many research laboratories, this presents an obstacle, especially in resource limited environments.FindingsWe present IDseq, an open source cloud-based metagenomics pipeline and service for global pathogen detection and monitoring (https://idseq.net). The IDseq Portal accepts raw mNGS data, performs host and quality filtration steps, then executes an assembly-based alignment pipeline which results in the assignment of reads and contigs to taxonomic categories. The taxonomic relative abundances are reported and visualized in an easy-to-use web application to facilitate data interpretation and hypothesis generation. Furthermore, IDseq supports environmental background model generation and automatic internal spike-in control recognition, providing statistics which are critical for data interpretation. IDseq was designed with the specific intent of detecting novel pathogens. Here, we benchmark novel virus detection capability using both synthetically evolved viral sequences, and real-world samples, including IDseq analysis of a nasopharyngeal swab sample acquired and processed locally in Cambodia from a tourist from Wuhan, China, infected with the recently emergent SARS-CoV-2.ConclusionThe IDseq Portal reduces the barrier to entry for mNGS data analysis and enables bench scientists, clinicians, and bioinformaticians to gain insight from mNGS datasets for both known and novel pathogens.","In solid tumors, increasing drug penetration promotes their regression and improves the therapeutic index of compounds. However, the heterogeneous extracellular matrix (ECM) acts a steric and interaction barrier that hinders effective transport of therapeutics, including nanomedicines. Specifically, the interactions between the ECM and surface physicochemical properties of nanomedicines (e.g. charge, hydrophobicity) impedes their diffusion and penetration. To address the challenges using existing surface chemistries, we used peptide-presenting phage libraries as a high-throughput approach to screen and identify peptides as coatings with desired physicochemical properties that improve diffusive transport through the tumor microenvironment. Through iterative screening against the ECM and identification by next-generation DNA sequencing and analysis, we selected individual clones and measured their transport by diffusion assays. Here, we identified a net-neutral charge, hydrophilic peptide P4 that facilitates significantly higher diffusive transport of phage than negative control through in vitro tumor ECM. Through alanine mutagenesis, we confirmed that the hydrophilicity, charge, and their spatial ordering impact diffusive transport. P4 phage clone exhibited almost 200-fold improved uptake in ex vivo pancreatic tumor xenografts compared to the negative control. Nanoparticles coated with P4 exhibited ~40-fold improvement in diffusivity in pancreatic tumor tissues, and P4-coated particles demonstrated less hindered diffusivity through the ECM compared to particles functionalized with gold standard poly(ethylene) glycol or iRGD peptide ligand. By leveraging the power of molecular diversity using phage display, we can greatly expand the chemical space of surface chemistries that can improve the transport of nanomedicines through the complex tumor microenvironment to ultimately improve their efficacy.","SARS-CoV-2 emerged by the end of 2019 to rapidly spread in 2020. At present, it is of utmost importance to understand the virus biology and to rapidly assess the potential of existing drugs and develop new active compounds. While some animal models for such studies are under development, most of the research is carried out in the Vero E6 cells. Here, we propose fully differentiated human airway epithelium cultures as a model for studies on the SARS-CoV-2. Further, we also provide basic characteristics of the system.","During the decoding of a subset of mRNAs a proportion of ribosomes productively shift to the -1 reading frame at a specific, slippage-prone site. While the great majority of occurrences of the Programmed -1 Ribosomal Frameshifting (-1 PRF) involve viruses, other mobile elements or retroelements, a dramatic instance of utilized human cellular -1 frameshifting in a non-retroelement derived mRNA has been reported. The mRNA for which -1 frameshifting was claimed due to a stimulatory pseudoknot is that which encodes immune functioning C-C chemokine receptor 5 (CCR5), the HIV-1 co-receptor. The publication ascribed the importance of the reporter CCR5 mRNA frameshifting directing translating ribosomes to a premature termination codon leading to mRNA destabilization via the nonsense-mediated decay pathway (NMD), rather than creating a C-terminally unique peptide. The same lab estimates that up to 10% of eukaryotic genes may be regulated by this frameshift-NMD mechanism and point to the potential for novel therapeutic strategies aimed at treating disease by -1 PRF.","AbstractThe ability to rapidly assemble and prototype cellular circuits is vital for biological research and its applications in biotechnology and medicine. Current methods that permit the assembly of DNA circuits in mammalian cells are laborious, slow, expensive and mostly not permissive of rapid prototyping of constructs. Here we present the Mammalian ToolKit (MTK), a Golden Gate-based cloning toolkit for fast, reproducible and versatile assembly of large DNA vectors and their implementation in mammalian models. The MTK consists of a curated library of characterized, modular parts that can be easily mixed and matched to combinatorially assemble one transcriptional unit with different characteristics, or a hierarchy of transcriptional units weaved into complex circuits. MTK renders many cell engineering operations facile, as showcased by our ability to use the toolkit to generate single-integration landing pads, to create and deliver libraries of protein variants and sgRNAs, and to iterate through Cas9-based prototype circuits. As a biological proof of concept, we used the MTK to successfully design and rapidly construct in mammalian cells a challenging multicistronic circuit encoding the Ebola virus (EBOV) replication complex. This construct provides a non-infectious biosafety level 2 (BSL2) cellular assay for exploring the transcription and replication steps of the EBOV viral life cycle in its host. Its construction also demonstrates how the MTK can enable important and time sensitive applications such as the rapid testing of pharmacological inhibitors of emerging BSL4 viruses that pose a major threat to human health.","Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is the cause of the ongoing pandemic of coronavirus disease 2019 (COVID-19), a public health emergency of international concern declared by the World Health Organization (WHO). An immuno-informatics approach along with comparative genomic was applied to design a multi-epitope-based peptide vaccine against SARS-CoV-2 combining the antigenic epitopes of the S, M and E proteins. The tertiary structure was predicted, refined and validated using advanced bioinformatics tools. The candidate vaccine showed an average of \u2265 90.0% world population coverage for different ethnic groups. Molecular docking of the chimeric vaccine peptide with the immune receptors (TLR3 and TLR4) predicted efficient binding. Immune simulation predicted significant primary immune response with increased IgM and secondary immune response with high levels of both IgG1 and IgG2. It also increased the proliferation of T-helper cells and cytotoxic T-cells along with the increased INF-\u03b3 and IL-2 cytokines. The codon optimization and mRNA secondary structure prediction revealed the chimera is suitable for high-level expression and cloning. Overall, the constructed recombinant chimeric vaccine candidate demonstrated significant potential and can be considered for clinical validation to fight against this global threat, COVID-19.","Developing methods to reconstruct transmission histories for viral outbreaks could provide critical information to support locating sources of disease transmission. Phylogenetic methods used to measure the degree of relatedness among sequenced viral samples have proven useful in identifying potential outbreak sources. The complex nature of infectious disease, however, makes it difficult to assign a rigorously defined quantitative confidence value assessing the likelihood of a true direct transmission event using genetic data alone. A new method is presented to calculate a confidence value assessing the likelihood of a transmission event using both phylogenetic inference and limited knowledge of incubation and infectious duration times. The method is applied to simulations of a foot and mouth disease (FMD) outbreak to demonstrate how the combination of both phylogenetic and epidemiology data can be used to strengthen the assessment of the likelihood of direct transmission over methods using just phylogenetic data or infection timing data alone. The method is applied to a previous FMD outbreak to identify areas where over confidence in previously inferred direct transmission may exist. Combining knowledge from viral evolution and epidemiology within a single integrated transmission inference framework is an important approach to assess the potential likelihood of transmission events and makes clear how specific features of a virus' spread through the course of an outbreak will directly determine the potential for confidence in inferred host transmission links.","AbstractBackgroundThe Tasmanian devil is an endangered carnivorous marsupial threatened by devil facial tumour disease (DFTD). While research on DFTD has been extensive, little is known about the viruses present in devils, and whether any of these are of potential conservation relevance for this endangered species.MethodsUsing both metagenomics based on virus-like particle (VLP) enrichment and sequence-independent amplification (VLP metagenomics), and meta-transcriptomics based on bulk RNA sequencing, we characterised and compared the faecal viromes of captive and wild Tasmanian devils.ResultsA total of 54 devil faecal samples collected from captive (n = 2) and wild (n = 4) populations were processed for virome characterisation using both approaches. We detected many novel, highly divergent viruses, including vertebrate viruses, bacteriophage and other dietary associated plant and insect viruses. In total, 18 new vertebrate viruses, including novel sapelovirus, astroviruses, bocaviruses, papillomaviruses and gammaherpesvirus were identified, as well as known mammalian pathogens including rabbit haemorrhagic disease virus 2 (RHDV2). Captive devils showed significantly lower levels of viral diversity than wild devils. Comparison of the two methodological approaches revealed substantial differences in the number and types of viruses detected, with meta-transcriptomics mainly identifying RNA viruses, and VLP metagenomics largely identifying DNA viruses.ConclusionThis study has greatly expanded our knowledge of eukaryotic viruses in the Tasmanian devil and provides important baseline information that will contribute to the conservation and captive management of this endangered species. In addition, our results showed that a combination of VLP metagenomics and meta-transcriptomics may be a more comprehensive approach to virome characterisation than either method alone.","The entire world has recently witnessed the commencement of coronavirus disease 19 (COVID-19) pandemic. It is caused by a novel coronavirus (n-CoV) generally distinguished as Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2). It has exploited human vulnerabilities to coronavirus outbreak. SARS-CoV-2 promotes fatal chronic respiratory disease followed by multiple organ failure which ultimately puts an end to human life. No proven vaccine for n-CoV is available till date in spite of significant research efforts worldwide. International Committee on Taxonomy of Viruses (ICTV) has reached to a consensus that the virus SARS-CoV-2 is highly genetically similar to Severe Acute Respiratory Syndrome Coronavirus (SARS-CoV) outbreak of 2003. It has been reported that SARS-CoV has ~89% genetic similarities with n-CoV. With this hypothesis, the current work focuses on the identification of spreader nodes in SARS-CoV protein interaction network. Various network characteristics like edge ratio, neighborhood density and node weight have been explored for defining a new feature spreadability index by virtue of which spreader nodes and edges are identified. The selected top spreader nodes having high spreadability index have been also validated by Susceptible-Infected-Susceptible (SIS) disease model. Initially, the proposed method is applied on a synthetic protein interaction network followed by SARS-CoV-human protein interaction network. Hence, key spreader nodes and edges (ranked edges) are unmasked in SARS-CoV proteins and its connected level 1 and level 2 human proteins. The new network attribute spreadability index along with generated SIS values of selected top spreader nodes when compared with the other network centrality based methodologies like Degree centrality (DC), Closeness centrality (CC), Local average centrality (LAC) and Betweeness centrality (BC) is found to perform relatively better than the existing-state-of-art.","AbstractThe 2015-2016 emergence of Zika virus (ZIKV) in the Americas, and recognition that ZIKV infection during pregnancy can result in birth defects, revealed a need for small animal models to study ZIKV pathogenic mechanisms and evaluate candidate vaccines and antivirals. Mice would be an attractive system for such studies, but ZIKV replicates poorly in laboratory mice because it fails to antagonize murine STAT2 and STING. To address this, most ZIKV pathogenesis studies have used mice with impaired interferon signaling (e.g. Ifnar1\u2212/\u2212 or treatment with IFNAR1-blocking antibodies). However, using mice with severe defects in innate antiviral signaling confounds studies of viral pathogenic mechanisms. Collaborative Cross (CC) mice have proven to be a valuable system for developing new mouse pathogenesis models for viral infections that are not well modeled in conventional laboratory mouse lines. To test whether CC mice could provide an immune-competent model for ZIKV pathogenesis, we infected CC lines with ZIKV and assessed weight loss, viremia, and production of neutralizing antibodies. We tested 21 CC lines (CC001, CC002, CC003, CC004, CC005, CC006, CC011, CC012, CC013, CC019, CC024, CC028, CC040, CC041, CC042, CC046, CC051, CC059, CC061, CC068, and CC072, 13 of which have non-functional alleles of the flavivirus restriction factor Oas1b) and 3 ZIKV strains (MR766, H/PF/2013, and a mouse-adapted variant of Dakar 41525). ZIKV infection did not induce weight loss compared to mock-infected controls and accordingly only low levels of viral RNA were detected in serum. Only a subset of mice developed neutralizing antibodies to ZIKV, likely due to overall low levels of infection and viremia. Our results are consistent with other studies demonstrating poor ZIKV infection in interferon-intact mice and suggest that the tested CC lines do not include polymorphic host genes that greatly increase susceptibility to ZIKV infection.","ABSTRACTEpilepsy is a neurological disorder with complex etiologies and genetic architecture. Animal models have a critical role in understanding the pathophysiology of epilepsy. Here we studied epilepsy utilizing a genetic reference population of Collaborative Cross (CC) mice with publicly available whole genome sequences. We measured multiple epilepsy traits in 35 CC strains, and we identified novel animal models that exhibit extreme outcomes in seizure susceptibility, seizure propagation, epileptogenesis, and sudden unexpected death in epilepsy. We performed QTL mapping in an F2 population and identified seven novel and one previously identified loci associated with seizure sensitivity. We combined whole genome sequence and hippocampal gene expression to pinpoint biologically plausible candidate genes and candidate variants associated with seizure sensitivity. These resources provide a powerful toolbox for studying complex features of seizures and for identifying genes associated with particular seizure outcomes, and hence will facilitate the development of new therapeutic targets for epilepsy.","AbstractThe recent emergence of Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) causing COVID-19 is a major burden for health care systems worldwide. It is important to address if the current infection control instructions based on active ingredients are sufficient. We therefore determined the virucidal activity of two alcohol-based hand rub solutions for hand disinfection recommended by the World Health Organization (WHO), as well as commercially available alcohols. Efficient SARS-CoV-2 inactivation was demonstrated for all tested alcohol-based disinfectants. These findings show the successful inactivation of SARS-CoV-2 for the first time and provide confidence in its use for the control of COVID-19.ImportanceThe current COVID-19 outbreak puts a huge burden on the world\u2019s health care systems. Without effective therapeutics or vaccines being available, effective hygiene measure are of utmost importance to prevent viral spreading. It is therefore crucial to evaluate current infection control strategies against SARS-CoV-2. We show the inactivation of the novel coronavirus for the first time and endorse the importance of disinfectant-based hand hygiene to reduce SARS-CoV-2 transmission.","AbstractTo explore potential intermediate host of a novel coronavirus is vital to rapidly control continuous COVID-19 spread. We found genomic and evolutionary evidences of the occurrence of 2019-nCoV-like coronavirus (named as Pangolin-CoV) from dead Malayan Pangolins. Pangolin-CoV is 91.02% and 90.55% identical at the whole genome level to 2019-nCoV and BatCoV RaTG13, respectively. Pangolin-CoV is the lowest common ancestor of 2019-nCoV and RaTG13. The S1 protein of Pangolin-CoV is much more closely related to 2019-nCoV than RaTG13. Five key amino-acid residues involved in the interaction with human ACE2 are completely consistent between Pangolin-CoV and 2019-nCoV but four amino-acid mutations occur in RaTG13. It indicates Pangolin-CoV has similar pathogenic potential to 2019-nCoV, and would be helpful to trace the origin and probable intermediate host of 2019-nCoV.","Rapid screening and low-cost diagnosis play a crucial role in choosing the correct course of intervention e.g., drug therapy, quarantine, no action etc. when dealing with highly infectious pathogens. This is especially important if the disease-causing agent has no effective treatment, such as the novel coronavirus SARS-CoV-2 (the pathogen causing COVID-19), and shows no or similar symptoms to other common infections. We report a silicon-based integrated Point-of-Need (PoN) transducer (TriSilix) that can chemically-amplify and detect pathogen-specific sequences of nucleic acids (NA) quantitatively in real-time. Unlike other silicon-based technologies, TriSilix can be produced at wafer-scale in a standard laboratory; we have developed a series of methodologies based on metal-assisted chemical (wet) etching, electroplating, thermal bonding and laser-cutting to enable a cleanroom-free low-cost fabrication that does not require processing in an advanced semiconductor foundry. TriSilix is, therefore, resilient to disruptions in the global supply chain as the devices can be produced anywhere in the world. To create an ultra-low-cost device, the architecture proposed exploits the intrinsic properties of silicon and integrates three modes of operation in a single chip: i) electrical (Joule) heater, ii) temperature sensor (i.e. thermistor) with a negative temperature coefficient that can provide the precise temperature of the sample solution during reaction and iii) electrochemical sensor for detecting target NA. Using TriSilix, the sample solution can be maintained at a single, specific temperature (needed for isothermal amplification of NA such as Recombinase Polymerase Amplification (RPA) or cycled between different temperatures (with a precision of \u00b11.3 \u00b0C) for Polymerase Chain Reaction (PCR) while the exact concentration of amplicons is measured quantitatively and in real-time electrochemically. A single 4-inch Si wafer yields 37 TriSilix chips of 10\u00d710\u00d70.65 mm in size and can be produced in 7 hours, costing ~0.35 USD per device. The system is operated digitally, portable and low power \u2014 capable of running up to 35 tests with a 4000 mAh battery (a typical battery capacity of a modern smartphone). We were able to quantitatively detect a 563-bp fragment (Insertion Sequence IS900) of the genomic DNA of M. avium subsp. paratuberculosis (extracted from cultured field samples) through PCR in real-time with a Limit-of-Detection of 20 fg, equivalent to a single bacterium, at the 30th cycle. Using TriSilix, we also detected the cDNA from SARS-CoV-2 (1 pg), through PCR, with high specificity against SARS-CoV (2003).","The pathogenic Middle East respiratory syndrome coronavirus (MERS-CoV), severe acute respiratory syndrome coronavirus (SARS-CoV-1) and COVID-19 coronavirus (SARS-CoV-2) have all emerged into the human population with devastating consequences. These viruses make use of a large envelope protein called spike (S) to engage host cell receptors and catalyze membrane fusion. Because of the vital role that these S proteins play, they represent a vulnerable target for the development of therapeutics to combat these highly pathogenic coronaviruses. Here, we describe the isolation and characterization of single-domain antibodies (VHHs) from a llama immunized with prefusion-stabilized coronavirus spikes. These VHHs are capable of potently neutralizing MERS-CoV or SARS-CoV-1 S pseudotyped viruses. The crystal structures of these VHHs bound to their respective viral targets reveal two distinct epitopes, but both VHHs block receptor binding. We also show cross-reactivity between the SARS-CoV-1 S-directed VHH and SARS-CoV-2 S, and demonstrate that this cross-reactive VHH is capable of neutralizing SARS-CoV-2 S pseudotyped viruses as a bivalent human IgG Fc-fusion. These data provide a molecular basis for the neutralization of pathogenic betacoronaviruses by VHHs and suggest that these molecules may serve as useful therapeutics during coronavirus outbreaks.","The ongoing SARS-CoV-2 pandemic has caused an unprecedented need for rapid diagnostic screening [1]. The primary assay employed is a reverse transcription quantitative polymerase chain reaction (RT-qPCR) assay that requires the use of an RNA extraction kit [2,3]. In the United States, the Centers for Disease Control and Prevention (CDC) protocol requires the use of the Qiagen QIAamp DSP Viral RNA Mini kit. The current shortage of this RNA extraction kit during this pandemic has resulted in a severe bottleneck in testing capacity. To address this problem, we tested two alternative strategies: the use of alternative RNA extraction kits or a direct RT-qPCR assay that omits an RNA extraction step altogether. We found that the Qiagen RNeasy Mini kit and the Qiagen RNeasy Micro kit could be substituted for the QIAamp Viral RNA Mini kit. Importantly, we report here that the RT-qPCR assay can be performed directly on patient sample material from a nasal swab, without the need for an RNA extraction step of any kind. Collectively, our findings provide viable options to circumvent supply chain issues in COVID-19 testing. Further, the ability to omit the RNA extraction step from RT-qPCR screening protocols would drastically ease supply chokepoints of COVID-19 screening and should be applicable throughout the world. We would note that our findings are preliminary and based on a single pooled nasopharyngeal swab sample from two previously confirmed positive COVID-19 patients. But due to the urgent need for high volume COVID-19 screening, we wanted to make these findings available immediately while we conduct replicate studies using additional patient samples.","AbstractThe fast and untraceable virus mutations take lives of thousands of people before the immune system can produce the inhibitory antibody. Recent outbreak of novel coronavirus infected and killed thousands of people in the world. Rapid methods in finding peptides or antibody sequences that can inhibit the viral epitopes of COVID-19 will save the life of thousands. In this paper, we devised a machine learning (ML) model to predict the possible inhibitory synthetic antibodies for Corona virus. We collected 1933 virus-antibody sequences and their clinical patient neutralization response and trained an ML model to predict the antibody response. Using graph featurization with variety of ML methods, we screened thousands of hypothetical antibody sequences and found 8 stable antibodies that potentially inhibit COVID-19. We combined bioinformatics, structural biology, and Molecular Dynamics (MD) simulations to verify the stability of the candidate antibodies that can inhibit the Corona virus.","Motivation: In molecular epidemiology, the identification of clusters of transmissions typically requires the alignment of viral genomic sequence data. However, existing methods of multiple sequence alignment scale poorly with respect to the number of sequences. Results: ViralMSA is a user-friendly reference-guided multiple sequence alignment tool that was built to enable the alignment of ultra-large viral genome datasets. It scales linearly with the number of sequences, and it is able to align tens of thousands of full viral genomes in seconds. Availability: ViralMSA is freely available at https://github.com/niemasd/ViralMSA as an open-source software project.","The SARS-CoV-2 pandemic is an urgent global public health emergency and warrants investigating molecular and structural studies addressing the dynamics of viral proteins involved in host cell adhesion. The recent comparative genomic studies highlight the insertion of Furin protease site in the SARS-CoV-2 spike glycoprotein alerting possible modification in the viral spike protein and its eventual entry to host cell and presence of Furin site implicated to virulence. Here we structurally show how Furin interacts with the SARS-CoV-2 spike glycoprotein homotrimer at S1/S2 region, which underlined the mechanism and mode of action, which is a key for host cell entry. Unravelling the structural features of the biding site opens the arena in rising bonafide antibodies targeting to block the Furin cleavage and have great implications in the development of Furin inhibitors or therapeutics.","Novel coronavirus 2019 (2019-nCoV), also known as SARS-CoV-2), leads high morbidity and mortality in global epidemics. Four structural protein (surface glycoprotein (QIQ22760.1), envelop glycoprotein (QIQ22762.1), nucleocapsid phosphoprotein (QIQ22768.1) and membrane glycoprotein (QIQ22763.1)) of SARS-CoV-2 are extracted from the NCBI database and further analyzed with ExPASy ProtParam tool. Phenyl alanine and serine are absolutely missing in all proteins. Histidine and glutamic acid is absent in envelope protein and cystein lacks in nucleocapsid phosphoprotein. Lucien is the highest in envelope, surface and membrane glycoprotein that is an optimal environment for rapid virus fixation on host cell's surface to the receptor molecule. Transmembrane region prediction was performed by SOSUI server. For all structural proteins, except nucleocapsid Phosphoprotein, the trans-membrane prediction indicates that the virus can enter the host easily. Domain analysis was done by SMART tool. Domain information helps in the function of the viral protein. Lastly, the 3D structure prediction was carried out by Swiss Model and the result validation was achieved by PROCHECK. This fairly simple method may help us understand how antivirals and vaccines could be produced against it Such models are the starting point of the community for structural drug and vaccine designs as well as virtual computational screening.","A disease outbreak named COVID-19, caused by a RNA virus, SARS-CoV-2 has become pandemic with a magnitude, which is daunting to all public health institutions in the absence of specific antiviral treatment. Surface glycoprotein and nucleocapsid phosphoprotein are two important proteins of this virus facilitating its entry into host cell and genome replication. Small interfering RNA (siRNA) is a prospective tool of the RNA interference (RNAi) pathway for the control of human viral infections by suppressing viral gene expression through hybridization and neutralization of target complementary mRNA. So, in this study, the power of RNA interference technology was harnessed to develop siRNA molecules against specific target genes namely, nucleocapsid phosphoprotein gene and surface glycoprotein gene. Conserved sequence from 139 SARS-CoV-2 strains from around the globe was collected to construct 78 siRNA that can inactivate nucleocapsid phosphoprotein and surface glycoprotein genes. Finally, through a rigorous filtering process 8 siRNA molecules were selected with exerts the best action. These predicted siRNAs should effectively silence the genes of SARS-CoV-2 during siRNA mediated treatment assisting in the response against SARS-CoV-2.","AbstractBackground Zika virus (ZIKV) emerged in Latin America &amp; the Caribbean (LAC) region in 2013, and has had serious implications for population health in the region. In 2016, the World Health Organization declared the ZIKV outbreak a Public Health Emergency of International Concern following a cluster of associated neurological disorders and neonatal malformations. In 2017, Zika cases declined, but future incidence in LAC remains uncertain due to gaps in our understanding, considerable variation in surveillance and a lack of a comprehensive collation of data from affected countries.Methods Our analysis combines information on confirmed and suspected Zika cases across LAC countries and a spatio-temporal dynamic transmission model for ZIKV infection to determine key transmission parameters and projected incidence in 91 major cities within 35 countries. Seasonality was determined by spatio-temporal estimates of Aedes aegypti vector capacity. We used country and state-level data from 2015 to mid-2017 to infer key model parameters, country-specific disease reporting rates, and the 2018 projected incidence. A 10-fold cross-validation approach was used to validate parameter estimates to out-of-sample epidemic trajectories.Results There was limited transmission in 2015, but in 2016 and 2017 there was sufficient opportunity for wide-spread ZIKV transmission in most cities, resulting in the depletion of susceptible individuals. We predict that the highest number of cases in 2018 within some Brazilian States (Sao Paulo and Rio de Janeiro), Colombia and French Guiana, but the estimated number of cases were no more than a few hundred. Model estimates of the timing of the peak in incidence were correlated (p&lt;0.05) with the reported peak in incidence. The reporting rate varied across countries, with lower reporting rates for those with only confirmed cases compared to those who reported both confirmed and suspected cases.Conclusions The findings suggest that the ZIKV epidemic is by and large over, with incidence projected to be low in most cities in LAC in 2018. Local low levels of transmission are probable but the estimated rate of infection suggests that most cities have a population with high levels of herd immunity.","AbstractSince December 2019, the outbreak of a new coronavirus, named 2019-nCoV, has greatly threatened the public health in China and raised great concerns worldwide. No specific treatment for this infection is currently available. We previously reported that teicoplanin, a glycopeptide antibiotic which has routinely been used in the clinic to treat bacterial infection with low toxicity, significantly inhibits the invasion of cells by Ebola virus, SARS-CoV and MERS-CoV, via specifically inhibiting the activity of cathepsin L. Here, we tested the efficacy of teicoplanin against 2019-nCoV virus infection and found that teicoplanin potently prevents the entrance of 2019-nCoV-Spike-pseudoviruses into the cytoplasm, with an IC50 of 1.66 \u03bcM. Although the inhibitory effect upon the replication of wildtype viruses ex vivo and in vivo remains to be determined, our preliminary result indicates that the potential antiviral activity of teicoplanin could be applied for the treatment of 2019-nCoV virus infection.","AbstractTo extend the frontier of genome editing and enable the radical redesign of mammalian genomes, we developed a set of dead-Cas9 base editor (dBE) variants that allow editing at tens of thousands of loci per cell by overcoming the cell death associated with DNA double-strand breaks (DSBs) and single-strand breaks (SSBs). We used a set of gRNAs targeting repetitive elements \u2013 ranging in target copy number from about 31 to 124,000 per cell. dBEs enabled survival after large-scale base editing, allowing targeted mutations at up to ~13,200 and ~2610 loci in 293T and human induced pluripotent stem cells (hiPSCs), respectively, three orders of magnitude greater than previously recorded. These dBEs can overcome current on-target mutation and toxicity barriers that prevent cell survival after large-scale genome engineering.One Sentence SummaryBase editing with reduced DNA nicking allows for the simultaneous editing of &gt;10,000 loci in human cells.","The CRISPR/Cas12a RNA-guided complexes have a tremendous potential for nucleic acid detection due to its ability to indiscriminately cleave ssDNA once bound to a target DNA. However, the current CRISPR/Cas12a systems are limited to detecting DNA in a picomolar detection limit without an amplification step. Here, we developed a platform with engineered crRNAs and optimized conditions that enabled us to detect DNA, DNA/RNA heteroduplex and methylated DNA with higher sensitivity, achieving a limit of detection of in femtomolar range without any target pre-amplification step. By extending the 3 prime- or 5 prime-ends of the crRNA with different lengths of ssDNA, ssRNA, and phosphorothioate ssDNA, we discovered a new self-catalytic behavior and an augmented rate of LbCas12a-mediated collateral cleavage activity as high as 3.5-fold compared to the wild-type crRNA. We applied this sensitive system to detect as low as 25 fM dsDNA from the PCA3 gene, an overexpressed biomarker in prostate cancer patients, in simulated urine over 6 hours. The same platform was used to detect as low as ~700 fM cDNA from HIV, 290 fM RNA from HCV, and 370 fM cDNA from SARS-CoV-2, all within 30 minutes without a need for target amplification. With isothermal amplification of SARS-CoV-2 RNA using RT-LAMP, the modified crRNAs were incorporated in a paper-based lateral flow assay that could detect the target with up to 23-fold higher sensitivity within 40-60 minutes.","AbstractBase J, \u03b2-D-glucosyl-hydroxymethyluracil, is a modification of thymine DNA base involved in RNA Polymerase (Pol) II transcription termination in kinetoplastid protozoa. Little is understood regarding how specific thymine residues are targeted for J-modification or the mechanism of J regulated transcription termination. To identify proteins involved in J-synthesis, we expressed a tagged version of the J-glucosyltransferase (JGT) in Leishmania tarentolae, and identified four co-purified proteins by mass spectrometry: protein phosphatase (PP1), a homolog of Wdr82, a potential PP1 regulatory protein (PNUTS) and a protein containing a J-DNA binding domain (named JBP3). Gel shift studies indicate JBP3 is a J-DNA binding protein. Reciprocal tagging, co-IP and sucrose gradient analyses indicate PP1, JGT, JBP3, Wdr82 and PNUTS form a multimeric complex in kinetoplastids, similar to the mammalian PTW/PP1 complex involved in transcription termination via PP1 mediated dephosphorylation of Pol II. Using RNAi and analysis of Pol II termination by RNA-seq and RT-PCR, we demonstrate that ablation of PNUTS, JBP3 and Wdr82 lead to defects in Pol II termination at the 3\u2019-end of polycistronic gene arrays in Trypanosoma brucei. Mutants also contain increased antisense RNA levels upstream of promoters, suggesting an additional role of the complex in regulating termination of bi-directional transcription. In addition, PNUTS loss causes derepression of silent Variant Surface Glycoprotein genes important for host immune evasion. Our results provide the first direct mechanistic link between base J and regulation of Pol II termination and suggest a novel molecular model for the role of the CTD of Pol II in terminating polycistronic transcription in trypanosomatids.Author SummaryTrypanosoma brucei is an early-diverged parasitic protozoan that causes African sleeping sickness in humans. The genome of T. brucei is organized into polycistronic gene clusters that contain multiple genes that are co-transcribed from a single promoter. We have recently described the presence of a modified DNA base J and variant of histone H3 (H3.V) at transcription termination sites within gene clusters where the loss of base J and H3.V leads to read-through transcription and the expression of downstream genes. We now identify a novel stable multimeric complex containing a J binding protein (JBP3), base J glucosyltransferase (JGT), PP1 phosphatase, PP1 interactive-regulatory protein (PNUTS) and Wdr82, which we refer to as PJW/PP1. A similar complex (PTW/PP1) has been shown to be involved in Pol II termination in humans and yeast. We demonstrate that PNUTS, JBP3 and Wdr82 mutants lead to read-through transcription in T. brucei. Our data suggest the PJW/PP1 complex regulates termination by recruitment to termination sites via JBP3-base J interactions and dephosphorylation of specific proteins (including Pol II and termination factors) by PP1. These findings significantly expand our understanding of mechanisms underlying transcription termination in eukaryotes, including divergent organisms that utilize polycistronic transcription and novel epigenetic marks such as base J and H3.V. The studies also provide the first direct mechanistic link between J modification of DNA at termination sites and regulated Pol II termination and gene expression in kinetoplastids.","AbstractAnimal relocation data has recently become considerably more ubiquitous, finely structured (collection frequencies measured in minutes) and co-variate rich (physiology of individuals, environmental and landscape information, and accelerometer data). To better understand the impacts of ecological interactions, individual movement and disease on global change ecology, including wildlife management and conservation, it is important to have simulators that will provide demographic, movement, and epidemiology null models against which to compare patterns observed in empirical systems. Such models may then be used to develop quantitative narratives that enhance our intuition and understanding of the relationship between population structure and generative processes: in essence, along with empirical and experimental narratives, quantitative narratives are used to advance ecological epistemology. Here we describe a simulator that accounts for the influence of consumer-resource interactions, existence of social groups anchored around a central location, territoriality, group-switching behavior, and disease dynamics on population size. We use this simulator to develop new and reinforce existing quantitative narratives and point out areas for future study.Author summaryThe health and viability of species are of considerable concern to all nature lovers. Population models are central to our efforts to assess the numerical and ecological status of species and threats posed by climate change. Models, however, are crude caricatures of complex ecological systems. So how do we construct reliable assessment models able to capture processes essential to predicating the impacts of global change on population viability without getting tied up in their vast complexities? We broach this question and demonstrate how models focusing at the level of the individual (i.e., agent-based models) are tools for developing robust, narratives to augment narratives arising purely from empirical data sources and experimental outcomes. We do this in the context of nesting social groups, foraging for food, while exhibiting territoriality and group-switching behavior; and, we evaluate the impact of disease on the viability of such populations.","AbstractHealth planners use forecasts of key metrics associated with influenza-like-illness (ILI); near-term weekly incidence, week of season onset, week of peak, and intensity of peak. Here, we describe our participation in a weekly prospective ILI forecasting challenge for the United States for the 2016-17 season and subsequent evaluation of our performance. We implemented a metapopulation model framework with 32 model variants. Variants differed from each other in their assumptions about: the force-of-infection (FOI); use of uninformative priors; the use of discounted historical data for not-yet-observed time points; and the treatment of regions as either independent or coupled. Individual model variants were chosen subjectively as the basis for our weekly forecasts; however, a subset of coupled models were only available part way through the season. Most frequently, during the 2016-17 season, we chose; FOI variants with both school vacations and humidity terms; uninformative priors; the inclusion of discounted historical data for not-yet-observed time points; and coupled regions (when available). Our near-term weekly forecasts substantially over-estimated incidence early in the season when coupled models were not available. However, our forecast accuracy improved in absolute terms and relative to other teams once coupled solutions were available. In retrospective analysis, we found that the 2016-17 season was not typical: on average, coupled models performed better when fit without historically augmented data. Also, we tested a simple ensemble model for the 2016-17 season and found that it underperformed our subjective choice for all forecast targets. In this study, we were able to improve accuracy during a prospective forecasting exercise by coupling dynamics between regions. Although reduction of forecast subjectivity should be a long-term goal, some degree of human intervention is likely to improve forecast accuracy in the medium-term in parallel with the systematic consideration of more sophisticated ensemble approaches.Author summaryIt is estimated that there are between 3 and 5 million worldwide annual seasonal cases of severe influenza illness, and between 290 000 and 650 000 respiratory deaths [1]. Influenza-like-illness (ILI) describes a set of symptoms and is a practical way for health-care workers to easily estimate likely influenza cases. The Centers for Disease Control (CDC) collects and disseminates ILI information, and has, for the last several years, run a forecasting challenge (the CDC Flu Challenge) for modelers to predict near-term weekly incidence, week of season onset, week of peak, and intensity of peak. We have developed a modeling framework that accounts for a range of mechanisms thought to be important for influenza transmission, such as climatic conditions, school vacations, and coupling between different regions. In this study we describe our forecast procedure for the 2016-17 season and highlight which features of our models resulted in better or worse forecasts. Most notably, we found that when the dynamics of different regions are coupled together, the forecast accuracy improves. We also found that the most accurate forecasts required some level of forecaster interaction, that is, the procedure could not be completely automated without a reduction in accuracy.","SUMMARYMany viruses globally shut off host gene expression to inhibit activation of cell-intrinsic antiviral responses. However, host shutoff is not indiscriminate, since viral proteins and host proteins required for viral replication are still synthesized during shutoff. The molecular determinants of target selectivity in host shutoff remain incompletely understood. Here, we report that the influenza A virus shutoff factor PA-X usurps RNA splicing to selectively target host RNAs for destruction. PA-X preferentially degrades spliced mRNAs, both transcriptome-wide and in reporter assays. Moreover, proximity-labeling proteomics revealed that PA-X interacts with cellular proteins involved in RNA splicing. The interaction with splicing contributes to target discrimination and is unique among viral host shutoff nucleases. This novel mechanism sheds light on the specificity of viral control of host gene expression and may provide opportunities for development of new host-targeted antivirals.","AbstractDecoding disease mechanisms for addressing unmet clinical need demands the rapid assimilation of the exponentially growing biomedical knowledge. These are either inherently unstructured and non-conducive to current computing paradigms or siloed into structured databases requiring specialized bioinformatics. Despite the recent renaissance in unsupervised neural networks for deciphering unstructured natural languages and the availability of numerous bioinformatics resources, a holistic platform for real-time synthesis of the scientific literature and seamless triangulation with deep omic insights and real-world evidence has not been advanced. Here, we introduce the nferX platform that makes the highly unstructured biomedical knowledge computable and supports the seamless visual triangulation with statistical inference from diverse structured databases. The nferX platform will accelerate and amplify the research potential of subject-matter experts as well as non-experts across the life science ecosystem (https://academia.nferx.com/).","AbstractSARS-CoV-2 emerged in China at the end of 2019 and has rapidly become a pandemic with over 400,000 recorded COVID-19 cases and greater than 19,000 recorded deaths by March 24th, 2020 (www.WHO.org) (1). There are no FDA approved antivirals or vaccines for any coronavirus, including SARS-CoV-2 (2). Current treatments for COVID-19 are limited to supportive therapies and off-label use of FDA approved drugs (3). Rapid development and human testing of potential antivirals is greatly needed. A potentially quicker way to test compounds with antiviral activity is through drug re-purposing (2, 4). Numerous drugs are already approved for use in humans and subsequently there is a good understanding of their safety profiles and potential side effects, making them easier to test in COVID-19 patients. Here, we present data on 20 FDA approved drugs tested for antiviral activity against SARS-CoV-2 that we have previously found to inhibit SARS-CoV and MERS-CoV (4). We find that 17 of these also inhibit SARS-CoV-2 at a range of IC50 values at non-cytotoxic concentrations. From these we specifically followed up with hydroxychloroquine sulfate and chloroquine phosphate.","AbstractWuhan coronavirus, called 2019-nCoV, is a newly emerged virus that infected more than 9692 people and leads to more than 213 fatalities by January 30, 2020. Currently, there is no effective treatment for this epidemic. However, the viral protease of a coronavirus is well-known to be essential for its replication and thus is an effective drug target. Fortunately, the sequence identity of the 2019-nCoV protease and that of severe-acute respiratory syndrome virus (SARS-CoV) is as high as 96.1%. We show that the protease inhibitor binding sites of 2019-nCoV and SARS-CoV are almost identical, which means all potential anti-SARS-CoV chemotherapies are also potential 2019-nCoV drugs. Here, we report a family of potential 2019-nCoV drugs generated by a machine intelligence-based generative network complex (GNC). The potential effectiveness of treating 2019-nCoV by using some existing HIV drugs is also analyzed.","AbstractThe behavioral response of individuals to an epidemic, and the timing of their behavior change, can have important consequences for epidemic dynamics. While the importance of behavior-change for epidemic dynamics is broadly recognized, the approaches to modeling the coupled dynamics of epidemics and behavior change have been limited. An important mode of behavior change in epidemics is a reduction in potentially infectious contacts. We develop a model for endogenous change of the effective contact rate of Susceptible-Infectious-Susceptible (SIS) epidemic model by positing a dynamic utility function associated with the expected number of contacts people have in the population. This utility function trades off the ideal number of social contacts with the expected cost of becoming infected. Our analysis of this simple, deterministic model reveals the existence of a non-zero endemic equilibrium, oscillatory dynamics under some parametric conditions, and complex dynamic regimes that shift under small parameter perturbations. Time-lag between current epidemic conditions and behavioral response is essential for the more exotic dynamics. These results suggest that reducing this lag may reduce both the final-size of the epidemic and the uncertainty associated with epidemic projections.","AbstractQuantification of aerosolized influenza virus is used for determining inhalation exposure. Several bioaerosol samplers and analytical methods have been used; however, the detection and quantification of influenza virus among aerosol samples remains challenging. Therefore, improved viral aerosol measurement methods are needed. This study evaluated influenza virus recovery among three filter types polytetrafluoroethylene, polyvinylchloride and polystyrene. Polytetrafluoroethylene, polyvinylchloride are fabricated filter materials and commonly used in the scientific literature to sample for viral aerosols. A novel, electrospun polystyrene filter material may improve viral aerosol recovery during filter-based air sampling. The filter materials were compared across the following conditions: treated with or without air, filter wash buffer (HBSS or PBS), and viral RNA extraction method (QIAamp Viral RNA Mini Kit or Trizol). Twenty trials were completed in a chamber and samples were analyzed using RT-qPCR. Viral recovery was significantly different (p-value &lt; .0001) by filter type. Polystyrene filter use resulted in recovery of the most viral RNA. Air sampling did not affect the recovery of viral RNA from the filter materials (p-values &gt; 0.05). Viral RNA concentrations were significantly different across extraction methods for all comparisons (p-values &lt; 0.05). Our results demonstrated that the novel polystyrene filter material resulted in the highest concentration of extracted RNA compared to the commonly used polytetrafluoroethylene and polyvinylchloride, which we speculate may be related to the chemical composition of the filter material (e.g., polystyrene is an aromatic hydrocarbon whereas polytetrafluoroethylene and polyvinylchloride contain more polar, and thus potentially reactive, carbon-halogen bonds). Air sampling did not have an effect on viral RNA recovery. Using Hanks Balanced Salt Solution with QIAamp Viral RNA Mini Kit, and Phosphate-buffered saline with the Trizol extraction, resulted in the most viral RNA recovery.","The COVID-19 pandemic is overwhelming healthcare systems worldwide. A significant portion of COVID-19 patients develop pneumonia and acute respiratory distress syndrome (ARDS), necessitating ventilator support. Some health systems do not have the capacity to accommodate this surge in ventilator demand, leading to shortages and inevitable mortality. Some clinicians have, of necessity, jerry-rigged ventilators to support multiple patients, but these devices lack protected air streams or individualized controls for each patient. Moreover, some have not been tested under conditions of ARDS. We have developed the Individualized System for Augmenting Ventilator Efficacy (iSAVE), a rapidly deployable platform to more safely use a single ventilator to simultaneously support multiple critically-ill patients. The iSAVE enables patient-specific volume and pressure control and incorporates safety features to mitigate cross-contamination between patients and flow changes due to patient interdependencies within the respiratory circuit. Here we demonstrate through simulated and in vivo pig evaluation the capacity of the iSAVE to support a range of respiratory clinical states. By leveraging off-the-shelf components that are readily available to intensive care unit (ICU) caregivers, the iSAVE could potentially be translated for human application to expand the ventilation capacity of hospitals using existing ventilators, minimizing the need to procure additional ventilators.","Many of the proteins produced by SARS-CoV-2 have related counterparts across the Severe Acute Respiratory Syndrome (SARS-CoV) family. One such protein is non-structural protein 9 (Nsp9), which is thought to mediate both viral replication and virulence. Current understanding suggests that Nsp9 is involved in viral genomic RNA reproduction. Nsp9 is thought to bind RNA via a fold that is unique to this class of betacoronoaviruses although the molecular basis for this remains ill-defined. We sought to better characterise the SARS-CoV-2 Nsp9 protein and subsequently solved its X-ray crystal structure, in an apo-form and, unexpectedly, in a peptide-bound form with a sequence originating from a rhinoviral 3C protease sequence (LEVL). The structure of the SARS-CoV-2 Nsp9 revealed the high level of structural conservation within the Nsp9 family. The exogenous peptide binding site is close to the dimer interface and impacted on the relative juxtaposition of the monomers within the homodimer. Together we have established a protocol for the production of SARS-CoV-2 Nsp9, determined its structure and identified a peptide-binding site that may warrant further study from the perspective of understanding Nsp9 function.","ABSTRACTIn 2013, U.S. swine producers were confronted with the disruptive emergence of porcine epidemic diarrhoea (PED). Movement of animals among farms is hypothesised to have played a role in the spread of PED among farms. Via this or other mechanisms, the rate of spread may also depend on the geographic density of farms and climate. To evaluate such effects on a large scale, we analyse state-level counts of outbreaks with variables describing the distribution of farm sizes and types, aggregate flows of animals among farms, and an index of climate. Our first main finding is that it is possible for a correlation analysis to be sensitive to transmission model parameters. This finding is based on a global sensitivity analysis of correlations on simulated data that included a biased and noisy observation model based on the available PED data. Our second main finding is that flows are significantly associated with the reports of PED outbreaks. This finding is based on correlations of pairwise relationships and regression modeling of total and weekly outbreak counts. These findings illustrate how variation in population structure may be employed along with observational data to improve understanding of disease spread.","AbstractIn December 2019, the first cases of a novel coronavirus infection were diagnosed in Wuhan, China. Due to international travel and human-to-human transmission, the virus spread rapidly inside and outside of China. Currently, there is no effective antiviral treatment for COVID-19, therefore research efforts are focused on the rapid development of vaccines and antiviral drugs. The SARS-CoV-2 Mpro protease constitutes one of the most attractive antiviral drug targets. To address this emerging problem, we have synthesized a combinatorial library of fluorogenic substrates with glutamine in the P1 position. We used it to determine the substrate preferences of the SARS-CoV and SARS-CoV-2 proteases, using natural and a large panel of unnatural amino acids. The results of our work provide a structural framework for the design of inhibitors as antiviral agents or diagnostic tests.","AbstractMany recombinant vector vaccines are capable of replication within the host. They consist of a fully competent vector backbone engineered to express an antigen from a foreign transgene. From the perspective of viral replication, the transgene is not only dispensable but may even be intrinsically detrimental. Thus vaccine revertants that delete the transgene may evolve to dominate the within-host population and in doing so reduce the antigenicity of the vaccine. We apply mathematical and computational models to study this process, including the dynamics of vaccine and revertant growth plus the dynamics of innate and adaptive immunity. Although the selective basis of vaccine evolution is easy to comprehend, the immunological consequences are not. One complication is that, despite possible fitness differences between vaccine and revertant, the opportunity for vaccine evolution is limited by the short period of growth before the viral population is cleared. Even less obvious, revertant per se does not interfere with immunity to vaccine except as the revertant suppresses vaccine abundance; the magnitude of this interference depends on mechanisms and timing of viral suppression. Adaptive immunity targeting the foreign antigen is also a possible basis of vaccine inferiority, but it is not worsened by vaccine evolution. Overall, we find that within-host vaccine evolution can sometimes matter to the adaptive immune response targeting the foreign antigen, but even when it does matter, simple principles of vaccine design and the control of inoculum composition can largely mitigate the effects.Author SummaryRecombinant vector vaccines are live replicating viruses that are engineered to carry extra genes derived from a pathogen \u2013 and these produce proteins against which we want to generate immunity. These genes may evolve to be lost during the course of replication within an individual, and there is a concern that this can severely limit the vaccine\u2019s efficacy. The dynamics of this process are studied here with mathematical models. The potential for vaccine evolution is somewhat reduced by the short-term growth of the vaccine population before it is suppressed by the immune response. Even when within-host evolution can be a problem, the models show that increasing the vaccine inoculum size or ensuring that the inoculum is mostly pure vaccine can largely avoid the loss of immunity arising from evolution.","ABSTRACTThe novel coronavirus SARS-CoV-2 was identified as the causative agent of the ongoing pandemic COVID 19. COVID-19-associated deaths are mainly attributed to severe pneumonia and respiratory failure. Recent work demonstrated that SARS-CoV-2 binds to angiotensin converting enzyme 2 (ACE2) in the lung. To better understand ACE2 abundance and expression patterns in the lung we interrogated our in-house single-cell RNA-sequencing dataset containing 70,085 EPCAM+ lung epithelial cells from paired normal and lung adenocarcinoma tissues. Transcriptomic analysis revealed a diverse repertoire of airway lineages that included alveolar type I and II, bronchioalveolar, club/secretory, quiescent and proliferating basal, ciliated and malignant cells as well as rare populations such as ionocytes. While the fraction of lung epithelial cells expressing ACE2 was low (1.7% overall), alveolar type II (AT2, 2.2% ACE2+) cells exhibited highest levels of ACE2 expression among all cell subsets. Further analysis of the AT2 compartment (n = 27,235 cells) revealed a number of genes co-expressed with ACE2 that are important for lung pathobiology including those associated with chronic obstructive pulmonary disease (COPD; HHIP), pneumonia and infection (FGG and C4BPA) as well as malarial/bacterial (CD36) and viral (DMBT1) scavenging which, for the most part, were increased in smoker versus light or non-smoker cells. Notably, DMBT1 was highly expressed in AT2 cells relative to other lung epithelial subsets and its expression positively correlated with ACE2. We describe a population of ACE2-positive AT2 cells that co-express pathogen (including viral) receptors (e.g. DMBT1) with crucial roles in host defense thus comprising plausible phenotypic targets for treatment of COVID-19.","ABSTRACTCOVID-19 has caused a major epidemic worldwide, however, much is yet to be known about the epidemiology and evolution of the virus. One reason is that the challenges underneath sequencing HCoV-19 directly from clinical samples have not been completely tackled. Here we illustrate the application of amplicon and hybrid capture (capture)-based sequencing, as well as ultra-high-throughput metatranscriptomic (meta) sequencing in retrieving complete genomes, inter-individual and intra-individual variations of HCoV-19 from clinical samples covering a range of sample types and viral load. We also examine and compare the bias, sensitivity, accuracy, and other characteristics of these approaches in a comprehensive manner. This is, to date, the first work systematically implements amplicon and capture approaches in sequencing HCoV-19, as well as the first comparative study across methods. Our work offers practical solutions for genome sequencing and analyses of HCoV-19 and other emerging viruses.","AbstractAs a heritable sequence-specific adaptive immune system, CRISPR-Cas is a powerful force shaping strain diversity in host-virus systems. While the diversity of CRISPR alleles has been explored, the associated structure and dynamics of host-virus interactions has not. We develop theory on the role of CRISPR immunity in mediating the interplay between host-virus interaction structure and eco-evolutionary dynamics in a computational model and three natural systems. We show that the structures of networks describing who infects whom and who is protected from whom are modular and weighted-nested, respectively. The dynamic interplay between these networks influences transitions between dynamical regimes of virus diversification and host control, leading to eventual virus extinction. The three empirical systems exhibit weighted nestedness, a pattern our theory shows is indicative of viral control by hosts. Previously missing from studies of microbial host-pathogen systems, the protection network plays a key role in the coevolutionary dynamics.","AbstractThe 2019 novel coronavirus (COVID19 / Wuhan coronavirus), officially named as Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2), is a positive-sense single-stranded RNA coronavirus. SARS-CoV-2 causes the contagious COVID19 disease also known as 2019-nCoV acute respiratory disease and has led to the ongoing 2019\u201320 pandemic COVID19 outbreak. The effective countermeasures against SARS-CoV-2 infection requires the design and development of specific and effective vaccine candidate. In the present study we have screened 38 CTL, 42 HTL and 12 B cell epitopes by utilizing different in silico tools. The screened epitopes were further validated for their binding with their respective HLA allele binders and TAP (Transporter associated with antigen processing) molecule by molecular docking. The screened epitopes were further utilized to design novel two multi-epitope vaccines (MEVs) composed of CTL, HTL and B cell epitopes overlaps with potential to elicit humoral as well as cellular immune response against SARS-CoV-2. To enhance the immune response for our vaccine design, truncated (residues 10-153) Onchocerca volvulus activation-associated secreted protein-1 (Ov-ASP-1) has been utilized as an adjuvant at N terminal of both the MEVs. Further molecular models for both the MEVs were prepared and validated for their stable molecular interaction with Toll-Like Receptor 3 (TLR 3). The codon-optimized cDNA of both the MEVs were further analyzed for their potential of high level of expression in Human cell line. The present study proposes in-silico validated design of two MEVs against SARS-CoV-2 to elicit humoral as well as cellular immune response.Graphical abstractThe designed CTL (Cytotoxic T lymphocyte) and HTL (Helper T lymphocyte) multi-epitope vaccine (MEV) against COVID19 infection. Both the CTL and HTL MEVs models show a very sytable and very fitting conformational complex formation tendency with the Toll like receptor 3. CTL and HTL MEVs: ribbon; Toll like receptor 3: gray cartoon; Adjuvant [truncated (residues 10-53). Onchocerca volvulus activation-associated secrete protein-1]: orange ribbon; Epitopes: cyan ribbons regions; 6xHis Tag: magenta ribbon region","ABSTRACTAlong with tRNAs, enzymes that modify anticodon bases are a key aspect of translation across the tree of life. tRNA modifications extend wobble pairing, allowing specific (\u201ctarget\u201d) tRNAs to recognize multiple codons and cover for other (\u201cnon-target\u201d) tRNAs, often improving translation efficiency and accuracy. However, the detailed evolutionary history and impact of tRNA modifying enzymes has not been analyzed. Using ancestral reconstruction of five tRNA modifications across 1093 bacteria, we show that most modifications were ancestral to eubacteria, but were repeatedly lost in many lineages. Most modification losses coincided with evolutionary shifts in non-target tRNAs, often driven by increased bias in genomic GC and associated codon use, or by genome reduction. In turn, the loss of tRNA modifications stabilized otherwise highly dynamic tRNA gene repertoires. Our work thus traces the complex history of bacterial tRNA modifications, providing the first clear evidence for their role in the evolution of bacterial translation.","Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) causes the infectious disease COVID-19, which was first reported in Wuhan, China in December, 2019. Despite the tremendous efforts to control the disease, COVID-19 has now spread to over 100 countries and caused a global pandemic. SARS-CoV-2 is thought to have originated in bats; however, the intermediate animal sources of the virus are completely unknown. Here, we investigated the susceptibility of ferrets and animals in close contact with humans to SARS-CoV-2. We found that SARS-CoV-2 replicates poorly in dogs, pigs, chickens, and ducks, but efficiently in ferrets and cats. We found that the virus transmits in cats via respiratory droplets. Our study provides important insights into the animal reservoirs of SARS-CoV-2 and animal management for COVID-19 control.","ABSTRACTWe have developed a novel detection system which couples CRISPR-Cas recognition of target sequences, Cas mediated nucleic acid probe cleavage, and quantum dots as highly sensitive reporter molecules for instrument-free detection of viral nucleic acid targets. After target recognition and Cas mediated cleavage of biotinylated ssDNA probe molecules, the probe molecules are bound to magnetic particles. A complimentary ssDNA oligonucleotide quantum dot conjugate is then added, which only hybridizes to un-cleaved probes on the magnetic beads. After separation of hybridized from unhybridized quantum dot conjugates by magnetic sequestration, the signal is measured fluorometrically to provide a signal proportional to the cleaved probes and therefore the amount of target nucleic acid. To demonstrate the power of this assay, a 250 bp DNA target sequence matching a portion of the African swine fever virus (ASFV) genome is used and a detection limit of ~0.5 nM is achieved without target amplification using a simple portable ultraviolet flashlight. The positive samples are readily confirmed by visual inspection, completely avoiding the need for complicated devices and instruments. This work establishes the feasibility of a simple, instrument free assay for rapid nucleic acid screening in both hospitals and point-of-care settings.","AbstractHigh-density lipoproteins (HDL) are nanoparticles with &gt;80 associated proteins, phospholipids, cholesterol and cholesteryl esters. A comprehensive genetic analysis of the regulation of proteome of HDL isolated from a panel of 100 diverse inbred strains of mice, Hybrid Mouse Diversity Panel (HMDP), revealed widely varied HDL protein levels across the strains. Some of this variation was explained by local, cis-acting regulation, termed cis-protein quantitative trait loci. Variations in apolipoprotein A-II and apolipoprotein C-3 affected the abundance of multiple HDL proteins indicating a coordinated regulation. We identified modules of co-varying proteins and define a protein-protein interaction network describing the protein composition of the naturally occurring subspecies of HDL in mice. Sterol efflux capacity varied up to 3-fold across the strains and HDL proteins displayed distinct correlation patterns with macrophage and ABCA1 specific cholesterol efflux capacity and cholesterol exchange, suggesting that subspecies of HDL participate in discrete functions. The baseline and stimulated sterol efflux capacity phenotypes associated with distinct QTLs with smaller effect size suggesting a multi genetic regulation. Our results highlight the complexity of HDL particles by revealing high degree of heterogeneity and intercorrelation, some of which is associated with functional variation, supporting the concept that HDL-cholesterol alone is not an accurate measure of HDL\u2019s properties such as protection against CAD.","ABSTRACTCoronaviruses may produce severe acute respiratory syndrome (SARS). As a matter of fact, a new SARS-type virus, SARS-CoV-2, is responsible of a global pandemic in 2020 with unprecedented sanitary and economic consequences for most countries. In the present contribution we study, by all-atom equilibrium and enhanced sampling molecular dynamics simulations, the interaction between the SARS Unique Domain and RNA guanine quadruplexes, a process involved in eluding the defensive response of the host thus favoring viral infection of human cells. The results obtained evidence two stable binding modes with guanine quadruplexes, driven either by electrostatic (dimeric mode) or by dispersion (monomeric mode) interactions, are proposed being the dimeric mode the preferred one, according to the analysis of the corresponding free energy surfaces. The effect of these binding modes in stabilizing the protein dimer was also assessed, being related to its biological role in assisting SARS viruses to bypass the host protective response. This work also constitutes a first step of the possible rational design of efficient therapeutic agents aiming at perturbing the interaction between SARS Unique Domain and guanine quadruplexes, hence enhancing the host defenses against the virus.TOC GRAPHICS","ABSTRACTViruses maximize their genetic coding capacity through a variety of biochemical mechanisms including programmed ribosomal frameshifting (PRF), which facilitates the production of multiple proteins from a single transcript. PRF is typically stimulated by structural elements within the mRNA that generate mechanical tension between the transcript and ribosome. However, in this work we show that the forces generated by the cotranslational folding of the nascent polypeptide chain can also enhance PRF. Using an array of biochemical, cellular, and computational techniques, we first demonstrate that the Sindbis virus structural polyprotein forms two competing topological isomers during biosynthesis at the ribosome-translocon complex. We then show that the formation of one of these topological isomers is linked to PRF. Coarse-grained molecular dynamic simulations reveal that the translocon-mediated membrane integration of a transmembrane domain upstream from the ribosomal slip-site generates a force on the nascent polypeptide chain that scales with observed frameshifting. Together, our results demonstrate that cotranslational folding of this protein generates a tension that stimulates PRF. To our knowledge, this constitutes the first example in which the conformational state of the nascent chain has been linked to PRF. These findings raise the possibility that, in addition to RNA-mediated translational recoding, a variety of cotranslational folding and/ or binding events may also stimulate PRF.","AbstractRecently a novel coronavirus (SARS-CoV-2) emerged from Wuhan, China and has infected more than 571000 people leading to more than 26000 deaths. Since SARS-CoV-2 genome sequences show similarity with those of SARS, we sought to analyze all the available SARS-CoV-2 genomes based on the insights obtained from SARS genome specifically focusing on non-coding RNAs. Here, results are presented from the dual approach i.e identifying host encoded miRNAs that might regulate viral pathogenesis as well as identifying viral encoded miRNAs that might regulate host cell signaling pathways and aid in viral pathogenesis. Analysis utilizing first approach resulted in the identification of 10 host encoded miRNAs that could target the genome of both the viruses (SARS-CoV-2 and SARS reference genome). Interestingly our analysis revealed that there is significantly higher number of host miRNAs that could target SARS-CoV-2 genome as compared to the SARS reference genome. Results from second approach involving SARS-CoV-2 and SARS reference genome identified a set of virus encoded miRNAs which might regulate host signaling pathways. Our analysis further identified a similar \u201cGA\u201d rich motif in SARS-CoV-2 genome that was shown to play a vital role in lung pathogenesis during severe SARS infections. Hence, we successfully identified human and virus encoded miRNAs that might regulate pathogenesis of both these coronaviruses and the fact that more number of host miRNAs could target SARS-CoV-2 genomes possibly reveal as to why this virus follows mild pathogenesis in healthy individuals. We identified non-coding sequences in SARS-CoV-2 genomes that were earlier reported to contribute towards SARS pathology. The study provides insights into the overlapping sequences among these viruses for their effective inhibition as well as identifying new drug targets that could be used for development of new antivirals.","ABSTRACTNewcastle disease virus (NDV) causes severe infectious disease in poultry, and selectively kills tumor cells by inducing apoptosis. In this report, we revealed the mechanisms underlying NDV-induced apoptosis via investigation of endoplasmic reticulum (ER) stress-related unfolded protein response (UPR) in HeLa cells. We found that NDV infection induced the expression of pro-apoptotic transcription factor CHOP via PKR-eIF2\u03b1 pathway. Knock down and exogenous expression studies showed that CHOP promoted cell apoptosis by down-regulation of anti-apoptotic protein BCL-2 and MCL-1, promotion of pro-apoptotic JNK and p38 signaling, and suppression of pro-survival AKT signaling. Meanwhile, CHOP facilitated NDV proliferation. Furthermore, virus infection activated IRE1\u03b1, another ER stress sensor, thereby promoting the mRNA splicing of XBP1 and resulting in the translation of transcription factor XBP1s. XBP1s entered into cell nucleus, promoted the expression of ER chaperones and components of ER associated degradation (ERAD). Exogenous expression of XBP1s helped IBV proliferation, and silence of XBP1s reduced virus proliferation. Meanwhile, exogenous expression and knock down studies demonstrated that IRE1\u03b1 activated pro-apoptotic JNK signaling, promoted apoptosis and inflammation. In conclusion, our current study demonstrates that the induction of CHOP and activation of IRE1\u03b1-XBP1/JNK signaling cascades promote apoptosis and benefit NDV proliferation.IMPORTANCEIt is well known that NDV kills host animal and tumor cells by inducing cell apoptosis. Although several studies investigate the apoptotic phenomena in NDV-infected tumor cells, the molecular mechanisms underlying this oncolytic virus induced apoptosis is not well understood yet. In this study, we focus on characterization of the ER stress responses in NDV-infected tumor cells, and find that virus induces apoptosis by up-regulation or activation of several unfolded protein responses (UPR) related transcription factors and signaling: such as ATF4, CHOP and XBP1s, and pro-apoptotic kinases (IRE1\u03b1, JNK, p38). Moreover, activation of these transcription factors and signaling cascades helps virus proliferation. Our study dissects the UPR induced apoptosis in NDV-infected tumor cells, and provides the evidence that UPR favors NDV proliferation.","Viral gain-of-function mutations are commonly observed in the laboratory; however, it is unknown whether those mutations also evolve in nature. We identify two key residues in the host recognition protein of bacteriophage \u03bb that are necessary to exploit a new receptor; both residues repeatedly evolved among homologs from environmental samples. Our results provide evidence for widespread host-shift evolution in nature and a proof of concept for integrating experiments with genomic epidemiology.","The avian coronavirus infectious bronchitis virus (IBV) typically infects the respiratory tract of chickens, but can also spread to other organs. However, the mechanisms of virus dissemination are presently unclear. We show that peripheral blood-derived monocytes/macrophages from chickens (chPBMC) are productively infected by clinical strains of IBV, accompanied by induction of apoptosis. Our data suggest that chPBMCs play a role in the dissemination of IBV, and may be important for viral pathogenesis.","AbstractThe coronavirus outbreak took place in December 2019 and continues to spread worldwide. In the absence of an effective vaccine, inhibitor repurposing may seem a fruitful attempt. Here, we compared Mpros from SARS-CoV-2 and SARS-CoV. Despite a high level of sequence similarity, the binding sites of analysed proteins show major differences in both shape and size indicating that repurposing SARS drugs for COVID-19 may be futile. Furthermore, the analysis of the pockets\u2019 conformational changes during the simulation time indicates their flexibility, which dashes hopes for rapid and reliable drug design. Conversely, structural stability of the SARS-CoV-2 Mpro with respect to mutations of the binding cavity and adjacent flexible loops indicates that the protein\u2019s mutability will pose a further challenge to the rational design of small-molecule inhibitors. However, few residues contribute significantly to the protein stability and thus can be considered as key anchoring residues for Mpro inhibitor design.","AbstractOver the past 20 years, several coronaviruses have crossed the species barrier into humans, causing outbreaks of severe, and often fatal, respiratory illness. Since SARS- CoV was first identified in animal markets, global viromics projects have discovered thousands of coronavirus sequences in diverse animals and geographic regions. Unfortunately, there are few tools available to functionally test these novel viruses for their ability to infect humans, which has severely hampered efforts to predict the next zoonotic viral outbreak. Here we developed an approach to rapidly screen lineage B betacoronaviruses, such as SARS-CoV and the recent 2019-nCoV, for receptor usage and their ability to infect cell types from different species. We show that host protease processing during viral entry is a significant barrier for several lineage B viruses and that bypassing this barrier allows several lineage B viruses to enter human cells through an unknown receptor. We also demonstrate how different lineage B viruses can recombine to gain entry into human cells and confirm that human ACE2 is the receptor for the recently emerging 2019-nCoV.","AbstractHost shifts, where a pathogen invades and establishes in a new host species, are a major source of emerging infectious diseases. They frequently occur between related host species and often rely on the pathogen evolving adaptations that increase their fitness in the novel host species. To investigate genetic changes in novel hosts, we experimentally evolved replicate lineages of an RNA virus (Drosophila C Virus) in 19 different species of Drosophilidae and deep sequenced the viral genomes. We found a strong pattern of parallel evolution, where viral lineages from the same host were genetically more similar to each other than to lineages from other host species. When we compared viruses that had evolved in different host species, we found that parallel genetic changes were more likely to occur if the two host species were closely related. This suggests that when a virus adapts to one host it might also become better adapted to closely related host species. This may explain in part why host shifts tend to occur between related species, and may mean that when a new pathogen appears in a given species, closely related species may become vulnerable to the new disease.","AbstractStarting from December 2019, a novel coronavirus, named 2019-nCoV, was found to cause Severe Acute Respiratory (SARI) symptoms and rapid pandemic in China. With the hope to identify candidate drugs for 2019-nCoV, we adopted a computational approach to screen for available commercial medicines which may function as inhibitors for the Mpro of 2019-nCoV. Up to 10 commercial medicines that may form hydrogen bounds to key residues within the binding pocket of 2019-nCoV Mpro were identified, which may have higher mutation tolerance than lopinavir/ritonavir and may also function as inhibitors for other coronaviruses with similar Mpro binding sites and pocket structures.","AbstractThe outbreak of Corona Virus Disease 2019 (COVID-19) is highly infectious and transmitted mainly through human-to-human transmission via respiratory droplets and direct or close contact to the patients with SARS-CoV-2. The other potential transmission routes remain to be further researched. In some clinical cases, samples of tears and conjunctival secretions from both SARS-CoV[1] and SARS-CoV-2 patients with conjunctivitis[2] displayed detectable viral RNA. A previous study reported the case of a clinician who was infected with SARS-CoV-2 while working with patients under all safeguards except eye protection [2]. By contrast, no SARS-CoV-2 could be detected by RT-PCR in 114 conjunctival swabs samples from patients with COVID-19 pneumonia [4]. Anatomically, the linkage of the ocular with respiratory tissues is primarily by the nasolacrimal system [5]. The potential extra-respiratory transmissible routes of SARS-CoV-2 via ocular remained unclear. Whether ocular conjunctiva is one of the portals that SARS-CoV-2 enters the host needs to be further research by laboratory-confirmation for providing significant data to oversight and prevention in people.","ABSTRACTHuman astrovirus nonstructural protein nsP1a/4, located at the C-terminal end of nsP1a, is thought to be involved in the regulation of RNA replication and capsid maturation;however, its rolesviral growth and virulence are not well understood. We investigated the intracellular host proteins that interact with nsP1a and explored the potential roles of the interaction in the pathogenesis of human astrovirus infection. We screened 14 independent proteins with a cDNA library derived from Caco-2 cells using a yeast two-hybrid technique. Deletion analysis revealed that interaction between the nsP1a/4 domain and the large extracellular loop (LEL) domain of the human protein CD63 is necessary for astrovirus replication. The interaction was confirmed by glutathione-S-transferase (GST) pull-down assays and co-immunoprecipitation assays. Confocal microscopy showed that nsP1a/4 and CD63 co-localized in the cytoplasm of infected cells. Over expression of CD63 promoted viral RNA synthesis, whereas knockdown of CD63 markedly decreased viral RNA levels. Those results suggest that CD63 plays a critical role in human astrovirus RNA replication. The interaction between CD63 and nsP1a/4 provides a channel to further understand the roles of interactions between host and virus proteins in astrovirus infection and release.IMPORTANCEHuman astroviruses cause gastroenteritis in young children and immunocompromised patients. In this study, we provide evidence that nsP1a/4, a nonstructural protein located at the C-terminal end of the human astrovirus nsP1a polyprotein, interacts with the host protein CD63. Over expression of CD63 promoted viral RNA replication, whereas knockdown of CD63 decreased virus RNA replication, indicating that CD63 plays a critical role in the human astrovirus life cycle.","AbstractAirborne-mediated microbial diseases such as influenza and tuberculosis represent major public health challenges. A direct approach to prevent airborne transmission is inactivation of airborne pathogens, and the airborne antimicrobial potential of UVC ultraviolet light has long been established; however, its widespread use in public settings is limited because conventional UVC light sources are both carcinogenic and cataractogenic. By contrast, we have previously shown that far-UVC light (207\u2013222 nm) efficiently kills bacteria without harm to exposed mammalian skin. This is because, due to its strong absorbance in biological materials, far-UVC light cannot penetrate even the outer (non living) layers of human skin or eye; however, because bacteria and viruses are of micrometer or smaller dimensions, far-UVC can penetrate and inactivate them. We show for the first time that far-UVC efficiently kills airborne aerosolized viruses, a very low dose of 2 mJ/cm2 of 222-nm light inactivating &gt;95% of aerosolized H1N1 influenza virus. Continuous very low dose-rate far-UVC light in indoor public locations is a promising, safe and inexpensive tool to reduce the spread of airborne-mediated microbial diseases.","AbstractThe ongoing COVID-19 pandemic continues to pose a major public health burden around the world. The novel coronavirus, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), has infected over one million people worldwide as of April, 2020, and has led to the deaths of nearly 300,000 people. No approved vaccines or treatments in the USA currently exist for COVID-19, so there is an urgent need to develop effective countermeasures. The IMPDH inhibitor merimepodib (MMPD) is an investigational antiviral drug that acts as a noncompetitive inhibitor of IMPDH. It has been demonstrated to suppress replication of a variety of emerging RNA viruses. We report here that MMPD suppresses SARS-CoV-2 replication in vitro. After overnight pretreatment of Vero cells with 10 \u03bcM of MMPD, viral titers were reduced by 4 logs of magnitude, while pretreatment for 4 hours resulted in a 3-log drop. The effect is dose-dependent, and concentrations as low as 3.3 \u03bcM significantly reduced viral titers when the cells were pretreated prior to infection. The results of this study provide evidence that MMPD may be a viable treatment option for COVID-19.","Different forms of lung injury can activate senescence pathways. Senescence has been involved in chronic lung diseases, but its role in the acute phase is less known. Acute lung injury activates a large variety of pathogenetic mechanisms that can lead to respiratory insufficiency. In some cases, mechanical ventilation may help to maintain gas exchange, but positive airway pressures can cause regional overdistension of alveolar units and further damage. Here, we report that acute lung injury and alveolar overstretching activate a lung senescent program aimed to limit apoptosis. A systematic pooling of transcriptomic data from animal models of lung injury demonstrates the enrichment of a specific senescence signature. In a clinically relevant, murine model of acid aspiration and mechanical ventilation, we observed the activation of the Tp53/Cdkn1a (p21) senescence pathway, increases in senescence-associated heterochromatin foci in alveolar cells and markers of DNA damage, chromatin reorganization and changes in the nuclear envelope. Autopsy samples from lungs of critically ill patients showed the increase in senescence-associated heterochromatin foci. Cell stretch per se was also able to activate the senescence response in an ex-vivo model. Absence of Cdkn1a decreased the senescent response, but worsened lung injury by a significant increase in apoptosis. In opposite, treatment with lopinavir/ritonavir led to Cdkn1a overexpression and ameliorated cell apoptosis and lung injury. Collectively, these results suggest that acute injury promotes lung senescence. This response plays a protective role against acute lung injury in the short term, but its long-term consequences remain unexplored.","AbstractThe ecological success of social insects is based on division of labour, not only between queens and workers, but also among workers. Whether a worker tends the brood or forages is strongly influenced by age, fertility and nutritional status, with brood carers being younger, more fecund and corpulent. Here, we experimentally disentangle behaviour from age and fertility in Temnothorax longispinosus ant workers and analyse how these parameters are linked to whole-body gene expression. Our transcriptome analysis reveals four times more genes associated with behaviour than with age and only few fertility-associated genes. Brood carers exhibited an upregulation of genes involved in lipid biosynthesis, whereas foragers invested in metabolism. Additional simulations revealed that the experimental disassociation of co-varying factors reduces transcriptomic noise, potentially explaining discrepancies between transcriptomic studies on worker behaviour in other social insects. Our study highlights the influence of nutritional status on task choice in ant workers.","AbstractMiddle East respiratory syndrome coronavirus (MERS-CoV) continues to cause zoonotic infections and serious disease, primarily in the Arabian Peninsula, due to repeated spill-over from dromedary camels and subsequent nosocomial transmission. Approved MERS vaccines for use in animals or humans are not currently available. MERS-CoV replication requires the virus-encoded papain-like protease (PLpro) to cleave multiple sites in the viral replicase polyproteins, thereby releasing functional non-structural proteins. Additionally, PLpro is a deubiquitinating enzyme (DUB) that can remove ubiquitin(-like) moieties from substrates, presumably to counteract host antiviral responses. In previous work, we determined the crystal structure of MERS-CoV PLpro in complex with ubiquitin, facilitating the design of PLpro mutations that impair DUB activity without affecting viral polyprotein cleavage. Here, we introduced these DUB-inactivating mutations into the viral genome and examined their impact on MERS-CoV infection both in cell culture and in a lethal mouse model. Although overall replication of DUB-negative and wild-type (wt) recombinant MERS-CoV was comparable in multiple cell lines, infection with DUB-negative virus markedly increased mRNA levels for interferon (IFN)-\u03b2 and IFN-stimulated genes. Moreover, compared to a wt virus infection, the survival rate was significantly increased when DUB-negative MERS-CoV was used to infect transgenic mice expressing a human MERS-CoV receptor. Interestingly, DUB-negative and wt MERS-CoV replicated to the same titers in lungs of infected mice, but the DUB-negative virus was cleared faster, likely due to the observed accelerated and better-regulated innate immune responses, in contrast to delayed and subsequently excessive responses in wt virus-infected mice. This study provides the first direct evidence that the DUB activity of a coronaviral protease contributes to innate immune evasion and can profoundly enhance virulence in an animal model. Thus, reduction or removal of the innate immune-suppressive DUB activity of PLpros is a promising strategy for coronavirus attenuation in the context of rational vaccine development.Author SummaryAlthough zoonotic coronaviruses such as Middle East respiratory coronavirus (MERS-CoV) have pandemic potential, therapeutics and vaccines that counteract this public health threat are not currently available. Coronaviruses typically employ multiple strategies to evade the host\u2019s innate immune response, which may enhance clinical disease and/or reduce the efficacy of modified live vaccines. The MERS-CoV-encoded papain-like protease (PLpro) is not only crucial for the expression of functional replicase proteins, but has also been postulated to antagonize ubiquitination-dependent steps during the activation of the innate immune response. Here, we report the generation of engineered MERS-CoVs mutants in which PLpro\u2019s deubiquitinating (DUB) activity was specifically disrupted without affecting virus viability. In this manner, we could demonstrate that the DUB activity of PLpro suppresses the interferon response in MERS-CoV-infected cells. Strikingly, in the lungs of mice infected with DUB-negative MERS-CoV, innate immune responses were induced at an earlier stage of infection than in wt virus-infected mice. This group also showed a clearly increased survival, indicating that the DUB activity is an important MERS-CoV virulence factor. This proof-of-concept study establishes that the engineering of DUB-negative coronaviruses, which elicit a more effective immune response in the host, is a viable strategy for vaccine development.","ABSTRACTEffective countermeasures against the recent emergence and rapid expansion of the 2019-Novel Coronavirus (2019-nCoV) require the development of data and tools to understand and monitor viral spread and immune responses. However, little information about the targets of immune responses to 2019-nCoV is available. We used the Immune Epitope Database and Analysis Resource (IEDB) resource to catalog available data related to other coronaviruses, including SARS-CoV, which has high sequence similarity to 2019-nCoV, and is the best-characterized coronavirus in terms of epitope responses. We identified multiple specific regions in 2019-nCoV that have high homology to SARS virus. Parallel bionformatic predictions identified a priori potential B and T cell epitopes for 2019-nCoV. The independent identification of the same regions using two approaches reflects the high probability that these regions are targets for immune recognition of 2019-nCoV.ONE SENTENCE SUMMARYWe identified potential targets for immune responses to 2019-nCoV and provide essential information for understanding human immune responses to this virus and evaluation of diagnostic and vaccine candidates.","AbstractMotivationAccurately determining the atomic structure of proteins represents a fundamental problem in the field of structural bioinformatics. A solution would be significant as protein structure information could be utilized in the medical field, e.g. in the development of vaccines for new viruses. This paper focuses on predicting the protein structure based on 3D images of the proteins captured through cryogenic electron microscopes (cryo-EM). A fully automated computationally efficient protein structure prediction method would be particularly beneficial in the field of cryo-EM as the technology allows researchers to photograph multiple large protein complexes in a single study, which means that a fast prediction method could allow for a high throughput of derived protein structures. We present a deep learning approach, DeepTracer, for predicting locations of the backbone atoms, secondary structure elements, and the amino acid types. In order to connect the predicted amino acids into chains, we applied a modified traveling salesman algorithm.ResultsWe trained our deep learning model on experimental cryo-EM density maps and tested it on a set of 50 density maps. We found that our new approach predicted protein structures with an average RMSD value of 1.18 and a coverage of 87.5%. Furthermore, we detected secondary structure information for 87.2% of amino acids correctly. We also showed preliminarily that 25.2% of amino acid types could be predicted directly from the 3D cryo-EM density map, considering 20 different types in total. Finally, we noted that the prediction runtime of DeepTracer is significantly improved compared to other methods. It predicts a large protein complex structure of more than 30,000 amino acids in only 2 hours.AvailabilityThe repository of this project will be published.Contactdongsi@uw.eduSupplementary informationSupplementary data will be available at Bioinformatics online.","The 2019 novel coronavirus, SARS-CoV-2, is an emerging pathogen of critical significance to international public health. Knowledge of the interplay between molecular-scale virus-receptor interactions, single-cell viral replication, intracellular-scale viral transport, and emergent tissue-scale viral propagation is limited. Moreover, little is known about immune system-virus-tissue interactions and how these can result in low-level (asymptomatic) infections in some cases and acute respiratory distress syndrome (ARDS) in others, particularly with respect to presentation in different age groups or pre-existing inflammatory risk factors like diabetes. A critical question for treatment and protection is why it appears that the severity of infection may correlate with the initial level of virus exposure. Given the nonlinear interactions within and among each of these processes, multiscale simulation models can shed light on the emergent dynamics that lead to divergent outcomes, identify actionable \"choke points\" for pharmacologic interactions, screen potential therapies, and identify potential biomarkers that differentiate response dynamics. Given the complexity of the problem and the acute need for an actionable model to guide therapy discovery and optimization, we introduce a prototype of a multiscale model of SARS-CoV-2 dynamics in lung and intestinal tissue that will be iteratively refined. The first prototype model was built and shared internationally as open source code and interactive, cloud-hosted executables in under 12 hours. In a sustained community effort, this model will integrate data and expertise across virology, immunology, mathematical biology, quantitative systems physiology, cloud and high performance computing, and other domains to accelerate our response to this critical threat to international health.","AbstractBackground.T cell immunoglobulin mucin domain-1 (TIM-1) is a phosphatidylserine (PS) receptor, mediating filovirus entry into cells through interactions with PS on virions. TIM-1 expression has been implicated in Ebola virus (EBOV) pathogenesis; however, it remains unclear whether this is due to TIM-1 serving as a filovirus receptor in vivo or, as others have suggested, TIM-1 induces a cytokine storm elicited by T cell/virion interactions. Here, we use a BSL2 model virus that expresses EBOV glycoprotein and demonstrate the importance of TIM-1 as a virus receptor late during in vivo infection.Methodology/Principal findings.We used an infectious, recombinant vesicular stomatitis virus expressing EBOV glycoprotein (EBOV GP/rVSV) to assess the role of TIM-1 during in vivo infection. TIM-1-sufficient or TIM-1-deficient BALB/c interferon \u03b1/\u03b2 receptor-/- mice were challenged with EBOV GP/rVSV-GFP or G/rVSV-GFP. While G/rVSV caused profound morbidity and mortality in both mouse strains, TIM-1-deficient mice had significantly better survival than TIM-1-expressing mice following EBOV GP/rVSV challenge. EBOV GP/rVSV load in spleen was high and unaffected by expression of TIM-1. However, infectious virus in serum, liver, kidney and adrenal gland was reduced late in infection in the TIM-1-deficient mice, suggesting that virus entry via this receptor contributes to virus load. Consistent with higher virus loads, proinflammatory chemokines trended higher in organs from infected TIM-1-sufficient mice compared to the TIM-1-deficient mice, but proinflammatory cytokines were more modestly affected. To assess the role of T cells in EBOV GP/rVSV pathogenesis, T cells were depleted in TIM-1-sufficient and -deficient mice and the mice were challenged with virus. Depletion of T cells did not alter the pathogenic consequences of virus infection.Conclusions.Our studies provide evidence that at late times during EBOV GP/rVSV infection, TIM-1 increased virus load and associated mortality, consistent with an important role of this receptor in virus entry. This work suggests that inhibitors which block TIM-1/virus interaction may serve as effective antivirals, reducing virus load at late times during EBOV infection.Author summaryT cell immunoglobulin mucin domain-1 (TIM-1) is one of a number of phosphatidylserine (PS) receptors that mediate clearance of apoptotic bodies by binding PS on the surface of dead or dying cells. Enveloped viruses mimic apoptotic bodies by exposing PS on the outer leaflet of the viral membrane. While TIM-1 has been shown to serve as an adherence factor/receptor for filoviruses in tissue culture, limited studies have investigated the role of TIM-1 as a receptor in vivo. Here, we sought to determine if TIM-1 was critical for Ebola virus glycoprotein-mediated infection using a BSL2 model virus. We demonstrate that loss of TIM-1 expression results in decreased virus load late during infection and significantly reduced virus-elicited mortality. These findings provide evidence that TIM-1 serves as an important receptor for Ebola virus in vivo. Blocking TIM-1/EBOV interactions may be effective antiviral strategy to reduce viral load and pathogenicity at late times of EBOV infection.","AbstractBats are an important reservoir for pathogenic human respiratory and hemorrhagic viruses but only little is known about bat viruses causing gastroenteritis in humans, including rotavirus A strains (RVA). Only three RVA strains have been reported in bats in Kenya (straw-colored fruit bat) and in China (lesser horseshoe and a stoliczka\u2019s trident bat), being highly divergent from each other. To further elucidate the potential of bat RVAs to cause gastroenteritis in humans we started by investigating the genetic diversity of RVAs in fecal samples from 87 straw-colored fruit bats living in close contact with humans in Cameroon using metagenomics. Five samples contained significant numbers of RVA Illumina reads, sufficient to obtain their (near) complete genomes. A single RVA strain showed a close phylogenetic relationship with the Kenyan bat RVA strain in six gene segments, including VP7 (G25), whereas the other gene segments represented novel genotypes as ratified by the RCWG. The 4 other RVA strains were highly divergent from known strains (but very similar among each other) possessing all novel genotypes. Only the VP7 and VP4 genes showed a significant variability representing multiple novel G and P genotypes, indicating the frequent occurrence of reassortment events.Comparing these bat RVA strains with currently used human RVA screening primers indicated that several of the novel VP7 and VP4 segments would not be detected in routine epidemiological screening studies. Therefore, novel VP6 based screening primers matching both human and bat RVAs were developed and used to screen samples from 25 infants with gastroenteritis living in close proximity with the studied bat population. Although RVA infections were identified in 36% of the infants, Sanger sequencing did not indicate evidence of interspecies transmissions.This study identified multiple novel bat RVA strains, but further epidemiological studies in humans will have to assess if these viruses have the potential to cause gastroenteritis in humans.","ABSTRACTThe introduction of electronic patient records in the ambulance service provides new opportunities to monitor the population. Most patients presenting to British ambulance services are discharged at scene. Ambulance records are therefore an ideal data source for syndromic early event detection systems to monitor infectious disease in the prehospital population. It has been previously found that tympanic temperature records can be used to detect influenza outbreaks in emergency departments. This study investigated whether routine tympanic temperature readings collected by ambulance crews can be used to detect seasonal influenza. Here we show that these temperature readings do allow the detection of seasonal influenza before methods applied to conventional data sources. The counts of pyretic patients were used to calculate a sliding case ratio (CR) as a measurement to detect seasonal influenza outbreaks. This method does not rely on conventional thresholds and can be adapted to the data. The data collected correlated with seasonal influenza. The 2016/17 outbreak was detected with high specificity and sensitivity, up to 9 weeks before other surveillance programs. An unanticipated outbreak of E. coli was detected in the same dataset. Our results show that ambulance records can be a useful data source for biosurveillance systems. Two outbreaks caused by different infectious agents have been successfully detected. The routine ambulance records allowed to use tympanic temperature readings that can be used as surveillance tool for febrile diseases. Therefore, this method is a valuable addition to the current surveillance tools.","Scintillators exhibit visible luminescence, called scintillation, when irradiated with X-rays. Given that X-rays penetrate through biological tissues, X-ray-induced scintillation would enable remote optogenetic control of neural functions at any depths in the brain. Here we show that a yellow-emitting inorganic scintillator, Ce-doped Gd3(Al,Ga)5O12 (Ce:GAGG), can effectively activate red-shifted excitatory and inhibitory opsins. Using these scintillator-opsin combinations, we successfully activated and inhibited midbrain dopamine neurons of freely moving mice by X-ray irradiation, producing bi-directional modulation of place preference behavior. The Ce:GAGG crystal was biocompatible and could be implanted for a long period without progressive neuroinflammatory responses. Neither brain injury nor behavioral dysfunction was acutely induced by radiation during the behavioral tests. Thus, X-ray-induced scintillation allows wireless control of cellular functions in living animals, expanding X-ray applications to functional studies of biology and medicine.","AbstractBackground and PurposeImmunosuppression is the predominant cause of mortality for sepsis due to failure to eradicate invading pathogens. Unfortunately, no effective and specific drugs capable of reversing immunosuppression are available for clinical use. Increasing evidence implicates vitamin D receptor (VDR) involved in sepsis-induced immunosuppression. Herein, artesunate (AS) was discovered to reverse sepsis-induced immunosuppression and its molecular mechanism is investigated.Experimental ApproachEffect of artesunate on sepsis-induced immunosuppression was investigated in mice and in vitro. VDR was predicted to be an interacted candidate of AS by bioinformatics predict, then identified using PCR and immunoblotting. VDR, ATG16L1 and NF-\u03baB p65 were modified to investigate the alteration of AS\u2019s effect on pro-inflammatory cytokines release, bacteria clearance and autophagy activities in sepsis-induced immunosuppression.Key ResultsAS significantly reduced the mortality of cecal ligation and puncture (CLP)-induced sepsis immunosuppression mice challenged with Pseudomonas Aeruginosa, and enhanced proinflammatory cytokines release and bacterial clearance to reverse sepsis-induced immunosuppression in vivo and in vitro. Mechanically, AS interacted with VDR thereby inhibited the nuclear translocation of VDR, then influencing ATG16L1 transcription and subsequent autophagy activity. In addition, AS inhibited physical interaction between VDR and NF-\u03baB p65 in LPS tolerance macrophages, then promoted nuclear translocation of NF-\u03baB p65, which activated the transcription of NF-\u03baB p65 target genes such as pro-inflammatory cytokines.Conclusion and ImplicationsOur findings provide an evidence that AS interacted with VDR to reverse sepsis-induced immunosuppression in an autophagy and NF-\u03baB dependent way, highlighting a novel approach for sepsis treatment and drug repurposing of AS in the future.","AbstractObjectiveThe emerging and recent 2014 Ebola Virus Disease (EVD) outbreaks rang the bell to call upon efforts from globe to assist resource-constrained countries to strengthen public health surveillance system for early response. Malawi adopted the Integrated Disease Surveillance and Response (IDSR) strategy to develop its national surveillance system since 2002 and revised its guideline to fulfill the International Health Regulation (IHR) requirements in 2014. This study aimed to understand the state of IDSR implementation and differences between guideline and practice for future disease surveillance system strengthening.MethodsThis was a mixed-method observational study. Quantitative data were to analyze completeness and timeliness of surveillance system performance from national District Health Information System 2 (DHIS2). Qualitative data were collected through interviews with 29 frontline health service providers from the selected district and key informants of the IDSR system implementation and administration at district and national levels.FindingsThe current IDSR system showed relatively good completeness (76.4%) but poor timeliness (41.5%) of total expected monthly reports nationwide and zero weekly reports. The challenges of IDSR implementation revealed through qualitative data included lack of supervision, inadequate resources for training and difficulty to implement weekly report due to overwhelming paperwork at frontline health services.ConclusionsThe differences between IDSR technical guideline and actual practice were huge. The developing information technology infrastructure in Malawi and emerging mobile health (mHealth) technology can be opportunities for the country to overcome these challenges and improve surveillance system to have better timeliness for the outbreaks and unusual events detection.","AbstractA novel coronavirus, SARS-CoV-2, was recently identified in patients with an acute respiratory syndrome, COVID-19. To compare its pathogenesis with that of previously emerging coronaviruses, we inoculated cynomolgus macaques with SARS-CoV-2 or MERS-CoV and compared with historical SARS-CoV infections. In SARS-CoV-2-infected macaques, virus was excreted from nose and throat in absence of clinical signs, and detected in type I and II pneumocytes in foci of diffuse alveolar damage and mucous glands of the nasal cavity. In SARS-CoV-infection, lung lesions were typically more severe, while they were milder in MERS-CoV infection, where virus was detected mainly in type II pneumocytes. These data show that SARS-CoV-2 can cause a COVID-19-like disease, and suggest that the severity of SARS-CoV-2 infection is intermediate between that of SARS-CoV and MERS-CoV.One Sentence SummarySARS-CoV-2 infection in macaques results in COVID-19-like disease with prolonged virus excretion from nose and throat in absence of clinical signs.","AbstractSocial experiences are an important predictor of disease susceptibility and survival in humans and other social mammals. Chronic social stress is thought to generate a pro-inflammatory state characterized by elevated antibacterial defenses and reduced investment in antiviral defense. Here, we manipulated long-term social status in female rhesus macaques to show that social subordination alters the gene expression response to ex vivo bacterial and viral challenge. As predicted by current models, bacterial lipopolysaccharide polarizes the immune response such that low status corresponds to higher expression of genes in NF-\u03baB-dependent pro-inflammatory pathways and lower expression of genes involved in the antiviral response and type I interferon (IFN) signaling. Counter to predictions, however, low status drives more exaggerated expression of both NF-\u03baB and IFN-associated genes after cells are exposed to the viral mimic Gardiquimod. Status-driven gene expression patterns are not only linked to social status at the time of sampling, but also to social history (i.e., past social status), especially in unstimulated cells. However, for a subset of genes, we observed interaction effects in which females who fell in rank were more strongly affected by current social status than those who climbed the social hierarchy. Together, our results indicate that the effects of social status on immune cell gene expression depend on pathogen exposure, pathogen type, and social history \u2013 in support of social experience-mediated biological embedding in adulthood, even in the conventionally memory-less innate immune system.","AbstractThe emergence of a novel, highly pathogenic coronavirus, 2019-nCoV, in China, and its rapid national and international spread pose a global health emergency. Coronaviruses use their spike proteins to select and enter target cells and insights into nCoV-2019 spike (S)-driven entry might facilitate assessment of pandemic potential and reveal therapeutic targets. Here, we demonstrate that 2019-nCoV-S uses the SARS-coronavirus receptor, ACE2, for entry and the cellular protease TMPRSS2 for 2019-nCoV-S priming. A TMPRSS2 inhibitor blocked entry and might constitute a treatment option. Finally, we show that the serum form a convalescent SARS patient neutralized 2019-nCoV-S-driven entry. Our results reveal important commonalities between 2019-nCoV and SARS-coronavirus infection, which might translate into similar transmissibility and disease pathogenesis. Moreover, they identify a target for antiviral intervention.One sentence summaryThe novel 2019 coronavirus and the SARS-coronavirus share central biological properties which can guide risk assessment and intervention.","Motivation: DNA metabarcoding is a commonly applied technique used to infer the species composition of environmental samples. These samples can comprise hundreds of organisms that can be closely or very distantly related in the taxonomic tree of life. DNA metabarcoding combines polymerase chain reaction (PCR) and next-generation sequencing (NGS), whereby a short, homologous sequence of DNA is amplified and sequenced from all members of the community. Sequences are then taxonomically identified based on their match to a reference database. Ideally, each species of interest would have a unique DNA barcode. This short, variable sequence needs to be flanked by relatively conserved regions that can be used as primer binding sites. Appropriate PCR primer pairs would match to a broad evolutionary range of taxa, such that we only need a few to achieve high taxonomic coverage. At the same time however, the DNA barcodes between primer pairs should be different to allow us to distinguish between species to improve resolution. This poses an interesting optimization problem. More specifically: Given a set of references \u211c = {\u211c1, \u211c2, ..., \u211cm}, the problem is to find a primer set P balancing both: high taxonomic coverage and high resolution. This goal can be captured by filtering for frequent primers and ranking by coverage or variation, i.e. the number of unique barcodes. Here we present the software PriSeT, an offline primer discovery tool that is capable of processing large libraries and is robust against mislabeled or low quality references. It tackles the computationally expensive steps with linear runtime filters and efficient encodings. Results: We first evaluated PriSeT on references (mostly 18S rRNA genes) from 19 clades covering eukaryotic organisms that are typical for freshwater plankton samples. PriSeT recovered several published primer sets as well as additional, more chemically suitable primer sets. For these new sets, we compared frequency, taxon coverage, and transcript variation with published primer sets. For 11 clades we found de novo primer pairs that cover more taxa than the published ones, and for six clades de novo primers resulted in greater sequence (i.e., DNA barcode) variation. We also applied PriSeT to 19 SARS-CoV-2 genomes and computed 114 new primer pairs with the additional constraint that the sequences have no co-occurrences in other taxa. These primer sets would be suitable for empirical testing.","SARS-COV-2 has recently emerged as a new public health threat. Herein, we report that the FDA-approved gold drug, auranofin, inhibits SARS-COV-2 replication in human cells at low micro molar concentration. Treatment of cells with auranofin resulted in a 95% reduction in the viral RNA at 48 hours after infection. Auranofin treatment dramatically reduced the expression of SARS-COV-2-induced cytokines in human cells. These data indicate that auranofin could be a useful drug to limit SARS-CoV-2 infection and associated lung injury due to its anti-viral, anti-inflammatory and anti-ROS properties. Auranofin has a well-known toxicity profile and is considered safe for human use.","BackgroundThe ongoing yellow fever (YF) epidemic in Angola strains the global vaccine supply, prompting WHO to adopt dose sparing for its vaccination campaign in Kinshasa in July-August 2016. Although a 5-fold fractional-dose vaccine is similar to standard-dose vaccine in safety and immunogenicity, efficacy is untested. There is an urgent need to ensure the robustness of fractional-dose vaccination by elucidating the conditions under which dose fractionation would reduce transmission.MethodsWe estimate the effective reproductive number for YF in Angola using disease natural history and case report data. With simple mathematical models of YF transmission, we calculate the infection attack rate (IAR, the proportion of population infected over the course of an epidemic) under varying levels of transmissibility and five-fold fractional-dose vaccine efficacy for two vaccination scenarios: (i) random vaccination in a hypothetical population that is completely susceptible; (ii) the Kinshasa vaccination campaign in July-August 2016 with different age cutoff for fractional-dose vaccines.FindingsWe estimate the effective reproductive number early in the Angola outbreak was between 5\u00b72 and 7\u00b71. If vaccine action is all-or-nothing (i.e. a proportion VE of vaccinees receives complete and the remainder receive no protection), n-fold fractionation can dramatically reduce IAR as long as efficacy VE exceeds 1/n. This benefit threshold becomes more stringent if vaccine action is leaky (i.e. the susceptibility of each vaccinee is reduced by a factor that is equal to the vaccine efficacy VE). The age cutoff for fractional-dose vaccines chosen by the WHO for the Kinshasa vaccination campaign (namely, 2 years) provides the largest reduction in IAR if the efficacy of five-fold fractional-dose vaccines exceeds 20%.InterpretationDose fractionation is a very effective strategy for reducing infection attack rate that would be robust with a large margin for error in case fractional-dose VE is lower than expected.FundingNIH-MIDAS, HMRF-Hong Kong","AbstractProteins are dynamic molecules that constitute the molecular machinery of living cells. Single particle cryo-EM excels as an imaging technique to solve static structures of proteins, but existing computational 3D reconstruction methods for cryo-EM data have not been effective in practice for solving structures or modelling motion of flexible proteins. We introduce 3D variability analysis (3DVA), an algorithm that models the conformational landscape of a molecule as a linear subspace, and is able to fit the model to experimental cryo-EM data at high resolution. 3DVA makes it possible, for the first time, to resolve and visualize detailed molecular motions of both large and small proteins. Through extensive experimental results on cryo-EM data, we demonstrate the ability of 3DVA to resolve multiple flexible motions of \u03b1-helices in the sub-50 kDa transmembrane domain of a GPCR complex, bending modes of a sodium ion channel, five types of symmetric and symmetry-breaking flexibility in a proteasome, and large motions in a spliceosome complex. We also show that 3DVA can uncover discrete conformational states of heterogenous mixtures of ribosome particles from a single sample. The results demonstrate that 3DVA enables new biological insight to be extracted from single particle cryo-EM data. 3DVA is implemented in the cryoSPARC software package, and has already been used successfully in several notable structural studies.","AbstractSevere acute respiratory syndrome-related coronavirus 2 (SARS-CoV-2) has resulted in a pandemic and continues to spread around the globe at an unprecedented rate. To date, no effective therapeutic is available to fight its associated disease, COVID-19. Our discovery of a novel insertion of glycosaminoglycan (GAG)-binding motif at S1/S2 proteolytic cleavage site (681-686 (PRRARS)) and two other GAG-binding-like motifs within SARS-CoV-2 spike glycoprotein (SGP) led us to hypothesize that host cell surface GAGs might be involved in host cell entry of SARS-CoV-2. Using a surface plasmon resonance direct binding assay, we found that both monomeric and trimeric SARS-CoV-2 spike more tightly bind to immobilized heparin (KD = 40 pM and 73 pM, respectively) than the SARS-CoV and MERS-CoV SGPs (500 nM and 1 nM, respectively). In competitive binding studies, the IC50 of heparin, tri-sulfated non-anticoagulant heparan sulfate, and non-anticoagulant low molecular weight heparin against SARS-CoV-2 SGP binding to immobilized heparin were 0.056 \u03bcM, 0.12 \u03bcM, and 26.4 \u03bcM, respectively. Finally, unbiased computational ligand docking indicates that heparan sulfate interacts with the GAG-binding motif at the S1/S2 site on each monomer interface in the trimeric SARS-CoV-2 SGP, and at another site (453-459 (YRLFRKS)) when the receptor-binding domain is in an open conformation. Our study augments our knowledge in SARS-CoV-2 pathogenesis and advances carbohydrate-based COVID-19 therapeutic development.","SummaryPorcine reproductive and respiratory syndrome virus (PRRSv) remains widespread in the North American pig population. Despite improvements in virus characterization, it is unclear whether PRRSv infections are a product of viral circulation within a farm, within production systems (local) or across production systems (external). Here we examined the dissemination dynamics of PRRSv and the processes facilitating its spread within and among pig farms in three production systems. Overall, PRRSv genetic diversity declined since 2018, while phylodynamic results support frequent transmission across-production systems. We found that PRRSv dissemination occurred mostly through transmission between farms of different production companies, which were predominant for several months, especially from November until May when PRRSv tends to peak in the studied region. Within production systems, dissemination occurred mainly through regular pig flow, (from sow to nursery and then to finisher farms); nevertheless, an important flux of PRRSv dissemination from finisher to sow and nursery farms highlighted the importance of downstream farms as sources of the virus. High density of farms and farms near major roads were more likely to infect by PRRSv, whereas higher elevation and denser vegetation acted as dissemination barriers. Although it is difficult to disentangle the exact route of PRRSv transmission, results evidenced that dissemination among commercially unrelated pig production systems was intense, reinforcing the importance of farm proximity on PRRSv spread. Thus, consideration of farm location and their geographic characteristics may help to forecast dissemination. The understanding of PRRSv transmission routes has the potential to inform targeted strategies for its prevention and control. Further studies are needed to quantify the relative contribution of PRRSv transmission routes.","The outbreak of the 2019-nCoV coronavirus causing severe acute respiratory syndrome which can be fatal, especially in elderly population, has been declared a pandemicby the World Health Organization. Many biotechnology laboratories are rushing to develop therapeutic antibodies and antiviral drugs for treatment of this viral disease. The viral CoV spike (S) glycoprotein is one of the main targets for pharmacological intervention. Its receptor-binding domain (RBD) interacts with the human ACE2 receptor ensuring the entry of the viral genomes into the host cell. In this work, we report onthe differences in the binding of the RBD of the previous coronavirus SARS-CoV and of the newer 2019-nCoV coronavirus to the human ACE2 receptor using atomistic molecular dynamics techniques. Our results show major mutations in the 2019-nCoV RBD with respect to the SARS-CoV RBD occurring at the interface of RBD-ACE2 complex. These mutations make the 2019-nCoV RBD protein backbone much more flexible, hydrophobic interactions are reduced and additional polar/charged residues appear at the interface. We observe that higher flexibility of the 2019-nCoV RBD with respect to the SARS-CoV RBD leads to a bigger binding interface between the 2019-nCoV RBD and ACE2 and to about 20% more contacts between them in comparison with SARS-CoV. Taken together, the 2019-nCoV RBD shows more stable binding interface and higher binding affinity for the ACE2 receptor. The mutations not only stabilize the binding interface, they also lead to overall more stable 2019-nCoV RBD protein structure, even far from the binding interface. Our results on the molecular differences in the binding between the two viruses can provide important inputs for development of appropriate antiviral treatments of the new viruses, addressing the necessity of ongoing pandemics.","Background: GenBank contains over 3 million viral sequences. The National Center for Biotechnology Information (NCBI) previously made available a tool for validating and annotating influenza virus sequences that is used to check submissions to GenBank. Before this project, there was no analogous tool in use for non-influenza viral sequence submissions. Results: We developed a system called VADR (Viral Annotation DefineR) that validates and annotates viral sequences in GenBank submissions. The annotation system is based on the analysis of the input nucleotide sequence using models built from curated RefSeqs. Hidden Markov models are used to classify sequences by determining the RefSeq they are most similar to, and feature annotation from the RefSeq is mapped based on a nucleotide alignment of the full sequence to a covariance model. Predicted proteins encoded by the sequence are validated with nucleotide-to-protein alignments using BLAST. The system identifies 43 types of \"alerts\" that (unlike the previous BLAST-based system) provide deterministic and rigorous feedback to researchers who submit sequences with unexpected characteristics. VADR has been integrated into GenBank's submission processing pipeline allowing for viral submissions passing all tests to be accepted and annotated automatically, without the need for any human (GenBank indexer) intervention. Unlike the previous submission-checking system, VADR is freely available (https://github.com/nawrockie/vadr) for local installation and use. VADR has been used for Norovirus submissions since May 2018 and for Dengue virus submissions since January 2019. Other viruses with high numbers of submissions will be added incrementally. Conclusion: VADR improves the speed with which non-flu virus submissions to GenBank can be checked and improves the content and quality of the GenBank annotations. The availability and portability of the software allow researchers to run the GenBank checks prior to submitting their viral sequences, and thereby gain confidence that their submissions will be accepted immediately without the need to correspond with GenBank staff. Reciprocally, the adoption of VADR frees GenBank staff to spend more time on services other than checking routine viral sequence submissions.","SummaryHere we propose a vaccination strategy for SARS-CoV-2 based on identification of both highly conserved regions of the virus and newly acquired adaptations that are presented by MHC class I and II across the vast majority of the population, are highly dissimilar from the human proteome, and are predicted B cell epitopes. We present 65 peptide sequences that we expect to result in a safe and effective vaccine which can be rapidly tested in DNA, mRNA, or synthetic peptide constructs. These include epitopes that are contained within evolutionarily divergent regions of the spike protein reported to increase infectivity through increased binding to the ACE2 receptor, and within a novel furin cleavage site thought to increase membrane fusion. This vaccination strategy specifically targets unique vulnerabilities of SARS-CoV-2 and should engage a robust adaptive immune response in the vast majority of the human population.","AbstractFor optimal exploitation of nanoparticles (NPs) in biomedicine, and to predict nanotoxicity, detailed knowledge on the cellular responses to cell-bound or internalized NPs is imperative. The outcome of NP-cell interaction is dictated by the type and magnitude of the NP insult and the cellular response. Here, we have systematically studied the impact of minor differences in NP composition on cellular stress responses and viability by using highly similar poly(alkylcyanoacrylate) (PACA) particles. Surprisingly, PACA particles differing only in their alkyl side chains; butyl (PBCA), ethylbutyl (PEBCA), or octyl (POCA), respectively, induced different stress responses and modes of cell death in human cell lines. POCA particles induced endoplasmic reticulum stress and apoptosis. In contrast, PBCA and PEBCA particles induced lipid peroxidation by depletion of the main cellular antioxidant glutathione (GSH), in a manner depending on the levels of the GSH precursor cystine, and transcription of the cystine transporter SLC7A11 regulated by ATF4 and Nrf2. Intriguingly, these particles activated the recently discovered cell death mechanism ferroptosis, which constitutes a promising alternative for targeting multidrug-resistant cancer stem-like cells. Of the two, PBCA was the strongest inducer. In summary, our findings highlight the cellular sensitivity to nanoparticle composition and have important implications for the choice of PACA monomer in therapeutical settings.","ABSTRACTDuring outbreaks of emerging infections, the lack of effective drugs and vaccines increases reliance on non-pharmacologic public health interventions and behavior change to limit human-to-human transmission. Interventions that increase the speed with which infected individuals remove themselves from the susceptible population are paramount, particularly isolation and hospitalization. Ebola virus disease (EVD), Severe Acute Respiratory Syndrome (SARS), and Middle East Respiratory Syndrome (MERS) are zoonotic viruses that have caused significant recent outbreaks with sustained human-to-human transmission. This investigation quantified changing mean removal rates (MRR) and days from symptom onset to hospitalization (DSOH) of infected individuals from the population in seven different outbreaks of EVD, SARS, and MERS, to test for statistically significant differences in these metrics between outbreaks. We found that epidemic week and viral serial interval were correlated with the speed with which populations developed and maintained health behaviors in each outbreak.","ABSTRACTClassifying the mechanisms of antibiotic failure has led to the development of new treatment strategies for killing bacteria. Among the currently described mechanisms, which include resistance, persistence and tolerance, we propose bacterial defiance as a form of antibiotic failure specific to prodrugs. As a prototypic model of a bacteria-activated prodrug, we construct cationic antimicrobial peptides (AMP), which are charge neutralized until activated by a bacterial protease. This construct successfully eliminated the vast majority of bacteria populations, while localizing activity to bacterial membranes and maintaining low active drug concentration. However, we observed defiant bacteria populations, which survive in the presence of identical drug concentration and exposure time. Using a multi-rate kinetic feedback model, we show that bacteria switch between susceptibility and defiance under clinically relevant environmental (e.g., hyperthermia) and genetic (e.g., downregulated protease expression) conditions. From this model, we derive a dimensionless quantity (Bacterial Advantage Heuristic, BAH) \u2013 representing the balance between bacterial proliferation and prodrug activation \u2013 that perfectly classifies bacteria as defiant or susceptible across a broad range of conditions. To apply this concept to other classes of prodrugs, we expand this model to include both linear and nonlinear terms and use general pharmacokinetic parameters (e.g., half-life, EC50, etc.). Taken together, this model reveals an analogous dimensionless quantity (General Advantage Key, GAK), which can applied to prodrugs with different activation mechanisms. We envision that these studies will enable the development of more effective prodrugs to combat antibiotic resistance.","AbstractThe Arenaviridae is a large family of viruses causing both acute and persistent infections and causing significant public health concerns in afflicted regions. A \u201ctrademark\u201d of infection is the quick and efficient immuno-suppression mediated in part by a 3\u2019-5\u2019 RNA exonuclease domain (ExoN) of the Nucleoprotein (NP). Mopeia virus, the eastern African counterpart of Lassa virus, carries such ExoN domain, but does not suppress the host innate immunity. We have recently reported the crystal structure of the Mopeia virus ExoN domain, which presents a conserved fold and active site. In the present study, we show that the ExoN activity rules out a direct link between ExoN activity and alteration of the host innate immunity. We found that the Arenavirus ExoN, however, is able to excise mis-incorporated bases present at the 3\u2019-end of double stranded RNA. ExoN(-) arenaviruses cultured in cells dampened in innate immunity still replicated in spite of a significant reduction in the viral charge over several passages. The remaining ExoN(-) virus population showed an increased base substitution rate on a narrow nucleotide spectrum, linking the ExoN activity to genome editing. Since, the Arenavirus ExoN belongs to the same nuclease family as that of the nsp14 coronavirus ExoN ; which has been recently shown to promote viral RNA synthesis proofreading; we propose that Arenavirus ExoN is involved in a \u201climited RNA editing\u201d mechanism mainly controlled by structural constraints and a low mutational/fitness ratio.Author summaryOnly Arenaviridae and Coronaviridae encode a 3\u2019-5\u2019 RNA exonuclease domain (ExoN) in their genome. This activity is either used to counteract the innate immunity response during viral infection or to ensure genome stability during replication. Mopeia virus (MOPV), the eastern African counterpart of Lassa virus, carries such ExoN domain, but does not suppress the host innate immunity. We studied MOPV ExoN activity both in vitro and in cellula to assess the role of ExoN MOPV and found that the Arenaviral ExoN is fully active on dsRNA, and is able like the one of Coronaviridae to excise a mismatched base. We measured genetic stability and found evidence of a limited spectrum of RNA synthesis proofreading mechanism, together with a strongly impacted viral replication. We propose that the Arenaviral ExoN is involved in a functional check of the conserved RNA structures of the viral genome.","We have analyzed the cell-type-specific expression of the renin-angiotensin system (RAS) components across 141 cell types or subtypes as defined by single-cell RNA-seq (scRNA-seq) analysis. ACE2, one of the components of RAS, also facilitates SARS-CoV-2 entry into cells in cooperation with its associated protease TMPRSS2. Therefore, our analysis also contributes to the understanding of SARS-CoV-2 infection, spreading of the virus throughout the body, and potential viral interference with RAS in COVID-19 patients.","AbstractThe detection of nucleic acids by pattern recognition receptors is an ancient and conserved component of the innate immune system. Notably, RNA virus genomes are sensed by mammalian cytosolic RIG-I\u2013like receptors, thereby activating interferon-stimulated gene (ISG) expression to restrict viral replication. However, recent evidence indicates that the cGAS-STING DNA sensing pathway also protects against RNA viruses. So far, the mechanisms responsible for DNA sensing of RNA viruses, which replicate without known DNA intermediates, remain unclear. By using cGAS gene knockout and reconstitution in human and mouse cell cultures, we discovered that DNA sensing and cGAMP synthase activities are required for cGAS-mediated restriction of vesicular stomatitis virus and Sindbis virus. The level of cGAMP produced in response to RNA virus infection was below the threshold of detection, suggesting that only transient and/or low levels of cGAMP are produced during RNA virus infections. To clarify the DNA ligands that activate cGAS activity, we confirmed that cGAS binds mitochondrial DNA in the cytosol of both uninfected and infected cells; however, the amount of cGAS-associated mitochondrial DNA did not change in response to virus infection. Rather, a variety of pre-existing cytosolic DNAs, including mitochondrial DNA and endogenous cDNAs, may serve as stimuli for basal cGAS activation. Importantly, cGAS knockout and reconstitution experiments demonstrated that cGAS drives low-level ISG expression at steady state. We propose that cGAS-STING restricts RNA viruses by promoting a preparatory immune activation state within cells, likely primed by endogenous cellular DNA ligands.ImportanceMany medically important RNA viruses are restricted by the cGAS-STING DNA-sensing pathway of innate immune activation. Since these viruses replicate without DNA intermediates, it is unclear what DNA ligand(s) are responsible for triggering this pathway. We show here that cGAS\u2019s DNA binding and signaling activities are required for RNA virus restriction, similar to the mechanisms by which it restricts DNA viruses. Furthermore, we confirmed that cGAS continuously binds host DNA, which was unaffected by RNA virus infection. Finally, cGAS expression correlated with the low-level expression of interferon-stimulated genes in uninfected cells, both in vitro and in vivo. We propose that cGAS-mediated sensing of endogenous DNA ligands contributes to RNA virus restriction by establishing a baseline of innate immune activation.","ABSTRACTHuman coronaviruses OC43 and HKU1 are respiratory pathogen of zoonotic origin that have gained worldwide distribution. OC43 apparently emerged from a bovine coronavirus (BCoV) spill-over. All three viruses attach to 9-O-acetylated sialoglycans via spike protein S with hemagglutinin-esterase HE acting as a receptor-destroying enzyme. In BCoV, an HE lectin domain promotes esterase activity towards clustered substrates. OC43 and HKU1, however, lost HE lectin function as an adaptation to humans. Replaying OC43 evolution, we knocked-out BCoV HE lectin function and performed forced evolution-population dynamics analysis. Loss of HE receptor-binding selected for second-site mutations in S, decreasing S binding affinity by orders of magnitude. Irreversible HE mutations selected for cooperativity in virus swarms with low-affinity S minority variants sustaining propagation of high-affinity majority phenotypes. Salvageable HE mutations induced successive second-site substitutions in both S and HE. Apparently, S and HE are functionally interdependent and co-evolve to optimize the balance between attachment and release. This mechanism of glycan-based receptor usage, entailing a concerted, fine-tuned activity of two envelope protein species, is unique among CoVs, but reminiscent of that of influenza A viruses (IAVs). Apparently, general principles fundamental to virion-sialoglycan interactions prompted convergent evolution of two important groups of human and animal pathogens.","AbstractFor understanding the evolution of social behavior in microbes, mathematical theory can aid empirical research but is often only used as a qualitative heuristic. How to properly formulate social evolution theory has also been contentious. Here we evaluate kin and multilevel selection theory as tools for analyzing microbial data. We reanalyze published datasets that share a common experimental design and evaluate these theories in terms of data visualization, statistical performance, biological interpretation, and quantitative comparison across systems. We find that the canonical formulations of both kin and multilevel selection are almost always poor analytical tools because they use statistical regressions that are poorly specified for the strong selection and nonadditive fitness effects common in microbial systems. Analyzing both individual and group fitness outcomes helps clarify the biology of selection. We also identify analytical practices in empirical research that suggest how theory might better handle the challenges of microbial data. A quantitative, data-driven approach thus shows how kin and multilevel selection theory both have substantial room for improvement as tools for understanding social evolution in all branches of life."],"authors":["Marco R. Straus; Jonathan T. Kinder; Michal Segall; Rebecca Ellis Dutch; Gary R. Whittaker","Zhe Li; Xin Li; Yi-You Huang; Yaoxing Wu; Lingli Zhou; Runduo Liu; Deyan Wu; Lei Zhang; Hao Liu; Ximing Xu; Yuxia Zhang; Jun Cui; Xin Wang; Hai-Bin Luo","Elisa Oberbeckmann; Vanessa Niebauer; Shinya Watanabe; Lucas Farnung; Manuela Moldt; Andrea Schmid; Patrick Cramer; Craig L. Peterson; Sebastian Eustermann; Karl-Peter Hopfner; Philipp Korber","Andr\u00e9s Pizzorno; Blandine Padey; Thomas Julien; Sophie Trouillet-Assant; Aur\u00e9lien Traversier; Elisabeth Errazuriz-Cerda; Julien Fouret; Julia Dubois; Alexandre Gaymard; Fran\u00e7ois-Xavier Lescure; Victoria Duli\u00e8re; Pauline Brun; Samuel Constant; Julien Poissy; Bruno Lina; Yazdan Yazdanpanah; Olivier Terrier; Manuel Rosa-Calatrava","Dora Pinto; Young-Jun Park; Martina Beltramello; Alexandra C. Walls; M. Alejandra Tortorici; Siro Bianchi; Stefano Jaconi; Katja Culap; Fabrizia Zatta; Anna De Marco; Alessia Peter; Barbara Guarino; Roberto Spreafico; Elisabetta Cameroni; James Brett Case; Rita E. Chen; Colin Havenar-Daughton; Gyorgy Snell; Amalio Telenti; Herbert W. Virgin; Antonio Lanzavecchia; Michael S. Diamond; Katja Fink; David Veesler; Davide Corti","Bikash K. Bhandari; Paul P. Gardner; Chun Shen Lim","Mathias Kuhring; Joerg Doellinger; Andreas Nitsche; Thilo Muth; Bernhard Y. Renard","Xiong Ding; Kun Yin; Ziyue Li; Changchun Liu","Yujia Xiang; Quan Zou; Lilin Zhao","No\u00e9mie Aurine; Camille Baquerre; Maria Gaudino; Christian Jean; Claire Dumont; Sylvie Rival-Gervier; Cl\u00e9mence Kress; Branka Horvat; Bertrand Pain","Lele Zhao; Mansha Seth Pasricha; Dragos Stemate; Alvin Crespo-Bellido; Jacqueline Gagnon; Siobain Duffy","Martina F Scallan; Catherine Dempsey; John MacSharry; Isabelle O'Callaghan; Paula M O'Connor; Conor P Hogan; Edel Durack; Paul D. Cotter; Sarah Hudson; Humphrey A Moynihan; Brigid Lucey","David Enard; Dmitri A. Petrov","Olga Kurskaya; Tatyana Ryabichenko; Natalya Leonova; Weifeng Shi; Hongtao Bi; Kirill Sharshov; Eugenia Kazachkova; Ivan Sobolev; Elena Prokopyeva; Tatyana Kartseva; Alexander Alekseev; Alexander Shestopalov","Jethro Herberg; Honglei Huang; Marie L. Thezenas; Victoria Janes; Michael Carter; Stuart Gormley; Melisa S. Hamilton; Benedikt Kessler; Michael Levin; Climent Casals-Pascual","Cheng Wang; Shaobo Wang; Daixi Li; Xia Zhao; Songling Han; Tao Wang; Gaomei Zhao; Yin Chen; Fang Chen; Jianqi Zhao; Liting Wang; Wei Sun; Yi Huang; Yongping Su; Dongqing Wei; Jinghong Zhao; Junping Wang","Elizabeth F. Maughan; Ersilia Nigro; Adam Pennycuick; Kate H.C. Gowers; Celine Denais; Sandra G\u00f3mez-L\u00f3pez; Kyren A. Lazarus; Colin R. Butler; Dani Do Hyang Lee; Jessica C. Orr; Vitor Hugo Teixeira; Benjamin E. Hartley; Richard J. Hewitt; Chadwan Al Yaghchi; Gurpreet S. Sandhu; Martin A. Birchall; Christopher O'Callaghan; Claire M. Smith; Paolo De Coppi; Robert E. Hynds; Sam M. Janes","Humberto J Debat","Paola Giusti-Rodr\u00edguez; James G. Xenakis; James J. Crowley; Randal J. Nonneman; Daniela M. DeCristo; Allison Ryan; Corey R. Quackenbush; Darla R. Miller; Ginger D. Shaw; Vasyl Zhabotynsky; Patrick F. Sullivan; Fernando Pardo-Manuel de Villena; Fei Zou","Kariem Ezzat; Maria Pernemalm; Sandra P\u00e5lsson; Thomas C. Roberts; Peter J\u00e4rver; Aleksandra Dondalska; Burcu Bestas; Michal J. Sobkowiak; Bettina Lev\u00e4nen; Magnus Sk\u00f6ld; Elizabeth A. Thompson; Osama Saher; Otto K. Kari; Tatu Lajunen; Eva Sverremark Ekstr\u00f6m; Caroline Nilsson; Yevheniia Ishchenko; Tarja Malm; Matthew J.A. Wood; Ultan F. Power; Sergej Masich; Anders Lind\u00e9n; Johan K. Sandberg; Janne Lehti\u00f6; Anna-Lena Spetz; Samir EL Andaloussi","Rebecca Kahn; Matt Hitchings; Rui Wang; Steven Bellan; Marc Lipsitch","Minji Park; Yong-Joon Cho; Donggyu Kim; Chul-Su Yang; Shi Mun Lee; Thomas L. Dawson; Satoshi Nakamizo; Kenji Kabashima; Yang Won Lee; Won Hee Jung","Lihong Zhan; Peter Dongmin Sohn; Yungui Zhou; Yaqiao Li; Li Gan","Aleksandra Milewska; Katherine Falkowski; Magdalena Kalinska; Ewa Bielecka; Antonina Naskalska; Pawel Mak; Adam Lesner; Marek Ochman; Maciej Urlik; Jan Potempa; Tomasz Kantyka; Krzysztof Pyrc","Abraham G. Beyene; Kristen Delevich; Jackson Travis Del Bonis-O\u2019Donnell; David J. Piekarski; Wan Chen Lin; A. Wren Thomas; Sarah J. Yang; Polina Kosillo; Darwin Yang; Linda Wilbrecht; Markita P. Landry","Ekta Gupta; Rupesh Kumar Mishra; Ravi Ranjan Kumar Niraj","Carly Adams; David Young; Paul A Gasta\u00f1aduy; Prabasaj Paul; Zach Marsh; Aron J Hall; Benjamin A Lopman","Lucile Broseus; Aubin Thomas; Andrew J. Oldfield; Dany Severac; Emeric Dubois; William Ritchie","David Brann; Tatsuya Tsukahara; Caleb Weinreb; Darren W. Logan; Sandeep Robert Datta","Zhengqiao Zhao; Bahrad A. Sokhansanj; Gail L. Rosen","Corey T Watson; Karyn Meltz Steinberg; Tina A Graves-Lindsay; Rene L Warren; Maika Malig; Jacqueline E Schein; Richard K Wilson; Rob Holt; Evan Eichler; Felix Breden","Abhishek Singh; Mukesh Thakur; Lalit Kumar Sharma; Kailash Chandra","Xavier Hernandez-Alias; Martin Schaefer; Luis Serrano","Pei-Hui Wang; Yun Cheng","Jiasu Xu; Jin Wang; Zecheng Zhong; Xiaosong Su; Kunyu Yang; Zhongfu Chen; Dongxu Zhang; Tingdong Li; Yingbin Wang; Shiyin Zhang; Shengxiang Ge; Jun Zhang; Ningshao Xia","Massimiliano S Tagliamonte; Nabil Adid; Giovanni Chillemi; Marco Salemi; Carla N Mavian","Kentaro Itokawa; Tsuyoshi Sekizuka; Masanori Hashino; Rina Tanaka; Makoto Kuroda","Sk Shahid Nadim; Indrajit Ghosh; Joydev Chattopadhyay","Jing Qi; Yang Zhou; Jiao Hua; Liying Zhang; Jialin Bian; Beibei Liu; Zicen Zhao; Shuilin Jin","Femke Maij; Christian Seegelke; W Pieter Medendorp; Tobias Heed","Yuan-Lin Kang; Yi-Ying Chou; Paul W Rothlauf; Zhuoming Liu; Silvia Piccinotti; Timothy K Soh; David Cureton; James Brett Case; Rita E Chen; Michael S Diamond; Sean P.J. Whelan; Tom Kirchhausen","Robert J. H. Ross; R. E. Baker; C. A. Yates","Arunachalam Ramaiah; Deisy Contreras; Vineela Gangalapudi; Masumi Sameer Padhye; Jie Tang; Vaithilingaraja Arumugaswami","Kinza Rian; Marina Esteban-Medina; Marta R Hidalgo; Cankut Cubuk; Matias M Falco; Carlos Loucera; Devrim Gunyel; Marek Ostaszewski; Maria Pena-Chilet; Joaquin Dopazo","Jingyue Ju; Xiaoxu Li; Shiv Kumar; Steffen Jockusch; Minchen Chien; Chuanjuan Tao; Irina Morozova; Sergey Kalachikov; Robert N. Kirchdoerfer; James J. Russo","Zhiqiang Zheng; Vanessa M. Monteil; Sebastian Maurer-Stroh; Chow Wenn Yew; Carol Leong; Nur Khairiah Mohd-Ismail; Suganya Cheyyatraivendran Arularasu; Vincent Tak Kwong Chow; Raymond Lin Tzer Pin; Ali Mirazimi; Wanjin Hong; Yee-Joo Tan","Katie M. Campbell; Gabriela Steiner; Daniel K. Wells; Antoni Ribas; Anusha Kalbasi","Wai-Kit Ming; Jian Huang; Casper J. P. Zhang","Jingyue Ju; Shiv Kumar; Xiaoxu Li; Steffen Jockusch; James J. Russo","Stephanie Pfaender; Katrina B. Mar; Eleftherios Michailidis; Annika Kratzel; Dagny Hirt; Philip V\u2019kovski; Wenchun Fan; Nadine Ebert; Hanspeter Stalder; Hannah Kleine-Weber; Markus Hoffmann; H. Heinrich Hoffmann; Mohsan Saeed; Ronald Dijkman; Eike Steinmann; Mary Wight-Carter; Natasha W. Hanners; Stefan P\u00f6hlmann; Tom Gallagher; Daniel Todt; Gert Zimmer; Charles M. Rice; John W. Schoggins; Volker Thiel","Alistair Martin; Jama Nateqi; Stefanie Gruarin; Nicolas Munsch; Isselmou Abdarahmane; Bernhard Knapp","Andrew J. McNamara; Pranav Danthi","Xiaoyang Ji; Chunming Zhang; Yubo Zhai; Zhonghai Zhang; Chunli Zhang; Yiqing Xue; Guangming Tan; Gang Niu","Nicola Clementi; Elena Criscuolo; Roberta Antonia Diotti; Roberto Ferrarese; Matteo Castelli; Roberto Burioni; Massimo Clementi; Nicasio Mancini","Jacob W. Myerson; Priyal N. Patel; Nahal Habibi; Landis R. Walsh; Yi-Wei Lee; David C. Luther; Laura T. Ferguson; Michael H. Zaleski; Marco E. Zamora; Oscar A. Marcos-Contreras; Patrick M. Glassman; Ian Johnston; Elizabeth D. Hood; Tea Shuvaeva; Jason V. Gregory; Raisa Y. Kiseleva; Jia Nong; Kathryn M. Rubey; Colin F. Greineder; Samir Mitragotri; George S. Worthen; Vincent M. Rotello; Joerg Lahann; Vladimir R. Muzykantov; Jacob S. Brenner","Jakub M Bartoszewicz; Anja Seidel; Bernhard Y Renard","Yong Zhang; Wanjun Zhao; Yonghong Mao; Shisheng Wang; Yi Zhong; Tao Su; Meng Gong; Xiaofeng Lu; Jingqiu Cheng; Hao Yang","Samuel Minot; Stephen D Turner; Krista L Ternus; Dana R Kadavy","Qingxin Zhang; Qingshun Zhao","Justina Jankauskait\u0117; Brian Jim\u00e9nez-Garc\u00eda; Justas Dapk\u016bnas; Juan Fern\u00e1ndez-Recio; Iain H. Moal","Aran Singanayagam; Joseph Footitt; Benjamin T Kasdorf; Matthias Marczynski; Michael T Cross; Lydia J Finney; Maria-Belen Trujillo Torralbo; Maria Calderazzo; Jie Zhu; Julia Aniscenko; Thomas B Clarke; Philip L Molyneaux; Nathan W Bartlett; Miriam F Moffatt; William O Cookson; Jadwiga Wedzicha; Christopher M Evans; Oliver Lieleg; Patrick Mallia; Sebastian L Johnston","Ashutosh Kumar; Muneeb A. Faiq; Vikas Pareek; Khursheed Raza; Ravi K. Narayan; Pranav Prasoon; Pavan Kumar; Maheswari Kulandhasamy; Chiman Kumari; Kamla Kant; Himanshu N. Singh; Rizwana Qadri; Sada N. Pandey; Santosh Kumar","Matthew J. Brownsword; Nicole Doyle; Mich\u00e8le Brocard; Nicolas Locker; Helena J. Maier","Mohammad Saeed; Alejandro Ib\u00e1\u00f1ez-Costa; Alejandra Mar\u00eda Pati\u00f1o-Trives; Eduardo Collantes Estevez; Mar\u00eda \u00c1ngeles Aguirre; Chary L\u00f3pez-Pedrera","Lars S Jermiin; Renee A Catullo; Barbara R Holland","Marek Prachar; Sune Justesen; Daniel B Steen-Jensen; Stephan P Thorgrimsen; Erik Jurgons; Ole Winther; Frederik Otzen Bagger","Nelly Mostajo Berrospi; Marie Lataretu; Sebastian Krautwurst; Florian Mock; Daniel Desir\u00f2; Kevin Lamkiewicz; Maximilian Collatz; Andreas Schoen; Friedemann Weber; Manja Marz; Martin H\u00f6lzer","Shi Zhao; Qianyin Lin; Jinjun Ran; Salihu S Musa; Guangpu Yang; Weiming Wang; Yijun Lou; Daozhou Gao; Lin Yang; Daihai He; Maggie H Wang","Ali Gibran; Runzhen Zhao; Mo Zhang; Krishan G. Jain; Jianjun Chang; Satoshi Komatsu; Xiaohui Fang; Beiyun Zhou; Jiurong Liang; Dianhua Jiang; Mistuo Ikebe; Michael A Matthay; Hong-Long Ji","Xiaojun Li; Elena E. Giorgi; Manukumar Honnayakanahalli Marichann; Brian Foley; Chuan Xiao; Xiang-peng Kong; Yue Chen; Bette Korber; Feng Gao","M. Gordon Joyce; Rajeshwer S. Sankhala; Wei-Hung Chen; Misook Choe; Hongjun Bai; Agnes Hajduczki; Lianying Yan; Spencer L. Sterling; Caroline E. Peterson; Ethan C. Green; Clayton Smith; Natalia de Val; Mihret Amare; Paul Scott; Eric D. Laing; Christopher C. Broder; Morgane Rolland; Nelson L. Michael; Kayvon Modjarrad","Hongyu Yuan; Lizhen Ma; Lijie Zhang; Xiaoying Li; Chun Xia","Andres S. Espindola; William Schneider; Kitty F. Cardwell; Yisel Carrillo; Peter R. Hoyt; Stephen M. Marek; Hassan Melouk; Carla D. Garzon","Mingwang Shen; Zhihang Peng; Yanni Xiao; Lei Zhang","Intikhab alam; Allan Kamau; Maxat Kulmanov; Stefan T. Arold; Arnab Pain; Takashi Gojobori; Carlos M. Duarte","Andrew Rambaut; Edward C Holmes; Verity Hill; Aine OToole; John McCrone; Chris Ruis; Louis du Plessis; Oliver Pybus","L\u00e9a Joffrin; Steven M. Goodman; David A. Wilkinson; Beza Ramasindrazana; Erwan Lagadec; Yann Gomard; Gildas Le Minter; Andr\u00e9a Dos Santos; M. Corrie Schoeman; Rajendraprasad Sookhareea; Pablo Tortosa; Simon Julienne; Eduardo S. Gudo; Patrick Mavingui; Camille Lebarbenchon","Gregory F Albery; Evan A Eskew; Noam Ross; Kevin J Olival","Dorothea Bestle; Miriam Ruth Heindl; Hannah Limburg; Thuy Van Lam van; Oliver Pilgram; Hong Moulton; David A. Stein; Kornelia Hardes; Markus Eickmann; Olga Dolnik; Cornelius Rohde; Stephan Becker; Hans-Dieter Klenk; Wolfgang Garten; Torsten Steinmetzer; Eva B\u00f6ttcher-Friebertsh\u00e4user","Changhai Lei; Wenyan Fu; Kewen Qian; Tian Li; Sheng Zhang; Min Ding; Shi Hu","V. Chaudhary; N. Rajput; A. B. Shrivastav; D. W. Tonkyn","Shu-Miaw Chaw; Jui-Hung Tai; Shi-Lun Chen; Chia-Hung Hsieh; Sui-Yuan Chang; Shiou-Hwei Yeh; Wei-Shiung Yang; Pei-Jer Chen; Hurng-Yi Wang","Chunlong Ma; Brett Hurst; Yanmei Hu; Tommy Szeto; Bart Tarbet; Jun Wang","Shen Lin; Runnan Shen; Jingdong He; Xinhao Li; Xushun Guo","Anastassia K. Pogoutse; Trevor F. Moraes","Saymon Akther; Edgaras Bezrucenkovas; Brian Sulkow; Christopher Panlasigui; Li Li; Weigang Qiu; Lia Di","Myra Hosmillo; Jia Lu; Michael R. McAllaster; James B. Eaglesham; Xinjie Wang; Edward Emmott; Patricia Domingues; Yasmin Chaudhry; Timothy J Fitzmaurice; Matthew K.H. Tung; Marc Panas; Gerald McInerney; Nicholas Locker; Craig B. Willen; Ian Goodfellow","Rustom Antia; Hasan Ahmed; James J Bull","Steven Henikoff; Jorja G. Henikoff","Jonathan Dushoff; Sang Woo Park","CV Herst; S Burkholz; J Sidney; A Sette; PE Harris; S Massey; T Brasel; E Cunha-Neto; DS Rosa; WCH Chao; R Carback; T Hodge; L Wang; S Ciotlos; P Lloyd; R Rubsamen","Ramya Rangan; Ivan N. Zheludev; Rhiju Das","Jun Wu; Dingyun Song; Zhongwen Li; Baojie Guo; Yani Xiao; Wenjing Liu; Lingmin Liang; Chunjing Feng; Tingting Gao; Zai Wang; Jianyan Wen; Shengnan Yang; Peipei Liu; Lei Wang; Yukai Wang; Liang Peng; Guihai Feng; Shiqiang Huang; Glyn Nigel Stacey; Wei Li; Yan Huo; Ronghua Jin; Qi Zhou; Liu Wang; Baoyang Hu; Huaping Dai; Jie Hao","Tam L. Nguyen; Carrie E. Perlman","Shuaiyao Lu; Yuan Zhao; Wenhai Yu; Yun Yang; Jiahong Gao; Junbin Wang; Dexuan Kuang; Mengli Yang; Jing Yang; Chunxia Ma; Jingwen Xu; Haiyan Li; Siwen Zhao; Jingmei Li; Haixuan Wang; Haiting Long; Jingxian Zhou; Fangyu Luo; Kaiyun Ding; Daoju Wu; Yong Zhang; Yingliang Dong; Yuqin Liu; Yingqiu Zheng; Xiaochen Lin; Li Jiao; Huanying Zheng; Qing Dai; Qiangmin Sun; Yunzhang Hu; Changwen Ke; Hongqi Liu; Xiaozhong Peng","Lisa Caroline Crossman","Kuiama Lewandowski; Yifei Xu; Steven T. Pullan; Sheila F. Lumley; Dona Foster; Nicholas Sanderson; Alison Vaughan; Marcus Morgan; Nicole Bright; James Kavanagh; Richard Vipond; Miles Carroll; Anthony C. Marriott; Karen E Gooch; Monique Andersson; Katie Jeffery; Timothy EA Peto; Derrick W. Crook; A Sarah Walker; Philippa C. Matthews","Arinjay Banerjee; Jalees A. Nasir; Patrick Budylowski; Lily Yip; Patryk Aftanas; Natasha Christie; Ayoob Ghalami; Kaushal Baid; Amogelang R. Raphenya; Jeremy A Hirota; Matthew S. Miller; Allison J McGeer; Mario A Ostrowski; Robert A. Kozak; Andrew G McArthur; Karen Mossman; Samira Mubareka","M\u00e9lanie Legrand; Sophie Bachellier-Bassi; Keunsook K. Lee; Yogesh Chaudhari; H\u00e9l\u00e8ne Tournu; Laurence Arbogast; H\u00e9l\u00e8ne Boyer; Murielle Chauvel; Vitor Cabral; Corinne Maufrais; Audrey Nesseir; Irena Maslanka; Emmanuelle Permal; Tristan Rossignol; Louise A. Walker; Ute Zeidler; Sadri Znaidi; Floris Schoeters; Charlotte Majgier; Renaud A. Julien; Laurence Ma; Magali Tichit; Christiane Bouchier; Patrick Van Dijck; Carol A. Munro; Christophe d\u2019Enfert","Cecylia S. Lupala; Xuanxuan Li; Jian Lei; Hong Chen; Jianxun Qi; Haiguang Liu; Xiao-dong Su","Claire Meissner-Bernard; Yulia Dembitskaya; Laurent Venance; Alexander Fleischmann","Jeanne M. Sisk; Matthew B. Frieman; Carolyn E. Machamer","Said Mougari; Nisrine Chelkha; Dehia Sahmi-Bounsiar; Fabrizio Di Pinto; Philippe Colson; Jonatas Abrahao; Bernard La Scola","Wu Liu; Mehmet U. Caglar; Zhangming Mao; Andrew Woodman; Jamie J. Arnold; Claus O. Wilke; Craig E. Cameron","Xiaojing Chi; Xiuying Liu; Conghui Wang; Xinhui Zhang; Lili Ren; Qi Jin; Jianwei Wang; Wei Yang","Monika Abedin Sigg; Tabea Menchen; Jeffery Johnson; Chanjae Lee; Semil P. Choksi; Galo Garcia; Henriette Busengdal; Gerard Dougherty; Petra Pennekamp; Claudius Werner; Fabian Rentzsch; Nevan Krogan; John B. Wallingford; Heymut Omran; Jeremy F. Reiter","Arunkumar Arumugam; Season S Wong","Vincent Navratil; Lo\u00efc Lionnard; Sonia Longhi; Christophe Combet; Abdel Aouacheria","Furong Qi; Shen Qian; Shuye Zhang; Zheng Zhang","Jason W. Westerbeck; Carolyn E. Machamer","Wanda J. Lyon; Zachary K. Smith; Brian Grier; James Baldwin; Clarise R. Starr","Haogao Gu; Daniel Chu; Malik Peiris; Leo L.M. Poon","Santrupti Nerli; Nikolaos G Sgourakis","Aran Singanayagam; Su-Ling Loo; Maria Calderazzo; Lydia J Finney; Maria-Belen Trujillo Torralbo; Eteri Bakhsoliani; Jason Girkin; Punnam Veerati; Prabuddha S Pathinayake; Kristy S Nichol; Andrew Reid; Joseph Foottit; Sebastian L Johnston; Nathan W Bartlett; Patrick Mallia","Aditi Joshi; Sushmita Paul","Alisha Chitrakar; Sneha Rath; Jesse Donovan; Kaitlin Demarest; Yize Li; Raghavendra Rao Sridhar; Susan R. Weiss; Sergei V. Kotenko; Ned S. Wingreen; Alexei Korennykh","Phuoc Truong; Santiago Garcia-Vallve; Pere Puigbo","Lu Guo; Xuehan Sun; Xinge Wang; Chen Liang; Haiping Jiang; Qingqin Gao; Moyu Dai; Bin Qu; Sen Fang; Yihuan Mao; Yangcan Chen; Guihai Feng; Qi Gu; Liu Wang; Ruiqi Rachel Wang; Qi Zhou; Wei Li","Sen Mei; Chong Xie; Hao Mi; Chuang Xue; Qiang Guo; Guang-Qing Du; Guo-Bei Li; Cai-Xia Li; Ya-Nan Qu; Ming-Hao Xiong; Yang Jiang; Tian-Wei Tan; Shang-Tian Yang; Li-Hai Fan","Hong Zhou; Xing Chen; Tao Hu; Juan Li; Hao Song; Yanran Liu; Peihan Wang; Di Liu; Jing Yang; Edward C. Holmes; Alice C. Hughes; Yuhai Bi; Weifeng Shi","Serge Morand; Bruno Andreas Walther","Samantha N. Davis; Patricia Wu; Esra D. Camci; Julian A. Simon; Edwin W Rubel; David W. Raible","Adam M. Dinan; Sarah Keep; Erica Bickerton; Paul Britton; Andrew E. Firth; Ian Brierley","A. Bal; M. Pichon; C. Picard; JS. Casalegno; M. Valette; I. Schuffenecker; L. Billard; S. Vallet; G. Vilchez; V. Cheynet; G. Oriol; S. Assant; Y. Gillet; B. Lina; K. Brengel-Pesce; F. Morfin; L. Josset","Serena H. Chen; M. Todd Young; John Gounley; Christopher Stanley; Debsindhu Bhowmik","Paul A. Rowley; Brandon Ho; Sarah Bushong; Arlen Johnson; Sara L. Sawyer","Salvatore Di Giorgio; Filippo Martignano; Maria Gabriella Torcia; Giorgio Mattiuz; Silvestro G. Conticello","Khader Shameer; Kipp W. Johnson; Ben Readhead; Benjamin S. Glicksberg; Claire McCallum; Amjesh Revikumar; Jamie S. Hirsch; Kevin Bock; John Chelico; Negin Hajizadeh; Michael Oppenheim; Joel T. Dudley","Esther S. Brielle; Dina Schneidman-Duhovny; Michal Linial","Harvey B Pollard; Bette S. Pollard; John R. Pollard","John R. Shorter; Paul L. Maurizio; Timothy A. Bell; Ginger D. Shaw; Darla R. Miller; Terry J. Gooch; Jason S. Spence; Leonard McMillan; William Valdar; Fernando Pardo-Manuel de Villena","Hang Xiao; Xin-Tian Nie; Xiao-Xia Ji; Shu-ping Yan; Bin Zhu; Yuan-Shu Zhang","Megan C. Cohan; Ammon E. Posey; Steven J. Grigsby; Anuradha Mittal; Alex S. Holehouse; Paul J. Buske; Petra A. Levin; Rohit V. Pappu","Prerana Shrestha; Pinar Ayata; Pedro Herrero-Vidal; Francesco Longo; Alexandra Gastone; Joseph E. Ledoux; Nathaniel Heintz; Eric Klann","James T. Van Leuven; Martina M. Ederer; Katelyn Burleigh; LuAnn Scott; Randall A. Hughes; Vlad Codrea; Andrew D. Ellington; Holly Wichman; Craig Miller","Andra Waagmeester; Egon L. Willighagen; Andrew I. Su; Martina Kutmon; Jose Emilio Labra Gayo; Daniel Fern\u00e1ndez-\u00c1lvarez; Peter J. Schaap; Lisa M. Verhagen; Jasper J. Koehorst","Youngchang Kim; Robert Jedrzejczak; Natalia I. Maltseva; Michael Endres; Adam Godzik; Karolina Michalska; Andrzej Joachimiak","Nils C. Gassen; Jan Papies; Thomas Bajaj; Frederik Dethloff; Jackson Emanuel; Katja Weckmann; Daniel E. Heinz; Nicolas Heinemann; Martina Lennarz; Anja Richter; Daniela Niemeyer; Victor M. Corman; Patrick Giavalisco; Christian Drosten; Marcel A. M\u00fcller","Dhurvas Chandrasekaran Dinesh; Dominika Chalupska; Jan Silhan; Vaclav Veverka; Evzen Boura","Fan Wu; Su Zhao; Bin Yu; Yan-Mei Chen; Wen Wang; Yi Hu; Zhi-Gang Song; Zhao-Wu Tao; Jun-Hua Tian; Yuan-Yuan Pei; Ming-Li Yuan; Yu-Ling Zhang; Fa-Hui Dai; Yi Liu; Qi-Min Wang; Jiao-Jiao Zheng; Lin Xu; Edward C. Holmes; Yong-Zhen Zhang","Han Suh Kang; Katelyn McNair; Daniel A. Cuevas; Barbara A. Bailey; Anca M. Segall; Robert A. Edwards","Nahui Olin Medina-Ch\u00e1vez; Mariette Viladomat-Jasso; Gabriela Olmedo-\u00c1lvarez; Luis E Eguiarte; Valeria Souza; Susana De la Torre-Zavala","Muhammet Uslupehlivan; Ecem Sener","Moritz U. G. Kraemer; T. Alex Perkins; Derek A.T. Cummings; Rubeena Zakar; Simon I. Hay; David L. Smith; Robert C. Reiner","Dima Kagan; Jacob Moran-Gilad; Michael Fire","Wanchao Yin; Chunyou Mao; Xiaodong Luan; Dan-Dan Shen; Qingya Shen; Haixia Su; Xiaoxi Wang; Fulai Zhou; Wenfeng Zhao; Minqi Gao; Shenghai Chang; Yuan-Chao Xie; Guanghui Tian; He-Wei Jiang; Sheng-Ce Tao; Jingshan Shen; Yi Jiang; Hualiang Jiang; Yechun Xu; Shuyang Zhang; Yan Zhang; H. Eric Xu","Amer Elias; Itay Spector; Natasha Gritsenko; Yael Zilberstein; Rena Gorovits; Gali Prag; Mikhail Kolot","William T Gibson; Daniel M Evans; Jianghong An; Steven JM Jones","Houcemeddine Othman; Zied Bouslama; Jean-Tristan Brandenburg; Jorge da Rocha; Yosr Hamdi; Kais Ghedira; Najet-Srairi Abid; Scott Hazelhurst","Zhen Zhao; Haodong Cui; Wenxing Song; Xiaoling Ru; Wenhua Zhou; Xuefeng Yu","Rahila Sardar; Deepshikha Satish; Shweta Birla; Dinesh Gupta","Chenglong Xiong; Lufang Jiang; Yue Chen; Qingwu Jiang","Jiajie Jessica Xu; Zuha Yousuf; Zhonghua Ouyang; Eric Kennedy; Patrick A. Lester; Tara Martin; Tim M. Bruns","Saahir Khan; Rie Nakajima; Aarti Jain; Rafael Ramiro de Assis; Al Jasinskas; Joshua M. Obiero; Oluwasanmi Adenaiye; Sheldon Tai; Filbert Hong; Donald K. Milton; Huw Davies; Philip L. Felgner","Vineet D. Menachery; Lisa E. Gralinski; Hugh D. Mitchell; Kenneth H. Dinnon; Sarah R. Leist; Boyd L. Yount; Rachel L. Graham; Eileen T. McAnarney; Kelly G. Stratton; Adam S. Cockrell; Kari Debbink; Amy C. Sims; Katrina M. Waters; Ralph S. Baric","Rafael R. de Assis; Aarti Jain; Rie Nakajima; Algis Jasinskas; Jiin Felgner; Joshua M. Obiero; Oluwasanmi Adenaiye; Sheldon Tai; Filbert Hong; Philip Norris; Mars Stone; Graham Simmons; Anil Bagri; Martin Schreiber; Andreas Buser; Andreas Holbro; Manuel Battegay; Donald K. Milton; Huw Davies; Laurence M. Corash; Michael P. Busch; Philip L. Felgner; Saahir Khan","Meng Yuan; Nicholas C. Wu; Xueyong Zhu; Chang-Chun D. Lee; Ray T. Y. So; Huibin Lv; Chris K. P. Mok; Ian A. Wilson","Andrea Torneri; Amin Azmon; Christel Faes; Eben Kenah; Gianpaolo Scalia Tomba; Jacco Wallinga; Niel Hens","Brent De Wijngaert; Shemaila Sultana; Chhaya Dharia; Hans Vanbuel; Jiayu Shen; Daniel vasilchuk; Sergio E Martinez; Eaazhisai Kandiah; Smita S Patel; Kalyan Das","Charles L Howe; Reghann G. LaFrance-Corey; Emma N Goddery; Kanish Mirchia","Ana Requena-M\u00e9ndez; Joaquin Salas-Coronas; Fernando Salvador; Joan Gomez-Junyent; Judith Villar-Garcia; Miguel Santin; Carme Mu\u00f1oz; Ana Gonzalez-Cordon; Maria Teresa Cabezas Fern\u00e1ndez; Elena Sulleiro; Maria del Mar Arenas; Dolors Somoza; Jose Vazquez-Villegas; Bego\u00f1a Trevi\u00f1o; Esperanza Rodr\u00edguez; Maria Eugenia Valls; Carme Subir\u00e1; Jose Mu\u00f1oz","Gloria P. Larson; Vy Tran; Shu\u01d0q\u00ecng Y\u00fa; Y\u00edngy\u00fan Ca\u00ec; Christina A. Higgins; Danielle M. Smith; Steven F. Baker; Sheli R. Radoshitzky; Jens H. Kuhn; Andrew Mehle","Alexander Rosenberg; Iuliana V. Ene; Maayan Bibi; Shiri Zakin; Ella Shtifman Segal; Naomi Ziv; Alon M. Dahan; Arnaldo L. Colombo; Richard J. Bennett; Judith Berman","Daniel R. Monaco; Sanjay V. Kottapalli; Tiezheng Yuan; Florian P. Breitwieser; Danielle E. Anderson; Limin Wijaya; Kevin Tan; Wan Ni Chia; Kai Kammers; Mario Caturegli; Kathleen Waugh; Marian Rewers; Lin-Fa Wang; H. Benjamin Larman","Ramya Rangan; Andrew M. Watkins; Wipapat Kladwang; Rhiju Das","Hazel Stewart; Katherine Brown; Adam M. Dinan; Nerea Irigoyen; Eric J. Snijder; Andrew E. Firth","Anniina Vihervaara; Dig Bijay Mahat; Samu V. Himanen; Malin A.H. Blom; John T. Lis; Lea Sistonen","Salik Hussain; Collin G Johnson; Joseph Sciurba; Xianglin Meng; Vandy P Stober; Caini Liu; Jaime M Cyphert-Daly; Katarzyna Bulek; Wen Qian; Alma Solis; Yosuke Sakamachi; Carol S Trempus; Jim J Aloor; Kym M Gowdy; W. Michael Foster; John W Hollingsworth; Robert M Tighe; Xiaoxia Li; Michael B Fessler; Stavros Garantziotis","Katherine E. Arden; Ristan M. Greer; Claire Y.T. Wang; Ian M. Mackay","Matteo Chiara; David S. Horner; Graziano Pesole","Siew-Kim Khoo; James Read; Kimberley Franks; Guicheng Zhang; Joelene Bizzintino; Laura Coleman; Christopher McCrae; Lisa \u00d6berg; Niamh Troy; Franciska Prastanti; Janet Everard; Stephen Oo; Meredith L Borland; Rose A Maciewicz; Peter N Le Sou\u00ebf; Ingrid A Laing; Anthony Bosco","Mohammad Rubayet Hasan; Hassan Al Mana; Virginia Young; Patrick Tang; Eva Thomas; Rusung Tan; Peter Tilley","Toshio Kamiya; Takashi Masuko; Dasiel Oscar Borroto-Escuela; Haruo Okado; Hiroyasu Nakata","Yuri I. Wolf; Darius Kazlauskas; Jaime Iranzo; Adriana Luc\u00cda-Sanz; Jens H. Kuhn; Mart Krupovic; Valerian V. Dolja; Eugene V. Koonin","M. Verosloff; J. Chappell; K. L. Perry; J. R. Thompson; J. B. Lucks","Jiao-Mei Huang; Syed Sajid Jan; Xiaobin Wei; Yi Wan; Songying Ouyang","Xiaoyan Pan; Pengfei Zhou; Tiejiong Fan; Yan Wu; Jing Zhang; Xiaoyue Shi; Weijuan Shang; Lijuan Fang; Xiaming Jiang; Jian Shi; Yuan Sun; Shaojuan Zhao; Rui Gong; Ze Chen; Gengfu Xiao","C Langelier; MS Zinter; K Kalantar; GA Yanik; S Christenson; B Odonovan; C White; M Wilson; A Sapru; CC Dvorak; S Miller; CY Chiu; JL DeRisi","Justine Jia Wen Seow; Rhea Pai; Archita Mishra; Edwin Shepherdson; Tony Kiat Hon Lim; Brian KP Goh; Jerry KY Chan; Pierce KH Chow; Florent Ginhoux; Ramanuj DasGupta; Ankur Sharma","Na Zhao; Jian Wang; Yong Yu; Jun-Yan Zhao; Duan-Bing Chen","Timothy R. Abbott; Girija Dhamdhere; Yanxia Liu; Xueqiu Lin; Laine Goudy; Leiping Zeng; Augustine Chemparathy; Stephen Chmura; Nicholas S. Heaton; Robert Debs; Tara Pande; Drew Endy; Marie La Russa; David B. Lewis; Lei S. Qi","Laura Riva; Shuofeng Yuan; Xin Yin; Laura Martin-Sancho; Naoko Matsunaga; Sebastian Burgstaller-Muehlbacher; Lars Pache; Paul P. De Jesus; Mitchell V. Hull; Max Chang; Jasper Fuk-Woo Chan; Jianli Cao; Vincent Kwok-Man Poon; Kristina Herbert; Tu-Trinh Nguyen; Yuan Pu; Courtney Nguyen; Andrey Rubanov; Luis Martinez-Sobrido; Wen-Chun Liu; Lisa Miorin; Kris M. White; Jeffrey R. Johnson; Christopher Benner; Ren Sun; Peter G. Schultz; Andrew Su; Adolfo Garcia-Sastre; Arnab K. Chatterjee; Kwok-Yung Yuen; Sumit K. Chanda","Lindsay Meyers; Christine C. Ginocchio; Aimie N. Faucett; Frederick S. Nolte; Per H. Gesteland; Amy Leber; Diane Janowiak; Virginia Donovan; Jennifer Dien Bard; Silvia Spitzer; Kathleen A. Stellrecht; Hossein Salimnia; Rangaraj Selvarangan; Stefan Juretschko; Judy A. Daly; Jeremy C. Wallentine; Kristy Lindsey; Franklin Moore; Sharon L. Reed; Maria Aguero-Rosenfeld; Paul D. Fey; Gregory A. Storch; Steve J. Melnick; Christine C. Robinson; Jennifer F. Meredith; Camille V. Cook; Robert K. Nelson; Jay D. Jones; Samuel V. Scarpino; Benjamin M. Althouse; Kirk M. Ririe; Bradley A. Malin; Mark A. Poritz","Yan Li; Jinyong Zhang; Ning Wang; Haibo Li; Yun Shi; Gang Guo; Kaiyun Liu; Hao Zeng; Quanming Zou","Konstans Wells; Serge Morand; Maya Wardeh; Matthew Baylis","Courtney R. Sullivan; Catharine A. Mielnik; Sinead M. O\u2019Donovan; Adam J. Funk; Eduard Bentea; Erica A.K. DePasquale; Zhexing Wen; Vahram Haroutunian; Pavel Katsel; Amy J. Ramsey; Jarek Meller; Robert E. McCullumsmith","Darko Bosnakovski; Erik A. Toso; Olivia O. Recht; Anja Cucak; Abhinav K Jain; Michelle C. Barton; Michael Kyba","James Brett Case; Yize Li; Ruth Elliott; Xiaotao Lu; Kevin W. Graepel; Nicole R. Sexton; Everett Clinton Smith; Susan R. Weiss; Mark R. Denison","Cara E. Brook; Mike Boots; Kartik Chandran; Andrew P. Dobson; Christian Drosten; Andrea L. Graham; Bryan T. Grenfell; Marcel A. M\u00fcller; Melinda Ng; Lin-Fa Wang; Anieke van Leeuwen","Renhong Yan; Yuanyuan Zhang; Yingying Guo; Lu Xia; Qiang Zhou","Karthi Balasubramanian; Nithin Nagaraj","Maoz Gelbart; Adi Stern","Yong Jia; Gangxu Shen; Yujuan Zhang; Keng-Shiang Huang; Hsing-Ying Ho; Wei-Shio Hor; Chih-Hui Yang; Chengdao Li; Wei-Lung Wang","Linlin Bao; Wei Deng; Hong Gao; Chong Xiao; Jiayi Liu; Jing Xue; Qi Lv; Jiangning Liu; Pin Yu; Yanfeng Xu; Feifei Qi; Yajin Qu; Fengdi Li; Zhiguang Xiang; Haisheng Yu; Shuran Gong; Mingya Liu; Guanpeng Wang; Shunyi Wang; Zhiqi Song; Wenjie Zhao; Yunlin Han; Linna Zhao; Xing Liu; Qiang Wei; Chuan Qin","Cindy Kundlacz; Marie Pourcelot; Aurore Fablet; Rayane Amaral Da Silva Moraes; Thibaut L\u00e9ger; Bastien Morlet; Cyril Viarouge; Corinne Sailleau; Mathilde Turpaud; Axel Gorlier; Emmanuel Breard; Sylvie Lecollinet; Piet A. van Rijn; Stephan Zientara; Damien Vitour; Gr\u00e9gory Caignard","Sang Woo Park; David Champredon; Jonathan Dushoff","Zebin Lin; Qian Fang; Jinlian Mai; Lishi Zhou; Yu Qian; Tian Cai; Zhenhua Chen; Ping Wang; Bin Lin","V\u00e1clav Vop\u00e1lensk\u00fd; Michal S\u00fdkora; Tom\u00e1\u0161 Ma\u0161ek; Martin Posp\u00ed\u0161ek","Saba Ismail; Sajjad Ahmad; Syed Sikander Azam","Asaf Poran; Dewi Harjanto; Matthew Malloy; Michael S. Rooney; Lakshmi Srinivasan; Richard B. Gaynor","Julien Riou; Chiara Poletto; Pierre-Yves Bo\u00eblle","Ahmed Shiek; Prabu Paramasivam; Kamal Raj; Vishal Kumar","Chengxin Zhang; Wei Zheng; Xiaoqiang Huang; Eric W. Bell; Xiaogen Zhou; Yang Zhang","Bhrugesh Joshi; Vishvajit Bakarola; Parth Shah; Ramar Krishnamurthy","Danielle E. Goodman; Carla D. Pretto; Tomas A. Krepostman; Kelly E. Carnahan; Katherine R. Spindler","Ramasubramanian Sundaramoorthy; Amanda L. Hughes; Hassane El-Mkami; David Norman; Tom Owen-Hughes","Chek Meng Poh; Guillaume Carissimo; Wang Bei; Siti Naqiah Amrun; Cheryl Yi-Pin Lee; Rhonda Sin-Ling Chee; Nicholas Kim-Wah Yeo; Wen-Hsin Lee; Yee-Sin Leo; Mark I-Cheng Chen; Seow-Yen Tan; Louis Yi Ann Chai; Shirin Kalimuddin; Siew-Yee Thien; Barnaby Edward Young; David C. Lye; Cheng-I Wang; Laurent Renia; Lisa FP Ng","Anthony E. Ambrosini; Nikhil Deshmukh; Michael J. Berry; Lynn W. Enquist; Ian B. Hogue","Casper K Lumby; Nuno R Nene; Christopher J R Illingworth","A. Sina Booeshaghi; Lior Pachter","Martin Mikl; Yitzhak Pilpel; Eran Segal","Vineet D. Menachery; Lisa E. Gralinski; Hugh D. Mitchell; Kenneth H. Dinnon; Sarah R. Leist; Boyd L. Yount; Eileen T. McAnarney; Rachel L. Graham; Katrina M. Waters; Ralph S. Baric","Lim Heo; Michael Feig","Qiang Huang; Andreas Herrmann","K. Reddisiva Prasanth; Minato Hirano; W. Samuel Fagg; Eileen T. McAnarney; Chao Shan; Xuping Xie; Adam Hage; Colette A. Pietzsch; Alexander Bukreyev; Ricardo Rajsbaum; Pei-Yong Shi; Mark T. Bedford; Shelton S. Bradrick; Vineet Menachery; Mariano A. Garcia-Blanco","Natalia Fintelman-Rodrigues; Carolina Q. Sacramento; Carlyle Ribeiro Lima; Franklin Souza da Silva; Andr\u00e9 C. Ferreira; Mayara Mattos; Caroline S. de Freitas; Vinicius Cardoso Soares; Suelen da Silva Gomes Dias; Jairo R. Temerozo; Milene Miranda; Aline R. Matos; Fernando A. Bozza; Nicolas Carels; Carlos Roberto Alves; Marilda M. Siqueira; Patr\u00edcia T. Bozza; Thiago Moreno L. Souza","Xuesen Zhao; Danying Chen; Robert Szabla; Mei Zheng; Guoli Li; Pengcheng Du; Shuangli Zheng; Xinglin Li; Chuan Song; Rui Li; Ju-Tao Guo; Murray Junop; Hui Zeng; Hanxin Lin","Claude Pasquier; Alain Robichon","Jinfang Yu; Shuyuan Qiao; Runyu Guo; Xinquan Wang","Ching-Fu Tu; Chin-kai Chuang; Kai-Hsuan Hsiao; Chien-Hong Chen; Chie-Min Chen; Su-Hei Peng; Yu-Hsiu Su; Ming-Tang Chiou; Chon-Ho Yen; Shau-Wen Hung; Tien-Shuh Yang; Chuan-Mu Chen","Ottar N. Bj\u00f8rnstad; Bryan T. Grenfell; Cecile Viboud; Aaron A. King","Linlin Zhang; Daizong Lin; Xinyuanyuan Sun; Katharina Rox; Rolf Hilgenfeld","Carolina Beltran-Pavez; Chantal L. Marquez; Gabriela Munoz; Fernando Valiente-Echeverria; Aldo Gaggero; Ricardo Soto-Rifo; Gonzalo P. Barriga","Abigail L. Coughtrie; Denise E. Morris; Rebecca Anderson; Nelupha Begum; David W. Cleary; Saul N. Faust; Johanna M. Jefferies; Alex R. Kraaijeveld; Michael V. Moore; Mark A. Mullee; Paul J. Roderick; Andrew Tuck; Robert N. Whittaker; Ho Ming Yuen; C. Patrick Doncaster; Stuart C. Clarke","Devon Chandler-Brown; Anna M. Bueno; Oguzhan Atay; David S. Tsao","Christina J. Castro; Rachel L. Marine; Edward Ramos; Terry Fei Fan Ng","Wayne M. Getz; Richard Salter; Oliver Muellerklein; Hyun S. Yoon; Krti Tallam","Maria Nevot; Ana Jordan-Paiz; Gl\u00f2ria Martrus; Cristina Andr\u00e9s; Damir Garc\u00eda-Cehic; Josep Gregori; Sandra Franco; Josep Quer; Miguel Angel Martinez","Qi Liu; Amita Gupta; Ayse Okesli-Armlovich; Wenjie Qiao; Curt R. Fischer; Mark Smith; Jan E. Carette; Michael C. Bassik; Chaitan Khosla","Hyejeong Kim; Victor D. Ellis; Andrew Woodman; Yan Zhao; Jamie J. Arnold; Craig E. Cameron","Leah Cuthbertson; Vanessa Craven; Lynne Bingle; William O.C.M. Cookson; Mark L. Everard; Miriam F. Moffatt","Kun Sun; Liuqi Gu; Li Ma; Yunfeng Duan","V Vern Lee; Louise M. Judd; Aaron R. Jex; Kathryn E. Holt; Christopher J. Tonkin; Stuart A. Ralph","Bing Zhao; Chao Ni; Ran Gao; Yuyan Wang; Li Yang; Jinsong Wei; Ting Lv; Jianqing Liang; Qisheng Zhang; Wei Xu; Youhua Xie; Xiaoyue Wang; Zhenghong Yuan; Junbo Liang; Rong Zhang; Xinhua Lin","Jing Xing; Rama Shankar; Aleksandra Drelich; Shreya Paithankar; Eugene Chekalin; Thomas Dexheimer; Surender Rajasekaran; Chien-Te Kent Tseng; Bin Chen","Kathryn D. Tuttle; Ross Minter; Katherine A. Waugh; Paula Araya; Michael Ludwig; Colin Sempeck; Keith Smith; Zdenek Andrysik; Matthew A. Burchill; Beth A.J. Tamburini; David J. Orlicky; Kelly D. Sullivan; Joaquin M. Espinosa","Vojo Deretic; Graham S Timmins","Prashali Bansal; Johannes Madlung; Kristina Schaaf; Boris Macek; Fulvia Bono","Natalia B. Hubbs; Mareena M. Whisby-Pitts; Jonathan L. McMurry","Ian M. Silverman; Sager J. Gosai; Nicholas Vrettos; Shawn W. Foley; Nathan D. Berkowitz; Zissimos Mourelatos; Brian D. Gregory","Vinicio Armijos-Jaramillo; Justin Yeager; Claire Muslin; Yunierkis Perez-Castillo","L. Tomova; K. Wang; T. Thompson; G. Matthews; A. Takahashi; K. Tye; R. Saxe","Michael D Vahey; Daniel A Fletcher","Tao Liu; Jianxiong Hu; Jianpeng Xiao; Guanhao He; Min Kang; Zuhua Rong; Lifeng Lin; Haojie Zhong; Qiong Huang; Aiping Deng; Weilin Zeng; Xiaohua Tan; Siqing Zeng; Zhihua Zhu; Jiansen Li; Dexin Gong; Donghua Wan; Shaowei Chen; Lingchuan Guo; Yan Li; Limei Sun; Wenjia Liang; Tie Song; Jianfeng He; Wenjun Ma","Jacob Kames; David Dillon Holcomb; Ofer Kimchi; Michael DiCuccio; Nobuko Hamasaki-Katagiri; Tony Wang; Anton A Komar; Aikaterini Alexaki; Chava Kimchi-Sarfaty","Jingjiao Li; Mingquan Guo; Xiaoxu Tian; Chengrong Liu; Xin Wang; Xing Yang; Ping Wu; Zixuan Xiao; Yafei Qu; Yue Yin; Joyce Fu; Zhaoqin Zhu; Zhenshan Liu; Chao Peng; Tongyu Zhu; Qiming Liang","Adam M. Dinan; Nina I. Lukhovitskaya; Ingrida Olendraite; Andrew E. Firth","Florian Mock; Adrian Viehweger; Emanuel Barth; Manja Marz","Xiao Huang; Jasper Z. Williams; Ryan Chang; Zhongbo Li; Eric Gai; David M. Patterson; Yu Wei; Wendell A. Lim; Tejal A. Desai","Nerea Irigoyen; Adam M. Dinan; Luke W. Meredith; Ian Goodfellow; Ian Brierley; Andrew E. Firth","Andres J. Gonzalez; Emmanuel C. Ijezie; Onesmo B. Balemba; Tanya A. Miura","Michele Tizzoni; Andr\u00e9 Panisson; Daniela Paolotti; Ciro Cattuto","Sneha Rath; Eliza Prangley; Jesse Donovan; Kaitlin Demarest; Yigal Meir; Ned Wingreen; Alexei Korennykh","Neha Jain; Subodh Kumar Mishra; Uma Shankar; Arpita Tawani; Ankit Jaiswal; Tarun Kumar Sharma; Prashant Kodgire; Amit Kumar","Sarah Sierra Smaga; Chaoyi Xu; Brady James Summers; Katherine Marie Digianantonio; Juan Roberto Perilla; Yong Xiong","Mert Gur; Elhan Taka; Sema Zeynep Yilmaz; Ceren Kilinc; Umut Aktas; Mert Golcuk","Jiayu Shang; Yanni Sun","Kris V Parag; Christl A Donnelly","Gytis Dudas; Luiz Max Carvalho; Andrew Rambaut; Trevor Bedford; Ali M. Somily; Mazin Barry; Sarah S. Al Subaie; Abdulaziz A. BinSaeed; Fahad A. Alzamil; Waleed Zaher; Theeb Al Qahtani; Khaldoon Al Jerian; Scott J.N. McNabb; Imad A. Al-Jahdali; Ahmed M. Alotaibi; Nahid A. Batarfi; Matthew Cotten; Simon J. Watson; Spela Binter; Paul Kellam","Guido Papa; Luca Venditti; Francesca Arnoldi; Elisabeth M. Schraner; Christiaan Potgieter; Alexander Borodavka; Catherine Eichwald; Oscar R. Burrone","Bingbing He; Wenbo Peng; Jia Huang; Hang Zhang; Yingsi Zhou; Xiali Yang; Jing Liu; Zhijie Li; Chunlong Xu; Mingxing Xue; Hui Yang; Pengyu Huang","Huan Wang; Yingjie Sun; Xiang Mao; Chunchun Meng; Lei Tan; Cuiping Song; Xusheng Qiu; Chan Ding; Ying Liao","Shay Leary; Silvana Gaudieri; Abha Chopra; Suman Pakala; Eric Alves; Mina John; Suman Das; Simon Mallal; Phillips Jane Phillips","Feria Hikmet; Loren M\u00e9ar; Mathias Uhl\u00e9n; Cecilia Lindskog","Christoph Muus; Malte D Luecken; Gokcen Eraslan; Avinash Waghray; Graham Heimberg; Lisa Sikkema; Yoshihiko Kobayashi; Eeshit Dhaval Vaishnav; Ayshwarya Subramanian; Christopher Smillie; Karthik Jagadeesh; Elizabeth Thu Duong; Evgenij Fiskin; Elena Torlai Triglia; Christophe Becavin; Meshal Ansari; Peiwen Cai; Brian Lin; Justin Buchanan; Sijia Chen; Jian Shu; Adam L Haber; Hattie Chung; Daniel T Montoro; Taylor Adams; Hananeh Aliee; Samuel J Allon; Zaneta Andrusivova; Ilias Angelidis; Orr Ashenberg; Kevin Bassler; Christophe Becavin; Inbal Benhar; Joseph Bergenstrahle; Ludvig Bergenstrahle; Liam Bolt; Emelie Braun; Linh T Bui; Mark Chaffin; Evgeny Chichelnitskiy; Joshua Chiou; Thomas M Conlon; Michael S Cuoco; Marie Deprez; David S Fischer; Astrid Gillich; Joshua Gould; Minzhe Guo; Austin J Gutierrez; Arun C Habermann; Tyler Harvey; Peng He; Xiaomeng Hou; Lijuan Hu; Alok Jaiswal; Peiyong Jiang; Theodoros Kapellos; Christin S Kuo; Ludvig Larsson; Michael A Leney-Greene; Kyungtae Lim; Monika Litvinukova; Ji Lu; Leif S Ludwig; Wendy Luo; Henrike Maatz; Elo Maddissoon; Lira Mamanova; Kasidet Manakongtreecheep; Charles-Hugo Marquette; Ian Mbano; Alexi M McAdams; Ross J Metzger; Ahmad N Nabhan; Sarah K Nyquist; Jose Ordovas-Montanes; Lolita Penland; Olivier B Poirion; Segio Poli; CanCan Qi; Daniel Reichart; Ivan Rosas; Jonas Schupp; Rahul Sinha; Rene V Sit; Kamil Slowikowski; Michal Slyper; Neal Smith; Alex Sountoulidis; Maximilian Strunz; Dawei Sun; Carlos Talavera-Lopez; Peng Tan; Jessica Tantivit; Kyle J Travaglini; Nathan R Tucker; Katherine Vernon; Marc H Wadsworth; Julia Waldman; Xiuting Wang; Wenjun Yan; Ali Onder Yildirim; William Zhao; Carly G K Ziegler; Aviv Regev","Fayyaz Minhas; Dimitris Grammatopoulos; Lawrence Young; Imran Amin; David Snead; Neil Anderson; Asa Ben-Hur; Nasir Rajpoot","Wenhao Dai; Bing Zhang; Xia-Ming Jiang; Haixia Su; Jian Li; Yao Zhao; Xiong Xie; Zhenming Jin; Jingjing Peng; Fengjiang Liu; Chunpu Li; You Li; Fang Bai; Haofeng Wang; Xi Cheng; Xiaobo Cen; Shulei Hu; Xiuna Yang; Jiang Wang; Xiang Liu; Gengfu Xiao; Hualiang Jiang; Zihe Rao; Leike Zhang; Yechun Xu; Haitao Yang; Hong Liu","Ahmed Hassan Fahal; Amir Abushouk; Amre Nasr; Emad Masuadi; Gamal Allam; Emmanuel E. Siddig","Zhijian Xu; Cheng Peng; Yulong Shi; Zhengdan Zhu; Kaijie Mu; Xiaoyu Wang; Weiliang Zhu","Miles D. Miller-Dickson; Victor A. Meszaros; Francis Baffour-Awuah; Salvador Almagro-Moreno; C. Brandon Ogbunugafor","Simon H. Tausch; Tobias P. Loka; Jakob M. Schulze; Andreas Andrusch; Jeanette Klenner; Piotr Wojciech Dabrowski; Martin S. Lindner; Andreas Nitsche; Bernhard Y. Renard","Liangzhong Lim; Amrita Roy; Jianxing Song","Tingting Li; Qingbing Zheng; Hai Yu; Dinghui Wu; Wenhui Xue; Yuyun Zhang; Xiaofen Huang; Lizhi Zhou; Zhigang Zhang; Zhenghui Zha; Tingting Chen; Zhiping Wang; Jie Chen; Hui Sun; Tingting Deng; Yingbin Wang; Yixin Chen; Qinjian Zhao; Jun Zhang; Ying Gu; Shaowei Li; Ningshao Xia","Kevin W. Graepel; Xiaotao Lu; James Brett Case; Nicole R. Sexton; Everett Clinton Smith; Mark R. Denison","Jose-Antonio Rodriguez-Rodriguez; Kara L. McKinley; Vitali Sikirzhytski; Jennifer Corona; John Maciejowski; Alexey Khodjakov; Iain M. Cheeseman; Prasad V. Jallepalli","Jessica A. Plante; Kenneth S. Plante; Lisa E. Gralinski; Anne Beall; Martin T. Ferris; Daniel Bottomly; Richard Green; Shannon K. McWeeney; Mark T. Heise; Ralph S. Baric; Vineet D. Menachery","Marcus Ludwig; Louis-F\u00e9lix Nothias; Kai D\u00fchrkop; Irina Koester; Markus Fleischauer; Martin A. Hoffmann; Daniel Petras; Fernando Vargas; Mustafa Morsy; Lihini Aluwihare; Pieter C. Dorrestein; Sebastian B\u00f6cker","A. Reiser; D. Wosch\u00e9e; N. Mehrotra; R. Krzyszto\u0144; H. H. Strey; J. O. R\u00e4dler","Lee Worden; Rae Wannier; Nicole A. Hoff; Kamy Musene; Bernice Selo; Mathias Mossoko; Emile Okitolonda-Wemakoy; Jean Jacques Muyembe-Tamfum; George W. Rutherford; Thomas M. Lietman; Anne W. Rimoin; Travis C. Porco; J. Daniel Kelly","Brielle C Stark; Alexandra Basilakos; Gregory Hickok; Chris Rorden; Leonardo Bonilha; Julius Fridriksson","Renhong Yan; Yuanyuan Zhang; Yaning Li; Lu Xia; Qiang Zhou","V\u2019kovski Philip; Gerber Markus; Kelly Jenna; Pfaender Stephanie; Ebert Nadine; Braga Lagache Sophie; Simillion Cedric; Portmann Jasmine; Stalder Hanspeter; Gaschen V\u00e9ronique; Bruggmann Remy; Stoffel Michael; Heller Manfred; Dijkman Ronald; Thiel Volker","Shuai Xia; Meiqin Liu; Chao Wang; Wei Xu; Qiaoshuai Lan; Siliang Feng; Feifei Qi; Linlin Bao; Lanying Du; Shuwen Liu; Chuan Qin; Fei Sun; Zhengli Shi; Yun Zhu; Shibo Jiang; Lu Lu","Thin Nguyen; Hang Le; Thomas P. Quinn; Thuc Le; Svetha Venkatesh","Serge P.J.M. Horbach","Ibne Raihan Zunaid; Stefania Pacini; Marco Ruggiero","Katherine E. Roberts; Jarrod D. Hadfield; Manmohan D. Sharma; Ben Longdon","Yasunori Watanabe; Joel D. Allen; Daniel Wrapp; Jason S. McLellan; Max Crispin","AJ Venkatakrishnan; Arjun Puranik; Akash Anand; David Zemmour; Xiang Yao; Xiaoying Wu; Ramakrishna Chilaka; Dariusz K Murakowski; Kristopher Standish; Bharathwaj Raghunathan; Tyler Wagner; Enrique Garcia-Rivera; Hugo Solomon; Abhinav Garg; Rakesh Barve; Anuli Anyanwu-Ofili; Najat Khan; Venky Soundararajan","Riccardo Delli ponti; Alexandros Armaos; Stefanie Marti; Gian Gaetano Tartaglia","L\u00e9a Joffrin; Axel O. G. Hoarau; Erwan Lagadec; Patrick Mavingui; Camille Lebarbenchon","Jonatan Luciano-Jaramillo; Flavio Sandoval-Garc\u00eda; M\u00f3nica V\u00e1zquez-Del Mercado; Yanet Karina Guti\u00e9rrez-Mercado; Rosa Elena Navarro-Hern\u00e1ndez; Erika Aurora Mart\u00ednez-Garc\u00eda; Oscar Pizano-Mart\u00ednez; Fernanda Isadora Corona-Meraz; Jacinto Ba\u00f1uelos-Pineda; Jorge Floresvillar-Mosqueda; Beatriz Teresita Mart\u00edn-M\u00e1rquez","William E. Diehl; Mehmet H. Guney; Pyae Phyo Kyawe; Judith M. White; Massimo Pizzato; Jeremy Luban","Adriana Larrea-Sarmiento; Anne M. Alvarez; James P. Stack; Mohammad Arif","Manish Tiwari; Divya Mishra","Rasmus Nielsen; Hongru Wang; Lenore Pipes","Shutoku Matsuyama; Miyuki Kawase; Naganori Nao; Kazuya Shirato; Makoto Ujike; Wataru Kamitani; Masayuki Shimojima; Shuetsu Fukushi","Chunyun Sun; Long Chen; Ji Yang; Chunxia Luo; Yanjing Zhang; Jing Li; Jiahui Yang; Jie Zhang; Liangzhi Xie","Ratul Chowdhury; Costas D Maranas","Andrea Vandelli; Michele Monti; Edoardo Milanetti; Riccardo Delli Ponti; Gian Gaetano Tartaglia","Syed Faraz Ahmed; Ahmed A. Quadeer; Matthew R. McKay","Yingyun Cai; Shuiqing Yu; Rohit Jangra; Elena Postnikova; Jiro Wada; Robert Tesh; Sean Whelan; Michael Lauck; Michael Wiley; Courtney Finch; Sheli Radoshitzky; David O'Connor; Gustavo Palacios; Kartik Chandran; Charles Chiu; Jens Kuhn","Jillian N Whelan; Joshua Hatterschide; David M. Renner; Beihua Dong; Robert H Silverman; Susan R Weiss","N.V Rajeshkumar; Shinichi Yabuuchi; Shweta G. Pai; Anirban Maitra; Manuel Hidalgo; Chi V. Dang","Juan Cristina; Natalia Echeverr\u00eda; Fabiana Gambaro; Alvaro Fajardo; Pilar Moreno","S. Willis; J. Masel","Zheng Zhang; Zena Cai; Zhiying Tan; Congyu Lu; Gaihua Zhang; Yousong Peng","Xueting Qiu; Justin Bahl","Yonghua Wu","Vineet D. Menachery; Kenneth H. Dinnon; Boyd L. Yount; Eileen T. McAnarney; Lisa E. Gralinski; Andrew Hale; Rachel L. Graham; Trevor Scobey; Simon J. Anthony; Lingshu Wang; Barney Graham; Scott H. Randell; W. Ian Lipkin; Ralph S. Baric","Zhizhou Tan; Gabriel Gonzalez; Jinliang Sheng; Jianmin Wu; Fuqiang Zhang; Lin Xu; Peisheng Zhang; Aiwei Zhu; Yonggang Qu; Changchun Tu; Michael J. Carr; Biao He","Nathan D. Grubaugh; Sharada Saraf; Karthik Gangavarapu; Alexander Watts; Amanda L. Tan; Rachel J. Oidtman; Jason T. Ladner; Glenn Oliveira; Nathaniel L. Matteson; Moritz U.G. Kraemer; Chantal B.F. Vogels; Aaron Hentoff; Deepit Bhatia; Danielle Stanek; Blake Scott; Vanessa Landis; Ian Stryker; Marshall R. Cone; Edgar W. Kopp; Andrew C. Cannons; Lea Heberlein-Larson; Stephen White; Leah D. Gillis; Michael J. Ricciardi; Jaclyn Kwal; Paola K. Lichtenberger; Diogo M. Magnani; David I. Watkins; Gustavo Palacios; Davidson H. Hamer; Lauren M. Gardner; T. Alex Perkins; Guy Baele; Kamran Khan; Andrea Morrison; Sharon Isern; Scott F. Michael; Kristian G. Andersen","Monique R. Ambrose; Adam J. Kucharski; Pierre Formenty; Jean-Jacques Muyembe-Tamfum; Anne W. Rimoin; James O. Lloyd-Smith","Martin Bartas; V\u00e1clav Br\u00e1zda; Nat\u00e1lia Boh\u00e1lov\u00e1; Alessio Cantara; Adriana Voln\u00e1; Tereza Stachurov\u00e1; Kate\u0159ina Malachov\u00e1; Eva B. Jagelsk\u00e1; Ot\u00edlia Porubiakov\u00e1; Ji\u0159\u00ed \u010cerve\u0148; Petr Pe\u010dinka","Kyle J Travaglini; Ahmad N Nabhan; Lolita Penland; Rahul Sinha; Astrid Gillich; Rene V Sit; Stephen Chang; Stephanie D Conley; Yasuo Mori; Jun Seita; Gerald J Berry; Joseph B Shrager; Ross J Metzger; Christin S Kuo; Norma Neff; Irving L Weissman; Stephen R Quake; Mark A Krasnow","Kathryn S. Evans; Erik C. Andersen","Tayaza Fadason; Sreemol Gokuladhas; Evgeniia Golovina; Daniel Ho; Sophie Farrow; Denis Nyaga; Hong Pan; Neerja Karnani; Conroy Wong; Antony Cooper; William Schierding; Justin M. O\u2019Sullivan","Jonathan L Schmid-Burgk; David Li; David Feldman; Mikolaj Slabicki; Jacob Borrajo; Jonathan Strecker; Brian Cleary; Aviv Regev; Feng Zhang","Jasmim Leal; Xinquan Liu; Xiujuan Peng; Rashmi P. Mohanty; Dhivya Arasappan; Dennis Wylie; Sarah H. Schwartz; Jason J. Fullmer; Bennie C. McWilliams; Hugh D. C. Smyth; Debadyuti Ghosh","Ling Leng; Jie Ma; Leike Zhang; Wei Li; Lei Zhao; Yunping Zhu; Zhihong Wu; Ruiyuan Cao; Wu Zhong","Domenico Benvenuto; Marta Giovannetti; Alessandra Ciccozzi; Silvia Spoto; Silvia Angeletti; Massimo Ciccozzi","Francisco D\u00edez Fuertes; Mar\u00eda Iglesias Caballero; Sara Monz\u00f3n; Pilar Jim\u00e9nez; Sarai Varona; Isabel Cuesta; \u00c1ngel Zaballos; Michael M. Thomson; Mercedes Jim\u00e9nez; Javier Garc\u00eda P\u00e9rez; Francisco Pozo; Mayte P\u00e9rez Olmeda; Jos\u00e9 Alcam\u00ed; Inmaculada Casas","Diogo de Moraes; Brunno Vivone Buquete Paiva; Sarah Santiloni Cury; Jo\u00e3o Pessoa Ara\u00fajo Junior; Marcelo Alves da Silva Mori; Robson Francisco Carvalho","Max Hebditch; Jim Warwicker","Lamia Wahba; Nimit Jain; Andrew Z. Fire; Massa J. Shoura; Karen L. Artiles; Matthew J. McCoy; Dae-Eun Jeong","Jonathon A. Siva-Jothy; Pedro F. Vale","Timokratis Karamitros; Gethsimani Papadopoulou; Maria Bousali; Anastasios Mexias; Sotiris Tsiodras; Andreas Mentis","Lifeng Zhou; Arun Richard Chandrasekaran; Jibin Abraham Punnoose; Gaston Bonenfant; Stephon Charles; Oksana Levchenko; Pheonah Badu; Cassandra Cavaliere; Cara T. Pager; Ken Halvorsen","Norio Yamamoto; Shutoku Matsuyama; Tyuji Hoshino; Naoki Yamamoto","Amy Dighe; Thibaut Jombart; Maria D. Van Kerkhove; Neil Ferguson","Lu Lu; Liam Brierley; Gail Robertson; Feifei Zhang; Samantha Lycett; Donald Smith; Margo Chase-Topping; Peter Simmonds; Mark Woolhouse","Adriaan H. de Wilde; A. Linda Boomaars-van der Zanden; Anja W. M. de Jong; Montserrat Barc\u00e9na; Eric J. Snijder; Clara C. Posthuma","Emma S Chambers; Milica Vukmanovic-Stejic; Barbara B Shih; Hugh Trahair; Priya Subramanian; Oliver P Devine; James Glanville; Derek Gilroy; Malcom Rustin; Tom C Freeman; Neil A Mabbot; Arne N Akbar","Jordi Mestres","Nicholas Fountain-Jones; Gustavo Machado; Scott Carver; Craig Packer; Mariana Mendoza; Meggan E Craft","Miguel Corona; Belen Branchiccela; Shayne Madella; Yanping Chen; Jay Evans","David Enard; Le Cai; Carina Gwenapp; Dmitri A Petrov","Qizhi Sun; Mohamed I. Gatie; Gregory M. Kelly","Eva Dervas; Jussi Hepojoki; Teemu Smura; Barbara Pr\u00e4hauser; Katharina Windbichler; Sandra Blumich; Antonio Ramis; Udo Hetzel; Anja Kipar","Liam Brierley; Amy B. Pedersen; Mark E. J. Woolhouse","Xiankun Zeng; Jun Liu; April Babka; Brian Kearney; Sheli Radoshitzky; Jens Kuhn","Kumari G. Lokugamage; Adam Hage; Craig Schindewolf; Ricardo Rajsbaum; Vineet D. Menachery","Jan Slingenbergh","Conor Jenkins; Ben Orsburn","Angela Steinauer; Jonathan R. LaRochelle; Rebecca Wissner; Samuel Berry; Alanna Schepartz","Eric J. Snijder; Ronald W.A.L. Limpens; Adriaan H. de Wilde; Anja W. M. de Jong; Jessika C. Zevenhoven-Dobbe; Helena J. Maier; F.G.A. Faas; Abraham J. Koster; Montserrat B\u00e1rcena","Erica T. Prates; Michael R. Garvin; Mirko Pavicic; Piet Jones; Manesh Shah; Christiane Alvarez; David Kainer; Omar Demerdash; B Kirtley Amos; Armin Geiger; John Pestian; Kang Jin; Alexis Mitelpunkt; Eric Bardes; Bruce Aronow; Daniel Jacobson","Saud H Aman; Mohammed I Aljaber; Abdullah I Alwehaibi; Fahad H Aman; Hamad A Algaeed; Sultan M Almasoud; Majed A Alahmari; Omair Al Hussain; Ahmed Elhag","Minchen Chien; Thomas K. Anderson; Steffen Jockusch; Chuanjuan Tao; Shiv Kumar; Xiaoxu Li; James J. Russo; Robert Kirchdoerfer; Jingyue Ju","Siying Ye; Chris Cowled; Cheng-Hon Yap; John Stambas","Lulin Zhou; Zubiao Niu; Xiaoyi Jiang; Zhengrong Zhang; You Zheng; Zhongyi Wang; Yichao Zhu; Lihua Gao; Xiaoning Wang; Qiang Sun","Edison Ong; Mei U Wong; Anthony Huffman; Yongqun He","Gytis Dudas; Andrew Rambaut","Sushmita Sridhar; Sally Forrest; Iain Kean; Jamie Young; Josefin Bartholdson Scott; Mailis Maes; Joana Pereira-Dias; Surendra Parmar; Matthew Routledge; Lucy Rivett; Gordon Dougan; Michael Weekes; Martin Curran; Ian Goodfellow; Stephen Baker","Ilda D\u2019Annessa; Filippo Marchetti; Giorgio Colombo","Efstathios S Giotis; George Carnell; Erik F Young; Saleena Ghanny; Patricia Soteropoulos; Wendy S Barclay; Michael A Skinner; Nigel Temperton","Michelle Wille; Mang Shi; Marcel Klaassen; Aeron C. Hurt; Edward C. Holmes","Ilana L. Brito; Thomas Gurry; Shijie Zhao; Katherine Huang; Sarah K. Young; Terrence P. Shea; Waisea Naisilisili; Aaron P. Jenkins; Stacy D. Jupiter; Dirk Gevers; Eric J. Alm","Karen N. Barnard; Brian R. Wasik; Justin R. LaClair; Wendy S. Weichert; Brynn K. Lawrence; Colin R. Parrish","Nicholas G Reich; Justin Lessler; Jay K Varma; Neil M Vora","Matthew C. Wong; Sara J. Javornik Cregeen; Nadim J. Ajami; Joseph F. Petrosino","Yuanmei Zhu; Danwei Yu; Hongxia Yan; Huihui Chong; Yuxian He","Mathilde Richard; Adinda Kok; Dennis de Meulder; Theo M. Bestebroer; Mart M. Lamers; Nisreen M.A. Okba; Martje Fentener van Vlissingen; Barry Rockx; Bart L. Haagmans; Marion P.G. Koopmans; Ron A.M. Fouchier; Sander Herfst","Dupas Enora; Briand Martial; Jacques Marie-Agn\u00e8s; Cesbron Sophie","Natacha Ogando; Timmothy Dalebout; Jessika C. Zevenhoven-Dobbe; Ronald W.A.L. Limpens; Yvonne van der Meer; Leon Caly; Julian Druce; Jutte de Vries; Marjolein Kikkert; Montse Barcena; Igor Sidorov; Eric J Snijder","Harlan Barker; Seppo Parkkila","Richard E. Grewelle","Aleksandra Milewska; Ying Chi; Artur Szczepanski; Emilia Barreto-Duran; Kevin Liu; Dan Liu; Xiling Guo; Yiyue Ge; Jingxin Li; Lunbiao Cui; Marek Ochman; Maciej Urlik; Sylwia Rodziewicz-Motowidlo; Fengcai Zhu; Krzysztof Szczubialka; Maria Nowakowska; Krzysztof Pyrc","Joseph C. Ward; Lidia Lasecka-Dykes; Chris Neil; Oluwapelumi Adeyemi; Sarah Gold; Niall McLean; Caroline Wright; Morgan R. Herod; David Kealy; Emma Warner; Donald P. King; Tobias J. Tuthill; David J. Rowlands; Nicola J. Stonehouse","Ali Mostafa Anwar; Saif M. Khodary","G. Zhang; S. Pomplun; A. R. Loftis; A. Loas; B. L. Pentelute","Bishajit Sarkar; Md. Asad Ullah; Fatema Tuz Johora; Masuma Afrin Taniya; Yusha Araf","Ivan Mercurio; Vincenzo Tragni; Francesco Busco; Anna De Grassi; Ciro Leonardo Pierri","Gozde S. Demirer; Huan Zhang; Natalie S. Goh; Roger Chang; Markita P. Landry","Simon P Ripperger; Sebastian Stockmaier; Gerald G Carter","Sean Ekins; Peter B. Madrid","Jamie A. Kelly; Jonathan D. Dinman","Ivan S. Kholodilov; Alexander G. Litov; Alexander S. Klimentov; Oxana A. Belova; Alexandra E. Polienko; Nikolai A. Nikitin; Alexey M. Shchetinin; Anna Y. Ivannikova; Lesley Bell-Sakyi; Alexander S. Yakovlev; Sergey V. Bugmyrin; Liubov A. Bespyatova; Larissa V. Gmyl; Svetlana V. Luchinina; Anatoly P. Gmyl; Vladimir A. Gushchin; Galina G. Karganova","Hayden C. Metsky; Catherine A. Freije; Tinna-Solveig F. Kosoko-Thoroddsen; Pardis C. Sabeti; Cameron Myhrvold","Anna Shcherbina; Darrell O. Ricke; Nelson Chiu","Weishan Huang; Sabrina Solouki; Chavez Carter; Song-Guo Zheng; Avery August","Shuvam Banerjee; Shrinjana Dhar; Sandip Bhattacharjee; Pritha Bhattacharjee","Meehyun Ko; So Young Chang; Soo Young Byun; Inhee Choi; Anne-Laure Pham Hung d\u2019Alexandry d\u2019Orengiani; David Shum; Ji-Young Min; Marc P. Windisch","Julianne R Brown; Denise O\u2019Sullivan; Rui PA Pereira; Alexandra S Whale; Eloise Busby; Jim Huggett; Kathryn Harris","Kevin W. Graepel; Maria L. Agostini; Xiaotao Lu; Nicole R. Sexton; Mark R. Denison","Marina Voinson; Alexandra Alvergne; Sylvain Billiard; Charline Smadi","Zhenming Jin; Yao Zhao; Yuan Sun; Bing Zhang; Haofeng Wang; Yan Wu; Yan Zhu; Chen Zhu; Tianyu Hu; Xiaoyu Du; Yinkai Duan; Jing Yu; Xiaobao Yang; Xiuna Yang; Xiang Liu; Luke W. Guddat; Gengfu Xiao; Leike Zhang; Haitao Yang; Zihe Rao","Yujun Li; Haimin Wang; Xiaojuan Tang; Danting Ma; Chengzhi Du; Yifei Wang; Hong Pan; Qing Zou; Jie Zheng; Liangde Xu; Michael Farzan; Guocai Zhong","Brandi Williamson; Friederike Feldmann; Benjamin Schwarz; Kimberly Meade-White; Danielle Porter; Jonathan Schulz; Neeltje van Doremalen; Ian Leighton; Claude Kwe Yinda; Lizzette Perez-Perez; Atsushi Okumura; Jamie Lovaglio; Patrick Hanley; Greg Saturday; Catharine Bosio; Sarah Anzick; Kent Barbian; Tomas Chilar; Craig Martens; Dana Scott; Vincent Munster; Emmie de Wit","Luis G. Nardin; Craig R. Miller; Benjamin J. Ridenhour; Stephen M. Krone; Paul Joyce; Bert O. Baumgaertner","Ernesto Estrada","Adil Doganay Duru; Renhua Sun; Eva B. Allerbring; Jesseka Chadderton; Nadir Kadri; Xiao Han; Hannes Uchtenhagen; Chaithanya Madhurantakam; Sara Pellegrino; Tatyana Sandalova; Per-\u00c5ke Nygren; Stephen J. Turner; Adnane Achour","Timothy A. Dinh; Ramja Sritharan; F. Donelson Smith; Adam B. Francisco; Rosanna K. Ma; Rodica P. Bunaciu; Matt Kanke; Charles G. Danko; Andrew P. Massa; John D. Scott; Praveen Sethupathy","Nanda Kishore Routhu; Sylvain D. Lehoux; Emily A. Rouse; Mehdi R. M. Bidokhti; Leila B. Giron; Alit Anzurez; St Patrick Reid; Mohamed Abdel-Mohsen; Richard D. Cummings; Siddappa N. Byrareddy","Ryan D. Chow; Sidi Chen","Brian D Quinlan; Huihui Mou; Lizhou Zhang; Yan Gao; Wenhui He; Amrita Ojha; Mark S Parcells; Guangxiang Luo; Wenhui Li; Guocai Zhong; Hyeryun Choe; Michael Farzan","Wilton T. Snead; Wade F. Zeno; Grace Kago; Ryan W. Perkins; J Blair Richter; Chi Zhao; Eileen M. Lafer; Jeanne C. Stachowiak","Giancarlo De Luca; Kim Van Kerckhove; Pietro Coletti; Chiara Poletto; Nathalie Bossuyt; Niel Hens; Vittoria Colizza","Neha Jain; Uma Shankar; Prativa Majee; Amit Kumar","Edoardo Milanetti; Mattia Miotto; Lorenzo Di Rienzo; Michele Monti; Giorgio Gosti; Giancarlo Ruocco","Juan Angel Patino-Galindo; Ioan Filip; Mohammed AlQuraishi; Raul Rabadan","Federica Rossetto; Maria Iglesias-Caballero; H. Christoph Liedtke; Ivan Gomez-Mestre; Jose M. Berciano; Gonzalo P\u00e9rez-Su\u00e1rez; Oscar de Paz; Juan E. Echevarr\u00eda; Inmaculada Casas; Javier Juste","Mandy Muller; Britt A. Glaunsinger","Nicholas M. Fountain-Jones; Craig Packer; Maude Jacquot; F. Guillaume Blanchet; Karen Terio; Meggan E. Craft","Carla Mavian; Simone Marini; Mattia Prosperi; Marco Salemi","Sisi Kang; Mei Yang; Zhongsi Hong; Liping Zhang; Zhaoxia Huang; Xiaoxue Chen; Suhua He; Ziliang Zhou; Zhechong Zhou; Qiuyue Chen; Yan Yan; Changsheng Zhang; Hong Shan; Shoudeng Chen","Kenneth Lyon; Luis U. Aguilera; Tatsuya Morisaki; Brian Munsky; Timothy J. Stasevich","Yanpeng Li; Emilia Gordon; Amanda Idle; Eda Altan; M. Alexis Seguin; Marko Estrada; Xutao Deng; Eric Delwart","Visnu Chaparro; Louis-Philippe Leroux; Laia Masvidal; Julie Lorent; Tyson E. Graber; Aude Zimmermann; Guillermo Arango Duque; Albert Descoteaux; Tommy Alain; Ola Larsson; Maritza Jaramillo","Irtiza A. Gilani; Kader K. Oguz; Huseyin Boyaci; Katja Doerschner","Mitchell Holland; Daniel Negr\u00f3n; Shane Mitchell; Nate Dellinger; Mychal Ivancich; Tyler Barrus; Sterling Thomas; Katharine W. Jennings; Bruce Goodwin; Shanmuga Sozhamannan","Deepak Kumar; Nitin Sharma; Murali Aarthy; Sanjeev Singh; Rajanish Giri","Gurjit S. Randhawa; Maximillian P.M. Soltysiak; Hadi El Roz; Camila P.E. de Souza; Kathleen A. Hill; Lila Kari","Justin D. Walter; Cedric A.J. Hutter; Iwan Zimmermann; Jennifer Earp; Pascal Egloff; Mich\u00e8le Sorgenfrei; Lea M. H\u00fcrlimann; Imre Gonda; Gianmarco Meier; Sille Remm; Sujani Thavarasah; Philippe Plattet; Markus A. Seeger","Alejandro Lopez-Rincon; Alberto Tonda; Lucero Mendoza-Maldonado; Eric Claassen; Johan Garssen; Aletta D. Kraneveld","Joana P. Campolina; Sandra G. Coelho; Anna Luiza Belli; Fernanda S. Machado; Luiz G. R. Pereira; Thierry R. Tomich; Wanessa A. Carvalho; Rodrigo O. S. Silva; Alessandra L. Voorsluys; David V. Jacob; Mariana M. Campos","Joe Pharaon; Chris T. Bauch","Eyal Maori; Yael Garbian; Vered Kunik; Rita Mozes-Koch; Osnat Malka; Haim Kalev; Niv Sabath; Ilan Sela; Sharoni Shafir","Giancarlo Cicconofri; Giovanni Noselli; Antonio DeSimone","Zachary J. Sandler; Michelle N. Vu; Vineet D. Menachery; Bryan C. Mounce","Emma Southall; Michael J. Tildesley; Louise Dyson","Konstans Wells; Rodrigo K. Hamede; Menna E. Jones; Paul A. Hohenlohe; Andrew Storfer; Hamish I. McCallum","Ross D. Overacker; Somdev Banerjee; George F. Neuhaus; Selena Milicevic Sephton; Alexander Herrmann; James A. Strother; Ruth Brack-Werner; Paul R. Blakemore; Sandra Loesgen","Sarah Krieg; Fabian Pott; Laura Eckei; Maud Verheirstraeten; Mareike B\u00fctepage; Barbara Lippok; Christine Goffinet; Bernhard L\u00fcscher; Patricia Verheugd","Enrico Lavezzo; Michele Berselli; Ilaria Frasson; Rosalba Perrone; Giorgio Pal\u00f9; Alessandra R. Brazzale; Sara N. Richter; Stefano Toppo","Lin Li; Ting Sun; Yufei He; Wendong Li; Yubo Fan; Jing Zhang","Zheng Zhang; Sifan Ye; Aiping Wu; Taijiao Jiang; Yousong Peng","Neeltje van Doremalen; Elaine Haddock; Friederike Feldmann; Kimberly Meade-White; Trenton Bushmaker; Robert Fischer; Atsushi Okumura; Patrick Hanley; Greg Saturday; Nick Edwards; Madeleine Clark; Teresa Lambe; Sarah Gilbert; Vincent Munster","Jatin Nandania; Gopal Peddinti; Alberto Pessia; Meri Kokkonen; Vidya Velagapudi","Carolina Corr\u00eaa Giron; Aatto Laaksonen; Fernando L. Barroso da Silva","Ke Wang; Wei Chen; Yu-Sen Zhou; Jian-Qi Lian; Zheng Zhang; Peng Du; Li Gong; Yang Zhang; Hong-Yong Cui; Jie-Jie Geng; Bin Wang; Xiu-Xuan Sun; Chun-Fu Wang; Xu Yang; Peng Lin; Yong-Qiang Deng; Ding Wei; Xiang-Min Yang; Yu-Meng Zhu; Kui Zhang; Zhao-Hui Zheng; Jin-Lin Miao; Ting Guo; Ying Shi; Jun Zhang; Ling Fu; Qing-Yi Wang; Huijie Bian; Ping Zhu; Zhi-Nan Chen","Arlin Stoltzfus; Ryan W. Norris","Reham M. Elhassan; Nagla M. Alsony; Khadeejah M. Othman; Duaa T. Izz-Aldin; Tamadour A. Alhaj; Abdelrahman A. Ali; Lena A. Abashir; Omar H. Ahmed; Mohammed A. Hassan","Yichun Wang; Usha Kadiyala; Zhibei Qu; Paolo Elvati; Christopher Altheim; Nicholas A. Kotov; Angela Violi; J. Scott VanEpps","Saima Sultana Tithi; Roderick V. Jensen; Liqing Zhang","Yujuan Xiong; Zhen-Zhen Li; Qizhen Zhuang; Yan Chao; Fei Li; Yi-Yuan Ge; Yi Wang; Pei-Feng Ke; Xianzhang Huang","Mst Rubaiat Nazneen Akhand; Kazi Faizul Azim; Syeda Farjana Hoque; Mahmuda Akther Moli; Bijit Das Joy; Hafsa Akter; Ibrahim Khalil Afif; Nadim Ahmed; Mahmudul Hasan","Xufang Deng; Yafang Chen; Anna M. Mielech; Matthew Hackbart; Kristina R. Kesely; Robert C. Mettelman; Amornrat O\u2019Brien; Mackenzie E. Chapman; Andrew D. Mesecar; Susan C. Baker","Shannon C. Brady; Stefan Zdraljevic; Karol W. Bisaga; Robyn E. Tanny; Daniel E. Cook; Daehan Lee; Ye Wang; Erik C. Andersen","Wei Lin; Longfei Hu; Yan Zhang; Joshua D. Ooi; Ting Meng; Peng Jin; Xiang Ding; Longkai Peng; Lei Song; Zhou Xiao; Xiang Ao; Xiangcheng Xiao; Qiaoling Zhou; Ping Xiao; Jue Fan; Yong Zhong","Robert N. Kirchdoerfer; Mahesh Bhandari; Olnita Martini; Leigh M. Sewall; Sandhya Bangaru; Kyoung-Jin Yoon; Andrew B. Ward","Felix Gr\u00fcnberger; Robert Kn\u00fcppel; Michael J\u00fcttner; Martin Fenk; Andreas Borst; Robert Reichelt; Winfried Hausner; J\u00f6rg Soppa; Sebastien Ferreira-Cerca; Dina Grohmann","Jennifer N Rauch; Eric Valois; Sabrina C Solley; Friederike Braig; Ryan S Lach; Naomi J Baxter; Kenneth S Kosik; Carolina Arias; Diego Acosta-Alvear; Maxwell Z Wilson","Alexander M. Price; Katharina E. Hayer; Daniel P. Depledge; Angus C. Wilson; Matthew D. Weitzman","Kevin Dick; Kyle K Biggar; James R Green","Jacob Peter Matson; Amy M. House; Gavin D. Grant; Huaitong Wu; Joanna Perez; Jeanette Gowen Cook","P-Y Kobres; JP Chretien; MA Johansson; J Morgan; P-Y Whung; H Mukundan; SY Del Valle; BM Forshey; TM Quandelacy; M Biggerstaff; C Viboud; S Pollett","Fabiana Renzi; Dario Ghersi","Daniel L. Hurdiss; Ieva Drulyte; Yifei Lang; Tatiana M. Shamorkina; Matti F. Pronker; Frank J.M. van Kuppeveld; Joost Snijder; Raoul J. de Groot","David A Wilkinson; Lea Joffrin; Camille Lebarbenchon; Patrick Mavingui","David S. Booth; Heather Szmidt-Middleton; Nicole King","A.J.W. Haasnoot; M.W. Schilham; S.S.M. Kamphuis; P.C.E. Hissink Muller; A. Heiligenhaus; D. Foell; R.A. Ophoff; T.R.D.J. Radstake; A.I. Den Hollander; T.H.C.M. Reinards; S. Hiddingh; N. Schalij-Delfos; E.P.A.H. Hoppenreijs; M.A.J. van Rossum; C. Wouters; R.K. Saurenmann; N. Wulffraat; R. ten Cate; J.H. de Boer; S.L. Pulit; J.J.W. Kuiper","Lara Urban; Andre Holzer; J Jotautas Baronas; Michael Hall; Philipp Braeuninger-Weimer; Michael J Scherm; Daniel J Kunz; Surangi N Perera; Daniel E Martin-Herranz; Edward T Tipper; Susannah J Salter; Maximilian R Stammnitz","J. Darr; M. Lassi; Archana Tomar; R. Gerlini; F. Scheid; M. Hrab\u011b de Angelis; M. Witting; R. Teperino","MS Zinter; CC Dvorak; MY Mayday; K Iwanaga; NP Ly; ME McGarry; GD Church; LE Faricy; CM Rowan; JR Hume; ME Steiner; ED Crawford; C Langelier; K Kalantar; ED Chow; S Miller; K Shimano; A Melton; GA Yanik; A Sapru; JL DeRisi","Ming Wu; Lin Xu; Bo Tan; Di Huang; Meijie Yuan; Chaoyang Ye","Chenyu Li; David N. Debruyne; Julia Spencer; Vidushi Kapoor; Lily Y. Liu; Bo Zhou; Lucie Lee; Rounak Feigelman; Grayson Burdon; Jeffrey Liu; Alejandra Oliva; Adam Borcherding; Hongdong Tan; Alexander E. Urban; Guoying Liu; Zhitong Liu","Joana Damas; Graham M. Hughes; Kathleen C. Keough; Corrie A. Painter; Nicole S. Persky; Marco Corbo; Michael Hiller; Klaus-Peter Koepfli; Andreas R. Pfenning; Huabin Zhao; Diane P. Genereux; Ross Swofford; Katherine S. Pollard; Oliver A. Ryder; Martin T. Nweeia; Kerstin Lindblad-Toh; Emma C. Teeling; Elinor K. Karlsson; Harris A. Lewin","Daniel J Butler; Christopher Mozsary; Cem Meydan; David C Danko; Jonathan Foox; Joel Rosiene; Alon Shaiber; Ebrahim Afshinnekoo; Matthew MacKay; Fritz J Sedlazeck; Nikolay A Ivanov; Maria A Sierra; Diana Pohle; Michael Zeitz; Vijendra Ramlall; Undina Gisladottir; Craig D Westover; Krista Ryon; Benjamin Young; Chandrima Bhattacharya; Phyllis Ruggiero; Bradley W Langhorst; Nathan A Tanner; Justyn Gawrys; Dmitry Meleshko; Dong Xu; Jenny Xiang; Angelika Iftner; Daniela Bezdan; John Sipley; Lin Cong; Arryn Craney; Priya Velu; Ari Melnick; Iman A Hajirasouliha; Thomas Iftner; Mirella Salvatore; Massimo Loda; Lars F Westblade; Shawn Levy; Melissa Cushing; Nicholas P Tatonetti; Marcin Imielinski; Hanna Rennert; Christopher Mason","Robert N. Kirchdoerfer; Andrew B. Ward","Gregor Sturm; Tamas G Szabo; Georgios Fotakis; Marlene Haider; Dietmar Rieder; Zlatko Trajanoski; Francesca Finotello","Yasunori Watanabe; Zachary T. Berndsen; Jayna Raghwani; Gemma E. Seabright; Joel D. Allen; Jason S. McLellan; Ian A. Wilson; Thomas A. Bowden; Andrew B. Ward; Max Crispin","Giovanni Colavizza; Rodrigo Costas; Vincent A. Traag; Nees Jan van Eck; Thed van Leeuwen; Ludo Waltman","Roshan Kumar; Helianthous Verma; Nirjara Singhvi; Utkarsh Sood; Vipin Gupta; Mona Singh; Rashmi Sharma; Princy Hira; Shekhar Nagar; Chandni Talwar; Namita Nayyar; Shailly Anand; Charu Dogra Rawat; Mansi Verma; Ram Kishan Negi; Yogendra Singh; Rup Lal","G\u00e1bor Erd\u0151s; B\u00e1lint M\u00e9sz\u00e1ros; Dana Reichmann; Zsuzsanna Doszt\u00e1nyi","Randall Toy; Pallab Pradhan; Vijayeetha Ramesh; Nelson C. Di Paolo; Blake Lash; Jiaying Liu; Emmeline L. Blanchard; Philip J. Santangelo; Dmitry M. Shayakhmetov; Krishnendu Roy","William Fitzsimmons; Robert J. Woods; John T. McCrone; Andrew Woodman; Jamie J. Arnold; Madhumita Yennawar; Richard Evans; Craig E. Cameron; Adam S. Lauring","Stephen A. Wells","Xiaofeng Xu; Jinlong Bei; Yibo Xuan; Jianyuan Chen; Defu Chen; Stephen C. Barker; Samuel Kelava; Bingjun He; Shan Gao; Ze Chen","Jan Zaucha; Charlotte A. Softley; Michael Sattler; Grzegorz M. Popowicz","Sara Cleemput; Wim Dumon; Vagner Fonseca; Wasim Abdool Karim; Marta Giovanetti; Luiz Carlos Alcantara; Koen Deforche; Tulio de Oliveira","Chunxi Zeng; Xucheng Hou; Jingyue Yan; Chengxiang Zhang; Wenqing Li; Weiyu Zhao; Shi Du; Yizhou Dong","Laura A. Wetzel; Tera C. Levin; Ryan E. Hulett; Daniel Chan; Grant A. King; Reef Aldayafleh; David S. Booth; Monika Abedin Sigg; Nicole King","Nila Roy Choudhury; Gregory Heikel; Maryia Trubitsyna; Peter Kubik; Jakub Stanislaw Nowak; Shaun Webb; Sander Granneman; Christos Spanos; Juri Rappsilber; Alfredo Castello; Gracjan Michlewski","Bo Ram Beck; Bonggun Shin; Yoonjung Choi; Sungsoo Park; Keunsoo Kang","Rodrigo Noriega; Matthew Samore","Yoya Vashi; Vipin Jagrit; Sachin Kumar","Cl\u00e1udia Pereira; Rita M. Reis; Jos\u00e9 B. Gama; Dhanya K. Cheerambathur; Ana X. Carvalho; Reto Gassmann","Jun-Gu Kang; Kyeongseok Jeon; Hooncheol Choi; Yuri Kim; Hong-Il Kim; Hyo-Jin Ro; Yong Bok Seo; Jua Shin; Junho Chung; Yoon Kyung Jeon; Yang Soo Kim; Keun Hwa Lee; Nam-Hyuk Cho","Sangeun Jeon; Meehyun Ko; Jihye Lee; Inhee Choi; Soo Young Byun; Soonju Park; David Shum; Seungtaek Kim","Monica Rosas Lemus; George Minasov; Ludmilla Shuvalova; Nicole L. Inniss; Olga Kiryukhina; Grant Wiersum; Youngchang Kim; Robert Jedrzejczak; Michael Enders; Lukasz Jaroszewski; Adam Godzik; Andrzej Joachimiak; Karla J Satchell","Sanchita Bhadra; Andre C Maranhao; Andrew D Ellington","Phillip Davis; John Bagnoli; David Yarmosh; Alan Shteyman; Lance Presser; Sharon Altmann; Shelton Bradrick; Joseph A. Russell","Corey M Peak; Lauren M Childs; Yonatan H Grad; Caroline O Buckee","Asiedu Ebenezer","Joost Snijder; Andrew J. Borst; Annie Dosey; Alexandra C. Walls; Anika Burrell; Vijay S. Reddy; Justin M. Kollman; David Veesler","Lucia de Noronha; Camila Zanluca; Marion Burger; Andreia Akemi Suzukawa; Marina Azevedo; Patricia Z. Rebutini; Iolanda Maria Novadzki; Laurina Setsuko Tanabe; Mayra Marinho Presibella; Claudia Nunes Duarte dos Santos","Salman L. Butt; Eric C. Erwood; Jian Zhang; Holly S. Sellers; Kelsey Young; Kevin K. Lahmers; James B. Stanton","Naoto Hori; Natalia A. Denesyuk; D. Thirumalai","Duc Duy Nguyen; Kaifu Gao; Jiahui Chen; Rui Wang; Guo-Wei Wei","Lyudmila Kovalchuke; Eugene V. Mosharov; Oren A. Levy; Lloyd A. Greene","Joan C Smith; Jason Meyer Sheltzer","Morgan Chabanon; Padmini Rangamani","Jacob E. Choby; Hanna B. Buechi; Allison J. Farrand; Eric P. Skaar; Matthew F. Barber","Lin WANG; Xiang Li","David E. Ochayon; Ayad Ali; Pablo C. Alarcon; Durga Krishnamurthy; Leah C. Kottyan; Michael Borchers; Stephen N. Waggoner","Chin-Yi Chu; Xing Qiu; Matthew N. McCall; Lu Wang; Anthony Corbett; Jeanne Holden-Wiltse; Christopher Slaunwhite; Qian Wang; Christopher Anderson; Alex Grier; Steven R. Gill; Gloria S. Pryhuber; Ann R. Falsey; David J. Topham; Mary T. Caserta; Edward E. Walsh; Thomas J Mariani","Matteo De Chiara; Benjamin Barr\u00e9; Karl Persson; Amadi Onyetuga Chioma; Agurtzane Irizar; Joseph Schacherer; Jonas Warringer; Gianni Liti","Gary R. Whittaker; Nicole M. Andr\u00e9; Jean Kaoru Millet","Sarah F. McGough; Michael A. Johansson; Marc Lipsitch; Nicolas A. Menzies","Oliver C. Grant; David Montgomery; Keigo Ito; Robert J. Woods","Yu Jin Jung; Gun-Soo Park; Jun Hye Moon; Keunbon Ku; Seung-Hwa Beak; Seil Kim; Edmond Changkyun Park; Daeui Park; Jong-Hwan Lee; Cheol Woo Byeon; Joong Jin Lee; Jin-Soo Maeng; Seong Jun Kim; Seung Il Kim; Bum-Tae Kim; Min Jun Lee; Hong Gi Kim","Jos\u00e9 L. Mart\u00ednez; Francesca Arnoldi; Elisabeth M. Schraner; Catherine Eichwald; Daniela Silva-Ayala; Eunjoo Lee; Elizabeth Sztul; \u00d3scar R. Burrone; Susana L\u00f3pez; Carlos F. Arias","Qian Guo; Mo Li; Chunhui Wang; Peihong Wang; Zhencheng Fang; Jie tan; Shufang Wu; Yonghong Xiao; Huaiqiu Zhu","Georgia M. Cook; Katherine Brown; Krzysztof Franaszek; Nathan A. Moore; Stuart G. Siddell; Ian Brierley; Andrew E. Firth; Nerea Irigoyen","Jrhau Lung; Yu-Shih Lin; Yao-Hsu Yang; Yu-Lun Chou; Geng-He Chang; Ming-Shao Tsai; Cheng-Ming Hsu; Reming-Albert Yeh; Li-Hsin Shu; Yu-Ching Cheng; Hung Te Liu; Ching-Yuan Wu","Pamela C. De La Cruz-Rivera; Mohammed Kanchwala; Hanquan Liang; Ashwani Kumar; Lin-Fa Wang; Chao Xing; John W. Schoggins","B. van der Hee; O. Madsen; H. Smidt; J.M. Wells","Xiaoqiang Chai; Longfei Hu; Yan Zhang; Weiyu Han; Zhou Lu; Aiwu Ke; Jian Zhou; Guoming Shi; Nan Fang; Jia Fan; Jiabin Cai; Jue Fan; Fei Lan","Jessica E. Manning; Jennifer A. Bohl; Sreyngim Lay; Sophana Chea; Ly Sovann; Yi Sengdoeurn; Seng Heng; Chan Vuthy; Katrina Kalantar; Vida Ahyong; Michelle Tan; Jonathan Sheu; Cristina M. Tato; Joseph L. DeRisi; Laurence Baril; Veasna Duong; Philippe Dussart; Erik A. Karlsson","Syed Mohammad Lokman; Md. Rasheduzzaman; Asma Salauddin; Rocktim Barua; Afsana Yeasmin Tanzina; Meheadi Hasan Rumi; Md. Imran Hossain; Amam Zonaed Siddiki; Adnan Mannan; Md. Mahbub Hasan","Denisa Bojkova; Jake E. McGreig; Katie-May McLaughlin; Stuart G. Masterson; Marek Widera; Verena Kr\u00e4hling; Sandra Ciesek; Mark N. Wass; Martin Michaelis; Jindrich Cinatl","Maciej F Boni; Philippe Lemey; Xiaowei Jiang; Tommy Tsan-Yuk Lam; Blair Perry; Todd Castoe; Andrew Rambaut; David L Robertson","Nishi Kumari; Ayush Upadhyay; Kishan Kalia; Rakesh Kumar; Kanika Tuteja; Priyanka Rani Paul; Eugenia Covernton; Tina Sharma; Vinod Scaria; Anshu Bhardwaj","Hulda R. Jonsdottir; Sabrina Marti; Dirk Geerts; Regulo Rodriguez; Volker Thiel; Ronald Dijkman","Giovanni Franzo; Mattia Cecchinato; Giovanni Tosi; Laura Fiorentini; Francesca Faccin; Claudia Maria Tucciarone; Tiziana Trogu; Ilaria Barbieri; Paola Massi; Ana Moreno","Emily K. Mantlo; Natalya Bukreyeva; Junki Maruyama; Slobodan Paessler; Cheng Huang","Zehua Zeng; Zhi Luo; Hongwu Du","Marcos Arribas Perez; Oscar H. Moriones; Neus G. Bast\u00fas; Victor Puntes; Andrew Nelson; Paul A. Beales","Gun-Soo Park; Keunbon Ku; Seung-Hwa Beak; Seong Jun Kim; Seung Il Kim; Bum-Tae Kim; Jin-Soo Maeng","Kun Yao; Xufeng Fu; Du Xing; Yan Li; Bing Han; Chen Zexi; Shanshan Yang; Ran Wei; Jiaqi Zhou; Qinghua Cui","KaLynn Harlow; Christina R. Ferreira; Tiago J.P. Sobreira; Theresa Casey; Kara Stewart","Zhaozhong Zhu; Chao-Ting Xiao; Yunshi Fan; Zena Cai; Congyu Lu; Gaihua Zhang; Taijiao Jiang; Yongjun Tan; Yousong Peng","Longxian Lv; Gaolei Li; Jinhui Chen; Xinle Liang; Yudong Li","Sujin Lee; Minh Trang Nguyen; Michael G. Currier; Joe B. Jenkins; Elizabeth A. Strobert; Adriana E. Kajon; Ranjna Madan-Lala; Yury A. Bochkov; James E. Gern; Krishnendu Roy; Xiaoyan Lu; Dean D. Erdman; Paul Spearman; Martin L. Moore","Shi Zhao; Salihu S. Musa; Hao Fu; Daihai He; Jing Qin","Susmita Roy","Hema Kothari; Corey M. Williams; Chantel McSkimming; Mythili Vigneshwar; Eli R. Zunder; Coleen A. McNamara","James T. VanLeuven; Benjamin J. Ridenhour; Craig R. Miller; Tanya A. Miura","Quan Li; Zanxia Cao; Proton Rahman","Matthias T\u00e4ufer","Valeria Lulla; Andrew E. Firth","Amrita Banerjee; Dipannita Santra; Smarajit Maiti","Saba Al Heialy; Mahmood Hachim; Abiola Senok; Ahmad Abou Tayoun; Rifat Hamoudi; Alawi Alsheikh-Ali; Qutayba Hamid","Eric W. Stawiski; Devan Diwanji; Kushal Suryamohan; Ravi Gupta; Frederic A. Fellouse; J. Fah Sathirapongsasuti; Jiang Liu; Ying-Ping Jiang; Aakrosh Ratan; Monika Mis; Devi Santhosh; Sneha Somasekar; Sangeetha Mohan; Sameer Phalke; Boney Kuriakose; Aju Antony; Jagath R. Junutula; Stephan C. Schuster; Natalia Jura; Somasekar Seshagiri","Sanchita Bhadra; Timothy E Riedel; Simren Lakhotia; Nicholas D Tran; Andrew D Ellington","Arun Sharma; Gustavo Garcia; Vaithilingaraja Arumugaswami; Clive N Svendsen","St\u00e9phane Aris-Brosou; Louis Parent; Neke Ibeh","Hayden C. Metsky; Katherine J. Siddle; Adrianne Gladden-Young; James Qu; David K. Yang; Patrick Brehio; Andrew Goldfarb; Anne Piantadosi; Shirlee Wohl; Amber Carter; Aaron E. Lin; Kayla G. Barnes; Damien C. Tully; Bj\u00f6rn Corleis; Scott Hennigan; Giselle Barbosa-Lima; Yasmine R. Vieira; Lauren M. Paul; Amanda L. Tan; Kimberly F. Garcia; Leda A. Parham; Ikponmwonsa Odia; Philomena Eromon; Onikepe A. Folarin; Augustine Goba; Etienne Simon-Lori\u00e8re; Lisa Hensley; Angel Balmaseda; Eva Harris; Douglas Kwon; Todd M. Allen; Jonathan A. Runstadler; Sandra Smole; Fernando A. Bozza; Thiago M. L. Souza; Sharon Isern; Scott F. Michael; Ivette Lorenzana; Lee Gehrke; Irene Bosch; Gregory Ebel; Donald Grant; Christian Happi; Daniel J. Park; Andreas Gnirke; Pardis C. Sabeti; Christian B. Matranga","KV Parag; CA Donnelly","Hongbo Liu; Fei Ye; Qi Sun; Hao Liang; Chunmei Li; Roujian Lu; Baoying Huang; Wenjie Tan; Luhua Lai","Nardus Mollentze; Daniel G. Streicker; Pablo R. Murcia; Katie Hampson; Roman Biek","Alexa C. Robitaille; Elise Caron; Nicolas Zucchini; Esp\u00e9rance Mukawera; Damien Adam; M\u00e9lissa K. Mariani; Ana\u00efs G\u00e9linas; Audray Fortin; Emmanuelle Brochiero; Nathalie Grandvaux","Longlong Si; Haiqing Bai; Melissa Rodas; Wuji Cao; Crystal Yur Oh; Amanda Jiang; Atiq Nurani; Danni Y Zhu; Girija Goyal; Sarah Gilpin; Rachelle Prantil-Baun; Donald E. Ingber","Pan Shen; Qijian Zheng; Aishi Xu; Junhua Liu; Yushan Hou; Chao Gao; Chunyan Tian; Fuchu He; Dong Yang","C I Mendes; E Lizarazo; M P Machado; D N Silva; A Tami; M Ramirez; N Couto; J W A Rossen; J A Carri\u00e7o","Ryo Imai; Makoto Nagai; Shoichi Sakaguchi; Tsuneyuki Masuda; Moegi Kuroda; Mami Oba; Yukie Katayama; Yuki Naoi; Shinobu Tsuchiaka; Tsutomu Omatsu; Hiroshi Yamazato; Shinji Makino; Tetsuya Mizutani","Lopez-Rincon Alejandro; Martinez-Archundia Marlet; Martinez-Ruiz Gustavo Ulises; Tonda Alberto","S Pooransingh; S Mohammed; K Melville; C Mohammed; M Mohammed; R Mohammed; W Mootoo; M Morris; D Motilal; B Bhagwandeen; I Dialsingh","Mpumelelo Casper Rikhotso; Ronewa Khumela; Jean Pierre Kabue; Afsatou Ndama Traor\u00e9; Natasha Potgieter","Allen Wang; Joshua A Chiou; Olivier B Poirion; Justin Buchanan; Michael J Valdez; Jamie M Verheyden; Xiaomeng Hou; Minzhe Guo; Jacklyn M Newsome; Parul Kudtarkar; Dina A Faddah; Kai Zhang; Randee E Young; Justinn Barr; Ravi Misra; Heidie Huyck; Lisa Rogers; Cory Poole; Jeffery A Whitsett; Gloria Pryhuber; Yan Xu; Kyle J Gaulton; Sebastian Preissl; Xin Sun","Ali Punjani; Haowei Zhang; David J. Fleet","Sanchita Bhadra; Timothy E. Riedel; Miguel A. Salda\u00f1a; Shivanand Hegde; Nicole Pederson; Grant L. Hughes; Andrew D. Ellington","Satria P. Sajuthi; Peter DeFord; Nathan D. Jackson; Michael T. Montgomery; Jamie L. Everman; Cydney L. Rios; Elmar Pruesse; James D. Nolin; Elizabeth G. Plender; Michael E. Wechsler; Angel CY Mak; Celeste Eng; Sandra Salazar; Vivian Medina; Eric M. Wohlford; Scott Huntsman; Deborah A. Nickerson; Soren Germer; Michael C. Zody; Gon\u00e7alo Abecasis; Hyun Min Kang; Kenneth M. Rice; Rajesh Kumar; Sam Oh; Jose Rodriguez-Santana; Esteban G. Burchard; Max A. Seibold","Sean M. Boyles; Carla N. Mavian; Esteban Finol; Maria Ukhanova; Caroline J. Stephenson; Gabriela Hamerlinck; Seokyoung Kang; Caleb Baumgartner; Mary Geesey; Israel Stinton; Kate Williams; Derrick K. Mathias; Mattia Prosperi; Volker Mai; Marco Salemi; Eva A. Buckner; John A. Lednicky; Adam R. Rivers; Rhoel R. Dinglasan","Huibin Lv; Nicholas C. Wu; Owen Tak-Yin Tsang; Meng Yuan; Ranawaka A. P. M. Perera; Wai Shing Leung; Ray T. Y. So; Jacky Man Chun Chan; Garrick K. Yip; Thomas Shiu Hong Chik; Yiquan Wang; Chris Yau Chung Choi; Yihan Lin; Wilson W. Ng; Jincun Zhao; Leo L. M. Poon; J. S. Malik Peiris; Ian A. Wilson; Chris K. P. Mok","Kenneth N. Hass; Mengdi Bao; Qian He; Myeongkee Park; Peiwu Qin; Ke Du","Amir Saberi; Anastasia A. Gulyaeva; John L. Brubacher; Phillip A. Newmark; Alexander E. Gorbalenya","Shu Mei Teo; Danny Mok; Kym Pham; Merci Kusel; Michael Serralha; Niamh Troy; Barbara J Holt; Belinda J Hales; Michael L Walker; Elysia Hollams; Yury H Bochkov; Kristine Grindle; Sebastian L Johnston; James E Gern; Peter D Sly; Patrick G Holt; Kathryn E Holt; Michael Inouye","Clayton M. Carey; Sarah E. Apple; Zoe A. Hilbert; Michael S. Kay; Nels C. Elde","Yongjun Tan; Theresa Schneider; Matthew Leong; L Aravind; Dapeng Zhang","Jie Sun; Nick Goplen","Alexander J. Austin; James D. J. Gilbert","Boris Krichel; Sven Falke; Rolf Hilgenfeld; Lars Redecke; Charlotte Uetrecht","Mehdi Baratchian; Jeff McManus; Mike Berk; Fumihiko Nakamura; Serpil Erzurum; Sanjay Mukhopadhyay; Judy Drazba; John Peterson; Ben Gaston; Nima Sharifi","Sourav Haldar; Kenta Okamoto; Peter Kasson","Bing-peng Lin; Mei Zhong; Hong-bin Gao; Kai-bin Wu; Ming-xiao Liu; Chang Liu; Xin-hong Wang; Jian-ming Chen; Learn-han Lee; Cui-ling Qi; Lin-hu Ge; Li-jing Wang","Sanchita Bhadra; Miguel A. Salda\u00f1a; Hannah Grace Han; Grant L. Hughes; Andrew D. Ellington","Miyssa I. Abdelmageed; Abdelrahman H. Abdelmoneim; Mujahed I. Mustafa; Nafisa M. Elfadol; Naseem S. Murshed; Shaza W. Shantier; Abdelrafie M. Makhawi","Tommy Tsan-Yuk Lam; Marcus Ho-Hin Shum; Hua-Chen Zhu; Yi-Gang Tong; Xue-Bing Ni; Yun-Shi Liao; Wei Wei; William Yiu-Man Cheung; Wen-Juan Li; Lian-Feng Li; Gabriel M Leung; Edward C. Holmes; Yan-Ling Hu; Yi Guan","Amrita Roy; Liangzhong Lim; Shagun Srivastava; Jianxing Song","Thomas Desautels; Adam Zemla; Edmond Lau; Magdalena Franco; Daniel Faissol","Julien Riou; Christian L. Althaus","Camilla T O Benfield; Farrell MacKenzie; Markus Ritzefeld; Michela Mazzon; Stuart Weston; Edward Tate; Boon Han Teo; Sarah E Smith; Paul Kellam; Edward C Holmes; Mark Marsh","Susanna K. P. Lau; Rachel Y. Y. Fan; Hayes K. H. Luk; Longchao Zhu; Joshua Fung; Kenneth S. M. Li; Emily Y.M. Wong; Syed Shakeel Ahmed; Jasper F. W. Chan; Raven K. H. Kok; Kwok-Hung Chan; Ulrich Wernery; Kwok-Yung Yuen; Patrick C. Y. Woo","Lucile Marescot; Mathias Franz; Sarah Benhaiem; Heribert Hofer; Marion East; Stephanie Kramer-Schadt","Azeem Mehmood Butt; Izza Nasrullah; Raheel Qamar; Yigang Tong","Pan Liu; Michael Ryczko; Xinfang Xie; Aftab Taiyab; Heather Sheardown; Susan E Quaggin; Jing Jin","Meytal Galilee; Akram Alian","Chad N. Brocker; Donghwan Kim; Tisha Melia; Kritika Karri; Thomas J. Velenosi; Shogo Takahashi; Jessica A. Bonzo; David J. Waxman; Frank J. Gonzalez","J. Daniel Kelly; Lee Worden; Rae Wannier; Nicole A. Hoff; Patrick Mukadi; Cyrus Sinai; Sarah Ackley; Xianyun Chen; Daozhou Gao; Bernice Selo; Mathais Mossoko; Emile Okitolonda-Wemakoy; Eugene T. Richardson; George W. Rutherford; Thomas M. Lietman; Jean Jacques Muyembe-Tamfum; Anne W. Rimoin; Travis C. Porco","Aiping Wu; Peihua Niu; Lulan Wang; Hangyu Zhou; Xiang Zhao; Wenling Wang; Jingfeng Wang; Chengyang Ji; Xiao Ding; Xianyue Wang; Roujian Lu; Sarah Gold; Saba Aliyari; Shilei Zhang; Ellee Vikram; Angela Zou; Emily Lenh; Janet Chen; Fei Ye; Na Han; Yousong Peng; Haitao Guo; Guizhen Wu; Taijiao Jiang; Wenjie Tan; Genhong Cheng","Anton Nekrutenko; Sergei L Kosakovsky Pond","Leili Zhang; David R. Bell; Binquan Luan; Ruhong Zhou","Andrew C Nelson; Benjamin Auch; Matthew Schomaker; Daryl M Gohl; Patrick Grady; Darrell Johnson; Robyn Kincaid; Kylene E Karnuth; Jerry Daniel; Jessica K Fiege; Elizabeth J Fay; Tyler Bold; Ryan A Langlois; Kenneth B Beckman; Sophia Yohe","Colline Lapina; Mathieu Rodic; Denis Peschanski; SALMA MESMOUDI","D. Molodenskiy; H. Mertens; D. Svergun","Natali Ozber; Paolo Margaria; Charles T. Anderson; Massimo Turina; Cristina Rosa","Zhenming Jin; Xiaoyu Du; Yechun Xu; Yongqiang Deng; Meiqin Liu; Yao Zhao; Bing Zhang; Xiaofeng Li; Leike Zhang; Chao Peng; Yinkai Duan; Jing Yu; Lin Wang; Kailin Yang; Fengjiang Liu; Rendi Jiang; Xinglou Yang; Tian You; Xiaoce Liu; Xiuna Yang; Fang Bai; Hong Liu; Xiang Liu; Luke W. Guddat; Wenqing Xu; Gengfu Xiao; Chengfeng Qin; Zhengli Shi; Hualiang Jiang; Zihe Rao; Haitao Yang","T. Kuhn; T. Kaufmann; N.T. Doan; L.T. Westlye; J. Jones; R.A. Nunez; S.Y. Bookheimer; E.J. Singer; C.H. Hinkin; A.D. Thames","Ethan Fast; Russ B. Altman; Binbin Chen","Dannielle Wellington; Zixi Yin; Liwei Zhang; Jessica Forbester; Kerry Kite; Henry Laurenson-Schafer; Shokouh Makvandi-Nejad; Boquan Jin; Emma Bowes; Krishnageetha Manoharan; David Maldonado-Perez; Clare Verill; Ian Humphreys; Tao Dong","Masaki Tagashira","Thanasis Vergoulis; Ilias Kanellos; Serafeim Chatzopoulos; Danae Pla Karidi; Theodore Dalamagas","William Rodriguez; Aman Srivastav; Mandy Muller","Haixia Su; Sheng Yao; Wenfeng Zhao; Minjun Li; Jia Liu; Weijuan Shang; Hang Xie; Changqiang Ke; Meina Gao; Kunqian Yu; Hong Liu; Jingshan Shen; Wei Tang; Leike Zhang; Jianping Zuo; Hualiang Jiang; Fang Bai; Yan Wu; Yang Ye; Yechun Xu","Isobel Routledge; Shengjie Lai; Katherine E Battle; Azra C Ghani; Manuel Gomez-Rodriguez; Kyle B Gustafson; Swapnil Mishra; Joshua L Proctor; Andrew J Tatem; Zhongjie Li; Samir Bhatt","Simon P. Jochems; Fernando Marcon; Beatriz F. Carniel; Mark Holloway; Elena Mitsi; Emma Smith; Jenna F. Gritzfeld; Carla Sol\u00f3rzano; Jes\u00fas Rein\u00e9; Sherin Pojar; Elissavet Nikolaou; Esther L. German; Angie Hyder-Wright; Helen Hill; Caz Hales; Wouter A.A de Steenhuijsen Piters; Debby Bogaert; Hugh Adler; Seher Zaidi; Victoria Connor; Jamie Rylance; Helder I. Nakaya; Daniela M. Ferreira","Amal Alex; Valentina Piano; Soumitra Polley; Marchel Stuiver; Stephanie Voss; Giuseppe Ciossani; Katharina Overlack; Beate Voss; Sabine Wohlgemuth; Arsen Petrovic; Yao-Wen Wu; Philipp Selenko; Andrea Musacchio; Stefano Maffini","Derrick Deming; Karen Lee; Tracey McSherry; Ronnie R. Wei; Tim Edmunds; Scott C. Garman","Alex Rodriguez-Palacios; Mathew Conger; Fabio Cominelli","Tristan Bitard-Feildel; Isabelle Callebaut","Dandan Liu; Lei Ge; Qing Wang; Jiarui Su; Xingxiang Chen; Chunfeng Wang; Kehe Huang","Rajanish Giri; Taniya Bhardwaj; Meenakshi Shegane; Bhuvaneshwari R. Gehi; Prateek Kumar; Kundlik Gadhave","Hualong Xiong; Yangtao Wu; Jiali Cao; Ren Yang; Jian Ma; Xiaoyang Qiao; Xiangyang Yao; Baohui Zhang; Yali Zhang; Wangheng Hou; Yang Shi; Jingjing Xu; Liang Zhang; Shaojuan Wang; Baorong Fu; Ting Yang; Shengxiang Ge; Jun Zhang; Quan Yuan; Baoying Huang; Zhiyong Li; Tianying Zhang; Ningshao Xia","Daniel J. Becker; Kelly A. Speer; Alexis M. Brown; M. Brock Fenton; Alex D. Washburne; Sonia Altizer; Daniel G. Streicker; Raina K. Plowright; Vladimir E. Chizhikov; Nancy B. Simmons; Dmitriy V. Volokhov","David N. Frick; Rajdeep S. Virdi; Nemanja Vuksanovic; Narayan Dahal; Nicholas R Silvaggi","Lei Han; Xiaoyu Wei; Chuanyu Liu; Giacomo Volpe; Zhifeng Wang; Taotao Pan; Yue Yuan; Ying Lei; Yiwei Lai; Carl Ward; Yeya Yu; Mingyue Wang; Quan Shi; Tao Wu; Liang Wu; Ya Liu; Chunqing Wang; Yuanhang Zhang; Haixi Sun; Hao Yu; Zhenkun Zhuang; Tingting Tang; Yunting Huang; Haorong Lu; Liqin Xu; Jiangshan Xu; Mengnan Cheng; Yang Liu; Chi Wai Wong; Tao Tan; Weizhi Ji; Patrick H. Maxwell; Huanming Yang; Jian Wang; Shida Zhu; Shiping Liu; Xun Xu; Yong Hou; Miguel A. Esteban; Longqi Liu","Alessia Landi; Muriel Mari; Tobias Wolf; Svenja Kleiser; Christine Gretzmeier; Isabel Wilhelm; Dimitra Kiritsi; Roland Th\u00fcnauer; Roger Geiger; Alexander Nystr\u00f6m; Fulvio Reggiori; Julie Claudinon; Winfried R\u00f6mer","Soon Keong Wee; Suppiah Paramalingam Sivalingam; Eric Peng Huat Yap","Yonggang Zhou; Binqing Fu; Xiaohu Zheng; Dongsheng Wang; Changcheng Zhao; Yingjie qi; Rui Sun; Zhigang Tian; Xiaoling Xu; Haiming Wei","George Taiaroa; Daniel Rawlinson; Leo Featherstone; Miranda Pitt; Leon Caly; Julian Druce; Damian Purcell; Leigh Harty; Thomas Tran; Jason Roberts; Mike Catton; Deborah Williamson; Lachlan Coin; Sebastian Duchene","Chuang Liu; Yang Yang; Yuanzhu Gao; Chenguang Shen; Bin Ju; Congcong Liu; Xian Tang; Jinli Wei; Xiaomin Ma; Weilong Liu; Shuman Xu; Yingxia Liu; Jing Yuan; Jing Wu; Zheng Liu; Zheng Zhang; Peiyi Wang; Lei Liu","Sang Woo Park; David Champredon; Joshua S. Weitz; Jonathan Dushoff","Chentong Li; Jinhu Xu; Jiawei Liu; Yicang Zhou","Bin Fang; Linlin Liu; Xiao Yu; Xiang Li; Guojun Ye; Juan Xu; Ling Zhang; Faxian Zhan; Guiming Liu; Tao Pan; Yilin Shu; Yongzhong Jiang","Allison J Cox; Fillan Grady; Gabriel Velez; Vinit B Mahajan; Polly J Ferguson; Andrew Kitchen; Benjamin W Darbro; Alexander G Bassuk","Halim Maaroufi","Anamika Basu; Anasua Sarkar; Ujjwal Maulik","Andrew D. Davidson; Maia Kavangh Williamson; Sebastian Lewis; Deborah Shoemark; Miles W Carroll; Kate Heesom; Maria Zambon; Joanna Ellis; Phillip A Lewis; Julian A Hiscox; David A Matthews","Xuesen Zhao; Shuangli Zheng; Danying Chen; Mei Zheng; Xinglin Li; Guoli Li; Hanxin Lin; Jinhong Chang; Hui Zeng; Ju-Tao Guo","Robert C. Cope; Joshua V. Ross","Jerome R Lon; Yunmeng Bai; Bingxu Zhong; Fuqiang Cai; Hongli Du","Mohd. Azhar; Rhythm Phutela; Asgar Hussain Ansari; Dipanjali Sinha; Namrata Sharma; Manoj Kumar; Meghali Aich; Saumya Sharma; Khushboo Singhal; Harsha Lad; Pradeep K. Patra; Govind Makharia; Giriraj Ratan Chandak; Debojyoti Chakraborty; Souvik Maiti","Yuan Liu; Zhi Ning; Yu Chen; Ming Guo; Yingle Liu; Nirmal Kumar Gali; Li Sun; Yusen Duan; Jing Cai; Dane Westerdahl; Xinjin Liu; Kin-fai Ho; Haidong Kan; Qingyan Fu; Ke Lan","Monika Litvinukova; Carlos Talavera-Lopez; Henrike Maatz; Daniel Reichart; Catherine L. Worth; Eric L. Lindberg; Masatoshi Kanda; Krzysztof Polanski; Eirini S. Fasouli; Sara Samari; Kenny Roberts; Elizabeth Tuck; Matthias Heinig; Daniel DeLaughter; Barbara McDonough; Hiroko Wakimoto; Joshua M. Gorham; Emily Nadelmann; Krishnaa T. Mahbubani; Kourosh Saeb-Parsy; Giannino Patone; Joseph J Boyle; Hongbo Zhang; Hao Zhang; Anissa Viveiros; Gavin Oudit; Omer Bayraktar; J. G. Seidman; Christine Seidman; Michela Noseda; Norbert Hubner; Sarah A. Teichmann","Ki Hyun Kim; Jinhee Kim; Meehyun Ko; June Young Chun; Hyori Kim; Seungtaek Kim; Ji-Young Min; Wan Beom Park; Myoung-don Oh; Junho Chung","Sebastian J. Schreiber; Ruian Ke; Claude Loverdo; Miran Park; Priyanna Ahsan; James O. Lloyd-Smith","Paul R Grant; Melanie A Turner; Gee Yen Shin; Eleni Nastouli; Lisa J Levett","jose-carlos Valle-Casuso; Delphine Gaudaire; Lydie Martin-Faivre; anthony madeline; Patrick Dallemagne; Stephane Pronost; Helene Munier-Lehmann; Stephan Zientara; Pierre-Olivier Vidalain; Aymeric Hans","Cody S. Nelson; Tori Huffman; Eduardo Cisneros de la Rosa; Guanhua Xie; Nathan Vandergrift; Robert F. Pass; Justin Pollara; Sallie R. Permar","Renata C Fleith; Harriet V Mears; Edward Emmott; Stephen C Graham; Daniel S Mansur; Trevor R Sweeney","Maximilian Krause; Adnan M. Niazi; Kornel Labun; Yamila N. Torres Cleuren; Florian S. M\u00fcller; Eivind Valen","Spyridon Megremis; Thomas Walker; Xiaotong He; James O'Sullivan; William E.R. Ollier; Hector Chinoy; Neil Pendleton; Antony Payton; Lynne Hampson; Ian Hampson; Janine Lamb","Ricardo Guimar\u00e3es Amaral; Silvana Vieira Floresta Gomes; \u00c2ngelo Roberto Antoniolli; Maria Claudia dos Santos Luciano; Cl\u00e1udia do \u00d3 Pessoa; Luciana Nalone Andrade; Patr\u00edcia Severino; Geraldo C\u00e9lio Brand\u00e3o; Larissa Mendes Bomfim; Daniel Pereira Bezerra; Jorge Maur\u00edcio David; Adriana Andrade Carvalho","Xin Zeng; Lingfang Li; Jing Lin; Xinlei Li; Bin Liu; Yang Kong; Shunze Zeng; Jianhua Du; Huahong Xiao; Tao Zhang; Shelin Zhang; Jianghai Liu","Vinay S. Mahajan; Faisal Alsufyani; Hamid Mattoo; Ian Rosenberg; Shiv Pillai","Daniel Wrapp; Nianshuang Wang; Kizzmekia S. Corbett; Jory A. Goldsmith; Ching-Lin Hsieh; Olubukola Abiona; Barney S. Graham; Jason S. McLellan","Revital Bronstein; Elizabeth E. Capowski; Sudeep Mehrotra; Alex D. Jansen; Daniel Navarro-Gomez; Mathew Maher; Emily Place; Riccardo Sangermano; Kinga M. Bujakowska; David M. Gamm; Eric A. Pierce","Steffen Jockusch; Chuanjuan Tao; Xiaoxu Li; Thomas K. Anderson; Minchen Chien; Shiv Kumar; James J. Russo; Robert Kirchdoerfer; Jingyue Ju","Niccolo Alfano; Anisha Dayaram; Jan Axtner; Kyriakos Tsangaras; Marie-Louise Kampmann; Azlan Mohamed; Seth Timothy Wong; M. Thomas P. Gilbert; Andreas Wilting; Alex Daivd Greenwood","Francisco Acosta-Reyes; Ritam Neupane; Joachim Frank; Israel S. Fern\u00e1ndez","Marco R. Straus; Tiffany Tang; Alex L. Lai; Annkatrin Flegel; Miya Bidon; Jack H. Freed; Susan Daniel; Gary R. Whittaker","Mushtaq Hussain; Nusrat Jabeen; Anusha Amanullah; Ayesha Ashraf Baig; Basma Aziz; Sanya Shabbir; Fozia Raza","Gal Haimovich; Tsviya Olender; Camila Baez; Jeffrey E Gerst","Leon Fodoulian; Joel Tuberosa; Daniel Rossier; Basile N. Landis; Alan Carleton; Ivan Rodriguez","Rebecca J Rockett; Alicia Arnott; Connie Lam; Rosemarie Sadsad; Verlaine Timms; Karen-Ann Gray; John-Sebastian Eden; Sheryl Le Chang; Mailie Gall; Jenny Draper; Eby Sim; Nathan L Bachmann; Ian Carter; Kerri Basile; Roy Byun; Matthew V O Sullivan; Sharon C-A Chen; Susan Maddocks; Tania C Sorrell; Dominic E Dwyer; Edward C Holmes; Jen Kok; Mikhail Prokopenko; Vitali Sintchenko","Romel Rosales Ramirez; Juan E. Ludert","Charles J Sande; Jacqueline M Waeni; James M Njunge; Martin N Mutunga; Elijah Gicheru; Nelson K Kibinge; Agnes Gwela","Ella N Hartenian; Britt A Glaunsinger","Sandra Smieszek; Bart Przychodzen; Mihael H Polymeropoulos","Maddalena Dilucca; Athanasia Pavlopoulou; Sergio Forcelloni; ANDREA GIANSANTI; Alexandros G Georgakilas","Brandon Malone; Boris Simovski; Clement Moline; Jun Cheng; Marius Gheorghe; Hugues Fontenelle; Ioannis Vardaxis; Simen Tennoe; Jenny-Ann Malmberg; Richard Stratford; Trevor Clancy","Arunachalam Ramaiah; Vaithilingaraja Arumugaswami","Enora Dupas; Bruno Legendre; Val\u00e9rie Olivier; Fran\u00e7oise Poliakoff; Charles Manceau; Amandine Cunty","Angela Bauch; Johann Pellet; Tina Schleicher; Xiao Yu; Andrea Gelemanovi\u0107; Cosimo Cristella; Pieter L. Fraaij; Ozren Polasek; Charles Auffray; Dieter Maier; Marion Koopmans; Menno D. de Jong","Yu-Nong Gong; Kuo-Chien Tsao; Mei-Jen Hsiao; Chung-Guei Huang; Peng-Nien Huang; Po-Wei Huang; Kuo-Ming Lee; Yi-Chun Liu; Shu-Li Yang; Rei-Lin Kuo; Ming-Tsan Liu; Ji-Rong Yang; Cheng-Hsun Chiu; Cheng-Ta Yang; Shin-Ru Shih; Guang-Wu Chen","Shefali Dobhal; Gamze Boluk; Brooke Babler; Michael J. Stulberg; John Rascoe; Mark Nakhla; Toni A. Chapman; Alex B. Crockford; Michael Melzer; Anne M. Alvarez; Mohammad Arif","Carlos Farkas; Francisco Fuentes-Villalobos; Jose Luis Garrido; Jody J Haigh; Mar\u00eda In\u00e9s Barr\u00eda","Matthew S. Faber; James T. Van Leuven; Martina M. Ederer; Yesol Sapozhnikov; Zo\u00eb L. Wilson; Holly A. Wichman; Timothy A. Whitehead; Craig R. Miller","Alisher M Kariev; Michael E Green","Yan Gao; Liming Yan; Yucen Huang; Fengjiang Liu; Yao Zhao; Lin Cao; Tao Wang; Qianqian Sun; Zhenhua Ming; Lianqi Zhang; Ji Ge; Litao Zheng; Ying Zhang; Haofeng Wang; Yan Zhu; Chen Zhu; Tianyu Hu; Tian Hua; Bing Zhang; Xiuna Yang; Jun Li; Haitao Yang; Zhijie Liu; Wenqing Xu; Luke W. Guddat; Quan Wang; Zhiyong Lou; Zihe Rao","Deguo Wang","Hongxin Guan; Youwang Wang; Abdullah F.U.H. Saeed; Jinyu Li; Syed Sajid Jan; Vanja Per\u010dulija; Yu Li; Ping Zhu; Songying Ouyang","Xiaoqiang Huang; Robin Pearce; Yang Zhang","Dongwan Kim; Joo-Yeon Lee; Jeong-Sun Yang; Jun Won Kim; V. Narry Kim; Hyeshik Chang","Courtney Mycroft-West; Dunhao Su; Stefano Elli; Scott Guimond; Gavin Miller; Jeremy Turnbull; Edwin Yates; Marco Guerrini; David Fernig; Marcelo Lima; Mark Skidmore","Manon Laporte; Annelies Stevaert; Valerie Raeymaekers; Talitha Boogaerts; Inga Nehlmeier; Winston Chiu; Mohammed Benkheil; Bart Vanaudenaerde; Stefan P\u00f6hlmann; Lieve Naesens","Zhe Liu; Huanying Zheng; Runyu Yuan; Mingyue Li; Huifang Lin; Jingju Peng; Qianlin Xiong; Jiufeng Sun; Baisheng Li; Jie Wu; Ruben J.G. Hulswit; Thomas A. Bowden; Andrew Rambaut; Nick Loman; Oliver G Pybus; Changwen Ke; Jing Lu","Brian G. Pierce; Zhen-Yong Keck; Ruixue Wang; Patrick Lau; Kyle Garagusi; Khadija Elkholy; Eric A. Toth; Richard A. Urbanowicz; Johnathan D. Guest; Pragati Agnihotri; Melissa C. Kerzic; Alexander Marin; Alexander K. Andrianov; Jonathan K. Ball; Roy A. Mariuzza; Thomas R. Fuerst; Steven K.H. Foung","Xiaotian Tan; Cory Lin; Jie Zhang; Maung Kyaw Khaing Oo; Xudong Fan","Lucie Ke\u0161nerov\u00e1; Olivier Emery; Micha\u00ebl Troilo; Joanito Liberti; Berra Erkosar; Philipp Engel","Warren R. Francis; Gert W\u00f6rheide","Takako I. Jones; Guo-Liang Chew; Pamela Barraza-Flores; Spencer Schreier; Monique Ramirez; Ryan D. Wuebbles; Dean J. Burkin; Robert K. Bradley; Peter L. Jones","R.N. Thompson","Sandeep Kumar; Jinmei Li; Jiwoong Park; Sydney K Hart; Nicholas L Bean; Niki J Song; Ariella Coler-Reilly; Nicholas C Jacobs; Damon T Burrow; Anastasiia Onyshchenko Pendergrass; Tanya H Pierre; India C Bradley; Malini Varadarajan; Jan E Carette; Thijn R Brummelkamp; Roland E Dolle; Timothy R Peterson","William Binning; Aja E. Hogan-Cann; Diana Yae Sakae; Matthew Maksoud; Valeriy Ostapchenko; Mohammed Al-Onaizi; Sara Matovic; Wei-Yang Lu; Marco A. M. Prado; Wataru Inoue; Vania F. Prado","Aaron S. Mendez; Carolin Vogt; Jens Bohne; Britt A. Glaunsinger","Lucia Grenga; Fabrice Gallais; Olivier Pible; Jean-Charles Gaillard; Duarte Gouveia; H\u00e9l\u00e8ne Batina; Niza Bazaline; Sylvie Ruat; Karen Culotta; Guylaine Miotello; St\u00e9phanie Debroas; Marie-Anne Roncato; G\u00e9rard Steinmetz; Charlotte Foissard; Anne Desplan; B\u00e9atrice Alpha-Bazin; Christine Almunia; Fabienne Gas; Laurent Bellanger; Jean Armengaud","Gage Kahl Moreno; Katarina M Braun; Peter J Halfmann; Trent M Prall; Kasen K Riemersma; Amelia K Haj; Joseph Lalli; Kelsey R Florek; Thomas C Friedrich; Yoshihiro Kawaoka; David H O'Connor","M.K. PrasannaKumar; P. Buela Parivallal; C. Manjunath; H.B. Mahesh; Karthik S Narayan; Gopal Venkatesh Babu; K. Priyanka; M.E. Puneeth; K.T. Rangaswamy","Tamina Park; Sang-Yeop Lee; Seil Kim; Mi Jeong Kim; Hong Gi Kim; Sangmi Jun; Seung Il Kim; Bum Tae Kim; Edmond Changkyun Park; Daeui Park","Damian Kao; Alvina G. Lai; Evangelia Stamataki; Silvana Rosic; Nikolaos Konstantinides; Erin Jarvis; Alessia Di Donfrancesco; Natalia Pouchkina-Stantcheva; Marie S\u00e9mon; Marco Grillo; Heather Bruce; Suyash Kumar; Igor Siwanowicz; Andy Le; Andrew Lemire; Michael B. Eisen; Cassandra Extavour; William E. Browne; Carsten Wolff; Michalis Averof; Nipam H. Patel; Peter Sarkies; Anastasios Pavlopoulos; A. Aziz Aboobaker","Hannah K. Frank; David Enard; Scott D. Boyd","Roberto Balbont\u00edn; Nelson Fraz\u00e3o; Isabel Gordo","Wenbin Ji; Yibo Luo; Ejaz Ahmad; Song-Tao Liu","Yiyue Ge; Tingzhong Tian; Suling Huang; Fangping Wan; Jingxin Li; Shuya Li; Hui Yang; Lixiang Hong; Nian Wu; Enming Yuan; Lili Cheng; Yipin Lei; Hantao Shu; Xiaolong Feng; Ziyuan Jiang; Ying Chi; Xiling Guo; Lunbiao Cui; Liang Xiao; Zeng Li; Chunhao Yang; Zehong Miao; Haidong Tang; Ligong Chen; Hainian Zeng; Dan Zhao; Fengcai Zhu; Xiaokun Shen; Jianyang Zeng","Collin D. Heer; Daniel J. Sanderson; Yousef M.O. Alhammad; Mark S. Schmidt; Samuel A.J. Trammell; Stanley Perlman; Michael S. Cohen; Anthony R. Fehr; Charles Brenner","Rui Xiong; Leike Zhang; Shiliang Li; Yuan Sun; Minyi Ding; Yong Wang; Yongliang Zhao; Yan Wu; Weijuan Shang; Xiaming Jiang; Jiwei Shan; Zihao Shen; Yi Tong; Liuxin Xu; Chen Yu; Yingle Liu; Gang Zou; Dimitri Lavillete; Zhenjiang Zhao; Rui Wang; Lili Zhu; Gengfu Xiao; Ke Lan; Honglin Li; Ke Xu","Michael Jay Corley; Christopher Sugai; Michael Schotsaert; Robert E. Schwartz; Lishomwa C Ndhlovu","Christian Ihling; Dirk T\u00e4nzler; Sven Hagemann; Astrid Kehlen; Stefan H\u00fcttelmaier; Andrea Sinz","Luis M. Colon-Perez; Kristen R. Ibanez; Mallory Suarez; Kristin Torroella; Kelly Acuna; Edward Ofori; Yona Levites; David E. Vaillancourt; Todd E. Golde; Paramita Chakrabarty; Marcelo Febo","Akshaya Ramesh; Sara Nakielny; Jennifer Hsu; Mary Kyohere; Oswald Byaruhanga; Charles de Bourcy; Rebecca Egger; Boris Dimitrov; Yun-Fang Juan; Jonathan Sheu; James Wang; Katrina Kalantar; Charles Langelier; Theodore Ruel; Arthur Mpimbaza; Michael R. Wilson; Philip J. Rosenthal; Joseph L. DeRisi","Sarah V Leavitt; Robyn S Lee; Paola Sebastiani; Charles R. Horsburgh; Helen E Jenkins; Laura F White","Smita Gopinath; Myoungjoo V. Kim; Tasfia Rakib; Patrick W. Wong; Michael van Zandt; Natasha A. Barry; Tsuneyasu Kaisho; Andrew L. Goodman; Akiko Iwasaki","Asif Shajahan; Nitin T. Supekar; Anne S. Gleinich; Parastoo Azadi","Zhaozhong Zhu; Yunshi Fan; Zena Cai; Zheng Zhang; Congyu Lu; Taijiao Jiang; Gaihua Zhang; Yousong Peng","H\u00e9ctor Cervera; Silvia Ambr\u00f3s; Guillermo P. Bernet; Guillermo Rodrigo; Santiago F. Elena","Junan Zhu; Kristina Rivera; Dror Baron","Xiaofeng Xu; Haishuo Ji; Zhi Cheng; Xiufeng Jin; Xue Yao; Yanqiang Liu; Qiang Zhao; Tao Zhang; Jishou Ruan; Wenjun Bu; Ze Chen; Shan Gao","Korbinian B\u00f6sl; Aleksandr Ianevski; Thoa T. Than; Petter I. Andersen; Suvi Kuivanen; Mona Teppor; Eva Zusinaite; Uga Dumpis; Astra Vitkauskiene; Rebecca J. Cox; Hannimari Kallio-Kokko; Anders Bergqvist; Tanel Tenson; Valentyn Oksenych; Magnar Bj\u00f8r\u00e5s; Marit W. Anthonsen; David Shum; Mari Kaarb\u00f8; Olli Vapalahti; Marc P. Windisch; Giulio Superti-Furga; Berend Snijder; Denis Kainov; Richard K. Kandasamy","Aravinth Kumar Jayabalan; Diane E. Griffin; Anthony K. L. Leung","Joshua A. Hayward; Mary Tachedjian; Adam Johnson; Tamsin B. Gordon; Jie Cui; Glenn A. Marsh; Michelle L. Baker; Lin-Fa Wang; Gilda Tachedjian","Ping Liu; Jing-Zhe Jiang; Xiu-Feng Wan; Yan Hua; Xiaohu Wang; Fanghui Hou; Jing Chen; Jiejian Zou; Jinping Chen","Hubert Bassene; Sophie Edouard; Georges Diatta; Jean Christophe Lagier; Oleg Mediannikov; Florence Fenollar; Aldiouma Diallo; El Hadji Ba; Didier Raoult; Philippe Parola; Michel Drancourt; Cheikh Sokhna","Michael M. Lutz; Megan P. Worth; Meleana M. Hinchman; John S. L. Parker; Emily D. Ledgerwood","Tristan de Jong; Victor Guryev; Yury M. Moshkin","Ellen D. Zhong; Tristan Bepler; Bonnie Berger; Joseph H Davis","Elizabeth T. Cirulli; Stephen Riffle; Alexandre Bolze; Nicole L. Washington","Lin Lv; Xiaoqing Xie; Qiyu Gong; Ru Feng; Xiaokui Guo; Bing Su; Lei Chen","James Zhu; Jiwoong Kim; Xue Xiao; Yunguan Wang; Danni Luo; Ran Chen; Lin Xu; He Zhang; Guanghua Xiao; John Schoggins; Xiaowei Zhan; Tao Wang; Yang Xie","David E. Gordon; Gwendolyn M. Jang; Mehdi Bouhaddou; Jiewei Xu; Kirsten Obernier; Matthew J O'Meara; Jeffrey Z. Guo; Danielle L. Swaney; Tia A. Tummino; Ruth Huttenhain; Robyn Kaake; Alicia L. Richards; Beril Tutuncuoglu; Helene Foussard; Jyoti Batra; Kelsey Haas; Maya Modak; Minkyu Kim; Paige Haas; Benjamin J. Polacco; Hannes Braberg; Jacqueline M. Fabius; Manon Eckhardt; Margaret Soucheray; Melanie Brewer; Merve Cakir; Michael J. McGregor; Qiongyu Li; Zun Zar Chi Naing; Yuan Zhou; Shiming Peng; Ilsa T. Kirby; James E. Melnyk; John S Chorba; Kevin Lou; Shizhong A. Dai; Wenqi Shen; Ying Shi; Ziyang Zhang; Inigo Barrio-Hernandez; Danish Memon; Claudia Hernandez-Armenta; Christopher J.P. Mathy; Tina Perica; Kala B. Pilla; Sai J. Ganesan; Daniel J. Saltzberg; Rakesh Ramachandran; Xi Liu; Sara B. Rosenthal; Lorenzo Calviello; Srivats Venkataramanan; Yizhu Lin; Stephanie A. Wankowicz; Markus Bohn; Phillip P. Sharp; Raphael Trenker; Janet M. Young; Devin A. Cavero; Joseph Hiatt; Theo Roth; Ujjwal Rathore; Advait Subramanian; Julia Noack; Mathieu Hubert; Ferdinand Roesch; Thomas Vallet; Bj\u00f6rn Meyer; Kris M. White; Lisa Miorin; Oren S. Rosenberg; Kliment A. Verba; David Agard; Melanie Ott; Michael Emerman; Davide Ruggero; Adolfo Garc&amp;iacute-Sastre; Natalia Jura; Mark von Zastrow; Jack Taunton; Olivier Schwartz; Marco Vignuzzi; Christophe d'Enfert; Shaeri Mukherjee; Matt Jacobson; Harmit S. Malik; Danica G Fujimori; Trey Ideker; Charles S Craik; Stephen Floor; James S. Fraser; John Gross; Andrej Sali; Tanja Kortemme; Pedro Beltrao; Kevan Shokat; Brian K. Shoichet; Nevan J. Krogan","Sarvenaz Sarabipour; Benjamin Schwessinger; Fiona N. Mumoki; Aneth D. Mwakilili; Aziz Khan; Humberto J. Debat; Pablo J. S\u00e1ez; Samantha Seah; Tomislav Mestrovic","Priyanka Saha; Arup Kumar Banerjee; Prem Prakash Tripathi; Amit Kumar Srivastava; Upasana Ray","Guang Li; Sharon E. Fox; Brian Summa; Carola Wenk; Aibek Akmatbekov; Jack L. Harbert; Richard S. Vander Heide; J. Quincy Brown","Chang Liu; Ze Chen; Wenyuan Shen; Deshui Yu; Siyu Li; Yue Hu; Haishuo Ji; Wenjun Bu; Qingsong Wang; Shan Gao","D. Gregory Sullens; Kayla Gilley; Kendall Jensen; Melanie J. Sekeres","Zhaoqian Su; Yinghao Wu","Adrian Egli; Nina Goldman; Nicola F. M\u00fcller; Myrta Brunner; Daniel W\u00fcthrich; Sarah Tschudin-Sutter; Emma Hodcroft; Richard Neher; Claudia Saalfrank; James Hadfield; Trevor Bedford; Mohammedyaseen Syedbasha; Thomas Vogel; No\u00e9mie Augustin; Jan Bauer; Nadine Sailer; Nadezhda Amar-Sliwa; Daniela Lang; Helena M.B. Seth-Smith; Annette Blaich; Yvonne Hollenstein; Olivier Dubuis; Michael N\u00e4gele; Andreas Buser; Christian H. Nickel; Nicole Ritz; Andreas Zeller; Tanja Stadler; Manuel Battegay; Rita Schneider-Sliwa","Javier A. Jaimes; Nicole M. Andr\u00e9; Jean K. Millet; Gary R. Whittaker","Fran\u00e7ois Ferron; Humberto Julio Debat; Etienne Decroly; Bruno Canard","Wenjun Zhang; Zeliang Chen; Yi Lu; Zhongmin Guo; Yanhong Qi; Guoling Wang; Jiahai Lu","Ignacio Ortea; Jens-Ole Bock","Erik Procko","Monica Sentmanat; Evguenia Kouranova; Xiaoxia Cui","MJ Morwitzer; A Corona; L Zinzula; E Fanunza; C Nigri; S Distinto; C Vornholt; V Kumar; E Tramontano; SP Reid","Hongzhu Cui; Ziyang Gao; Ming Liu; Senbao Lu; Sun Mo; Winnie Mkandawire; Oleksandr Narykov; Suhas Srinivasan; Dmitry Korkin","Alexander E. Gorbalenya; Susan C. Baker; Ralph S. Baric; Raoul J. de Groot; Christian Drosten; Anastasia A. Gulyaeva; Bart L. Haagmans; Chris Lauber; Andrey M Leontovich; Benjamin W. Neuman; Dmitry Penzar; Stanley Perlman; Leo L.M. Poon; Dmitry Samborskiy; Igor A. Sidorov; Isabel Sola; John Ziebuhr","Valentin Mitin; Lidia Tumanova; Natalia Botnariuc","Chuang Guo; Bin Li; Huan Ma; Xiaofang Wang; Pengfei Cai; Qiaoni Yu; Lin Zhu; Liying Jin; Chen Jiang; Jingwen Fang; Qian Liu; Dandan Zong; Wen Zhang; Yichen Lu; Kun Li; Xuyuan Gao; Binqing Fu; Lianxin Liu; Xiaoling Ma; Jianping Weng; Haiming Wei; Tengchuan Jin; Jun Lin; Kun Qu","Jiao Chen; Jiayu Shang; Jianrong Wang; Yanni Sun","Amrita Roy; Liangzhong Lim; Jianxing Song","Tianmu Chen; Jia Rui; Qiupeng Wang; Zeyu Zhao; Jing-An Cui; Ling Yin","Feng-Biao Biao; Chuan Dong; Hong-Li Hua; Shuo Liu; Hao Luo; Hong-Wan Zhang; Yan-Ting Jin; Kai-Yue Zhang","Anna R. Sjodin; Michael R. Willig; Simon J. Anthony","Sofia Morfopoulou; Vincent Plagnol","Christos Nicolaides; Demetris Avraam; Luis Cueto-Felgueroso; Marta C. Gonz\u00e1lez; Ruben Juanes","Jos\u00e9-Mar\u00eda D\u00edez; Carolina Romero; Rodrigo Gajardo","Karen N. Barnard; Brynn K. Alford-Lawrence; David W. Buchholz; Brian R. Wasik; Justin R. LaClair; Hai Yu; Rebekah Honce; Stefan Ruhl; Petar Pajic; Erin K. Daugherity; Xi Chen; Stacey L. Schultz-Cherry; Hector C. Aguilar; Ajit Varki; Colin R. Parrish","John-Sebastian Eden; Rebecca Rockett; Ian Carter; Hossinur Rahman; Joep de Ligt; James Hadfield; Matthew Storey; Xiaoyun Ren; Rachel Tulloch; Kerri Basile; Jessica Wells; Roy Byun; Nicky Gilroy; Matthew V O\u2019Sullivan; Vitali Sintchenko; Sharon C Chen; Susan Maddocks; Tania C Sorrell; Edward C Holmes; Dominic E Dwyer; Jen Kok","Timothy P. Sheahan; Amy C. Sims; Shuntai Zhou; Rachel L. Graham; Collin S. Hill; Sarah R. Leist; Alexandra Sch\u00e4fer; Kenneth H. Dinnon; Stephanie A. Montgomery; Maria L. Agostini; Andrea J. Pruijssers; James D. Chapell; Ariane J. Brown; Gregory R. Bluemling; Michael G. Natchus; Manohar Saindane; Alexander A. Kolykhalov; George Painter; Jennifer Harcourt; Azaibi Tamin; Natalie J. Thornburg; Ronald Swanstrom; Mark R. Denison; Ralph S. Baric","Gennadi V. Glinsky","Ryan M. Carey; Jenna R. Freund; Benjamin M. Hariri; Nithin D. Adappa; James N. Palmer; Robert J. Lee","Adrian Viehweger; Sebastian Krautwurst; Kevin Lamkiewicz; Ramakanth Madhugiri; John Ziebuhr; Martin H\u00f6lzer; Manja Marz","Amanda D. Melin; Mareike C. Janiak; Frank Marrone; Paramjit S. Arora; James P. Higham","Richard K. Babbs; Jacob A. Beierle; Julia C. Kelliher; Rose Medeiros; Jeya Anandakumar; Anyaa Shah; Emily J. Yao; Melanie M. Chen; Camron D. Bryant","Peng Shao; Yingji Shan","Chengcai Lai; Lihui Liu; Qinghua Liu; Sijie Cheng; Keyu Wang; Lingna Zhao; Min Xia; Cheng Wang; Hongjing Gu; Yueqiang Duan; Zhongpeng Zhao; Lili Zhang; Ziyang Liu; Jianjun Luo; Jianxun Song; Penghui Yang; Runsheng Chen; Xiliang Wang","Bowen Tang; Fengming He; Dongpeng Liu; Meijuan Fang; Zhen Wu; Dong Xu","Tesfahun Dessale; Krishna Chaithanya Batchu; Diogo Barardo; Ng Li Fang; Vanessa Yuk Man Lam; Linfan Xiao; Markus R. Wenk; Nicholas S. Tolwinski; Jan Gruber","Ruixia Yuan; Zhidong Teng; Jinhui Li","Danielle E Anderson; Jin Cui; Qian Ye; Baoying Huang; Wenhong Zu; Jing Gong; Weiqiang Liu; So Young Kim; Biao Guo Yan; Kristmundur Sigmundsson; Xiao Fang Lim; Fei Ye; Peihua Niu; Xuming Zhou; Wenjie Tan; Lin-Fa Wang; Xu Tan","Ze Chen; Xiaofeng Xu; Xiaojun Yang; Weihao Dou; Xiufeng Jin; Haishuo Ji; Guangyuan Liu; Jianxun Luo; Hong Yin; Gao Shan","Linlin Zhang; Daizong Lin; Yuri Kusov; Yong Nian; Qingjun Ma; Jiang Wang; Albrecht von Brunn; Pieter Leyssen; Kristina Lanko; Johan Neyts; Adriaan de Wilde; Eric J. Snijder; Hong Liu; Rolf Hilgenfeld","Nicola F Fletcher; Luke W Meredith; Emma L Tidswell; Steven R Bryden; Daniel Gon\u00e7alves-Carneiro; Yasmin Chaudhry; Claire Shannon-Lowe; Michael A Folan; Daniella A Lefteri; Marieke Pingen; Dalan Bailey; Clive S McKimmie; Alan W Baird","Katrina L. Kalantar; Tiago Carvalho; Charles F.A. de Bourcy; Boris Dimitrov; Greg Dingle; Rebecca Egger; Julie Han; Olivia B. Holmes; Yun-Fang Juan; Ryan King; Andrey Kislyuk; Maria Mariano; Lucia V. Reynoso; David Rissato Cruz; Jonathan Sheu; Jennifer Tang; James Wang; Mark A. Zhang; Emily Zhong; Vida Ahyong; Sreyngim Lay; Sophana Chea; Jennifer A. Bohl; Jessica E. Manning; Cristina M. Tato; Joseph L. DeRisi","Rashmi Mohanty; Xinquan Liu; Jae You Kim; Xiujuan Peng; Sahil Bhandari; Jasmim Leal; Dhivya Arasappan; Dennis C. Wylie; Tony Dong; Debadyuti Ghosh","Aleksandra Milewska; Anna Kula-Pacurar; Jakub Wadas; Agnieszka Suder; Artur Szczepanski; Agnieszka Dabrowska; Katarzyna Owczarek; Marek Ochman; Tomasz Stacel; Zenon Rajfur; Pawel Piotr Labaj; Wojciech Branicki; Krzysztof Pyrc","Yousuf A Khan; Gary Loughran; John F. Atkins","Jo\u00e3o Pedro Fonseca; Alain R. Bonny; G. Renuka Kumar; Andrew H. Ng; Jason Town; Qiu Chang Wu; Elham Aslankoohi; Susan Y. Chen; Patrick Harrigan; Lindsey C. Osimiri; Amy L. Kistler; Hana El-Samad","M. Shaminur Rahman; M. Nazmul Hoque; M. Rafiul Islam; Salma Akter; A. S. M. Rubayet-Ul-Alam; Mohammad Anwar Siddique; Otun Saha; Md. Mizanur Rahaman; Munawar Sultana; M. Anwar Hossain","Stephen P Velsko; Jonathan E Allen","Rowena Chong; Mang Shi; Catherine E Grueber; Edward C Holmes; Carolyn Hogg; Katherine Belov; Vanessa R Barrs","Sovan Saha; Piyali Chatterjee; Subhadip Basu; Mita Nasipuri","Melissa D. Mattocks; Kenneth S. Plante; Ethan J. Fritch; Ralph S. Baric; Martin T. Ferris; Mark T. Heise; Helen M. Lazear","Bin Gu; John R. Shorter; Lucy H. Williams; Timothy A. Bell; Pablo Hock; Katherine A. Dalton; Yiyun Pan; Darla R. Miller; Ginger D. Shaw; Brian C. Cooley; Benjamin D. Philpot; Fernando Pardo-Manuel de Villena","Annika Kratzel; Daniel Todt; Philip V\u2019kovski; Silvio Steiner; Mitra L. Gultom; Tran Thi Nhu Thao; Nadine Ebert; Melle Holwerda; J\u00f6rg Steinmann; Daniela Niemeyer; Ronald Dijkman; G\u00fcnter Kampf; Christian Drosten; Eike Steinmann; Volker Thiel; Stephanie Pfaender","Tao Zhang; Qunfu Wu; Zhigang Zhang","Estefania Nunez-Bajo; Michael Kasimatis; Yasin Cotur; Tarek Asfour; Alex Collins; Ugur Tanriverdi; Max Grell; Matti Kaisti; Guglielmo Senesi; Karen Stevenson; Firat Guder","Daniel Wrapp; Dorien De Vlieger; Kizzmekia S Corbett; Gretel M Torres; Wander Van Breedam; Kenny Roose; Loes van Schie; Markus Hoffmann; Stefan P\u00f6hlmann; Barney S Graham; Nico Callewaert; Bert Schepens; Xavier Saelens; Jason S McLellan","Emily A Bruce; Scott Tighe; Jessica J Hoffman; Pheobe Laaguiby; Diana L Gerrard; Sean A. Diehl; Debra G. B. Leonard; Christopher D. Huston; Beth D. Kirkpatrick; Jessica W Crothers; Julie Dragon; Jason W Botten","Rishikesh Magar; Prakarsh Yadav; Amir Barati Farimani","Niema Moshiri","Naveen Vankadari","Sumaira Gulzar; Saqib Hussain","Umar Faruq Chowdhury; Mohammad Umer Sharif Shohan; Kazi Injamamul Hoque; Mirza Ashikul Beg; Mohammad Ali Moni; Mohammad Kawsar Sharif Siam","Kathleen M O\u2019Reilly; Rachel Lowe; W John Edmunds; Philippe Mayaud; Adam Kucharski; Rosalind M Eggo; Sebastian Funk; Deepit Bhatia; Kamran Khan; Moritz U Kraemar; Annelies Wilder-Smith; Laura C Rodrigues; Patricia Brasil; Eduardo Massad; Thomas Jaenisch; Simon Cauchemez; Oliver J Brady; Laith Yakob","Junsong Zhang; Xiancai Ma; Fei Yu; Jun Liu; Fan Zou; Ting Pan; Hui Zhang","Cory J. Smith; Oscar Castanon; Khaled Said; Verena Volf; Parastoo Khoshakhlagh; Amanda Hornick; Raphael Ferreira; Chun-Ting Wu; Marc G\u00fcell; Shilpa Garg; Hannu Myllykallio; George M. Church","Long T. Nguyen; Brianna M Smith; Piyush K. Jain","Rudo Kieft; Yang Zhang; Alexandre P. Marand; Jose Dagoberto Moran; Robert Bridger; Lance Wells; Robert J. Schmitz; Robert Sabatini","Wayne M. Getz; Richard Salter; Krti Tallam","Michal Ben-Nun; Pete Riley; James Turtle; David P. Bacon; Steven Riley","Lea Gaucherand; Brittany K. Porter; Summer K. Schmaling; Christopher Harley Rycroft; Yuzo Kevorkian; Craig McCormick; Denys A. Khaperskyy; Marta Maria Gaglia","Tyler Wagner; Samir Awasthi; Gayle Wittenberg; AJ Venkatakrishnan; Dan Tarjan; Anuli Anyanwu-Ofili; Andrew Badley; John Halamka; Christopher Flores; Najat Khan; Rakesh Barve; Venky Soundararajan","Stuart Weston; Rob Haupt; James Logue; Krystal Matthews; Matthew B. Frieman","Kaifu Gao; Duc Duy Nguyen; Rui Wang; Guo-Wei Wei","Ronan F. Arthur; James H. Jones; Matthew H. Bonds; Marcus W. Feldman","Terrance A. Thedell; Corey L. Boles; David M. Cwiertny; Jiajie Qian; Grant D. Brown; Matthew W. Nonnenmann","Shriya Srinivasan; Khalil Ramadi; Franceso Vicario; Declan Gwynne; Alison Hayward; Robert Langer; Joseph Frassica; Rebecca Baron; Giovanni Traverso","Dene Littler; Benjamin Gully; Rhys N Colson; Jamie Rossjohn","Eamon B. O\u2019Dea; Harry Snelson; Shweta Bansal","Wioletta Rut; Katarzyna Groborz; Linlin Zhang; Xinyuanyuan Sun; Mikolaj Zmudzinski; Rolf Hilgenfeld; Marcin Drag","James Bull; Scott L. Nuismer; Rustom Antia","Guangchun Han; Ansam Sinjab; Warapen Treekitkarnmongkol; Patrick Brennan; Kieko Hara; Kyle Chang; Elena Bogatenkova; Beatriz Sanchez-Espiridion; Carmen Behrens; Boning Gao; Luc Girard; Jianjun Zhang; Boris Sepesi; Tina Cascone; Lauren Byers; Don L. Gibbons; Jichao Chen; Seyed Javad Moghaddam; Edwin J. Ostrin; Junya Fujimoto; Jerry Shay; John V. Heymach; John D. Minna; Steven Dubinett; Paul A. Scheet; Ignacio I. Wistuba; Edward Hill; Shannon Telesco; Christopher Stevenson; Avrum E. Spira; Linghua Wang; Humam Kadara","Minfeng Xiao; Xiaoqing Liu; Jingkai Ji; Min Li; Jiandong Li; Lin Yang; Wanying Sun; Peidi Ren; Guifang Yang; Jincun Zhao; Tianzhu Liang; Huahui Ren; Tian Chen; Huanzi Zhong; Wenchen Song; Yanqun Wang; Ziqing Deng; Yanping Zhao; Zhihua Ou; Daxi Wang; Jielun Cai; Xinyi Cheng; Taiqing Feng; Honglong Wu; Yanping Gong; Huanming Yang; Jian Wang; Xun Xu; Shida Zhu; Fang Chen; Yanyan Zhang; Weijun Chen; Yimin Li; Junhua Li","Shai Pilosof; Sergio A. Alcala-Corona; Tong Wang; Ted Kim; Sergei Maslov; Rachel Whitaker; Mercedes Pascual","Sukrit Srivastava; Sonia Verma; Mohit Kamthania; Rupinder Kaur; Ruchi Kiran Badyal; Ajay Kumar Saxena; Ho-Joon Shin; Kailash Pandey; Michael Kolbe","Gaurav D Diwan; Deepa Agashe","Hualan Chen","Mengdi Bao; Erik Jensen; Yu Chang; Grant Korensky; Ke Du","Nathalie Pamir; Calvin Pan; Deanna L. Plubell; Patrick M. Hutchins; Chongren Tang; Jake Wimberger; Angela Irwin; Thomas Q. de Aguiar Vallim; Jay W. Heinecke; Aldons J. Lusis","C\u00e9cilia Hognon; Tom Miclot; Cristina Garcia Iriepa; Antonio Franc\u00e9s-Monerris; Stephanie Grandemange; Alessio Terenzi; Marco Marazzi; Giampaolo Barone; Antonio Monari","Haley R. Harrington; Matthew H. Zimmer; Laura M. Chamness; Veronica Nash; Wesley D. Penn; Thomas F. Miller; Suchetana Mukhopadhyay; Jonathan P. Schlebach","Neha Periwal; Sankritya Sarma; Pooja Arora; Vikas Sood","Yanrong Li; Ying Liao; Qiaona Niu; Feng Gu; Yingjie Sun; Chunchun Meng; Lei Tan; Cuiping Song; Xusheng Qiu; Chan Ding","Rohan Maddamsetti; Daniel T. Johnson; Stephanie J. Spielman; Katherine L. Petrie; Debora S. Marks; Justin R. Meyer","Yueting Zhang; Gary Whittaker","Maria Bz\u00f3wka; Karolina Mitusi\u0144ska; Agata Raczy\u0144ska; Aleksandra Samol; Jack Tuszy\u0144ski; Artur G\u00f3ra","Michael Letko; Vincent Munster","Ben Longdon; Jonathan P Day; Joel M Alves; Sophia CL Smith; Thomas M Houslay; John E McGonigle; Lucia Tagliaferri; Francis M Jiggins","Xin Liu; Xiu-Jie Wang","Wei Deng; Linlin Bao; Hong Gao; Zhiguang Xiang; Yajin Qu; Zhiqi Song; Shunran Gong; Jiayi Liu; Jiangning Liu; Pin Yu; Feifei Qi; Yanfeng Xu; Fengli Li; Chong Xiao; Qi Lv; Jing Xue; Qiang Wei; Mingya Liu; Guanpeng Wang; Shunyi Wang; Haisheng Yu; Xing Liu; Wenjie Zhao; Yunlin Han; Chuan Qin","wei zhao; Nian liu; Xiao Li Tao; Chun hong Zheng; Xiang yu Li; Man Yu; Yong gang Li","David Welch; Manuela Buonanno; Veljko Grilj; Igor Shuryak; Connor Crickmore; Alan W. Bigelow; Gerhard Randers-Pehrson; Gary W. Johnson; David J. Brenner","Natalya Bukreyeva; Emily K. Mantlo; Rachel A. Sattler; Cheng Huang; Slobodan Paessler; Jerry Zeldis","Jorge Blazquez-Prieto; Covadonga Huidobro; Ines Lopez-Alonso; Laura Amado-Rodriguez; Paula Martin-Vicente; Cecilia Lopez-Martinez; Irene Crespo; Cristina Pantoja; Pablo J Fernandez-Marcos; Manuel Serrano; Jacob I Sznajder; Guillermo M Albaiceta","Philip Kohlmeier; Austin R Alleman; Romain Libbrecht; Susanne Foitzik; Barbara Feldmeyer","Robert C. M. Knaap; Ra\u00fal Fern\u00e1ndez-Delgado; Tim J. Dalebout; Nadia Oreshkova; Peter J. Bredenbeek; Luis Enjuanes; Isabel Sola; Eric J. Snijder; Marjolein Kikkert","Alba Grifoni; John Sidney; Yun Zhang; Richard H Scheuermann; Bjoern Peters; Alessandro Sette","Jonas Pfab; Dong Si","Yafei Wang; Randy Heiland; Morgan Craig; Courtney L. Davis; Ashlee N Ford Versypt; Adrianne Jenner; Jonathan Ozik; Nicholson Collier; Chase Cockrell; Andrew Becker; Gary An; James A. Glazier; Aarthi Narayanan; Amber M Smith; Paul Macklin","Bethany Brunton; Kai Rogers; Elisabeth K. Phillips; Rachel B. Brouillette; Ruayda Bouls; Noah S. Butler; Wendy Maury","Claude Kwe Yinda; Mark Zeller; N\u00e1dia Concei\u00e7\u00e3o-Neto; Piet Maes; Ward Deboutte; Leen Beller; Elisabeth Heylen; Stephen Mbigha Ghogomu; Marc Van Ranst; Jelle Matthijnssens","Thilo Reich; Marcin Budka","Takanori Matsubara; Takayuki Yanagida; Noriaki Kawaguchi; Takashi Nakano; Junichiro Yoshimoto; Satoshi P. Tsunoda; Shin-ichiro Horigane; Shuhei Ueda; Sayaka Takemoto-Kimura; Hideki Kandori; Akihiro Yamanaka; Takayuki Yamashita","Shenglan Shang; Jiaqi Wu; Xiaoli Li; Xin Liu; Pan Li; Chunli Zheng; Yonghua Wang; Songqing Liu; Jiang Zheng; Hong Zhou","Tsung-Shu Joseph Wu; Matthew Kagoli; Jens Johan Kaasb\u00f8ll; Gunnar Aksel Bjune","Barry Rockx; Thijs Kuiken; Sander Herfst; Theo Bestebroer; Mart M. Lamers; Dennis de Meulder; Geert van Amerongen; Judith van den Brand; Nisreen M.A. Okba; Debby Schipper; Peter van Run; Lonneke Leijten; Ernst Verschoor; Babs Verstrepen; Jan Langermans; Christian Drosten; Martje Fentener van Vlissingen; Ron Fouchier; Rik de Swart; Marion Koopmans; Bart L. Haagmans","Joaquin Sanz; Paul L. Maurizio; Noah Snyder-Mackler; Noah D. Simons; Tawni Voyles; Jordan Kohn; Vasiliki Michopoulos; Mark Wilson; Jenny Tung; Luis B. Barreiro","Markus Hoffmann; Hannah Kleine-Weber; Nadine Kr\u00fcger; Marcel M\u00fcller; Christian Drosten; Stefan P\u00f6hlmann","Marie Hoffmann; Michael T. Monaghan; Knut Reinert","Hussin Rothan; Shannon Stone; Janhavi Natekar; Pratima Kumari; Komal Arora; Mukesh Kumar","Joseph T. Wu; Corey M. Peak; Gabriel M. Leung; Marc Lipsitch","Ali Punjani; David J. Fleet","So Young Kim; Weihua Jin; Amika Sood; David W. Montgomery; Oliver C. Grant; Mark M. Fuster; Li Fu; Jonathan S. Dordick; Robert J. Woods; Fuming Zhang; Robert J. Linhardt","Manuel Jara; David A. Rasmussen; Cesar A. Corzo; Gustavo Machado","Toan The Nguyen; Hien Thi Thu Lai; Ly Hai Nguyen; Duc Nguyen-Manh; Agata The Kranjc Pietrucci","Alejandro A Sch\u00e4ffer; Eneida Hatcher; Linda Yankie; Lara Shonkwiler; J Rodney Brister; Ilene Karsch-Mizrachi; Eric P Nawrocki","Mark Yarmarkovich; John M. Warrington; Alvin Farrel; John M. Maris","Marzena Szwed; Tonje S\u00f8nstevold; Anders \u00d8verbye; Nikolai Engedal; Beata Grallert; \u00ddrr A M\u00f8rch; Tore-Geir Iversen; Tore Skotland; Kirsten Sandvig; Maria L Torgersen","Evans K. Lodge; Annakate M. Schatz; John M. Drake","Brandon Alexander Holt; Gabriel A. Kwong","Elsie Yekwa; Chutima Aphibanthammakit; Xavier Carnec; Bruno Coutard; Caroline Picard; Bruno Canard; Sylvain Baize; Fran\u00e7ois Ferron","Hemant Suryawanshi; Pavel Morozov; Thomas Tuschl; Thangamani Muthukumar; Benjamin R. tenOever; Masashi Yamaji; Zev Williams","Michael T. Parker; Smita Gopinath; Corey E. Perez; Melissa M. Linehan; Jason M. Crawford; Akiko Iwasaki; Brett D. Lindenbach","Yifei Lang; Wentao Li; Zeshi Li; Danielle Koerhuis; Arthur C.S. van den Burg; Erik Rozemuller; Berend-Jan Bosch; Frank J.M. van Kuppeveld; Geert-Jan P.H. Boons; Eric G. Huizinga; Hilde M. van der Schaar; Raoul J. de Groot","jeff smith; R. Fredrik Inglis"],"desc":{"__ndarray__":"AgAAAAYAAAADAAAAAgAAAAIAAAAFAAAABQAAAAMAAAAFAAAAAgAAAAMAAAAFAAAABQAAAAIAAAAFAAAAAgAAAAQAAAAFAAAACgAAAAIAAAAIAAAAAgAAAAQAAAACAAAACgAAAAcAAAAIAAAABQAAAAQAAAAFAAAAAAAAAAcAAAAFAAAABAAAAAMAAAAGAAAABQAAAAgAAAAEAAAACgAAAAIAAAAIAAAAAgAAAAUAAAACAAAAAgAAAAcAAAAIAAAAAgAAAAMAAAAFAAAAAgAAAAQAAAACAAAAAQAAAAUAAAACAAAABQAAAAMAAAAFAAAAAQAAAAQAAAACAAAABAAAAAUAAAAHAAAABQAAAAgAAAAEAAAABgAAAAYAAAACAAAABQAAAAkAAAAGAAAACQAAAAkAAAAKAAAAAgAAAAIAAAAJAAAABgAAAAIAAAAGAAAAAwAAAAUAAAADAAAACAAAAAMAAAAIAAAAAgAAAAAAAAABAAAAAQAAAAEAAAAFAAAABQAAAAIAAAADAAAABgAAAAoAAAACAAAAAgAAAAIAAAACAAAAAwAAAAMAAAACAAAABAAAAAIAAAALAAAABgAAAAcAAAABAAAAAgAAAAMAAAAFAAAABQAAAAMAAAAGAAAACAAAAAIAAAAAAAAABQAAAAYAAAADAAAABQAAAAUAAAAGAAAABAAAAAoAAAAEAAAAAwAAAAoAAAAAAAAABQAAAAMAAAACAAAAAgAAAAYAAAAFAAAABQAAAAYAAAAJAAAACQAAAAMAAAADAAAABgAAAAYAAAADAAAAAgAAAAkAAAABAAAACQAAAAIAAAAJAAAABgAAAAgAAAADAAAAAQAAAAkAAAADAAAACgAAAAUAAAAAAAAAAAAAAAMAAAABAAAABQAAAAUAAAAFAAAABQAAAAMAAAAFAAAAAwAAAAYAAAACAAAABQAAAAQAAAAIAAAAAgAAAAIAAAAFAAAABgAAAAkAAAAKAAAAAwAAAAIAAAAIAAAABgAAAAUAAAAFAAAABgAAAAEAAAADAAAACAAAAAQAAAADAAAABwAAAAcAAAAIAAAAAgAAAAYAAAAFAAAAAwAAAAMAAAACAAAAAgAAAAgAAAAEAAAAAAAAAAIAAAAAAAAABgAAAAMAAAABAAAABgAAAAIAAAAGAAAAAgAAAAgAAAACAAAAAwAAAAsAAAAFAAAABQAAAAgAAAACAAAAAwAAAAMAAAALAAAABAAAAAUAAAAEAAAAAgAAAAEAAAACAAAAAwAAAAMAAAADAAAABgAAAAoAAAADAAAACQAAAAYAAAACAAAABQAAAAUAAAADAAAAAAAAAAEAAAAJAAAAAwAAAAMAAAACAAAABgAAAAUAAAAIAAAACQAAAAMAAAADAAAAAwAAAAIAAAAEAAAABAAAAAUAAAACAAAAAgAAAAYAAAAIAAAABQAAAAkAAAACAAAAAgAAAAMAAAAKAAAABQAAAAMAAAAJAAAABQAAAAYAAAACAAAAAgAAAAUAAAAFAAAAAgAAAAoAAAACAAAABAAAAAAAAAAJAAAACgAAAAIAAAALAAAAAgAAAAUAAAACAAAABgAAAAYAAAACAAAABwAAAAEAAAACAAAAAQAAAAkAAAAAAAAABQAAAAUAAAAGAAAAAgAAAAkAAAAJAAAACAAAAAUAAAAEAAAACgAAAAQAAAAFAAAAAwAAAAQAAAAGAAAACQAAAAQAAAACAAAABQAAAAMAAAAFAAAAAwAAAAIAAAAJAAAACQAAAAIAAAABAAAAAgAAAAUAAAABAAAACgAAAAMAAAACAAAABQAAAAIAAAACAAAABQAAAAUAAAADAAAAAgAAAAQAAAAJAAAAAwAAAAMAAAAEAAAABwAAAAUAAAALAAAABgAAAAMAAAAFAAAABQAAAAMAAAAJAAAABgAAAAIAAAABAAAACwAAAAIAAAAEAAAABQAAAAIAAAADAAAABgAAAAYAAAAHAAAABgAAAAMAAAAKAAAAAgAAAAAAAAADAAAABQAAAAUAAAAEAAAABgAAAAIAAAAFAAAAAgAAAAgAAAACAAAABgAAAAEAAAAIAAAAAgAAAAIAAAADAAAAAgAAAAQAAAACAAAAAwAAAAgAAAAHAAAABgAAAAYAAAAJAAAAAwAAAAUAAAAJAAAAAgAAAAAAAAALAAAAAwAAAAUAAAAFAAAAAwAAAAUAAAADAAAABQAAAAoAAAAIAAAAAwAAAAgAAAACAAAACAAAAAgAAAADAAAAAwAAAAUAAAAHAAAABQAAAAEAAAAFAAAABgAAAAIAAAAFAAAABwAAAAMAAAAFAAAABQAAAAcAAAACAAAACgAAAAQAAAADAAAABQAAAAUAAAADAAAABQAAAAMAAAAJAAAABgAAAAIAAAAFAAAAAwAAAAoAAAALAAAAAwAAAAUAAAABAAAABQAAAAYAAAAFAAAAAwAAAAQAAAACAAAABQAAAAIAAAADAAAAAwAAAAMAAAAGAAAABQAAAAUAAAAFAAAAAwAAAAMAAAADAAAABgAAAAUAAAAHAAAAAwAAAAIAAAACAAAAAgAAAAMAAAAFAAAACAAAAAAAAAADAAAAAwAAAAUAAAAAAAAABgAAAAMAAAAEAAAACAAAAAMAAAAIAAAAAwAAAAsAAAADAAAAAgAAAAkAAAAGAAAAAwAAAAIAAAAGAAAAAwAAAAYAAAADAAAABAAAAAYAAAAFAAAABgAAAAIAAAAJAAAABQAAAAIAAAAJAAAAAgAAAAYAAAAKAAAAAwAAAAMAAAADAAAAAgAAAAYAAAABAAAACAAAAAYAAAAEAAAAAgAAAAYAAAAIAAAAAAAAAAcAAAAEAAAABgAAAAMAAAACAAAACgAAAAUAAAAIAAAAAgAAAAoAAAADAAAAAgAAAAMAAAAFAAAAAgAAAAUAAAAJAAAACwAAAAQAAAAFAAAAAwAAAAQAAAAFAAAAAgAAAAMAAAADAAAACwAAAAMAAAAGAAAAAQAAAAoAAAADAAAABAAAAAMAAAAEAAAAAwAAAAcAAAAGAAAAAwAAAAYAAAAIAAAAAgAAAAIAAAAIAAAACQAAAAEAAAADAAAAAwAAAAkAAAAGAAAABQAAAAYAAAAFAAAABAAAAAUAAAADAAAAAgAAAAUAAAAHAAAABAAAAAUAAAAFAAAAAwAAAAIAAAAJAAAAAgAAAAMAAAACAAAABQAAAAAAAAACAAAAAgAAAAIAAAAKAAAAAwAAAAQAAAADAAAAAwAAAAQAAAAFAAAAAQAAAAgAAAAIAAAACQAAAAUAAAAGAAAABwAAAAIAAAACAAAACAAAAAcAAAADAAAABQAAAAQAAAACAAAACAAAAAMAAAACAAAAAgAAAAMAAAAFAAAABQAAAAIAAAADAAAAAgAAAAYAAAAAAAAAAwAAAAUAAAADAAAAAgAAAAYAAAACAAAABAAAAAkAAAACAAAACwAAAAMAAAACAAAABgAAAAcAAAAHAAAACwAAAAUAAAACAAAACwAAAAYAAAADAAAAAAAAAAMAAAADAAAABgAAAAYAAAAFAAAAAwAAAAMAAAAGAAAAAgAAAAIAAAAFAAAABQAAAAEAAAAJAAAAAwAAAAoAAAADAAAAAgAAAAUAAAALAAAABgAAAAUAAAAGAAAAAwAAAAMAAAABAAAAAgAAAAIAAAAEAAAAAwAAAAoAAAAFAAAACAAAAAIAAAACAAAAAgAAAAMAAAAFAAAAAwAAAAIAAAADAAAAAwAAAAYAAAAJAAAAAgAAAAAAAAAFAAAABAAAAAIAAAAHAAAAAgAAAAkAAAAGAAAAAQAAAAsAAAAKAAAABgAAAAkAAAAGAAAAAwAAAAgAAAACAAAABgAAAAMAAAADAAAABgAAAAUAAAADAAAABAAAAAUAAAAJAAAACAAAAAUAAAAFAAAABQAAAAgAAAACAAAAAgAAAAkAAAABAAAABQAAAAQAAAAFAAAABgAAAAoAAAAIAAAAAwAAAAIAAAADAAAACAAAAAIAAAAFAAAAAgAAAAEAAAAFAAAAAwAAAAIAAAAAAAAAAwAAAAcAAAAIAAAACAAAAAgAAAACAAAACgAAAAIAAAAGAAAAAwAAAAMAAAADAAAABQAAAAUAAAAGAAAABwAAAAIAAAAJAAAAAgAAAAMAAAADAAAAAwAAAAgAAAAIAAAAAwAAAAQAAAACAAAABgAAAAgAAAAKAAAABQAAAAMAAAAJAAAAAgAAAAgAAAAEAAAABQAAAAgAAAAHAAAABQAAAAEAAAADAAAACgAAAAYAAAAAAAAAAgAAAAMAAAAAAAAAAgAAAAYAAAACAAAABQAAAAYAAAABAAAAAwAAAAgAAAACAAAAAQAAAAMAAAACAAAABwAAAAUAAAAFAAAAAQAAAAsAAAAJAAAAAwAAAAMAAAAJAAAAAQAAAAoAAAACAAAABQAAAAIAAAAIAAAABQAAAAYAAAAJAAAABgAAAAUAAAAHAAAAAwAAAAkAAAADAAAAAwAAAAQAAAADAAAAAgAAAAUAAAA=","dtype":"int32","shape":[848]},"journal":["NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN","NaN"],"labels":["C-2","C-6","C-3","C-2","C-2","C-5","C-5","C-3","C-5","C-2","C-3","C-5","C-5","C-2","C-5","C-2","C-4","C-5","C-10","C-2","C-8","C-2","C-4","C-2","C-10","C-7","C-8","C-5","C-4","C-5","C-0","C-7","C-5","C-4","C-3","C-6","C-5","C-8","C-4","C-10","C-2","C-8","C-2","C-5","C-2","C-2","C-7","C-8","C-2","C-3","C-5","C-2","C-4","C-2","C-1","C-5","C-2","C-5","C-3","C-5","C-1","C-4","C-2","C-4","C-5","C-7","C-5","C-8","C-4","C-6","C-6","C-2","C-5","C-9","C-6","C-9","C-9","C-10","C-2","C-2","C-9","C-6","C-2","C-6","C-3","C-5","C-3","C-8","C-3","C-8","C-2","C-0","C-1","C-1","C-1","C-5","C-5","C-2","C-3","C-6","C-10","C-2","C-2","C-2","C-2","C-3","C-3","C-2","C-4","C-2","C-11","C-6","C-7","C-1","C-2","C-3","C-5","C-5","C-3","C-6","C-8","C-2","C-0","C-5","C-6","C-3","C-5","C-5","C-6","C-4","C-10","C-4","C-3","C-10","C-0","C-5","C-3","C-2","C-2","C-6","C-5","C-5","C-6","C-9","C-9","C-3","C-3","C-6","C-6","C-3","C-2","C-9","C-1","C-9","C-2","C-9","C-6","C-8","C-3","C-1","C-9","C-3","C-10","C-5","C-0","C-0","C-3","C-1","C-5","C-5","C-5","C-5","C-3","C-5","C-3","C-6","C-2","C-5","C-4","C-8","C-2","C-2","C-5","C-6","C-9","C-10","C-3","C-2","C-8","C-6","C-5","C-5","C-6","C-1","C-3","C-8","C-4","C-3","C-7","C-7","C-8","C-2","C-6","C-5","C-3","C-3","C-2","C-2","C-8","C-4","C-0","C-2","C-0","C-6","C-3","C-1","C-6","C-2","C-6","C-2","C-8","C-2","C-3","C-11","C-5","C-5","C-8","C-2","C-3","C-3","C-11","C-4","C-5","C-4","C-2","C-1","C-2","C-3","C-3","C-3","C-6","C-10","C-3","C-9","C-6","C-2","C-5","C-5","C-3","C-0","C-1","C-9","C-3","C-3","C-2","C-6","C-5","C-8","C-9","C-3","C-3","C-3","C-2","C-4","C-4","C-5","C-2","C-2","C-6","C-8","C-5","C-9","C-2","C-2","C-3","C-10","C-5","C-3","C-9","C-5","C-6","C-2","C-2","C-5","C-5","C-2","C-10","C-2","C-4","C-0","C-9","C-10","C-2","C-11","C-2","C-5","C-2","C-6","C-6","C-2","C-7","C-1","C-2","C-1","C-9","C-0","C-5","C-5","C-6","C-2","C-9","C-9","C-8","C-5","C-4","C-10","C-4","C-5","C-3","C-4","C-6","C-9","C-4","C-2","C-5","C-3","C-5","C-3","C-2","C-9","C-9","C-2","C-1","C-2","C-5","C-1","C-10","C-3","C-2","C-5","C-2","C-2","C-5","C-5","C-3","C-2","C-4","C-9","C-3","C-3","C-4","C-7","C-5","C-11","C-6","C-3","C-5","C-5","C-3","C-9","C-6","C-2","C-1","C-11","C-2","C-4","C-5","C-2","C-3","C-6","C-6","C-7","C-6","C-3","C-10","C-2","C-0","C-3","C-5","C-5","C-4","C-6","C-2","C-5","C-2","C-8","C-2","C-6","C-1","C-8","C-2","C-2","C-3","C-2","C-4","C-2","C-3","C-8","C-7","C-6","C-6","C-9","C-3","C-5","C-9","C-2","C-0","C-11","C-3","C-5","C-5","C-3","C-5","C-3","C-5","C-10","C-8","C-3","C-8","C-2","C-8","C-8","C-3","C-3","C-5","C-7","C-5","C-1","C-5","C-6","C-2","C-5","C-7","C-3","C-5","C-5","C-7","C-2","C-10","C-4","C-3","C-5","C-5","C-3","C-5","C-3","C-9","C-6","C-2","C-5","C-3","C-10","C-11","C-3","C-5","C-1","C-5","C-6","C-5","C-3","C-4","C-2","C-5","C-2","C-3","C-3","C-3","C-6","C-5","C-5","C-5","C-3","C-3","C-3","C-6","C-5","C-7","C-3","C-2","C-2","C-2","C-3","C-5","C-8","C-0","C-3","C-3","C-5","C-0","C-6","C-3","C-4","C-8","C-3","C-8","C-3","C-11","C-3","C-2","C-9","C-6","C-3","C-2","C-6","C-3","C-6","C-3","C-4","C-6","C-5","C-6","C-2","C-9","C-5","C-2","C-9","C-2","C-6","C-10","C-3","C-3","C-3","C-2","C-6","C-1","C-8","C-6","C-4","C-2","C-6","C-8","C-0","C-7","C-4","C-6","C-3","C-2","C-10","C-5","C-8","C-2","C-10","C-3","C-2","C-3","C-5","C-2","C-5","C-9","C-11","C-4","C-5","C-3","C-4","C-5","C-2","C-3","C-3","C-11","C-3","C-6","C-1","C-10","C-3","C-4","C-3","C-4","C-3","C-7","C-6","C-3","C-6","C-8","C-2","C-2","C-8","C-9","C-1","C-3","C-3","C-9","C-6","C-5","C-6","C-5","C-4","C-5","C-3","C-2","C-5","C-7","C-4","C-5","C-5","C-3","C-2","C-9","C-2","C-3","C-2","C-5","C-0","C-2","C-2","C-2","C-10","C-3","C-4","C-3","C-3","C-4","C-5","C-1","C-8","C-8","C-9","C-5","C-6","C-7","C-2","C-2","C-8","C-7","C-3","C-5","C-4","C-2","C-8","C-3","C-2","C-2","C-3","C-5","C-5","C-2","C-3","C-2","C-6","C-0","C-3","C-5","C-3","C-2","C-6","C-2","C-4","C-9","C-2","C-11","C-3","C-2","C-6","C-7","C-7","C-11","C-5","C-2","C-11","C-6","C-3","C-0","C-3","C-3","C-6","C-6","C-5","C-3","C-3","C-6","C-2","C-2","C-5","C-5","C-1","C-9","C-3","C-10","C-3","C-2","C-5","C-11","C-6","C-5","C-6","C-3","C-3","C-1","C-2","C-2","C-4","C-3","C-10","C-5","C-8","C-2","C-2","C-2","C-3","C-5","C-3","C-2","C-3","C-3","C-6","C-9","C-2","C-0","C-5","C-4","C-2","C-7","C-2","C-9","C-6","C-1","C-11","C-10","C-6","C-9","C-6","C-3","C-8","C-2","C-6","C-3","C-3","C-6","C-5","C-3","C-4","C-5","C-9","C-8","C-5","C-5","C-5","C-8","C-2","C-2","C-9","C-1","C-5","C-4","C-5","C-6","C-10","C-8","C-3","C-2","C-3","C-8","C-2","C-5","C-2","C-1","C-5","C-3","C-2","C-0","C-3","C-7","C-8","C-8","C-8","C-2","C-10","C-2","C-6","C-3","C-3","C-3","C-5","C-5","C-6","C-7","C-2","C-9","C-2","C-3","C-3","C-3","C-8","C-8","C-3","C-4","C-2","C-6","C-8","C-10","C-5","C-3","C-9","C-2","C-8","C-4","C-5","C-8","C-7","C-5","C-1","C-3","C-10","C-6","C-0","C-2","C-3","C-0","C-2","C-6","C-2","C-5","C-6","C-1","C-3","C-8","C-2","C-1","C-3","C-2","C-7","C-5","C-5","C-1","C-11","C-9","C-3","C-3","C-9","C-1","C-10","C-2","C-5","C-2","C-8","C-5","C-6","C-9","C-6","C-5","C-7","C-3","C-9","C-3","C-3","C-4","C-3","C-2","C-5"],"links":["10.1101/752592","10.1101/2020.03.23.004580","10.1101/2020.02.28.969618","10.1101/2020.03.31.017889","10.1101/2020.04.07.023903","10.1101/2020.02.15.951012","10.1101/812313","10.1101/2020.03.19.998724","10.1101/2020.04.01.019562","10.1101/846410","10.1101/394080","10.1101/2020.04.05.026435","10.1101/2020.03.18.997346","10.1101/353037","10.1101/623819","10.1101/2020.03.29.013490","10.1101/2020.04.20.027144","10.1101/307678","10.1101/761353","10.1101/246785","10.1101/295337","10.1101/2019.12.17.880526","10.1101/722090","10.1101/2020.03.01.971499","10.1101/356543","10.1101/2020.04.13.039198","10.1101/707356","10.1101/2020.01.10.901728","10.1101/2020.03.25.009084","10.1101/2020.04.07.030759","10.1101/006866","10.1101/2020.04.15.040618","10.1101/2020.04.06.027557","10.1101/2020.02.24.963348","10.1101/2020.04.07.029934","10.1101/2020.03.30.015685","10.1101/2020.03.10.985150","10.1101/486720","10.1101/2020.04.16.045690","10.1101/549832","10.1101/2020.04.21.053058","10.1101/041509","10.1101/061788","10.1101/2020.04.12.025577","10.1101/2020.03.12.989186","10.1101/2020.03.06.980037","10.1101/2020.03.30.016931","10.1101/2020.01.27.922443","10.1101/2020.01.30.927574","10.1101/2020.03.05.979260","10.1101/2020.03.25.008805","10.1101/843680","10.1101/2020.02.27.967588","10.1101/2020.03.29.014407","10.1101/2020.04.15.037564","10.1101/2020.01.29.925354","10.1101/2020.03.28.013276","10.1101/001727","10.1101/2020.02.20.958785","10.1101/341735","10.1101/706804","10.1101/2020.04.14.040204","10.1101/819482","10.1101/2020.04.17.047332","10.1101/400648","10.1101/2020.03.20.000794","10.1101/738526","10.1101/2020.01.23.916395","10.1101/2020.03.24.006270","10.1101/2020.03.20.000885","10.1101/2020.03.15.992883","10.1101/2020.01.15.908608","10.1101/365254","10.1101/2020.01.23.916726","10.1101/2020.02.17.952895","10.1101/2020.04.17.046086","10.1101/742866","10.1101/732255","10.1101/2020.04.15.042085","10.1101/2020.02.01.929976","10.1101/360271","10.1101/2020.04.12.038554","10.1101/2020.04.20.051581","10.1101/2020.01.31.929695","10.1101/730739","10.1101/2020.04.10.036343","10.1101/571455","10.1101/2020.03.22.002188","10.1101/2020.04.15.043083","10.1101/2020.03.02.974048","10.1101/2020.02.25.963546","10.1101/2020.03.27.012906","10.1101/2020.04.15.042119","10.1101/2020.04.08.031526","10.1101/2020.04.08.031807","10.1101/2020.04.20.046920","10.1101/676155","10.1101/2020.04.11.037382","10.1101/261628","10.1101/2020.03.24.005561","10.1101/297226","10.1101/246991","10.1101/780841","10.1101/606715","10.1101/2020.04.14.042010","10.1101/153437","10.1101/2020.04.06.028811","10.1101/2020.04.09.033522","10.1101/2020.02.16.951913","10.1101/440628","10.1101/436600","10.1101/2020.02.15.950568","10.1101/2020.03.23.004176","10.1101/632372","10.1101/2020.04.14.041301","10.1101/476341","10.1101/2020.04.20.050047","10.1101/2020.04.10.023358","10.1101/378786","10.1101/2020.03.02.974139","10.1101/2020.04.20.049866","10.1101/2020.04.14.041731","10.1101/612614","10.1101/367367","10.1101/2020.04.17.047548","10.1101/069799","10.1101/2020.03.02.973255","10.1101/333021","10.1101/2020.03.10.986398","10.1101/2020.04.09.034983","10.1101/458877","10.1101/2020.03.12.988634","10.1101/301622","10.1101/704965","10.1101/2020.02.10.942847","10.1101/2020.04.05.026336","10.1101/2020.03.02.968388","10.1101/2020.04.15.997254","10.1101/2020.04.02.022194","10.1101/2020.01.24.919183","10.1101/114819","10.1101/766709","10.1101/2020.03.25.007898","10.1101/018481","10.1101/2020.03.17.995795","10.1101/2020.04.08.032763","10.1101/147736","10.1101/2020.04.05.026633","10.1101/2020.03.04.976027","10.1101/2020.02.22.961268","10.1101/2020.03.21.001586","10.1101/2020.01.30.926477","10.1101/868398","10.1101/2020.03.24.006544","10.1101/173286","10.1101/2020.04.15.043364","10.1101/2020.03.13.991570","10.1101/568485","10.1101/2020.04.13.038620","10.1101/208587","10.1101/852558","10.1101/592485","10.1101/206359","10.1101/333625","10.1101/2020.04.14.041962","10.1101/296996","10.1101/576959","10.1101/792705","10.1101/334334","10.1101/2020.03.30.016790","10.1101/222950","10.1101/455568","10.1101/065250","10.1101/451740","10.1101/498998","10.1101/2020.03.16.993816","10.1101/2020.04.07.029884","10.1101/102798","10.1101/2020.03.23.002832","10.1101/2020.01.28.922757","10.1101/2020.03.13.991307","10.1101/2020.04.16.044016","10.1101/157156","10.1101/2020.01.28.922922","10.1101/421511","10.1101/338392","10.1101/118315","10.1101/182196","10.1101/696195","10.1101/2020.02.19.956946","10.1101/2020.03.24.006007","10.1101/190769","10.1101/2020.04.09.034942","10.1101/2020.03.13.990226","10.1101/562421","10.1101/683326","10.1101/2020.03.26.010694","10.1101/325316","10.1101/2020.04.05.026005","10.1101/2020.04.06.027805","10.1101/300954","10.1101/2020.04.20.050138","10.1101/2020.02.04.933135","10.1101/2020.03.30.014555","10.1101/584680","10.1101/290874","10.1101/2020.03.30.015461","10.1101/729830","10.1101/302331","10.1101/2020.04.02.021451","10.1101/469692","10.1101/309591","10.1101/2020.03.25.008904","10.1101/2020.02.01.930537","10.1101/2020.03.24.005900","10.1101/2020.04.04.020925","10.1101/2020.04.19.048710","10.1101/2020.04.08.031856","10.1101/2020.02.23.961912","10.1101/488510","10.1101/2019.12.19.882175","10.1101/2020.02.17.952879","10.1101/2020.03.28.013508","10.1101/099069","10.1101/2020.04.07.029199","10.1101/815480","10.1101/191601","10.1101/315366","10.1101/2020.03.24.992230","10.1101/769224","10.1101/181982","10.1101/2020.03.30.015644","10.1101/2020.02.16.946699","10.1101/2020.03.16.990317","10.1101/2020.04.07.030734","10.1101/2020.04.07.024455","10.1101/2020.03.29.008631","10.1101/2020.01.08.899146","10.1101/509141","10.1101/294488","10.1101/2020.03.21.001933","10.1101/2020.03.25.006643","10.1101/512467","10.1101/2020.01.25.919787","10.1101/2020.03.30.016832","10.1101/2020.03.31.019216","10.1101/664342","10.1101/575571","10.1101/587105","10.1101/112904","10.1101/326546","10.1101/346411","10.1101/484675","10.1101/357046","10.1101/444067","10.1101/2020.04.17.047324","10.1101/2020.03.26.009001","10.1101/703751","10.1101/173211","10.1101/660217","10.1101/2020.02.17.945014","10.1101/352898","10.1101/2020.04.10.029454","10.1101/2020.03.31.016048","10.1101/2020.04.19.049254","10.1101/2020.04.13.039792","10.1101/2020.03.25.996348","10.1101/502609","10.1101/2020.01.27.921627","10.1101/525089","10.1101/402370","10.1101/078543","10.1101/2020.03.16.994152","10.1101/175562","10.1101/297580","10.1101/2020.02.19.957118","10.1101/842740","10.1101/607986","10.1101/461285","10.1101/544841","10.1101/2020.02.17.951848","10.1101/417907","10.1101/2020.03.09.983247","10.1101/684662","10.1101/2020.04.18.045963","10.1101/2020.03.03.975524","10.1101/358564","10.1101/2020.03.26.010322","10.1101/2020.03.24.005702","10.1101/284869","10.1101/774224","10.1101/631879","10.1101/780577","10.1101/566281","10.1101/2020.04.16.043273","10.1101/2020.04.20.052019","10.1101/2020.03.11.987016","10.1101/2020.02.16.951723","10.1101/2020.03.30.015891","10.1101/2020.03.28.013789","10.1101/2020.02.03.933226","10.1101/507350","10.1101/852194","10.1101/2020.03.31.018556","10.1101/070656","10.1101/229690","10.1101/597963","10.1101/228692","10.1101/2020.03.04.975995","10.1101/768663","10.1101/627158","10.1101/496901","10.1101/677021","10.1101/2020.04.09.031252","10.1101/742320","10.1101/2020.03.09.984393","10.1101/2020.04.14.042002","10.1101/2020.04.06.025635","10.1101/659540","10.1101/2020.04.16.045799","10.1101/2020.01.24.915157","10.1101/2020.04.20.050039","10.1101/2020.04.07.030767","10.1101/2020.04.21.053967","10.1101/2020.02.08.939660","10.1101/733915","10.1101/2020.03.27.009480","10.1101/2020.01.12.902452","10.1101/2020.04.06.026476","10.1101/574103","10.1101/771394","10.1101/587261","10.1101/2020.04.02.020081","10.1101/2020.03.30.016485","10.1101/569012","10.1101/667931","10.1101/029397","10.1101/483289","10.1101/2020.04.10.036640","10.1101/581512","10.1101/2020.04.21.042911","10.1101/2020.03.07.982264","10.1101/492603","10.1101/2020.03.08.980383","10.1101/374926","10.1101/2020.03.24.005298","10.1101/2020.04.06.028712","10.1101/474205","10.1101/2020.03.18.997585","10.1101/324723","10.1101/2020.04.06.028522","10.1101/2020.03.20.000141","10.1101/020834","10.1101/2020.04.14.041319","10.1101/2020.03.13.990267","10.1101/507467","10.1101/528174","10.1101/540252","10.1101/650341","10.1101/156497","10.1101/2020.02.07.939207","10.1101/2020.03.26.009233","10.1101/2020.04.16.044503","10.1101/699371","10.1101/2020.04.20.049924","10.1101/2020.04.13.038752","10.1101/2020.03.10.986109","10.1101/2020.03.29.014183","10.1101/2020.01.10.901801","10.1101/2020.04.01.019463","10.1101/2020.03.19.999318","10.1101/2020.02.05.935072","10.1101/2020.04.17.046185","10.1101/564427","10.1101/2020.03.30.015545","10.1101/2020.03.09.984856","10.1101/2020.03.13.991083","10.1101/862573","10.1101/2020.02.26.967026","10.1101/046532","10.1101/359547","10.1101/2020.04.06.027854","10.1101/2020.02.25.965582","10.1101/2020.04.17.047118","10.1101/618249","10.1101/213579","10.1101/2020.04.09.033233","10.1101/2020.04.10.032342","10.1101/2020.04.15.043166","10.1101/069013","10.1101/2020.04.03.023887","10.1101/862144","10.1101/2020.01.18.911297","10.1101/564542","10.1101/2020.04.07.030684","10.1101/2020.04.10.036418","10.1101/276147","10.1101/230565","10.1101/2020.03.26.009209","10.1101/2020.03.24.006197","10.1101/2020.02.10.942748","10.1101/626309","10.1101/155457","10.1101/370841","10.1101/2020.04.01.020594","10.1101/2020.03.06.977876","10.1101/478040","10.1101/2020.03.24.005876","10.1101/2019.12.20.884338","10.1101/2020.03.06.981142","10.1101/2020.04.08.031963","10.1101/530600","10.1101/2020.02.03.932350","10.1101/2020.04.16.045419","10.1101/2020.03.13.990242","10.1101/2020.03.17.995209","10.1101/293936","10.1101/299800","10.1101/2020.03.15.991331","10.1101/2020.01.21.914929","10.1101/2020.04.02.021576","10.1101/392324","10.1101/525105","10.1101/2020.01.07.896977","10.1101/344127","10.1101/2020.02.25.965434","10.1101/2020.02.27.967885","10.1101/2020.04.13.036293","10.1101/352468","10.1101/2020.04.05.026377","10.1101/2020.03.14.988345","10.1101/027722","10.1101/534008","10.1101/550285","10.1101/196998","10.1101/2020.03.26.010975","10.1101/2020.04.15.036285","10.1101/782409","10.1101/565218","10.1101/2020.02.08.939892","10.1101/2020.02.18.955195","10.1101/2019.12.18.880849","10.1101/2020.04.20.052159","10.1101/2019.12.13.876037","10.1101/2020.03.29.014381","10.1101/558783","10.1101/634832","10.1101/2020.04.06.028647","10.1101/2020.03.25.998963","10.1101/2020.03.02.974311","10.1101/343111","10.1101/140954","10.1101/2020.02.06.936302","10.1101/670398","10.1101/291864","10.1101/2020.04.01.019943","10.1101/2020.03.12.988246","10.1101/2020.04.16.045302","10.1101/2020.04.20.048066","10.1101/551986","10.1101/2020.04.10.035865","10.1101/2020.02.20.957472","10.1101/2020.04.20.046144","10.1101/2020.03.25.006213","10.1101/412692","10.1101/647305","10.1101/309880","10.1101/2020.03.10.986190","10.1101/2020.03.24.004796","10.1101/2020.04.20.050393","10.1101/2020.01.31.928796","10.1101/2020.04.01.019877","10.1101/384453","10.1101/200410","10.1101/2020.01.31.929547","10.1101/2020.04.03.024216","10.1101/2020.04.08.013516","10.1101/282707","10.1101/787879","10.1101/2020.03.20.999730","10.1101/2020.04.17.047498","10.1101/2020.03.29.013342","10.1101/2020.02.28.969782","10.1101/072652","10.1101/2020.01.23.917559","10.1101/094623","10.1101/370528","10.1101/634600","10.1101/048801","10.1101/2020.02.05.936013","10.1101/484857","10.1101/2020.03.28.013672","10.1101/635466","10.1101/339705","10.1101/003889","10.1101/777847","10.1101/641795","10.1101/2020.02.08.939314","10.1101/101774","10.1101/663823","10.1101/2020.04.07.030445","10.1101/2020.02.25.964775","10.1101/619924","10.1101/2020.01.21.914044","10.1101/292979","10.1101/2020.03.26.009803","10.1101/167999","10.1101/600940","10.1101/2020.02.03.931766","10.1101/2020.03.02.968818","10.1101/2020.04.07.030924","10.1101/2020.04.03.024257","10.1101/2020.03.30.015008","10.1101/2020.03.12.977918","10.1101/694380","10.1101/402800","10.1101/2020.04.02.022764","10.1101/2020.02.26.961938","10.1101/623678","10.1101/2020.03.09.983064","10.1101/111567","10.1101/593392","10.1101/460832","10.1101/2020.02.27.969006","10.1101/053967","10.1101/602706","10.1101/2020.04.20.052290","10.1101/2020.04.19.047274","10.1101/090936","10.1101/2020.04.14.041434","10.1101/2020.04.06.028431","10.1101/661579","10.1101/2020.04.02.021725","10.1101/2020.04.17.046938","10.1101/2020.04.07.024752","10.1101/2020.04.13.039941","10.1101/2020.04.21.051912","10.1101/539239","10.1101/279570","10.1101/835181","10.1101/2020.04.10.035824","10.1101/2020.04.09.033928","10.1101/163360","10.1101/2020.04.13.039917","10.1101/2020.04.15.041848","10.1101/628073","10.1101/510131","10.1101/353201","10.1101/346999","10.1101/830281","10.1101/2020.04.12.037580","10.1101/2019.12.15.877092","10.1101/291849","10.1101/2020.04.09.034454","10.1101/2020.01.24.919282","10.1101/2020.03.15.993097","10.1101/2020.03.17.994137","10.1101/299776","10.1101/012070","10.1101/2020.03.29.014761","10.1101/2020.03.04.977736","10.1101/2020.04.13.040196","10.1101/397802","10.1101/860049","10.1101/2020.04.21.051201","10.1101/2020.04.12.038109","10.1101/2020.04.18.047951","10.1101/424440","10.1101/2020.02.04.934232","10.1101/2020.02.13.945485","10.1101/078113","10.1101/2020.04.03.024885","10.1101/2020.01.23.917351","10.1101/737841","10.1101/326538","10.1101/2020.04.11.036806","10.1101/044594","10.1101/2020.03.30.016147","10.1101/301721","10.1101/675785","10.1101/331447","10.1101/2020.02.29.971101","10.1101/2020.02.21.959973","10.1101/418830","10.1101/2020.04.02.022186","10.1101/2020.04.06.027318","10.1101/714303","10.1101/724716","10.1101/2020.02.26.964882","10.1101/265199","10.1101/2020.02.19.955484","10.1101/2019.12.20.885590","10.1101/2020.03.05.975797","10.1101/2020.04.11.037093","10.1101/506410","10.1101/2020.04.13.038687","10.1101/628842","10.1101/347161","10.1101/631358","10.1101/212837","10.1101/2020.04.06.028688","10.1101/249995","10.1101/852608","10.1101/2020.03.13.990598","10.1101/2020.04.08.026948","10.1101/2019.12.21.885921","10.1101/2020.03.31.014639","10.1101/2020.04.10.022103","10.1101/629972","10.1101/2020.04.17.042366","10.1101/2020.02.12.945576","10.1101/2020.03.05.976167","10.1101/2020.03.02.972927","10.1101/312397","10.1101/2020.02.29.965418","10.1101/2020.03.04.976662","10.1101/496133","10.1101/2020.04.01.020941","10.1101/2020.02.27.967422","10.1101/2020.03.22.002204","10.1101/2020.04.02.021469","10.1101/571547","10.1101/2020.04.03.022723","10.1101/2020.04.07.028167","10.1101/2020.03.08.982637","10.1101/2020.04.03.024075","10.1101/434530","10.1101/066688","10.1101/2020.04.06.028316","10.1101/2020.04.10.035402","10.1101/246884","10.1101/261776","10.1101/588343","10.1101/2020.03.25.007534","10.1101/337253","10.1101/2020.04.19.049643","10.1101/469254","10.1101/2020.02.11.944462","10.1101/766717","10.1101/2020.04.03.022939","10.1101/2020.03.26.009993","10.1101/606236","10.1101/2019.12.18.881391","10.1101/2020.04.21.052639","10.1101/2020.04.20.050088","10.1101/2020.03.31.013268","10.1101/2020.04.19.048751","10.1101/370932","10.1101/840090","10.1101/585984","10.1101/2020.04.05.026187","10.1101/2020.03.25.006569","10.1101/2020.04.21.052084","10.1101/2020.01.29.925867","10.1101/582288","10.1101/2020.04.17.044743","10.1101/2020.03.29.014290","10.1101/847590","10.1101/2020.04.09.034462","10.1101/798546","10.1101/371914","10.1101/2020.03.16.993386","10.1101/2020.04.21.052530","10.1101/2020.03.04.976258","10.1101/2020.03.28.013607","10.1101/2020.03.12.988865","10.1101/2020.02.29.971093","10.1101/736678","10.1101/2020.03.31.015941","10.1101/2020.04.15.044073","10.1101/2020.04.20.052233","10.1101/703512","10.1101/068627","10.1101/471094","10.1101/2020.01.24.919159","10.1101/2020.04.10.035683","10.1101/2020.01.27.921809","10.1101/320929","10.1101/2020.04.17.046193","10.1101/2020.04.20.051011","10.1101/328674","10.1101/2020.02.22.951178","10.1101/065789","10.1101/2020.04.20.051656","10.1101/756767","10.1101/212704","10.1101/2020.03.11.986836","10.1101/2020.04.17.047480","10.1101/2020.03.11.983056","10.1101/2020.04.13.039263","10.1101/2020.04.18.047878","10.1101/562348","10.1101/385005","10.1101/612945","10.1101/248617","10.1101/2020.04.01.020966","10.1101/599043","10.1101/206789","10.1101/2020.04.06.026765","10.1101/444117","10.1101/548909","10.1101/629881","10.1101/2020.04.08.031203","10.1101/2020.02.18.954628","10.1101/502575","10.1101/555367","10.1101/2020.04.10.035717","10.1101/2020.03.27.003871","10.1101/2020.04.07.030544","10.1101/2020.04.16.043224","10.1101/2020.02.08.939553","10.1101/2020.03.22.002386","10.1101/2020.04.02.022079","10.1101/2020.04.07.029132","10.1101/2020.04.11.037473","10.1101/185876","10.1101/2020.04.15.043877","10.1101/2020.02.20.958272","10.1101/2020.04.03.023135","10.1101/2020.02.10.942185","10.1101/639369","10.1101/2020.02.11.944728","10.1101/2020.03.26.009605","10.1101/2020.03.16.994236","10.1101/2020.04.02.022384","10.1101/726935","10.1101/2020.02.10.942136","10.1101/2020.02.07.937862","10.1101/089128","10.1101/2020.04.08.029769","10.1101/704288","10.1101/074559","10.1101/2020.01.19.911669","10.1101/084129","10.1101/856518","10.1101/007476","10.1101/530618","10.1101/2020.04.07.029017","10.1101/800300","10.1101/2020.03.15.992818","10.1101/2020.03.19.997890","10.1101/848762","10.1101/2020.01.09.900555","10.1101/483693","10.1101/2020.04.09.034967","10.1101/865600","10.1101/2020.01.28.923169","10.1101/623132","10.1101/2020.03.03.972133","10.1101/153205","10.1101/331678","10.1101/2020.03.29.014209","10.1101/485060","10.1101/2020.02.10.936898","10.1101/2020.03.29.009464","10.1101/2020.04.07.030551","10.1101/659524","10.1101/2020.03.20.999029","10.1101/513333","10.1101/506188","10.1101/2020.03.30.015164","10.1101/010389","10.1101/443457","10.1101/2020.04.12.038216","10.1101/695510","10.1101/690917","10.1101/2020.03.10.986711","10.1101/2020.02.19.950253","10.1101/2020.03.23.002931","10.1101/2020.03.26.010165","10.1101/2020.03.20.001008","10.1101/2020.03.14.992156","10.1101/2020.04.20.052068","10.1101/2020.04.10.036533","10.1101/2020.04.21.053199","10.1101/2020.04.10.036335","10.1101/323915","10.1101/2020.02.05.935387","10.1101/574020","10.1101/2020.04.13.036079","10.1101/753004","10.1101/791400","10.1101/309021","10.1101/442996","10.1101/2020.04.03.020602","10.1101/2020.03.25.008482","10.1101/2020.01.30.927889","10.1101/2020.04.14.028407","10.1101/441154","10.1101/2020.03.28.012617","10.1101/2020.03.28.013920","10.1101/017178","10.1101/2020.03.07.981928","10.1101/545087","10.1101/2020.04.16.045617","10.1101/2020.03.16.993584","10.1101/850800","10.1101/2020.04.01.019299","10.1101/251322","10.1101/2020.03.30.015347","10.1101/2020.04.10.036558","10.1101/405811","10.1101/2020.04.07.029447","10.1101/790444","10.1101/2020.03.31.018499","10.1101/300129","10.1101/242495","10.1101/041558","10.1101/2020.02.27.968008","10.1101/2020.01.22.915660","10.1101/226175","10.1101/2020.01.29.924100","10.1101/2020.03.13.990036","10.1101/331835","10.1101/240408","10.1101/2020.04.07.028589","10.1101/2020.03.24.005983","10.1101/267336","10.1101/751578","10.1101/2020.02.12.946087","10.1101/2020.02.12.946772","10.1101/2020.04.02.019075","10.1101/466102","10.1101/054072","10.1101/462341","10.1101/798702","10.1101/2020.02.26.966143","10.1101/363713","10.1101/2020.03.17.995639","10.1101/552356","10.1101/2020.01.31.929042","10.1101/2020.04.06.027961","10.1101/2020.04.14.041228","10.1101/053421","10.1101/2020.04.08.032466","10.1101/2020.04.14.041459","10.1101/2020.04.09.034181","10.1101/2020.04.21.053009","10.1101/852657","10.1101/2020.03.31.018978","10.1101/304618","10.1101/2020.01.27.921536","10.1101/556951","10.1101/541698","10.1101/2020.04.11.034603","10.1101/434027","10.1101/2020.04.03.003699","10.1101/742122"],"titles":["SPINT2 inhibits proteases involved in activation of both influenza viruses and metapneumoviruses","FEP-based screening prompts drug repositioning against COVID-19","Ruler elements in chromatin remodelers set nucleosome array spacing and phasing","Characterization and treatment of SARS-CoV-2 in nasal and bronchial human airway epithelia","Structural and functional analysis of a potent sarbecovirus neutralizing antibody","Solubility-Weighted Index: fast and accurate prediction of protein solubility","An iterative and automated computational pipeline for untargeted strain-level identification using MS/MS spectra from pathogenic samples","All-in-One Dual CRISPR-Cas12a (AIOD-CRISPR) Assay: A Case for Rapid, Ultrasensitive and Visual Detection of Novel Coronavirus SARS-CoV-2 and HIV virus","VPTMdb: a viral post-translational modification database","Reprogrammed Pteropus Bat Stem Cells Present Distinct Immune Signature and are Highly Permissive for Henipaviruses","Existing host range mutations constrain further emergence of RNA viruses","Validation of a Lysis Buffer Containing 4 M Guanidinium Thiocyanate (GITC)/ Triton X-100 for Extraction of SARS-CoV-2 RNA for COVID-19 Testing: Comparison of Formulated Lysis Buffers Containing 4 to 6 M GITC, Roche External Lysis Buffer and Qiagen RTL Lysis Buffer","Ancient RNA virus epidemics through the lens of recent adaptation in human genomes","Viral etiology of Acute Respiratory Infections in Hospitalized Children in Novosibirsk City, Russia (2013 \u2013 2017)","Lipocalin-2 is a Sensitive and Specific Marker of Bacterial Infection in Children","Lectin-like Intestinal Defensin Inhibits 2019-nCoV Spike binding to ACE2","Cell-intrinsic differences between human airway epithelial cells from children and adults","Expanding the size limit of RNA viruses: Evidence of a novel divergent nidovirus in California sea hare, with a ~35.9 kb virus genome","Antipsychotic behavioral phenotypes in the mouse Collaborative Cross recombinant inbred inter-crosses (RIX)","The Viral Protein Corona Directs Viral Pathogenesis and Amyloid Aggregation","Analyzing Vaccine Trials in Epidemics with Mild and Asymptomatic Infection","A novel totivirus alters gene expression and vacuolar morphology in Malassezia cells and induces a TLR3-mediated inflammatory immune response","A Mac2-positive progenitor-like microglial population survives independent of CSF1R signaling in adult mouse brain","Kallikrein 13: a new player in coronaviral infections","Imaging Striatal Dopamine Release Using a Non-Genetically Encoded Near-Infrared Fluorescent Catecholamine Nanosensor","Identification of potential vaccine candidates against SARS-CoV-2, A step forward to fight novel coronavirus 2019-nCoV: A Reverse Vaccinology Approach","Quantifying the roles of vomiting, diarrhea, and residents vs. staff in norovirus transmission in U.S. nursing home outbreaks","TALC: Transcription Aware Long Read Correction","Non-neural expression of SARS-CoV-2 entry genes in the olfactory epithelium suggests mechanisms underlying anosmia in COVID-19 patients","Characterizing geographical and temporal dynamics of novel coronavirus SARS-CoV-2 using informative subtype markers","Sequencing of the human IG light chain loci from a hydatidiform mole BAC library reveals locus-specific signatures of genetic diversity","Designing a multi-epitope peptide-based vaccine against SARS-CoV-2","Translational adaptation of human viruses to the tissues they infect","Increasing Host Cellular Receptor\u2014Angiotensin-Converting Enzyme 2 (ACE2) Expression by Coronavirus may Facilitate 2019-nCoV Infection","Room-temperature-storable PCR Mixes for SARS-CoV-2 Detection","Re-insights into origin and adaptation of SARS-CoV-2","A proposal of alternative primers for the ARTIC Network's multiplex PCR to improve coverage of SARS-CoV-2 genome sequencing","Global dynamics of a general vector-borne disease model with two transmission routes","The scRNA-seq expression profiling of the receptor ACE2 and the cellular protease TMPRSS2 reveals human organs susceptible to COVID-19 infection","External location of touch is constructed post-hoc based on limb choice","Inhibition of PIKfyve kinase prevents infection by EBOV and SARS-CoV-2","How domain growth is implemented determines the long term behaviour of a cell population through its effect on spatial correlations","Dysregulation of Long Non-coding RNA (lncRNA) Genes and Predicted lncRNA-Protein Interactions during Zika Virus Infection","Mechanistic modeling of the SARS-CoV-2 disease map","Nucleotide Analogues as Inhibitors of SARS-CoV Polymerase","Monoclonal antibodies for the S2 subunit of spike of SARS-CoV cross-react with the newly-emerged SARS-CoV-2","Prediction of SARS-CoV-2 epitopes across 9360 HLA class I alleles","Breaking down of the healthcare system: Mathematical modelling for controlling the novel coronavirus (2019-nCoV) outbreak in Wuhan, China","Nucleotide Analogues as Inhibitors of Viral Polymerases","LY6E impairs coronavirus fusion and confers immune control of viral disease","An artificial intelligence-based first-line defence against COVID-19: digitally screening citizens for risks via a chatbot","Loss of IKK subunits limits NF-\u03baB signaling in reovirus infected cells","TWIRLS, an automated topic-wise inference method based on massive literature, suggests a possible mechanism via ACE2 for the pathological changes in the human host after coronavirus infection","Combined prophylactic and therapeutic use maximizes hydroxychloroquine anti-SARS-CoV-2 effects in vitro","Supramolecular Organization Predicts Protein Nanoparticle Delivery to Neutrophils for Acute Lung Inflammation Diagnosis and Treatment","Interpretable detection of novel human viruses from genome sequencing data","Site-specific N-glycosylation Characterization of Recombinant SARS-CoV-2 Spike Proteins using High-Resolution Mass Spectrometry","SIANN: Strain Identification by Alignment to Near Neighbors","Inactivating porcine coronavirus before nuclei acid isolation with the temperature higher than 56 \u00b0C damages its genome integrity seriously","SKEMPI 2.0: An updated benchmark of changes in protein-protein binding energy, kinetics and thermodynamics upon mutation","MUC5AC drives COPD exacerbation severity through amplification of virus-induced airway inflammation","Relevance of enriched expression of SARS-CoV-2 binding receptor ACE2 in gastrointestinal tissue with pathogenesis of digestive symptoms, diabetes-associated mortality, and disease recurrence in COVID-19 patients","Infectious bronchitis virus regulates cellular stress granule signaling","Genomic convergence of locus-based GWAS meta-analysis identifies DDX11 as a novel Systemic Lupus Erythematosus gene","A new phylogenetic protocol: Dealing with model misspecification and confirmation bias in molecular phylogenetics","COVID-19 Vaccine Candidates: Prediction and Validation of 174 SARS-CoV-2 Epitopes","A comprehensive annotation and differential expression analysis of short and long non-coding RNAs in 16 bat genomes","Preliminary estimation of the basic reproduction number of novel coronavirus (2019-nCoV) in China, from 2019 to 2020: A data-driven analysis in the early phase of the outbreak","Fibrinolytic niche is requested for alveolar type 2 cell-mediated alveologenesis and injury repair","Emergence of SARS-CoV-2 through Recombination and Strong Purifying Selection","A Cryptic Site of Vulnerability on the Receptor Binding Domain of the SARS-CoV-2 Spike Glycoprotein","Crystal structure of the giant panda MHC class I complex: first insights into the viral peptide presentation profile in the bear family","Inferring the presence of aflatoxin-producing Aspergillus flavus strains using RNA sequencing and electronic probes as a transcriptomic screening tool","Modelling the epidemic trend of the 2019 novel coronavirus outbreak in China","Functional pangenome analysis suggests inhibition of the protein E as a readily available therapy for COVID-2019","A dynamic nomenclature proposal for SARS-CoV-2 to assist genomic epidemiology","Bat coronavirus phylogeography in the Western Indian Ocean","Predicting the global mammalian viral sharing network using phylogeography","TMPRSS2 and furin are both essential for proteolytic activation and spread of SARS-CoV-2 in human airway epithelial cells and provide promising drug targets","Potent neutralization of 2019 novel coronavirus by recombinant ACE2-Ig","Risk of disease spillover from dogs to wild carnivores in Kanha Tiger Reserve, India","The origin and underlying driving forces of the SARS-CoV-2 outbreak","Boceprevir, GC-376, and calpain inhibitors II, XII inhibit SARS-CoV-2 viral replication by targeting the viral main protease","Molecular Modeling Evaluation of the Binding Effect of Ritonavir, Lopinavir and Darunavir to Severe Acute Respiratory Syndrome Coronavirus 2 Proteases","Transferrin binding protein B and Transferrin binding protein A 2 expand the transferrin recognition range of Histophilus somni","CoV Genome Tracker: tracing genomic footprints of Covid-19 pandemic","Noroviruses subvert the core stress granule component G3BP1 to promote viral VPg-dependent translation","Using directed attenuation to enhance vaccine immunity","Profiling the epigenome at home","Speed and strength of an epidemic intervention","An Effective CTL Peptide Vaccine for Ebola Zaire Based on Survivors\u2019 CD8+ Targeting of a Particular Nucleocapsid Protein Epitope with Potential Implications for COVID-19 Vaccine Design","RNA genome conservation and secondary structure in SARS-CoV-2 and SARS-related viruses","Immunity-and-Matrix-Regulatory Cells Derived from Human Embryonic Stem Cells Safely and Effectively Treat Mouse Lung Injury and Fibrosis","Sulforhodamine B and exogenous surfactant effects on alveolar surface tension in acute respiratory distress syndrome models","Comparison of SARS-CoV-2 infections among 3 species of non-human primates","Leverging Deep Learning to Simulate Coronavirus Spike proteins has the potential to predict future Zoonotic sequences","Metagenomic Nanopore sequencing of influenza virus direct from clinical respiratory samples","Isolation, sequence, infectivity and replication kinetics of SARS-CoV-2","Generating genomic platforms to study Candida albicans pathogenesis","Computational simulations reveal the binding dynamics between human ACE2 and the receptor binding domain of SARS-CoV-2 spike protein","Encoding of odor fear memories in the mouse olfactory cortex","Coronavirus S protein-induced fusion is blocked prior to hemifusion by Abl kinase inhibitors","First evidence of host range expansion in virophages and its potential impact on giant viruses and host cells","More than efficacy revealed by single-cell analysis of antiviral therapeutics","Humanized Single Domain Antibodies Neutralize SARS-CoV-2 by Targeting Spike Receptor Binding Domain","Evolutionary proteomics uncovers ciliary signaling components","The Potential Use of Unprocessed Sample for RT-qPCR Detection of COVID-19 without an RNA Extraction Step","The severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) envelope (E) protein harbors a conserved BH3-like motif","Single cell RNA sequencing of 13 human tissues identify cell types and receptors of human coronaviruses","The Infectious Bronchitis Virus Coronavirus Envelope Protein Alters Golgi pH to Protect Spike Protein and Promote Release of Infectious Virus","Evaluating an Upper Respiratory Disease Panel on the Portable MinION Sequencer","Multivariate Analyses of Codon Usage of SARS-CoV-2 and other betacoronaviruses","Structure-based modeling of SARS-CoV-2 peptide/HLA-A02 antigens","Anti-microbial immunity is impaired in COPD patients with frequent exacerbations","Phylogenetic Analysis of the Novel Coronavirus Reveals Important Variants in Indian Strains","Realtime 2-5A kinetics suggests interferons \u03b2 and \u03bb evade global arrest of translation by RNase L","An unsupervised algorithm for host identification in flaviviruses","SARS-CoV-2 detection with CRISPR diagnostics","The permeabilized SecY protein-translocation channel can serve as a nonspecific sugar transporter","A novel bat coronavirus reveals natural insertions at the S1/S2 cleavage site of the Spike protein and a possible recombinant origin of HCoV-19","The accelerated infectious disease risk in the Anthropocene: more outbreaks and wider global spread","Chloroquine Kills Hair Cells in Zebrafish Lateral Line and Murine Cochlear Cultures: Implications for Ototoxicity","Comparative analysis of gene expression in virulent and attenuated strains of infectious bronchitis virus at sub-codon resolution","Quality control implementation for universal characterization of DNA and RNA viruses in clinical respiratory samples using single metagenomic next-generation sequencing workflow","Distinct Structural Flexibility within SARS-CoV-2 Spike Protein Reveals Potential Therapeutic Targets","XRN1 is a Species-Specific Virus Restriction Factor in Yeasts","Evidence for RNA editing in the transcriptome of 2019 Novel Coronavirus","Rapid Therapeutic Recommendations in the Context of a Global Public Health Crisis using Translational Bioinformatics Approaches: A proof-of-concept study using Nipah Virus Infection","The SARS-CoV-2 exerts a distinctive strategy for interacting with the ACE2 human receptor","Classical drug digitoxin inhibits influenza cytokine storm, with implications for COVID-19 therapy.","A diallel of the mouse Collaborative Cross founders reveals strong strain-specific maternal effects on litter size","Establishing Prokaryotic Expression System of Angiotensin-Converting Enzyme 2 (ACE2) gene in pigs","Evolved sequence features within the intrinsically disordered tail influence FtsZ assembly and bacterial cell division","Chemogenetic evidence that rapid neuronal de novo protein synthesis is required for consolidation of long-term memory","\u03a6X174 Attenuation by Whole Genome Codon Deoptimization","A protocol for adding knowledge to Wikidata, a case report","Crystal structure of Nsp15 endoribonuclease NendoU from SARS-CoV-2","Analysis of SARS-CoV-2-controlled autophagy reveals spermidine, MK-2206, and niclosamide as putative antiviral therapeutics","Structural basis of RNA recognition by the SARS-CoV-2 nucleocapsid phosphoprotein","Complete genome characterisation of a novel coronavirus associated with severe human respiratory disease in Wuhan, China","Prophage genomics reveals patterns in phage genome organization and replication","Diversity of Archaea Domain in Cuatro Cienegas Basin: Archaean Domes","Glycoinformatics approach for identifying target positions to inhibit initial binding of SARS-CoV-2 S1 protein to the host cell","Big city, small world: Density, contact rates, and transmission of dengue across Pakistan.","Scientometric Trends for Coronaviruses and Other Emerging Viral Infections","Structural Basis for the Inhibition of the RNA-Dependent RNA Polymerase from SARS-CoV-2 by Remdesivir","An anti-cancer binary system activated by bacteriophage HK022 Integrase","ACE 2 Coding Variants: A Potential X-linked Risk Factor for COVID-19 Disease","In silico study of the spike protein from SARS-CoV-2 interaction with ACE2: similarity with SARS-CoV, hot-spot analysis and effect of the receptor polymorphism","A simple magnetic nanoparticles-based viral RNA extraction method for efficient detection of SARS-CoV-2","Comparative analyses of SAR-CoV2 genomes from different geographical locations and other coronavirus family genomes reveals unique features potentially consequential to host-virus interaction and pathogenesis","Evolution and variation of 2019-novel coronavirus","Anesthetic agents affect urodynamic parameters and anesthetic depth at doses necessary to facilitate preclinical testing in felines","Analysis of Serologic Cross-Reactivity Between Common Human Coronaviruses and SARS-CoV-2 Using Coronavirus Antigen Microarray","MERS-CoV NSP16 necessary for IFN resistance and viral pathogenesis","Analysis of SARS-CoV-2 Antibodies in COVID-19 Convalescent Plasma using a Coronavirus Antigen Microarray","A highly conserved cryptic epitope in the receptor-binding domains of SARS-CoV-2 and SARS-CoV","Realized generation times: contraction and impact of infectious period, reproduction number and population size","Cryo-EM structures reveal transcription initiation steps by yeast mitochondrial RNA polymerase","Neuronal CCL2 expression drives inflammatory monocyte infiltration into the brain during acute virus infection","High Prevalence of Strongyloidiasis in Spain: A Hospital-Based Study","EPS8 facilitates uncoating of influenza A virus","Clearing the FoG: Antifungal tolerance is a subpopulation effect that is distinct from resistance and is associated with persistent candidemia","Deconvoluting Virome-Wide Antiviral Antibody Profiling Data","De novo 3D models of SARS-CoV-2 RNA elements and small-molecule-binding RNAs to guide drug discovery","The transcriptional and translational landscape of equine torovirus","Stress-Induced Transcriptional Memory Accelerates Promoter-Proximal Pause-Release and Decelerates Termination over Mitotic Divisions","TLR5 participates in the TLR4 receptor complex and biases towards MyD88-dependent signaling in environmental lung injury","Genotypic diversity, circulation patterns, and co-detections among rhinoviruses in Queensland, 2001","Comparative genomics suggests limited variability and similar evolutionary patterns between major clades of SARS-Cov-2","Airway IRF7hi versus IRF7lo molecular response patterns determine clinical phenotypes in children with acute wheezing","A novel real-time PCR assay panel for detection of common respiratory pathogens in a convenient, strip-tube array format","Design: An assay based on single-polypeptide-chain heterodimeric A2AR/D2R and non-oligomerized fusions for in vivo analysis of their allosteric receptor-receptor interactions","Origins and Evolution of the Global RNA Virome","PLANT-Dx: A Molecular Diagnostic for Point of Use Detection of Plant Pathogens","Evidence of the Recombinant Origin and Ongoing Mutations in Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2)","Immunoglobulin fragment F(ab\u2019)2 against RBD potently neutralizes SARS-CoV-2 in vitro","Metagenomic Sequencing Detects Respiratory Pathogens in Hematopoietic Cellular Transplant Patients","scRNA-seq reveals ACE2 and TMPRSS2 expression in TROP2+ Liver Progenitor Cells: Implications in COVID-19 associated Liver Dysfunction","Spreading predictability in complex networks","Development of CRISPR as a prophylactic strategy to combat novel coronavirus and influenza","A Large-scale Drug Repositioning Survey for SARS-CoV-2 Antivirals","Automated collection of pathogen-specific diagnostic data for real-time syndromic epidemiological studies","Therapeutic Drugs Targeting 2019-nCoV Main Protease by High-Throughput Screening","Distinct spread of DNA and RNA viruses among mammals amid prominent role of domestic species","Connectivity analyses of bioenergetic changes in schizophrenia: Identification of novel treatments","p53 is not necessary for DUX4 pathology","Mouse hepatitis virus nsp14 exoribonuclease activity is required for resistance to innate immunity","Accelerated viral dynamics in bat cell lines, with implications for zoonotic emergence","Structural basis for the recognition of the 2019-nCoV by human ACE2","Automatic Identification of SARS Coronavirus using Compression-Complexity Measures","Evolutionary rate shifts suggest species-specific adaptation events in HIV-1 and SIV","Analysis of the mutation dynamics of SARS-CoV-2 reveals the spread history and emergence of RBD mutant with lower ACE2 binding affinity","Reinfection could not occur in SARS-CoV-2 infected rhesus macaques","Novel Function of Bluetongue Virus NS3 Protein in Regulation of the MAPK/ERK Signaling Pathway","Inferring generation-interval distributions from contact-tracing data","The Nucleocapsid Protein of SARS-CoV-2 Abolished Pluripotency in Human Induced Pluripotent Stem Cells","Messenger RNAs transcribed from yeast linear cytoplasmic plasmids possess unconventional 5\u2019 and 3\u2019 UTRs and suggest a novel mechanism of translation","Immuno-informatics Characterization SARS-CoV-2 Spike Glycoprotein for Prioritization of Epitope based Multivalent Peptide Vaccine","Sequence-based prediction of vaccine targets for inducing T cell responses to SARS-CoV-2 utilizing the bioinformatics predictor RECON","Improving early epidemiological assessment of emerging Aedes-transmitted epidemics using historical data","Interplay of host regulatory network on SARS-CoV-2 binding and replication machinery","Protein structure and sequence re-analysis of 2019-nCoV genome does not indicate snakes as its intermediate host or the unique similarity between its spike protein insertions and HIV-1","deepMINE - Natural Language Processing based Automatic Literature Mining and Research Summarization for Early-Stage Comprehension in Pandemic Situations specifically for COVID-19","Enhanced replication of mouse adenovirus type 1 following virus-induced degradation of protein kinase R (PKR)","Structure of the chromatin remodelling enzyme Chd1 bound to a ubiquitinylated nucleosome","Potent neutralizing antibodies in the sera of convalescent COVID-19 patients are directed against conserved linear epitopes on the SARS-CoV-2 spike protein","Alpha Herpesvirus Egress and Spread from Neurons Uses Constitutive Secretory Mechanisms Independent of Neuronal Firing Activity","A novel framework for inferring parameters of transmission from viral sequence data","Decrease in ACE2 mRNA expression in aged mouse lung","High-throughput interrogation of programmed ribosomal frameshifting in human cells","Combination attenuation offers strategy for live-attenuated coronavirus vaccines","Modeling of Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) Proteins by Machine Learning and Physics-Based Refinement","Fast assessment of human receptor-binding capability of 2019 novel coronavirus (2019-nCoV)","Topoisomerase III-\u00df is required for efficient replication of positive-sense RNA viruses","Atazanavir inhibits SARS-CoV-2 replication and pro-inflammatory cytokine production","Broad and differential animal ACE2 receptor usage by SARS-CoV-2","SARS-CoV-2 might manipulate against its host the immunity RNAi/Dicer/Ago system Does mitochondria collapse upon COVID-19 infection?","Cryo-EM structures of HKU2 and SADS-CoV spike glycoproteins and insights into coronavirus evolution","Lessening of porcine epidemic diarrhoea virus susceptibility in piglets after editing of the CMP-N-glycolylneuraminic acid hydroxylase gene with CRISPR/Cas9 to nullify N-glycolylneuraminic acid expression","Comparison of alternative models of human movement and the spread of disease","X-ray Structure of Main Protease of the Novel Coronavirus SARS-CoV-2 Enables Design of \u03b1-Ketoamide Inhibitors","SARS-CoV-2 detection from nasopharyngeal swab samples without RNA extraction","Epidemiological and ecological modelling reveal diversity in upper respiratory tract microbial population structures from a cross-sectional community swabbing study","A Highly Scalable and Rapidly Deployable RNA Extraction-Free COVID-19 Assay by Quantitative Sanger Sequencing","The effect of variant interference on de novo assembly for viral deep sequencing","Modeling Epidemics: A Primer and Numerus Software Implementation","HIV-1 Protease Evolvability is Affected by Synonymous Nucleotide Recoding","Enhancing the Antiviral Efficacy of RNA-Dependent RNA Polymerase Inhibition by Combination with Modulators of Pyrimidine Metabolism","RNA-dependent RNA polymerase speed and fidelity are not the only determinants of the mechanism or efficiency of recombination","The impact of persistent bacterial bronchitis on the pulmonary microbiome of children","Atlas of ACE2 gene expression in mammals reveals novel insights in transmission of SARS-Cov-2","Direct nanopore sequencing of mRNA reveals landscape of transcript isoforms in apicomplexan parasites","Recapitulation of SARS-CoV-2 Infection and Cholangiocyte Damage with Human Liver Organoids","Reversal of Infected Host Gene Expression Identifies Repurposed Drug Candidates for COVID-19","JAK1 inhibition blocks lethal sterile immune responses: implications for COVID-19 therapy","Azithromycin and ciprofloxacin have a chloroquine-like effect on respiratory epithelial cells","An interaction network of RNA-binding proteins involved in Drosophila oogenesis","Kinetic Analysis of Bacteriophage Sf6 Binding to Outer Membrane Protein A Using Whole Virions","Isolation and sequencing of AGO-bound RNAs reveals characteristics of mammalian stem-loop processing in vivo","SARS-CoV-2, an evolutionary perspective of interaction with human ACE2 reveals undiscovered amino acids necessary for complex stability","The need to connect: Acute social isolation causes neural craving responses similar to hunger","Influenza A virus surface proteins are organized to help penetrate host mucus","Time-varying transmission dynamics of Novel Coronavirus Pneumonia in China","Sequence analysis of SARS-CoV-2 genome reveals features important for vaccine design","Virus-host interactome and proteomic survey of PMBCs from COVID-19 patients reveal potential virulence factors influencing SARS-CoV-2 pathogenesis","A case for a reverse-frame coding sequence in a group of positive-sense RNA viruses","VIDHOP, viral host prediction with Deep Learning","DNA-scaffolded biomaterials enable modular and tunable control of cell-based cancer immunotherapies","The translational landscape of Zika virus during infection of mammalian and insect cells","Attenuation of influenza A virus disease severity by viral co-infection in a mouse model","The impact of news exposure on collective attention in the United States during the 2016 Zika epidemic","Concerted 2-5A-Mediated mRNA Decay and Transcription Reprogram Protein Synthesis in dsRNA Response","G-quadruplex stabilization in the ions and maltose transporters inhibit Salmonella enterica growth and virulence","MxB restricts HIV-1 by targeting the tri-hexamer interface of the viral capsid","Exploring Conformational Transition of 2019 Novel Coronavirus Spike Glycoprotein Between Its Closed and Open States Using Molecular Dynamics Simulations","CHEER: hierarCHical taxonomic classification for viral mEtagEnomic data via deep leaRning","Adaptive Estimation for Epidemic Renewal and Phylogenetic Skyline Models","MERS-CoV spillover at the camel-human interface","Recombinant rotaviruses rescued by reverse genetics reveal the role of NSP5 hyperphosphorylation in the assembly of viral factories","Modulation of metabolic functions through Cas13d-mediated gene knockdown in liver","Infectious bronchitis virus attaches to lipid rafts and enters cells via clathrin mediated endocytosis","Three adjacent nucleotide changes spanning two residues in SARS-CoV-2 nucleoprotein: possible homologous recombination from the transcription-regulating sequence","The protein expression profile of ACE2 in human tissues","Integrated analyses of single-cell atlases reveal age, gender, and smoking status associations with cell type-specific expression of mediators of SARS-CoV-2 viral entry and highlights inflammatory programs in putative target cells","Improving COVID-19 Testing Efficiency using Guided Agglomerative Sampling","Structure-Based Design, Synthesis and Biological Evaluation of Peptidomimetic Aldehydes as a Novel Series of Antiviral Drug Candidates Targeting the SARS-CoV-2 Main Protease","The Role of Interleukin-1 cytokine family (IL-1\u03b2, IL-37) and interleukin-12 cytokine family (IL-12, IL-35) in eumycetoma infection pathogenesis","Nelfinavir was predicted to be a potential inhibitor of 2019-nCov main protease by an integrative approach combining homology modelling, molecular docking and binding free energy calculation","Waterborne, abiotic and other indirectly transmitted (W.A.I.T.) infections are defined by the dynamics of free-living pathogens and environmental reservoirs","PathoLive - Real-time pathogen identification from metagenomic Illumina datasets","Identification of a Zika NS2B-NS3pro pocket susceptible to allosteric inhibition by small molecules including qucertin rich in edible plants","Characterization of the SARS-CoV-2 Spike in an Early Prefusion Conformation","Proofreading-deficient coronaviruses adapt over long-term passage for increased fidelity and fitness without reversion of exoribonuclease-inactivating mutations","The RZZ complex integrates spindle checkpoint maintenance with dynamic expansion of unattached kinetochores","Mucin 4 Protects Female Mice from Coronavirus Pathogenesis","ZODIAC: database-independent molecular formula annotation using Gibbs sampling reveals unknown small molecules","Correlation of mRNA delivery timing and protein expression in lipid-based transfection","Real-time projections of epidemic transmission and estimation of vaccination impact during an Ebola virus disease outbreak in Northeastern Democratic Republic of Congo","Neural organization of speech production: A lesion-based study of error patterns in connected speech","Structure of dimeric full-length human ACE2 in complex with B0AT1","Determination of host cell proteins constituting the molecular microenvironment of coronavirus replicase complexes by proximity-labeling","Inhibition of SARS-CoV-2 infection (previously 2019-nCoV) by a highly potent pan-coronavirus fusion inhibitor targeting its spike protein that harbors a high capacity to mediate membrane fusion","GraphDTA: Predicting drug\u2013target binding affinity with graph neural networks","Pandemic Publishing: Medical journals drastically speed up their publication process for Covid-19","Significance of hydrophobic and charged sequence similarities in sodium-bile acid cotransporter and vitamin D-binding protein macrophage activating factor","Changes in temperature alter susceptibility to a virus following a host shift","Site-specific analysis of the SARS-CoV-2 glycan shield","Knowledge synthesis from 100 million biomedical documents augments the deep expression profiling of coronavirus receptors","A method for RNA structure prediction shows evidence for structure in lncRNAs","Astrovirus in Reunion Free-tailed Bat (Mormopterus francoismoutoui)","Downregulation of hippocampal NR2A/2B subunits related to cognitive impairment in a pristane-induced lupus BALB/c mice","Influence of different glycoproteins and of the virion core on SERINC5 antiviral activity","Synergetic effect of non-complementary 5\u2019 AT-rich sequences on the development of a multiplex TaqMan real-time PCR for specific and robust detection of Clavibacter michiganensis and C. michiganensis subsp. nebraskensis","Investigating the genomic landscape in novel coronavirus (2019-nCoV) genomes to identify non-synonymous mutations for use in diagnosis and drug design","Synonymous mutations and the molecular evolution of SARS-Cov-2 origins","The inhaled corticosteroid ciclesonide blocks coronavirus RNA replication by targeting viral NSP15","SARS-CoV-2 and SARS-CoV Spike-RBD Structure and Receptor Binding Comparison and Potential Implications on Neutralizing Antibody and Vaccine Development","Biophysical characterization of the SARS-CoV2 spike protein binding with the ACE2 receptor explains increased COVID-19 pathogenesis","Structural analysis of SARS-CoV-2 and prediction of the human interactome","Preliminary identification of potential vaccine targets for the COVID-19 coronavirus (SARS-CoV-2) based on SARS-CoV immunological studies","Human, Nonhuman Primate, and Bat Cells Are Broadly Susceptible to Tibrovirus Particle Cell Entry","The host antiviral ribonuclease L protein supports Zika virus replication factory formation to enhance infectious virus production","Fatal toxicity of chloroquine or hydroxychloroquine with metformin in mice","Genome-wide Prediction of microRNAs in Zika virus Genomes Reveals Possible Interactions with Human Genes Involved in the Nervous System Development","Gene birth contributes to structural disorder encoded by overlapping genes","Identification of viruses with the potential to infect human","Structurally informed evolutionary models improve phylogenetic reconstruction for emerging, seasonal, and pandemic influenza viruses","Strong evolutionary convergence of receptor-binding protein spike between COVID-19 and SARS-related coronaviruses","Trypsin treatment unlocks barrier for zoonotic coronaviruses infection","Extensive genetic diversity of bat-borne polyomaviruses reveals inter-family host-switching events","International travelers and genomics uncover a \u2018hidden\u2019 Zika outbreak","Quantifying transmission of emerging zoonoses: Using mathematical models to maximize the value of surveillance data","In-depth Bioinformatic Analyses of Human SARS-CoV-2, SARS-CoV, MERS-CoV, and Other Nidovirales Suggest Important Roles of Noncanonical Nucleic Acid Structures in Their Lifecycles","A molecular cell atlas of the human lung from single cell RNA sequencing","A single locus underlies variation in Caenorhabditis elegans chemotherapeutic responses","A transcription regulatory network within the ACE2 locus may promote a pro-viral environment for SARS-CoV-2 by modulating expression of host factors","LAMP-Seq: Population-Scale COVID-19 Diagnostics Using a Compressed Barcode Space","A combinatorial biomolecular strategy to identify peptides for improved transport across the sputum of cystic fibrosis patients and the underlying epithelia","Potential microenvironment of SARS-CoV-2 infection in airway epithelial cells revealed by Human Protein Atlas database analysis","The 2019-new coronavirus epidemic: evidence for virus evolution","Phylodynamics of SARS-CoV-2 transmission in Spain","Prediction of SARS-CoV interaction with host proteins during lung aging reveals a potential role for TRIB3 in COVID-19","protein-sol pKa: prediction of electrostatic frustration, with application to coronaviruses","Identification of a Pangolin Niche for a 2019-nCoV-like Coronavirus via an Extensive Meta-metagenomic Search","Dissecting genetic and sex-specific host heterogeneity in pathogen transmission potential","SARS-CoV-2 exhibits intra-host genomic plasticity and low-frequency polymorphic quasispecies","Programmable low-cost DNA-based platform for viral RNA detection","Nelfinavir inhibits replication of severe acute respiratory syndrome coronavirus 2 in vitro.","A systematic review of MERS-CoV (Middle East Respiratory Syndrome Coronavirus) seroprevalence and viral RNA prevalence in dromedary camels: implications for animal vaccination","Evolutionary origins of epidemic potential among human RNA viruses","Inhibition of arterivirus RNA synthesis by cyclophilin inhibitors is counteracted by mutations in replicase transmembrane subunits","Monocyte-derived Prostaglandin E2 inhibits antigen-specific cutaneous immunity during ageing","The target landscape of N4-hydroxycytidine based on its chemical neighborhood","How to make more from exposure data? An integrated machine learning pipeline to predict pathogen exposure","Decoupling the effects of nutrition, age and behavioral caste on honey bee physiology and immunity","Viruses are a dominant driver of protein adaptation in mammals","Serum-dependent and independent regulation of PARP2","Python nidoviruses, more than respiratory pathogens","Tissue Tropism and Transmission Ecology Predict Virulence of Human RNA Viruses","Molecular Detection of SARS-CoV-2 in Formalin Fixed Paraffin Embedded Specimens","SARS-CoV-2 is sensitive to type I interferon pretreatment","Animal virus ecology and evolution are shaped by the virus host-body infiltration and colonization pattern.","In silico approach to accelerate the development of mass spectrometry-based proteomics methods for detection of viral proteins: Application to COVID-19","HOPS-dependent endosomal fusion required for efficient cytosolic delivery of therapeutic peptides and small proteins","A unifying structural and functional model of the coronavirus replication organelle: tracking down RNA synthesis","Functional Immune Deficiency Syndrome via Intestinal Infection in COVID-19","Awareness of MERS-CoV among Staff Members of Prince Sultan Military Medical City in Riyadh, Saudi Arabia","Nucleotide Analogues as Inhibitors of SARS-CoV-2 Polymerase","Deep sequencing of primary human lung epithelial cells challenged with H5N1 influenza virus reveals a proviral role for CEACAM1","Systemic analysis of tissue cells potentially vulnerable to SARS-CoV-2 infection by the protein-proofed single-cell RNA profiling of ACE2, TMPRSS2 and Furin proteases","COVID-19 coronavirus vaccine design using reverse vaccinology and machine learning","MERS-CoV recombination: implications about the reservoir and potential for adaptation","A blueprint for the implementation of a validated approach for the detection of SARS-Cov2 in clinical samples in academic facilities","Differential Antibody Recognition by Novel SARS-CoV-2 and SARS-CoV Spike Protein Receptor Binding Domains: Mechanistic Insights and Implications for the Design of Diagnostics and Therapeutics","The MHC class-II HLA-DR receptor mediates bat influenza A-like H17N10 virus entry into mammalian cells","Virome heterogeneity and connectivity in waterfowl and shorebird communities","Transmission of human-associated microbiota along family and social networks","Expression of 9-O- and 7,9-O-acetyl modified sialic acid in cells and their effects on influenza viruses","Quantifying the Risk and Cost of Active Monitoring for Infectious Diseases","Evidence of recombination in coronaviruses implicating pangolin origins of nCoV-2019","Design of potent membrane fusion inhibitors against SARS-CoV-2, an emerging coronavirus with high fusogenic activity","SARS-CoV-2 is transmitted via contact and via the air between ferrets","Novel tetraplex qPCR assays for simultaneous detection and identification of Xylella fastidiosa subspecies in plant tissues","SARS-coronavirus-2 replication in Vero E6 cells: replication kinetics, rapid adaptation and cytopathology","Bioinformatic characterization of angiotensin-converting enzyme 2, the entry receptor for SARS-CoV-2","Larger viral genome size facilitates emergence of zoonotic diseases","HTCC as a highly effective polymeric inhibitor of SARS-CoV-2 and MERS-CoV","The RNA pseudoknots in foot-and-mouth disease virus are dispensable for genome replication but essential for the production of infectious virus","Insights into The Codon Usage Bias of 13 Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) Isolates from Different Geo-locations","The first-in-class peptide binder to the SARS-CoV-2 spike protein","The Essential Facts of Wuhan Novel Coronavirus Outbreak in China and Epitope-based Vaccine Designing against COVID-19","Protein structure analysis of the interactions between SARS-CoV-2 spike protein and the human ACE2 receptor: from conformational changes to novel neutralizing antibodies","Nanotubes effectively deliver siRNA to intact plant cells and protect siRNA against nuclease degradation","Sickness behaviour reduces network centrality in wild vampire bats","Tilorone: A Broad-Spectrum Antiviral For Emerging Viruses","Structural and functional conservation of the programmed -1 ribosomal frameshift signal of SARS-CoV-2","First isolation and characterisation of Alongshan virus in Russia","CRISPR-based surveillance for COVID-19 using genomically-comprehensive machine learning design","Evaluating performance of metagenomic characterization algorithms using in silico datasets generated with FASTQSim","Beyond type 1 regulatory T cells: co-expression of LAG3 and CD49b in IL-10-producing T cell lineages","Decoding the lethal effect of SARS-CoV-2 (novel coronavirus) strains from global perspective: molecular pathogenesis and evolutionary divergence","Screening of FDA-approved drugs using a MERS-CoV clinical isolate from South Korea identifies potential therapeutic options for COVID-19","Comparison of SARS-CoV2 N gene real-time RT-PCR targets and commercially available mastermixes","Fitness barriers limit reversion of a proofreading-deficient coronavirus","Stochastic dynamics of an epidemics with recurrent spillovers from an endemic reservoir","Structural basis for the inhibition of COVID-19 virus main protease by carmofur, an antineoplastic drug","Potential host range of multiple SARS-like coronaviruses and an improved ACE2-Fc variant that is potent against both SARS-CoV-2 and SARS-CoV-1","Clinical benefit of remdesivir in rhesus macaques infected with SARS-CoV-2","Planning horizon affects prophylactic decision-making and epidemic dynamics","Topological Analysis of SARS CoV-2 Main Protease","Tuning antiviral CD8 T-cell response via proline-altered peptide ligand vaccination","Hotspots of aberrant enhancer activity in fibrolamellar carcinoma reveal molecular mechanisms of oncogenesis and intrinsic drug resistance","Glycosylation of Zika Virus Is Important in Host-Virus Interaction and Pathogenesis","The aging transcriptome and cellular landscape of the human lung in relation to SARS-CoV-2","The SARS-CoV-2 receptor-binding domain elicits a potent neutralizing response without antibody-dependent enhancement","BAR scaffolds drive membrane fission by crowding disordered domains","The impact of regular school closure on seasonal influenza epidemics: a data-driven spatial transmission model for Belgium","Scrutinizing the SARS-CoV-2 protein information for the designing an effective vaccine encompassing both the T-cell and B-cell epitopes","In-Silico evidence for two receptors based strategy of SARS-CoV-2","Recombination and lineage-specific mutations led to the emergence of SARS-CoV-2","Mating strategy is determinant of Adenovirus prevalence in European bats","Nuclease escape elements protect messenger RNA against cleavage by multiple viral endonucleases","Chronic infections can shape epidemic exposure: Pathogen co-occurrence networks in the Serengeti lions","A snapshot of SARS-CoV-2 genome availability up to 30th March, 2020 and its implications","Crystal structure of SARS-CoV-2 nucleocapsid protein RNA binding domain reveals potential unique drug targeting sites","Live-cell single RNA imaging reveals bursts of translational frameshifting","Multiple known and a novel parvovirus associated with an outbreak of feline diarrhea and vomiting","Translational profiling of macrophages infected with Leishmania donovani identifies mTOR- and eIF4A-sensitive immune-related transcripts","Assessing Topographic Structural Connectivity of the Human Basal Ganglia and Thalamus","BioLaboro: A bioinformatics system for detecting molecular assay signature erosion and designing new assays in response to emerging and reemerging pathogens","Mechanistic insights into Zika virus NS3 helicase inhibition by Epigallocatechin-3-gallate: Supplementary Files","Machine learning using intrinsic genomic signatures for rapid classification of novel pathogens: COVID-19 case study","Synthetic nanobodies targeting the SARS-CoV-2 receptor-binding domain","Accurate Identification of SARS-CoV-2 from Viral Genome Sequences using Deep Learning","Effects of a blend of essential oils in milk replacer on performance, rumen fermentation, blood parameters and health scores of dairy heifers","The Influence Of Social Behavior On Competition Between Virulent Pathogen Strains","A transmissible RNA pathway in honey bees","The biomechanical role of extra-axonemal structures in shaping the flagellar beat of Euglena","Novel ionophores active against La Crosse virus identified through rapid antiviral screening","Prospects for detecting early warning signals in discrete event sequence data: application to epidemiological incidence data","Individual and temporal variation in pathogen load predicts long-term impacts of an emerging infectious disease","Biological Evaluation of Molecules of the azaBINOL Class as Antiviral Agents: Specific Inhibition of HIV-1 RNase H Activity by 7-Isopropoxy-8-(naphth-1-yl)quinoline","Mono-ADP-ribosylation by ARTD10 restricts Chikungunya virus replication by interfering with the proteolytic activity of nsP2","G-quadruplex forming sequences in the genome of all known human viruses: a comprehensive guide","Epitope-based peptide vaccine design and target site characterization against novel coronavirus disease caused by SARS-CoV-2","Prediction of receptorome for human-infecting virome","A single dose of ChAdOx1 MERS provides broad protective immunity against a variety of MERS-CoV strains","Validation and automation of a high-throughput multi-targeted method for semi-quantification of endogenous metabolites from different biological matrices using tandem mass spectrometry","On the interactions of the receptor-binding domain of SARS-CoV-1 and SARS-CoV-2 spike proteins with monoclonal antibodies and the receptor ACE2","SARS-CoV-2 invades host cells via a novel route: CD147-spike protein","On the causes of evolutionary transition:transversion bias","Computational vaccinology approach: Designing an efficient multi-epitope peptide vaccine against Cryptococcus neoformans var. grubii\u2019s heat shock 70KDa protein","Anti-biofilm Activity of Graphene Quantum Dots via Self-Assembly with Bacterial Amyloid Proteins","Fast Virome Explorer: A Pipeline for Virus and Phage Identification and Abundance Profiling in Metagenomics Data","Comparative performance of four nucleic acid amplification tests for SARS-CoV-2 virus","Genome based Evolutionary study of SARS-CoV-2 towards the Prediction of Epitope Based Chimeric Vaccine","Structure-Guided Mutagenesis Alters Deubiquitinating Activity and Attenuates Pathogenesis of a Murine Coronavirus","A nematode-specific gene underlies bleomycin-response variation in Caenorhabditis elegans","Single-cell Analysis of ACE2 Expression in Human Kidneys and Bladders Reveals a Potential Route of 2019-nCoV Infection","Structure and immune recognition of the porcine epidemic diarrhea virus spike protein","Nanopore-based native RNA sequencing provides insights into prokaryotic transcription, operon structures, rRNA maturation and modifications","A Scalable, Easy-to-Deploy, Protocol for Cas13-Based Detection of SARS-CoV-2 Genetic Material","Novel splicing and open reading frames revealed by long-read direct RNA sequencing of adenovirus transcripts","Computational Prediction of the Comprehensive SARS-CoV-2 vs. Human Interactome to Guide the Design of Therapeutics","Intrinsic checkpoint deficiency during cell cycle re-entry from quiescence","A systematic review and evaluation of Zika virus forecasting and prediction research during a public health emergency of international concern","ACE2 fragment as a decoy for novel SARS-Cov-2 virus","Cryo-EM structure of coronavirus-HKU1 haemagglutinin esterase reveals architectural changes arising from prolonged circulation in humans","Partial RdRp sequences offer a robust method for Coronavirus subgenus classification","Choanoflagellate transfection illuminates their cell biology and the ancestry of animal septins","An amino acid motif in HLA-DR\u03b21 distinguishes patients with uveitis in juvenile idiopathic arthritis","Freshwater monitoring by nanopore sequencing","In-vivo targeted tagging of RNA isolates cell specific transcriptional responses to environmental stimuli and identifies liver-to-adipose RNA transfer","Pulmonary Metagenomic Sequencing Suggests Missed Infections in Immunocompromised Children","Remdesivir inhibits renal fibrosis in obstructed kidneys","High sensitivity detection of coronavirus SARS-CoV-2 using multiplex PCR and a multiplex-PCR-based metagenomic method","Broad Host Range of SARS-CoV-2 Predicted by Comparative and Structural Analysis of ACE2 in Vertebrates","Host, Viral, and Environmental Transcriptome Profiles of the Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2)","Structure of the SARS-CoV NSP12 polymerase bound to NSP7 and NSP8 co-factors","Scirpy: A Scanpy extension for analyzing single-cell T-cell receptor sequencing data","Vulnerabilities in coronavirus glycan shields despite extensive glycosylation","A scientometric overview of CORD-19","Comparative Genomic Analysis of Rapidly Evolving SARS CoV-2 Viruses Reveal Mosaic Pattern of Phylogeographical Distribution","Large-scale analysis of redox-sensitive conditionally disordered protein regions reveal their widespread nature and key roles in high-level eukaryotic processes","Modification of primary amines to higher order amines reduces in vivo hematological and immunotoxicity of cationic nanocarriers through TLR4 and complement pathways","A speed-fidelity trade-off determines the mutation rate and virulence of an RNA virus","Rigidity, normal modes and flexible motion of a SARS-CoV-2 (COVID-19) protease structure","Full-length genome sequence of segmented RNA virus from ticks was obtained using small RNA sequencing data","Deep learning model can predict water binding sites on the surface of proteins using limited-resolution data","Genome Detective Coronavirus Typing Tool for rapid identification and characterization of novel coronavirus genomes","Leveraging mRNAs sequences to express SARS-CoV-2 antigens in vivo","Glycosyltransferases promote development and prevent promiscuous cell aggregation in the choanoflagellate S. rosetta","RNA-binding activity of TRIM25 is mediated by its PRY/SPRY domain and is required for ubiquitination","Predicting commercially available antiviral drugs that may act on the novel coronavirus (2019-nCoV), Wuhan, China through a drug-target interaction deep learning model","Increasing testing throughput and case detection with a pooled-sample Bayesian approach in the context of COVID-19","Understanding the B and T cells epitopes of spike protein of severe respiratory syndrome coronavirus-2: A computational way to predict the immunogens","Self-assembly of the RZZ complex into filaments drives kinetochore expansion in the absence of microtubule attachment","Vaccination with single plasmid DNA encoding IL-12 and antigens of severe fever with thrombocytopenia syndrome virus elicits complete protection in IFNAR knockout mice","Identification of antiviral drug candidates against SARS-CoV-2 from FDA-approved drugs","The crystal structure of nsp10-nsp16 heterodimer from SARS CoV-2 in complex with S-adenosylmethionine","A one-enzyme RT-qPCR assay for SARS-CoV-2, and procedures for reagent production","Vorpal: A novel RNA virus feature-extraction algorithm demonstrated through interpretable genotype-to-phenotype linear models","Containing Emerging Epidemics: a Quantitative Comparison of Quarantine and Symptom Monitoring","Designing Effective small interfering RNA for Post-Transcriptional Silencing of Human GREM1: A Comprehensive Bioinformatics Approach","Vitrification after multiple rounds of sample application and blotting improves particle density on cryo-electron microscopy grids","Zika Virus Infection at Different Pregnancy Stages: Anatomopathological Findings, Target Cells and Viral Persistence in Placental Tissues","Real-time, MinION-based, amplicon sequencing for lineage typing of infectious bronchitis virus from upper respiratory samples","Salt Effects on the Thermodynamics of a Frameshifting RNA Pseudoknot under Tension","Potentially highly potent drugs for 2019-nCoV","Stress-induced phospho-ubiquitin formation causes parkin degradation","Cigarette smoke triggers the expansion of a subpopulation of respiratory epithelial cells that express the SARS-CoV-2 receptor ACE2","Geometric coupling of helicoidal ramps and curvature-inducing proteins in organelle membranes","Molecular basis for the evolution of species-specific hemoglobin capture by pathogenic Staphylococcus","Spatial epidemiology of networked metapopulation: An overview","Interleukin-33 promotes type 1 cytokine expression via p38 MAPK in human natural killer cells","Insufficiency in airway interferon activation defines clinical severity to infant RSV infection","Domestication reprogrammed the budding yeast life cycle","Spike-based phylogenetically defined clades within the Alphacoronavirus 1 species","Nowcasting by Bayesian Smoothing: A flexible, generalizable model for real-time epidemic tracking","3D Models of glycosylated SARS-CoV-2 spike protein suggest challenges and opportunities for vaccine development","Comparative analysis of primer-probe sets for the laboratory confirmation of SARS-CoV-2","The guanine nucleotide exchange factor GBF1 participates in rotavirus replication","Host and infectivity prediction of Wuhan 2019 novel coronavirus using deep learning algorithm","Probing the unfolded protein response to mouse hepatitis coronavirus infection through RNA sequencing and ribosome profiling","The potential SARS-CoV-2 entry inhibitor","The IFN response in bat cells consists of canonical and non-canonical ISGs with unique temporal expression kinetics","Congruence of location-specific transcriptional programs in intestinal organoids during long-term culture","Specific ACE2 Expression in Cholangiocytes May Cause Liver Damage After 2019-nCoV Infection","Rapid metagenomic characterization of a case of imported COVID-19 in Cambodia","Exploring the genomic and proteomic variations of SARS-CoV-2 spike glycoprotein: a computational biology approach","SARS-CoV-2 and SARS-CoV differ in their cell tropism and drug sensitivity profiles","Evolutionary origins of the SARS-CoV-2 sarbecovirus lineage responsible for the COVID-19 pandemic","A Combined Evidence Approach to Prioritize Nipah Virus Inhibitors","Establishment of primary transgenic human airway epithelial cell cultures to study respiratory virus \u2013 host interactions","GI-16 lineage (624/I or Q1), there and Back Again: the history of one of the major threat for poultry farming of our era","Potent Antiviral Activities of Type I Interferons to SARS-CoV-2 Infection","Predictions for the binding domain and potential new drug targets of 2019-nCoV","Isolated occurrences of membrane perturbation by mechanosensing from weakly aggregating silver nanoparticles","Development of Reverse Transcription Loop-mediated Isothermal Amplification (RT-LAMP) Assays Targeting SARS-CoV-2","PGC-1\u03b1 coordinates with Bcl-2 to control cell cycle in U251 cells through reducing ROS","Lipidome profiles of postnatal day 2 vaginal swabs reflect fat composition of gilt\u2019s postnatal diet","Homologous Recombination as an Evolutionary Force in African Swine Fever Viruses","Comparative genomic analysis revealed specific mutation pattern between human coronavirus SARS-CoV-2 and Bat-SARSr-CoV RaTG13","50-valent inactivated rhinovirus vaccine is broadly immunogenic in rhesus macaques","Large-scale Lassa fever outbreaks in Nigeria: quantifying the association between disease reproduction number and local rainfall","Dynamical asymmetry exposes 2019-nCoV prefusion spike","Identification of Human Immune Cell Subtypes Most Vulnerable to IL-1\u03b2-induced Inflammatory Signaling Using Mass Cytometry","Lung epithelial cells have virus-specific and shared gene expression responses to infection by diverse respiratory viruses","Genetic Variability of Human Angiotensin-Converting Enzyme 2 (hACE2) Among Various Ethnic Populations","Rapid, large-scale, and effective detection of COVID-19 via non-adaptive testing","A hidden gene in astroviruses encodes a cell-permeabilizing protein involved in virus release","Energetics based epitope screening in SARS CoV-2 (COVID 19) spike glycoprotein by Immuno-informatic analysis aiming to a suitable vaccine development","Regulation of angiotensin converting enzyme 2 (ACE2) in obesity: implications for COVID-19","Human ACE2 receptor polymorphisms predict SARS-CoV-2 susceptibility","High-surety isothermal amplification and detection of SARS-CoV-2, including with crude enzymes","Human iPSC-Derived Cardiomyocytes are Susceptible to SARS-CoV-2 Infection","Episodic diversifying selection and intragenomic interactions shape the evolution of DNA and RNA viruses","Capturing diverse microbial sequence with comprehensive and scalable probe design","Optimising Renewal Models for Real-Time Epidemic Prediction and Estimation","Scutellaria baicalensis extract and baicalein inhibit replication of SARS-CoV-2 and its 3C-like protease in vitro","Dynamics of viral index infections in novel hosts","DUSP1 regulates apoptosis and cell migration, but not the JIP1-protected cytokine response, during Respiratory Syncytial Virus and Sendai Virus infection","Human organs-on-chips as tools for repurposing approved drugs as potential influenza and COVID19 therapeutics in viral pandemics","Novel non-transposable-element regulation patterns of KZFP family reveal new drivers of its rapid evolution","DEN-IM: Dengue Virus identification from shotgun and targeted metagenomics","A novel defective recombinant porcine enterovirus G virus carrying a porcine torovirus papain-like cysteine protease gene and a putative anti-apoptosis gene in place of viral structural protein genes","Ensemble Feature Selection and Meta-Analysis of Cancer miRNA Biomarkers","Knowledge and attitudes of Ebola among the general public of Trinidad and Tobago during the 2014-15 West Africa outbreak","Human Bocavirus Prevalence In Children With Acute Gastroenteritis From Rural Communities In The Northen Region Of South Africa","Single Nucleus Multiomic Profiling Reveals Age-Dynamic Regulation of Host Genes Associated with SARS-CoV-2 Infection","Non-uniform refinement: Adaptive regularization improves single particle cryo-EM reconstruction","Direct nucleic acid analysis of mosquitoes for high fidelity species identification and detection of Wolbachia using a cellphone","Type 2 and interferon inflammation strongly regulate SARS-CoV-2 related gene expression in the airway epithelium","Under-the-radar dengue virus infections in natural populations of Aedes aegypti mosquitoes","Cross-reactive antibody response between SARS-CoV-2 and SARS-CoV infections","Integrated Micropillar Polydimethylsiloxane Accurate CRISPR Detection (IMPACT) System for Rapid Viral DNA Sensing","A planarian nidovirus expands the limits of RNA genome size","The infant airway microbiome in health and disease impacts later asthma development","Conflicts with diarrheal pathogens trigger rapid evolution of an intestinal signaling axis","Novel Immunoglobulin Domain Proteins Provide Insights into Evolution and Pathogenesis Mechanisms of SARS-Related Coronaviruses","Tissue-resident CD8+ T cells drive age-associated chronic lung sequelae following viral pneumonia","The geometry of dependence: solitary bee larvae prioritize carbohydrate over protein in parentally provided pollen","Processing of the SARS-CoV pp1a/ab nsp7-10 region","No evidence that androgen regulation of pulmonary TMPRSS2 explains sex-discordant COVID-19 outcomes","Precise triggering and chemical control of single-virus fusion within endosomes","Significant expression of FURIN and ACE2 on oral epithelial cells may facilitate the efficiency of 2019-nCov entry","Multiplex logic processing isothermal diagnostic assays for an evolving virus","Design of multi epitope-based peptide vaccine against E protein of human COVID-19: An immunoinformatics approach","Identification of 2019-nCoV related coronaviruses in Malayan pangolins in southern China","Unique properties of Zika NS2B-NS3pro complexes as decoded by experiments and MD simulations","Rapid in silico design of antibodies targeting SARS-CoV-2 using machine learning and supercomputing","Pattern of early human-to-human transmission of Wuhan 2019-nCoV","Comparative analysis reveals adaptive evolution of bat IFITMs and a novel antiviral determinant","Replication of MERS and SARS coronaviruses in bat cells offers insights to their ancestral origins","Keeping the kids at home can limit the persistence of contagious pathogens in social animals","Zika virus outbreak in the Americas: Is Aedes albopictus an overlooked culprit?","New Soluble Angiopoietin Analog of C4BP-ANG1 Prevents Pathological Vascular Leakage","Multimerization of HIV-1 integrase hinges on conserved SH3-docking platforms","Long non-coding RNA Gm15441 attenuates hepatic inflammasome activation in response to metabolic stress","Real-time projections of Ebola outbreak size and duration with and without vaccine use in \u00c9quateur, Democratic Republic of Congo, as of May 27, 2018","Mutations, Recombination and Insertion in the Evolution of 2019-nCoV","No more business as usual: agile and effective responses to emerging pathogen threats require open data and open analytics","Exploring the Binding Mechanism between Human Profilin (PFN1) and Polyproline-10 through Binding Mode Screening","Analytical Validation of a COVID-19 qRT-PCR Detection Assay Using a 384-well Format and Three Extraction Methods","The potential genetic network of human brain SARS-CoV-2 infection","An automated data processing and analysis pipeline for transmembrane proteins in detergent solutions","The role of post-Golgi transport pathways and sorting motifs in the plasmodesmal targeting of the movement protein (MP) of Ourmia melon virus (OuMV)","Structure of Mpro from COVID-19 virus and discovery of its inhibitors","An Augmented Aging Process in Brain White Matter in HIV","Potential T-cell and B-cell Epitopes of 2019-nCoV","Low basal expression and slow induction of IFITM3 puts immune cells at risk of influenza A infection","PhyloFold: Precise and Swift Prediction of RNA Secondary Structures to Incorporate Phylogeny among Homologs","BIP4COVID19: Releasing impact metrics data for articles relevant to COVID-19","C19ORF66 broadly escapes viral-induced endonuclease cleavage and restricts Kaposi\u2019s Sarcoma Associated Herpesvirus (KSHV)","Discovery of baicalin and baicalein as novel, natural product inhibitors of SARS-CoV-2 3CL protease in vitro","Tracking progress towards malaria elimination in China: estimates of reproduction numbers and their spatiotemporal variation","Inflammation induced by influenza virus impairs innate control of human pneumococcal carriage","Electroporated recombinant proteins as tools for in vivo functional complementation, imaging, and chemical biology","The molecular basis for Pompe disease revealed by the structure of human acid \u03b1-glucosidase","Nonmedical Masks in Public for Respiratory Pandemics: Droplet Retention by Two-Layer Textile Barrier Fully Protects Germ-free Mice from Bacteria in Droplets","HCAtk and pyHCA: A Toolkit and Python API for the Hydrophobic Cluster Analysis of Protein Sequences","Deoxynivalenol Promotes Porcine Epidemic Diarrhea Virus Infection and Aggravates Gut Barrier Injury","Dark Proteome of Newly Emerged SARS-CoV-2 in Comparison with Human and Bat Coronaviruses","Robust neutralization assay based on SARS-CoV-2 S-bearing vesicular stomatitis virus (VSV) pseudovirus and ACE2-overexpressed BHK21 cells","Ecological and evolutionary drivers of hemoplasma infection and bacterial genotype sharing in a Neotropical bat community","Variable Macro X Domain of SARS-CoV-2 Retains the Ability to Bind ADP-ribose","Single-cell atlas of a non-human primate reveals new pathogenic mechanisms of COVID-19","Pseudomonas aeruginosa lectin LecB impairs keratinocyte fitness by abrogating growth factor signalling","Rapid direct nucleic acid amplification test without RNA extraction for SARS-CoV-2 using a portable PCR thermocycler","Aberrant pathogenic GM-CSF+ T cells and inflammatory CD14+CD16+ monocytes in severe pulmonary syndrome patients of a new coronavirus","Direct RNA sequencing and early evolution of SARS-CoV-2","Viral Architecture of SARS-CoV-2 with Post-Fusion Spike Revealed by Cryo-EM","A practical generation interval-based approach to inferring the strength of epidemics from their speed","The within-host viral kinetics of SARS-CoV-2","Genome-wide data inferring the evolution and population demography of the novel pneumonia coronavirus (SARS-CoV-2)","In trans variant calling reveals enrichment for compound heterozygous variants in genes involved in neuronal development and growth.","The Spike Protein S1 Subunit of SARS-CoV-2 Contains an LxxIxE-like Motif that is Known to Recruit the Host PP2A-B56 Phosphatase","Strategies for vaccine design for corona virus using Immunoinformatics techniques","Characterisation of the transcriptome and proteome of SARS-CoV-2 using direct RNA sequencing and tandem mass spectrometry reveals evidence for a cell passage induced in-frame deletion in the spike glycoprotein that removes the furin-like cleavage site.","LY6E Restricts the Entry of Human Coronaviruses, including the currently pandemic SARS-CoV-2","Identification of the relative timing of infectiousness and symptom onset for outbreak control","Prediction and Evolution of B Cell Epitopes of Surface Protein in SARS-CoV-2","Rapid, field-deployable nucleobase detection and identification using FnCas9","Aerodynamic Characteristics and RNA Concentration of SARS-CoV-2 Aerosol in Wuhan Hospitals during COVID-19 Outbreak","Cells and gene expression programs in the adult human heart","An anti-Gn glycoprotein antibody from a convalescent patient potently inhibits the infection of severe fever with thrombocytopenia syndrome virus","Cross-scale dynamics and the evolutionary emergence of infectious diseases","Extraction-free COVID-19 (SARS-CoV-2) diagnosis by RT-PCR to increase capacity for national testing programmes during a pandemic","Replication of Equine arteritis virus is efficiently suppressed by purine and pyrimidine biosynthesis inhibitors","HCMV glycoprotein B subunit vaccine efficacy was mediated by non-neutralizing antibody effector functions","IFIT3 and IFIT2/3 promote IFIT1-mediated translation inhibition by enhancing binding to non-self RNA","tailfindr: Alignment-free poly(A) length measurement for Oxford Nanopore RNA and DNA sequencing","Microbial and autoantibody immunogenic repertoires in TIF1\u03b3 autoantibody positive dermatomyositis","Evaluation of the cytotoxic potential of extracts from the genus Passiflora cultived in Brazil against cancer cells","Blocking antibodies against SARS-CoV-2 RBD isolated from a phage display antibody library using a competitive biopanning strategy","Alterations in sialic-acid O-acetylation glycoforms during murine erythrocyte development","Cryo-EM Structure of the 2019-nCoV Spike in the Prefusion Conformation","A combined RNA-seq and whole genome sequencing approach for identification of non-coding pathogenic variants in single families","Triphosphates of the Two Components in DESCOVY and TRUVADA are Inhibitors of the SARS-CoV-2 Polymerase","Non-invasive surveys of mammalian viruses using environmental DNA","The Israeli Acute Paralysis Virus IRES captures host ribosomes by mimicking a ribosomal state with hybrid tRNAs","Ca2+ ions promote fusion of Middle East Respiratory Syndrome coronavirus with host cells and increase infectivity","Structural Basis of SARS-CoV-2 Spike Protein Priming by TMPRSS2","Identification and enrichment of SECReTE cis-acting RNA elements in the Coronaviridae and other (+) single-strand RNA viruses","SARS-CoV-2 receptor and entry genes are expressed by sustentacular cells in the human olfactory neuroepithelium","REVEALING COVID-19 TRANSMISSION BY SARS-CoV-2 GENOME SEQUENCING AND AGENT BASED MODELLING","The dengue virus non-structural protein 1 (NS1) is secreted from mosquito cells in association with the intracellular cholesterol transporter chaperone caveolin complex","In-silico immune cell deconvolution of the airway proteomes of infants with pneumonia reveals a link between reduced airway eosinophils and an increased risk of mortality","Herpesvirus infection reduces Pol II occupancy of host promoters but spares viral promoters","Amantadine disrupts lysosomal gene expression; potential therapy for COVID19","Codon usage and evolutionary rates of the 2019-nCoV genes","Artificial intelligence predicts the immunogenic landscape of SARS-CoV-2: toward universal blueprints for vaccine designs","Insights into Cross-species Evolution of Novel Human Coronavirus 2019-nCoV and Defining Immune Determinants for Vaccine Development","Comparison of real-time PCR and droplet digital PCR for the detection of Xylella fastidiosa in plants","Informing epidemic (research) responses in a timely fashion by knowledge management - a Zika virus use case","Sequence variation among SARS-CoV-2 isolates in Taiwan","Comparative genomics reveals signature regions used to develop a robust and sensitive multiplex TaqMan real-time qPCR assay to detect the genus Dickeya and Dickeya dianthicola","Insights on early mutational events in SARS-CoV-2 virus reveal founder effects across geographical regions","Saturation mutagenesis genome engineering of infective \u03a6X174 bacteriophage via unamplified oligo pools and golden gate assembly","The Role of Proton Transport in Gating Current in a Voltage Gated Ion Channel, as Shown by Quantum Calculations","Structure of RNA-dependent RNA polymerase from 2019-nCoV, a major antiviral drug target","One-pot Detection of COVID-19 with Real-time Reverse-transcription Loop-mediated Isothermal Amplification (RT-LAMP) Assay and Visual RT-LAMP Assay","Cryo-electron microscopy structure of the SADS-CoV spike glycoprotein provides insights into an evolution of unique coronavirus spike proteins","Computational Design of Peptides to Block Binding of the SARS-CoV-2 Spike Protein to Human ACE2","The architecture of SARS-CoV-2 transcriptome","The 2019 coronavirus (SARS-CoV-2) surface protein (Spike) S1 Receptor Binding Domain undergoes conformational change upon heparin binding","Evidence for influenza B virus hemagglutinin adaptation to the human host: high cleavability, acid-stability and preference for cool temperature","Identification of a common deletion in the spike protein of SARS-CoV-2","Structure-based design of hepatitis C virus E2 glycoprotein improves serum binding and cross-neutralization","Rapid and quantitative detection of COVID-19 markers in micro-liter sized samples","Gut microbiota structure differs between honey bees in winter and summer","Similar ratios of introns to intergenic sequence across animal genomes","Transgenic mice expressing tunable levels of DUX4 develop characteristic facioscapulohumeral muscular dystrophy-like pathophysiology ranging in severity","2019-20 Wuhan coronavirus outbreak: Intense surveillance is vital for preventing sustained transmission in new locations","Sphingolipid Biosynthesis Inhibition As A Host Strategy Against Diverse Pathogens","Chronic hM3Dq signaling in microglia ameliorates neuroinflammation in male mice","Site specific target binding controls RNA cleavage efficiency by the Kaposi\u2019s sarcoma-associated herpesvirus endonuclease SOX","Shotgun proteomics of SARS-CoV-2 infected cells and its application to the optimisation of whole viral particle antigen production for vaccines","Limited SARS-CoV-2 diversity within hosts and following passage in cell culture","Early in planta detection of Xanthomonas axonopodis pv. punicae in pomegranate using enhanced loop-mediated isothermal amplification assay","Spike protein binding prediction with neutralizing antibodies of SARS-CoV-2","The genome of the crustacean Parhyale hawaiensis: a model for animal development, regeneration, immunity and lignocellulose digestion","Exceptional diversity and selection pressure on SARS-CoV and SARS-CoV-2 host receptor in bats compared to other mammals","DNA breaks-mediated cost reveals RNase HI as a new target for selectively eliminating antibiotic resistance","Coordination between discrete Mitotic Arrest Deficient 1 (MAD1) domains is required for efficient mitotic checkpoint signaling","A data-driven drug repositioning framework discovered a potential therapeutic agent targeting COVID-19","Coronavirus Infection and PARP Expression Dysregulate the NAD Metabolome: A Potentially Actionable Component of Innate Immunity","Novel and potent inhibitors targeting DHODH, a rate-limiting enzyme in de novo pyrimidine biosynthesis, are broad-spectrum antiviral against RNA viruses including newly emerged coronavirus SARS-CoV-2","Comparative in vitro transcriptomic analyses of COVID-19 candidate therapy hydroxychloroquine suggest limited immunomodulatory evidence of SARS-CoV-2 host response genes.","Mass Spectrometric Identification of SARS-CoV-2 Proteins from Gargle Solution Samples of COVID-19 Patients","Increased Neurite Orientation-Dispersion and Density in the TgCRND8 Mouse Model of Amyloidosis: Inverse Relation with Functional Connectome Clustering and Modulation by Interleukin-6","Etiology of fever in Ugandan children: identification of microbial pathogens using metagenomic next-generation sequencing and IDseq, a platform for unbiased metagenomic analysis","Estimating the Relative Probability of Direct Transmission between Infectious Disease Patients","Microbiota-independent antiviral protection conferred by aminoglycoside antibiotics","Deducing the N- and O-glycosylation profile of the spike protein of novel coronavirus SARS-CoV-2","Prediction of antiviral drugs against African Swine Fever Viruses based on protein-protein interaction analysis","Viral fitness predicts the magnitude and direction of perturbations in the infected host transcriptome","Noisy Pooled PCR for Virus Testing","Using pan RNA-seq analysis to reveal the ubiquitous existence of 5\u2019 end and 3\u2019 end small RNAs","Critical Nodes of Virus\u2013Host Interaction Revealed Through an Integrated Network Analysis","Alphavirus nsP3 ADP-ribosylhydrolase Activity Disrupts Stress Granule Formation","Bats Possess Unique Variants of the Antiviral Restriction Factor Tetherin","Are pangolins the intermediate host of the 2019 novel coronavirus (2019-nCoV) ?","FilmArray Respiratory Panel Integrated in a field Point Of Care dispositive for the diagnosis of respiratory tract infections in rural areas in Senegal","Mammalian orthoreovirus infection is enhanced in cells pre-treated with sodium arsenite","Discovery of pharmaceutically-targetable pathways and prediction of survivorship for pneumonia and sepsis patients from the view point of ensemble gene noise","CryoDRGN: Reconstruction of heterogeneous structures from cryo-electron micrographs using neural networks","Revealing variants in SARS-CoV-2 interaction domain of ACE2 and loss of function intolerance through analysis of &gt;200,000 exomes","Transcriptional Difference between SARS-COV-2 and other Human Coronaviruses Revealed by Sub-genomic RNA Profiling","The immune vulnerability landscape of the 2019 Novel Coronavirus, SARS-CoV-2","A SARS-CoV-2-Human Protein-Protein Interaction Map Reveals Drug Targets and Potential Drug-Repurposing","Evaluating features of scientific conferences: A call for improvements","A virus that has gone viral: Amino acid mutation in S protein of Indian isolate of Coronavirus COVID-19 might impact receptor binding and thus infectivity","Multiscale 3-dimensional pathology findings of COVID-19 diseased lung using high-resolution cleared tissue microscopy","Complemented palindrome small RNAs first discovered from SARS coronavirus","Girls gone wild: Social isolation induces hyperactivity and exploration in aged female mice","A Multiscale and Comparative Model for Receptor Binding of 2019 Novel Coronavirus and the Implication of its Life Cycle in Host Cells","High-resolution influenza mapping of a city reveals socioeconomic determinants of transmission within and between urban quarters","Structural modeling of 2019-novel coronavirus (nCoV) spike protein reveals a proteolytically-sensitive activation loop as a distinguishing feature compared to SARS-CoV and related SARS-like coronaviruses","Identification of a Nidovirales Orf1a N7-guanine cap Methyltransferase signature-sequence as a genetic marker of large genome Tobaniviridae","A Generalized Discrete Dynamic Model for Human Epidemics","Re-analysis of SARS-CoV-2 infected host cell proteomics time-course data by impact pathway analysis and network analysis. A potential link with inflammatory response.","The sequence of human ACE2 is suboptimal for binding the S spike protein of SARS coronavirus 2","One-step RNA extraction for RT-qPCR detection of 2019-nCoV","Mutation of Ebola virus VP35 Ser129 uncouples interferon antagonist and replication functions","Structural genomics and interactomics of 2019 Wuhan novel coronavirus, 2019-nCoV, indicate evolutionary conserved functional regions of viral proteins","Severe acute respiratory syndrome-related coronavirus: The species and its viruses \u2013 a statement of the Coronavirus Study Group","MYCOPLASMA FAUCIUM AND BREAST CANCER","Tocilizumab treatment in severe COVID-19 patients attenuates the inflammatory storm incited by monocyte centric immune interactions revealed by single-cell analysis","A binning tool to reconstruct viral haplotypes from assembled contigs","Identification of quercetin from fruits to immediately fight Zika","A mathematical model for simulating the transmission of Wuhan novel Coronavirus","Accurate prediction of human essential genes using only nucleotide composition and association information","Quantitative Delineation of Herpesviruses in Bats for use in Ecological Studies","Bayesian mixture analysis for metagenomic community profiling.","Hand-hygiene mitigation strategies against global disease spreading through the air transportation network","Currently available intravenous immunoglobulin (Gamunex\u00ae-C and Flebogamma\u00ae DIF) contains antibodies reacting against SARS-CoV-2 antigens","The effects of modified sialic acids on mucus and erythrocytes on influenza A virus HA and NA functions","An emergent clade of SARS-CoV-2 linked to returned travellers from Iran","An orally bioavailable broad-spectrum antiviral inhibits SARS-CoV-2 and multiple endemic, epidemic and bat coronavirus","Impacts of genomic networks governed by human-specific regulatory sequences and genetic loci harboring fixed human-specific neuro-regulatory single nucleotide mutations on phenotypic traits of Modern Humans","Altered polarization of PAR-2 signaling during airway epithelial remodeling","Direct RNA nanopore sequencing of full-length coronavirus genomes provides novel insights into structural variants and enables modification analysis","Comparative ACE2 variation and primate COVID-19 risk","The demyelinating agent cuprizone induces a male-specific reduction in binge eating in the binge-prone C57BL/6NJ strain","Beware of asymptomatic transmission: Study on 2019-nCoV prevention and control measures based on extended SEIR model","Long noncoding RNA AVAN promotes antiviral innate immunity by interacting with TRIM25 and enhancing the transcription of FOXO3a","AI-aided design of novel targeted covalent inhibitors against SARS-CoV-2","Doubling healthy lifespan using drug synergy","Complex dynamics in an SIS epidemic model induced by nonlinear incidence","Orthogonal genome-wide screenings in bat cells identify MTHFD1 as a target of broad antiviral therapy","DNA segment of African Swine Fever Virus first detected in hard ticks from sheep and bovine","Alpha-ketoamides as broad-spectrum inhibitors of coronavirus and enterovirus replication","A novel antiviral formulation inhibits a range of enveloped viruses.","IDseq \u2013 An Open Source Cloud-based Pipeline and Analysis Service for Metagenomic Pathogen Detection and Monitoring","Identification of peptide coatings that enhance diffusive transport of nanoparticles through the tumor microenvironment","Replication of SARS-CoV-2 in human respiratory epithelium","Contesting the evidence for -1 frameshifting in immune-functioning C-C chemokine receptor 5 (CCR5) - the HIV-1 co-receptor","A Toolkit for Rapid Modular Construction of Biological Circuits in Mammalian Cells","Epitope-based chimeric peptide vaccine design against S, M and E proteins of SARS-CoV-2 etiologic agent of global pandemic COVID-19: an in silico approach","Mapping a viral phylogeny onto outbreak trees to improve host transmission inference","Characterisation of the faecal virome of captive and wild Tasmanian devils using virus-like particles metagenomics and meta-transcriptomics","Detection of spreader nodes and ranking of interacting edges in Human-SARS-CoV protein interaction network","Zika virus infection in Collaborative Cross mice","Collaborative Cross Mouse Populations as a Resource for the Study of Epilepsy","Efficient inactivation of SARS-CoV-2 by WHO-recommended hand rub formulations and alcohols","Pangolin homology associated with 2019-nCoV","Ultra-Low-Cost Integrated Silicon-based Transducer for On-Site, Genetic Detection of Pathogens","Structural Basis for Potent Neutralization of Betacoronaviruses by Single-domain Camelid Antibodies","RT-qPCR DETECTION OF SARS-CoV-2 RNA FROM PATIENT NASOPHARYNGEAL SWAB USING QIAGEN RNEASY KITS OR DIRECTLY VIA OMISSION OF AN RNA EXTRACTION STEP","Potential Neutralizing Antibodies Discovered for Novel Corona Virus Using Machine Learning","ViralMSA: Massively scalable reference-guided multiple sequence alignment of viral genomes","Structural interactions between pandemic SARS-CoV-2 spike glycoprotein and human Furin protease","Bioinformatics Study on Structural Protein of Severe Acute Respiratory Syndrome Coronavirus 2 (Sars-Cov-2) For Better Understanding the Vaccine Development","A Computational Approach to Design Potential siRNA Molecules as a Prospective Tool for Silencing Nucleocapsid Phosphoprotein and Surface Glycoprotein Gene of SARS-CoV-2","Projecting the end of the Zika virus epidemic in Latin America: a modelling analysis","Teicoplanin potently blocks the cell entry of 2019-nCoV","Enabling large-scale genome editing by reducing DNA nicking","Enhancement of trans-cleavage activity of Cas12a with engineered crRNA enables amplified nucleic acid detection","Identification of a Novel Base J Binding Protein Complex Involved in RNA Polymerase II Transcription Termination in Trypanosomes","A quantitative narrative on movement, disease and patch exploitation in nesting agent groups","National and Regional Influenza-Like-Illness Forecasts for the USA","The influenza A virus endoribonuclease PA-X usurps host mRNA processing machinery to limit host gene expression","Real-time biomedical knowledge synthesis of the exponentially growing world wide web using unsupervised neural networks","FDA approved drugs with broad anti-coronaviral activity inhibit SARS-CoV-2 in vitro","Machine intelligence design of 2019-nCoV drugs","Complex dynamics induced by delayed adaptive behavior during epidemics","Comparisons of a Novel Air Sampling Filter Material, Wash Buffers and Extraction Methods in the Detection and Quantification of Influenza Virus","Individualized System for Augmenting Ventilator Efficacy (iSAVE): A Rapidly deployable system to expand ventilator capacity","Crystal structure of the SARS-CoV-2 non-structural protein 9, Nsp9","Using heterogeneity in the population structure of U.S. swine farms to compare transmission models for porcine epidemic diarrhoea","Substrate specificity profiling of SARS-CoV-2 Mpro protease provides basis for anti-COVID-19 drug design","Recombinant vector vaccines and within-host evolution","Single-cell analysis of human lung epithelia reveals concomitant expression of the SARS-CoV-2 receptor ACE2 with multiple virus receptors and scavengers in alveolar type II cells","Multiple approaches for massively parallel sequencing of HCoV-19 (SARS-CoV-2) genomes directly from clinical samples","The network structure and eco-evolutionary dynamics of CRISPR-induced immune diversification","Structural basis to design multi-epitope vaccines against Novel Coronavirus 19 (COVID19) infection, the ongoing pandemic emergency: an in silico approach","The Evolutionary History and Impact of Bacterial tRNA Modifications","Susceptibility of ferrets, cats, dogs, and different domestic animals to SARS-coronavirus-2","Magnetic Bead-Quantum Dot (MB-Qdot) CRISPR Assay for Instrument-Free Viral DNA Detection","Genetic control of the HDL proteome","Role of RNA Guanine Quadruplexes in Favoring the Dimerization of SARS Unique Domain in Coronaviruses","Cotranslational Folding Stimulates Programmed Ribosomal Frameshifting in the Alphavirus Structural Polyprotein","In-silico analysis of SARS-CoV-2 genomes: Insights from SARS encoded non-coding RNAs","CHOP and IRE1\u03b1-XBP1/JNK signaling promote Newcastle Disease Virus induced apoptosis and benefit virus proliferation","Viral gain-of-function experiments uncover residues under diversifying selection in nature","Productive infection of field strains of avian coronavirus infectious bronchitis virus in chicken peripheral blood-derived monocyte","Molecular Dynamics Simulations Indicate the SARS-CoV-2 Mpro Is Not a Viable Target for Small-Molecule Inhibitors Design","Functional assessment of cell entry and receptor usage for lineage B \u03b2-coronaviruses, including 2019-nCoV","Host shifts result in parallel genetic changes when viruses evolve in closely related species","Potential inhibitors for 2019-nCoV coronavirus M protease from clinically approved medicines","Rhesus macaques can be effectively infected with SARS-CoV-2 via ocular conjunctival route","Host protein CD63 enhances viral RNA replication by interacting with human astrovirus nonstructural protein nsP1a/4","Far-UVC light: A new tool to control the spread of airborne-mediated microbial diseases","The IMPDH inhibitor merimepodib suppresses SARS-CoV-2 replication in vitro","Cellular senescence limits acute lung injury induced by mechanical ventilation","Gene expression is stronger associated with behaviour than with age and fertility in ant workers","The deubiquitinating activity of Middle East respiratory syndrome coronavirus papain-like protease delays the innate immune response and enhances virulence in a mouse model","Candidate targets for immune responses to 2019-Novel Coronavirus (nCoV): sequence homology- and bioinformatic-based predictions","DeepTracer: Predicting Backbone Atomic Structure from High Resolution Cryo-EM Density Maps of Protein Complexes","Rapid community-driven development of a SARS-CoV-2 tissue simulator","TIM-1 Serves as a Nonredundant Receptor for Ebola Virus, Enhancing Viremia and Pathogenesis","Evidence for reassortment of highly divergent novel rotaviruses from bats in Cameroon, without evidence for human interspecies transmissions","A Proof Of Concept For A Syndromic Surveillance System Based On Routine Ambulance Records In The South-west Of England, For The Influenza Season 2016/17","Remote control of neural function by X-ray-induced scintillation","Artesunate interacts with Vitamin D receptor to reverse mouse model of sepsis-induced immunosuppression via enhancing autophagy","Integrated Disease Surveillance and Response (IDSR) in Malawi: Implementation Gaps and Challenges for Timely Alert","Comparative Pathogenesis Of COVID-19, MERS And SARS In A Non-Human Primate Model","Social history and exposure to pathogen signals modulate social status effects on gene regulation in rhesus macaques","The novel coronavirus 2019 (2019-nCoV) uses the SARS-coronavirus receptor ACE2 and the cellular protease TMPRSS2 for entry into target cells","PriSeT: Efficient De Novo Primer Discovery","The FDA-approved gold drug Auranofin inhibits novel coronavirus (SARS-COV-2) replication and attenuates inflammation in human cells","Fractional Dosing of Yellow Fever Vaccine to Extend Supply: A Modeling Study","3D Variability Analysis: Directly resolving continuous flexibility and discrete heterogeneity from single particle cryo-EM images","Glycosaminoglycan binding motif at S1/S2 proteolytic cleavage site on spike glycoprotein may facilitate novel coronavirus (SARS-CoV-2) host cell entry","On reproductive and respiratory syndrome virus dissemination across pig production systems in the United States","Elucidating the differences in the molecular mechanism of receptor binding between 2019-nCoV and the SARS-CoV viruses using computational tools","VADR: validation and annotation of virus sequence submissions to GenBank","A SARS-CoV-2 Vaccination Strategy Focused on Population-Scale Immunity","The alkyl side chain of PACA nanoparticles dictates the impact on cellular stress responses and the mode of particle-induced cell death","Protective Population Behavior Change in Outbreaks of Emerging Infectious Disease","Bacterial defiance as a form of prodrug failure","Arenaviridae exoribonuclease presents genomic RNA edition capacity","Cell-Type-Specific Expression of Renin-Angiotensin-System Components in the Human Body and Its Relevance to SARS-CoV-2 Infection","Innate Immune Priming by cGAS as a Preparatory Countermeasure Against RNA Virus Infection","Coronavirus hemagglutinin-esterase and spike proteins co-evolve for functional balance and optimal virion avidity","Kin and group selection are both flawed but useful data analysis tools"],"x":{"__ndarray__":"HF8YwUYd10GM+MTBhGtLwQdbwUF8lGfAZZTjQBqphUHrI3XALah7wTAMwsE6ZrBBTYwIQLqFxUDjFH1ApBylQc3oMz3c4p5AdPXPwdZ/T8HtDRdBXeDhwGvGvcEEtAfBDyeJwY8BkMEN/mtBMiUKQTtblb7gTw5B/1+pPhCFl8GXmfq/EKjNwH6Pk0ElM3xBuJRjQRAdyD/1UkzAtBWdwby6W0GQcqo/lNhDwJ0kt78XMJDBUE5NwV3loMHotFdB87yPweozTsHLYiRBfiqBwUcBAsAIVnrBhTdxwbhsPb8fr/BBd+UGQXWamUHXSJXA8/3UPxBjmcBeLFvBRnwWwDQu20B6WqTBlV0RQFf4SEH+M3vBli9kQQTvv0FwJ+tBfJ7CQILRZUERVPJA+6oUQV/1kUATX9w/ZLfxwDezLMGNPMBALfkqQTM3AUIGsMRBvNpQwYEiD0FNSaPBnj2mwBEVsEHPzQVBTXeAwQsGBcJVB4XBZ9pxwVij4UHtaIE/ggpLQZ6xQ8Gn98rBJde+QeEnlMHxWg7BdN68wbLElMEbAjjB83Ndwf0ym0HSt+VAq7MJwPrJD8G/Kj1BN+QqwOsaocGMO+o/OsQlQZlAksHptm7ALWGKQdCPFMGMVm9BtB/HQDQvjsFZ1gXCoVVGQeZvyEFUm7XBZs7YQClqlMG7RLxB6zuLwUChz8EqPLDAdKEQwWKioMHJhYu/UOfQwHNiAUIw8IrBghT5QToAbkH6IGU/ESvoQIFb30F0psFABzWawCDKCkKCPa7BMFPpwULArkEk+ZZBI7IjQYQBc0G7FnLBc3otQcRxN8H8fC1Bzfa0QVIiCUEQLAJC6UuTwQ1JA0Fe/FfBEJ6zwXTPpECSBAXCqEQFwht8y8GQgJPBx++xQNZeFkG3GPQ/r3dNQb02/sAYxJVA8IKDQbh0YEGSLjLBQxzDQPQoBcCL5GlAhIekwXLuicFUxxpBRaTIQRs8C0DoSqDBiPKLwQyNh8EvqZBAVIy0QV4/DUDrAHtA5GAmQdjx30Eii5bBZCsQQXjTfMFFG8XBKlnaQYT/osGOoe1AuX4tv7l4QEEbGNPATCaNwV0aA0I9r1HBjTqvwYcBO0BfCTTA90bywXVuNcFYwyrAep6rQZg2mMGmwW7B966aQY+uZkDIiNBBWR7iwKn+nUBXBOpBh3maQQCKvUAxoZpBr2M3QXePLUA5W1FAo8lHwWE3rsH4WJxAVseKwACYDEHW+tTA5RqNwZiBlsGJB4TB5fSxwQ/eKME9Vs/BjeuuQddFl8EbkS7BPctEQRgAD8Afp48/ZxBVvYKTar/TfXHBppwFwv+aLsGHh9xAPQKVwdBEsMHx0aLADDbKQR2sT79xAfE/oGk6QQxJZsE+ja7B+fZhwcGOAEEjyk3AkiKAv0DhFD4Zfe1B5XO4wA3Qy0FG549A94UYQdS+lUGVastBQxknwSEH/8EWdzzB/cWVPw2SRsFZoT1BV1KZwW/1s0HtepzAIbB6QS2TEcDEqrLAL2ecwFEDQj6u3e5B/KCIwMDGEkDuhYNAJ3WrwVuR9MCaR0dBix8aQQzqNkHOdI7BNN4twZn4uEGhR7FAHXWSwZjDgUH7mkXB7mntwNbr6D/81Ii+tQgAwCBSwEBoX01BK6VSwYfenkDvLfZA67oBQbqCgUAegy4+zYXiwYCNWsAb33VBUKtcwbXtr8BQl75AbowUQQxnI8AbHBjA1NUBQeSAGb/WzQNBqCeOQYMSlcFPVdJAlhytQFDqpsGaiM7BOwSSv7FrRsBWOoVBpRI1QHqLicF+XjpAPrU6wH2+0kH5+1nB0HUPP6cWokCpBjvBB96dwBJ8tMA04jZBioeQwRvQZsHUOSLA+nh0wcG4OUHJ37FBVbm0QQjTRsHi/EVAhArSQEV0J8HVVRtBjPVqQfkQeEHzmNhBs7RPQdDpm8EYqHfA9slkQFDGeMEEl7TBYRMYwNb7sUFK2IbBy9y5QXufTsF7BJC+tFRhwdIm8sH5BxRAkl95QVucBkGkr8bBCm8jQcmujcEV5ZBB4EcmwTarkUA9/+9BmzmKQV197kEUBVZAIseHQTIQ6EHzp53BmkX+QUXhDsAIth/BMaoywZLIwEDq0orB4TyrQcWjQ0HD1SVAWAq+wVXzekCnuhZBZI/2QZms9cHqpWZAkWiMwfIkksEeGwNBO+XhQU0QLD+ewiHBLOLxPrLUrMG0EXhAsNX4wQeoBz2H82bBVwuAQCpgyUA8JkTB/yyiwTZDRkDStYnBkKmMwI4g6EGdIRTB9kG4QSjXRcGq00w/89mRwfm5IcGmjhhB3vWJQd92hME7MKfB883iwVray79o6dVBtCYEQXVJg0FIywlBXWPbP3RZhsHz839B763BQRZ15UEnK7tA3dBGwZ/S28GFGBRB/v2pwDxKyECR/OrAtIpqQTQ0oEGbPy1BBZ0JQqkCvsHQKu1BQ9O6wITwE0FoOYzAyi6Dwcj+s8GvNdZBZj0LwQ3KUcAiITFBg7aswYcUycF/0a7Bjc3AQRfsfEGdto3B2Vb9wRDggcGdd4/Bq1r/Qaf3kUEgeUc+Fu4KQQqVX0AWDOvAP60wwYRJMkGRHphB4FfEQdf3icGYWhXAt09uvhkfSMFce6hAt36QwbozEEA8ZcfB8dF0QY/6BEFvHu1BRXiOQQiWcMFX2GlB5OGNwd7irkEcsnfB8qyUwNTqdUG1FkFBpW1oQfz8y8A4OjlBGA6Hv68IEMGqgARB5B5pwZZ1kUG87rHBtd18QYAI5MAYCbLBvphOP7SrOkFCbHnBXa/yQJAgxEFB3svBao5ewdPOyEEK4oO+/TICwmGPhcH0lFzAuFadQe/DdkFXeHjBqGFPQFvlKkGnUCFAo637QTE6ID8Hy3vBWumAwU1gVsCqQBpBH5UzQK1GMsCSSQ9BhifeQBJwor+JcMPA3Lh7QSjcZ78ft9lAYrWyQTBfl0Ezk6lAZrWWQBBLQ8E7QeZA7TvJwRXH+cENdAJChMY8wE7XN8EDy2bA9dl2Qb8NjsEsR05BNdyVQT1CvkGneT5BMcjowIrkgUH2yMRAkm1/wCpMhsEPe/RBEUaewVbNPUEQIWBBJ9oDQdl31UE2fqZBOySjwJHE48B4VyDB/dTsQW8mmcHUppHBnAXFwc6Dpz8zgsXAD42+wdYX90FKxxVBezpMP4kE/cHI0vFBtT9VwUyEYsAWaODA65rkQJZLGcFFkOI/wJ/+QU5z2r/aB1zBcPOaQd2q0MHtBu9AzVNWwRL6FEH8HNdAI8AGQfbr38EPEYBBm3+UwVqidUBpCfvAlr82QEkZhcFLmYxB9go+QfXvSj+qJcHAaxplQPJ2okF3cIjBLqVtwe+OrMES/P1AWGSfQCr1t8AxZTjBYcUawSmPykHvYBTA/VKQwbs9PkFURBfAt+cXwXq+kEGvgJhAW74Wv+FRI0HqLI/AjS7EP2+RycHWllpBUAoawMJsnMEdoonBi4FUQRD2BcBcIEFBnP1HQcJZWUE2RsjBHySbQUs0C0KBunlBK9nQQcxfuUFjNtZAQWspwWWFD8GvKhxBBIdTwf0jP8EX0vzBS7mOPzeqmcHLsSpBGwaYwTShksGrfrnB+gxnQNB2YkEFHl1BsfutQcWjXj+yyptBUvK8waCp+sGRxojBQo9lwasCdsHTSAPAEu1pQKY9k8EPYJVAs9gOQT0nUsEEwPVBpSQBv1jYJsBbvI2+CELdwUM39L7rYIXBSAw3wSx6gEG349FAo5N/weagjsDSS7XAhWHowdpxxkAna5fBQQCJP5SvgsBH/TBBBv9AwYFyDsH5o53BkjOnQZFWAEGn5n1B7yezQPnjvUA9X6+/A7SZQRIepUG+sJ/BnCIAQakhjEAdYd/AdRfMwRUxI0EvuJVBNdVfQRBWS8Dsh01Al5wVQcDWmEAXAGxBY4AmwaORJUGI6jbBGMsxwIRHgcGrYPlAnQWhQVCgn8GUaFpBwediweFT5kFhQazBdgihP6INmcEiiwjBYKn4QWleW8HVORFBpJ1cwa+0a8EEm/LBmU27wTQmjMHQ3F1AB8QrQVfBhUF4eDbByencwe3qYEGZrXBBGuCUQa4swkEvRKJBmU24v2VNCUF3PZlBgcS1QEkgnUCmpPdASCtnQR9bu8GY845BLmfLwUrEOkA8FYZA+ZmxwRI1nsAGC3zBdSTEQUd+f0AFfSfBdJZewb1o/UEUO6tAvjsAQjsUpcCRJZi/uRVRQfpZM0Cq/ZnBLLu5Px0r5UHQ7ZhBJyHRwT1yz0G8wvbB2Q4nQFjElcGnhYZADnRNwZKL10GDlYhBgJpkQNp5ykHYAeJBVkGuwUZeLsEcgG/Bf4+GwaVJ9sFcxD3BP76Ywa/qYMCeBVbANPAnwd7fY0CpbxhBn6KMwZFPjMH5Dw1BCIjoQV11u78GXZRB/QcvQN22gsGkHRlBZ6fAwPAwjkGMRfBA+InCQTkxAkCFwJnByNyDwWsuIUGMMrfBKoyswQiZBsBUbYzBGV6rwYwsXD8=","dtype":"float32","shape":[848]},"x_backup":{"__ndarray__":"HF8YwUYd10GM+MTBhGtLwQdbwUF8lGfAZZTjQBqphUHrI3XALah7wTAMwsE6ZrBBTYwIQLqFxUDjFH1ApBylQc3oMz3c4p5AdPXPwdZ/T8HtDRdBXeDhwGvGvcEEtAfBDyeJwY8BkMEN/mtBMiUKQTtblb7gTw5B/1+pPhCFl8GXmfq/EKjNwH6Pk0ElM3xBuJRjQRAdyD/1UkzAtBWdwby6W0GQcqo/lNhDwJ0kt78XMJDBUE5NwV3loMHotFdB87yPweozTsHLYiRBfiqBwUcBAsAIVnrBhTdxwbhsPb8fr/BBd+UGQXWamUHXSJXA8/3UPxBjmcBeLFvBRnwWwDQu20B6WqTBlV0RQFf4SEH+M3vBli9kQQTvv0FwJ+tBfJ7CQILRZUERVPJA+6oUQV/1kUATX9w/ZLfxwDezLMGNPMBALfkqQTM3AUIGsMRBvNpQwYEiD0FNSaPBnj2mwBEVsEHPzQVBTXeAwQsGBcJVB4XBZ9pxwVij4UHtaIE/ggpLQZ6xQ8Gn98rBJde+QeEnlMHxWg7BdN68wbLElMEbAjjB83Ndwf0ym0HSt+VAq7MJwPrJD8G/Kj1BN+QqwOsaocGMO+o/OsQlQZlAksHptm7ALWGKQdCPFMGMVm9BtB/HQDQvjsFZ1gXCoVVGQeZvyEFUm7XBZs7YQClqlMG7RLxB6zuLwUChz8EqPLDAdKEQwWKioMHJhYu/UOfQwHNiAUIw8IrBghT5QToAbkH6IGU/ESvoQIFb30F0psFABzWawCDKCkKCPa7BMFPpwULArkEk+ZZBI7IjQYQBc0G7FnLBc3otQcRxN8H8fC1Bzfa0QVIiCUEQLAJC6UuTwQ1JA0Fe/FfBEJ6zwXTPpECSBAXCqEQFwht8y8GQgJPBx++xQNZeFkG3GPQ/r3dNQb02/sAYxJVA8IKDQbh0YEGSLjLBQxzDQPQoBcCL5GlAhIekwXLuicFUxxpBRaTIQRs8C0DoSqDBiPKLwQyNh8EvqZBAVIy0QV4/DUDrAHtA5GAmQdjx30Eii5bBZCsQQXjTfMFFG8XBKlnaQYT/osGOoe1AuX4tv7l4QEEbGNPATCaNwV0aA0I9r1HBjTqvwYcBO0BfCTTA90bywXVuNcFYwyrAep6rQZg2mMGmwW7B966aQY+uZkDIiNBBWR7iwKn+nUBXBOpBh3maQQCKvUAxoZpBr2M3QXePLUA5W1FAo8lHwWE3rsH4WJxAVseKwACYDEHW+tTA5RqNwZiBlsGJB4TB5fSxwQ/eKME9Vs/BjeuuQddFl8EbkS7BPctEQRgAD8Afp48/ZxBVvYKTar/TfXHBppwFwv+aLsGHh9xAPQKVwdBEsMHx0aLADDbKQR2sT79xAfE/oGk6QQxJZsE+ja7B+fZhwcGOAEEjyk3AkiKAv0DhFD4Zfe1B5XO4wA3Qy0FG549A94UYQdS+lUGVastBQxknwSEH/8EWdzzB/cWVPw2SRsFZoT1BV1KZwW/1s0HtepzAIbB6QS2TEcDEqrLAL2ecwFEDQj6u3e5B/KCIwMDGEkDuhYNAJ3WrwVuR9MCaR0dBix8aQQzqNkHOdI7BNN4twZn4uEGhR7FAHXWSwZjDgUH7mkXB7mntwNbr6D/81Ii+tQgAwCBSwEBoX01BK6VSwYfenkDvLfZA67oBQbqCgUAegy4+zYXiwYCNWsAb33VBUKtcwbXtr8BQl75AbowUQQxnI8AbHBjA1NUBQeSAGb/WzQNBqCeOQYMSlcFPVdJAlhytQFDqpsGaiM7BOwSSv7FrRsBWOoVBpRI1QHqLicF+XjpAPrU6wH2+0kH5+1nB0HUPP6cWokCpBjvBB96dwBJ8tMA04jZBioeQwRvQZsHUOSLA+nh0wcG4OUHJ37FBVbm0QQjTRsHi/EVAhArSQEV0J8HVVRtBjPVqQfkQeEHzmNhBs7RPQdDpm8EYqHfA9slkQFDGeMEEl7TBYRMYwNb7sUFK2IbBy9y5QXufTsF7BJC+tFRhwdIm8sH5BxRAkl95QVucBkGkr8bBCm8jQcmujcEV5ZBB4EcmwTarkUA9/+9BmzmKQV197kEUBVZAIseHQTIQ6EHzp53BmkX+QUXhDsAIth/BMaoywZLIwEDq0orB4TyrQcWjQ0HD1SVAWAq+wVXzekCnuhZBZI/2QZms9cHqpWZAkWiMwfIkksEeGwNBO+XhQU0QLD+ewiHBLOLxPrLUrMG0EXhAsNX4wQeoBz2H82bBVwuAQCpgyUA8JkTB/yyiwTZDRkDStYnBkKmMwI4g6EGdIRTB9kG4QSjXRcGq00w/89mRwfm5IcGmjhhB3vWJQd92hME7MKfB883iwVray79o6dVBtCYEQXVJg0FIywlBXWPbP3RZhsHz839B763BQRZ15UEnK7tA3dBGwZ/S28GFGBRB/v2pwDxKyECR/OrAtIpqQTQ0oEGbPy1BBZ0JQqkCvsHQKu1BQ9O6wITwE0FoOYzAyi6Dwcj+s8GvNdZBZj0LwQ3KUcAiITFBg7aswYcUycF/0a7Bjc3AQRfsfEGdto3B2Vb9wRDggcGdd4/Bq1r/Qaf3kUEgeUc+Fu4KQQqVX0AWDOvAP60wwYRJMkGRHphB4FfEQdf3icGYWhXAt09uvhkfSMFce6hAt36QwbozEEA8ZcfB8dF0QY/6BEFvHu1BRXiOQQiWcMFX2GlB5OGNwd7irkEcsnfB8qyUwNTqdUG1FkFBpW1oQfz8y8A4OjlBGA6Hv68IEMGqgARB5B5pwZZ1kUG87rHBtd18QYAI5MAYCbLBvphOP7SrOkFCbHnBXa/yQJAgxEFB3svBao5ewdPOyEEK4oO+/TICwmGPhcH0lFzAuFadQe/DdkFXeHjBqGFPQFvlKkGnUCFAo637QTE6ID8Hy3vBWumAwU1gVsCqQBpBH5UzQK1GMsCSSQ9BhifeQBJwor+JcMPA3Lh7QSjcZ78ft9lAYrWyQTBfl0Ezk6lAZrWWQBBLQ8E7QeZA7TvJwRXH+cENdAJChMY8wE7XN8EDy2bA9dl2Qb8NjsEsR05BNdyVQT1CvkGneT5BMcjowIrkgUH2yMRAkm1/wCpMhsEPe/RBEUaewVbNPUEQIWBBJ9oDQdl31UE2fqZBOySjwJHE48B4VyDB/dTsQW8mmcHUppHBnAXFwc6Dpz8zgsXAD42+wdYX90FKxxVBezpMP4kE/cHI0vFBtT9VwUyEYsAWaODA65rkQJZLGcFFkOI/wJ/+QU5z2r/aB1zBcPOaQd2q0MHtBu9AzVNWwRL6FEH8HNdAI8AGQfbr38EPEYBBm3+UwVqidUBpCfvAlr82QEkZhcFLmYxB9go+QfXvSj+qJcHAaxplQPJ2okF3cIjBLqVtwe+OrMES/P1AWGSfQCr1t8AxZTjBYcUawSmPykHvYBTA/VKQwbs9PkFURBfAt+cXwXq+kEGvgJhAW74Wv+FRI0HqLI/AjS7EP2+RycHWllpBUAoawMJsnMEdoonBi4FUQRD2BcBcIEFBnP1HQcJZWUE2RsjBHySbQUs0C0KBunlBK9nQQcxfuUFjNtZAQWspwWWFD8GvKhxBBIdTwf0jP8EX0vzBS7mOPzeqmcHLsSpBGwaYwTShksGrfrnB+gxnQNB2YkEFHl1BsfutQcWjXj+yyptBUvK8waCp+sGRxojBQo9lwasCdsHTSAPAEu1pQKY9k8EPYJVAs9gOQT0nUsEEwPVBpSQBv1jYJsBbvI2+CELdwUM39L7rYIXBSAw3wSx6gEG349FAo5N/weagjsDSS7XAhWHowdpxxkAna5fBQQCJP5SvgsBH/TBBBv9AwYFyDsH5o53BkjOnQZFWAEGn5n1B7yezQPnjvUA9X6+/A7SZQRIepUG+sJ/BnCIAQakhjEAdYd/AdRfMwRUxI0EvuJVBNdVfQRBWS8Dsh01Al5wVQcDWmEAXAGxBY4AmwaORJUGI6jbBGMsxwIRHgcGrYPlAnQWhQVCgn8GUaFpBwediweFT5kFhQazBdgihP6INmcEiiwjBYKn4QWleW8HVORFBpJ1cwa+0a8EEm/LBmU27wTQmjMHQ3F1AB8QrQVfBhUF4eDbByencwe3qYEGZrXBBGuCUQa4swkEvRKJBmU24v2VNCUF3PZlBgcS1QEkgnUCmpPdASCtnQR9bu8GY845BLmfLwUrEOkA8FYZA+ZmxwRI1nsAGC3zBdSTEQUd+f0AFfSfBdJZewb1o/UEUO6tAvjsAQjsUpcCRJZi/uRVRQfpZM0Cq/ZnBLLu5Px0r5UHQ7ZhBJyHRwT1yz0G8wvbB2Q4nQFjElcGnhYZADnRNwZKL10GDlYhBgJpkQNp5ykHYAeJBVkGuwUZeLsEcgG/Bf4+GwaVJ9sFcxD3BP76Ywa/qYMCeBVbANPAnwd7fY0CpbxhBn6KMwZFPjMH5Dw1BCIjoQV11u78GXZRB/QcvQN22gsGkHRlBZ6fAwPAwjkGMRfBA+InCQTkxAkCFwJnByNyDwWsuIUGMMrfBKoyswQiZBsBUbYzBGV6rwYwsXD8=","dtype":"float32","shape":[848]},"y":{"__ndarray__":"2E3bwJfkRj9GUUTBK0qpwQgIMEH8BtVASmTIwMfqksF7yABB4kJuwToYgcHTZHXBUs5MQQqGhMFLi23BezboQFTFtMGl78JABr6MQI9HPcB0duBBAJUnwZgePEFleOvAXcyPQN5a0EFdaflBTTijwT/txcGCOkxBfygmwUMyy0EI1UtBRJnWwUowg8HWAphAhR4zwUsE9UE0VdfBVxLmQCWVoD/1O+xBPI0vwaVz2sAJ/PrB1LQhQVyuqUFLm8tBekX3wTVF+MBDPqpBPBJIwTYA/8CTUKvBnZnDPWYT/0AUoB1BEubOwNaFhsFfprk/XiuOwWsb0cH7XCnBsp5bwdonR0FpnbFBHrQlwYbP1UE2lKG/mCATQTXJUkHJmiHA+RmHwO2PykG5qE5AKHdUQSbbkkGRPI9B8NXkwBXQBEG8OJ9B0n0wQft648DFF4HAYMCUwBzqNEHWvSPBwzOiQR56icECOfRBTiRyQdv0wb/I+nG/8K75vkzpo0HY0/ZABew0wUX3DMG2QWHBEC7VP7UgmEBl76vAYvqAwaF+n8EfxhtBZ4uKwC4rZMGA6S5A+Q3ZwYa8ocBY7x/BU59kQXR2tUHT0ovB7KqjQGD6RMHC5WhBbpCCwZZLWT/HsQVBGonhQQDEl8EjeKTAMR4zwRLLx0CPclzB2ISHv9/RycFLe1BAxC49wLg1lUA3+unB+0QWv7Pef0CO919BOMLHQbN/kb/YPqDBKJZ/v4OOL0FzWy1BAQIuwdEL6EAU2cxBOAzQQXAjbD6l/LrAj94GQFCQrT+1x33B2YmHQIxcSkFzt11AnoHgwcJTjcHMgeDBSmY1QdHE70ENpro/a1sQQAbCo0GrieHAQFsSwJ5+IsFt+by/JCigwDlXAsFxg189aBd5wTLLIkHXHoPBCXNlwWfk977NNQFB9C+Owe+fI0ENKyJBMHlTwdAQxcEQJ+RBECnywDUnu8GOW7JBs/GMwNN8kUEIoj5AHNq/wMzMhMGtIr9B0EPsQJhf60A5/1FBjZccQXzLqUGipSDB5ID2QdDby8CM/lnBoqhGQL6MpkGcjs5BGQTcwEUWCUHXN9BB8ogywefsHz87KVpBH3iEQFEFv0HPNbPBM3ujwHnujcFCzW5AlAWHQL/K/MAov77BBG0NP2TQIUAc1xJBn7VLwQl93EGUDtrAVl5GwVmWasHmNFXByqbTwMax7kGWczRBu1zGwQfGgcEdD4HBJqfgwd0kqsGFJYzBTkPCwfr65L/+R43AE8AWwcvdn0DheyfB+92YPxV03EAWVDw/WpXZQQYjV0Fn9NzAUvk5QVxi9kDSl4I/Txu7wM64c8FpKsZBOD9HwfSSaMBgWEQ/IkqyQG7xCEHL99pB+5lpQcyrI8EJmLnAwFQKwRabCUEtY9DB7Pa9wf6/vkGFiNnA6psrwQvRXcD+qPhBVoj5wBIc68GBWDRBZMOjwSirMMGPL2TBRsPOQaHZgT4mcu9BCgPtQPIA8kD4ppXAg9oKQI1s2UCbOsxB+Vv+vmMslkFxKShBSobCwcJIZkCrK5BBZRKRQPAde8C/iIvBAVvDQIUjOEEwBKrBMUU3QUFpKUB6pktAd+GnQftCAEKLUETB7BiBwT5dqz81uk1BeYAVQULUSUGHXipBeTGNwVwehEFq8rxBXwDZQbfEVEC987zB3UadQEZrm8HZDFzBoDa2QCU50sGatzhBqPduQdKmpMETt7k/86GYwL7GmEE6x/9AucCewYH2tMFIX59BZ3heQbhcgME6hy5BuwsVwM6yHkEHpAFCL9lLQWhTFsGWh4m/r/InQfXWn0HDx4LBFDDHvpxbx8AVCL2/ituTwBynycG/yaJBlrP9wfxuX8F4587BmRGpQZJMZUGM4HLB8uWkQGpRA8Hh7oRBTQkYwd2TF8G8wMZBan4DQRknCUC9N51BKdeFwWUahcHef8zB5NdfQVsVmcHbWmTBDuRqQVfRYkDN+9JBIZO0QGaMnz50Ap9BFYGzwSLCvMCMSiHAEEhwwSJ62sBLWzBB2m2tQA7+ssG6ykzBBQakwcJb9EHwR87Awwz6PsLjrkE/1PBBJrxqwMo+NcAu9qzAx9kfQUXLo8FK2htBYAKCv2S82kHzBtFBW0KoQP3MNkGuXJNB3rwnwTxwoEEP5mJBOizYvlpsi8AB4Li/N7pjwVF+10CwdInAZqFKPSLWA0H3xPNAixnaQA241T7F/vpB1ReFwXDc70Eh2qfBZaPfQcJJ7UEFlsDBrhI4wSYIg0BQIsVB+FQIQewyr0EB1lFA6V6SQI4VEkEtuF5Bl5/GQX46hbwz5dPAQ/BgwcaizEH+5G7Bw7OdQIvO18HQriVBKdqqwcwCdsFv1rHBKUeXwCSQ9MA3cv9B+GtzQLPDJUEirw9BZZGXvy+/J0BPpxHBwJZtwXHxUcHQOIfBJxVCwdDf875t4EfBHoaZvXcwRkGfjihBknfQQZCJi0DsruG/woIovTmUhsGTh5S+gXaCQa/wl0BVdO9Apw/zwIFHkMCnGi7BCgGgwE2mZ8FlW7VBTesrwdzqbUEu37PBUBaRv0cYa8G7g/5AYNviQfPcwT9yXi5An2FEwTTyIsG4+oVBU/vGwBRzDMHFj7jBtmDvQSltqMCFPOVBo8IOwKrtgcFVyWHAMXmuQD5C0EGCcxZB/ZpSwXJdGsHLOU5BTdliwcZvR0AzdmPB1XlwwZtUxEHuoBPBWoSZQH1q6cD24VJB1pkewFeaRsHS2V1BUUytwU7d00C7DRpABHmIwaopQMG4oaU/FHHIvn8THUH3a2VBiP3XQWqMx0B5JzZBb9FowSg3R0BX1MJB+GegwAAIvEECEJ/BuAyIP9L/nMGkzdDAsno3QXJz78Dor9VB3pDjwPEnk0EvREvBZ7CGwLDgMr/7RZPAGj7rv997/0AUKqhBupJ1wUMur8Hp45tAJ+CgwbCjpcEaV2BBwK46QWRllcEQ+rZAQm18wZOHqcADVJdA6mEiQWRQfsGJQEzAOJKzwTJjN8BW49PBP7OhwSyovkF0D95AiBHrwUnblUC+K9VBVvJbwD03B0FY4e5BDmtqQbRcsj6xiB7ACNGTwNhD70GXePBAs2MyQVaW6j+xgFfB9IjYwfgCXUDNm0zArVPAwF8qAUFBIrBBXF8kQQGmd0C+u8xBJQQlwWO4s8CHK89BcKd/wdwqKcEXkkrAmRZnQOOgHb8tMkDBHSsCQCvcHEG6MphB1G9Bv8SkycGfnz7AyO1bwSOEO0HzBbXBrtoLQZk99EFoo/1BaHNxQSxrPUA01YtAYvm+Qb9AvMDlEovA87HKQUmdtkFTNJzBvWakQW3Bv8Hib5C+ZP8AQrhIXcFaUKbBAp9iQQtFNcG/AKfBn1Igwa/FP8HnPyhBTsQfwTndJUEHt0/Bo0T8wRQvA8E5vwdA9hWKwBKqg0Dtl1lAiofJwevdgUHVahjBlClkwQcDCMEQKCu/GExxQQ9dpUH+RKhBhQGBwaGzhsAnXMpAPDqLwQ3QDUHhI3HB8o99QdRxdD7Sd4/Bbg8YQZU36j/ws7HBol/GQFx/4sCKjAdBF89IQeC2NkFZwoLB8bkowfFYZcC4NtVBHXbJwM8PUUD5ND3BDD7fwN2QHsFMbIzB5BotQbFKKcG6sHu/RvxewBN8NMGnO8bB1zV7wWxHu8EDKJTBDRvrwG+RwECEn2XBQYzWQUtGXMG9Zh9B4xecwPoLeMHdkMNBOm0nwfJOncCTESHBD5GzwPVqGUFZ6nzBzdM0wWjBH0GRgqBAnXULQIKiir+tOKNBRPzYwCDZzkFKKYpA8E/mP6PLgEG6vaRAQPO8QKzxlUGpJdFASuSqQBO/+kHWXOXApW4DQK6sZcHYyzXBOj8pQJI5DUGb7yPBXJxHQRTixcAnIevBYTvUQYg380AzOItBKmbEwK9W6kHiUplBGzYXwYtDg0HVF3zB9Y1fwVx2UMBy9a3Br+9fvm0PmUCbOs1BuptVwa+YucALtnq/2A/2QbGZ6cDI34NBaJjMwPf8nsHDFvLAVlK2QILHk8GhCsHAnyi8wFhayUGAb8ZBZcSqvTFJbsDnZ1LBfUF5QPYfm0E+oS9B+z2QwRe2SkE4Y0/Bj9qXQOyibMDsgJFA+IWxPzhKxD/94MVBAE+3PzKjs8BiVJjBUJoHwXkh+EEmHNJBHqoiwQLHvMEOkrPB/c3FwHH48UHcK0RAlKpRQDp11L/YDMlBcmfAwNdRokGZarfB4DohwRgIAkJkr8VBFxQ7QZ7Go0F13pbB6jRFQK+rqT+GrqHAjkAMQCRkHsFX+D5B+Kgkwe0ooL5l/SE/2G1mQf9umMAv+KlBHJ1OwZBIQECbhLXB7l25v8RlfcGZmnzB0GmuQX0BkUCno9nA15ZnwYZJR8DtT75BdUR2QKf9k7/XLa9B7vGjQXMRg8GfdQ5BmCDdQAukoMHwjuRB3fCeQBeLekDD1ohBsTchQCRhtUAv+6lBEMQFwQRVzEE3OCbArgB2wXxr0cFw6TLB7fuPwcztZkE=","dtype":"float32","shape":[848]},"y_backup":{"__ndarray__":"2E3bwJfkRj9GUUTBK0qpwQgIMEH8BtVASmTIwMfqksF7yABB4kJuwToYgcHTZHXBUs5MQQqGhMFLi23BezboQFTFtMGl78JABr6MQI9HPcB0duBBAJUnwZgePEFleOvAXcyPQN5a0EFdaflBTTijwT/txcGCOkxBfygmwUMyy0EI1UtBRJnWwUowg8HWAphAhR4zwUsE9UE0VdfBVxLmQCWVoD/1O+xBPI0vwaVz2sAJ/PrB1LQhQVyuqUFLm8tBekX3wTVF+MBDPqpBPBJIwTYA/8CTUKvBnZnDPWYT/0AUoB1BEubOwNaFhsFfprk/XiuOwWsb0cH7XCnBsp5bwdonR0FpnbFBHrQlwYbP1UE2lKG/mCATQTXJUkHJmiHA+RmHwO2PykG5qE5AKHdUQSbbkkGRPI9B8NXkwBXQBEG8OJ9B0n0wQft648DFF4HAYMCUwBzqNEHWvSPBwzOiQR56icECOfRBTiRyQdv0wb/I+nG/8K75vkzpo0HY0/ZABew0wUX3DMG2QWHBEC7VP7UgmEBl76vAYvqAwaF+n8EfxhtBZ4uKwC4rZMGA6S5A+Q3ZwYa8ocBY7x/BU59kQXR2tUHT0ovB7KqjQGD6RMHC5WhBbpCCwZZLWT/HsQVBGonhQQDEl8EjeKTAMR4zwRLLx0CPclzB2ISHv9/RycFLe1BAxC49wLg1lUA3+unB+0QWv7Pef0CO919BOMLHQbN/kb/YPqDBKJZ/v4OOL0FzWy1BAQIuwdEL6EAU2cxBOAzQQXAjbD6l/LrAj94GQFCQrT+1x33B2YmHQIxcSkFzt11AnoHgwcJTjcHMgeDBSmY1QdHE70ENpro/a1sQQAbCo0GrieHAQFsSwJ5+IsFt+by/JCigwDlXAsFxg189aBd5wTLLIkHXHoPBCXNlwWfk977NNQFB9C+Owe+fI0ENKyJBMHlTwdAQxcEQJ+RBECnywDUnu8GOW7JBs/GMwNN8kUEIoj5AHNq/wMzMhMGtIr9B0EPsQJhf60A5/1FBjZccQXzLqUGipSDB5ID2QdDby8CM/lnBoqhGQL6MpkGcjs5BGQTcwEUWCUHXN9BB8ogywefsHz87KVpBH3iEQFEFv0HPNbPBM3ujwHnujcFCzW5AlAWHQL/K/MAov77BBG0NP2TQIUAc1xJBn7VLwQl93EGUDtrAVl5GwVmWasHmNFXByqbTwMax7kGWczRBu1zGwQfGgcEdD4HBJqfgwd0kqsGFJYzBTkPCwfr65L/+R43AE8AWwcvdn0DheyfB+92YPxV03EAWVDw/WpXZQQYjV0Fn9NzAUvk5QVxi9kDSl4I/Txu7wM64c8FpKsZBOD9HwfSSaMBgWEQ/IkqyQG7xCEHL99pB+5lpQcyrI8EJmLnAwFQKwRabCUEtY9DB7Pa9wf6/vkGFiNnA6psrwQvRXcD+qPhBVoj5wBIc68GBWDRBZMOjwSirMMGPL2TBRsPOQaHZgT4mcu9BCgPtQPIA8kD4ppXAg9oKQI1s2UCbOsxB+Vv+vmMslkFxKShBSobCwcJIZkCrK5BBZRKRQPAde8C/iIvBAVvDQIUjOEEwBKrBMUU3QUFpKUB6pktAd+GnQftCAEKLUETB7BiBwT5dqz81uk1BeYAVQULUSUGHXipBeTGNwVwehEFq8rxBXwDZQbfEVEC987zB3UadQEZrm8HZDFzBoDa2QCU50sGatzhBqPduQdKmpMETt7k/86GYwL7GmEE6x/9AucCewYH2tMFIX59BZ3heQbhcgME6hy5BuwsVwM6yHkEHpAFCL9lLQWhTFsGWh4m/r/InQfXWn0HDx4LBFDDHvpxbx8AVCL2/ituTwBynycG/yaJBlrP9wfxuX8F4587BmRGpQZJMZUGM4HLB8uWkQGpRA8Hh7oRBTQkYwd2TF8G8wMZBan4DQRknCUC9N51BKdeFwWUahcHef8zB5NdfQVsVmcHbWmTBDuRqQVfRYkDN+9JBIZO0QGaMnz50Ap9BFYGzwSLCvMCMSiHAEEhwwSJ62sBLWzBB2m2tQA7+ssG6ykzBBQakwcJb9EHwR87Awwz6PsLjrkE/1PBBJrxqwMo+NcAu9qzAx9kfQUXLo8FK2htBYAKCv2S82kHzBtFBW0KoQP3MNkGuXJNB3rwnwTxwoEEP5mJBOizYvlpsi8AB4Li/N7pjwVF+10CwdInAZqFKPSLWA0H3xPNAixnaQA241T7F/vpB1ReFwXDc70Eh2qfBZaPfQcJJ7UEFlsDBrhI4wSYIg0BQIsVB+FQIQewyr0EB1lFA6V6SQI4VEkEtuF5Bl5/GQX46hbwz5dPAQ/BgwcaizEH+5G7Bw7OdQIvO18HQriVBKdqqwcwCdsFv1rHBKUeXwCSQ9MA3cv9B+GtzQLPDJUEirw9BZZGXvy+/J0BPpxHBwJZtwXHxUcHQOIfBJxVCwdDf875t4EfBHoaZvXcwRkGfjihBknfQQZCJi0DsruG/woIovTmUhsGTh5S+gXaCQa/wl0BVdO9Apw/zwIFHkMCnGi7BCgGgwE2mZ8FlW7VBTesrwdzqbUEu37PBUBaRv0cYa8G7g/5AYNviQfPcwT9yXi5An2FEwTTyIsG4+oVBU/vGwBRzDMHFj7jBtmDvQSltqMCFPOVBo8IOwKrtgcFVyWHAMXmuQD5C0EGCcxZB/ZpSwXJdGsHLOU5BTdliwcZvR0AzdmPB1XlwwZtUxEHuoBPBWoSZQH1q6cD24VJB1pkewFeaRsHS2V1BUUytwU7d00C7DRpABHmIwaopQMG4oaU/FHHIvn8THUH3a2VBiP3XQWqMx0B5JzZBb9FowSg3R0BX1MJB+GegwAAIvEECEJ/BuAyIP9L/nMGkzdDAsno3QXJz78Dor9VB3pDjwPEnk0EvREvBZ7CGwLDgMr/7RZPAGj7rv997/0AUKqhBupJ1wUMur8Hp45tAJ+CgwbCjpcEaV2BBwK46QWRllcEQ+rZAQm18wZOHqcADVJdA6mEiQWRQfsGJQEzAOJKzwTJjN8BW49PBP7OhwSyovkF0D95AiBHrwUnblUC+K9VBVvJbwD03B0FY4e5BDmtqQbRcsj6xiB7ACNGTwNhD70GXePBAs2MyQVaW6j+xgFfB9IjYwfgCXUDNm0zArVPAwF8qAUFBIrBBXF8kQQGmd0C+u8xBJQQlwWO4s8CHK89BcKd/wdwqKcEXkkrAmRZnQOOgHb8tMkDBHSsCQCvcHEG6MphB1G9Bv8SkycGfnz7AyO1bwSOEO0HzBbXBrtoLQZk99EFoo/1BaHNxQSxrPUA01YtAYvm+Qb9AvMDlEovA87HKQUmdtkFTNJzBvWakQW3Bv8Hib5C+ZP8AQrhIXcFaUKbBAp9iQQtFNcG/AKfBn1Igwa/FP8HnPyhBTsQfwTndJUEHt0/Bo0T8wRQvA8E5vwdA9hWKwBKqg0Dtl1lAiofJwevdgUHVahjBlClkwQcDCMEQKCu/GExxQQ9dpUH+RKhBhQGBwaGzhsAnXMpAPDqLwQ3QDUHhI3HB8o99QdRxdD7Sd4/Bbg8YQZU36j/ws7HBol/GQFx/4sCKjAdBF89IQeC2NkFZwoLB8bkowfFYZcC4NtVBHXbJwM8PUUD5ND3BDD7fwN2QHsFMbIzB5BotQbFKKcG6sHu/RvxewBN8NMGnO8bB1zV7wWxHu8EDKJTBDRvrwG+RwECEn2XBQYzWQUtGXMG9Zh9B4xecwPoLeMHdkMNBOm0nwfJOncCTESHBD5GzwPVqGUFZ6nzBzdM0wWjBH0GRgqBAnXULQIKiir+tOKNBRPzYwCDZzkFKKYpA8E/mP6PLgEG6vaRAQPO8QKzxlUGpJdFASuSqQBO/+kHWXOXApW4DQK6sZcHYyzXBOj8pQJI5DUGb7yPBXJxHQRTixcAnIevBYTvUQYg380AzOItBKmbEwK9W6kHiUplBGzYXwYtDg0HVF3zB9Y1fwVx2UMBy9a3Br+9fvm0PmUCbOs1BuptVwa+YucALtnq/2A/2QbGZ6cDI34NBaJjMwPf8nsHDFvLAVlK2QILHk8GhCsHAnyi8wFhayUGAb8ZBZcSqvTFJbsDnZ1LBfUF5QPYfm0E+oS9B+z2QwRe2SkE4Y0/Bj9qXQOyibMDsgJFA+IWxPzhKxD/94MVBAE+3PzKjs8BiVJjBUJoHwXkh+EEmHNJBHqoiwQLHvMEOkrPB/c3FwHH48UHcK0RAlKpRQDp11L/YDMlBcmfAwNdRokGZarfB4DohwRgIAkJkr8VBFxQ7QZ7Go0F13pbB6jRFQK+rqT+GrqHAjkAMQCRkHsFX+D5B+Kgkwe0ooL5l/SE/2G1mQf9umMAv+KlBHJ1OwZBIQECbhLXB7l25v8RlfcGZmnzB0GmuQX0BkUCno9nA15ZnwYZJR8DtT75BdUR2QKf9k7/XLa9B7vGjQXMRg8GfdQ5BmCDdQAukoMHwjuRB3fCeQBeLekDD1ohBsTchQCRhtUAv+6lBEMQFwQRVzEE3OCbArgB2wXxr0cFw6TLB7fuPwcztZkE=","dtype":"float32","shape":[848]}},"selected":{"id":"1139"},"selection_policy":{"id":"1140"}},"id":"1084","type":"ColumnDataSource"},{"attributes":{"data_source":{"id":"1084"},"glyph":{"id":"1122"},"hover_glyph":null,"muted_glyph":null,"nonselection_glyph":{"id":"1123"},"selection_glyph":null,"view":{"id":"1125"}},"id":"1124","type":"GlyphRenderer"},{"attributes":{"bottom_units":"screen","fill_alpha":0.5,"fill_color":"lightgrey","left_units":"screen","level":"overlay","line_alpha":1.0,"line_color":"black","line_dash":[4,4],"line_width":2,"render_mode":"css","right_units":"screen","top_units":"screen"},"id":"1112","type":"BoxAnnotation"},{"attributes":{},"id":"1139","type":"Selection"},{"attributes":{"callback":null,"point_policy":"follow_mouse","tooltips":[["Title","@titles{safe}"],["labels","@labels{safe}"]]},"id":"1085","type":"HoverTool"}],"root_ids":["1087"]},"title":"Bokeh Application","version":"2.0.1"}}
        </script>
        <script type="text/javascript">
          (function() {
            var fn = function() {
              Bokeh.safely(function() {
                (function(root) {
                  function embed_document(root) {
                    
                  var docs_json = document.getElementById('1278').textContent;
                  var render_items = [{"docid":"72ff5b54-16ef-4148-b5df-f79ee92fabad","root_ids":["1087"],"roots":{"1087":"72c29b4b-5dd3-4d6c-8154-5e52b5f613f3"}}];
                  root.Bokeh.embed.embed_items(docs_json, render_items);
                
                  }
                  if (root.Bokeh !== undefined) {
                    embed_document(root);
                  } else {
                    var attempts = 0;
                    var timer = setInterval(function(root) {
                      if (root.Bokeh !== undefined) {
                        clearInterval(timer);
                        embed_document(root);
                      } else {
                        attempts++;
                        if (attempts > 100) {
                          clearInterval(timer);
                          console.log("Bokeh: ERROR: Unable to run BokehJS code because BokehJS library is missing");
                        }
                      }
                    }, 10, root)
                  }
                })(window);
              });
            };
            if (document.readyState != "loading") fn();
            else document.addEventListener("DOMContentLoaded", fn);
          })();
        </script>
    
  </body>
  
</html>